{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Requirement already satisfied: requests in /home/jack/projects/Metadata/.conda/lib/python3.11/site-packages (2.32.3)\n",
      "Requirement already satisfied: biopython in /home/jack/projects/Metadata/.conda/lib/python3.11/site-packages (1.84)\n",
      "Requirement already satisfied: charset-normalizer<4,>=2 in /home/jack/projects/Metadata/.conda/lib/python3.11/site-packages (from requests) (3.4.0)\n",
      "Requirement already satisfied: idna<4,>=2.5 in /home/jack/projects/Metadata/.conda/lib/python3.11/site-packages (from requests) (3.10)\n",
      "Requirement already satisfied: urllib3<3,>=1.21.1 in /home/jack/projects/Metadata/.conda/lib/python3.11/site-packages (from requests) (2.2.3)\n",
      "Requirement already satisfied: certifi>=2017.4.17 in /home/jack/projects/Metadata/.conda/lib/python3.11/site-packages (from requests) (2024.8.30)\n",
      "Requirement already satisfied: numpy in /home/jack/projects/Metadata/.conda/lib/python3.11/site-packages (from biopython) (1.26.4)\n"
     ]
    }
   ],
   "source": [
    "!pip install requests biopython\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio import Entrez\n",
    "import xml.etree.ElementTree as ET\n",
    "import time\n",
    "import os \n",
    "\n",
    "# Always tell NCBI who you are\n",
    "Entrez.email = \"riboseq@gmail.com\"  # Replace with your email"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_pubmed_id_from_bioproject(bioproject_id, max_retries=3, delay=1):\n",
    "    for attempt in range(max_retries):\n",
    "        try:\n",
    "            # First, use esearch to get the BioProject ID\n",
    "            handle = Entrez.esearch(db=\"bioproject\", term=bioproject_id)\n",
    "            record = Entrez.read(handle)\n",
    "            handle.close()\n",
    "\n",
    "            if record[\"IdList\"]:\n",
    "                project_id = record[\"IdList\"][0]\n",
    "                \n",
    "                # Then use elink to find related PubMed entries\n",
    "                handle = Entrez.elink(dbfrom=\"bioproject\", db=\"pubmed\", id=project_id)\n",
    "                record = Entrez.read(handle)\n",
    "                handle.close()\n",
    "\n",
    "                # Check if there are any linked PubMed IDs\n",
    "                if record[0]['LinkSetDb']:\n",
    "                    pubmed_id = record[0]['LinkSetDb'][0]['Link'][0]['Id']\n",
    "                    return pubmed_id\n",
    "            \n",
    "            return None  # No linked PubMed ID found\n",
    "\n",
    "        except Exception as e:\n",
    "            if attempt < max_retries - 1:\n",
    "                print(f\"Attempt {attempt + 1} failed. Retrying in {delay} seconds...\")\n",
    "                time.sleep(delay)\n",
    "            else:\n",
    "                print(f\"Failed to retrieve data after {max_retries} attempts.\")\n",
    "                print(f\"Error: {str(e)}\")\n",
    "                return None\n",
    "\n",
    "\n",
    "def get_bioproject_metadata(bioproject_id, max_retries=3, delay=1, save_xml=False, xml_file_path=None):\n",
    "    for attempt in range(max_retries):\n",
    "            # First, use esearch to get the BioProject ID\n",
    "        handle = Entrez.esearch(db=\"bioproject\", term=bioproject_id)\n",
    "        record = Entrez.read(handle)\n",
    "        handle.close()\n",
    "\n",
    "        if record[\"IdList\"]:\n",
    "            project_id = record[\"IdList\"][0]\n",
    "            \n",
    "            # Then use efetch to get the full BioProject record\n",
    "            handle = Entrez.efetch(db=\"bioproject\", id=project_id, retmode=\"xml\")\n",
    "            xml_content = handle.read()\n",
    "            handle.close()\n",
    "\n",
    "            # Save XML to file if requested\n",
    "            if save_xml:\n",
    "                if xml_file_path is None:\n",
    "                    xml_file_path = f\"bioproject_{bioproject_id}.xml\"\n",
    "                \n",
    "                            \n",
    "                with open(xml_file_path, 'wb') as xml_file:\n",
    "                    xml_file.write(xml_content)\n",
    "                print(f\"XML content saved to {xml_file_path}\")\n",
    "\n",
    "            root = ET.fromstring(xml_content)\n",
    "            \n",
    "            # Extract project-level metadata\n",
    "            metadata = {}\n",
    "            \n",
    "            # Project ID\n",
    "            project = root.find(\".//Project\")\n",
    "            if project is not None:\n",
    "                metadata['ProjectID'] = project.get('accession')\n",
    "            \n",
    "            # Project Type\n",
    "            project_type = root.find(\".//ProjectType\")\n",
    "            if project_type is not None:\n",
    "                metadata['ProjectType'] = project_type.find('ProjectTypeSubmission').get('ProjectDataTypeSet')\n",
    "            \n",
    "            # Project Name\n",
    "            name = root.find(\".//Project/ProjectDescr/Name\")\n",
    "            if name is not None:\n",
    "                metadata['ProjectName'] = name.text\n",
    "            \n",
    "            # Project Title\n",
    "            title = root.find(\".//Project/ProjectDescr/Title\")\n",
    "            if title is not None:\n",
    "                metadata['ProjectTitle'] = title.text\n",
    "            \n",
    "            # Project Description\n",
    "            description = root.find(\".//Project/ProjectDescr/Description\")\n",
    "            if description is not None:\n",
    "                metadata['ProjectDescription'] = description.text\n",
    "            \n",
    "            # Relevance\n",
    "            relevance = root.find(\".//Project/ProjectDescr/Relevance\")\n",
    "            if relevance is not None:\n",
    "                metadata['Relevance'] = relevance.text\n",
    "            \n",
    "            # Submitter\n",
    "            submitter = root.find(\".//Submission/Description/Organization/Name\")\n",
    "            if submitter is not None:\n",
    "                metadata['Submitter'] = submitter.text\n",
    "            \n",
    "            # Submission Date\n",
    "            submitted = root.find(\".//Submission\")\n",
    "            if submitted is not None:\n",
    "                metadata['SubmissionDate'] = submitted.get('submitted')\n",
    "            \n",
    "            return metadata\n",
    "        else:\n",
    "            print(f\"No BioProject found with ID: {bioproject_id}\")\n",
    "            return None\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "No PubMed ID found for BioProject PRJDB10544 or an error occurred\n"
     ]
    }
   ],
   "source": [
    "# Example usage\n",
    "bioproject_id = \"PRJDB10544\"  # Replace with your BioProject ID\n",
    "\n",
    "pubmed_id = get_pubmed_id_from_bioproject(bioproject_id)\n",
    "if pubmed_id:\n",
    "    print(f\"The PubMed ID for BioProject {bioproject_id} is: {pubmed_id}\")\n",
    "else:\n",
    "    print(f\"No PubMed ID found for BioProject {bioproject_id} or an error occurred\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "XML content saved to bioproject_PRJDB10544.xml\n",
      "\n",
      "BioProject Metadata:\n",
      "ProjectID: None\n",
      "ProjectType: None\n",
      "ProjectName: Arabidopsis thaliana strain:T87 cultured cells\n",
      "ProjectTitle: Ribosome profiling in Arabidopsis T87 cultured cell\n",
      "ProjectDescription: In gene expression, DNA is transcribed to mRNA, and mRNA is translated to protein. In these steps, translation is one of the important steps to determine the protein abundance. To evaluate translation efficiencies, ribosome profiling is reported in previous study. This method is based on deep sequencing of ribosome-protected mRNA fragments, and provides a \"snapshot\" of all the ribosomes abundance at a specific time point. This information provide the ribosome position or ribosome occupancy on mRNAs. Thus, we conducted ribosome profiling using 3 day after inoculation of Arabidopsis T87 cultured cell to evaluate translation efficiencies and analyze the relationships between sequence feature and translation process.\n",
      "Relevance: \n",
      "                \n",
      "Submitter: Graduate School of Science and Technology, Nara Institute of Science and Technology\n",
      "SubmissionDate: 2020-09-10\n"
     ]
    }
   ],
   "source": [
    "metadata = get_bioproject_metadata(bioproject_id, save_xml=True, xml_file_path=f\"bioproject_{bioproject_id}.xml\")\n",
    "if metadata:\n",
    "    print(\"\\nBioProject Metadata:\")\n",
    "    for key, value in metadata.items():\n",
    "        print(f\"{key}: {value}\")\n",
    "else:\n",
    "    print(f\"Failed to retrieve metadata for BioProject {bioproject_id}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "metadata": {},
   "outputs": [],
   "source": [
    "import gzip\n",
    "import urllib.request\n",
    "import io"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "metadata": {},
   "outputs": [],
   "source": [
    "def download_and_parse_geo_soft(gse_id):\n",
    "    # Construct the URL for the SOFT file\n",
    "    url = f\"https://ftp.ncbi.nlm.nih.gov/geo/series/{gse_id[:-3]}nnn/{gse_id}/soft/{gse_id}_family.soft.gz\"\n",
    "    \n",
    "    try:\n",
    "        # Download the gzipped file\n",
    "        with urllib.request.urlopen(url) as response:\n",
    "            with gzip.GzipFile(fileobj=io.BytesIO(response.read())) as unzipped:\n",
    "                content = unzipped.read().decode('utf-8')\n",
    "    \n",
    "        # Parse the SOFT file content\n",
    "        series_info = {}\n",
    "        in_series_section = False\n",
    "        \n",
    "        for line in content.split('\\n'):\n",
    "            if line.startswith('^SERIES'):\n",
    "                in_series_section = True\n",
    "                continue\n",
    "            elif line.startswith('^'):\n",
    "                in_series_section = False\n",
    "            \n",
    "            if in_series_section and line.startswith('!'):\n",
    "                key, value = line.split(' = ', 1)\n",
    "                key = key.lstrip('!')\n",
    "                series_info[key] = value.strip()\n",
    "        \n",
    "        return series_info\n",
    "    \n",
    "    except Exception as e:\n",
    "        print(f\"An error occurred: {e}\")\n",
    "        return None"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Series Information for GSE241473:\n",
      "title: Yeast eIF2A plays a minimal role in translation initiation in vivo\n",
      "geo_accession: GSE241473\n",
      "status: Public on Jan 03 2024\n",
      "submission_date: Aug 22 2023\n",
      "last_update_date: Apr 02 2024\n",
      "pubmed_id: 38266075\n",
      "summary: Earlier investigations have associated mammalian eIF2A with Met-tRNAi binding to the 40S subunit and its recruitment to specialized mRNAs in a GTP-independent manner. Additionally, eIF2A has been implicated in non-AUG start codon initiation, particularly under conditions where eIF2 function is attenuated by phosphorylation of its α-subunit during stress or starvation. However, the precise role of eIF2A in vivo translation remains unclear. Moreover, it's uncertain if the conserved ortholog in budding yeast can functionally substitute for eIF2 during stress. To address these questions, we conducted ribosome profiling on a yeast deletion mutant lacking eIF2A, alongside isogenic wild-type (WT) cells, both in the presence or absence of eIF2α phosphorylation induced by amino acid starvation.\n",
      "overall_design: This study includes a total of 16 libraries comprised of 8 RNA-Seq libraries (mRNA) and 8 ribosome footprint profiling libraries (ribo). The samples used for library preparation were sourced from two biological replicates of both the BY4742 (WT) and yeast eIF2A∆ deletion mutant F2247 (eIF2A∆). The strains were cultured at 30ºC in synthetic complete (SC) medium to log-phase (A600 = 0.5–0.6). Additionally, the WT and eIF2A∆ strains were grown in SC media lacking isoleucine and valine (SC-Ile/Val) to log-phase (A600 = 0.5–0.6). Subsequently, they were treated with the antimetabolite Sulfometuron methyl (SM) at 1 µg/mL for 25 min to induce eIF2α phosphorylation.\n",
      "type: Other\n",
      "contributor: Alan,G,Hinnebusch\n",
      "sample_id: GSM7729576\n",
      "contact_name: Swati,,Gaikwad\n",
      "contact_laboratory: Hinnebusch, Alan\n",
      "contact_department: Section on Nutrient Control Of Gene Expression\n",
      "contact_institute: National Institutes of Health\n",
      "contact_address: Building 6, Room 233\n",
      "contact_city: Bethesda\n",
      "contact_state: MD\n",
      "contact_zip/postal_code: 20892-2716\n",
      "contact_country: USA\n",
      "supplementary_file: ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE241nnn/GSE241473/suppl/GSE241473_DESeq2_eIF2Adelta_vs_WT.xlsx\n",
      "platform_id: GPL17342\n",
      "platform_taxid: 4932\n",
      "sample_taxid: 4932\n",
      "relation: BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1008247\n"
     ]
    }
   ],
   "source": [
    "gse_id = \"GSE241473\"\n",
    "\n",
    "# Get metadata and save XML\n",
    "series_metadata = download_and_parse_geo_soft(gse_id)\n",
    "\n",
    "if series_metadata:\n",
    "    print(f\"Series Information for {gse_id}:\")\n",
    "    for key, value in series_metadata.items():\n",
    "        print(f\"{key[7:]}: {value}\")\n",
    "else:\n",
    "    print(f\"Failed to retrieve series information for {gse_id}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_192057/419387699.py:1: DtypeWarning: Columns (67,76,77,80,82) have mixed types. Specify dtype option on import or set low_memory=False.\n",
      "  rdp_metadata = pd.read_csv(\"https://rdp.ucc.ie/static2/RiboSeqOrg_Metadata_v2024.10.csv\")\n"
     ]
    }
   ],
   "source": [
    "rdp_metadata = pd.read_csv(\"https://rdp.ucc.ie/static2/RiboSeqOrg_Metadata_v2024.10.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 76,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(15465, 89)"
      ]
     },
     "execution_count": 76,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rdp_metadata.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 79,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(748, 2)"
      ]
     },
     "execution_count": 79,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bioproject_gses = rdp_metadata[['BioProject', 'GEO']].dropna().drop_duplicates().reset_index(drop=True)\n",
    "bioproject_gses.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 83,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0 PRJNA1004241 GSE240563 {'Series_title': 'Regulation by the RNA-binding protein Unkempt at its effector interface  [Ribo]', 'Series_geo_accession': 'GSE240563', 'Series_status': 'Public on Mar 17 2024', 'Series_submission_date': 'Aug 10 2023', 'Series_last_update_date': 'May 01 2024', 'Series_pubmed_id': '38605040', 'Series_summary': 'How RNA-binding proteins (RBPs) convey regulatory instructions to the core effectors of RNA processing is unclear. Here we document the existence and functions of a multivalent RBP–effector interface. We show that the effector interface of a conserved RBP with an essential role in metazoan development, Unkempt, is mediated by a novel type of ‘dual-purpose’ peptide motifs that can contact two different surfaces of interacting proteins. Unexpectedly, we find that the multivalent contacts do not merely serve effector recruitment but are required for the accuracy of RNA recognition by Unkempt. Systems analyses reveal that multivalent RBP–effector contacts can repurpose the principal activity of an effector for a different function, as we demonstrate for reuse of the central eukaryotic mRNA decay factor CCR4-NOT in translational control. Our study establishes the molecular assembly and functional principles of an RBP–effector interface.', 'Series_overall_design': 'Comparisons between uninduced and WT, 3M, or dPAM2 UNK-expressing cells were performed. The log2FoldChange between the conditions were expressed as log2(Translation efficiency ratio): log2TER.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Jernej,,Murn', 'Series_sample_id': 'GSM7701775', 'Series_contact_name': 'Shiyang,,He', 'Series_contact_email': 'she050@ucr.edu', 'Series_contact_laboratory': 'Murn lab', 'Series_contact_department': 'Biochemistry', 'Series_contact_institute': 'UCR', 'Series_contact_address': '3401 Watkins Drive, Boyce Hall', 'Series_contact_city': 'Riverside', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '92521', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE240nnn/GSE240563/suppl/GSE240563_Ribo_seq.normalized.counts.tsv.gz', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1004241'}\n",
      "1 PRJNA1008247 GSE241473 {'Series_title': 'Yeast eIF2A plays a minimal role in translation initiation in vivo', 'Series_geo_accession': 'GSE241473', 'Series_status': 'Public on Jan 03 2024', 'Series_submission_date': 'Aug 22 2023', 'Series_last_update_date': 'Apr 02 2024', 'Series_pubmed_id': '38266075', 'Series_summary': \"Earlier investigations have associated mammalian eIF2A with Met-tRNAi binding to the 40S subunit and its recruitment to specialized mRNAs in a GTP-independent manner. Additionally, eIF2A has been implicated in non-AUG start codon initiation, particularly under conditions where eIF2 function is attenuated by phosphorylation of its α-subunit during stress or starvation. However, the precise role of eIF2A in vivo translation remains unclear. Moreover, it's uncertain if the conserved ortholog in budding yeast can functionally substitute for eIF2 during stress. To address these questions, we conducted ribosome profiling on a yeast deletion mutant lacking eIF2A, alongside isogenic wild-type (WT) cells, both in the presence or absence of eIF2α phosphorylation induced by amino acid starvation.\", 'Series_overall_design': 'This study includes a total of 16 libraries comprised of 8 RNA-Seq libraries (mRNA) and 8 ribosome footprint profiling libraries (ribo). The samples used for library preparation were sourced from two biological replicates of both the BY4742 (WT) and yeast eIF2A∆ deletion mutant F2247 (eIF2A∆). The strains were cultured at 30ºC in synthetic complete (SC) medium to log-phase (A600 = 0.5–0.6). Additionally, the WT and eIF2A∆ strains were grown in SC media lacking isoleucine and valine (SC-Ile/Val) to log-phase (A600 = 0.5–0.6). Subsequently, they were treated with the antimetabolite Sulfometuron methyl (SM) at 1 µg/mL for 25 min to induce eIF2α phosphorylation.', 'Series_type': 'Other', 'Series_contributor': 'Alan,G,Hinnebusch', 'Series_sample_id': 'GSM7729576', 'Series_contact_name': 'Swati,,Gaikwad', 'Series_contact_laboratory': 'Hinnebusch, Alan', 'Series_contact_department': 'Section on Nutrient Control Of Gene Expression', 'Series_contact_institute': 'National Institutes of Health', 'Series_contact_address': 'Building 6, Room 233', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '20892-2716', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE241nnn/GSE241473/suppl/GSE241473_DESeq2_eIF2Adelta_vs_WT.xlsx', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1008247'}\n",
      "2 PRJNA1017399 GSE243206 {'Series_title': 'Depletion of mRNA cap-binding protein eIF4E1 drives bradyzoite formation in Toxoplasma gondii [RIBO-seq]', 'Series_geo_accession': 'GSE243206', 'Series_status': 'Public on Nov 02 2023', 'Series_submission_date': 'Sep 14 2023', 'Series_last_update_date': 'Jun 26 2024', 'Series_pubmed_id': '38747593', 'Series_summary': 'Ingestion of Toxoplasma gondii results in life-long infection due to its ability to convert from the rapidly disseminating tachyzoite stage to the chronic, encysted bradyzoite stage.  The control of mRNA translation has been suggested to play a key role in the signaling required to trigger bradyzoite formation. In eukaryotes, translational control primarily operates at two key points during the assembly of translation initiation factors. The phosphorylation of eIF2 affects mRNA start codon recognition and promotes differentiation in a variety of parasites. Modulating eIF4F function is the second pan-eukaryote regulatory point but remains unexplored in Toxoplasma. Here, we uncover the role that eIF4F-centric regulation plays in regulating bradyzoite formation by targeting the cap-binding subunit, eIF4E1. We discover that eIF4E1 coordinates two eIF4F complexes and binds the 5’-end of all mRNAs transcribed in tachyzoites, many of which show evidence of stemming from heterogenous transcriptional start sites. Together, this indicates that eIF4E1 is the predominant cap-binding protein in Toxoplasma tachyzoites. We also demonstrate that eIF4E1 knockdown or its chemical inhibition triggers the efficient formation of bradyzoites in the absence of other stresses and that stress-induced bradyzoites reduce their eIF4E1 expression. This study unearths a role for eIF4F-centric translational control in controlling Toxoplasma differentiation, suggesting that control of cap-dependent translation regulates the process of bradyzoite formation.', 'Series_overall_design': 'expression changes at the transcriptional and translational levels upon IAA-induced eIF4E1 depletion in Toxoplasma gondii tachyzoites', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'William,J,Sullivan Jr.', 'Series_sample_id': 'GSM7780706', 'Series_contact_name': 'Michael,James,Holmes', 'Series_contact_email': 'holmesmi@iu.edu', 'Series_contact_department': 'Pharmacology & Toxicology', 'Series_contact_institute': 'Indiana University School of Medicine', 'Series_contact_address': '635 Barnhill Drive', 'Series_contact_city': 'Indianapolis', 'Series_contact_state': 'IN', 'Series_contact_zip/postal_code': '46202', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE243nnn/GSE243206/suppl/GSE243206_RAW.tar', 'Series_platform_id': 'GPL23466', 'Series_platform_taxid': '5811', 'Series_sample_taxid': '5811', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1017399'}\n",
      "3 PRJNA1022492 GSE244378 {'Series_title': 'Combined Dietary and Pharmacological Depletion of Polyamines Targets Oncogenic Translation in Neuroblastoma', 'Series_geo_accession': 'GSE244378', 'Series_status': 'Public on Feb 09 2024', 'Series_submission_date': 'Sep 29 2023', 'Series_last_update_date': 'Feb 09 2024', 'Series_pubmed_id': '38260457', 'Series_summary': 'Cancer growth is fueled by nutrients obtained from circulation and local biosynthesis. Isolating and exploiting tumoral nutrient dependencies to potentiate current anti-cancer therapies is undergoing active research. We used unbiased metabolomics on patient samples and identified reprogramming of the arginine-proline-glutamine axis in MYCN amplified neuroblastoma. Tumoral acquisition of these non-essential amnio acids, as revealed by stable isotope tracing, was primarily by import, while extra-tumoral deamination of arginine indirectly feeds tumor ornithine via circulation. Dietary depletion of proline and arginine reduced systemic ornithine, the direct precursor of polyamine biosynthesis, and in combination with the clinically approved polyamine biosynthesis inhibitor, difluoromethylornithine, enhanced their depletion to improve survival in a MYCN-driven neuroblastoma mouse model. Mechanistically, ribosome profiling indicated specific translation defects, with ribosomal pausing at adenine-ending codons, to cause reprogramed protein biosynthesis affecting cell cycle and inducing neuronal differentiation. This work provides proof of concept for combined small molecule and nutrient depletion therapy for simultaneous targeting of translation in cancers dependent on external nutrient uptake.', 'Series_overall_design': 'Assessment of translation of tumors through ribosome profiling and transcriptomics. Each sample are taken from tumor found from TH-MYCN mice under pharmacological and dietary intervention. Each sample represents a different mouse. Control samples are the mice with a defined control diet (CD) and no pharmacological treatment.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Raphael,J,Morscher', 'Series_sample_id': 'GSM7814515', 'Series_contact_name': 'Sarah,,Cherkaoui', 'Series_contact_email': 'sarah.cherkaoui@kispi.uzh.ch', 'Series_contact_phone': '+41789601707', 'Series_contact_laboratory': 'Morscher lab', 'Series_contact_department': 'Oncology', 'Series_contact_institute': \"University Children's Hospital Zurich\", 'Series_contact_address': 'Steinwiesstrasse 75', 'Series_contact_city': 'Zurich', 'Series_contact_zip/postal_code': '8032', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE244nnn/GSE244378/suppl/GSE244378_RPKMsRiboRNASeq.xlsx', 'Series_platform_id': 'GPL24247', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1022492'}\n",
      "4 PRJNA1024513 GSE244763 {'Series_title': 'FUST1 is a heat-responsive switch for global translation in plants', 'Series_geo_accession': 'GSE244763', 'Series_status': 'Public on Oct 10 2023', 'Series_submission_date': 'Oct 05 2023', 'Series_last_update_date': 'Oct 10 2023', 'Series_summary': 'Adequate reprogramming of global translation under stress is of critical importance to cell survival in eukaryotes. The phosphorylation of eukaryotic initiation factor-2α is a major pathway for stress-induced translational arrest in animals. Here we report that eIF2α phosphorylation is not induced by heat in Arabidopsis. Instead, we identify an uncharacterized protein, FUST1, that can directly sense heat and initiate translational shutdown in Arabidopsis. FUST1 exhibits heat-dependent condensation both in vivo and in vitro, which is mainly driven by its prion-like domain (PrLD). Molecular dynamic simulation reveals that PrLD undergoes conformational rearrangements and engages more inter-amino acid interactions as temperature increases. Mutations that block this conformational change also diminish FUST1 condensation in vitro and in vivo and impair heat tolerance. FUST1 condensates preferentially partition mRNAs of greater length via electrostatic interactions, recruit translation repressors and RNA decapping and deadenylation factors. Importantly, disruption of FUST1 condensation dramatically compromises translational arrest under heat. As a result, FUST1 condensation precedes and is necessary for the assembly of heat stress granules. These findings thus uncover FUST1 as a molecular switch for translation under heat stress and shed light on engineering of heat-adaptable crops.', 'Series_overall_design': 'Ten-day-old Arabidopsis seedlings with genotype Col-0 and fust1 were either treated with physiological temperature (22 ℃), or subjected to heatshock (37 ℃). For each genotype-temperature combination, 2 biological replicates were profiled with RNA-seq and Ribo-seq.', 'Series_type': 'Other', 'Series_contributor': 'Changxuan,,Li', 'Series_sample_id': 'GSM7829234', 'Series_contact_name': 'Xiaofeng,,Fang', 'Series_contact_email': 'xffang@mail.tsinghua.edu.cn', 'Series_contact_institute': 'Tsinghua University', 'Series_contact_address': 'Hai dian', 'Series_contact_city': 'Beijing', 'Series_contact_zip/postal_code': '100084', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE244nnn/GSE244763/suppl/GSE244763_Ribo-seq-counts.txt.gz', 'Series_platform_id': 'GPL26208', 'Series_platform_taxid': '3702', 'Series_sample_taxid': '3702', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1024513'}\n",
      "5 PRJNA1027912 GSE245380 {'Series_title': 'KDM5-mediated transcriptional activation of ribosomal protein genes alters translation efficiency to regulate mitochondrial metabolism.', 'Series_geo_accession': 'GSE245380', 'Series_status': 'Public on Nov 08 2023', 'Series_submission_date': 'Oct 13 2023', 'Series_last_update_date': 'Jul 31 2024', 'Series_pubmed_id': '38597673', 'Series_summary': 'Genes encoding the KDM5 family of transcriptional regulators are disrupted in individuals with intellectual disability (ID). To understand the link between KDM5 and ID, we generated five Drosophila strains harboring missense alleles analogous to those observed in patients. These alleles showed effects on neuroanatomical development, cognition, and other behaviors, in addition to a transcriptional signature that included the downregulation of many ribosomal protein genes. A similar transcriptional profile was observed in KDM5C knockout human glutamatergic neurons derived from induced pluripotent stem cells (iPSCs), suggesting a conserved role for KDM5 proteins in regulating ribosomal protein genes. Loss of ribosomal protein gene expression resulted in changes to neuronal ribosome composition. Moreover, we find that the translation efficiency of mRNAs required for mitochondrial metabolism was particularly affected upon reduction KDM5 in Drosophila neurons. Altered mitochondrial activity was confirmed through metabolomic studies that revealed decreased citric acid cycle activity. KDM5 therefore plays a key role in maintaining mitochondrial function that, when altered, could contribute to cognitive and behavioral phenotypes.', 'Series_overall_design': 'To understand how patient Claes-Jensen Syndrome (CJS) associated variants in KDM5C modeled in Drosophila affect transriptional programs in neurons, we created 5 CJS variant strains as well as a synthetic demethylase-dead strain and performed paired-end RNA-Sequencing. All variants were compared to WT KDM5. We dicovered that ribosome protein genes were downregulted acorss all ID variants. To understand how loss of KDM5 function affected translation efficiency, we performed Ribo-Seq on control and kdm5-knockdown neurons.', 'Series_type': 'Other', 'Series_contributor': 'Julie,,Secombe', 'Series_sample_id': 'GSM7841616', 'Series_contact_name': 'Matanel,,Yheskel', 'Series_contact_email': 'matanel.yheskel@einsteinmed.edu', 'Series_contact_laboratory': 'Julie Secombe', 'Series_contact_department': 'Genetics', 'Series_contact_institute': 'Albert Einstein College of Medicine', 'Series_contact_address': '1300 Morris Park', 'Series_contact_city': 'Bronx', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '10461', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE245nnn/GSE245380/suppl/GSE245380_counts.csv.gz', 'Series_platform_id': 'GPL17275', 'Series_platform_taxid': '7227', 'Series_sample_taxid': '7227', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1027912'}\n",
      "6 PRJNA1031631 GSE246139 {'Series_title': 'Integrated multiplexed assays of variant effect reveal determinants of catechol-O-methyltransferase gene expression', 'Series_geo_accession': 'GSE246139', 'Series_status': 'Public on Nov 06 2023', 'Series_submission_date': 'Oct 24 2023', 'Series_last_update_date': 'Jun 12 2024', 'Series_pubmed_id': '38355921', 'Series_summary': 'Multiplexed assays of variant effect are powerful methods to profile the consequences of rare variants on gene expression and organismal fitness. Yet, few studies have integrated several multiplexed assays to map variant effects on gene expression in coding sequences. Here, we pioneered a multiplexed assay based on polysome profiling to measure variant effects on translation at scale, uncovering single-nucleotide variants that increase and decrease ribosome load. By combining high-throughput ribosome load data with multiplexed mRNA and protein abundance readouts, we mapped the cis-regulatory landscape of thousands of catechol-O-methyltransferase (COMT) variants from RNA to protein and found numerous coding variants that alter COMT expression. Finally, we trained machine learning models to map signatures of variant effects on COMT gene expression and uncovered both directional and divergent impacts across expression layers. Our analyses reveal expression phenotypes for thousands of variants in COMT and highlight variant effects on both single and multiple layers of expression. Our findings prompt future studies that integrate several multiplexed assays for the readout of gene expression', 'Series_overall_design': 'Amplicon targeted library preparation was used to assay variant abundance of thousands of COMT variants in gDNA, total RNA, and fractionated polysomal RNA by Next-Generation Sequencing', 'Series_type': 'Other', 'Series_contributor': 'Can,,Cenik', 'Series_sample_id': 'GSM7857099', 'Series_contact_name': 'Can,,Cenik', 'Series_contact_email': 'ccenik@austin.utexas.edu', 'Series_contact_phone': '512-232-2874', 'Series_contact_laboratory': 'Cenik', 'Series_contact_department': 'Molecular Biosciences', 'Series_contact_institute': 'The University of Texas at Austin', 'Series_contact_address': 'Moffett Molecular Biology Building 2.122 2500 Speedway', 'Series_contact_city': 'Austin', 'Series_contact_state': 'TX', 'Series_contact_zip/postal_code': '78712', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE246nnn/GSE246139/suppl/GSE246139_RAW.tar', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1031631'}\n",
      "7 PRJNA1031690 GSE246163 {'Series_title': 'The Ribosome Assembly Factor Reh1 is Released from the Polypeptide Exit Tunnel in the Pioneer Round of Translation', 'Series_geo_accession': 'GSE246163', 'Series_status': 'Public on Jan 24 2024', 'Series_submission_date': 'Oct 24 2023', 'Series_last_update_date': 'Oct 14 2024', 'Series_summary': 'In S. cerevisiae, the ribosome assembly factor Reh1 binds to pre-60S subunits at a late stage during their cytoplasmic maturation. Unlike canonical assembly factors, which associate exclusively with pre-60S subunits, we observed that Reh1 sediments with polysomes in addition to free 60S subunits. We therefore investigated the intriguing possibility that Reh1 remains associated with 60S subunits after the release of the anti-association factor Tif6 and after subunit joining. Here, we show that Reh1-bound nascent 60S subunits associate with 40S subunits to form actively translating ribosomes.', 'Series_overall_design': 'Selective ribosome profiling of  Reh1-bound ribosomes in S.cerevisiae.', 'Series_type': 'Other', 'Series_contributor': 'Arlen,,Johnson', 'Series_sample_id': 'GSM8566364', 'Series_contact_name': 'Can,,Cenik', 'Series_contact_email': 'ccenik@austin.utexas.edu', 'Series_contact_phone': '512-232-2874', 'Series_contact_laboratory': 'Cenik', 'Series_contact_department': 'Molecular Biosciences', 'Series_contact_institute': 'The University of Texas at Austin', 'Series_contact_address': 'Moffett Molecular Biology Building 2.122 2500 Speedway', 'Series_contact_city': 'Austin', 'Series_contact_state': 'TX', 'Series_contact_zip/postal_code': '78712', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE246nnn/GSE246163/suppl/GSE246163_Reh1_60.ribo.h5', 'Series_platform_id': 'GPL27812', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1031690'}\n",
      "8 PRJNA1045657 GSE248724 {'Series_title': 'Impact of eIF2α phosphorylation on the translational landscape of mouse embryonic stem cells [Ribo-Seq]', 'Series_geo_accession': 'GSE248724', 'Series_status': 'Public on Dec 29 2023', 'Series_submission_date': 'Nov 27 2023', 'Series_last_update_date': 'Apr 04 2024', 'Series_pubmed_id': '38159280', 'Series_summary': 'The integrated stress response (ISR) is critical for cell survival under stress. In response to diverse environmental cues, eIF2αbecomes phosphorylated, engendering a dramatic change in mRNA translation. The activation of ISR plays a pivotal role in the early embryogenesis but the eIF2-dependent translational landscape in pluripotent embryonic stem cells (ESC) is largely unexplored. We employ a multi-omics approachconsisting of ribosome profiling, proteomics, and metabolomics inwild-type(eIF2α+/+)andphosphorylation-deficient mutant eIF2α(eIF2αA/A)mouse ESCs (mESCs) to investigate phosphorylated (p)-eIF2α-dependent translational control of naïve pluripotency. We show a transient increase in p-eIF2α in the naïve epiblast layer of E4.5 embryos. Absence of eIF2α phosphorylation engenders an exit from naive pluripotency following 2i (two chemical inhibitors of MEK1/2 and GSK3α/β) withdrawal. p-eIF2α controls translation of mRNAs encoding proteins which govern pluripotency, chromatin organization, and glutathione synthesis.Thus, p-eIF2αacts as a key regulator of the naïve pluripotency gene regulatory network.', 'Series_overall_design': 'To delineate the mechanisms by which phosphorylation of eIF2α controls the naïve state of pluripotency, we performed ribosome profiling (Ribo-Seq) and RNA-Seq  on eIF2αA/A (KI) and eIF2α+/+ (WT) mouse embryonic stem cells (mESCs) grown in LIF/2i and LIF conditions', 'Series_type': 'Other', 'Series_contributor': 'Nahum,,Sonenberg', 'Series_sample_id': 'GSM7919151', 'Series_contact_name': 'Mehdi,,Amiri', 'Series_contact_email': 'm.amiri.biology@gmail.com', 'Series_contact_department': 'Biochemistry', 'Series_contact_institute': 'McGill University', 'Series_contact_address': '1160 Pine Avenue West', 'Series_contact_city': 'Montreal', 'Series_contact_state': 'Quebec', 'Series_contact_zip/postal_code': 'H3A 1A3', 'Series_contact_country': 'Canada', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE248nnn/GSE248724/suppl/GSE248724_raw_read_count_riboseq.csv.gz', 'Series_platform_id': 'GPL24247', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1045657'}\n",
      "9 PRJNA1046391 GSE248888 {'Series_title': 'IGHMBP2 deletion suppresses translation and activates the integrated stress response [Ribo-Seq]', 'Series_geo_accession': 'GSE248888', 'Series_status': 'Public on Dec 19 2023', 'Series_submission_date': 'Nov 29 2023', 'Series_last_update_date': 'Jun 06 2024', 'Series_pubmed_id': '38803225', 'Series_summary': 'IGHMBP2 is a non-essential, superfamily 1 DNA/RNA helicase with incompletely understood biological function. Mutagenesis in IGHMBP2 is found in patients with rare neuromuscular diseases SMARD1 and CMT2S. IGHMBP2 is implicated in translational and transcriptional regulation via biochemical association with ribosomal proteins, pre-rRNA processing factors, and tRNA-related species. To uncover the cellular consequences of perturbing IGHMBP2, we generated full and partial IGHMBP2 deletion K562 cell lines. We performed Ribo-seq and RNA-seq and identified diverse gene expression changes due to IGHMBP2 deletion, including ATF4 upregulation.', 'Series_overall_design': 'We performed Ribo-seq and RNA-seq using two replicates per samples including K562 CRISPRi parental cells and K562 CRISPRi monoclonal cell lines with partial \"HET\" (two separate clones) versus full deletion \"KO\" (two separate clones) of IGHMBP2. We performed differential expression analysis among \"HET\" and \"KO\" genotypes relative to parental cell lines as the control.', 'Series_type': 'Other', 'Series_contributor': 'Stephen,,Floor', 'Series_sample_id': 'GSM7921733', 'Series_contact_name': 'Jesslyn,,Park', 'Series_contact_email': 'jesslyn.park@ucsf.edu, stephen.floor@ucsf.edu', 'Series_contact_laboratory': 'Stephen Floor', 'Series_contact_department': 'Cell and Tissue Biology', 'Series_contact_institute': 'University of California, San Francisco', 'Series_contact_address': '513 Parnassus Ave', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94143', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE248nnn/GSE248888/suppl/GSE248888_RAW.tar', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1046391'}\n",
      "10 PRJNA1047529 GSE249133 {'Series_title': 'A novel oncomicropeptide encoded by DSP-AS1 exhibits proneoplastic properties in lung adenocarcinoma by facilitating YTHDF1-induced translation initiation through mediation of mRNA circularization', 'Series_geo_accession': 'GSE249133', 'Series_status': 'Public on Dec 04 2023', 'Series_submission_date': 'Dec 01 2023', 'Series_last_update_date': 'Dec 05 2023', 'Series_summary': 'Functional micropeptides can hide inside RNA previously annotated as non-coding, their roles in the tumorigenesis of lung cancer remain largely unknown. Here, combining Ribo-seq, mass spectrometry and bioinformatics, we characterize a 46 amino-acid length oncogenic micropeptide encoded by lncRNA DSP-AS1, that we named Desmoplakin Associated MicroPeptide (DAMP), in lung adenocarcinoma (LUAD). DAMP is aberrantly overexpressed in LUAD and confers an unfavorable prognosis, its pro-neoplastic properties with respect to augmented proliferation, survival and invasiveness were validated in vitro and in vivo. DAMP colocalizes with translation machinery and induces translational reprogramming both by elevating global protein synthesis and by selectively upregulating the translation of mRNA group encoding oncogenic factors in LUAD. Mechanistically, DAMP simultaneously binds cap-binding subunit eIF4G and YTHDF1, a m6A reader that could fuel translation initiation by recruiting eIF3 to methylated mRNA. The above interactions lead to the formation of a “closed loop” between YTHDF1 and eIF4G bridged by DAMP, which makes YTHDF1 to be spatially in proximity with translation initiation complex and successfully loading of ribosomal 40S subunit onto target mRNAs. Targeting DAMP exhibited significant anti-cancer effects in vitro and in vivo. Our findings not only unravel the oncogenic role of a previously unrecognized micropeptide through orchestrating mRNA translation but also provide a strong rationale in the design of anti-cancer therapy targeting translation machinery in LUAD.', 'Series_overall_design': 'To further clarify the mRNA regulated by novel peptide DAMP and investigate their translation dynamics, LUAD cell line H1975 with or without DAMP knockout were subjected to ribo-seq. Coupled mRNA-seq was carried out to control for the difference in ribosome-protected fragments due to differences in mRNA abundances resulting from altered RNA transcription.', 'Series_type': 'Other', 'Series_contributor': 'Yongxin,,Zhou', 'Series_sample_id': 'GSM7927220', 'Series_contact_name': 'zhenchuan,,liu', 'Series_contact_email': 'liuzhenchuan1991@hotmail.com', 'Series_contact_department': 'Department of Thoracic Surgery', 'Series_contact_institute': 'Shanghai Tongji Hospital Affiliated with Tongji University', 'Series_contact_address': 'No. 389, Xincun Road', 'Series_contact_city': 'Shanghai', 'Series_contact_zip/postal_code': '200065', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE249nnn/GSE249133/suppl/GSE249133_RAW.tar', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1047529'}\n",
      "11 PRJNA1049048 GSE249448 {'Series_title': 'Translation in Bacillus subtilis is spatially and temporally coordinated during sporulation [Ribo-seq]', 'Series_geo_accession': 'GSE249448', 'Series_status': 'Public on Dec 06 2023', 'Series_submission_date': 'Dec 05 2023', 'Series_last_update_date': 'Dec 06 2023', 'Series_summary': 'Translational control during the intricate process of sporulation in Bacillus subtilis as a response to nutrient limitation is still underexplored. Here, we employed a comprehensive approach including RNA-seq, ribosome profiling and fluorescence microscopy to dissect the translational landscape of B. subtilis during sporulation. We identified two events of translation silencing and described the spatiotemporal changes in the subcellular location of translational machinery during sporulation. Using a triple knock-out strain (3KO) of zinc-independents paralogs of three zinc-dependent ribosomal proteins L31, L33 and S14, we investigated the potential regulatory role of ribosome during sporulation. The 3KO strain exhibited delayed sporulation, reduced germination efficiency, and dysregulated translation including expression of key metabolic and sporulation-related genes as well as disruptions in translation silencing, particularly in late sporulation.', 'Series_overall_design': 'RNA was isolated from sporulating Bacillus subtilis WT and 3KO (delta RpmEB, RpmGC, RpsNB) strains at eight timepoints. Cultures were treated with chloramphenicol, and mRNA and ribosomal footprints were purified. rRNA depletion was performed, and cDNA libraries were prepared using NEBNext Multiplex Small RNA Library Prep Set for Illumina. Sequencing was carried out on an Illumina NextSeq500, generating single-end 50bp reads.', 'Series_type': 'Other', 'Series_contributor': 'Agata,L,Starosta', 'Series_sample_id': 'GSM7947741', 'Series_contact_name': 'Agata,L.,Starosta', 'Series_contact_email': 'agata.starosta@ibb.waw.pl', 'Series_contact_laboratory': 'Laboratory of Translatomics', 'Series_contact_institute': 'Institute of Biochemistry and Biophysics Polish Academy of Sciences', 'Series_contact_address': 'Pawińskiego 5a', 'Series_contact_city': 'Warsaw', 'Series_contact_zip/postal_code': '02-106', 'Series_contact_country': 'Poland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE249nnn/GSE249448/suppl/GSE249448_countsTPM_CDSandUTR_Ribo.xlsx', 'Series_platform_id': 'GPL24109', 'Series_platform_taxid': '1423', 'Series_sample_taxid': '1423', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1049048'}\n",
      "12 PRJNA1051101 GSE249895 {'Series_title': 'Changes in transcriptome and translatome upon YB-1 or YB-3 expression in double knockout HEK293T ΔYB-1ΔYB-3 cells', 'Series_geo_accession': 'GSE249895', 'Series_status': 'Public on Jan 18 2024', 'Series_submission_date': 'Dec 11 2023', 'Series_last_update_date': 'Feb 15 2024', 'Series_pubmed_id': '38339016', 'Series_summary': 'In this study, we generated transcriptome profiles from HEK293T,  HEK293T ΔYB-1ΔYB-3, and HEK293T ΔYB-1ΔYB-3 with the restored expression of YB-1 and/or YB-3. The data are used to explore YB-1 and YB-3 influence on transcriptome and translatome and their functional interchangeability in transcription and translation.', 'Series_overall_design': 'RNA-Seq and Ribo-Seq data obtained from HEK293T, HEK293T ΔYB-1ΔYB-3, and HEK293T ΔYB-1ΔYB-3 with restored YB-1 and/or YB-3 expression in normal conditions.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Andrey,I,Buyan', 'Series_sample_id': 'GSM7966626', 'Series_contact_name': 'Ivan,V,Kulakovskiy', 'Series_contact_institute': 'Institute of Protein Research', 'Series_contact_address': 'Institutskaya 4', 'Series_contact_city': 'Pushchino', 'Series_contact_zip/postal_code': '142290', 'Series_contact_country': 'Russia', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE249nnn/GSE249895/suppl/GSE249895_readcounts.tsv.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1051101'}\n",
      "13 PRJNA1051495 GSE249958 {'Series_title': 'Nucleophosmin sustains WNT-driven cell proliferation and tumor initiation in vivo', 'Series_geo_accession': 'GSE249958', 'Series_status': 'Public on Feb 15 2024', 'Series_submission_date': 'Dec 12 2023', 'Series_last_update_date': 'Feb 15 2024', 'Series_summary': 'Nucleophosmin (NPM1) is a nucleolar protein frequently overexpressed in many solid tumours with little known about its functional role to support carcinogenesis. Here, we identify Npm1 as a commonly upregulated gene following Apc loss in highly proliferative tissues, and high expression correlates with increased proliferation in CRC patients. We show that NPM1 is dispensable for adult epithelial tissue homeostasis, however, it has a profound effect on WNT-driven intestinal and liver tumourigenesis, leading to reduced proliferation and extension of survival. Mechanistically, NPM1 depletion triggers integrated stress response (ISR) and p53 pathway activation. P53 deletion, or inhibition of the ISR effects, rescue proliferation following NPM1 loss. Overall, our results demonstrate that NPM1 supports WNT-driven cell proliferation and tumour initiation by attenuating p53 pathway activation or induction of ISR. Being dispensable for homeostasis, NPM1 presents as a promising target for cancer therapy in WNT-driven tumours with functional p53, including difficult to treat KRAS-mutant CRC and hepatic cancers.', 'Series_overall_design': 'To investigate the role of Nucleophosmin (Npm1) on colorectal cancer, we conditionally deleted it in an APC null/KRAS mutant cancer model. Epithelial extracts three days post-KO were harvested from mice. Ribo-seq and total cytoplasmic RNA sequencing were performed to address the effects on translation as well as transcript levels. The experiment was performed in four independent mice. Replicates are indicated by _1/_2/_3/_4. CRUK Scotland Institute Mouse IDs are indicated.', 'Series_type': 'Other', 'Series_contributor': 'Martin,,Bushell', 'Series_sample_id': 'GSM7968161', 'Series_contact_name': 'Chiara,,Giacomelli', 'Series_contact_institute': 'CRUK Scotland Institute', 'Series_contact_address': 'Garscube Estate, Switchback Road', 'Series_contact_city': 'Glasgow City', 'Series_contact_state': 'Scotland', 'Series_contact_zip/postal_code': 'G61 1BD', 'Series_contact_country': 'United Kingdom', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE249nnn/GSE249958/suppl/GSE249958_Totals_APC-KRAS_KO_batchCorrected_normalised_counts.csv.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1051495'}\n",
      "14 PRJNA1057253 GSE252063 {'Series_title': 'Modeling neurodevelopmental disorder-associated human AGO1 mutations in C. elegans Argonaute alg-1. [ribosome profiling]', 'Series_geo_accession': 'GSE252063', 'Series_status': 'Public on Dec 30 2023', 'Series_submission_date': 'Dec 26 2023', 'Series_last_update_date': 'Mar 30 2024', 'Series_pubmed_id': '38412125', 'Series_web_link': 'https://www.pnas.org/doi/10.1073/pnas.2308255121?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed', 'Series_summary': 'MicroRNAs (miRNA) associate with Argonaute proteins and negatively regulate gene expression by base pairing with complementary sequences in the 3’ UTRs of target genes. De novo coding variants in the human Argonaute gene AGO1 were reported to cause neurodevelopmental disorder (NDD) with intellectual disability (ID). Most of the altered amino acids are conserved between the miRNA associated Argonautes in H. sapiens and C. elegans, suggesting that the human AGO1 mutations could disrupt evolutionarily conserved functions in miRNA biogenesis or target repression. We genetically modeled four human AGO1 mutations in C. elegans by introducing identical mutations into the C. elegans AGO1 homolog, ALG-1. These alg-1 NDD mutations caused phenotypes in C. elegans indicative of disrupted miRNA processing, miRISC formation, and/or target repression. We show that the alg-1 NDD mutations are antimorphic as they cause developmental and molecular phenotypes stronger than those exhibited by the alg-1 null mutants, likely by sequestrating functional miRNA silencing complex (miRISC) components into non-functional complexes that fail to confer robust gene repression. The alg-1 NDD mutations cause allele-specific disruptions in mature miRNA profiles, both in overall abundances and ALG-1 NDD association, accompanied by perturbation of downstream gene expression, including altered translational efficiency and/or mRNA abundance. The perturbed genes include those with human orthologs whose dysfunction is associated with NDD. These cross-clade genetic studies illuminate fundamental Argonaute functions and provide insights into the conservation of miRNA-mediated post-transcriptional regulatory mechanisms.', 'Series_overall_design': 'To profile the translatome and transcriptome of the alg-1 NDD mutants, we performed ribosome profiling (Ribo-seq) and RNAseq of the four alg-1 NDD mutants, the alg-1 null mutants, and WT C. elegans', 'Series_type': 'Other', 'Series_contributor': 'Anna,,Zinovyeva', 'Series_sample_id': 'GSM7993214', 'Series_contact_name': 'Ye,,Duan', 'Series_contact_email': 'ye.duan@umassmed.edu', 'Series_contact_phone': '6178176037', 'Series_contact_laboratory': 'Victor Ambros', 'Series_contact_institute': 'Umass Medical School', 'Series_contact_address': '373 Plantation Street, Biotech Two, Suite 306', 'Series_contact_city': 'Worcester', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '01605', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE252nnn/GSE252063/suppl/GSE252063_alg-1.ribosome.profiling.raw.reads.csv.gz', 'Series_platform_id': 'GPL19757', 'Series_platform_taxid': '6239', 'Series_sample_taxid': '6239', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1057253'}\n",
      "15 PRJNA1062064 GSE252692 {'Series_title': 'The transcriptional and translational landscape of HCoV-OC43 infection', 'Series_geo_accession': 'GSE252692', 'Series_status': 'Public on Jan 17 2024', 'Series_submission_date': 'Jan 07 2024', 'Series_last_update_date': 'Jan 18 2024', 'Series_summary': 'The coronavirus HCoV-OC43 circulates continuously in the human population and is a frequent cause of the common cold. Here, we generated a high-resolution atlas of the transcriptional and translational landscape of OC43 at various timepoints following infection of human lung fibroblasts. Using ribosome profiling, we quantified the relative expression of the canonical open reading frames (ORFs) and identified several unannotated ORFs, including short upstream ORFs and a putative ORF nested inside the M gene. In parallel, we analyzed the cellular response to infection. Endoplasmic reticulum (ER) stress response genes are transcriptionally and translationally induced, but conventional antiviral genes remain mostly suppressed. At the same time, we observed widespread aberrant translation across cellular transcripts, including over 3′UTRs and of noncoding transcripts normally targeted by the nonsense mediated decay pathway. Taken together, our work provides a genomic resource for further study of OC43 and the cellular response to infection.', 'Series_overall_design': 'Human lung fibroblast (MRC5) cells were infected with the coronavirus OC43, and samples were collected for transcriptomic (RNAseq) and translatomic (Riboseq) analysis at various timepoints following infection.', 'Series_type': 'Other', 'Series_contributor': 'Stefan,,Bresson', 'Series_sample_id': 'GSM8005301', 'Series_contact_name': 'Stefan,,Bresson', 'Series_contact_email': 'stefan.bresson@gmail.com', 'Series_contact_laboratory': 'Tollervey Lab', 'Series_contact_department': 'Wellcome Trust Centre for Cell Biology', 'Series_contact_institute': 'University of Edinburgh', 'Series_contact_address': 'Max Born Crescent, Swann 5.1', 'Series_contact_city': 'Edinburgh', 'Series_contact_state': 'Scotland', 'Series_contact_zip/postal_code': 'EH9 3BF', 'Series_contact_country': 'United Kingdom', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE252nnn/GSE252692/suppl/GSE252692_RAW.tar', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1062064'}\n",
      "16 PRJNA1062387 GSE252752 {'Series_title': 'N6-methyladenosine reader YTHDF1 promotes stemness and therapeutic resistance in hepatocellular carcinoma by enhancing NOTCH1 expression', 'Series_geo_accession': 'GSE252752', 'Series_status': 'Public on Jan 11 2024', 'Series_submission_date': 'Jan 08 2024', 'Series_last_update_date': 'Jan 11 2024', 'Series_summary': 'N6-methyladenosine (m6A) RNA modification is the most common and conserved epigenetic modification in mRNA and has been shown to play important roles in cancer biology. As the m6A reader YTHDF1 has been reported to promote progression of hepatocellular carcinoma (HCC), it represents a potential therapeutic target. In this study, we evaluated the clinical significance of YTHDF1 using human HCC samples and found that YTHDF1 was significantly upregulated in HCCs with high stemness scores and was positively associated with recurrence and poor prognosis. Analysis of hepatoma spheroids revealed that YTHDF1 was highly expressed in liver cancer stem cells (CSCs). Stem cell-specific conditional Ythdf1 knock-in (CKI) mice treated with diethylnitrosamine showed elevated tumor burden compared with wild type mice. YTHDF1 promoted CSC renewal and resistance to the multiple tyrosine kinase inhibitors lenvatinib and sorafenib in patient-derived organoids and HCC cell lines, which could be abolished by catalytically inactive mutant YTHDF1. RNA immunoprecipitation sequencing, m6A methylated RNA immunoprecipitation sequencing, ribosome profiling, and RNA sequencing identified NOTCH1 as a direct downstream of YTHDF1. YTHDF1 bound to m6A modified NOTCH1 mRNA to enhance its stability and translation, which led to increased NOTCH1 target genes expression. NOTCH1 overexpression rescued HCC stemness in YTHDF1 deficient cells in vitro and in vivo. Lipid nanoparticles targeting YTHDF1 significantly enhanced the efficacy of lenvatinib and sorafenib in HCC in vivo. Taken together, YTHDF1 drives HCC stemness and drug resistance through a YTHDF1-m6A-NOTCH1 epitranscriptomic axis, and YTHDF1 is a potential therapeutic target for treating HCC.', 'Series_overall_design': 'To investigate the molecular mechanism that silencing of YTHFD1 increased sensitivities against lenvatinib/sorafenib, we performed RNA-seq on tumor tissues from orthotopic xenograft implants treated with LNP-siYTHDF1 in combination with either lenvatinib or sorafenib.', 'Series_type': 'Other', 'Series_contributor': 'Yifan,,Wu', 'Series_sample_id': 'GSM8006354', 'Series_contact_name': 'Xinyue,,Zhang', 'Series_contact_email': 'zhangxy68@mail2.sysu.edu.cn', 'Series_contact_institute': 'Sun Yat-sen University First Affiliated Hospital', 'Series_contact_address': 'zhongshan road 2', 'Series_contact_city': 'Guangzhou', 'Series_contact_state': 'guangdong', 'Series_contact_zip/postal_code': '510000', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE252nnn/GSE252752/suppl/GSE252752_processed_data_file.xlsx', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1062387'}\n",
      "17 PRJNA1070982 GSE254531 {'Series_title': 'The functional small RNA interactome reveals targets for the vancomycin-responsive sRNA RsaOI in vancomycin tolerant Staphylococcus aureus [Ribo-seq]', 'Series_geo_accession': 'GSE254531', 'Series_status': 'Public on Mar 18 2024', 'Series_submission_date': 'Jan 30 2024', 'Series_last_update_date': 'May 01 2024', 'Series_pubmed_id': '38534138', 'Series_summary': 'Small RNAs have been found to control a broad range of bacterial phenotypes including tolerance to antibiotics. Vancomycin tolerance in multidrug resistance Staphylococcus aureus is correlated with dysregulation of small RNAs although their contribution to antibiotic tolerance in poorly understood. RNA-RNA interactome profiling techniques are expanding our understanding of sRNA-mRNA interactions in bacteria; however, determining the function of these interactions for hundreds of sRNA-mRNA pairs is a major challenge. At steady-state, protein and mRNA abundances are often highly correlated and lower than expected protein abundance may indicate translational repression of an mRNA. To identify sRNA-mRNA interactions that regulate mRNA translation, we examined the correlation between gene transcript abundance, ribosome occupancy, and protein levels. We used the machine learning technique self-organising maps (SOMS) to cluster genes with similar transcription and translation patterns and identified a cluster of mRNAs that appeared to be post-transcriptionally repressed. By integrating our clustering with sRNA-mRNA interactome data generated in vancomycin tolerant S. aureus by RNase III-CLASH, we identified sRNAs that may be mediating translational repression. We have confirmed sRNA-dependant post-transcriptional repression of several mRNAs in this cluster. Two of these interactions are mediated by RsaOI, a sRNA that is highly upregulated by vancomycin. We demonstrate regulation of HPr and the cell-wall autolysin Atl. These findings suggest RsaOI coordinates carbon metabolism and cell wall turnover during vancomycin treatment.', 'Series_overall_design': 'This study uses a metaomics approach to assess gene transcription and translation, and identify genes that are post-transcriptionally regulated. Ribo-seq was used the assess translational of genes and is presented in this entry. Ribo-seq was performed on biological duplicate cultures grown in Muller Hinton media to an OD of 0.8.', 'Series_type': 'Other', 'Series_contributor': 'Winton,,Wu', 'Series_sample_id': 'GSM8045619', 'Series_contact_name': 'Jai,Justin,Tree', 'Series_contact_email': 'j.tree@unsw.edu.au', 'Series_contact_phone': '+61 2 938 59142', 'Series_contact_laboratory': 'Tree lab', 'Series_contact_department': 'School of Biotechnology and Biomolecular Sciences', 'Series_contact_institute': 'University of New South Wales', 'Series_contact_address': 'Rm s110 Bldg F25, UNSW, Gate 11 Botany St', 'Series_contact_city': 'Sydney', 'Series_contact_state': 'NSW', 'Series_contact_zip/postal_code': '2033', 'Series_contact_country': 'Australia', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE254nnn/GSE254531/suppl/GSE254531_RAW.tar', 'Series_platform_id': 'GPL34143', 'Series_platform_taxid': '546342', 'Series_sample_taxid': '546342', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1070982'}\n",
      "18 PRJNA1074242 GSE255244 {'Series_title': 'Ex vivo activation of the GCN2 pathway metabolically reprograms T cells leading to enhanced adoptive cell therapy', 'Series_geo_accession': 'GSE255244', 'Series_status': 'Public on Feb 17 2024', 'Series_submission_date': 'Feb 07 2024', 'Series_last_update_date': 'Apr 17 2024', 'Series_pubmed_id': '38460518', 'Series_summary': 'The manipulation of T cell metabolism to enhance anti-tumor activity is an area of active investigation. Here, we report that activating the amino acid starvation response in effector CD8+ T cells ex vivo using the General Control Non-depressible 2 (GCN2) agonist halofuginone (halo) enhances oxidative metabolism and effector function. Further characterization revealed that halo-treated CD8+ T cells increased expression of the large neutral amino acid (LNAA) transporter CD98 as well as the co-stimulatory marker 4-1BB. Mechanistically, we identified autophagy coupled with the CD98-mTOR axis as key downstream mediators of the phenotype induced by halo treatment. The adoptive transfer of halo-treated CD8+ T cells into mice bearing well-established tumors led to robust tumor control and curative responses. Halo-treated T cells also synergized in vivo with a 4-1BB agonistic antibody to control tumor growth in a mouse model resistant to immunotherapy.  Importantly, treatment of human CD8+ T cells with halo resulted in similar metabolic and functional reprogramming. These findings demonstrate that activating the amino acid starvation response with the GCN2 agonist halofuginone can enhance T cell metabolism, effector function and anti-tumor activity, with the potential to augment existing clinical immunotherapeutic approaches.', 'Series_overall_design': '12 CD8+ T-cell Samples profiled by RIBO-Seq: 3 Halofuginone (HALO)-treated samples, Ribosome-Protected RNA Fragments (RPF) profiled; 3 Control samples, RPF-profiled; 3 Halo-treated samples, total RNA profiled; 3 control samples, total RNA profiled.', 'Series_type': 'Other', 'Series_contributor': 'Sayad,,Azin', 'Series_sample_id': 'GSM8067257', 'Series_contact_name': 'Pamela,,Ohashi', 'Series_contact_department': 'Tumor Immunotherapy Program', 'Series_contact_institute': 'University Health Network', 'Series_contact_address': '610 University Ave', 'Series_contact_city': 'Toronto', 'Series_contact_state': 'ON', 'Series_contact_zip/postal_code': 'M5G 2C4', 'Series_contact_country': 'Canada', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE255nnn/GSE255244/suppl/GSE255244_stpaul_saibil_all_samples_FPKM.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1074242'}\n",
      "19 PRJNA1088434 GSE261690 {'Series_title': '7q11.23 CNV alters ribosomal biogenesis, mTOR and neuronal intrinsic excitability [ribosome footprinting]', 'Series_geo_accession': 'GSE261690', 'Series_status': 'Public on Mar 20 2024', 'Series_submission_date': 'Mar 15 2024', 'Series_last_update_date': 'Aug 09 2024', 'Series_pubmed_id': '39007270', 'Series_summary': 'Copy number variations (CNVs) at 7q11.23 cause Williams-Beuren (WBS) and 7q microduplication syndromes (7Dup). Both neurodevelopmental disorders (NDDs) feature mild to moderate intellectual disability, accompanied by symmetrically opposite neurocognitive and behavioral features. Although significant progress has been made in understanding the molecular mechanisms underlying 7q11.23-related pathophysiologies, the interplay between different layers of gene expression and the propagation of opposite symmetry arising from CNV remains elusive.  Here, we uncovered 7q11.23 dosage-dependent symmetrically opposite dynamics in neuronal differentiation and intrinsic excitability. By integrating transcriptomics, translatomics and proteomics of patient-derived and isogenic induced neurons, we found that genes related to neuronal transmission follow 7q11.23 dosage and are transcriptionally controlled, while translational factors and ribosomal genes are post-transcriptionally buffered. Consistently, we found a deregulated mTOR pathway, with phospho-S6 (pS6) being downregulated in WBS and upregulated in 7Dup. Surprisingly phospho-4E-BP (p4E-BP) was going in the opposite direction, reflecting the total 4E-BP levels, thus highlighting the different roles of pS6 and p4E-BP during neurogenesis. Therefore, our work emphasizes the importance of multilayer research when investigating complex NDDs, highlighting ribosomal biogenesis and paving the way for novel treatments by uncoupling the role of pS6 and p4E-BPs in NDDs.', 'Series_overall_design': 'Using a 7q11.23 duplication lines, isogenic lines with 2 (CTL) and 1 (WBS) copies of the 7q11.23 were generated via CRISPR. All three lines were differentiated to iNeurons in triplicates, before ribosome footprinting.', 'Series_type': 'Other', 'Series_contributor': 'Giuseppe,,Testa', 'Series_sample_id': 'GSM8148833', 'Series_contact_name': 'Alessandro,,Vitriolo', 'Series_contact_email': 'alessandro.vitriolo@fht.org', 'Series_contact_department': 'Neurogenomics', 'Series_contact_institute': 'Fondazione Human Technopole', 'Series_contact_address': 'Via Rita Levi Montalcini 1', 'Series_contact_city': 'Milano', 'Series_contact_state': 'MI', 'Series_contact_zip/postal_code': '20157', 'Series_contact_country': 'Italy', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE261nnn/GSE261690/suppl/GSE261690_riboFoot.SE.rds.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1088434'}\n",
      "20 PRJNA1099224 GSE263777 {'Series_title': 'Two distinct Trypanosoma eIF4F complexes co-exist, bind different mRNAs and are regulated during nutritional stress', 'Series_geo_accession': 'GSE263777', 'Series_status': 'Public on Apr 14 2024', 'Series_submission_date': 'Apr 11 2024', 'Series_last_update_date': 'Apr 14 2024', 'Series_summary': 'Many eIF4F subunits and PABP paralogues are found in trypanosomes: six eIF4E, five eIF4G, one eIF4A and two PABPs. They are expressed simultaneously and assemble into different complexes, contrasting the situation in metazoans that use distinct complexes in different cell types or developmental stages. Each eIF4F complex has its own proteins, mRNAs and, consequently, a distinct function. We set out to study the function and regulation of the two major eIF4F complexes of Trypanosoma cruzi. We identified the associated proteins and mRNAs of eIF4E3 and eIF4E4 in cells in exponential growth and in nutritional stress. Upon stress, eIF4G/eIF4A and PABP remain associated to the eIF4E, but the association with other 43S pre-initiation factors decreases, indicating that ribosome attachment is impaired. Most eIF4E3-associated mRNAs encode for metabolic proteins, while eIF4E4 associate to mRNAs encoding ribosomal proteins. Interestingly, for both eIF4E3/4, more mRNAs were associated in stressed cells than in non-stressed cells, even though stress causes ribosome disassembly. In summary, trypanosomes have two co-existing eIF4F complexes involved in translation of distinct mRNA cohorts important for growth. Under stress conditions, both complexes exit translation but remain bound to their mRNA targets.', 'Series_overall_design': 'We analysed RNAs associated to Trypanosoma cruzi eIF4E3 and eIF4E4 using RIP-seq in exponential growth and nutritional stress. We also analysed polyA total RNA and sequenced polysomal RNA (Ribo-seq). in these two conditions.', 'Series_type': 'Other', 'Series_contributor': 'Fabíola,,Barbieri Holetz', 'Series_sample_id': 'GSM8198435', 'Series_contact_name': 'Bernardo,,Papini Gabiatti', 'Series_contact_email': 'bgabiatti@gmail.com', 'Series_contact_institute': 'University of Würzburg', 'Series_contact_address': 'Sanderring 2', 'Series_contact_city': 'Würzburg', 'Series_contact_zip/postal_code': '97070', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE263nnn/GSE263777/suppl/GSE263777_raw_counts.txt.gz', 'Series_platform_id': 'GPL34379', 'Series_platform_taxid': '5693', 'Series_sample_taxid': '5693', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1099224'}\n",
      "21 PRJNA142427 GSE25626 {'Series_title': \"Initiation Pausing of 80S Ribosome on mRNA Is Governed by 5'UTR and Responsive to mTORC1 Signaling (RNA-Seq)\", 'Series_geo_accession': 'GSE25626', 'Series_status': 'Public on Nov 12 2011', 'Series_submission_date': 'Nov 27 2010', 'Series_last_update_date': 'May 15 2019', 'Series_summary': 'Recent studies have revealed that the mRNA translation is punctuated by ribosomal pauses through the body of transcripts.  However, little is known about its physiological significance and regulatory aspects.  Here we present a multi-dimensional ribosome profiling approach to quantify the dynamics of initiation and elongation of 80S ribosomes across the entire transcriptome in mammalian cells.  We show that a subset of transcripts have a significant pausing of 80S ribosome around the start codon, creating a major barrier to the commitment of translation elongation.  Intriguingly, genes encoding ribosome proteins themselves exhibit an exceptionally high initiation pausing on their transcripts.  Our studies also reveal that the initiation pausing is dependent on the 5’ untranslated region (5’ UTR) of mRNAs and subject to the regulation of mammalian target of rapamycin complex 1 (mTORC1).  Thus, the initiation pausing of 80S ribosome represents a novel regulatory step in translational control mediated by nutrient signaling pathway.', 'Series_overall_design': 'Monitor the translational status of transcriptome in mammalian cells under different conditions', 'Series_type': 'Other', 'Series_contributor': 'Shu-Bing,,Qian', 'Series_sample_id': 'GSM691013', 'Series_contact_name': 'Shu-Bing,,Qian', 'Series_contact_email': 'sq38@cornell.edu', 'Series_contact_phone': '(607)254-3397', 'Series_contact_department': 'Division of Nutritional Sciences', 'Series_contact_institute': 'Cornell University', 'Series_contact_address': '301 Biotech Bldg.', 'Series_contact_city': 'Ithaca', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '14853', 'Series_contact_country': 'USA', 'Series_contact_web_link': 'http://qian.human.cornell.edu', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE25nnn/GSE25626/suppl/GSE25626_RAW.tar', 'Series_platform_id': 'GPL9185', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA142427'}\n",
      "22 PRJNA144149 GSE30839 {'Series_title': 'Ribosome\\xa0Profiling\\xa0of Mouse Embryonic Stem Cells Reveals the Complexity of Mammalian Proteomes', 'Series_geo_accession': 'GSE30839', 'Series_status': 'Public on Nov 03 2011', 'Series_submission_date': 'Jul 21 2011', 'Series_last_update_date': 'Sep 11 2019', 'Series_pubmed_id': '23810193', 'Series_summary': 'The ability to sequence genomes has far outstripped approaches for deciphering the  information they encode. Here we present a suite of techniques, based on ribosome profiling (the deep-sequencing of ribosome-protected mRNA fragments), to provide genome-wide maps of protein  synthesis as well as a pulse-chase strategy for determining rates of translation elongation. We exploit  the propensity of harringtonine to cause ribosomes to accumulate at sites of translation initiation  together with a machine learning algorithm to define protein products systematically. Analysis of  translation in mouse embryonic stem cells reveals thousands of strong pause sites and novel  translation products. These include amino-terminal extensions and truncations and upstream open  reading frames with regulatory potential, initiated at both AUG and non-AUG codons, whose  translation changes after differentiation. We also define a new class of short, polycistronic ribosome-associated coding RNAs (sprcRNAs) that encode small proteins. Our studies reveal an unanticipated  complexity to mammalian proteomes.', 'Series_overall_design': 'Examination of translation in mouse embryonic stem cells and during differentiation into embryoid bodies', 'Series_type': 'Other', 'Series_contributor': 'Jonathan,S,Weissman', 'Series_sample_id': 'GSM765302', 'Series_contact_name': 'Nicholas,T,Ingolia', 'Series_contact_email': 'ingolia@berkeley.edu, nick@ingolia.org', 'Series_contact_phone': '510 664 7071', 'Series_contact_laboratory': 'Ingolia', 'Series_contact_department': 'Molecular and Cell Biology', 'Series_contact_institute': 'University of California, Berkeley', 'Series_contact_address': '1 Barker Hall # 3202', 'Series_contact_city': 'Berkeley', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94720', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE30nnn/GSE30839/suppl/GSE30839_readme.txt', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA144149'}\n",
      "23 PRJNA147613 GSE32060 {'Series_title': 'Widespread regulation of translation by elongation pausing in heat shock', 'Series_geo_accession': 'GSE32060', 'Series_status': 'Public on Jan 03 2013', 'Series_submission_date': 'Sep 12 2011', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '23290915', 'Series_summary': 'RNA-seq and ribosome footprinting libraries of mouse 3T3 and human 293T cellsrelated to Shalgi et al.  2013', 'Series_overall_design': '36bases paired-end RNA-seq, and ribosome footprinting libraries for: 3T3 cells - Control, 8 hours of mild heat shock (42) and 2 hours of severe heat shock (44) - in replicates, as well as 3T3 cells treated by mild followed by severe heat shock. In addition, 3T3 cells treated with Hsp70 inhibitor VER-155008 (Massey et al. 2010), and 293T cells transfected with Hspa1a or GFP, before and after 2 hours of severe heat shock.', 'Series_type': 'Other', 'Series_contributor': 'Christopher,B,Burge', 'Series_sample_id': 'GSM1063578', 'Series_contact_name': 'Reut,,Shalgi', 'Series_contact_email': 'reutshalgi@technion.ac.il', 'Series_contact_department': 'Faculty of Medicine', 'Series_contact_institute': 'Technion', 'Series_contact_address': '1 Efron st.', 'Series_contact_city': 'Haifa', 'Series_contact_zip/postal_code': '31096', 'Series_contact_country': 'Israel', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE32nnn/GSE32060/suppl/GSE32060_RAW.tar', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP017942'}\n",
      "24 PRJNA150397 GSE34743 {'Series_title': 'Ribosome profiling of early zebrafish embryos -- miRNA-mediated regulation during embryogenesis causes translational repression before mRNA decay', 'Series_geo_accession': 'GSE34743', 'Series_status': 'Public on Mar 15 2012', 'Series_submission_date': 'Dec 27 2011', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '22422859', 'Series_summary': 'MicroRNAs regulate gene expression through deadenylation, repression and mRNA decay. However, the contribution of each mechanism in non-steady-state situations remains unclear.  We monitored the impact of miR-430 on ribosome occupancy of endogenous mRNAs in wild type and dicer mutants lacking mature miR-430. Our results indicate that miR-430 reduces the number of ribosomes on target mRNAs before causing mRNA decay. Translational repression occurs before complete deadenylation, and disrupting deadenylation using an internal poly(A) tail did not block target repression. Finally, we observe that ribosome density along the length of the target mRNA remains constant, suggesting that translational repression occurs by reducing the initiation rate rather than reducing elongation or causing ribosomal drop-off. In summary, our results show that miR-430 regulates translation initiation before inducing mRNA decay.', 'Series_overall_design': 'Time course parallel ribosome profiling and input mRNA quantification in wildtype and MZdicer mutant embryos', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Antonio,J,Giraldez', 'Series_sample_id': 'GSM854443', 'Series_contact_name': 'Antonio,J.,Giraldez', 'Series_contact_email': 'antonio.giraldez@yale.edu', 'Series_contact_phone': '203 785 5423', 'Series_contact_laboratory': 'Giraldez Lab', 'Series_contact_department': 'Genetics', 'Series_contact_institute': 'Yale University', 'Series_contact_address': '333 Cedar Street', 'Series_contact_city': 'New Haven', 'Series_contact_state': 'CT', 'Series_contact_zip/postal_code': '06510', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE34nnn/GSE34743/suppl/GSE34743_RAW.tar', 'Series_platform_id': 'GPL9319', 'Series_platform_taxid': '7955', 'Series_sample_taxid': '7955', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA150397'}\n",
      "25 PRJNA152347 GSE35469 {'Series_title': 'The translational landscape of mTOR signaling steers cancer initiation and metastasis', 'Series_geo_accession': 'GSE35469', 'Series_status': 'Public on Feb 28 2012', 'Series_submission_date': 'Feb 01 2012', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '22367541', 'Series_summary': 'The mammalian target of rapamycin (mTOR) kinase is a master regulator of protein synthesis that couples nutrient sensing to cell growth and cancer. However, the downstream translationally regulated nodes of gene expression that may direct cancer development are poorly characterized. Employing ribosome profiling we uncover specialized translation of the prostate cancer genome by oncogenic mTOR signaling, revealing a remarkably specific repertoire of genes involved in cell proliferation, metabolism, and invasion. We extend these findings by functionally characterizing a class of translationally controlled pro-invasion mRNAs that we show direct prostate cancer invasion and metastasis downstream of oncogenic mTOR signaling. We further develop a clinically relevant ATP site inhibitor of mTOR, INK128, which reprograms this gene expression signature with therapeutic benefit for prostate cancer metastasis, for which there is presently no cure. Together, these findings significantly extend our understanding of how the “cancerous” translation machinery steers specific cancer cell behaviors and may be therapeutically targeted.', 'Series_overall_design': 'Examination of mRNA translation in human prostate cancer upon differential inhibition of the mTOR signaling pathway.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'D,,Ruggero', 'Series_sample_id': 'GSM869047', 'Series_contact_name': 'Andrew,Caleb,Hsieh', 'Series_contact_email': 'andrew.hsieh@ucsf.edu', 'Series_contact_phone': '415-514-4827', 'Series_contact_fax': '415-514-4826', 'Series_contact_laboratory': 'Davide Ruggero', 'Series_contact_department': 'Urology', 'Series_contact_institute': 'UCSF', 'Series_contact_address': '1450 3rd Street, HD Rm 319', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE35nnn/GSE35469/suppl/GSE35469_RAW.tar', 'Series_platform_id': 'GPL9115', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA152347'}\n",
      "26 PRJNA152495 GSE35641 {'Series_title': 'The anti-Shine-Dalgarno sequence drives translational pausing and codon choice in bacteria', 'Series_geo_accession': 'GSE35641', 'Series_status': 'Public on Apr 01 2012', 'Series_submission_date': 'Feb 08 2012', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '22456704', 'Series_summary': 'Protein synthesis by ribosomes takes place on a linear substrate but at variable speeds. Transient pausing of ribosomes can impact a variety of co-translational processes, including protein targeting and folding. These pauses are influenced by the sequence of the mRNA. Thus redundancy in the genetic code allows the same protein to be translated at different rates. However, our knowledge of both the position and the mechanism of translational pausing in vivo is highly limited. Here we present a genome-wide analysis of translational pausing in bacteria using ribosome profiling-deep sequencing of ribosome-protected mRNA fragments. This approach enables high-resolution measurement of ribosome density profiles along most transcripts at unperturbed, endogenous expression levels. Unexpectedly, we found that codons decoded by rare tRNAs do not lead to slow translation under nutrient-rich conditions. Instead, Shine-Dalgarno-(SD) like features within coding sequences cause pervasive translational pausing. Using an orthogonal ribosome possessing an altered anti-SD sequence, we demonstrated that pausing is due to hybridization between mRNA and the 16S rRNA of the translating ribosome. In protein coding sequences, internal SD sequences are disfavoured, which leads to biased usage, avoiding codons and codon pairs that resemble canonical SD sites. Our results indicate that internal SD-like sequences are a major determinant of translation rates and a global driving force for the coding of bacterial genomes.', 'Series_overall_design': 'Identification of translation pause sites in vivo using ribosome profiling', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Jonathan,S,Weissman', 'Series_sample_id': 'GSM872399', 'Series_contact_name': 'Gene-Wei,,Li', 'Series_contact_email': 'gene-wei.li@ucsf.edu', 'Series_contact_institute': 'UCSF', 'Series_contact_address': '1700 4th Street, BH404', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE35nnn/GSE35641/suppl/GSE35641_RAW.tar', 'Series_platform_id': 'GPL15206', 'Series_platform_taxid': '679895', 'Series_sample_taxid': '679895', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA152495'}\n",
      "27 PRJNA156379 GSE34438 {'Series_title': 'Ribosome profiling of wild-type and loc1- cells in S. cerevisiae', 'Series_geo_accession': 'GSE34438', 'Series_status': 'Public on Jan 01 2014', 'Series_submission_date': 'Dec 14 2011', 'Series_last_update_date': 'May 15 2019', 'Series_summary': 'We performed ribosome profiling of wild-type and loc1- cells.  We found that approximately 1500 genes exhibit increased translational efficiencies in the absence of Loc1, indicating that Loc1 is required for their translational repression.', 'Series_overall_design': 'Examination of translational efficiencies by comparing ribosome footprints and mRNA abundances in two different cell lines.', 'Series_type': 'Other', 'Series_contributor': 'Frederick,P,Roth', 'Series_sample_id': 'GSM849112', 'Series_contact_name': 'Suzanne,,Komili', 'Series_contact_institute': 'Harvard University', 'Series_contact_address': '44 Binney St.', 'Series_contact_city': 'Boston', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02115', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE34nnn/GSE34438/suppl/GSE34438_RAW.tar', 'Series_platform_id': 'GPL13272', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA156379'}\n",
      "28 PRJNA172014 GSE39911 {'Series_title': 'Transcriptome-wide Regulation of Splicing and mRNA Localization by Muscleblind Proteins', 'Series_geo_accession': 'GSE39911', 'Series_status': 'Public on Aug 17 2012', 'Series_submission_date': 'Aug 06 2012', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '22901804', 'Series_summary': \"The Muscleblind-like (Mbnl) family of RNA-binding proteins plays important roles in muscle and eye development and in Myotonic Dystrophy (DM), where expanded CUG or CCUG repeats functionally deplete Mbnl proteins. We identified transcriptome-wide functional and biophysical targets of Mbnl proteins in brain, heart, muscle, and myoblasts using RNA sequencing and crosslinking/immunoprecipitation-sequencing approaches. This analysis identified several hundred splicing events whose regulation depended on Mbnl function, in a pattern indicative of functional interchangeability between Mbnl1 and Mbnl2. A nucleotide resolution RNA map associated repression or activation of exon splicing with Mbnl binding near either 3' splice site or near the downstream 5' splice site, respectively. Transcriptomic analysis of sub-cellular compartments uncovered a global role for Mbnls in regulating localization of mRNAs encoding membrane, synaptic and other proteins in both mouse and Drosophila cells, and Mbnls also contribute to protein secretion. These findings hold several new implications for DM pathogenesis.\", 'Series_overall_design': 'To assess global functions of Muscleblind proteins, RNA-Seq was performed using WT and Mbnl1 KO brain, heart, and muscle (5 mice each). Additionally, C2C12 mouse myoblasts were depleted of Mbnl1, Mbnl2, or both. Subcellular fractionation experiments were performed to analyze mRNA localization following depletion of Mbnl1 and Mbnl2 in C2C12 mouse myoblasts, and following depletion of Mbnl in Drosophila S-2R+ cells. CLIP-Seq was also performed against Mbnl1 in mouse brain, heart, muscle, and C2C12 myoblasts. Finally, ribosome footprinting was performed with C2C12 mouse myoblasts that were depleted of Mbnl1, Mbnl2, or both.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Christopher,B,Burge', 'Series_sample_id': 'GSM981234', 'Series_contact_name': 'Eric,T,Wang', 'Series_contact_institute': 'University of Florida', 'Series_contact_address': '2033 Mowry Rd', 'Series_contact_city': 'Gainesville', 'Series_contact_state': 'FL', 'Series_contact_zip/postal_code': '32610', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE39nnn/GSE39911/suppl/GSE39911_tissue_RPKM.txt.gz', 'Series_platform_id': 'GPL13304', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP014709'}\n",
      "29 PRJNA173092 GSE40209 {'Series_title': 'Regulatory impact of RNA secondary structure across the Arabidopsis thaliana transcriptome', 'Series_geo_accession': 'GSE40209', 'Series_status': 'Public on Nov 15 2012', 'Series_submission_date': 'Aug 17 2012', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '23150631', 'Series_summary': 'The secondary structure of an RNA molecule plays an integral role in its maturation, regulation, processing, and functionality. However, the global influence of this feature on plant gene expression is still for the most part unclear. Here, we use a high-throughput, sequencing-based, structure-mapping approach in conjunction with transcriptome-wide sequencing of polyA+-selected (RNA-seq), small (smRNA-seq), and ribosome-bound (ribo-seq) RNA populations to investigate the impact of RNA secondary structure on gene expression regulation in Arabidopsis. From this analysis, we find that highly unpaired and paired RNAs are strongly correlated with euchromatic and heterochromatic epigenetic histone modifications, respectively, providing further evidence that secondary structure is necessary for RNA-mediated posttranscriptional regulatory pathways. Additionally, we uncover key structural patterns across protein-coding transcripts that indicate RNA folding demarcates regions of protein translation and likely affects microRNA-mediated regulation of mRNAs in this model plant. We also reveal that RNA folding is significantly anti-correlated with overall transcript abundance, which is likely due to the increased propensity of highly structured mRNAs to be degraded and/or processed into smRNAs. Finally, we find that secondary structure affects mRNA translation, suggesting that this feature regulates plant gene expression at multiple levels. Overall, our findings provide the first global assessment of RNA folding and its significant regulatory effects in a plant transcriptome.', 'Series_overall_design': 'Single-stranded RNA sequencing (ssRNA-seq) and ribosome-bound RNA sequencing (ribo-seq) in immature buds. A single replicate of each library.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Brian,D,Gregory', 'Series_sample_id': 'GSM988392', 'Series_contact_name': 'Fan,,Li', 'Series_contact_department': 'Pediatrics', 'Series_contact_institute': 'University of California Los Angeles', 'Series_contact_address': '675 Charles E Young Drive South', 'Series_contact_city': 'Los Angeles', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '90024', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE40nnn/GSE40209/suppl/GSE40209_RAW.tar', 'Series_platform_id': 'GPL13222', 'Series_platform_taxid': '3702', 'Series_sample_taxid': '3702', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP014909'}\n",
      "30 PRJNA177820 GSE41590 {'Series_title': 'Genome-wide translational changes induced by the prion [PSI+]', 'Series_geo_accession': 'GSE41590', 'Series_status': 'Public on Jul 18 2014', 'Series_submission_date': 'Oct 15 2012', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25043188', 'Series_summary': 'Prions are infectious proteins that can adopt a structural conformation different from that of the normal protein. This change of conformation is then propagated among other molecules of the same protein. Prions are associated with neurodegenerative diseases in mammals, but are also found in fungi (in the yeast Saccharomyces cerevisiae and the filamentous fungus Podospora anserina), in which they control heritable traits. They are widespread in wild yeast strains, suggesting a biologically important role. [PSI+] is one of the most widely studied yeast prions. It corresponds to an aggregated conformation of the translational release factor, eRF3, which suppresses nonsense codons. [PSI+] modifies cellular fitness, inducing various phenotypes, depending on the genetic background. However, the genes displaying [PSI+]-controlled expression remain largely unknown. We used the recently described ribosome profiling approach to identify genes displaying changes in expression in the presence of [PSI+]. This made it possible to determine the positions of all active ribosomes within the genome, in both [PSI+] and [PSI-] isogenic strains. Comparisons of the translatomes and transcriptomes of the two strains revealed that the primary effect of [PSI+] was to repress genes involved in the stress response. Thus, we provide the first description of the global translational effect of [PSI+] and a new genetic explanation of the phenotypic differences between [PSI-] and [PSI+] strains under stress conditions.', 'Series_overall_design': 'Comparisons of the translatomes of the two strains\\xa0: a  [PSI+] strain (two replicates) and a [PSI+] overexpressing SUP35-Cter strain.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Olivier,,Namy', 'Series_sample_id': 'GSM1347626', 'Series_contact_name': 'Olivier,,Namy', 'Series_contact_email': 'olivier.namy@i2bc.paris-saclay.fr', 'Series_contact_laboratory': 'Genomic, structure and Translation', 'Series_contact_department': 'I2BC', 'Series_contact_institute': 'CNRS, Université Paris-Sud', 'Series_contact_address': 'Rue Gregor Mendel, bat 400', 'Series_contact_city': 'Orsay', 'Series_contact_zip/postal_code': '91400', 'Series_contact_country': 'France', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE41nnn/GSE41590/suppl/GSE41590_results_htseq-count.txt.gz', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP016508'}\n",
      "31 PRJNA178171 GSE41785 {'Series_title': 'Global analysis of Upf1 in mESCs reveals expanded scope of nonsense-mediated mRNA decay', 'Series_geo_accession': 'GSE41785', 'Series_status': 'Public on Jun 12 2013', 'Series_submission_date': 'Oct 23 2012', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '23766421', 'Series_summary': \"Nonsense-mediated mRNA decay (NMD) is a conserved RNA surveillance pathway that is an important modulator of disease pathology and is required for embryonic development.  Despite significant research effort, the rules that govern NMD remain incompletely understood. Here we used a combined¬ approach, integrating RNA-Seq, ribosome footprinting, and CLIP-Seq analysis of the essential NMD factor Upf1, to provide a more complete picture of the role of NMD in modulating gene expression in murine embryonic stem cells (mESCs).  We show that presence of an exon-exon junction ≥50 nucleotides (nt) downstream of a termination codon (dEJ) contributes to NMD independently of 3' UTR length, but has stronger effects in genes with shorter 3' UTRs.  We also map translated upstream open reading frames (uORFs) in mESCs and show that they are associated with NMD regulation, especially of genes encoding transcription factors, and we find that lowly translated mRNAs can escape NMD.  Finally, we identify over 200 direct binding targets of Upf1 and describe a pathway of Upf1-dependent gene regulation reliant on Upf1 binding to the 3' UTR and independent of presence of a dEJ.   Together, these analyses characterize known and discover novel determinants of NMD and establish a broader role in mESC gene regulation for Upf1.\", 'Series_overall_design': 'mRNA-Seq analysis of wildtype (2 samples), translationally inhibited (by cycloheximide treatment, 2 samples),  control-depleted (2 samples),  and Upf1-depleted (4 samples) mouse embryonic stem cells (mESCs); CLIP-Seq analysis of Upf1 (5 samples, and 5 samples of IgG control CLIP-Seq); Ribosome footprint profiling of wildtype (1 sample), control-depleted (1 sample), and Upf1-depleted (1 sample) mESCs', 'Series_type': 'Other', 'Series_contributor': 'Christopher,B,Burge', 'Series_sample_id': 'GSM1024313', 'Series_contact_name': 'Jessica,,Hurt', 'Series_contact_email': 'hurt@mit.edu', 'Series_contact_laboratory': 'Christopher Burge', 'Series_contact_department': 'Biology', 'Series_contact_institute': 'Massachusetts Institute of Technology', 'Series_contact_address': '77 Massachusetts Avenue', 'Series_contact_city': 'Cambridge', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02139', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE41nnn/GSE41785/suppl/GSE41785_v6.5_NoCHX_merge_isoforms.fpkm_tracking.gz', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP016625'}\n",
      "32 PRJNA187087 GSE43703 {'Series_title': 'Translational landscape of photomorphogenic Arabidopsis', 'Series_geo_accession': 'GSE43703', 'Series_status': 'Public on Nov 08 2013', 'Series_submission_date': 'Jan 23 2013', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '24179124', 'Series_summary': 'We report the application of ribosome profiling method to map ribosome locations on transcripts in de-etiolating Arabidopsis seedlings', 'Series_overall_design': 'Examine steady-state mRNA and ribosome-protected mRNA abundance in 4-day-old etiolated seedlings with/without 4-h light treatment', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Shu-Hsing,,Wu', 'Series_sample_id': 'GSM1226369', 'Series_contact_name': 'Ming-Jung,,Liu', 'Series_contact_email': 'mjliu@gate.sinica.edu.tw', 'Series_contact_institute': 'Academia sinica', 'Series_contact_address': '128 Sec. 2, Academia Rd, Nankang', 'Series_contact_city': 'Taipei', 'Series_contact_zip/postal_code': '11529', 'Series_contact_country': 'Taiwan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE43nnn/GSE43703/suppl/GSE43703_processed_data_repeat2_samples.txt.gz', 'Series_platform_id': 'GPL13222', 'Series_platform_taxid': '3702', 'Series_sample_taxid': '3702', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP018118'}\n",
      "33 PRJNA193543 GSE45366 {'Series_title': 'Loss of a Conserved tRNA Anticodon Modification Perturbs Cellular Signaling', 'Series_geo_accession': 'GSE45366', 'Series_status': 'Public on Aug 01 2013', 'Series_submission_date': 'Mar 20 2013', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '23935536', 'Series_summary': 'Transfer RNA (tRNA) modifications enhance the efficiency, specificity and fidelity of translation in all organisms. The anticodon modification mcm5s2U34 is required for normal growth and stress resistance in yeast; mutants lacking this modification have numerous phenotypes. Mutations in the homologous human genes are linked to neurological disease. The yeast phenotypes can be ameliorated by overexpression of specific tRNAs, suggesting that the modifications are necessary for efficient translation of specific codons. We determined the in vivo ribosome distributions at single codon resolution in yeast strains lacking mcm5s2U. We found accumulations at AAA, CAA, and GAA codons, suggesting that translation is slow when these codons are in the ribosomal A site, but these changes appeared too small to affect protein output. Instead, we observed activation of the GCN4-mediated stress response by a non- canonical pathway. Thus, loss of mcm5s2U causes global effects on gene expression due to perturbation of cellular signaling.', 'Series_overall_design': 'WT yeast and mutants lacking anticodon tRNA modifications were grown in YPD, and subjected to ribosome footprint profiling (ribo-seq) and RNA-seq of poly-A selected RNA. Dataset contains biological replicates for WT, Δncs6 and Δuba4. Technical replicates were also performed for all RNA-seq datasets (using a different poly-A selection method).', 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'W,V,Gilbert', 'Series_sample_id': 'GSM1103199', 'Series_contact_name': 'Boris,,Zinshteyn', 'Series_contact_email': 'zinshteyn.boris@gmail.com', 'Series_contact_laboratory': 'Green', 'Series_contact_department': 'Molecular Biology and Genetics', 'Series_contact_institute': 'Johns Hopkins University School of Medicine', 'Series_contact_address': '725 N Wolfe St, PCTB 714', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'Maryland', 'Series_contact_zip/postal_code': '21205', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE45nnn/GSE45366/suppl/GSE45366_MSUM_combined_rpkms.txt.gz', 'Series_platform_id': 'GPL13272', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP019833'}\n",
      "34 PRJNA197001 GSE46038 {'Series_title': 'The effects of dietary selenium on selenocysteine incorporation and selenoprotein expression', 'Series_geo_accession': 'GSE46038', 'Series_status': 'Public on Sep 17 2013', 'Series_submission_date': 'Apr 13 2013', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '23696641', 'Series_summary': 'The goal of this study was to determine the effects of dietary selenium levels on translational control of selenoprotein synthesis in mouse liver.', 'Series_overall_design': 'Wild type mice and mice expressing a mutant Sec-tRNA gene (TrspA37G) were fed diets supplemented with 0, 0.1, or 2 ppm selenium for 6 weeks. Livers were harvested and ribosome and mRNA profiles were generated by deep-sequencing using the Illumina HiSeq 2000.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Michael,T,Howard', 'Series_sample_id': 'GSM1122216', 'Series_contact_name': 'Michael,T,Howard', 'Series_contact_email': 'mhoward@genetics.utah.edu', 'Series_contact_phone': '801-585-1927', 'Series_contact_laboratory': 'Howard', 'Series_contact_department': 'Human Genetics', 'Series_contact_institute': 'University of Utah', 'Series_contact_address': '15 N 2030 E', 'Series_contact_city': 'Salt Lake City', 'Series_contact_state': 'UT', 'Series_contact_zip/postal_code': '84112', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE46nnn/GSE46038/suppl/GSE46038_Seps.xlsx.gz', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP021058'}\n",
      "35 PRJNA198514 GSE46277 {'Series_title': 'Transcription in Pronuclei and One- to Four-Cell Embryos Drives Early Development in a Nematode', 'Series_geo_accession': 'GSE46277', 'Series_status': 'Public on Jan 02 2014', 'Series_submission_date': 'Apr 22 2013', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '24374308', 'Series_summary': 'A long-standing view of development is that transcription is silenced in the oocyte until early divisions in the embryo. The point at which major transcription is reactivated varies in different organisms, but is usually after the 2-cell stage. However, this model may not be universal. We used RNA-seq and exploited the protracted development of the parasitic nematode Ascaris suum, to provide a comprehensive time course of mRNA expression, degradation, and translation during early development. Surprisingly, we find that ~4,000 genes are transcribed prior to pronuclear fusion and in the 1-4 cell embryos. Intriguingly, we do not detect maternal contribution of many orthologs of maternal C. elegans mRNAs, but instead find these are newly transcribed in the A. suum zygote prior to pronuclear fusion. Ribosome profiling demonstrates that, in general, early embryonic mRNAs are not stored for subsequent translation, but are directly translated following their synthesis. The role of maternally contributed and zygotically transcribed genes differs between the nematodes A. suum and C. elegans despite the fact that the two nematodes appear to exhibit highly similar morphological patterns during early development. Our study indicates that major transcription can occur immediately after fertilization and prior to pronuclear fusion in metazoa, suggesting that newly transcribed genes appear to drive A. suum early development. Furthermore, the mechanisms used for controlling the timing of the expression of key conserved genes has been altered between the two nematodes, illustrating significant plasticity in the regulatory networks that play important roles in developmental outcomes in nematodes.', 'Series_overall_design': 'A total of 28 samples are used for this study. The RNAs for the library construction are derived from 0.5 – 1 ml of synchronized embryo stages corresponding to a minimum of 0.5 billion embryos derived from > 1,000 female worms. The RNAs for polysome analysis are from pooled fractions of polysome gradient. For each sample, the amount of RNA for all polysome fractions is equivalent to the total RNA from the sample. Thus, the data from polysome profiling and RNA transcriptomes are considered as biological replicates. The level of the RNAs between each neighboring stages during development are considered as the reference for each other. Different sequencing strategies were applied for multiple libraries.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Richard,E,Davis', 'Series_sample_id': 'GSM1264049', 'Series_contact_name': 'Jianbin,,Wang', 'Series_contact_email': 'Jianbin.Wang@ucdenver.edu', 'Series_contact_phone': '303-724-3227', 'Series_contact_laboratory': 'Richard E. Davis Lab', 'Series_contact_department': 'Biochemistry and Molecular Genetics', 'Series_contact_institute': 'University of Colorado Denver', 'Series_contact_address': '12801 East 17th Ave', 'Series_contact_city': 'Aurora', 'Series_contact_state': 'CO', 'Series_contact_zip/postal_code': '80045', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE46nnn/GSE46277/suppl/GSE46277_A_summ_mRNAs_polysomes_RPKM.txt.gz', 'Series_platform_id': 'GPL17050', 'Series_platform_taxid': '6253', 'Series_sample_taxid': '6253', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP021466'}\n",
      "36 PRJNA200706 GSE46512 {'Series_title': 'Ribosome Profiling over a Zebrafish Developmental Timecourse', 'Series_geo_accession': 'GSE46512', 'Series_status': 'Public on May 23 2013', 'Series_submission_date': 'Apr 30 2013', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '23698349', 'Series_summary': \"To experimentally-validate the non-coding status of annotated lncRNAs, we performed ribosome profiling over a developmental timecourse that matched our previously-published (Pauli et al. 2012) developmental transcriptome. We find that many previously-annotated lncRNAs appear to be translated, but in a pattern more akin to 5' leaders of coding genes.\", 'Series_overall_design': 'Ribosome profiling over 8 stages in early zebrafish development: 2-4 cell, 256 cell, 1K cell, Dome, Shield, Bud, 28hpf and 5dpf', 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'Alexander,,Schier', 'Series_sample_id': 'GSM1131537', 'Series_contact_name': 'Guo-Liang,,Chew', 'Series_contact_email': 'chew@fas.harvard.edu', 'Series_contact_laboratory': 'Schier', 'Series_contact_department': 'Dept of Molecular and Cellular Biology', 'Series_contact_institute': 'Harvard University', 'Series_contact_address': '16 Divinity Ave BL 1019', 'Series_contact_city': 'Cambridge', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02139', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE46nnn/GSE46512/suppl/GSE46512_zv9_lncLoci.gtf.gz', 'Series_platform_id': 'GPL14875', 'Series_platform_taxid': '7955', 'Series_sample_taxid': '7955', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP021915'}\n",
      "37 PRJNA206070 GSE47558 {'Series_title': 'Nanog, SoxB1 and Pou5f1/Oct4 regulate widespread zygotic gene activation during the maternal-to-zygotic transition', 'Series_geo_accession': 'GSE47558', 'Series_status': 'Public on Sep 22 2013', 'Series_submission_date': 'May 31 2013', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '24056933', 'Series_summary': 'Upon fertilization, maternal factors direct development in a transcriptionally silent embryo. At the maternal-to-zygotic transition (MZT), a universal step in animal development, unknown maternal factors trigger zygotic genome activation (ZGA). In zebrafish, ZGA is required for gastrulation and clearance of maternal mRNAs, which is achieved in part by the conserved microRNA miR-430. However, the precise factors that activate the zygotic program remain largely unknown. Here we show that Nanog, Pou5f1 and SoxB1 are required for genome activation in zebrafish. We identified several hundred genes directly activated by maternal factors, thus constituting the first wave of zygotic transcription in zebrafish. Ribosome profiling in the pre-MZT embryo revealed that nanog, sox19b and pou5f1 are the most highly translated transcription factor mRNAs. Combined loss of function for Nanog, SoxB1 and Pou5f1 resulted in developmental arrest prior to gastrulation, and a failure to activate >75% of zygotic genes. Furthermore, we found that Nanog binds the miR-430 locus and together with Pou5f1 and SoxB1 initiate miR-430 expression and activity. Our results demonstrate that maternal Nanog, Pou5f1 and SoxB1 are required to initiate the zygotic developmental program and in turn trigger the clearance of the maternal program by activating miR-430 expression.', 'Series_overall_design': 'Wild type and loss-of-function total mRNA sequencing of embryonic transcriptomes pre- and post-MZT; ribosome profiling pre-MZT', 'Series_type': 'Other', 'Series_contributor': 'Antonio,J,Giraldez', 'Series_sample_id': 'GSM1219147', 'Series_contact_name': 'Antonio,J.,Giraldez', 'Series_contact_email': 'antonio.giraldez@yale.edu', 'Series_contact_phone': '203 785 5423', 'Series_contact_laboratory': 'Giraldez Lab', 'Series_contact_department': 'Genetics', 'Series_contact_institute': 'Yale University', 'Series_contact_address': '333 Cedar Street', 'Series_contact_city': 'New Haven', 'Series_contact_state': 'CT', 'Series_contact_zip/postal_code': '06510', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE47nnn/GSE47558/suppl/GSE47558_total_RNA_intron_rpkm.txt.gz', 'Series_platform_id': 'GPL14875', 'Series_platform_taxid': '7955', 'Series_sample_taxid': '7955', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP023492'}\n",
      "38 PRJNA211752 GSE48785 {'Series_title': 'Global response to chemotherapy-induced apoptosis', 'Series_geo_accession': 'GSE48785', 'Series_status': 'Public on Oct 17 2013', 'Series_submission_date': 'Jul 11 2013', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '24171104', 'Series_summary': 'Here we use an integrated systems-level examination of transcription, translation, and proteolysis to explore how cancer cells struggle with a chemotherapeutic drug prior to succumbing to apoptosis. As a model system we study myeloma cells exposed to the proteasome inhibitor bortezomib, a first-line clinical treatment. Despite robust transcriptional changes, unbiased quantitative proteomics detects production of only a few critical anti-apoptotic proteins against a background of general translation inhibition. Ribosome profiling further reveals potential translational regulation of stress response genes following bortezomib treatment. Once the apoptotic machinery is engaged, degradation by caspases is largely independent of changes at the transcriptional level. Moreover, previously uncharacterized non-caspase proteolytic events also participate in cellular deconstruction. As suggested by these data, we find that inhibition of the anti-apoptotic response regulator HSF1 promotes cell death by bortezomib. Thus, monitoring global cellular dynamics after chemotherapy offers in-depth insight into apoptosis and can also guide potential therapeutic combinations.', 'Series_overall_design': 'We examined MM1.S myeloma cells exposed to 20 nM bortezomib across a time course with independent samples of poly(A) mRNA and ribosome footprints isolated at each of six time points (0h (untreated), 1.5h, 3h, 6h, 9h, 12h). Sequencing was performed on a Illumina HiSeq 2000 with single-end.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'James,A,Wells', 'Series_sample_id': 'GSM1184602', 'Series_contact_name': 'Arun,P.,Wiita', 'Series_contact_department': 'Laboratory Medicine', 'Series_contact_institute': 'University of California, San Francisco', 'Series_contact_address': '185 Berry St., Suite 290', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94107', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE48nnn/GSE48785/suppl/GSE48785_Sequencing_counts.xlsx.gz', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP027015'}\n",
      "39 PRJNA213675 GSE49339 {'Series_title': 'm6A-dependent regulation of messenger RNA stability', 'Series_geo_accession': 'GSE49339', 'Series_status': 'Public on Nov 29 2013', 'Series_submission_date': 'Jul 30 2013', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '24284625', 'Series_summary': 'N6-methyladenosine (m6A) is the most prevalent internal modification present in the mRNA of all higher eukaryotes. Here we present that m6A is selectively recognized by human YTH domain family (YTHDF2) protein to regulate mRNA degradation. By using crosslinking and immunoprecipitation, we have identified over 4000 substrate RNA of YTHDF2 with conserved core motif of G(m6A)C. We further estabilshed the role of YTHDF2 in RNA metabolism by a combination of ribosome profiling, RNA sequencing, m6A level quantification and cell-based imaging: the C-terminal domain of YTHDF2 selectively binds to m6A of mRNA and the N-terminal domain is responsive for localizing mRNA from translatable pool to processing body where mRNA decay occurs.', 'Series_overall_design': 'PAR-CLIP and RIP was used to identify YTHDF2 binding sites followed by ribosome profling and RNA seq to assess the consequences of YTHDF2 siRNA knock-down', 'Series_type': 'Other', 'Series_contributor': 'C,,He', 'Series_sample_id': 'GSM1197633', 'Series_contact_name': 'zhike,,lu', 'Series_contact_email': 'zhikelu@gmail.com', 'Series_contact_department': 'department of chemistry', 'Series_contact_institute': 'University of Chicago', 'Series_contact_address': '5801 South Ellis Avenue', 'Series_contact_city': 'Chicago', 'Series_contact_state': 'IL', 'Series_contact_zip/postal_code': '60637', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE49nnn/GSE49339/suppl/GSE49339_E-mRNA_Lifetime-rep2.xlsx.gz', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP028325'}\n",
      "40 PRJNA219205 GSE50870 {'Series_title': 'A Serine Sensor for Multicellularity in a Bacterium', 'Series_geo_accession': 'GSE50870', 'Series_status': 'Public on Oct 22 2013', 'Series_submission_date': 'Sep 13 2013', 'Series_last_update_date': 'Jul 23 2019', 'Series_pubmed_id': '24347549', 'Series_summary': 'We report the discovery of a simple environmental sensing mechanism for biofilm formation in the bacterium Bacillus subtilis that operates without the involvement of a dedicated RNA or protein. Certain serine codons, the four TCN codons, in the gene for the biofilm repressor SinR caused a lowering of SinR levels under biofilm-inducing conditions. Synonymous substitutions of these TCN codons with AGC or AGT impaired biofilm formation and gene expression. Conversely, switching AGC or AGT to TCN codons upregulated biofilm formation. Genome-wide ribosome profiling showed that ribosome density was higher at UCN codons than at AGC or AGU during biofilm formation. Serine starvation recapitulated the effect of biofilm-inducing conditions on ribosome occupancy and SinR production. As serine is one of the first amino acids to be exhausted at the end of exponential phase growth, reduced translation speed at serine codons may be exploited by other microbes in adapting to stationary phase.', 'Series_overall_design': '4 samples for ribosome profiling and 2 samples for total mRNA profiling', 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'Yunrong,,Chai', 'Series_sample_id': 'GSM1231419', 'Series_contact_name': 'Arvind,R,Subramaniam', 'Series_contact_email': 'rasi@fredhutch.org', 'Series_contact_phone': '6174968433', 'Series_contact_department': 'Molecular and Cellular Biology', 'Series_contact_institute': 'Harvard University', 'Series_contact_address': '52 Oxford St', 'Series_contact_city': 'Cambridge', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02138', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE50nnn/GSE50870/suppl/GSE50870_RAW.tar', 'Series_platform_id': 'GPL26938', 'Series_platform_taxid': '535026', 'Series_sample_taxid': '535026', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP029937'}\n",
      "41 PRJNA219480 GSE50983 {'Series_title': 'MAEL-dependent selective processing of pachytene piRNA precursors and translation of spermiogenic mRNAs', 'Series_geo_accession': 'GSE50983', 'Series_status': 'Public on Jul 25 2014', 'Series_submission_date': 'Sep 18 2013', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25063675', 'Series_summary': 'Piwi-interacting small RNAs (piRNAs) of fetal prospermatogonia of mice have been strongly implicated in transposon control. In contrast, little is known about biogenesis and function of abundant piRNAs from adult testes expressed in late spermatocytes and round spermatids. These so-called \"pachytene\" piRNAs are processed from long non-coding piRNA precursors and have no defined RNA targets in the transcriptome even though their binding partner Piwi, MIWI, is essential for spermiogenesis and fertility. Here we report that 129SvJae mice lacking Maelstrom (MAEL), a conserved piRNA pathway protein, exhibit spermiogenic arrest with defects in acrosome and flagellum formation. Further analysis revealed MAEL association with RNPs containing MIWI, TDRD6, and processed  intermediates of pachytene piRNA precursors of various length. Loss of MAEL causes a 10-fold drop in pachytene piRNA levels but an increase in piRNAs from abundantly expressed mRNAs. These results suggest a MAEL-dependent mechanism for the selective processing of pachytene piRNA precursor into piRNAs. Strikingly, ribosome profiling of Mael-null testes revealed that reduced piRNA production is accompanied by reduced translation of over 800 spermiogenic mRNAs including those encoding acrosome and flagellum proteins. In light of recent reports of piRNA-independent protection of translationally repressed mRNPs by MIWI and piRNA-dependent turnover of MIWI, we propose that pachytene piRNAs function by controlling the availably of MIWI for the translational repression of spermiogenic mRNAs.', 'Series_overall_design': 'piRNA sequencing, RNA immunoprecipitation, and expression measurements (RNA-Seq and ribosome profiling) in wild-type and Mael -/- testes', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Alex,,Bortvin', 'Series_sample_id': 'GSM1234255', 'Series_contact_name': 'Nicholas,T,Ingolia', 'Series_contact_email': 'nick@ingolia.org', 'Series_contact_phone': '410 246 3025', 'Series_contact_laboratory': 'Ingolia', 'Series_contact_department': 'Embryology', 'Series_contact_institute': 'Carnegie Institution', 'Series_contact_address': '3520 San Martin Drive', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21218', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE50nnn/GSE50983/suppl/GSE50983_RAW.tar', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP030014'}\n",
      "42 PRJNA221377 GSE51164 {'Series_title': 'Ribosome profiling of whi3 mutant yeast', 'Series_geo_accession': 'GSE51164', 'Series_status': 'Public on Sep 26 2013', 'Series_submission_date': 'Sep 25 2013', 'Series_last_update_date': 'Jun 28 2021', 'Series_pubmed_id': '24386402', 'Series_summary': 'In general, RNA-binding proteins act to modulate gene expression at transcript level through degradation or at protein level through translation.  To elucidate the effect of Whi3, a yeast RNA binding protein, on gene expression, we performed ribosome profiling experiment on whi3 mutant and wildtype cells.', 'Series_overall_design': 'Comparison ribosome profiling and RNA-seq data between whi3 mutant and wildtype cells', 'Series_type': 'Other', 'Series_contributor': 'Ying,,Cai', 'Series_sample_id': 'GSM1239962', 'Series_contact_name': 'Ying,,Cai', 'Series_contact_email': 'cai.ying@stonybrook.edu', 'Series_contact_laboratory': 'Bruce Futcher', 'Series_contact_department': 'Molecular Genetics and Microbiology', 'Series_contact_institute': 'Stony Brook University', 'Series_contact_address': 'Life Sciences Building Rm 363', 'Series_contact_city': 'Stony Brook', 'Series_contact_state': 'New York', 'Series_contact_zip/postal_code': '11794', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE51nnn/GSE51164/suppl/GSE51164_RAW.tar', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP030453'}\n",
      "43 PRJNA223060 GSE51424 {'Series_title': 'Cell type-specific ribosome profiling in vivo.', 'Series_geo_accession': 'GSE51424', 'Series_status': 'Public on Aug 25 2014', 'Series_submission_date': 'Oct 18 2013', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25122893', 'Series_summary': 'We report cell-type specific ribosome profiling in a mouse glioma model.', 'Series_overall_design': 'We report a strategy for cell-type specific ribosome profiling in vivo. Our strategy was applied to the characterization of a mouse glioma model.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Nicholas,,Hornstein', 'Series_sample_id': 'GSM1495253', 'Series_contact_name': 'Peter,,Sims', 'Series_contact_email': 'pas2182@columbia.edu', 'Series_contact_institute': 'Columbia Medical Center', 'Series_contact_address': '3960 Broadway blvd', 'Series_contact_city': 'New York', 'Series_contact_state': 'NEW YORK', 'Series_contact_zip/postal_code': '10032', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE51nnn/GSE51424/suppl/GSE51424_RAW.tar', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP031501'}\n",
      "44 PRJNA223608 GSE51584 {'Series_title': 'Expanded identification and characterization of mammalian circular RNAs', 'Series_geo_accession': 'GSE51584', 'Series_status': 'Public on Jul 08 2014', 'Series_submission_date': 'Oct 23 2013', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25070500', 'Series_summary': 'The recent reports of two circular RNAs (circRNAs) with strong potential to act as microRNA (miRNA) sponges suggest that circRNAs might play important roles in regulating gene expression. However, the global properties of circRNAs are not well understood. We developed a computational pipeline to identify circRNAs and quantify their relative abundance from RNA-seq data. Applying this pipeline to a large set of non-poly(A)-selected RNA-seq data from the ENCODE project, we annotated 7,112 human circRNAs that were estimated to comprise at least 10% of the transcripts accumulating from their loci. Most circRNAs are expressed in only a few cell types and at low abundance, but they are no more cell-type–specific than are mRNAs with similar overall expression levels. Although most circRNAs overlap protein-coding sequences, ribosome profiling provides no evidence for their translation. We also annotated 635 mouse circRNAs, and although 20% of them are orthologous to human circRNAs, the sequence conservation of these circRNA orthologs is no higher than that of their flanking linear exons. The previously proposed miR-7 sponge, CDR1as, is one of only two circRNAs with more miRNA sites than expected by chance, with the next best miRNA-sponge candidate deriving from a primate-specific zinc-finger gene, ZNF91. These results provide a new framework for future investigation of this intriguing topological isoform while raising doubts regarding a biological function of most circRNAs.', 'Series_overall_design': 'Note: The ENCODE data we used are under GEO SuperSeries GSE26284 (all samples labeled \"_cell_total\"). But they were not used in the processing of the U2OS data.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'David,P,Bartel', 'Series_sample_id': 'GSM1248736', 'Series_contact_name': 'Stephen,,Eichhorn', 'Series_contact_email': 'eichhorn@wi.mit.edu', 'Series_contact_laboratory': 'Bartel', 'Series_contact_department': 'Biology', 'Series_contact_institute': 'Whitehead Institute for Biomedical Research', 'Series_contact_address': '9 Cambridge Center', 'Series_contact_city': 'Cambridge', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02142', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE51nnn/GSE51584/suppl/GSE51584_U2OS.txt.gz', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP031849'}\n",
      "45 PRJNA229103 GSE52447 {'Series_title': 'Transcriptome-wide mapping of human Staufen1 binding sites', 'Series_geo_accession': 'GSE52447', 'Series_status': 'Public on Nov 25 2013', 'Series_submission_date': 'Nov 18 2013', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '24336223', 'Series_summary': 'Staufen1 (Stau1) is a double-stranded RNA (dsRNA) binding protein implicated in mRNA transport, regulation of translation, mRNA decay and stress granule homeostasis. Here we combined RNA-Immunoprecipitation in Tandem (RIPiT) with RNase footprinting, formaldehyde crosslinking, sonication-mediated RNA fragmentation and deep sequencing to map Staufen1 binding sites transcriptome-wide. We find that Stau1 binds complex secondary structures containing multiple short helices, many of which are formed by inverted Alu elements in annotated 3\\'UTRs or in \"strongly distal\" 3\\'UTRs extending far beyond the canonical polyadenylation signal. Stau1 also interacts with both actively translating ribosomes and with mRNA coding sequences (CDS) and 3\\'UTRs in proportion to their GC-content and internal secondary structure-forming propensity.  On mRNAs with high CDS GC-content, higher Stau1 levels lead to greater ribosome densities, suggesting a general role for Stau1 in modulating the ability of ribosomes to elongate through secondary structures located in CDS regions.', 'Series_overall_design': 'We used HEK293 cells expressing near endogenous levels of wild-type Flag-Stau1 (65KDa isoform with an N-Terminal Flag tag). As a control we used a mutant version of Stau1 that is not functional for dsRNA binding. Formaldehyde crosslinking experiments and RNase footprinting experiments were done in two biological replicates. All RNASeq, Ribosome footprinting and PAS-Seq were done in two biological replicates.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Melissa,J,Moore', 'Series_sample_id': 'GSM1267863', 'Series_contact_name': 'Alper,,Kucukural', 'Series_contact_email': 'alper.kucukural@umassmed.edu', 'Series_contact_phone': '7743124493', 'Series_contact_laboratory': 'Biocore', 'Series_contact_department': 'Program in Molecular Medicine', 'Series_contact_institute': 'UMass Medical School', 'Series_contact_address': '364 Plantation Street', 'Series_contact_city': 'Worcester', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '01605', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE52nnn/GSE52447/suppl/GSE52447_Stau1-WT_Sonication_Formaldehyde.bw', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP033119'}\n",
      "46 PRJNA230112 GSE52809 {'Series_title': 'Poly(A)-tail profiling reveals an embryonic switch in translational control', 'Series_geo_accession': 'GSE52809', 'Series_status': 'Public on Jan 29 2014', 'Series_submission_date': 'Nov 27 2013', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '24476825', 'Series_summary': \"Poly(A) tails enhance the stability and translation of most eukaryotic messenger RNAs, but difficulties in globally measuring poly(A)-tail lengths have impeded greater understanding of poly(A)-tail function. Here we describe poly(A)-tail length profiling by sequencing (PAL-seq) and apply it to measure tail lengths of millions of individual RNAs isolated from yeasts, cell lines, Arabidopsis thaliana leaves, mouse liver, and zebrafish and frog embryos. Poly(A)-tail lengths were conserved between orthologous mRNAs, with mRNAs encoding ribosomal proteins and other 'housekeeping' proteins tending to have shorter tails. As expected, tail lengths were coupled to translational efficiencies in early zebrafish and frog embryos. However, this strong coupling diminished at gastrulation and was absent in non-embryonic samples, indicating a rapid developmental switch in the nature of translational control. This switch complements an earlier switch to zygotic transcriptional control and explains why the predominant effect of microRNA mediated deadenylation concurrently shifts from translational repression to mRNA destabilization.\", 'Series_overall_design': '64 samples from a variety of species', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'David,P,Bartel', 'Series_sample_id': 'GSM1316827', 'Series_contact_name': 'Stephen,,Eichhorn', 'Series_contact_email': 'eichhorn@wi.mit.edu', 'Series_contact_laboratory': 'Bartel', 'Series_contact_department': 'Biology', 'Series_contact_institute': 'Whitehead Institute for Biomedical Research', 'Series_contact_address': '9 Cambridge Center', 'Series_contact_city': 'Cambridge', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02142', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE52nnn/GSE52809/suppl/GSE52809_Xla_st9.txt.gz', 'Series_platform_id': 'GPL17682', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP033369'}\n",
      "47 PRJNA230577 GSE52968 {'Series_title': 'Ribosome profiling study of dom34 and hbs1 knockout strains using short (16-nt) and long (28-nt) monosome-protected footprints and disome-protected footprints', 'Series_geo_accession': 'GSE52968', 'Series_status': 'Public on Dec 19 2013', 'Series_submission_date': 'Dec 04 2013', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '24581494', 'Series_summary': 'Ribosomes that stall before completing peptide synthesis must be recycled and returned to the cytoplasmic pool. The protein Dom34 and cofactors Hbs1 and Rli1 can dissociate stalled ribosomes in vitro, but the identity of targets in the cell is unknown. Here we use ribosome profiling methodology to reveal a high- resolution molecular characterization of Dom34 function in vivo. We show that Dom34 removes stalled ribosomes from mRNAs that are truncated but, in contrast, does not generally dissociate ribosomes on coding sequences known to trigger stalling, such as polyproline. We also show that Dom34 targets arrested ribosomes near the ends of 3 ́ UTRs. These ribosomes appear to gain access to the 3 ́ UTR via a mechanism that does not require decoding of the mRNA. These results suggest that Dom34 carries out the important task of rescuing ribosomes found in noncoding regions.', 'Series_overall_design': '25 samples are included in the study (2 mRNA-Seq samples and 23 ribosome footprint profiling samples). These include wild-type and dom34 or hbs1 knockout strains that were created in a variety of genetic backgrounds, treated with various agents in cell culture (e.g. diamide, 3-AT, or glucose starvation), treated differently during cell lysis (use of cycloheximide vs. other ribosome-stabilizing agents), or prepared in different ways after cell lysis (e.g. retention of short vs. long monosome-protected footprints or disome footprints).', 'Series_type': 'Other', 'Series_contributor': 'Rachel,,Green', 'Series_sample_id': 'GSM1279590', 'Series_contact_name': 'Nicholas,R,Guydosh', 'Series_contact_email': 'nicholas.guydosh@nih.gov', 'Series_contact_laboratory': 'LBG', 'Series_contact_department': 'NIDDK', 'Series_contact_institute': 'National Institutes of Health', 'Series_contact_address': '8 Center Dr, Bldg 8, 220', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '20892', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE52nnn/GSE52968/suppl/GSE52968_wt_combined_plus.wig.gz', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP033499'}\n",
      "48 PRJNA230610 GSE52976 {'Series_title': 'Integrative AUF1 PAR-CLIP analysis uncovers AUF1 roles in translation and genome integrity (Ribosome)', 'Series_geo_accession': 'GSE52976', 'Series_status': 'Public on Nov 04 2014', 'Series_submission_date': 'Dec 04 2013', 'Series_last_update_date': 'Jun 22 2020', 'Series_pubmed_id': '25366541', 'Series_summary': 'We report that AUF1 modulates global mRNA stability and translation, in turn promoting the maintenance of DNA integrity.', 'Series_overall_design': 'Please see individual series. In short, For AUF1 PAR-CLIP, the four isoforms of AUF1 (p37, p40, p42, and p45) tagged with a Flag epitope were expressed in HEK293 cells. For total RNA-Seq HEK293 cells were transfected with Control siRNA, AUF1 siRNA, Empty Vector, Flag-AUF1 p37, p40, p42, or p45 as well as WI-38 cells were collected at PDL 15 and 55 and also transfected with Control siRNA, AUF1 siRNA, HuR siRNA. For Ribo-Seq HeLa cells were transfected with Control siRNA, AUF1 siRNA, or HuR siRNA.', 'Series_type': 'Other', 'Series_contributor': 'Myriam,,Gorospe', 'Series_sample_id': 'GSM1279709', 'Series_contact_name': 'Supriyo,,De', 'Series_contact_laboratory': 'Computational Biology & Genomics Core', 'Series_contact_department': 'Laboratory of Genetics and Genomics', 'Series_contact_institute': 'NIA-IRP, NIH', 'Series_contact_address': '251 Bayview Blvd', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'Maryland', 'Series_contact_zip/postal_code': '21224', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE52nnn/GSE52976/suppl/GSE52976_RAW.tar', 'Series_platform_id': 'GPL9115', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP033503'}\n",
      "49 PRJNA232843 GSE53767 {'Series_title': 'Absolute quantification of protein production reveals principles underlying protein synthesis rates', 'Series_geo_accession': 'GSE53767', 'Series_status': 'Public on Apr 24 2014', 'Series_submission_date': 'Jan 02 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '24766808', 'Series_summary': 'A quantitative view of cellular functions requires precise measures of the rates of biomolecule production, especially proteins-the direct effectors of biological processes. Here we present a genome-wide approach, based on ribosome profiling, for measuring absolute protein synthesis rates. The resultant E. coli dataset transforms our understanding of the extent to which protein synthesis is precisely controlled to optimize function and efficiency. For example, members of multi-protein complexes are made in precise proportion to their stoichiometry, whereas components of functional modules are produced differentially according to their hierarchical role. Estimates of absolute protein abundance also reveal principles used to optimize design. These include how the level of different types of transcription factors is optimized for rapid response, and how a metabolic pathway (methionine biosynthesis) balances production cost with activity requirements. More broadly, our studies reveal how general principles, important both for understanding natural systems and for synthesizing new ones, emerge from global quantitative analyses of protein synthesis.', 'Series_overall_design': '4 samples of E. coli ribosome profiling and mRNA-seq, including biological replicates', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Jonathan,S,Weissman', 'Series_sample_id': 'GSM1300282', 'Series_contact_name': 'Gene-Wei,,Li', 'Series_contact_email': 'gene-wei.li@ucsf.edu', 'Series_contact_institute': 'UCSF', 'Series_contact_address': '1700 4th Street, BH404', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE53nnn/GSE53767/suppl/GSE53767_mrna-rdm-pooled_r.wig.gz', 'Series_platform_id': 'GPL18133', 'Series_platform_taxid': '562', 'Series_sample_taxid': '562', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP034871'}\n",
      "50 PRJNA237963 GSE54883 {'Series_title': 'The coding and noncoding architecture of the Caulobacter crescentus genome', 'Series_geo_accession': 'GSE54883', 'Series_status': 'Public on May 29 2014', 'Series_submission_date': 'Feb 11 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25078267', 'Series_summary': 'Caulobacter crescentus undergoes an asymmetric cell division controlled by a genetic circuit that cycles in space and time.  We provide a universal strategy for defining the coding potential of bacterial genomes by applying ribosome profiling, RNA-seq, global 5’-RACE, and liquid chromatography coupled with tandem mass spectrometry (LC-MS) data to the 4-megabase C. crescentus genome.  We mapped transcript units at single base-pair resolution using RNA-seq together with global 5’ RACE.  Additionally, using ribosome profiling and LC-MS, we mapped translation start sites and coding regions with near complete coverage.  We found most start codons lacked corresponding Shine-Dalgarno sites although ribosomes were observed to pause at internal Shine-Dalgarno sites within the ORF.  These data suggest a more prevalent use of the Shine-Dalgarno sequence for ribosome pausing rather than translation initiation in C. crescentus.  Overall 19% of the transcribed and translated genomic elements were newly identified or significantly improved by this approach providing a valuable genomic resource to elucidate the complete C. crescentus genetic circuitry that controls asymmetric cell division.', 'Series_overall_design': 'Ribosome profiling and RNA-seq data were collected in Caulobacter crescentus NA1000 cells grown in M2G and PYE media to map transcript and ORF features in the genome.', 'Series_type': 'Other', 'Series_contributor': 'Lucy,,Shapiro', 'Series_sample_id': 'GSM1326111', 'Series_contact_name': 'Jared,Michael,Schrader', 'Series_contact_email': 'schrader@wayne.edu', 'Series_contact_laboratory': 'Schrader Lab', 'Series_contact_department': 'Biological Sciences', 'Series_contact_institute': 'Wayne State University', 'Series_contact_address': '5047 Gullen Mall, Biological Sciences Building Room 2168', 'Series_contact_city': 'Detroit', 'Series_contact_state': 'MI', 'Series_contact_zip/postal_code': '48202', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE54nnn/GSE54883/suppl/GSE54883_RAW.tar', 'Series_platform_id': 'GPL18276', 'Series_platform_taxid': '565050', 'Series_sample_taxid': '565050', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP037582'}\n",
      "51 PRJNA238879 GSE55195 {'Series_title': 'Ribosome profiling data obtained from HEK293T cells 30 minutes after treatment with arsenite to a final concentration of 40 µM', 'Series_geo_accession': 'GSE55195', 'Series_status': 'Public on Jan 27 2015', 'Series_submission_date': 'Feb 20 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25621764', 'Series_summary': 'Eukaryotic cells rapidly reduce protein synthesis in response to various stress conditions. This can be achieved by the phosphorylation-mediated inactivation of a key translation initiation factor, eIF2. However, the persistent translation of certain mRNAs is required for deployment of an adequate stress response. We carried out ribosome profiling of cultured human cells under conditions of severe stress induced with sodium arsenite. Although this led to a ~4.5-fold general translational repression, the protein coding ORFs of certain individual mRNAs exhibited resistance to the inhibition. Nearly all resistant transcripts possess at least one efficiently translated uORF that repress translation of the main coding ORF under normal conditions. Site specific mutagenesis of two identified stress resistant mRNAs (PPP1R15B and IFRD1) demonstrated that a single uORF is sufficient for eIF2-mediated translation control in both cases. Phylogenetic analysis suggests that at least two regulatory uORFs (namely in SLC35A4 and MIEF1) encode functional protein products.', 'Series_overall_design': 'Ribosome profiling of sodium arsenite treated cells for examination of translational response to induction of eIF2 phosphoylation', 'Series_type': 'Other', 'Series_contributor': 'Pavel,V,Baranov', 'Series_sample_id': 'GSM1331349', 'Series_contact_name': 'Pavel,V,Baranov', 'Series_contact_email': 'p.baranov@ucc.ie', 'Series_contact_laboratory': 'Lapti', 'Series_contact_department': 'Biochemistry and Cell Biology', 'Series_contact_institute': 'University College Cork', 'Series_contact_address': 'Western Gateway Building, UCC', 'Series_contact_city': 'Cork', 'Series_contact_state': 'Co. Cork', 'Series_contact_zip/postal_code': 'NA', 'Series_contact_country': 'Ireland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE55nnn/GSE55195/suppl/GSE55195_SupplementalTable1.xlsx.gz', 'Series_platform_id': 'GPL15520', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP038695'}\n",
      "52 PRJNA244101 GSE56622 {'Series_title': 'Ribosome profiling upon glucose starvation in S. cerevisiae', 'Series_geo_accession': 'GSE56622', 'Series_status': 'Public on Jul 28 2014', 'Series_submission_date': 'Apr 08 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25119046', 'Series_summary': 'A universal feature of the response to stress and nutrient limitation is transcriptional upregulation of genes encoding proteins important for survival.  Interestingly, under many of these conditions overall protein synthesis levels are reduced, thereby dampening the stress response at the level of protein expression. For example, during glucose starvation in yeast, translation is rapidly and reversibly repressed, yet transcription of many stress- and glucose-repressed genes is increased. Using ribosome profiling and microscopy, we found that this transcriptionally upregulated gene set consists of two classes: (1) one producing mRNAs that are preferentially translated during glucose limitation and are diffusely localized in the cytoplasm – this class includes many heat shock protein mRNAs; and (2) another producing mRNAs that are poorly translated during glucose limitation, have high rates of translation initiation, and are concentrated in foci that co-localize with P bodies and stress granules – this class is enriched for glucose metabolism mRNAs. Remarkably, the information specifying differential localization and translation of these two classes of mRNAs is encoded in the promoter sequence – promoter responsiveness to heat shock factor (Hsf1) specifies diffuse cytoplasmic localization and preferential translation upon glucose starvation, whereas different promoter elements upstream of genes encoding poorly translated glucose metabolism mRNAs direct these mRNAs to RNA granules under glucose starvation. Thus, promoter sequences and transcription factor binding can influence not only mRNA levels, but also subcellular localization of mRNAs and the efficiency with which they are translated, enabling cells to tailor protein production to environmental conditions.', 'Series_overall_design': 'Examination of mRNA translation in S. cerevisiae upon glucose starvation.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': \"Erin,K,O'Shea\", 'Series_sample_id': 'GSM1387094', 'Series_contact_name': 'Brian,Matthew,Zid', 'Series_contact_email': 'zid@ucsd.edu', 'Series_contact_laboratory': 'Zid', 'Series_contact_department': 'Chemistry and Biochemistry', 'Series_contact_institute': 'University of California San Diego', 'Series_contact_address': '9500 Gilman Drive', 'Series_contact_city': 'La Jolla', 'Series_contact_state': 'California', 'Series_contact_zip/postal_code': '92093', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE56nnn/GSE56622/suppl/GSE56622_ZidProcessedDataAll.txt.gz', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP041039'}\n",
      "53 PRJNA244865 GSE56887 {'Series_title': 'RNA G-quadruplexes cause eIF4A-dependent oncogene translation in cancer', 'Series_geo_accession': 'GSE56887', 'Series_status': 'Public on Jul 27 2014', 'Series_submission_date': 'Apr 17 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25079319', 'Series_summary': \"The translational control of oncoprotein expression is implicated in many cancers. Here we report an eIF4A/DDX2 RNA helicase-dependent mechanism of translational control that contributes to oncogenesis and underlies the anticancer effects of Silvestrol and related compounds. For example, eIF4A promotes T-ALL development in vivo and is required for leukaemia maintenance. Accordingly, inhibition of eIF4A with Silvestrol has powerful therapeutic effects in vitro and in vivo. We use transcriptome-scale ribosome footprinting to identify the hallmarks of eIF4A-dependent transcripts. These include 5'UTR sequences such as the 12-mer guanine quartet (CGG)4 motif that can form RNA G-quadruplex structures. Notably, among the most eIF4A-dependent and Silvestrol-sensitive transcripts are a number of oncogenes, super-enhancer associated transcription factors, and epigenetic regulators. Hence, the 5'UTRs of selected cancer genes harbour a targetable requirement for the eIF4A RNA helicase.\", 'Series_overall_design': 'Comparison of ribosome-protected RNA for drug treated and DMSO treated KOPT-K1 cell, two replicates of ribosome-protected RNA sequencing and three replicates of RNA-seq.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Hans-Guido,,Wendel', 'Series_sample_id': 'GSM1370704', 'Series_contact_name': 'Yi,,Zhong', 'Series_contact_email': 'zhongyi@sjtu.edu.cn', 'Series_contact_institute': 'SJTU', 'Series_contact_address': '227 South Chongqing Rd.', 'Series_contact_city': 'Shanghai', 'Series_contact_zip/postal_code': '200025', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE56nnn/GSE56887/suppl/GSE56887_RAW.tar', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP041255'}\n",
      "54 PRJNA244941 GSE56924 {'Series_title': 'Ribosome profiling reveals an important role for translational control in circadian gene expression', 'Series_geo_accession': 'GSE56924', 'Series_status': 'Public on Aug 19 2015', 'Series_submission_date': 'Apr 18 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26338483', 'Series_summary': 'Physiological and behavioral circadian rhythms are driven by a conserved transcriptional/translational negative feedback loop in mammals. Although most core clock factors are transcription factors, post-transcriptional control introduces delays that are critical for circadian oscillations. Little work has been done on circadian regulation of translation, so to address this deficit we conducted ribosome profiling experiments in a human cell model for an autonomous clock. We found that most rhythmic gene expression occurs with little delay between transcription and translation, suggesting that the lag in the accumulation of some clock proteins relative to their mRNAs does not arise from regulated translation. Nevertheless, we found that translation occurs in a circadian fashion for many genes, sometimes imposing an additional level of control on rhythmically expressed mRNAs and, in other cases, conferring rhythms on non-cycling mRNAs.  Most cyclically transcribed RNAs are translated at one of two major times in a 24h day, while rhythmic translation of most non-cyclic RNAs is phased to a single time of day. Unexpectedly, we found that the clock also regulates the formation of cytoplasmic processing (P) bodies, which control the fate of mRNAs, suggesting circadian coordination of mRNA metabolism and translation.', 'Series_overall_design': 'U2-OS cell time course over one day of 12 wildtype total RNA-seq samples, 12 wildtype ribosome profiling samples, 12 Bmal1 knockdown total RNA-seq samples and 12 Bmal1 knockdown ribosome profiling samples. All samples have duplicates.', 'Series_type': 'Other', 'Series_contributor': 'Amita,,Sehgal', 'Series_sample_id': 'GSM1371490', 'Series_contact_name': 'Nicholas,,Lahens', 'Series_contact_department': 'ITMAT', 'Series_contact_institute': 'University of Pennsylvania', 'Series_contact_address': 'Smilow Center for Translational Research 10-110 3400 Civic Center Blvd, Bldg 421', 'Series_contact_city': 'Philadelphia', 'Series_contact_state': 'PA', 'Series_contact_zip/postal_code': '19104', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE56nnn/GSE56924/suppl/GSE56924_wt_totalRNA_quant_rep2.txt.gz', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP041298'}\n",
      "55 PRJNA245830 GSE57175 {'Series_title': 'Sequence selectivity of macrolide-induced translational regulation', 'Series_geo_accession': 'GSE57175', 'Series_status': 'Public on Nov 03 2014', 'Series_submission_date': 'Apr 29 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25313041', 'Series_summary': 'We sequenced a total of 12 cDNA libararies derived from fragmented total mRNA and ribosome protected mRNAs from azithromycin-treated and -untreated S. aureus samples. The data represented 2 independent biological replicates.', 'Series_overall_design': 'Examination of the impact of azithromycin on global translatome, mRNA abundance and the ribosome density along the transcripts.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'M,F,Yap', 'Series_sample_id': 'GSM1376717', 'Series_contact_name': 'Mee-Ngan,F.,Yap', 'Series_contact_email': 'frances.yap@northwestern.edu', 'Series_contact_phone': '312-503-3793', 'Series_contact_laboratory': 'Searle 3-430', 'Series_contact_department': 'Microbiology-Immunology', 'Series_contact_institute': 'Northwestern University Feinberg School of Medicine', 'Series_contact_address': '320 E Superior St', 'Series_contact_city': 'Chicago', 'Series_contact_state': 'IL', 'Series_contact_zip/postal_code': '60611', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE57nnn/GSE57175/suppl/GSE57175_RAW.tar', 'Series_platform_id': 'GPL18626', 'Series_platform_taxid': '93061', 'Series_sample_taxid': '93061', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP041549'}\n",
      "56 PRJNA246300 GSE57336 {'Series_title': 'Stage-specific regulation of translation revealed by ribosome profiling of Trypanosoma brucei', 'Series_geo_accession': 'GSE57336', 'Series_status': 'Public on May 24 2014', 'Series_submission_date': 'May 06 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25331479', 'Series_summary': 'Trypanosoma brucei subspecies infect humans and animals in sub-Saharan Africa.  This early diverging eukaryote shows many novel features in basic biological processes, including the use of polycistronic transcription to generate all protein-coding mRNAs.  Therefore we hypothesized that translational control provides a means to tune gene expression during parasite development in mammalian and fly hosts. We used ribosome profiling to examine genome-wide protein production in animal-derived slender bloodstream forms and cultured procyclic (insect midgut) forms.  About one-third of all CDSs showed statistically significant regulation of protein production between the two stages.  Of these, more than two-thirds showed a change in translation efficiency, but few were controlled by this parameter alone.  Ribosomal proteins were translated poorly, especially in animal-derived parasites.  A disproportionate number of metabolic enzymes were up-regulated at the mRNA level in procyclic forms, as were variant surface glycoproteins in bloodstream forms.   Comparison with cultured bloodstream forms from another strain identified stage-specific changes in protein production that transcend strain and growth conditions.  Genes with upstream ORFs had a lower mean translation efficiency but no evidence was seen for involvement of these ORFs in stage-regulation. Ribosome profiling revealed that differences in the production of specific proteins in T. brucei slender bloodstream and procyclic forms are more common than anticipated from analysis of mRNA abundance.  While in vivo and in vitro derived slender bloodstream forms of different strains are more similar to one another than to procyclic forms, they showed numerous differences at both the mRNA and protein production level.', 'Series_overall_design': 'Ribosome profiling and mRNA libraries were constructed in triplicate from in vitro  PCF and in vivo  BF lifestages of theT. brucei Treu927 and in vitro T. brucei  Lister427, to evaluate role of translational gene regulation', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Marilyn,,Parsons', 'Series_sample_id': 'GSM1379717', 'Series_contact_name': 'Gowthaman,,Ramasamy', 'Series_contact_email': 'Gowthaman.Ramasamy@seattlebiomed.org', 'Series_contact_phone': '206-256-7188', 'Series_contact_laboratory': 'Bioinformatics Core & Myler Lab', 'Series_contact_department': 'GHBC', 'Series_contact_institute': 'SEATTLE BIOMEDICAL RESEARCH INSTITUTE', 'Series_contact_address': '307 WESTLAKE AVE N', 'Series_contact_city': 'SEATTLE', 'Series_contact_state': 'WASHINGTON', 'Series_contact_zip/postal_code': '98109', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE57nnn/GSE57336/suppl/GSE57336_RAW.tar', 'Series_platform_id': 'GPL10599', 'Series_platform_taxid': '5691', 'Series_sample_taxid': '5691', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP041705'}\n",
      "57 PRJNA251587 GSE58207 {'Series_title': 'PROTEOFORMER: deep proteome coverage through ribosome profiling and MS integration', 'Series_geo_accession': 'GSE58207', 'Series_status': 'Public on Dec 01 2014', 'Series_submission_date': 'Jun 04 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25510491', 'Series_summary': 'An increasing amount of studies integrate mRNA sequencing data into MS-based proteomics to complement the translation product search space. We present the generation of a protein synthesis-based database from deep sequencing of ribosome-protected mRNA fragments. This approach increases the overall protein identification rates with 3% and 11% (improved and new identifications) for human and mouse respectively and enables proteome-wide detection of 5’-extended proteoforms, uORF translation and near-cognate translation start sites.', 'Series_overall_design': 'Ribosome profiling of lactimidomycin and cycloheximide treated HCT116 cells', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'D,,Gawron', 'Series_sample_id': 'GSM1403308', 'Series_contact_name': 'Gerben,,Menschaert', 'Series_contact_email': 'gerben.menschaert@ugent.be', 'Series_contact_laboratory': 'BioBix, Lab for Bioinformatics and Computational Genomics', 'Series_contact_institute': 'Ghent University, Faculty of BioScience Engineering', 'Series_contact_address': 'Coupure Links 653', 'Series_contact_city': 'Ghent', 'Series_contact_zip/postal_code': '9000', 'Series_contact_country': 'Belgium', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE58nnn/GSE58207/suppl/GSE58207_RAW.tar', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP042937'}\n",
      "58 PRJNA252511 GSE58402 {'Series_title': 'Ribosome Profiling in P. falciparum asexual blood stages', 'Series_geo_accession': 'GSE58402', 'Series_status': 'Public on Dec 01 2014', 'Series_submission_date': 'Jun 11 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25493618', 'Series_summary': 'The description of the transcriptome and proteome datasets of Plasmodium falciparum, the deadliest strain of human malaria, has been a tremendous resource for the understanding of the molecular physiology of this parasite. However, the underlying translational control that links global measurements of steady-state mRNA with protein levels is not well understood. Our work bridges this disconnect by measuring translation on a whole genome scale using ribosome profiling, providing the first measurements of new protein synthesis through the asexual blood phase developmental cycle.', 'Series_overall_design': 'Cultures corresponding to ring, early trophozoite, late trophozoite, schizont stage, merozoite were harvested for both mRNA and polysome isolation using cycloheximide to freeze ribosomes in place.', 'Series_type': 'Other', 'Series_contributor': 'Joseph,L,DeRisi', 'Series_sample_id': 'GSM1410300', 'Series_contact_name': 'Joseph,,DeRisi', 'Series_contact_laboratory': 'DeRisi', 'Series_contact_department': 'Biochemistry', 'Series_contact_institute': 'UCSF', 'Series_contact_address': '1700 4th Street', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94134', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE58nnn/GSE58402/suppl/GSE58402_RAW.tar', 'Series_platform_id': 'GPL16607', 'Series_platform_taxid': '5833', 'Series_sample_taxid': '5833', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP043116'}\n",
      "59 PRJNA252579 GSE58423 {'Series_title': 'Musashi proteins are post-transcriptional regulators of the epithelial-luminal cell state', 'Series_geo_accession': 'GSE58423', 'Series_status': 'Public on Jun 13 2014', 'Series_submission_date': 'Jun 12 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25380226', 'Series_summary': 'mRNA-seq and ribosome profiling of neural stem cells overexpressing or knocked out for Musashi RNA-binding proteins', 'Series_overall_design': 'Study of the global effects of Musashi (Msi) proteins on the transcriptome of embryonic neural stem cells. Neural stem cells were derived from brains of E12.5 or E13.5 embryos engineered to have inducible Msi1 or Msi2 genes, or from embryos with double floxed alleles of Msi1 and Msi2 carrying a Tamoxifen-induclble Cre (CreER). The overexpression mice were made using the Flp-in system (OpenBioSystems), where a cDNA of interest (in this case Msi1 or Msi2) is knocked into the Collagen (Col1A1) locus. The expression of the cDNA of interest is driven by m2rTTA that is knocked into the Rosa26 locus (R26). KH2 describes a strain containing the R26-m2rTTA but lacking Msi1 or Msi2 cDNA. MSI1 describes a strain containing R26-m2rTTA and Msi1 cDNA in Col1A1. MSI2 describes a strain containing R26-m2rTTA and Msi2 cDNA in Col1A1. C1 describes a strain lacking the CreER allele but containing double floxed alleles of Msi1/Msi2 (used as Tamoxifen control). C4 describes a strain carrying the CreER allele and double floxed alleles of Msi1/Msi2.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Christopher,B,Burge', 'Series_sample_id': 'GSM1410675', 'Series_contact_name': 'Yarden,,Katz', 'Series_contact_laboratory': 'Jaenisch/Burge lab', 'Series_contact_department': 'Dept. of Biology and Whitehead Institute', 'Series_contact_institute': 'MIT', 'Series_contact_address': '9 Cambridge Center', 'Series_contact_city': 'Cambridge', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02142', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE58nnn/GSE58423/suppl/GSE58423_ribo_oe1_ensGene.cds_only.rpkm.txt.gz', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP043153'}\n",
      "60 PRJNA253056 GSE58637 {'Series_title': 'MicL, a new σE-dependent sRNA, combats envelope stress by repressing synthesis of Lpp, the major outer membrane lipoprotein', 'Series_geo_accession': 'GSE58637', 'Series_status': 'Public on Jul 17 2014', 'Series_submission_date': 'Jun 18 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25030700', 'Series_summary': 'In enteric bacteria, the transcription factor σE maintains membrane homeostasis by inducing expression of proteins involved in membrane repair and of two small, regulatory RNAs (sRNAs) that downregulate synthesis of abundant membrane porins. Here, we describe the discovery of a third σE-dependent sRNA, MicL, transcribed from a promoter located within the coding sequence of the cutC gene. MicL is synthesized as a 308 nt primary transcript that is processed to an 80 nt form. Both forms possess features typical of Hfq-binding sRNAs, but surprisingly only target a single mRNA, which encodes the outer membrane lipoprotein Lpp, the most abundant protein of the cell. We show that the copper sensitivity phenotype previously ascribed to inactivation of the cutC gene is actually derived from the loss of MicL and elevated Lpp levels. This observation raises the possibility that other phenotypes currently attributed to protein defects are due to deficiencies in unappreciated regulatory RNAs. We also report that σE activity is sensitive to Lpp abundance and that MicL and Lpp comprise a new σE regulatory loop that opposes membrane stress. Together MicA, RybB and MicL allow σE to repress the expression of all abundant outer membrane proteins in response to stress.', 'Series_overall_design': '12 samples mRNA-seq data, 2 samples ribosome profiling data.  For mRNA-seq data, samples were gathered at the indicated time (in min) after induction of either vector (WT), long (MicL), and short (MicL-S) forms of MicL.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Gisela,,Storz', 'Series_sample_id': 'GSM1430432', 'Series_contact_name': 'Carol,A,Gross', 'Series_contact_email': 'cgross@cgl.ucsf.edu', 'Series_contact_institute': 'UCSF', 'Series_contact_address': '600 16th St, S376', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE58nnn/GSE58637/suppl/GSE58637_RAW.tar', 'Series_platform_id': 'GPL14548', 'Series_platform_taxid': '562', 'Series_sample_taxid': '562', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP043387'}\n",
      "61 PRJNA254361 GSE59095 {'Series_title': 'DDX3X regulation of global translation is impaired by medulloblastoma-associated mutations [RiboSeq]', 'Series_geo_accession': 'GSE59095', 'Series_status': 'Public on May 20 2016', 'Series_submission_date': 'Jul 03 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27180681', 'Series_summary': 'Whole-genome sequencing recently identified recurrent missense mutations in the RNA helicase DDX3X in pediatric medulloblastoma (MB) and other tumors. The normal function of DDX3X is poorly understood, and the consequences of its cancer-associated mutations have not been explored. Here we used genomic, biochemical, cell biological, and animal modeling approaches to investigate normal DDX3X function and the impact of cancer-associated DDX3X mutations. Cross-linking immunoprecipitation–high-throughput sequencing (CLIPseq) analyses revealed that DDX3X binds primarily to ~1000 mature mRNA targets at binding sites spanning the full mRNA length; their enrichment in the coding regions suggests that DDX3X plays a role in translational elongation. The association of wild-type DDX3X with polysomes is consistent with this observation. Cancer-associated mutations result in loss of DDX3X from polysomes and accumulation of mutant DDX3X in stress granules (cytoplasmic accumulations of translationally arrested mRNAs). Mutation-dependent redistribution of DDX3X to stress granules is also observed in a Drosophila model system and in MB tumor cells from patients carrying DDX3X mutations. Importantly, mRNAs targeted by DDX3X are enriched in translation factors, suggesting that DDX3X regulates translation both directly and indirectly. Indeed, depletion of DDX3X by RNAi or over-expression of mutant DDX3X significantly impairs global protein synthesis. Ribosome profiling confirmed this observation and showed a 5’ bias in ribosomal occupancy, further confirming the role of DDX3X in translational elongation. Together, our data show that DDX3X is a key regulator of translation and that this function is impaired by cancer-associated mutations. Finally, we found that medulloblastoma-related mutant DDX3X can efficiently bind the wild-type form suggesting that mutant DDX3X could exert a dominant negative effect in vivo.', 'Series_overall_design': 'Examination of translation at whole genome scale under conditions of overexpression of WT DDX3X or cancer-associated DDX3X mutant G325E', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'J,P,Taylor', 'Series_sample_id': 'GSM1427364', 'Series_contact_name': 'Yong-Dong,,Wang', 'Series_contact_email': 'michael.wang@stjude.org', 'Series_contact_phone': '9015954224', 'Series_contact_institute': \"St. Jude Children's Research Hospital\", 'Series_contact_address': '262 Danny Thomas Pl, MS1135', 'Series_contact_city': 'Memphis', 'Series_contact_state': 'TN', 'Series_contact_zip/postal_code': '38134', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE59nnn/GSE59095/suppl/GSE59095_RAW.tar', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP044064'}\n",
      "62 PRJNA257365 GSE60040 {'Series_title': 'Wide-spread disruption of transcription termination in HSV-1 infection: Next-generation sequencing of translational activityd by ribosome profiling', 'Series_geo_accession': 'GSE60040', 'Series_status': 'Public on May 26 2015', 'Series_submission_date': 'Aug 04 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25989971', 'Series_summary': 'Primary human foreskin fibroblasts (HFF) were infected with wild-type simplex virus 1 (HSV-1) strain 17 at a multiplicity of infection (MOI) of 10. Ribosome profiling was performed at various times during infection with minor modification to the protocol described in Stern-Ginossar N et al., Science 2012', 'Series_overall_design': 'Ribosome profiling was performed a 0, 1, 2, 4, 6 and 8 h post infection. Two biological replicates were analysed.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Lars,,Dölken', 'Series_sample_id': 'GSM1464106', 'Series_contact_name': 'Caroline,,Friedel', 'Series_contact_department': 'Institut für Informatik', 'Series_contact_institute': 'Ludwig-Maximilians-Universität München', 'Series_contact_address': 'Amalienstr. 17', 'Series_contact_city': 'München', 'Series_contact_zip/postal_code': '80333', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE60nnn/GSE60040/suppl/GSE60040_RAW.tar', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP045214'}\n",
      "63 PRJNA257463 GSE60095 {'Series_title': 'Ribosome Profiling Reveals Pervasive Translation Outside of Annotated Protein-Coding Genes', 'Series_geo_accession': 'GSE60095', 'Series_status': 'Public on Aug 21 2014', 'Series_submission_date': 'Aug 05 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25159147', 'Series_summary': \"Ribosome profiling suggests that ribosomes occupy many regions of the transcriptome thought to be non-coding, including 5' UTRs and lncRNAs. Apparent ribosome footprints outside of protein-coding regions raise the possibility of artifacts unrelated to translation, particularly when they occupy multiple, overlapping open reading frames (ORFs). Here we show hallmarks of translation in these footprints: co-purification with the large ribosomal subunit, response to drugs targeting elongation, trinucleotide periodicity, and initiation at early AUGs. We develop a metric for distinguishing between 80S footprints and nonribosomal sources using footprint size distributions, which validates the vast majority of footprints outside of coding regions. We present evidence for polypeptide production beyond annotated genes, including induction of immune responses following human cytomegalovirus (HCMV) infection. Translation is pervasive on cytosolic transcripts outside of conserved reading frames, and direct detection of this expanded universe of translated products enables efforts to understand how cells manage and exploit its consequences.\", 'Series_overall_design': 'Ribosome profiling to verify that true ribosome footprints shift in response to different elongation inhibitors (CHX vs Emetine) and co-purify with an affinity-tagged large ribosomal subunit (bound vs input)', 'Series_type': 'Other', 'Series_contributor': 'Jonathan,S,Weissman', 'Series_sample_id': 'GSM1464904', 'Series_contact_name': 'Nicholas,T,Ingolia', 'Series_contact_email': 'nick@ingolia.org', 'Series_contact_phone': '410 246 3025', 'Series_contact_laboratory': 'Ingolia', 'Series_contact_department': 'Embryology', 'Series_contact_institute': 'Carnegie Institution', 'Series_contact_address': '3520 San Martin Drive', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21218', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE60nnn/GSE60095/suppl/GSE60095_RAW.tar', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP045257'}\n",
      "64 PRJNA258156 GSE60384 {'Series_title': 'Extensive translation of small Open Reading Frames revealed by Poly-Ribo-Seq', 'Series_geo_accession': 'GSE60384', 'Series_status': 'Public on Sep 11 2014', 'Series_submission_date': 'Aug 13 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25144939', 'Series_summary': 'We present a genome-wide assessment of small open reading frames (smORF) translation by ribosomal profiling of polysomal fractions in Drosophila S2 cell. In this way, mRNAs bound by multiple ribosomes and hence actively translated can be isolated and distinguished from mRNAs bound by sporadic, putatively non-productive single ribosomes or ribosomal subunits.', 'Series_overall_design': 'Ribosomal profiling of large and small polysomal fractions in Drosophila S2 cells to assess translation of smORFs', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Juan,P,Couso', 'Series_sample_id': 'GSM1477474', 'Series_contact_name': 'Juan Pablo,,Couso', 'Series_contact_email': 'j.p.couso@sussex.ac.uk', 'Series_contact_institute': 'University of Sussex', 'Series_contact_address': 'Falmer', 'Series_contact_city': 'Brighton', 'Series_contact_zip/postal_code': 'BN1 9QG', 'Series_contact_country': 'United Kingdom', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE60nnn/GSE60384/suppl/GSE60384_S2.ribo.uorf.txt.gz', 'Series_platform_id': 'GPL16479', 'Series_platform_taxid': '7227', 'Series_sample_taxid': '7227', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP045475'}\n",
      "65 PRJNA259424 GSE60752 {'Series_title': 'Time resolved ribosome profiling study of oxygen and glucose deprivation of rat pheochromocytoma cells', 'Series_geo_accession': 'GSE60752', 'Series_status': 'Public on Apr 10 2015', 'Series_submission_date': 'Aug 26 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25943107', 'Series_summary': 'Oxygen and glucose metabolism plays a pivotal role in many (patho)physiological conditions. In particular, oxygen and glucose deprivation (OGD) occurs during ischemia and stroke, resulting in extensive tissue injury and cell death. We applied time-resolved ribosome profiling technique to assess early events at the level of gene expression in rat pheochromocytoma PC12 cells during short-term OGD. Most substantial alterations in transcripts levels and their translation were seen to occur in the first 20 minutes of OGD. The rapid adaptation of translation apparatus to OGD is global and involves altered elongation and initiation rates. We also observed salient and reproducible alterations in ribosome densities of individual mRNAs such as increased translation of particular upstream Open Reading Frames (uORFs); induced site-specific arrests of the ribosomes and synthesis of extended protein isoforms.', 'Series_overall_design': 'Ribosome profiling (with mRNA-seq sequencing) was carried out at 0,20,40 and 60 minutes of OGD. Two biological replicates were used.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Pavel,V,Baranov', 'Series_sample_id': 'GSM1487112', 'Series_contact_name': 'Pavel,V,Baranov', 'Series_contact_email': 'p.baranov@ucc.ie', 'Series_contact_laboratory': 'Lapti', 'Series_contact_department': 'Biochemistry and Cell Biology', 'Series_contact_institute': 'University College Cork', 'Series_contact_address': 'Western Gateway Building, UCC', 'Series_contact_city': 'Cork', 'Series_contact_state': 'Co. Cork', 'Series_contact_zip/postal_code': 'NA', 'Series_contact_country': 'Ireland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE60nnn/GSE60752/suppl/GSE60752_supplementary_data.xls.gz', 'Series_platform_id': 'GPL14844', 'Series_platform_taxid': '10116', 'Series_sample_taxid': '10116', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP045777'}\n",
      "66 PRJNA259844 GSE60930 {'Series_title': 'Genome-wide translational analysis of RAW264 macrophages by ribosome profiling', 'Series_geo_accession': 'GSE60930', 'Series_status': 'Public on Aug 31 2016', 'Series_submission_date': 'Aug 29 2014', 'Series_last_update_date': 'May 15 2019', 'Series_summary': 'Result and conclusion: We discovered where in the ribosome translational arrest ocurs and how this happens. Translational stall was striking at A-site and P-site of ribosomal complex. The other arrest site was observed 3 to 5 residues away from the peptidyl transfer center of the exit tunnel, in which ribosomal density and hydrophobicity showed a significant negative correlation.', 'Series_overall_design': 'mRNA and RPF of RAW264 macrophages were deep-sequenced with two independent biological replicates by Ion PGM sequencer', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Hiroaki,,Sako', 'Series_sample_id': 'GSM2054401', 'Series_contact_name': 'Hiroaki,,Sako', 'Series_contact_institute': 'Waseda University', 'Series_contact_address': '2-579-15', 'Series_contact_city': 'Tokorozawa', 'Series_contact_zip/postal_code': '359-1192', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE60nnn/GSE60930/suppl/GSE60930_RAW.tar', 'Series_platform_id': 'GPL16331', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP045893'}\n",
      "67 PRJNA261698 GSE61619 {'Series_title': 'The general mode of translation inhibition by macrolide antibiotics', 'Series_geo_accession': 'GSE61619', 'Series_status': 'Public on Nov 03 2014', 'Series_submission_date': 'Sep 22 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25349425', 'Series_summary': \"Macrolides are clinically important antibiotics thought to inhibit bacterial growth by impeding the passage of newly synthesized polypeptides through the nascent peptide exit tunnel of the bacterial ribosome. Recent data challenged this view by showing that macrolide antibiotics can differentially affect synthesis of individual proteins. In order to understand the general mechanism of macrolide action, we used genome-wide ribosome profiling and analyzed the redistribution of ribosomes translating highly expressed genes in bacterial cells treated with high concentrations of macrolide antibiotics. The metagene analysis indicated that inhibition of early rounds of translation, which would be characteristic of the conventional view of macrolide action, occurs only at a limited number of genes. Translation of most genes proceeds past the 5' proximal codons and can be arrested at more distal codons when the ribosome encounters specific short sequence motifs. The sequence motifs enriched in the sites of arrest are confined to the nascent peptide residues in the peptidyl transferase center but not to the peptide segments that contact the antibiotic molecule in the exit tunnel. This led to the conclusion that the general mode of macrolide action involves selective inhibition of peptide bond formation between specific combinations of donor and acceptor substrates. Additional factors operating in the living cell but not during in vitro protein synthesis may modulate site-specific action of macrolide antibiotics.\", 'Series_overall_design': 'Comparing ribosome distribution in bacterial cells treated with macrolide antibiotics against the control cells.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Neil,,Bahroos', 'Series_sample_id': 'GSM1509453', 'Series_contact_name': 'Krishna,,Kannan', 'Series_contact_email': 'uickrishna@gmail.com', 'Series_contact_laboratory': 'Alexander S. Mankin', 'Series_contact_department': 'Center for Pharmaceutical Biotechnology', 'Series_contact_institute': 'University of Illinois at Chicago', 'Series_contact_address': '900 S Ashland Avenue', 'Series_contact_city': 'Chicago', 'Series_contact_state': 'IL', 'Series_contact_zip/postal_code': '60607', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE61nnn/GSE61619/suppl/GSE61619_RAW.tar', 'Series_platform_id': 'GPL15206', 'Series_platform_taxid': '679895', 'Series_sample_taxid': '679895', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP047380'}\n",
      "68 PRJNA262009 GSE61742 {'Series_title': 'Impact of regulatory variation from RNA to protein', 'Series_geo_accession': 'GSE61742', 'Series_status': 'Public on Dec 19 2014', 'Series_submission_date': 'Sep 25 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25657249', 'Series_summary': 'Genetic variants that impact gene regulation are important contributors to human phenotypic variation. For this reason, considerable efforts have been made to identify genetic associations with differences in mRNA levels of nearby genes, namely, cis expression quantitative trait loci (eQTLs). The phenotypic consequences of eQTLs are presumably due, in most cases, to their ultimate effects on protein expression levels. Yet, only few studies have quantified the impact of genetic variation on proteins levels directly. It remains unclear how faithfully eQTLs are reflected at the protein level, and whether there is a significant layer of cis regulatory variation acting primarily on translation or steady state protein levels. To address these questions, we measured ribosome occupancy by high-throughput sequencing, and relative protein levels by high-resolution quantitative mass spectrometry, in a panel of lymphoblastoid cell lines (LCLs) in which we had previously measured transcript expression using RNA sequencing. We then mapped genetic variants that are associated with changes in transcript expression (eQTLs), ribosome occupancy (rQTLs), or protein abundance (pQTLs). Most of the QTLs we detected are associated with transcript expression levels, with consequent effects on ribosome and protein levels. However, we found that eQTLs tend to have significantly reduced effect sizes on protein levels, suggesting that their potential impact on downstream phenotypes is often attenuated or buffered. Additionally, we confirmed the presence of a class of cis QTLs that specifically affect protein abundance with little or no effect on mRNA levels; most of these QTLs have little effect on ribosome occupancy, and hence may arise from differences in post-translational regulation.', 'Series_overall_design': 'We measured level of translation transcriptome-wide in lymphoblastoid cell lines derived from 72 HapMap Yoruba individuals using ribosome profiling assay, for which we have transcript level, protein level (62 out of 72) and genotype information collected.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Michael,J,Ford', 'Series_sample_id': 'GSM1513258', 'Series_contact_name': 'Sidney,,Wang', 'Series_contact_email': 'siddisis@gmail.com', 'Series_contact_department': 'Human Genetics', 'Series_contact_institute': 'University of Chicago', 'Series_contact_address': '920 E. 58th Street, CLSC 317', 'Series_contact_city': 'Chicago', 'Series_contact_state': 'IL', 'Series_contact_zip/postal_code': '60637', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE61nnn/GSE61742/suppl/GSE61742_YRI_RPF_count_by_gene.table.txt.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP047476'}\n",
      "69 PRJNA262036 GSE61753 {'Series_title': 'The ribosomal protein Asc1/RACK1 is required for efficient translation of short mRNAs', 'Series_geo_accession': 'GSE61753', 'Series_status': 'Public on Apr 24 2016', 'Series_submission_date': 'Sep 25 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27117520', 'Series_summary': 'Translation is a core cellular process carried out by a highly conserved macromolecular machine, the ribosome. There has been remarkable evolutionary adaptation of this machine through the addition of eukaryote-specific ribosomal proteins whose individual effects on ribosome function are largely unknown. Here we show that eukaryote-specific Asc1/RACK1 is required for efficient translation of mRNAs with short open reading frames that show greater than average translational efficiency in diverse eukaryotes. ASC1 mutants in S. cerevisiae display compromised translation of specific functional groups, including cytoplasmic and mitochondrial ribosomal proteins, and display cellular phenotypes consistent with their gene-specific translation defects. Asc1-sensitive mRNAs are preferentially associated with the translational ‘closed loop’ complex comprised of eIF4E, eIF4G and Pab1, and depletion of eIF4G mimics the translational defects of ASC1 mutants. Together our results reveal a role for Asc1/RACK1 in a length-dependent initiation mechanism optimized for efficient translation of genes with important housekeeping functions.', 'Series_overall_design': '', 'Series_type': 'Other', 'Series_contributor': 'Wendy,V,Gilbert', 'Series_sample_id': 'GSM2101543', 'Series_contact_name': 'Wendy,V,Gilbert', 'Series_contact_email': 'wgilbert@mit.edu', 'Series_contact_laboratory': 'Wendy V. Gilbert', 'Series_contact_department': 'Biology', 'Series_contact_institute': 'Massachusetts Institute of Technology', 'Series_contact_address': '31 Ames St.', 'Series_contact_city': 'Cambridge', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02142', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE61nnn/GSE61753/suppl/GSE61753_yeast_rz_scaledcounts.txt.gz', 'Series_platform_id': 'GPL21667', 'Series_platform_taxid': '658763', 'Series_sample_taxid': '658763', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP047486'}\n",
      "70 PRJNA262909 GSE61996 {'Series_title': 'm6A mRNA Methylation Facilitates Resolution of Naïve Pluripotency Towards Differentiation (Ribo-seq)', 'Series_geo_accession': 'GSE61996', 'Series_status': 'Public on Nov 02 2014', 'Series_submission_date': 'Oct 02 2014', 'Series_last_update_date': 'Jan 10 2022', 'Series_pubmed_id': '34893620', 'Series_summary': 'Naïve and primed pluripotent states retain distinct molecular properties, yet limited knowledge exists on how their state transitions are regulated. Here we identify Mettl3, an N6-Methyladenosine (m6A) transferase, as a regulator for terminating murine naïve pluripotency. Mettl3 knockout pre-implantation epiblasts and naïve embryonic stem cells (ESCs) are depleted for m6A in mRNAs and yet, are viable. However, they fail to adequately terminate their naïve state, and subsequently undergo aberrant and restricted lineage priming at the post-implantation stage, leading to early embryonic lethality. m6A predominantly and directly reduces mRNA stability, including that of key naïve pluripotency promoting transcripts. This study highlights a critical role for an mRNA epigenetic modification in vivo, and identifies regulatory modules that functionally influence naïve and primed pluripotency in an opposing manner.', 'Series_overall_design': 'Ribosome footprint (Ribo-Seq) was measured from mouse embryonic stem cells and mouse embriod bodies, in WT and Mettl3-KO cell lines.', 'Series_type': 'Other', 'Series_contributor': 'Jacob,,Hanna', 'Series_sample_id': 'GSM1518043', 'Series_contact_name': 'Noa,,Novershtern', 'Series_contact_email': 'noa.novershtern@weizmann.ac.il', 'Series_contact_department': 'Molecular Genetics', 'Series_contact_institute': 'Weizmann Institute of Science', 'Series_contact_address': 'Weizmann Institute', 'Series_contact_city': 'Rehovot', 'Series_contact_zip/postal_code': '7610001', 'Series_contact_country': 'Israel', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE61nnn/GSE61996/suppl/GSE61996_norm_counts_cds_ribo.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP048597'}\n",
      "71 PRJNA263261 GSE62134 {'Series_title': 'HuR- dependent regulation of mRNA splicing is essential for the B cell antibody response [Ribo-Seq]', 'Series_geo_accession': 'GSE62134', 'Series_status': 'Public on Feb 13 2015', 'Series_submission_date': 'Oct 07 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25706746', 'Series_summary': 'Post-transcriptional regulation of mRNA by the RNA binding protein HuR is required in B cells for the germinal centre reaction and for the production of class-switched antibodies in response to T-independent antigens. Transcriptome-wide examination of RNA isoforms, abundance and translation in HuR-deficient B cells, together with direct measurements of HuR-RNA interaction,  revealed that HuR-dependent mRNA splicing affects hundreds of transcripts including the dihydrolipoyl succinyltransferase (Dlst), a subunit of the aketoglutaratedehydrogenase (aKGDH) enzyme. In the absence of HuR, defective mitochondrial metabolism results in high levels of reactive oxygen species and B cell death. Our study shows how post-transcriptional processes control the balance of energy metabolism required for B cell proliferation and differentiation.', 'Series_overall_design': 'Sequencing analysis of ribosome protected RNA fragments in ex vivo or LPS-activated splenic B cells was performed using ARTseq™ Ribosome Profiling Kit - Mammalian (Epicentre, RPHMR12126) and Illumina platform. Splenic B cells come from HuRf/f control or HuR cKO mice. 4-5 biological replicates per genotype and condition.', 'Series_type': 'Other', 'Series_contributor': 'Martin,,Turner', 'Series_sample_id': 'GSM1520231', 'Series_contact_name': 'Felix,,Krueger', 'Series_contact_email': 'fkrueger@altoslabs.com', 'Series_contact_department': 'Bioinformatics', 'Series_contact_institute': 'Altos Labs', 'Series_contact_address': 'Granta Park', 'Series_contact_city': 'Cambridge', 'Series_contact_zip/postal_code': 'CB21 6GP', 'Series_contact_country': 'United Kingdom', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE62nnn/GSE62134/suppl/GSE62134_Riboseq_Analysis_DESeq2_LPS_B_cells.txt.gz', 'Series_platform_id': 'GPL15103', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP048708'}\n",
      "72 PRJNA263582 GSE62247 {'Series_title': 'Cell fate determination by ubiquitin-dependent regulation of translation', 'Series_geo_accession': 'GSE62247', 'Series_status': 'Public on Sep 23 2015', 'Series_submission_date': 'Oct 10 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26399832', 'Series_summary': 'Metazoan development depends on accurate execution of differentiation programs that allow pluripotent stem cells to adopt specific fates. Differentiation is brought about by global changes to chromatin architecture and transcriptional networks, yet whether other regulatory events support cell fate determination is less well understood. Using human embryonic stem cell and Xenopus models, we identified the vertebrate-specific ubiquitin ligase Cul3KBTBD8 as an essential regulator of neural crest specification. Cul3KBTBD8 monoubiquitylates NOLC1 and its paralog TCOF1, whose mutation underlies the craniofacial disorder Treacher Collins Syndrome that is characterized by a loss of cranial neural crest cells. Ubiquitylation of NOLC1 and TCOF1 drives formation of a platform that connects RNA polymerase I with ribosome modification enzymes, thereby altering the translational program of differentiating cells to support the generation of neural crest cells. We conclude that the dynamic regulation of ribosome function is an important feature of cell fate determination.', 'Series_overall_design': 'Ribosome profiling and mRNA-Seq', 'Series_type': 'Other', 'Series_contributor': 'Michael,,Rape', 'Series_sample_id': 'GSM1782887', 'Series_contact_name': 'Shintaro,,Iwasaki', 'Series_contact_email': 'shintaro.iwasaki@riken.jp', 'Series_contact_department': 'Cluster for Pioneering Research', 'Series_contact_institute': 'RIKEN', 'Series_contact_address': 'S205 Bioscience Bldg. 2-1 Hirosawa', 'Series_contact_city': 'Wako', 'Series_contact_state': 'Saitama', 'Series_contact_zip/postal_code': '351-0198', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE62nnn/GSE62247/suppl/GSE62247_RAW.tar', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP048825'}\n",
      "73 PRJNA266705 GSE63120 {'Series_title': 'Widespread Co-translational RNA Decay Reveals Ribosome Dynamics', 'Series_geo_accession': 'GSE63120', 'Series_status': 'Public on Jun 04 2015', 'Series_submission_date': 'Nov 08 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26046441', 'Series_summary': 'It is generally assumed that mRNAs undergoing translation are protected from decay. Here, we show that mRNAs are, in fact, co-translationally degraded. This is a widespread and conserved process affecting most genes, where 5′–3′ transcript degradation follows the last translating ribosome, producing an in vivo ribosomal footprint. By sequencing the ends of 5′ phosphorylated mRNA degradation intermediates, we obtain a genome-wide drug-free measurement of ribosome dynamics. We identify general translation termination pauses in both normal and stress conditions. In addition, we describe novel codon-specific ribosomal pausing sites in response to oxidative stress that are dependent on the RNase Rny1. Our approach is simple and straightforward and does not require the use of translational inhibitors or in vitro RNA footprinting that can alter ribosome protection patterns.', 'Series_overall_design': 'RNA samples were quantified according to their 5’modification. Samples were quantified using 2 alternative methods. 5PSeq (measures 5’P RNAs) and 5PSeq-fragmented (measures the sequencing bias present in any sample by randomly fragmenting RNA, re-phosphorilating the 5’ends and subjecting the sample to 5PSeq; it is used as a negative control). Different strains of S. cerevisiae and S. pombe samples were analyzed. Cells were grown both in rich media (YPD or YES) and synthetic defined media (SD). Cells were subjected to different treatments such as inhibition of translation elongation with cyclohexamide (CHX), oxidative stress (0.2 mM H2O2) or inhibition of histidine biosyntesys (3-AT; 100mM 3-amino-1,2,4-triazole). S. cerevisiae samples grown in rich media were also analyzed after CHX treatment and sucrose fractionation into monosome and polyribosome fractionation. All samples were analyzed in biological duplicates.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Lars,M,Steinmetz', 'Series_sample_id': 'GSM1637660', 'Series_contact_name': 'Wu,,Wei', 'Series_contact_email': 'wuwei@embl.de', 'Series_contact_laboratory': 'Steinmetz Lab', 'Series_contact_department': 'Genome Biology Unit', 'Series_contact_institute': 'EMBL Heidelberg', 'Series_contact_address': 'EMBL Heidelberg,Meyerhofstraße 1', 'Series_contact_city': 'Heidelberg', 'Series_contact_zip/postal_code': '69117', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE63nnn/GSE63120/suppl/GSE63120_xrn_ypd_p.txt.gz', 'Series_platform_id': 'GPL17143', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP049641'}\n",
      "74 PRJNA268222 GSE63570 {'Series_title': 'High-throughput sequencing of pluripotent cells', 'Series_geo_accession': 'GSE63570', 'Series_status': 'Public on Apr 21 2015', 'Series_submission_date': 'Nov 21 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25896322', 'Series_summary': 'We report HERV-K rec iCLIP-seq binding data, ribosome profiling data, and RNA-seq from ELF1 naïve hESC and RNA-seq from NCCIT cells.', 'Series_overall_design': 'HERV-K Rec iCLIP-seq: 2 replicates in NCCIT. Ribosome profiling: 4 replicates each of Rec-overexpressing NCCIT vs. control NCCIT; RNAseq: 3 replicates each of HERV-K Rec siRNA vs. control siRNA in NCCIT; RNA-seq: 3 replicates each of ELF1 naïve hESC vs. primed hESC.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Joanna,,Wysocka', 'Series_sample_id': 'GSM1592573', 'Series_contact_name': 'Edward,J,Grow', 'Series_contact_laboratory': 'Cairns', 'Series_contact_department': 'Oncology', 'Series_contact_institute': 'University of Utah', 'Series_contact_address': '2000 Circle of Hope Drive', 'Series_contact_city': 'Salt Lake City', 'Series_contact_state': 'UT', 'Series_contact_zip/postal_code': '84112-0555', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE63nnn/GSE63570/suppl/GSE63570_Rec_siRNA_RNA-seq.txt.gz', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP050147'}\n",
      "75 PRJNA270712 GSE64304 {'Series_title': \"Parkinson's disease genes VPS35 and EIF4G1 interact genetically and converge on alpha-synuclein\", 'Series_geo_accession': 'GSE64304', 'Series_status': 'Public on Dec 19 2014', 'Series_submission_date': 'Dec 18 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25533483', 'Series_summary': \"Parkinson's disease (PD) is a common human neurodegenerative movement disorder. Studies of the genetic forms of PD have helped to reveal disease mechanisms. Functional interactions between some Parkinson's disease (PD) genes, like PINK1 and parkin, have been identified, but whether other ones interact remains elusive. Here we report an unexpected genetic interaction between two PD genes, VPS35 and EIF4G1. We provide evidence that EIF4G1 upregulation causes defects associated with protein misfolding. Expression of a sortilin protein rescues these defects, downstream of VPS35, suggesting a potential role for sortilins in PD. We also show interactions between VPS35, EIF4G1 and alpha-synuclein, a protein with a key role in the pathogenesis of both sporadic and familial PD. We extend our findings from yeast to an animal model and show these interactions are conserved in neurons. We also connect VPS35 impairments to neurodegeneration in alpha-synuclein transgenic mice. Our studies reveal unexpected genetic and functional interactions between two seemingly unrelated PD genes and functionally connect them to alpha-synuclein pathobiology in yeast, worms, and mouse. Finally, we provide a resource of candidate PD genes for future genetic and functional interrogation.\", 'Series_overall_design': 'Ribosome profiling (RiboSeq) of wild type and VPS35 deletion yeast strains, with or without overexpression of the TIF4631 initiation factor', 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'Aaron,D,Gitler', 'Series_sample_id': 'GSM1568346', 'Series_contact_name': 'Justin,William,Chartron', 'Series_contact_email': 'chartron@stanford.edu', 'Series_contact_laboratory': 'Frydman', 'Series_contact_department': 'Biology', 'Series_contact_institute': 'Stanford University', 'Series_contact_address': 'Clark Center E200, 318 Campus Drive', 'Series_contact_city': 'Stanford', 'Series_contact_state': 'California', 'Series_contact_zip/postal_code': '94305', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE64nnn/GSE64304/suppl/GSE64304_RAW.tar', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP051314'}\n",
      "76 PRJNA272597 GSE64962 {'Series_title': 'Improved transcription and translation with L-leucine stimulation of mTORC1', 'Series_geo_accession': 'GSE64962', 'Series_status': 'Public on Jan 15 2016', 'Series_submission_date': 'Jan 14 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26729373', 'Series_summary': 'Roberts syndrome (RBS) is a human developmental disorder caused by mutations in the cohesin acetyltransferase ESCO2. We previously reported that mTORC1 was inhibited and overall translation was reduced in RBS cells. Treatment of RBS cells with L-leucine partially rescued mTOR function and protein synthesis, correlating with increased cell division. In this study, we use RBS as a model for mTOR inhibition and analyze transcription and translation with ribosome profiling to determine genome-wide effects of L-leucine. The translational efficiency of many genes is increased with Lleucine in RBS cells including genes involved in ribosome biogenesis, translation, and mitochondrial function. snoRNAs are strongly upregulated in RBS cells, but decreased with L-leucine. Imprinted genes, including H19 and GTL2, are differentially expressed in RBS cells consistent with contribution to mTORC1 control. This study reveals dramatic effects of L-leucine stimulation of mTORC1 and supports that ESCO2 function is required for normal gene expression and translation.', 'Series_overall_design': '42 samples of human fibroblast cell lines with various genotypes (wt, corrected, and esco2 mutants) are treated with l-leucine or d-leucine (control) for 3 or 24 hours. Biological replicates are present.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Jennifer,,Gerton', 'Series_sample_id': 'GSM1585245', 'Series_contact_name': 'Madelaine,,Gogol', 'Series_contact_department': 'Computational Biology Core', 'Series_contact_institute': 'Stowers Institute', 'Series_contact_address': '1000 E. 50th Street', 'Series_contact_city': 'Kansas City', 'Series_contact_state': 'MO', 'Series_contact_zip/postal_code': '64110', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE64nnn/GSE64962/suppl/GSE64962_RAW.tar', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP052229'}\n",
      "77 PRJNA274997 GSE65778 {'Series_title': 'The Small Molecule ISRIB Reverses the Effects of eIF2α Phosphorylation on Translation and Stress Granule Assembly', 'Series_geo_accession': 'GSE65778', 'Series_status': 'Public on Feb 27 2015', 'Series_submission_date': 'Feb 09 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25719440', 'Series_summary': 'We recently identified ISRIB as a potent inhibitor of the integrated stress response (ISR). ISRIB renders cells resistant to the effects of eIF2α phosphorylation and enhances long-term memory in rodents (10.7554/eLife.00498).  Here we show by genome-wide in vivo ribosome profiling that translation of a restricted subset of mRNAs is induced upon ISR activation. ISRIB substantially reversed the translational effects elicited by phosphorylation of eIF2α and induced no major changes in translation or mRNA levels in unstressed cells. eIF2α phosphorylation-induced stress granule (SG) formation was blocked by ISRIB. Strikingly, ISRIB addition to stressed cells with pre-formed SGs induced their rapid disassembly, liberating mRNAs into the actively translating pool. Restoration of mRNA translation and modulation of SG dynamics may be an effective treatment of neurodegenerative diseases characterized by eIF2α phosphorylation, SG formation and cognitive loss.', 'Series_overall_design': 'Ribosome profiling with paired RNA-seq', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Peter,,Walter', 'Series_sample_id': 'GSM1606114', 'Series_contact_name': 'Anna,M,McGeachy', 'Series_contact_email': 'anna.mcgeachy@gmail.com', 'Series_contact_phone': '2242131825', 'Series_contact_laboratory': 'Ingolia', 'Series_contact_department': 'MCB', 'Series_contact_institute': 'UC Berkeley', 'Series_contact_address': '416 Barker Hall', 'Series_contact_city': 'Berkeley', 'Series_contact_state': 'California', 'Series_contact_zip/postal_code': '94720', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE65nnn/GSE65778/suppl/GSE65778_RAW.tar', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP053402'}\n",
      "78 PRJNA275305 GSE65885 {'Series_title': 'Ribosome profiling and RNA sequencing of MCF10A-ER-Src and fibroblast cell transformation', 'Series_geo_accession': 'GSE65885', 'Series_status': 'Public on Nov 13 2015', 'Series_submission_date': 'Feb 12 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26687005', 'Series_summary': 'We applied ribosome profiling and RNA sequencing to examine gene expression regulation during oncogenic cell transformation. One model involves normal mammary epithelial cells (MCF10A) containing ER-Src. Treatment of such cells with tamoxifen rapidly induces Src, thereby making it possible to kinetically follow the transition between normal and transformed cells. The other model consists of three isogenic cell lines derived from primary fibroblasts in a serial manner (Hahn et al., 1999). EH cell is immortalized by overexpression of telomerase (hTERT), and exhibits normal fibroblast morphology. EL cell expresses hTERT along with both large and small T antigens of Simian virus 40, and it displays an altered morphology but is not transformed. ELR cell expresses hTERT, T antigens, and an oncogenic derivative of Ras (H-RasV12).', 'Series_overall_design': 'Ribosome profiling and RNA sequencing in two cancer cell models', 'Series_type': 'Other', 'Series_contributor': 'Kevin,,Struhl', 'Series_sample_id': 'GSM1608296', 'Series_contact_name': 'Zhe,,Ji', 'Series_contact_email': 'zhe.ji@northwestern.edu', 'Series_contact_institute': 'Northwestern University', 'Series_contact_address': '303 E Superior St, Lurie 6-107', 'Series_contact_city': 'Chicago', 'Series_contact_state': 'ILLINOIS', 'Series_contact_zip/postal_code': '60611', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE65nnn/GSE65885/suppl/GSE65885_RAW.tar', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP054971'}\n",
      "79 PRJNA275386 GSE65912 {'Series_title': 'Integrative analysis of RNA, translation and protein levels reveals distinct regulatory variation across humans', 'Series_geo_accession': 'GSE65912', 'Series_status': 'Public on Feb 14 2015', 'Series_submission_date': 'Feb 13 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26297486', 'Series_summary': 'Elucidating the extent and consequences of genetic differences between humans is essential for understanding phenotypic diversity and personalized medicine. Although variation in RNA levels, transcription factor binding and chromatin have been explored, little is known about global variation in translation and its genetic determinants among humans. We used ribosome profiling, RNA sequencing, and mass spectrometry to perform an integrated analysis in lymphoblastoid cell lines from a diverse group of individuals. We find significant differences in RNA levels, translation, and protein abundance suggesting diverse mechanisms of personalized gene expression control. Combined analysis of RNA expression and ribosome occupancy improves the identification of individual protein level differences.  Finally, we identify genetic differences that specifically modulate ribosome occupancy - many of these differences lie close to start codons and upstream ORFs. Our results reveal a new level of gene expression variation among humans and indicate that genetic variants can cause changes in protein levels through effects on translation.', 'Series_overall_design': 'Ribosome profiling and RNA sequencing experiments from human lymphoblastoid cells', 'Series_type': 'Other', 'Series_contributor': 'Michael,P,Snyder', 'Series_sample_id': 'GSM1609444', 'Series_contact_name': 'Can,,Cenik', 'Series_contact_email': 'ccenik@stanford.edu', 'Series_contact_laboratory': 'Snyder', 'Series_contact_department': 'Genetics', 'Series_contact_institute': 'Stanford University', 'Series_contact_address': '300 Pasteur Dr, Alway Building Rm M344', 'Series_contact_city': 'Stanford', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94305', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE65nnn/GSE65912/suppl/GSE65912_RibosomeProfiling.csv.gz', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP055009'}\n",
      "80 PRJNA275789 GSE66054 {'Series_title': 'MicroRNA-203 represses selection and expansion of oncogenic HRas transformed tumor initiating cells [seq]', 'Series_geo_accession': 'GSE66054', 'Series_status': 'Public on Jul 23 2015', 'Series_submission_date': 'Feb 18 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26203562', 'Series_summary': 'In many mouse models of skin cancer, only a few tumors typically form although many cells competent for tumorigenesis receive the same oncogenic mutations. These observations suggest a selection process for defining tumor initiating cells. Here we use quantitative mRNA- and miR-Seq to determine the impact of HRasG12V on the transcriptome of keratinocytes. We discover that microRNA-203 is downregulated by HRasG12V. Using a knockout mouse model, we demonstrate that loss of microRNA-203 promotes selection and expansion of tumor-initiating cells. Conversely, restoration of microRNA-203 with an inducible model potently inhibits proliferation of these cells. We comprehensively identify microRNA-203 targets required for HRas-initiated tumorigenesis. These targets include important effectors of the Ras pathway and essential genes required for cell division. Together, this study establishes a role for the loss of microRNA-203 in promoting selection and expansion of HRas mutated cells and identifies a mechanism through which microRNA-203 antagonizes HRas-mediated tumorigenesis.', 'Series_overall_design': 'Identifying mRNA and microRNA networks regulated by oncogenic HRasG12V in primary keratinocytes through the use of 3Seq and small-RNA-Sequencing. Additionally we utilize ribosome-profiling, 3Seq, Microarray and Ago2-HITS-CLIP approaches to identify novel miR-203 target genes.', 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'Rui,,Yi', 'Series_sample_id': 'GSM1613700', 'Series_contact_name': 'Kent,Augustus,Riemondy', 'Series_contact_email': 'kent.riemondy@cuanschutz.edu', 'Series_contact_department': 'Biochemistry and Molecular Genetics', 'Series_contact_institute': 'University of Colorado School of Medicine', 'Series_contact_address': '12801 E 17th Ave', 'Series_contact_city': 'Aurora', 'Series_contact_state': 'Colorado', 'Series_contact_zip/postal_code': '80045', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE66nnn/GSE66054/suppl/GSE66054_riboseq_counts_103114.txt.gz', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP055150'}\n",
      "81 PRJNA276827 GSE66411 {'Series_title': 'Genome-wide analysis of translational efficiency reveals distinct but overlapping functions of yeast DEAD-box RNA helicases Ded1 and eIF4A', 'Series_geo_accession': 'GSE66411', 'Series_status': 'Public on Mar 02 2015', 'Series_submission_date': 'Mar 02 2015', 'Series_last_update_date': 'Jul 18 2019', 'Series_pubmed_id': '31299079', 'Series_summary': \"DEAD-box RNA helicases eIF4A and Ded1 are believed to promote translation initiation by resolving mRNA secondary structures that impede ribosome attachment at the mRNA 5’ end or subsequent scanning of the 5’UTR, but whether they perform distinct functions or act redundantly in vivo is poorly understood. We compared the effects of mutations in Ded1 or eIF4A on global translational efficiencies (TEs) in yeast by ribosome footprint profiling. Despite similar reductions in bulk translation, inactivation of a cold-sensitive Ded1 mutant substantially reduced the TEs of >600 mRNAs, whereas inactivation of a temperature-sensitive eIF4A mutant yielded <40 similarly impaired mRNAs. The broader requirement for Ded1 did not reflect more pervasive secondary structures at low temperature, as inactivation of temperature-sensitive and cold-sensitive ded1 mutants gave highly correlated results. Interestingly, Ded1-dependent mRNAs exhibit greater than average 5’UTR length and propensity for secondary structure, implicating Ded1 in scanning though structured 5' UTRs. Reporter assays confirmed that cap- distal stem-loop insertions increase dependence on Ded1 but not eIF4A for efficient translation. While only a small fraction of mRNAs is strongly dependent on eIF4A, this dependence is significantly correlated with requirements for Ded1 and 5’UTR features characteristic of Ded1- dependent mRNAs. Our findings suggest that Ded1 is critically required to promote scanning through secondary structures within 5’UTRs; and while eIF4A cooperates with Ded1 in this function, it also promotes a step of initiation common to virtually all yeast mRNAs.\", 'Series_overall_design': 'We compared the effects of mutations in Ded1 or eIF4A on global translational efficiencies (TEs) in yeast by ribosome footprint profiling.The study includes 32 samples, comprised of 16 mRNA-Seq samples and 16 ribosome footprint profiling samples, derived from biological replicates of 3 mutant strains, ded1-cs, ded1-ts and tif1-ts, and the corresponding wild-type strains. The tif1-ts mutant and its wild-type counterpart were analyzed at 30°C and 37°C.', 'Series_type': 'Other', 'Series_contributor': 'Alan,G,Hinnebusch', 'Series_sample_id': 'GSM1622019', 'Series_contact_name': 'Alan,G,Hinnebusch', 'Series_contact_email': 'ahinnebusch@nih.gov', 'Series_contact_institute': 'NICHD', 'Series_contact_address': '6 CENTER DR Room 230, MSC 0609', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '20814', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE66nnn/GSE66411/suppl/GSE66411_tif1-ts_37_deg.csv.gz', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP055707'}\n",
      "82 PRJNA277363 GSE66580 {'Series_title': 'Ribosome profiling reveals changes in translational status of soybean transcripts during immature cotyledon development', 'Series_geo_accession': 'GSE66580', 'Series_status': 'Public on Mar 23 2018', 'Series_submission_date': 'Mar 05 2015', 'Series_last_update_date': 'Jul 29 2019', 'Series_pubmed_id': '29570733', 'Series_summary': 'To understand translational capacity on a genome-wide scale across three developmental stages of immature soybean seed cotyledons, ribosome profiling was performed in combination with RNA sequencing and cluster analysis. Transcripts representing 216 unique genes demonstrated a higher level of translational activity in at least one stage by exhibiting higher translational efficiencies (TEs) in which there were relatively more ribosome footprint sequence reads mapping to the transcript than were present in the control total RNA sample. The majority of these transcripts were more translationally active at the early stage of seed development and included 12 unique serine or cysteine proteases and 16 2S albumin and low molecular weight cysteine-rich proteins that may serve as substrates for turnover and mobilization early in seed development. It would appear that the serine proteases and 2S albumins play a vital role in the early stages. In contrast, our investigation of profiles of 19 genes encoding high abundance seed storage proteins, such as glycinins, beta-conglycinins, lectin, and Kunitz trypsin inhibitors, showed that they all had similar patterns in which the TE values started at low levels and increased approximately 2 to 6-fold during development. The highest levels of these seed protein transcripts were found at the mid-developmental stage, whereas the highest ribosome footprint levels of only up to 1.6 TE were found at the late developmental stage.  These experimental findings suggest that the major seed storage protein coding genes are primarily regulated at the transcriptional level during normal soybean cotyledon development. Finally, our analyses also identified a total of 370 unique gene models that showed very low TE values including over 48 genes encoding ribosomal family proteins and 95 gene models that are related to energy and photosynthetic functions, many of which have homology to the chloroplast genome.', 'Series_overall_design': 'Construction of ribosome profiling libraries using cotyledons from 3 different developmental stages, with two biological replicates. Data was obtained also from two stages of seed coat development.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Lila,O,Vodkin', 'Series_sample_id': 'GSM1625841', 'Series_contact_name': 'Lila,O.,Vodkin', 'Series_contact_email': 'l-vodkin@illinois.edu', 'Series_contact_phone': '217-244-6147', 'Series_contact_laboratory': 'Lila Vodkin', 'Series_contact_department': 'Crop Sciences', 'Series_contact_institute': 'University of Illinois', 'Series_contact_address': '1201 W. Gregory Dr.', 'Series_contact_city': 'Urbana', 'Series_contact_state': 'IL', 'Series_contact_zip/postal_code': '61801', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE66nnn/GSE66580/suppl/GSE66580_RAW.tar', 'Series_platform_id': 'GPL15008', 'Series_platform_taxid': '3847', 'Series_sample_taxid': '3847', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP055880'}\n",
      "83 PRJNA277771 GSE66715 {'Series_title': 'Integrated Transcriptome and Proteome Analyses Reveal Organ-Specific Proteome Deterioration in Old Rats', 'Series_geo_accession': 'GSE66715', 'Series_status': 'Public on Sep 22 2015', 'Series_submission_date': 'Mar 09 2015', 'Series_last_update_date': 'Jul 26 2019', 'Series_pubmed_id': '27135913', 'Series_summary': 'Aging is associated with the decline of protein, cell, and organ function. Here, we use an integrated approach to characterize gene expression, bulk translation, and cell biology in the brains and livers of young and old rats. We identify 468 differences in protein abundance between young and old animals. The majority are a consequence of altered translation output, that is, the combined effect of changes in transcript abundance and translation efficiency. In addition, we identify 130 proteins whose overall abundance remains unchanged but whose sub-cellular localization, phosphorylation state, or splice-form varies. While some protein-level differences appear to be a generic property of the rats’ chronological age, the majority are specific to one organ. These may be a consequence of the organ’s physiology or the chronological age of the cells within the tissue. Taken together, our study provides an initial view of the proteome at the molecular, sub-cellular, and organ level in young and old rats.', 'Series_overall_design': 'RNA-Seq and ribosome profiling from matched young and old rat liver and brain', 'Series_type': 'Other', 'Series_contributor': 'Martin,,Beck', 'Series_sample_id': 'GSM1630481', 'Series_contact_name': 'Michael,,Harris', 'Series_contact_email': 'mike.harris@berkeley.edu', 'Series_contact_laboratory': 'Ingolia', 'Series_contact_department': 'MCB', 'Series_contact_institute': 'UC Berkeley', 'Series_contact_address': 'Barker Hall', 'Series_contact_city': 'Berkeley', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94720-0001', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE66nnn/GSE66715/suppl/GSE66715_RAW.tar', 'Series_citation': 'Ori A, Toyama BH, Harris MS, Bock T, Iskar M, Bork P, Ingolia NT, Hetzer MW, Beck, M. (2015). Integrated Transcriptome and Proteome Analyses Reveal Organ-Specific Proteome Deterioration in Old Rats. Cell Systems, Volume 1, Issue 3, p224-237, 23 September http://dx.doi.org/10.1016/j.cels.2015.08.012', 'Series_platform_id': 'GPL14844', 'Series_platform_taxid': '10116', 'Series_sample_taxid': '10116', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP056012'}\n",
      "84 PRJNA279785 GSE67387 {'Series_title': 'Optimization of codon translation rates via tRNA modifications maintains proteome integrity', 'Series_geo_accession': 'GSE67387', 'Series_status': 'Public on Jun 04 2015', 'Series_submission_date': 'Mar 28 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26052047', 'Series_summary': 'Proteins begin to fold as they emerge from translating ribosomes. The kinetics of ribosome transit along a given mRNA can influence nascent chain folding, but the extent to which individual codon translation rates impact proteome integrity remains unknown. Here, we show that slower decoding of discrete codons elicits widespread protein aggregation in vivo. Using ribosome profiling, we find that loss of anticodon wobble uridine (U34) modifications in a subset of tRNAs leads to ribosome pausing at their cognate codons in S. cerevisiae and C. elegans. Yeast cells lacking U34 modifications exhibit gene expression hallmarks of proteotoxic stress and accumulate aggregates of endogenous proteins with key cellular functions. Moreover, these cells are severely compromised in clearing stress-induced protein aggregates. Overexpression of hypomodified tRNAs alleviates ribosome pausing, concomitantly restoring protein homeostasis. Our findings demonstrate that modified U34 is an evolutionarily conserved accelerator of decoding and reveal an unanticipated role for tRNA anticodon modifications in maintaining proteome integrity.', 'Series_overall_design': 'Ribosome profiling of wild-type and tRNA modification-deficient yeast and nematodes. Yeast samples were generated in various growth conditions (rich medium versus stress induced by treatment with diamide or rapamycin) and paired mRNA-Seq was performed on a subset of samples. Dataset contains three biological replicates for yeast samples and two biological replicates for nematode samples.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Sebastian,A,Leidel', 'Series_sample_id': 'GSM1646112', 'Series_contact_name': 'Danny,D.,Nedialkova', 'Series_contact_email': 'danny.nedialkova@mpi-muenster.mpg.de', 'Series_contact_department': 'Max Planck Research Group for RNA Biology', 'Series_contact_institute': 'Max Planck Institute for Molecular Biomedicine', 'Series_contact_address': 'Von-Esmarch-Str. 54', 'Series_contact_city': 'Münster', 'Series_contact_zip/postal_code': '48149', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE67nnn/GSE67387/suppl/GSE67387_Scer_ribo_tKQE_rawCounts.txt.gz', 'Series_platform_id': 'GPL19959', 'Series_platform_taxid': '6239', 'Series_sample_taxid': '6239', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP056647'}\n",
      "85 PRJNA280809 GSE67741 {'Series_title': 'Ribosome profiling with translation inhibitors reveals pervasive translation in murine ES cells', 'Series_geo_accession': 'GSE67741', 'Series_status': 'Public on Jan 12 2016', 'Series_submission_date': 'Apr 10 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26764022', 'Series_summary': 'Ribosome profiling with translation inhibitors reveals pervasive translation in murine ES cells.', 'Series_overall_design': 'Ribosome profiling or stranded RNAseq (ribominus) with murine embryonic stem cells treated with either DMD-pateamineA, Puromycin, Harringtonine or vehicle (no drug control). Two replicates per condition.', 'Series_type': 'Other', 'Series_contributor': 'P,,Barbry', 'Series_sample_id': 'GSM1655068', 'Series_contact_name': 'Kevin,,Lebrigand', 'Series_contact_laboratory': 'Functional Genomics Platform of Nice-Sophia-Antipolis, France.', 'Series_contact_institute': 'IPMC/CNRS', 'Series_contact_address': '660 route des lucioles', 'Series_contact_city': 'Valbonne - Sophia-Antipolis', 'Series_contact_zip/postal_code': '06560', 'Series_contact_country': 'France', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE67nnn/GSE67741/suppl/GSE67741_RAW.tar', 'Series_platform_id': 'GPL18061', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP057051'}\n",
      "86 PRJNA281177 GSE67902 {'Series_title': 'Regulation of protein translation during mitosis', 'Series_geo_accession': 'GSE67902', 'Series_status': 'Public on Sep 01 2015', 'Series_submission_date': 'Apr 15 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26305499', 'Series_summary': 'Entry into and exit from mitosis is driven by precisely-timed changes in protein abundance, and involves transcriptional regulation and protein degradation. However, the role of translational regulation in modulating cellular protein content during mitosis remains poorly understood. Here, using ribosome profiling, we show that translational, rather than transcriptional regulation is the dominant mechanism for modulating protein synthesis at mitotic entry. The vast majority of regulated mRNAs are translationally repressed, which contrasts previous findings of selective mRNA translational activation at mitotic entry. One of the most pronounced translationally repressed genes in mitosis is Emi1, an inhibitor of the anaphase promoting complex (APC), which is degraded during mitosis. We show that Emi1 degradation is insufficient for full APC activation and that simultaneous translational repression is required. These results provide a genome-wide view of protein translation during mitosis and suggest that translational repression may be used to ensure complete protein inactivation', 'Series_overall_design': 'Ribosome profiling and mRNA-seq from 3 time points in the cell cycle', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Ronald,D,Vale', 'Series_sample_id': 'GSM1657731', 'Series_contact_name': 'Marvin,,Tanenbaum', 'Series_contact_email': 'marvin.tanenbaum@ucsf.edu', 'Series_contact_laboratory': 'Ron Vale', 'Series_contact_department': 'CMP', 'Series_contact_institute': 'UCSF', 'Series_contact_address': 'Genentech Hall, MC 2200, Room N316,   600 16th street', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'ca', 'Series_contact_zip/postal_code': '94158-2517', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE67nnn/GSE67902/suppl/GSE67902_Supplemental_table_S2_TE_analysis.txt.gz', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP057253'}\n",
      "87 PRJNA281481 GSE68008 {'Series_title': 'Methods comparison for ribosome profiling', 'Series_geo_accession': 'GSE68008', 'Series_status': 'Public on Jul 01 2015', 'Series_submission_date': 'Apr 17 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26164698', 'Series_summary': 'Assessment of the effecacy of direction extraction of ribosome-protected mRNA fragments for ribosome profiling', 'Series_overall_design': 'Immortalized human lymphocytes were analyzed by ribosome profiling protocols with or without ribosome purification', 'Series_type': 'Other', 'Series_contributor': 'David,W,Reid', 'Series_sample_id': 'GSM1660951', 'Series_contact_name': 'David,,Reid', 'Series_contact_laboratory': 'Nicchitta', 'Series_contact_department': 'Biochemistry', 'Series_contact_institute': 'Duke University', 'Series_contact_address': '436 Nanaline Duke Bldg., Box 3709 DUMC', 'Series_contact_city': 'Durham', 'Series_contact_state': 'NC', 'Series_contact_zip/postal_code': '27710', 'Series_contact_country': 'USA', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP057384'}\n",
      "88 PRJNA282016 GSE68200 {'Series_title': 'Ribosome Profiling Reveals Translational Control During the Caulobacter crescentus cell cycle', 'Series_geo_accession': 'GSE68200', 'Series_status': 'Public on Apr 20 2016', 'Series_submission_date': 'Apr 23 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27791168', 'Series_summary': 'The Caulobacter cell cycle includes sequential differentiation of the cell poles and an asymmetric cell division yielding distinct daughter cells.  RNA-seq and ribosome profiling were used to measure global messenger RNA (mRNA) abundance and translation levels throughout the Caulobacter cell cycle revealing translational control in 51% of cell cycle-regulated genes.   Ribosome profiling levels were temporally regulated for ten proteins encoding asymmetrically localized multi-protein assemblies that specify cell fate upon cell division, with five ORFs being regulated with a >2 fold contribution from the translation efficiency.  Individual ORFs within 198 operons undergo different cell cycle patterns of translation efficiency, allowing decoupling of the expression of these co-transcribed units.  This work reveals that, in addition to transcriptional control, there is prevalent translational control of cell cycle-regulated genes, providing a framework for a multi-level program of cell cycle control that generates daughter cells of different fates.', 'Series_overall_design': 'Ribosome profiling and RNA-seq data were collected in sychronized Caulobacter crescentus NA1000 cells grown in M2G to measure changes in mRNA levels and translation levels during the asymmetric cell division cycle.', 'Series_type': 'Other', 'Series_contributor': 'Shapiro,,Lucy', 'Series_sample_id': 'GSM1665599', 'Series_contact_name': 'Jared,Michael,Schrader', 'Series_contact_email': 'schrader@wayne.edu', 'Series_contact_laboratory': 'Schrader Lab', 'Series_contact_department': 'Biological Sciences', 'Series_contact_institute': 'Wayne State University', 'Series_contact_address': '5047 Gullen Mall, Biological Sciences Building Room 2168', 'Series_contact_city': 'Detroit', 'Series_contact_state': 'MI', 'Series_contact_zip/postal_code': '48202', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE68nnn/GSE68200/suppl/GSE68200_RAW.tar', 'Series_platform_id': 'GPL18276', 'Series_platform_taxid': '565050', 'Series_sample_taxid': '565050', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP057599'}\n",
      "89 PRJNA282960 GSE68265 {'Series_title': 'Dissection of the translational impacts of the PERK pathway', 'Series_geo_accession': 'GSE68265', 'Series_status': 'Public on Jan 01 2016', 'Series_submission_date': 'Apr 24 2015', 'Series_last_update_date': 'May 15 2019', 'Series_summary': 'Disruptions of protein homeostasis in the endoplasmic reticulum (ER) elicit activation of the unfolded protein response (UPR), a translation- and transcription-coupled proteostatic stress response pathway. The primary translational control arm of the UPR is initiated by the PERK-dependent phosphorylation of eIF2α, leading to a large-scale reprogramming of translation and subsequent activation of an ATF4-mediated transcriptional program. It has remained challenging, however, to accurately evaluate the contribution of each component of the eIF2α/ATF4 pathway to the remodelling of transcription and translation. Here, we have used mouse embryonic fibroblasts containing either a knock-in of the non-phosphorylatable eIF2α S51A mutant or knock-out for ATF4 by ribosome profiling and mRNA-seq to define the specific contributions of eIF2α phosphoryation and ATF4 in controlling the translational and transcriptional components of the UPR. These studies show that the transcriptional and translational targets of each P-eIF2α, ATF4, and the other UPR gene expression programs overlapped extensively, leading to a core set of UPR genes whose robust expression in response to ER stress is driven by multiple mechanisms. The identification of other, more factor-specific targets illustrated the potential for functional specialization of the UPR. As the UPR progressed temporally, cells employed distinct combinations of transcriptional and translational mechanisms, initiated by different factors, to adapt to ongoing stress. These effects were accompanied by a buffering effect where changes in mRNA levels were coupled to opposing changes in ribosome loading, a property which makes cooperation between transcription and translation essential to confer robust protein expression.', 'Series_overall_design': 'Mouse embryonic fibroblasts (MEFs) lacking components of the PERK pathway (eIF2a phosphorylation and ATF4) were subjected to ER stress and analyzed by ribosome profiling.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Christopher,,Nicchitta', 'Series_sample_id': 'GSM1674149', 'Series_contact_name': 'David,,Reid', 'Series_contact_laboratory': 'Nicchitta', 'Series_contact_department': 'Biochemistry', 'Series_contact_institute': 'Duke University', 'Series_contact_address': '436 Nanaline Duke Bldg., Box 3709 DUMC', 'Series_contact_city': 'Durham', 'Series_contact_state': 'NC', 'Series_contact_zip/postal_code': '27710', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE68nnn/GSE68265/suppl/GSE68265_translation_expression.xlsx', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP057991'}\n",
      "90 PRJNA283800 GSE68762 {'Series_title': 'Bacteria differently regulate mRNA abundance to specifically respond to various stresses', 'Series_geo_accession': 'GSE68762', 'Series_status': 'Public on Jul 01 2015', 'Series_submission_date': 'May 11 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27507888', 'Series_summary': 'Environmental stress is detrimental to cell viability and requires an adequate reprogramming of cellular activities to maximize cell survival. We present a global analysis of the response of Escherichia coli to acute heat and osmotic stress. We combine deep sequencing of total mRNA and ribosome-protected fragments to provide a genome-wide map of the stress response at transcriptional and translational level. For each type of stress, we observe a unique subset of genes that shape the stress-specific response. Upon temperature upshift, mRNAs with reduced folding stability up- and downstream of the start codon, and thus with more accessible initiation regions, are translationally favoured. Conversely, osmotic upshift causes a global reduction of highly-translated transcripts with high copy numbers, allowing reallocation of translation resources to undegraded and newly synthesised mRNAs', 'Series_overall_design': 'Comparing global transcriptional and translational control by mRNA-Seq and Ribosome Profiling (mRNA-Seq of ribosome protected fragments – RPF)', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Zoya,,Ignatova', 'Series_sample_id': 'GSM1680893', 'Series_contact_name': 'Alexander,,Bartholomaeus', 'Series_contact_email': 'bartholomaeus.alexander@gmail.com', 'Series_contact_laboratory': 'Section 3.7 Geomicrobiology', 'Series_contact_department': 'Department 3 Geochemie', 'Series_contact_institute': 'Helmholtz Centre Potsdam, GFZ German Research Centre for Geosciences', 'Series_contact_address': 'Telegrafenberg C', 'Series_contact_city': 'Potsdam', 'Series_contact_zip/postal_code': '14473', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE68nnn/GSE68762/suppl/GSE68762_RAW.tar', 'Series_platform_id': 'GPL20183', 'Series_platform_taxid': '1403831', 'Series_sample_taxid': '1403831', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA283800'}\n",
      "91 PRJNA284399 GSE69047 {'Series_title': 'Time-Resolved Proteomics Extends Ribosome Profiling-Based Measurements of Protein Synthesis Dynamics', 'Series_geo_accession': 'GSE69047', 'Series_status': 'Public on May 18 2017', 'Series_submission_date': 'May 19 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28578850', 'Series_summary': 'Ribosome profiling is a widespread tool for studying translational dynamics in human cells.  Its central assumption is that ribosome footprint density on a transcript quantitatively reflects protein synthesis.  Here, we test this assumption using pulsed-SILAC (pSILAC) high-accuracy targeted proteomics.  We focus on multiple myeloma cells exposed to bortezomib, a first-line chemotherapy and proteasome inhibitor. In the absence of drug effects, we found that direct measurement of protein synthesis by pSILAC correlated well with indirect measurement of synthesis from ribosome footprint density.  This correlation, however, broke down under bortezomib-induced stress. By developing a statistical model integrating longitudinal proteomic and mRNA-seq measurements, we found that proteomics could directly detect global alterations in translational rate caused by bortezomib; these changes are not detectable by ribosomal profiling alone.  Further, by incorporating pSILAC data into a gene expression model, we predict cell-stress specific proteome remodeling events.  These results demonstrate that pSILAC provides an important complement to ribosome profiling in measuring proteome dynamics.', 'Series_overall_design': 'Timecourse experiment with six points over 48hr after bortezomib exposure in MM.1S myeloma cells.  mRNA-seq and ribosome profiling data at each time point.', 'Series_type': 'Other', 'Series_contributor': 'A,P,Wiita', 'Series_sample_id': 'GSM1691207', 'Series_contact_name': 'Arun,P.,Wiita', 'Series_contact_department': 'Laboratory Medicine', 'Series_contact_institute': 'University of California, San Francisco', 'Series_contact_address': '185 Berry St., Suite 290', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94107', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE69nnn/GSE69047/suppl/GSE69047_mRNA-seq_and_footprint_reads_timecourse.xlsx', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP058501'}\n",
      "92 PRJNA285265 GSE69350 {'Series_title': 'Transcriptome and translatome landscape of Streptomyces coelicolor M145', 'Series_geo_accession': 'GSE69350', 'Series_status': 'Public on May 28 2016', 'Series_submission_date': 'May 28 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27251447', 'Series_summary': 'We have integrated nucleotide resolution genome-scale measurements of the transcriptome and translatome of the Streptomyces coelicolor A3(2), the model antibiotic-producing actinomycete. Our systematic study determined 3,473 transcription start sites, leading to discovery of a high proportion (~21%) of leaderless mRNAs and 230 non-coding RNAs; this enabled deduction of promoter architecture on a genome-scale. Ribosome profiling analysis revealed that the translation efficiency was negatively correlated for secondary metabolic genes. These results provide novel fundamental insights into translational regulation of secondary metabolism that enables rational synthetic biology approaches to awaken such ‘silent’ secondary metabolic pathways.', 'Series_overall_design': 'Profiles of primary transcripts, whole transcripts, and ribosome protected fragments (RPFs) of  Streptomyces coelicolor were generated by deep sequencing using Illumina Miseq.', 'Series_type': 'Other', 'Series_contributor': 'Byung-Kwan,,Cho', 'Series_sample_id': 'GSM1698373', 'Series_contact_name': 'Yujin,,Jeong', 'Series_contact_email': 'dbwls54@gmail.com', 'Series_contact_institute': 'KAIST', 'Series_contact_address': '291 Daehak-ro', 'Series_contact_city': 'Daejeon', 'Series_contact_zip/postal_code': '34141', 'Series_contact_country': 'South Korea', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE69nnn/GSE69350/suppl/GSE69350_TSS_processed.txt.gz', 'Series_platform_id': 'GPL20259', 'Series_platform_taxid': '1902', 'Series_sample_taxid': '1902', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP058830'}\n",
      "93 PRJNA285961 GSE69602 {'Series_title': 'Ribosome profiling analysis of Dengue Virus', 'Series_geo_accession': 'GSE69602', 'Series_status': 'Public on Sep 01 2016', 'Series_submission_date': 'Jun 05 2015', 'Series_last_update_date': 'Feb 10 2020', 'Series_pubmed_id': '29321322', 'Series_summary': 'Dengue virus is an + strand RNA virus. We have carried our infections of human cells with Dengue and analyzed the translation, replication, and localization of the Dengue RNA. This allowed for clear definition of the life cycle of the Dengue virus inside a host cell. We also assessed the host response to Dengue virus, finding that a large fraction of the translational response is due to Interferon function.', 'Series_overall_design': 'Translational and transcriptional analysis of the cellular response to Dengue virus infection', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Mariano,A,Garcia-Blanco', 'Series_sample_id': 'GSM1704618', 'Series_contact_name': 'David,,Reid', 'Series_contact_laboratory': 'Nicchitta', 'Series_contact_department': 'Biochemistry', 'Series_contact_institute': 'Duke University', 'Series_contact_address': '436 Nanaline Duke Bldg., Box 3709 DUMC', 'Series_contact_city': 'Durham', 'Series_contact_state': 'NC', 'Series_contact_zip/postal_code': '27710', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE69nnn/GSE69602/suppl/GSE69602_dengueExpression-fractionated.xlsx', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP059187'}\n",
      "94 PRJNA286218 GSE69699 {'Series_title': 'Genome-wide translational analysis of the effect of acute endurance exercise on mouse gastrocnemius', 'Series_geo_accession': 'GSE69699', 'Series_status': 'Public on Feb 16 2016', 'Series_submission_date': 'Jun 09 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26845575', 'Series_summary': 'Conclusion: There were diverse regulation between transcription and translation. Although many focused on overall protein synthesis to understand the effect of exercise, translational regulation of individual genes are required.', 'Series_overall_design': 'Transcriptional and translational profiles of mouse gastrocnemius with or without acute endurance exercise were generated using Ion PGM sequencer.', 'Series_type': 'Other', 'Series_contributor': 'Hiroaki,,Sako', 'Series_sample_id': 'GSM1707655', 'Series_contact_name': 'Hiroaki,,Sako', 'Series_contact_institute': 'Waseda University', 'Series_contact_address': '2-579-15', 'Series_contact_city': 'Tokorozawa', 'Series_contact_zip/postal_code': '359-1192', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE69nnn/GSE69699/suppl/GSE69699_RAW.tar', 'Series_platform_id': 'GPL16331', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP059282'}\n",
      "95 PRJNA286792 GSE69800 {'Series_title': 'Ribosome profiling analysis of GADD34 null cells', 'Series_geo_accession': 'GSE69800', 'Series_status': 'Public on Oct 01 2015', 'Series_submission_date': 'Jun 11 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27161320', 'Series_summary': 'Accumulation of misfolded proteins in the endoplasmic reticulum (ER) triggers the unfolded protein response (UPR), which results in the increased phosphorylation of the eukaryotic initiation factor, eIF2a, and widespread translational repression. Protein synthesis is subsequently restored following the stress-induced transcriptional upregulation of GADD34 (growth arrest and DNA damage transcript 34) protein, a regulator of an eIF2a phosphatase. Genome-wide ribosome foot-printing in WT and GADD34-/- MEFs established that GADD34 mRNA is translated in unstressed cells and identified numerous mRNAs, whose translation was dependent on GADD34 even in the absence of ER stress. Following UPR activation, temporal analyses showed that the translational profile in GADD34-/- MEFs was stalled, displaying a pattern that mirrors the early response to UPR in WT MEFs. Basal GADD34 expression is also required for de-repression of translation and displacement of ER-bound polysomes that occur in early UPR. Thus, the overall UPR response is delayed in the GADD34-/- MEFs, gradually recovering as CReP expression increased. These studies reveal a critical role for basal GADD34 in the propagation of UPR signals in MEFs and mice and suggest that delayed UPR signaling protects GADD34-/- mice from tunicamycin-induced renal toxicity.', 'Series_overall_design': 'Translational and transcriptional analysis mouse embryonic fibroblasts lacking GADD34 during ER stress', 'Series_type': 'Other', 'Series_contributor': 'Shirish,,Shenolikar', 'Series_sample_id': 'GSM1709051', 'Series_contact_name': 'David,,Reid', 'Series_contact_laboratory': 'Nicchitta', 'Series_contact_department': 'Biochemistry', 'Series_contact_institute': 'Duke University', 'Series_contact_address': '436 Nanaline Duke Bldg., Box 3709 DUMC', 'Series_contact_city': 'Durham', 'Series_contact_state': 'NC', 'Series_contact_zip/postal_code': '27710', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE69nnn/GSE69800/suppl/GSE69800_gadd34-summary-expression.xlsx', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP059390'}\n",
      "96 PRJNA287766 GSE70159 {'Series_title': 'Dietary selenium levels affect selenoprotein expression and support the interferon-γ and IL-6 immune response pathways in mice [Ribosome Profiling]', 'Series_geo_accession': 'GSE70159', 'Series_status': 'Public on Aug 06 2015', 'Series_submission_date': 'Jun 23 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26258789', 'Series_summary': 'Mice were fed Se-deficient or Se-adequate diets for 6 weeks. Liver and lung tissue were harvested and processed for RNA-Seq, ribosome profiling, and microarray analysis. From these studies, we identified changes in mRNA levels and translation of selenoprotein genes and genes regulated by interferon-gamma. Cytokine profiles of serum indicated that interferon-gamma and IL-6 levels were increased in the Se-adequate mice relative to Se-deficient mice.', 'Series_overall_design': 'Ribosome profiling of liver tissue from mice fed Se-deficient or Se-adequate diets', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Bradley,,Carlson', 'Series_sample_id': 'GSM1717858', 'Series_contact_name': 'Michael,T,Howard', 'Series_contact_email': 'mhoward@genetics.utah.edu', 'Series_contact_phone': '801-585-1927', 'Series_contact_laboratory': 'Howard', 'Series_contact_department': 'Human Genetics', 'Series_contact_institute': 'University of Utah', 'Series_contact_address': '15 N 2030 E', 'Series_contact_city': 'Salt Lake City', 'Series_contact_state': 'UT', 'Series_contact_zip/postal_code': '84112', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE70nnn/GSE70159/suppl/GSE70159_10423_EdgeR.txt.gz', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP059780'}\n",
      "97 PRJNA287879 GSE70211 {'Series_title': 'Rocaglamide A converts RNA helicase eIF4A into a sequence-specific translational repressor', 'Series_geo_accession': 'GSE70211', 'Series_status': 'Public on Jun 15 2016', 'Series_submission_date': 'Jun 24 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27309803', 'Series_summary': 'Rocaglamide A (RocA) typifies a novel class of protein synthesis inhibitors that selectively kill aneuploid tumor cells and repress translation of specific mRNAs. RocA targets eukaryotic initiation factor 4A (eIF4A), the prototypical DEAD-box RNA helicase, and its mRNA selectivity is proposed to reflect highly structured 5′ UTRs that are very dependent on eIF4A-mediated unwinding. Here, we show that secondary structure in 5′ UTRs is only a minor determinant for RocA selectivity and RocA does not repress translation by reducing eIF4A activity. Rather, in vitro and in vivo, RocA clamps eIF4A onto a specific sequence motif even after ATP hydrolysis. This artificially clamped eIF4A blocks 43S scanning, leading to premature, upstream translation initiation and reducing gene expression on transcripts bearing the RocA-eIF4A target sequence. In elucidating the mechanism of this lead anti-cancer compound and explaining its mRNA selectivity, we provide the first example of a drug stabilizing sequence-specific RNA-protein interactions.', 'Series_overall_design': 'We stably expressed SBP (streptavidin binding peptide)-tagged eIF4A in HEK 293T-REx cells and purified it via M270 streptavidin beads (life techonologies).', 'Series_type': 'Other', 'Series_contributor': 'Nicholas,T,Ingolia', 'Series_sample_id': 'GSM1720825', 'Series_contact_name': 'Shintaro,,Iwasaki', 'Series_contact_email': 'shintaro.iwasaki@riken.jp', 'Series_contact_department': 'Cluster for Pioneering Research', 'Series_contact_institute': 'RIKEN', 'Series_contact_address': 'S205 Bioscience Bldg. 2-1 Hirosawa', 'Series_contact_city': 'Wako', 'Series_contact_state': 'Saitama', 'Series_contact_zip/postal_code': '351-0198', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE70nnn/GSE70211/suppl/GSE70211_RAW.tar', 'Series_platform_id': 'GPL15228', 'Series_platform_taxid': '32630', 'Series_sample_taxid': '32630', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP059825'}\n",
      "98 PRJNA289569 GSE70802 {'Series_title': 'Profiles of ribosome-associated mRNAs regulated by expression of wild-type (WT) or R534H variant of DDX3 with or without Sodium Arsenite treatment [Ribo-Seq]', 'Series_geo_accession': 'GSE70802', 'Series_status': 'Public on Apr 21 2016', 'Series_submission_date': 'Jul 11 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27058758', 'Series_summary': 'By using ribosome profiling, we demonstrate that catalytic activity of the RNA helicase DDX3 is generally required for mediating translation repression under stress. Intriguingly, however, a cancer-related DDX3 variant DDX3 R534H selectively preserves translation of genes encoding core nucleosome components. Additionally, DDX3 variants also shift ORF usage on select genes, such as RPLP1 and stress-response factors as an added mechanism of translation regulation during stress. Thus, DDX3 through both extensive and selective interactions with RNA and the ribosomal machinery helps to remodel the translational landscape under stress and in cancer.', 'Series_overall_design': 'Cells expressing either DDX3 WT or R534H variant were treated with or without sodium arsenite and lysed in the presence of cyclohexime. Ribosome-protected mRNA fragments after RNase I treatment were recovered and sequenced.', 'Series_type': 'Other', 'Series_contributor': 'Yoon-Jae,,Cho', 'Series_sample_id': 'GSM1819228', 'Series_contact_name': 'Sekyung,,Oh', 'Series_contact_email': 'ohskjhmi@gmail.com', 'Series_contact_phone': '650-725-3354', 'Series_contact_department': 'Neurology and Neurological Sciences', 'Series_contact_institute': 'Stanford University School of Medicine', 'Series_contact_address': '1201 Welch Road, MSLS P224', 'Series_contact_city': 'Stanford', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94305', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE70nnn/GSE70802/suppl/GSE70802_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP060676'}\n",
      "99 PRJNA290912 GSE71333 {'Series_title': 'Genome-wide analysis of Musashi-2 targets reveals novel functions in governing epithelial cell migration', 'Series_geo_accession': 'GSE71333', 'Series_status': 'Public on Apr 01 2016', 'Series_submission_date': 'Jul 24 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27034466', 'Series_summary': '', 'Series_overall_design': 'Identification of direct Musashi-2 targets in keratinocytes through the use of RNA-Seq, Ribosome-Profiling, and Msi2-HITS-CLIP', 'Series_type': 'Other', 'Series_contributor': 'R,,Yi', 'Series_sample_id': 'GSM1832735', 'Series_contact_name': 'Kent,Augustus,Riemondy', 'Series_contact_email': 'kent.riemondy@cuanschutz.edu', 'Series_contact_department': 'Biochemistry and Molecular Genetics', 'Series_contact_institute': 'University of Colorado School of Medicine', 'Series_contact_address': '12801 E 17th Ave', 'Series_contact_city': 'Aurora', 'Series_contact_state': 'Colorado', 'Series_contact_zip/postal_code': '80045', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE71nnn/GSE71333/suppl/GSE71333_rnaseq_counts.txt.gz', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP061564'}\n",
      "100 PRJNA292112 GSE71808 {'Series_title': 'Post-translational buffering leads to convergent protein expression levels between primates', 'Series_geo_accession': 'GSE71808', 'Series_status': 'Public on May 10 2018', 'Series_submission_date': 'Aug 06 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '29950183', 'Series_summary': '', 'Series_overall_design': 'Using ribosome profiling assays, we measured levels of protein translation transcriptome-wide in lymphoblastoid cell lines derived from 4 HapMap Yoruba individuals, 5 chimpanzee individuals, and 5 rhesus macaque individuals. Note that the data uploaded for the 4 Yoruba individuals were generated by further sequencing the libraries generated as part of the Battle et al QTL mapping study (GSE61742). About 30~50% more data were generated for these individuals and the files uploaded here combined the new sequencing data with the old data from GSE61742 for each respective cell line.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Amy,,Mitrano', 'Series_sample_id': 'GSM2601312', 'Series_contact_name': 'Sidney,,Wang', 'Series_contact_email': 'siddisis@gmail.com', 'Series_contact_department': 'Human Genetics', 'Series_contact_institute': 'University of Chicago', 'Series_contact_address': '920 E. 58th Street, CLSC 317', 'Series_contact_city': 'Chicago', 'Series_contact_state': 'IL', 'Series_contact_zip/postal_code': '60637', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE71nnn/GSE71808/suppl/GSE71808_Primate_RPF_count_by_gene_GEO.table.txt.gz', 'Series_platform_id': 'GPL19148', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP062129'}\n",
      "101 PRJNA292778 GSE72030 {'Series_title': 'Global Translational Impacts of the Loss of the tRNA Modification t6A in Yeast', 'Series_geo_accession': 'GSE72030', 'Series_status': 'Public on Feb 15 2016', 'Series_submission_date': 'Aug 13 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26798630', 'Series_summary': 'The universal tRNA modification t6A is found at position 37 of nearly all tRNAs decoding ANN codons. The importance of t6A in decoding has been studied for the past 40 years, but the cellular role has remained unclear. In this work, we examined the genome-wide effect to translation during the absence of t6A. In accordance with the previous experiments using single gene reporters, absence of t6A led to a increase of upstream non-AUG initiation of translation and increased translation ambiguities. Surprisingly, absence of t6A did not lead to a global and cataclysmic frame-shifting, but instead occurred only in a few genes. Restoration of protein homeostasis through the overexpression of Unfolded Protein Response (UPR) factors or the addition of L-homoserine suppressed the slow growth phenotype seen in t6A-defcient strains.', 'Series_overall_design': '4 samples', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Valérie,,de Crécy-Lagard', 'Series_sample_id': 'GSM1850255', 'Series_contact_name': 'Olivier,,Namy', 'Series_contact_email': 'olivier.namy@i2bc.paris-saclay.fr', 'Series_contact_laboratory': 'Genomic, structure and Translation', 'Series_contact_department': 'I2BC', 'Series_contact_institute': 'CNRS, Université Paris-Sud', 'Series_contact_address': 'Rue Gregor Mendel, bat 400', 'Series_contact_city': 'Orsay', 'Series_contact_zip/postal_code': '91400', 'Series_contact_country': 'France', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE72nnn/GSE72030/suppl/GSE72030_RAW.tar', 'Series_platform_id': 'GPL18085', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP062368'}\n",
      "102 PRJNA292862 GSE72066 {'Series_title': 'Ribosome Profiling of mouse skin squamous tumours', 'Series_geo_accession': 'GSE72066', 'Series_status': 'Public on Jun 16 2016', 'Series_submission_date': 'Aug 14 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27306184', 'Series_summary': 'Purpose: Ribosome profiling has revolutionized systems-based analysis and which produces a “global snapshot” of all the ribosomes translationally active in a cell at a particular moment. The goals of this study are to first apply ribosome profiling to in vivo samples for the first time and in particular to stem cells and tumours and second to determine which mRNAs are being actively translated in these particular situations. Although much is known about gene expression regulation, little is known about how protein translation regulation can affect stem cell differentiation and tumour progression. Methods: several replicates of  ribosome and mRNA profiles of wild-type (WT) and NSun2 -/- mouse skin squamous tumours were generated by deep sequencing, using Illumina HiSeq platform. Results: Our analyses reveal that activation of stress response pathways in vivo drives both a global reduction of protein synthesis and altered translation of specific mRNAs that together promote stem cell functions and tumourigenesis.', 'Series_overall_design': 'Ribosome profiles of wild-type (WT) and NSun2 -/- mouse skin squamous tumours', 'Series_type': 'Other', 'Series_contributor': 'Sabine,,Dietmann', 'Series_sample_id': 'GSM1854042', 'Series_contact_name': 'Sabine,,Dietmann', 'Series_contact_institute': 'Washington University School of Medicine', 'Series_contact_address': '4565 McKinley Avenue', 'Series_contact_city': 'St Louis', 'Series_contact_state': 'MO', 'Series_contact_zip/postal_code': '63110', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE72nnn/GSE72066/suppl/GSE72066_RAW.tar', 'Series_platform_id': 'GPL9250', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP062409'}\n",
      "103 PRJNA294100 GSE72463 {'Series_title': 'Advancing Trypanosoma brucei genome annotation through ribosome profiling and spliced leader mapping', 'Series_geo_accession': 'GSE72463', 'Series_status': 'Public on Dec 22 2015', 'Series_submission_date': 'Aug 27 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26393539', 'Series_summary': 'Since the initial publication of the trypanosomatid genomes, curation has been ongoing. Here we apply the technique of ribosome profiling to Trypanosoma brucei, identifying 223 new coding regions by virtue of ribosome occupancy in the corresponding transcripts. A small number of these putative genes correspond to extra copies of previously annotated genes but 85% are novel. The median size of these novels CDSs is small (74 aa) indicating that past annotation work has excelled at detecting large CDSs. Of the unique CDSs discovered here, over half have candidate orthologues in other trypanosomatid genomes, most of which were not yet annotated as genes. Still, approximately one-third of the new CDSs were found only in T. brucei subspecies. When combined with RNA-seq and spliced leader mapping, we were able to definitively revise the start sites for 430 CDSs as compared to the current gene models. Such data also allowed us to use a structured approach to eliminate 701 putative genes as protein-coding. Finally, the data pointed to several regions of the genome that had sequence errors that altered coding region boundaries.', 'Series_overall_design': 'Ribosome profiling and mRNA libraries were constructed in triplicate from in vitro  PCF and in vivo  BF lifestages of theT. brucei Treu927 and in vitro T. brucei  Lister427, to evaluate role of translational gene regulation', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Peter,J,Myler', 'Series_sample_id': 'GSM1862971', 'Series_contact_name': 'Gowthaman,,Ramasamy', 'Series_contact_email': 'Gowthaman.Ramasamy@seattlebiomed.org', 'Series_contact_phone': '206-256-7188', 'Series_contact_laboratory': 'Bioinformatics Core & Myler Lab', 'Series_contact_department': 'GHBC', 'Series_contact_institute': 'SEATTLE BIOMEDICAL RESEARCH INSTITUTE', 'Series_contact_address': '307 WESTLAKE AVE N', 'Series_contact_city': 'SEATTLE', 'Series_contact_state': 'WASHINGTON', 'Series_contact_zip/postal_code': '98109', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE72nnn/GSE72463/suppl/GSE72463_RAW.tar', 'Series_platform_id': 'GPL10599', 'Series_platform_taxid': '5691', 'Series_sample_taxid': '5691', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP062914'}\n",
      "104 PRJNA295296 GSE72899 {'Series_title': 'Clarifying the translational pausing landscape in bacteria by ribosome profiling', 'Series_geo_accession': 'GSE72899', 'Series_status': 'Public on Jan 13 2016', 'Series_submission_date': 'Sep 10 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26776510', 'Series_summary': 'The rate of protein synthesis varies according to the mRNA sequence in ways that affect gene expression. Global analysis of translational pausing is now possible with ribosome profiling. Here, we revisit an earlier report that Shine-Dalgarno sequences are the major determinant of translational pausing in bacteria. Using refinements in the profiling method as well as biochemical assays, we find that SD motifs have little (if any) effect on elongation rates. We argue that earlier evidence of pausing arose from two factors. First, in previous analyses, pauses at Gly codons were difficult to distinguish from pauses at SD motifs. Second, and more importantly, the initial study preferentially isolated long ribosome-protected mRNA fragments that are enriched in SD motifs. These findings clarify the landscape of translational pausing in bacteria as observed by ribosome profiling.', 'Series_overall_design': 'Ribosome profiling (three replicates) and RNAseq (two replicates) of E. coli MG1655', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Allen,R,Buskirk', 'Series_sample_id': 'GSM1874192', 'Series_contact_name': 'Allen,R,Buskirk', 'Series_contact_email': 'buskirk@jhmi.edu', 'Series_contact_department': 'Molecular Biology and Genetics', 'Series_contact_institute': 'Johns Hopkins University School of Medicine', 'Series_contact_address': '725 N. Wolfe St', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21205', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE72nnn/GSE72899/suppl/GSE72899_RAW.tar', 'Series_platform_id': 'GPL18956', 'Series_platform_taxid': '511145', 'Series_sample_taxid': '511145', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP063562'}\n",
      "105 PRJNA295556 GSE72996 {'Series_title': 'An integrated approach combining RNA-seq, Ribo-seq and Proteomics identifies the coding potential of Mycobacterium abscessus', 'Series_geo_accession': 'GSE72996', 'Series_status': 'Public on Mar 30 2016', 'Series_submission_date': 'Sep 14 2015', 'Series_last_update_date': 'Jul 18 2019', 'Series_pubmed_id': '27495169', 'Series_summary': 'The objective of this study is to develop an integrated map for the M. abscessus ATCC19977 organism. Among the data types contributing to the development of this map are next-generation RNA-sequencing, differential RNA-sequencing, ribosome profiling, and mass spectrometry.', 'Series_overall_design': 'The design of the study consists of two biological samples, each with one technical replication; both biological samples were subjected to identical conditions.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Kevin,,Conlon', 'Series_sample_id': 'GSM1876371', 'Series_contact_name': 'Brendan,,Loftus', 'Series_contact_email': 'brendan.loftus@ucd.ie', 'Series_contact_department': 'School of Medicine & Medical Science', 'Series_contact_institute': 'University College Dublin', 'Series_contact_address': 'Conway Institute, Belfield, Dublin 4', 'Series_contact_city': 'Dublin', 'Series_contact_zip/postal_code': 'D4', 'Series_contact_country': 'Ireland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE72nnn/GSE72996/suppl/GSE72996_RAW.tar', 'Series_platform_id': 'GPL20911', 'Series_platform_taxid': '561007', 'Series_sample_taxid': '561007', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA295556'}\n",
      "106 PRJNA297039 GSE73357 {'Series_title': 'Heterogeneous ribosomes exist and selectively translate distinct subpools of mRNAs in stem cells', 'Series_geo_accession': 'GSE73357', 'Series_status': 'Public on Jun 15 2017', 'Series_submission_date': 'Sep 23 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28625553', 'Series_summary': 'Emerging studies have linked the ribosome to more selective control of gene regulation. However, an outstanding question is whether ribosome heterogeneity at the level of core ribosomal proteins (RPs) enables ribosomes to preferentially translate specific mRNAs genome-wide. Here, we measured the absolute abundance of RPs in translating ribosomes and profiled transcripts that are enriched or depleted from select subsets of ribosomes within embryonic stem cells. We find that heterogeneity in RP composition endows ribosomes with different selectivity for translating subpools of transcripts including those controlling metabolism, the cell cycle, and development. As a paradigm example, we show that mRNAs enriched in binding to RPL10A/uL1-containing ribosomes require RPL10A/uL1 for their efficient translation. Within several of these transcripts, we find this level of regulation is mediated, at least in part, by internal ribosome entry sites. Together, these results reveal a critical functional link between ribosome heterogeneity and the post-transcriptional circuitry of gene expression.', 'Series_overall_design': 'To identify the transcripts specifically translated by ribosomes demarcated by RPL10A or RPS25, we enriched for RPL10A or RPS25-containing ribosomes by immunoprecipitation (IP), and performed RPL10A-Ribo-Seq and RPS25-Ribo-Seq (2 replicates each), with the control being total Ribo-Seq (2 replicates each). To further assess the translation efficiencies of mRNAs upon Rpl10a knockdown at genome-wide scale, we performed RNA-Seq (2 replicates each) of purified RNAs from combined medium and heavy polysome fractions containing the most actively translating ribosomes (>=4 ribosomes along a mRNA molecule), and from all other fractions containing the free ribonucleoproteins (RNPs), 40S/60S ribosomal free subunits, 80S/monosomes as well as light polysomes (2-3 ribosomes per mRNA molecule).', 'Series_type': 'Other', 'Series_contributor': 'Maria,,Barna', 'Series_sample_id': 'GSM2579690', 'Series_contact_name': 'Zhen,,Shi', 'Series_contact_institute': 'Stanford', 'Series_contact_address': '279 Campus Drive', 'Series_contact_city': 'Stanford', 'Series_contact_state': 'California', 'Series_contact_zip/postal_code': '94305', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE73nnn/GSE73357/suppl/GSE73357_RAW.tar', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP064202'}\n",
      "107 PRJNA297338 GSE73565 {'Series_title': 'Complementary Post Transcriptional Regulatory Information is Detected by PUNCH-P and Ribosome Profiling', 'Series_geo_accession': 'GSE73565', 'Series_status': 'Public on Sep 28 2016', 'Series_submission_date': 'Sep 29 2015', 'Series_last_update_date': 'May 15 2019', 'Series_summary': 'The results reported here support the conjecture that for gaining a full post-transcriptional regulation picture one should integrate the two approaches.', 'Series_overall_design': 'Two Ribo-Seq experiments, which included parallel RNA-Seq, one with 3 replicates, and the other with one replicate, totalling 4 technical replicates for the G1 and M phases of the cell cycle, respectively (totalling 16 samples), were performed.', 'Series_type': 'Other', 'Series_contributor': 'Tamir,,Tuller', 'Series_sample_id': 'GSM1898021', 'Series_contact_name': 'Tamir,,Tuller', 'Series_contact_email': 'tamirtul@post.tau.ac.il', 'Series_contact_laboratory': 'Computational, Systems, and Synthetic Biology', 'Series_contact_department': 'Biomedical Engineering', 'Series_contact_institute': 'Tel Aviv University', 'Series_contact_address': 'P.O. Box 39040', 'Series_contact_city': 'Tel Aviv-Yafo', 'Series_contact_zip/postal_code': '6997801', 'Series_contact_country': 'Israel', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE73nnn/GSE73565/suppl/GSE73565_RC_Data.xlsx', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP064313'}\n",
      "108 PRJNA299160 GSE74139 {'Series_title': 'A regression-based analysis of ribosome-profiling data reveals a conserved complexity to mammalian translation', 'Series_geo_accession': 'GSE74139', 'Series_status': 'Public on Dec 03 2015', 'Series_submission_date': 'Oct 19 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26638175', 'Series_summary': 'A fundamental goal of genomics is to identify the complete set of expressed proteins. Automated annotation strategies rely on assumptions about protein-coding sequences (CDSs), e.g., they are conserved, do not overlap, and exceed a minimum length. However, an increasing number of newly discovered proteins violate these rules. Here we present an experimental and analytical framework, based on ribosome profiling and linear regression, for systematic identification and quantification of translation. Application of this approach to lipopolysaccharide-stimulated mouse dendritic cells and HCMV-infected human fibroblasts identifies thousands of novel CDSs, including micropeptides and variants of known proteins, that bear the hallmarks of canonical translation and exhibit comparable translation levels and dynamics to annotated CDSs. Remarkably, many translation events are identified in both mouse and human cells even when the peptide sequence is not conserved. Our work thus reveals an unexpected complexity to mammalian translation suited to provide both conserved regulatory or protein-based functions.', 'Series_overall_design': \"37 ribosome footprint samples from mouse BMDCs across a time course of LPS stimulation and in the presence or absence of translation inhibitors cycloheximide, harringtonine, or lactimidomycin. A mock (non-LPS stimulated) time course is also included, collected in the presence of cycloheximide. Two replicates are available for the LPS timecourse under CHX treatment. Reads include the 3' linker sequence CTGTAGGCACCATCAAT.\", 'Series_type': 'Other', 'Series_contributor': 'Jonathan,S,Weissman', 'Series_sample_id': 'GSM1911129', 'Series_contact_name': 'Alexander,P,Fields', 'Series_contact_email': 'alexander.fields@ucsf.edu', 'Series_contact_laboratory': 'Weissman', 'Series_contact_department': 'Cellular and Molecular Pharmacology', 'Series_contact_institute': 'University of California San Francisco', 'Series_contact_address': '1700 4th Street', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE74nnn/GSE74139/suppl/GSE74139_RAW.tar', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP064972'}\n",
      "109 PRJNA299306 GSE74197 {'Series_title': 'Ribosome hibernation factor promotes Staphyloccocal survival and differentially represses translation', 'Series_geo_accession': 'GSE74197', 'Series_status': 'Public on Mar 29 2016', 'Series_submission_date': 'Oct 20 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27001516', 'Series_summary': 'We sequenced a total of 16 cDNA libararies derived from fragmented total mRNA and ribosome protected mRNAs from S. aureus hpf mutant (NE838) and its parental strain JE2 grown in tryptic soy broth (TSB) or minimal medium (MM). The data represented 2 independent biological replicates.', 'Series_overall_design': 'Examination of the impact of hpf on global S.aureus translatome, mRNA abundance and the ribosome density along the transcripts.', 'Series_type': 'Other', 'Series_contributor': 'M.-N.,F,Yap', 'Series_sample_id': 'GSM1914390', 'Series_contact_name': 'Mee-Ngan,F.,Yap', 'Series_contact_email': 'frances.yap@northwestern.edu', 'Series_contact_phone': '312-503-3793', 'Series_contact_laboratory': 'Searle 3-430', 'Series_contact_department': 'Microbiology-Immunology', 'Series_contact_institute': 'Northwestern University Feinberg School of Medicine', 'Series_contact_address': '320 E Superior St', 'Series_contact_city': 'Chicago', 'Series_contact_state': 'IL', 'Series_contact_zip/postal_code': '60611', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE74nnn/GSE74197/suppl/GSE74197_RAW.tar', 'Series_platform_id': 'GPL17452', 'Series_platform_taxid': '1280', 'Series_sample_taxid': '1280', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP065033'}\n",
      "110 PRJNA299827 GSE74365 {'Series_title': 'Ribosomal profiling in Cystic Fibrosis Bronchial Epithelial cells', 'Series_geo_accession': 'GSE74365', 'Series_status': 'Public on May 18 2017', 'Series_submission_date': 'Oct 26 2015', 'Series_last_update_date': 'Apr 24 2024', 'Series_pubmed_id': '38580646', 'Series_summary': '', 'Series_overall_design': 'Ribosome profiling of cells at steady state. 3 biological replicates.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Zoya,,Ignatova', 'Series_sample_id': 'GSM1918969', 'Series_contact_name': 'Robert,,Rauscher', 'Series_contact_email': 'robert.rauscher@chemie.uni-hamburg.de', 'Series_contact_laboratory': 'Molecular Biology and Biochemistry', 'Series_contact_department': 'Biochemistry', 'Series_contact_institute': 'University Hamburg', 'Series_contact_address': 'Martin-Luther-King-Platz 6', 'Series_contact_city': 'Hamburg', 'Series_contact_zip/postal_code': '20146', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE74nnn/GSE74365/suppl/GSE74365_RAW.tar', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP065282'}\n",
      "111 PRJNA300274 GSE74393 {'Series_title': 'Cotranslational signal independent SRP preloading during membrane targeting', 'Series_geo_accession': 'GSE74393', 'Series_status': 'Public on Aug 03 2016', 'Series_submission_date': 'Oct 27 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27487213', 'Series_summary': 'Cotranslational targeting into the endoplasmic reticulum (ER) by the Signal Recognition Particle (SRP) is a key event determining polypeptide fate in eukaryotic cells. Here, we globally define the principles and mechanisms of SRP binding and ER targeting in vivo. Cotranslational targeting through SRP is the dominant route into the ER for all secretory proteins, regardless of targeting signal characteristics. Cytosolic SRP functions in a pioneer translation round that builds a membrane-resident mRNAs pool, explaining how low SRP levels suffice for the secretory load. SRP does not induce an elongation arrest; consequently, kinetic competition between targeting and translation elongation dictates which substrates are translocated post-translationally. Unexpectedly, SRP binds most secretory ribosomal complexes before targeting signals are synthesized. We show non-coding mRNA elements can promote signal-independent SRP pre-recruitment. Our study defines the complex kinetic interplay between elongation and determinants in the polypeptide and mRNA modulating SRP-substrate selection and membrane targeting in vivo.', 'Series_overall_design': 'Ribosome profiling (RiboSeq) and RNA-seq of subcellular fractions of ribosomes. Soluble and membrane bound ribosomes are separated by centrifugation, and SRP-bound ribosomes are immunoprecipitated from the soluble fraction. Polysomes and monosomes are separated by sucrose gradient ultracentrifugation.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Judith,,Frydman', 'Series_sample_id': 'GSM2241688', 'Series_contact_name': 'Justin,William,Chartron', 'Series_contact_email': 'chartron@stanford.edu', 'Series_contact_laboratory': 'Frydman', 'Series_contact_department': 'Biology', 'Series_contact_institute': 'Stanford University', 'Series_contact_address': 'Clark Center E200, 318 Campus Drive', 'Series_contact_city': 'Stanford', 'Series_contact_state': 'California', 'Series_contact_zip/postal_code': '94305', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE74nnn/GSE74393/suppl/GSE74393_SupplementaryTable1.xlsx', 'Series_platform_id': 'GPL21656', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP065322'}\n",
      "112 PRJNA301154 GSE74683 {'Series_title': 'A dynamic unfolded protein response contributes to the control of cortical neurogenesis', 'Series_geo_accession': 'GSE74683', 'Series_status': 'Public on Dec 11 2015', 'Series_submission_date': 'Nov 04 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26651292', 'Series_web_link': 'http://www.sciencedirect.com/science/article/pii/S1534580715007194', 'Series_summary': 'The cerebral cortex contains layers of neurons sequentially generated by distinct lineage-related progenitors. At the onset of corticogenesis, the first-born progenitors are apical progenitors (APs) whose asymmetric division give birth directly to neurons. Later, they switch to indirect neurogenesis by generating intermediate progenitors (IPs), which give rise to projection neurons of all cortical layers. While a direct lineage relationship between APs and IPs has been established, the molecular mechanism that controls their transition remains elusive. Our data suggest that interfering with codon translation speed triggers endoplasmic reticulum stress and the unfolded protein response (UPR), further impairing the generation of IPs and leading to microcephaly. Moreover, we demonstrate that a progressive downregulation of UPR in cortical progenitors acts as physiological signal to amplify IPs and promotes indirect neurogenesis. Thus, our findings reveal a hitherto unrecognized contribution of UPR to cell fate acquisition during mammalian brain development.', 'Series_overall_design': 'Ribosome profiling and RNA-Seq of forebrains from E14.5 mouse embryos from wild type animals and mutants carrying a conditional knockout of ELP3 in cortical progenitors', 'Series_type': 'Other', 'Series_contributor': 'Danny,D,Nedialkova', 'Series_sample_id': 'GSM1930058', 'Series_contact_name': 'Danny,D.,Nedialkova', 'Series_contact_email': 'danny.nedialkova@mpi-muenster.mpg.de', 'Series_contact_department': 'Max Planck Research Group for RNA Biology', 'Series_contact_institute': 'Max Planck Institute for Molecular Biomedicine', 'Series_contact_address': 'Von-Esmarch-Str. 54', 'Series_contact_city': 'Münster', 'Series_contact_zip/postal_code': '48149', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE74nnn/GSE74683/suppl/GSE74683_rawCounts_mRNA.txt.gz', 'Series_platform_id': 'GPL16173', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP065804'}\n",
      "113 PRJNA303186 GSE75322 {'Series_title': 'Accurate prediction of cellular co-translational folding indicates proteins can switch from post- to co-translational folding', 'Series_geo_accession': 'GSE75322', 'Series_status': 'Public on Jan 30 2016', 'Series_submission_date': 'Nov 23 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26887592', 'Series_summary': 'The rates at which domains fold and codons are translated are important factors in determining whether a nascent protein will co-translationally fold and function or misfold and malfunction. In this study, we develop a chemical kinetic model that calculates a protein domain’s co-translational folding curve using only the domain’s bulk folding and unfolding rates and codon translation rates. We show that this model accurately predicts the course of co-translational folding measured in vivo for four different protein molecules. As part of our chemical kinetic model, we assume that there is steady-state translation kinetics through the time-course of the experiment. To prove that this assumption is valid, we performed Ribo-Seq experiments on two biological replicates of yeast cells to compare their ribosome profiles. For genes with sufficient coverage across the open reading frames in both the replicates, we compare the ribosome profiles from both replicates and see a strong correlation. This implies there is a steady-state and our assumption is valid.', 'Series_overall_design': 'Ribosome profiling of two biological replicates of S288C yeast cells', 'Series_type': 'Other', 'Series_contributor': \"Edward,P,O'Brien\", 'Series_sample_id': 'GSM1949551', 'Series_contact_name': 'Nabeel,,Ahmed', 'Series_contact_email': 'nxa176@psu.edu', 'Series_contact_laboratory': \"O'Brien Lab\", 'Series_contact_department': 'The Huck Institutes of the Life Sciences', 'Series_contact_institute': 'Pennsylvania State University', 'Series_contact_address': '407 Chemistry Building', 'Series_contact_city': 'University Park', 'Series_contact_state': 'PENNSYLVANIA', 'Series_contact_zip/postal_code': '16802', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE75nnn/GSE75322/suppl/GSE75322_list_genes_correlations.tab.gz', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP066576'}\n",
      "114 PRJNA303958 GSE75290 {'Series_title': 'Thousands of novel translated open reading frames in humans inferred by ribosome footprint profiling.', 'Series_geo_accession': 'GSE75290', 'Series_status': 'Public on Dec 25 2015', 'Series_submission_date': 'Nov 22 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27232982', 'Series_summary': 'Accurate annotations of protein coding regions are essential for understanding how genetic information is translated into biological functions. The recent development of ribosome footprint profiling provides an important new tool for measuring translation. Here we describe riboHMM, a new method that uses ribosome footprint data along with gene expression and sequence information to accurately infer translated sequences. We applied our method to human lymphoblastoid cell lines and identified 7,863 previously unannotated coding sequences, including 445 translated sequences in pseudogenes and 2,442 translated upstream open reading frames. We observed an enrichment of harringtonine-treated ribosome footprints at the inferred initiation sites, validating many of the novel coding sequences. In aggregate, the novel sequences exhibit significant signatures of purifying selection indicative of protein-coding function, suggesting that many of the novel sequences are functional. We observed that nearly 40% of bicistronic transcripts showed significant negative correlation in the levels of translation of their two coding sequences, suggesting a key regulatory role for these novel translated sequences. Despite evidence for their functional importance, the novel peptide sequences were detected by mass spectrometry at a lower rate than predicted based on data from annotated proteins, thus suggesting that many of the novel peptide products may be relatively short-lived. Our work illustrates the value of ribosome profiling for improving coding annotations, and significantly expands the set of known coding regions.', 'Series_overall_design': 'Ribosome profiling assays were performed on cell lines treated with Harringtonine to generate data for validating translation initiatoin site prediction', 'Series_type': 'Other', 'Series_contributor': 'Arbel,,Harpak', 'Series_sample_id': 'GSM1948926', 'Series_contact_name': 'Sidney,,Wang', 'Series_contact_email': 'siddisis@gmail.com', 'Series_contact_department': 'Human Genetics', 'Series_contact_institute': 'University of Chicago', 'Series_contact_address': '920 E. 58th Street, CLSC 317', 'Series_contact_city': 'Chicago', 'Series_contact_state': 'IL', 'Series_contact_zip/postal_code': '60637', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE75nnn/GSE75290/suppl/GSE75290_novel_CDS_hg19.headers.txt.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP066598'}\n",
      "115 PRJNA305643 GSE75897 {'Series_title': 'Improved ribosome-footprint and mRNA measurements provide insights into dynamics and regulation of yeast translation', 'Series_geo_accession': 'GSE75897', 'Series_status': 'Public on Feb 10 2016', 'Series_submission_date': 'Dec 10 2015', 'Series_last_update_date': 'Jul 16 2019', 'Series_pubmed_id': '27692069', 'Series_summary': \"Ribosome-footprint profiling provides genome-wide snapshots of translation, but technical challenges can confound its analysis. Here, we use improved methods to obtain ribosome-footprint profiles and mRNA abundances that more faithfully reflect gene expression in Saccharomyces cerevisiae. Our results support proposals that both the beginning of coding regions and codons matching rare tRNAs are more slowly translated. They also indicate that emergent polypeptides with as few as three basic residues within a 10-residue window tend to slow translation. With the improved mRNA measurements, the variation attributable to translational control in exponentially growing yeast was less than previously reported, and most of this variation could be predicted with a simple model that considered mRNA abundance, upstream open reading frames, cap-proximal structure and nucleotide composition, and lengths of the coding and 5'-untranslated regions. Collectively, our results provide a framework for executing and interpreting ribosome-profiling studies and reveal key features of translational control in yeast.\", 'Series_overall_design': 'Ribosome-footprint profiling and RNA-seq (total RNA, poly(A) selected, RiboMinus treated, or Ribo-Zero treated) from log-phase S. cerevisiae. The study includes a reanalysis of the two Samples from GSE53313. The reanalyzed data is linked to the Series record.', 'Series_type': 'Third-party reanalysis', 'Series_contributor': 'David,P,Bartel', 'Series_sample_id': 'GSM1969535', 'Series_contact_name': 'David,Eric,Weinberg', 'Series_contact_email': 'david.weinberg@ucsf.edu', 'Series_contact_phone': '415-514-9723', 'Series_contact_laboratory': 'Weinberg', 'Series_contact_institute': 'University of California, San Francisco', 'Series_contact_address': '600 16th Street, GH S572F', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE75nnn/GSE75897/suppl/GSE75897_RiboZero_RPKMs.txt.gz', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP067237'}\n",
      "116 PRJNA306259 GSE76117 {'Series_title': 'Redefining the translational status of 80S monosomes', 'Series_geo_accession': 'GSE76117', 'Series_status': 'Public on Feb 11 2016', 'Series_submission_date': 'Dec 17 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26871635', 'Series_summary': 'Fully assembled ribosomes exist in two populations: polysomes and monosomes. While the former has been studied extensively, to what extent translation occurs on monosomes and its importance for overall translational output remains controversial. Here, we used ribosome profiling to examine the translational status of 80S monosomes in Saccharomyces cerevisiae. We found that the vast majority of 80S monosomes are elongating, not initiating. Further, most mRNAs exhibit some degree of monosome occupancy, with monosomes predominating on nonsense-mediated decay (NMD) targets, upstream open reading frames (uORFs), canonical ORFs shorter than ~590 nucleotides and ORFs for which the total time required to complete elongation is substantially shorter than that required for initiation. Importantly, mRNAs encoding low-abundance regulatory proteins tend to be enriched in the monosome fraction. Our data highlight the importance of monosomes for the translation of highly regulated mRNAs.', 'Series_overall_design': 'We examined the translational status of single 80S ribosomes using ribosome profiling, and compared these monosome footprints to both polysome ribosome footprints and general ribosome profiling.  RNASeq libraries were also prepared from the overall sample input.', 'Series_type': 'Other', 'Series_contributor': 'J,M,Moore', 'Series_sample_id': 'GSM1974457', 'Series_contact_name': 'Erin,,Heyer', 'Series_contact_laboratory': 'Transcriptomic Research', 'Series_contact_department': 'Genomics & Epigenetics', 'Series_contact_institute': 'Garvan Institute of Medical Research', 'Series_contact_address': '384 Victoria Street', 'Series_contact_city': 'Darlinghurst', 'Series_contact_state': 'NSW', 'Series_contact_zip/postal_code': '2010', 'Series_contact_country': 'Australia', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE76nnn/GSE76117/suppl/GSE76117_GeneCounts_allLibraries.csv.gz', 'Series_platform_id': 'GPL17143', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP067514'}\n",
      "117 PRJNA311041 GSE77617 {'Series_title': 'Operon mRNAs are organized into ORF-centric structures that predict translation efficiency', 'Series_geo_accession': 'GSE77617', 'Series_status': 'Public on Jan 30 2017', 'Series_submission_date': 'Feb 05 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28139975', 'Series_summary': 'Bacterial mRNAs are organized into operons consisting of discrete open reading frames (ORFs) in a single polycistronic mRNA. Individual ORFs on the mRNA are differentially translated, with rates varying as much as 100-fold. The signals controlling differential translation are poorly understood. Our genome-wide mRNA secondary structure analysis indicated that operonic mRNAs are comprised of ORF-wide units of secondary structure that vary across ORF boundaries such that adjacent ORFs on the same mRNA molecule are structurally distinct. ORF translation rate is strongly correlated with its mRNA structure in vivo, and correlation persists, albeit in a reduced form, with its structure when translation is inhibited and with that of in vitro refolded mRNA. These data suggests that intrinsic ORF mRNA structure encodes a rough blueprint for translation efficiency. This structure is then amplified by translation, in a self-reinforcing loop, to provide the structure that ultimately specifies the translation of each ORF.', 'Series_overall_design': 'mRNA-seq and ribosome profiling of E. coli MG1655 (wild-type) cells at 37°C and DMS-seq of in vivo modified mRNA from wild-type cells, kasugamycin-treated or untreated ΔgcvB cells, as well as in vitro modified mRNA refolded at 37°C or denatured at 95°C. Reads from replicates of the same experiment were pooled in wiggle files.', 'Series_type': 'Other', 'Series_contributor': 'Carol,A,Gross', 'Series_sample_id': 'GSM2462938', 'Series_contact_name': 'Yan,,Zhang', 'Series_contact_email': 'yan.jenny.zhang@gmail.com', 'Series_contact_laboratory': 'Carol Gross', 'Series_contact_department': 'Microbiology and Immunology', 'Series_contact_institute': 'UC San Francisco', 'Series_contact_address': '600 16th Street', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE77nnn/GSE77617/suppl/GSE77617_RAW.tar', 'Series_platform_id': 'GPL21433', 'Series_platform_taxid': '562', 'Series_sample_taxid': '562', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP069339'}\n",
      "118 PRJNA312799 GSE78163 {'Series_title': 'Ligation-Free Ribosome Profiling Reveals Cell Type-Specific Regulation of Protein Synthesis in the Brain', 'Series_geo_accession': 'GSE78163', 'Series_status': 'Public on Jun 07 2016', 'Series_submission_date': 'Feb 22 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27380875', 'Series_summary': 'Ribosome profiling has emerged as a powerful tool for genome-wide measurements of translation, but library construction requires multiple ligation steps and remains cumbersome relative to more conventional deep sequencing experiments. We report a new approach to ribosome profiling that does not require ligation. Library construction for ligation-free ribosome profiling can be completed in one day with as little as 1 ng of purified RNA footprints. We used ligation-free ribosome profiling to identify new patterns of cell type-specific translation in the brain and tested its ability to identify translational targets of mTOR signaling in the brain.', 'Series_overall_design': 'We report a strategy for ligation free cell-type specific ribosome profiling. Our strategy was utilized to explore cell-type specific expression in mouse brain.', 'Series_type': 'Other', 'Series_contributor': 'Peter,,Canoll', 'Series_sample_id': 'GSM2068582', 'Series_contact_name': 'Peter,,Sims', 'Series_contact_email': 'pas2182@columbia.edu', 'Series_contact_institute': 'Columbia Medical Center', 'Series_contact_address': '3960 Broadway blvd', 'Series_contact_city': 'New York', 'Series_contact_state': 'NEW YORK', 'Series_contact_zip/postal_code': '10032', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE78nnn/GSE78163/suppl/GSE78163_Matrix_NonNormalized_Exon_Cts.txt.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP070655'}\n",
      "119 PRJNA314507 GSE78960 {'Series_title': 'Modeling the Neuropathology of Tuberous Sclerosis with Human Stem Cells Reveals a Role for Inflammation and Angiogenic Growth Factors [Treatment]', 'Series_geo_accession': 'GSE78960', 'Series_status': 'Public on Jul 25 2016', 'Series_submission_date': 'Mar 07 2016', 'Series_last_update_date': 'Nov 08 2020', 'Series_pubmed_id': '27655340', 'Series_summary': 'Tuberous sclerosis complex (TSC) is a rare genetic disease characterized by mTOR hyperfunction induced benign tumor growths in multiple organs and neurological symptoms. Because the molecular pathology is highly complex and the etiology poorly understood we employed a defined human neuronal model with a single mTOR activating mutation to dissect the disease-relevant molecular responses driving the neuropathology. TSC2 deficient neural stem cells showed severely reduced neuronal functional maturation and characteristics of astrogliosis instead. Accordingly, transcriptome analysis uncovered an inflammatory response and increased metabolic activity, while ribosome profiling revealed excessive translation of ribosomal transcripts and higher synthesis rates of angiogenic growth factors. Treatment with mTOR inhibitors corrected translational alterations but not transcriptional dysfunction. These results extend our understanding of the molecular pathophysiology of TSC brain lesions, and suggest phenotype-tailored pharmacological treatment strategies.', 'Series_overall_design': 'Rapamycin, AZD-8055, and DMSO were given to two TSC+/+ cell lines and two TSC-/- cell lines after six weeks of differentiation. Cells are harvested after 3 hours treatment and are subject to ribosome profiling and RNA-seq analysis.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Anna,,Kiialainen', 'Series_sample_id': 'GSM2082580', 'Series_contact_name': 'Jitao David,,Zhang', 'Series_contact_email': 'jitao_david.zhang@roche.com', 'Series_contact_phone': '+41616886251', 'Series_contact_laboratory': 'Pharmaceutical Sciences', 'Series_contact_department': 'Roche Pharmaceutical Research and Early Development, Roche Innovation Center Basel', 'Series_contact_institute': 'F. Hoffmann-La Roche', 'Series_contact_address': 'Grenzacherstrasse 124', 'Series_contact_city': 'Basel', 'Series_contact_zip/postal_code': '4070', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE78nnn/GSE78960/suppl/GSE78960_TSC2-treatment_RPF.txt.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP071235'}\n",
      "120 PRJNA314508 GSE78959 {'Series_title': 'Modeling the Neuropathology of Tuberous Sclerosis with Human Stem Cells Reveals a Role for Inflammation and Angiogenic Growth Factors [Cell Model]', 'Series_geo_accession': 'GSE78959', 'Series_status': 'Public on Jul 25 2016', 'Series_submission_date': 'Mar 07 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27655340', 'Series_summary': 'Tuberous sclerosis complex (TSC) is a rare genetic disease characterized by mTOR hyperfunction induced benign tumor growths in multiple organs and neurological symptoms. Because the molecular pathology is highly complex and the etiology poorly understood we employed a defined human neuronal model with a single mTOR activating mutation to dissect the disease-relevant molecular responses driving the neuropathology. TSC2 deficient neural stem cells showed severely reduced neuronal functional maturation and characteristics of astrogliosis instead. Accordingly, transcriptome analysis uncovered an inflammatory response and increased metabolic activity, while ribosome profiling revealed excessive translation of ribosomal transcripts and higher synthesis rates of angiogenic growth factors. Treatment with mTOR inhibitors corrected translational alterations but not transcriptional dysfunction. These results extend our understanding of the molecular pathophysiology of TSC brain lesions, and suggest phenotype-tailored pharmacological treatment strategies.', 'Series_overall_design': 'Two TSC+/- cell lines and two TSC-/- cell lines were independently generated from wild-type human embryonic stem cells by genome editting with zinc finger nucleases. Two cell lines were handled in the same way but without any known human gene editted and they are used as negative controls. Two independent biological replicates of each of the six cell lines are profiled with ribosome profiling technique.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Anna,,Kiialainen', 'Series_sample_id': 'GSM2082532', 'Series_contact_name': 'Jitao David,,Zhang', 'Series_contact_email': 'jitao_david.zhang@roche.com', 'Series_contact_phone': '+41616886251', 'Series_contact_laboratory': 'Pharmaceutical Sciences', 'Series_contact_department': 'Roche Pharmaceutical Research and Early Development, Roche Innovation Center Basel', 'Series_contact_institute': 'F. Hoffmann-La Roche', 'Series_contact_address': 'Grenzacherstrasse 124', 'Series_contact_city': 'Basel', 'Series_contact_zip/postal_code': '4070', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE78nnn/GSE78959/suppl/GSE78959_TSC2-cellModel_RPF.txt.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP071232'}\n",
      "121 PRJNA316065 GSE79512 {'Series_title': 'Ribosome profiling-guided depletion of an mRNA improves CHO cell growth and recombinant product titers', 'Series_geo_accession': 'GSE79512', 'Series_status': 'Public on Dec 21 2016', 'Series_submission_date': 'Mar 23 2016', 'Series_last_update_date': 'Jan 13 2020', 'Series_pubmed_id': '31896772', 'Series_summary': 'We present the first comprehensive genome-wide view of CHO translational activity during recombinant expression by the application of ribosome profiling. The distribution of translational power in CHO cells was analyzed in the context of the recombinant mRNAs and the endogenous mRNA pool. In an antibody-producing CHO cell line, the recombinant mRNAs were found to be the most abundant transcripts and also sequestered a substantial amount of translating ribosomes (up to 15%). The recombinant mRNAs were translated as efficiently as the endogenous mRNAs, and changes in translation and transcription of the recombinant mRNAs were directly reflected in changes in specific productivity. Interestingly, improvements in bioprocess quality attributes were achieved by depleting the highly expressed and translated inert NeoR mRNA by siRNA-mediated knock-down. This resulted in improved cellular growth, which was accompanied by an 18% increase in antibody titers. This study is the first to carefully map the translatome of the CHO cell to the nucleotide level and to demonstrate how recombinant expression affects the cell on the translational level.', 'Series_overall_design': 'Examine translation of IgG at exponential phase (day3) and stationary phase (day6) in CHO cell', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Nathan,E,Lewis', 'Series_sample_id': 'GSM2096530', 'Series_contact_name': 'Nathan,,Lewis', 'Series_contact_email': 'n4lewis@ucsd.edu', 'Series_contact_phone': '8589975844', 'Series_contact_laboratory': 'System Biochemistry and Cell Engineering', 'Series_contact_department': 'Pediatrics', 'Series_contact_institute': 'UCSD', 'Series_contact_address': '9500 Gilman Drive', 'Series_contact_city': 'La Jolla', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '92093', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE79nnn/GSE79512/suppl/GSE79512_RiboSeq_rawCount.txt.gz', 'Series_platform_id': 'GPL20904', 'Series_platform_taxid': '10029', 'Series_sample_taxid': '10029', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP072205'}\n",
      "122 PRJNA316125 GSE79539 {'Series_title': 'Finding new proteins encoded by long non-coding RNAs in human cells', 'Series_geo_accession': 'GSE79539', 'Series_status': 'Public on Jul 20 2019', 'Series_submission_date': 'Mar 23 2016', 'Series_last_update_date': 'Sep 23 2019', 'Series_pubmed_id': '31340039', 'Series_summary': 'We performed ribosome profiling to assess the translation elongation on translating RNA in HBE, A549, and H1299. We detected the new proteins in the cells.', 'Series_overall_design': 'HBE, A549 and H1299 cells were cultured in the complete DMEM medium. We detected and analyzed the new proteins with ribosome profiling.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Gong,,Zhang', 'Series_sample_id': 'GSM2097524', 'Series_contact_name': 'Gong,,Zhang', 'Series_contact_email': 'zhanggong@jnu.edu.cn', 'Series_contact_laboratory': 'Translatomics Lab', 'Series_contact_department': 'Institute of Life and Health Engineering', 'Series_contact_institute': 'Jinan University', 'Series_contact_address': 'Huang-Pu Avenue West 601', 'Series_contact_city': 'Guangzhou', 'Series_contact_zip/postal_code': '510632', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE79nnn/GSE79539/suppl/GSE79539_Expression_Table.xlsx.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP072231'}\n",
      "123 PRJNA316463 GSE79622 {'Series_title': 'Ubiquitination of Stalled Ribosome Triggers Ribosome-associated Quality Control [SET1]', 'Series_geo_accession': 'GSE79622', 'Series_status': 'Public on Jul 31 2017', 'Series_submission_date': 'Mar 25 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28757607', 'Series_summary': 'Translation elongation rates are regulated to ensure proper conformation and biological function of proteins. Translation of either non-stop mRNA or transcripts coding for poly-basic sequences induces ribosome stalling, and the arrest product is degraded by the ribosome-mediated quality control system (RQC). During this process, the stalled ribosome is dissociated into subunits, and the polypeptide is ubiquitinated by the E3 ubiquitin ligase Listerin on the 60S large ribosomal subunit (LSU) leading to subsequent proteasomal degradation. However, it is largely unknown how stalled ribosomes are recognized and dissociated into subunits. Here we report that ubiquitination of the ribosomal protein uS10 by the E3 ubiquitin ligase Hel2 is required for the production of the RQC substrate. RQC-trigger (RQT) factors, a RNA helicase-family protein Slh1/Rqt2, ubiquitin binding protein Cue3/Rqt3 and yKR023W/Rqt4, were also required for the primary steps of RQC, and associated with Hel2-ribosome complexes. Rqt2-4 factors were dispensable for the ubiquitination of uS10 by Hel2/Rqt1 and associated with ribosomes independent of the ubiquitination of uS10. However, the ubiquitin-binding activity of Rqt3 were crucial to trigger RQC. Cryo-electron microscopy (cryo-EM) analysis revealed that Hel2 bound ribosomes are in an rotated state containing hybrid state AP- and PE-tRNAs. Furthermore, ribosome profiling revealed that short footprints, hallmarks of hybrid state ribosomes18, were accumulated at tandem CGA rare codons at the beginning of the poly arginine stalling sequence and long footprints at subsequent codons, respectively. Short footprints at CGA codons were decreased in rqt1 mutant but drastically increased in uS10 mutants defective in the ubiquitination or rqt2 mutant. Collectively, our results demonstrate   that Hel2 stabilizes ratcheted ribosomes leading to ubiquitination of uS10. Subsequently, Rqt2-4 factors target these hybrid state ribosomes specifically, allowing subsequent RQC reactions.', 'Series_overall_design': 'Ribosome,profiling', 'Series_type': 'Other', 'Series_contributor': 'Toshifumi,,Inada', 'Series_sample_id': 'GSM2099060', 'Series_contact_name': 'Shintaro,,Iwasaki', 'Series_contact_email': 'shintaro.iwasaki@riken.jp', 'Series_contact_department': 'Cluster for Pioneering Research', 'Series_contact_institute': 'RIKEN', 'Series_contact_address': 'S205 Bioscience Bldg. 2-1 Hirosawa', 'Series_contact_city': 'Wako', 'Series_contact_state': 'Saitama', 'Series_contact_zip/postal_code': '351-0198', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE79nnn/GSE79622/suppl/GSE79622_RAW.tar', 'Series_platform_id': 'GPL21656', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP072363'}\n",
      "124 PRJNA316618 GSE79664 {'Series_title': 'Regulation of poly(A) tail and translation during the somatic cell cycle', 'Series_geo_accession': 'GSE79664', 'Series_status': 'Public on Apr 04 2016', 'Series_submission_date': 'Mar 28 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27153541', 'Series_summary': 'Poly(A) tails are critical for mRNA stability and translation. However, recent studies have challenged this view, showing that poly(A) tail length and translation efficiency are decoupled in non-embryonic cells. Using TAIL-seq and ribosome profiling, we investigate poly(A) tail dynamics and translational control in the somatic cell cycle. We find dramatic changes in poly(A) tail lengths of cell cycle regulatory genes like CDK1, TOP2A, and FBXO5, explaining their translational repression in M phase. We also find that poly(A) tail length is coupled to translation when the poly(A) tail is <20 nucleotides. However, as most genes have >20 nucleotide poly(A) tails, their translation is regulated mainly via poly(A) tail length-independent mechanisms during the cell cycle. Specifically, we find that terminal oligopyrimidine (TOP) tract-containing transcripts escape global translational suppression in M phase and are actively translated. Our quantitative and comprehensive data provide a revised view of translational control in the somatic cell cycle.', 'Series_overall_design': 'HeLa cells were synchronized at S or M phase, and subject to RNA-seq, ribosome profiling and TAIL-seq analysis.', 'Series_type': 'Other', 'Series_contributor': 'V,N,Kim', 'Series_sample_id': 'GSM2100604', 'Series_contact_name': 'Jong-Eun,,Park', 'Series_contact_email': 'jepark87@gmail.com', 'Series_contact_institute': 'Seoul National University', 'Series_contact_address': '1 Gwanak-ro, Gwanak-gu', 'Series_contact_city': 'Seoul', 'Series_contact_zip/postal_code': '08826', 'Series_contact_country': 'South Korea', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE79nnn/GSE79664/suppl/GSE79664_TAIL_S.csv.gz', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP072459'}\n",
      "125 PRJNA317266 GSE79864 {'Series_title': 'MYC dependent mRNA translation shapes gene expression and cell biology', 'Series_geo_accession': 'GSE79864', 'Series_status': 'Public on Apr 30 2019', 'Series_submission_date': 'Apr 03 2016', 'Series_last_update_date': 'Aug 21 2020', 'Series_pubmed_id': '31142587', 'Series_summary': 'We report that MYC controls the translation of specific mRNAs in a sequence and RNA binding protein dependent manner. Specifically, using transcriptome-scale ribosome footprinting we identify 882 mRNAs whose translation depends on MYC. These mRNAs are highly significantly enriched for specific 5ÕUTR sequence motifs that bind the SRSF1/RBM42/HNRNPK complex. For example, MYC and SRFSF1/RBM42 dependent transcripts include many components of the electron transport chain and directly affect cellular respiration. Our results provide an integrated map of MYC driven gene expression programs that includes MYCÕs ability to control mRNA translation.', 'Series_overall_design': 'Human B-cell lymphoma P493 cells were treated with Tetracycline (0.1 uM) for 24 hrs followed by cycloheximide treatment for 10 minutes in triplicates. An aliquot of cells were used for total RNA preparation and remaining cells were processed for isolation of ribosome protected fragments.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Zhengqing,,Ouyang', 'Series_sample_id': 'GSM2745780', 'Series_contact_name': 'Kamini,,Singh', 'Series_contact_email': 'kamini.singh@einsteinmed.edu', 'Series_contact_phone': '7184302466', 'Series_contact_laboratory': 'Kamini Singh', 'Series_contact_department': 'Molecular Pharmacology', 'Series_contact_institute': 'Albert Einstein College of Medicine', 'Series_contact_address': '1300 Morris Park Ave, Golding 203', 'Series_contact_city': 'Bronx', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '10461', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE79nnn/GSE79864/suppl/GSE79864_Myc_Ribodiff.txt.gz', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP072804'}\n",
      "126 PRJNA318051 GSE80156 {'Series_title': 'A coding sequence-embedded principle governs translational reading frame fidelity', 'Series_geo_accession': 'GSE80156', 'Series_status': 'Public on Apr 12 2016', 'Series_submission_date': 'Apr 11 2016', 'Series_last_update_date': 'Oct 01 2019', 'Series_pubmed_id': '31549036', 'Series_summary': 'Upon initiation at an AUG start codon, the ribosome must maintain the correct reading frame for hundreds of codons in order to produce functional proteins. Although some sequence elements are able to trigger programmed ribosomal frameshifting (PRF), very little is known how the ribosome normally prevents spontaneous frameshift errors. Using high resolution ribosome profiling data sets, we discovered that the translating ribosome uses the 3’ end of 18S rRNA to scan the AUG-like codons after the decoding process. The internal mRNA:rRNA interaction not only contributes to predominant translational pausing, but also provides a post-decoding mechanism to safeguard the ribosome in the correct reading frame. Partially eliminating the AUG-like “sticky” codons in the reporter message leads to increased +1 frameshift errors. Remarkably, mutating the highly conserved CAU triplet of 18S rRNA globally changes codon “stickiness”. Further supporting the role of “sticky” sequences in reading frame maintenance, the codon composition of open reading frames is highly optimized across eukaryotic genomes by minimizing the appearance of AUG-like codons in the frame 2. These results suggest an important layer of information embedded within the protein coding sequences that instructs the ribosome to ensure reading frame fidelity during translation.', 'Series_overall_design': 'Investigating the role of 18S rRNA in controlling translational reading frame fidelity', 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'Shu-Bing,,Qian', 'Series_sample_id': 'GSM2113586', 'Series_contact_name': 'Ji,,Wan', 'Series_contact_email': 'jw896@cornell.edu', 'Series_contact_institute': 'Cornell University', 'Series_contact_address': '303 Biotech 526 Campus Road', 'Series_contact_city': 'Ithaca', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '14853', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE80nnn/GSE80156/suppl/GSE80156_RAW.tar', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP073116'}\n",
      "127 PRJNA318054 GSE80157 {'Series_title': 'Post-transcriptional regulation by the let-7 microRNA and the TRIM-NHL protein LIN41 [RNA-seq]', 'Series_geo_accession': 'GSE80157', 'Series_status': 'Public on Jul 17 2017', 'Series_submission_date': 'Apr 11 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28111013', 'Series_summary': 'We perform RNA sequencing and ribosome profiling time course experiments to examine the effect of fully dysregulating all let-7 targets (in let-7(n2853) animals), partially dysregulating only LIN41 (in lin-41(xe11) animals) or fully dysregulating all let-7 targets while partially dysregulating LIN41 in lin-41(xe11); let-7(n2853) double mutant animals. We conclude that effects on gene expression in let-7 mutant animals are largely and quantitatively explained by dysregulation of LIN41 as its primary target. Furthermore, we identify direct LIN41 target genes regulated on the level of translation or mRNA abundance.', 'Series_overall_design': 'Total RNA-sequencing time course experiments sampling synchronized worm populations of different genetic backgrounds every two hours over the course of development from late L2/early L3 stage to late L4/Young adult stage.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Helge,,Großhans', 'Series_sample_id': 'GSM2113629', 'Series_contact_name': 'Dimos,,Gaidatzis', 'Series_contact_email': 'd.gaidatzis@fmi.ch', 'Series_contact_institute': 'Friedrich Miescher Institute', 'Series_contact_address': 'Maulbeerstrasse 66', 'Series_contact_city': 'Basel', 'Series_contact_zip/postal_code': '4058', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE80nnn/GSE80157/suppl/GSE80157_ce_geneExpression_riboM_rawCounts.txt.gz', 'Series_platform_id': 'GPL13657', 'Series_platform_taxid': '6239', 'Series_sample_taxid': '6239', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP073117'}\n",
      "128 PRJNA321057 GSE81247 {'Series_title': 'The RNA-binding protein TTP is a global post-transcriptional regulator of feedback control in inflammation [Ribo-seq]', 'Series_geo_accession': 'GSE81247', 'Series_status': 'Public on May 25 2016', 'Series_submission_date': 'May 09 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27220464', 'Series_summary': \"RNA-binding proteins (RBPs) facilitate post-transcriptional control of eukaryotic gene expression at multiple levels. The RBP tristetraprolin (TTP/Zfp36) is a signal-induced phosphorylated anti-inflammatory protein guiding unstable mRNAs of pro-inflammatory proteins for degradation and preventing translation. Using iCLIP, we have identified numerous mRNA targets bound by wild-type TTP and by a non-MK2-phosphorylatable TTP mutant (TTP-AA) in 1h LPS-stimulated macrophages and correlated their interaction with TTP to changes at the level of mRNA abundance and translation in a transcriptome-wide manner. The close similarity of the transcriptome of TTP-deficient and TTP-expressing macrophages upon short LPS stimulation suggested an effective inactivation of TTP by MK2 under these conditions whereas retained RNA-binding capacity of TTP-AA to 3’UTRs caused profound changes in the transcriptome and translatome, altered NF-κB-activation and induced cell death. Increased TTP binding to the 3'UTR of feedback inhibitor mRNAs, such as Ier3, Dusp1 or Tnfaip3, in the absence of MK2-dependent TTP neutralization resulted in a strong reduction of their protein synthesis contributing to the deregulation of the NF-κB-signaling pathway. Taken together, our study uncovers a role for TTP in NF-κB-signaling and highlights the importance of fine-tuned TTP activity-regulation by MK2 in order to control feedback signaling during the inflammatory response.\", 'Series_overall_design': 'Comparison of the transcriptomes of TTP knockout macrophages inducibly expressing GFP, GFP-TTP or GFP-TTP-AA (S52A, S178A) phosphorylation mutant during 1h LPS stimulation. 3 biological replicates per genotype and condition.', 'Series_type': 'Other', 'Series_contributor': 'Matthias,,Gaestel', 'Series_sample_id': 'GSM2147489', 'Series_contact_name': 'Felix,,Krueger', 'Series_contact_email': 'fkrueger@altoslabs.com', 'Series_contact_department': 'Bioinformatics', 'Series_contact_institute': 'Altos Labs', 'Series_contact_address': 'Granta Park', 'Series_contact_city': 'Cambridge', 'Series_contact_zip/postal_code': 'CB21 6GP', 'Series_contact_country': 'United Kingdom', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE81nnn/GSE81247/suppl/GSE81247_Norm_counts_RiboSeq_DESeq.csv.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP074604'}\n",
      "129 PRJNA321185 GSE81283 {'Series_title': 'Translational contributions to tissue-specificity in rhythmic and constitutive gene expression', 'Series_geo_accession': 'GSE81283', 'Series_status': 'Public on Apr 08 2017', 'Series_submission_date': 'May 10 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28622766', 'Series_summary': '', 'Series_overall_design': 'A total of 48 mice were entrained under 12hours light:dark conditions for 2 weeks and also collected under 12hours light:dark. Mice were sacrificed every two hours during the 24 hours daily cycle. Two replicates per time point, each replicate is a pool of livers or kidneys from 2 animals.', 'Series_type': 'Other', 'Series_contributor': 'David,,Gatfield', 'Series_sample_id': 'GSM2149149', 'Series_contact_name': 'David,,Gatfield', 'Series_contact_email': 'david.gatfield@unil.ch', 'Series_contact_department': 'Center for Integrative Genomics', 'Series_contact_institute': 'University of Lausanne', 'Series_contact_address': 'Genopode', 'Series_contact_city': 'Lausanne', 'Series_contact_zip/postal_code': '1015', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE81nnn/GSE81283/suppl/GSE81283_RAW.tar', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP074765'}\n",
      "130 PRJNA321201 GSE81295 {'Series_title': 'Buffer composition affects ribosome footprint precision in Arabidopsis root and shoot', 'Series_geo_accession': 'GSE81295', 'Series_status': 'Public on Nov 04 2016', 'Series_submission_date': 'May 10 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27791167', 'Series_summary': 'The goal of this study is to test ionic strength and buffering capacity in polysome extraction buffer on ribosome footprints in Arabidopsis root and shoot.', 'Series_overall_design': 'Ribo-seq generated from Arabidopsis root and shoot prepared with four different polysome extraction buffers. All four buffers contain 2% polyoxyethylene 10 tridecyl ether, 1% deoxycholic acid, 1 mM DTT, 100 µg/mL cycloheximide, and 10 unit/mL DNase I, but with different ionic strength or buffering capacity listed below. Buffer A: 200 mM Tris-HCl (pH 8), 50 mM KCl, 25 mM MgCl2; buffer B: 200 mM Tris-HCl (pH 8), 40 mM KCl, 20 mM MgCl2; buffer C: 200 mM Tris-HCl (pH 8), 30 mM KCl, 15 mM MgCl2; buffer D: 100 mM Tris-HCl (pH 8), 40 mM KCl, 20 mM MgCl2. 8 libraries in total.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Philip,N,Benfey', 'Series_sample_id': 'GSM2149897', 'Series_contact_name': 'Polly,Yingshan,Hsu', 'Series_contact_laboratory': 'Philip Benfey', 'Series_contact_department': 'Biology', 'Series_contact_institute': 'Duke University', 'Series_contact_address': 'Duke University', 'Series_contact_city': 'Durham', 'Series_contact_state': 'NC', 'Series_contact_zip/postal_code': '27708', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE81nnn/GSE81295/suppl/GSE81295_ribo_count_CDS_4buffers.txt.gz', 'Series_platform_id': 'GPL13222', 'Series_platform_taxid': '3702', 'Series_sample_taxid': '3702', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP074779'}\n",
      "131 PRJNA321304 GSE81332 {'Series_title': 'Ribosome profiling in Arabidopsis root and shoot', 'Series_geo_accession': 'GSE81332', 'Series_status': 'Public on Nov 04 2016', 'Series_submission_date': 'May 11 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27791167', 'Series_summary': 'The goal of this study is to map genome-wide ribosome occupancy on mRNAs in Arabidopsis root and shoot to characterize translatome.', 'Series_overall_design': 'RNAseq and ribo-seq generated from three biological replicates of root and shoot harvested from 4-day old wild-type plant (Col); 12 libraries in total.', 'Series_type': 'Other', 'Series_contributor': 'Philip,N,Benfey', 'Series_sample_id': 'GSM2150578', 'Series_contact_name': 'Polly,Yingshan,Hsu', 'Series_contact_laboratory': 'Philip Benfey', 'Series_contact_department': 'Biology', 'Series_contact_institute': 'Duke University', 'Series_contact_address': 'Duke University', 'Series_contact_city': 'Durham', 'Series_contact_state': 'NC', 'Series_contact_zip/postal_code': '27708', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE81nnn/GSE81332/suppl/GSE81332_Shoot_translated_ORFs_filtered_sorted.bed.gz', 'Series_platform_id': 'GPL17639', 'Series_platform_taxid': '3702', 'Series_sample_taxid': '3702', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP074840'}\n",
      "132 PRJNA323467 GSE81932 {'Series_title': 'Ribosome profiling of synchronous, non-arrested yeast cells identifies translational control of lipid biosynthesis enzymes in the cell cycle', 'Series_geo_accession': 'GSE81932', 'Series_status': 'Public on Jan 09 2017', 'Series_submission_date': 'May 26 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28057705', 'Series_summary': 'The translational efficiency of mRNAs in cells progressing synchronously through the mitotic cell cycle and having to meet their growth requirements for cell division has not been interrogated. Here, we used ribosome profiling of a cell-size series from the time of cell birth, to identify for the first time yeast mRNAs under periodic translational control. Late in the cell cycle, there was coordinate translational activation of mRNAs encoding lipid biosynthesis enzymes. An upstream open reading frame (uORF) mediates the translational control of ACC1, the mRNA encoding the rate-limiting enzyme acetyl-CoA carboxylase. The ACC1 uORF also adjusts Acc1p protein levels in different nutrients. Mutating the ACC1 uORF delayed cell cycle progression, reduced cell size and suppressed the replicative longevity of cells lacking the Sch9p protein kinase, the yeast S6 kinase ortholog. These findings reveal unexpected links between lipogenesis and protein synthesis in mitotic cell divisions', 'Series_overall_design': 'We used centrifugal elutriation to isolate growing, highly synchronous, unbudded, early G1 cell populations and scored the elutriated cultures over time microscopically, recording the percentage of budded cells with a particle counter that directly measures their volume (channelyzer). Generation of both the Ribosome Profile (RP) and Transcriptional Profile (TP) libraries require at least ≥2E+09 yeast cells per sample. Such sample size could not be collected following a typical time-series experiment because only a small minority of the total cell population is in early G1 at any given moment. Instead, we allowed the entire cell population of each elutriated fraction to progress in the cell cycle until it reached a given cell size (e.g., 40 fl), at which point the sample was harvested. Elutriated cells of the same size were pooled together until enough cells per sample were obtained. This pool of same-size cells defined one experimental replicate (e.g., Sample01A). We harvested cells with 5fl cell size increments, from the time they are born (40 fl -- Sample01), until the end of the cell cycle (75 fl -- Sample08). In total we generated three independent Experimental Replicates for each cell size (A, B and C). We have: Sample01A, 01B and 01C (40 fl); Sample02A, 02B and 02C (45 fl), Sample03A, 03B and 03C (50 fl), Sample04A, 04B and 04C (55 fl), Sample05A, 05B and 05C (60 fl), Sample06A, 06B and 06C (65 fl), Sample07A, 07B and 07C (70 fl) and Sample08A, 08B and 08C (75 fl). Samples are divided into two series: Ribosome Profile (RP) and Transcriptional Profile (TP).', 'Series_type': 'Other', 'Series_contributor': 'Michael,,Polymenis', 'Series_sample_id': 'GSM2177931', 'Series_contact_name': 'Rodolfo,,Aramayo', 'Series_contact_email': 'raramayo@tamu.edu', 'Series_contact_laboratory': 'AramayoLab', 'Series_contact_department': 'Biology', 'Series_contact_institute': 'Texas A&M University', 'Series_contact_address': 'Room 412, Biological Sciences Building West (BSBW)', 'Series_contact_city': 'College Station', 'Series_contact_state': 'Texas', 'Series_contact_zip/postal_code': '77843-3258', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE81nnn/GSE81932/suppl/GSE81932_Dataset08.txt.gz', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP075766'}\n",
      "133 PRJNA323510 GSE81966 {'Series_title': 'eIF4B preferentially stimulates translation of long mRNAs with structured 5’UTRs and low closed-loop potential but weak dependence on eIF4G', 'Series_geo_accession': 'GSE81966', 'Series_status': 'Public on May 27 2016', 'Series_submission_date': 'May 26 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27601676', 'Series_summary': 'DEAD-box RNA helicases eIF4A and Ded1 promote translation by resolving mRNA secondary structures that impede preinitiation complex (PIC) attachment to mRNA or scanning. eIF4B is a cofactor for eIF4A but might also function independently of eIF4A. Ribosome profiling of mutants lacking eIF4B or with impaired eIF4A or Ded1 activity revealed that eliminating eIF4B reduces the relative translational efficiencies of many more genes than does inactivation of eIF4A, despite comparable reductions in bulk translation, and few genes display unusually strong requirements for both factors. However, either eliminating eIF4B or inactivating eIF4A preferentially impacts mRNAs with longer, more structured 5’UTRs. These findings reveal an eIF4A-independent role for eIF4B in addition to its function as eIF4A cofactor in promoting PIC attachment or scanning on structured mRNAs. eIF4B, eIF4A, and Ded1 mutations also preferentially impair translation of longer mRNAs in a fashion mitigated by the ability to form closed-loop mRNPs via eIF4F-Pab1 association, suggesting cooperation between closed-loop assembly and eIF4B/helicase functions. Remarkably, depleting eIF4G, the scaffold subunit of eIF4F, preferentially impacts short mRNAs with strong closed-loop potential and unstructured 5’UTRs, exactly the opposite features associated with hyperdependence on the eIF4B/helicases. We propose that short, highly efficient mRNAs preferentially depend on the stimulatory effects of eIF4G-dependent closed-loop assembly.', 'Series_overall_design': 'We compared the effects of mutations in eIF4B on global translational efficiencies (TEs) in yeast by ribosome footprint profiling. The study includes 16 samples, comprised of 8 mRNA-Seq samples and 8 ribosome footprint profiling samples, derived from biological replicates of tif3Δ mutant strain, and the corresponding TIF3 wild-type strain . These strains were analyzed following a temperature shift at 37°C or 15°C.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Alan,G,Hinnebusch', 'Series_sample_id': 'GSM2178830', 'Series_contact_name': 'Alan,G,Hinnebusch', 'Series_contact_email': 'ahinnebusch@nih.gov', 'Series_contact_institute': 'NICHD', 'Series_contact_address': '6 CENTER DR Room 230, MSC 0609', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '20814', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE81nnn/GSE81966/suppl/GSE81966_tif3_37_deg.csv.gz', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP075785'}\n",
      "134 PRJNA324409 GSE82232 {'Series_title': 'A systematic view on Influenza induced host shut-off', 'Series_geo_accession': 'GSE82232', 'Series_status': 'Public on Aug 15 2016', 'Series_submission_date': 'Jun 03 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27525483', 'Series_summary': 'Influenza A viruses (IAV) induces drastic host shut-off during infection. We use RNA-seq and ribosome profiling to systematically look at host genes RNA and translation levels along IAV infection. We show that host shutoff is mainly achieved by reduction in cellular mRNAs levels and that the high translation levels of IAV transcripts are driven by their strong accumulation compared to host transcripts. Interestingly, our systematic analysis reveals that host transcripts are affected differently by IAV infection and the extent of cellular mRNAs reduction is strongly related to transcripts’ length and GC content.', 'Series_overall_design': 'Ribosome profiling and mRNA-seq along IAV infection', 'Series_type': 'Other', 'Series_contributor': 'Noam,,Stern-Ginossar', 'Series_sample_id': 'GSM2188849', 'Series_contact_name': 'Idit,Anna,Gelbart', 'Series_contact_email': 'idit.gelbart@weizmann.ac.il', 'Series_contact_laboratory': 'Noam Stern-Ginossar', 'Series_contact_department': 'Molecular Genetics', 'Series_contact_institute': 'Weizmann Institute of Science', 'Series_contact_address': 'Weizmann Institute of Science', 'Series_contact_city': 'Rehovot', 'Series_contact_zip/postal_code': '76100', 'Series_contact_country': 'Israel', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE82nnn/GSE82232/suppl/GSE82232_RAW.tar', 'Series_platform_id': 'GPL21979', 'Series_platform_taxid': '11320', 'Series_sample_taxid': '11320', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP076102'}\n",
      "135 PRJNA325659 GSE83351 {'Series_title': 'Rapid Genome-wide Recruitment of RNA Polymerase II Drives Transcription, Splicing and Translation Events during T Cell Responses', 'Series_geo_accession': 'GSE83351', 'Series_status': 'Public on Mar 31 2017', 'Series_submission_date': 'Jun 14 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28423325', 'Series_summary': 'Activation of immune cells results in rapid functional changes, but how such fast changes are accomplished remains enigmatic. By combining time-courses of 4sU-Seq, RNA-Seq, ribosome profiling and RNA polymerase II (RNAPII) ChIP-Seq during T cell activation, we illustrate genome-wide temporal dynamics for ~10,000 genes. This approach reveals immediate-early as well as posttranscriptionally-regulated genes, but also coupled changes in transcription and translation for >90% of genes. Recruitment, rather than release of paused RNAPII, primarily mediates transcriptional changes. This coincides with a genome-wide temporary slowdown in cotranscriptional splicing, even for polyadenylated mRNAs that are localized at the chromatin. Subsequent splicing optimization correlates with increasing Ser-2 phosphorylation of the RNAPII carboxy-terminal domain (CTD) and activation of the positive transcription elongation factor complex (pTEFb). Thus, rapid de novo recruitment of RNAPII dictates the course of events during T cell activation, in particular transcription, splicing and consequently translation.', 'Series_overall_design': 'Analysis of temporal dynamics of transcription, splicing and translation in real-time resolution during the activation of T effector cells by RNAseq, 4sU-Seq and Ribosome Profiling', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Elke,,Glasmacher', 'Series_sample_id': 'GSM2200068', 'Series_contact_name': 'Caroline,,Friedel', 'Series_contact_department': 'Institut für Informatik', 'Series_contact_institute': 'Ludwig-Maximilians-Universität München', 'Series_contact_address': 'Amalienstr. 17', 'Series_contact_city': 'München', 'Series_contact_zip/postal_code': '80333', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE83nnn/GSE83351/suppl/GSE83351_FPKM_all.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP076551'}\n",
      "136 PRJNA327113 GSE83823 {'Series_title': 'Widespread and dynamic translational control of red blood cell development', 'Series_geo_accession': 'GSE83823', 'Series_status': 'Public on Dec 01 2016', 'Series_submission_date': 'Jun 28 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27899360', 'Series_summary': 'Cell development requires tight yet dynamic control of protein production. Here, we use murine erythropoiesis as a model to study translational regulatory dynamics during mammalian cell differentiation. We uncover pervasive translational control of protein synthesis, including widespread alternative translation initiation and termination, stoichiometric synthesis, and dynamic use of upstream reading frames. We thus unravel translational regulatory programs during erythropoiesis, comprising hundreds of mRNAs with dynamic translation efficiencies and their cell type-specific regulators. A major such program involves enhanced decoding of specific mRNAs depleted in terminally differentiating/enucleating cells with decreasing transcriptional capacity. Rbm38, an erythroid-specific RNA-binding protein, promotes translation of many such genes by recruiting the translation initiation factor eIF4G. Inhibiting Rbm38 blocks red cell production, and mice lacking Rbm38 develop anemia, demonstrating an essential role for Rbm38-mediated enhanced translation of irreplaceable mRNAs. These findings reveal critical roles for dynamic translational control in supporting specialized mammalian cell formation.', 'Series_overall_design': 'Matched RNA-seq and Ribo-seq profiles of erythroid progenitors purified from E14.5 mouse fetal livers after 0, 24, 33, and 48 hours of ex vivo differentiation, generated by deep sequencing, in duplicate, using the Illumina HiSeq2000 platform.', 'Series_type': 'Other', 'Series_contributor': 'Wenqian,,Hu', 'Series_sample_id': 'GSM2219157', 'Series_contact_name': 'Juan,R,Alvarez-Dominguez', 'Series_contact_department': 'Stem Cell and Regenerative Biology', 'Series_contact_institute': 'Harvard University', 'Series_contact_address': '7 Divinity Ave', 'Series_contact_city': 'Cambridge', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02138', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE83nnn/GSE83823/suppl/GSE83823_RAW.tar', 'Series_citation': 'Widespread and dynamic translational control of red blood cell development Juan R. Alvarez-Dominguez, Xu Zhang, Wenqian Hu Blood Jan 2016, blood-2016-09-741835; DOI: 10.1182/blood-2016-09-741835', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP077522'}\n",
      "137 PRJNA328042 GSE84112 {'Series_title': 'Global analysis of selenoprotein expression reveals differential roles for Secisbp2 in UGA redefinition and mRNA stabilization', 'Series_geo_accession': 'GSE84112', 'Series_status': 'Public on Oct 30 2016', 'Series_submission_date': 'Jul 06 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27956496', 'Series_summary': 'We describe the effects of knocking out the Secisbp2 and Trsp genes on selenoprotein expression and translation in mouse liver', 'Series_overall_design': 'Livers from wild type mice and mice that were deleted for the Secisbp2 or Sec-tRNA (Trsp) genes (conditional deletion in hepatocytes) were harvested and processed for RNA-Seq and ribosome profiling. RNA-Seq was used to monitor mRNA abundance and ribosome profiling was used to monitor translation initiation and UGA-Sec redefinition efficiency.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Michael,T,Howard', 'Series_sample_id': 'GSM2227384', 'Series_contact_name': 'Michael,T,Howard', 'Series_contact_email': 'mhoward@genetics.utah.edu', 'Series_contact_phone': '801-585-1927', 'Series_contact_laboratory': 'Howard', 'Series_contact_department': 'Human Genetics', 'Series_contact_institute': 'University of Utah', 'Series_contact_address': '15 N 2030 E', 'Series_contact_city': 'Salt Lake City', 'Series_contact_state': 'UT', 'Series_contact_zip/postal_code': '84112', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE84nnn/GSE84112/suppl/GSE84112_RPF_RNA.xlsx', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP078005'}\n",
      "138 PRJNA330982 GSE84746 {'Series_title': 'Dual randomization of oligonucleotides to reduce the bias in ribosome-profiling libraries', 'Series_geo_accession': 'GSE84746', 'Series_status': 'Public on Aug 01 2016', 'Series_submission_date': 'Jul 22 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27450428', 'Series_summary': 'Protein translation is at the heart of cellular metabolism and its in-depth characterization is key for many lines of research. Recently, ribosome profiling became the state-of-the-art method to quantitatively characterize translation dynamics at a transcriptome-wide level. However, the strategy of library generation affects its outcomes. Here, we present a modified ribosomeprofiling protocol starting from yeast, human cells and vertebrate brain tissue. We use a DNA linker carrying four randomized positions at its 5’  and a reverse-transcription (RT) primer with three randomized positions to reduce artifacts during library preparation. The use of seven randomized nucleotides allows to efficiently detect library-generation artifacts. We find that the effect of polymerase chain reaction (PCR) artifacts is relatively small for global analyses when sufficient input material is used. However, when input material is limiting, our strategy improves the sensitivity of gene-specific analyses. Furthermore, randomized nucleotides alleviate the skewed frequency of specific sequences at the 3’  end of ribosome-protected fragments (RPFs) likely resulting from ligase specificity. Finally, strategies that rely on dual ligation show a high degree of gene-coverage variation. Taken together, our approach helps to remedy two of the main problems associated with ribosome-profiling data. This will facilitate the analysis of translational dynamics and increase our understanding of the influence of RNA modifications on translation.', 'Series_overall_design': 'Ribosome profiling and mRNA-seq libraries from wt yeast comparing different library preparation approaches using different combinations of randomized and non-randomized linkers and RT primers.', 'Series_type': 'Other', 'Series_contributor': 'Sebastian,A,Leidel', 'Series_sample_id': 'GSM2249634', 'Series_contact_name': 'Benedikt,Sebastian,Nilges', 'Series_contact_email': 'benedikt.nilges@mpi-muenster.mpg.de', 'Series_contact_department': 'RNA Biology Group', 'Series_contact_institute': 'Max Planck Institute for Molecular Biomedicine', 'Series_contact_address': 'Von-Esmarch-Straße 54', 'Series_contact_city': 'Muenster', 'Series_contact_zip/postal_code': '48149', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE84nnn/GSE84746/suppl/GSE84746_RAW.tar', 'Series_platform_id': 'GPL19958', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP079402'}\n",
      "139 PRJNA336005 GSE85036 {'Series_title': 'Post-Translational Dosage Compensation Buffers Genetic Perturbations to Stoichiometry of Protein Complexes', 'Series_geo_accession': 'GSE85036', 'Series_status': 'Public on Dec 30 2016', 'Series_submission_date': 'Aug 01 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28121980', 'Series_summary': 'Understanding buffering mechanisms for various perturbations is essential for understanding robustness in cellular systems. Protein-level dosage compensation, which arises when changes in gene copy number do not translate linearly into protein level, is one mechanism for buffering against genetic perturbations. Here, we present an approach to identify genes with dosage compensation by increasing the copy number of individual genes using the genetic tug-of-war technique. Our screen of chromosome I suggests that dosage-compensated genes constitute approximately 10% of the genome and consist predominantly of subunits of multi-protein complexes. Importantly, because subunit levels are regulated in a stoichiometry-dependent manner, dosage compensation plays a crucial role in maintaining subunit stoichiometries. Indeed, we observed changes in the levels of a complex when its subunit stoichiometries were perturbed. We further analyzed compensation mechanisms using a proteasome-defective mutant as well as ribosome profiling, which provided strong evidence for compensation by ubiquitin-dependent degradation but not reduced translational efficiency. Thus, our study provides a systematic understanding of dosage compensation and highlights that this post-translational regulation is a critical aspect of robustness in cellular systems.', 'Series_overall_design': 'Ribosome profiling for Sample1-4. RNA-seq for Sample 5-8. Ribosome profiling and RNA-seq were performed upon the experimental condition where cells carry a high copy of Pop5 gene cloned into a multicopy plasmid and the control condition where they carry empty vector. For ribosome profiling, Sample 1, 2 (experimental condition) and Sample 3, 4 (control condition) are biological replicates of each condition. For RNA-seq, Sample 5, 6  (experimental condition) and Sample 7, 8  (control condition) are biological replicates of each condition.', 'Series_type': 'Other', 'Series_contributor': 'Hisao,,Moriya', 'Series_sample_id': 'GSM2256656', 'Series_contact_name': 'Hisao,,Moriya', 'Series_contact_email': 'hisaom@cc.okayama-u.ac.jp', 'Series_contact_department': 'Research Core for Interdisciplinary Sciences', 'Series_contact_institute': 'Okayama University', 'Series_contact_address': 'Tsushima-naka 3-1-1, Kita-Ku', 'Series_contact_city': 'Okayama', 'Series_contact_zip/postal_code': '700-8530', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE85nnn/GSE85036/suppl/GSE85036_RAW.tar', 'Series_platform_id': 'GPL21656', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP080744'}\n",
      "140 PRJNA336542 GSE85198 {'Series_title': 'CAN1 Arginine Permease Deficiency Extends Yeast Replicative Lifespan via Translational Activation of Stress Response Genes', 'Series_geo_accession': 'GSE85198', 'Series_status': 'Public on Feb 20 2017', 'Series_submission_date': 'Aug 04 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28228255', 'Series_summary': 'Transcriptional regulation plays an important role in the control of gene expression during aging. However, translation efficiency likely plays an equally important role in determining protein abundance, but has been relatively under studied in this context. Here we used RNA-seq and ribosome profiling to investigate the role of translational regulation in lifespan extension by CAN1 gene deletion in yeast. Through comparison of the transcriptional and translational changes in cells lacking CAN1 with other long-lived mutants, we were able to identify critical regulatory factors, including transcription factors and mRNA-binding proteins, that coordinate transcriptional and translational responses. Together, our data support a model in which deletion of CAN1 extends replicative lifespan through increased translation of proteins that facilitate cellular response to stress. This study extends our understanding of the importance of translational control in regulating stress resistance and longevity.', 'Series_overall_design': 'RNA-seq and ribosome profiling analyses of wild-type, can1, rpl22a, and tor1 knockout strains', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Vyacheslav,M,Labunskyy', 'Series_sample_id': 'GSM2260121', 'Series_contact_name': 'Vyacheslav,,Labunskyy', 'Series_contact_email': 'vlabuns@bu.edu', 'Series_contact_department': 'Department of Dermatology', 'Series_contact_institute': 'Boston University School of Medicine', 'Series_contact_address': '609 Albany Street', 'Series_contact_city': 'Boston', 'Series_contact_state': 'Massachusetts', 'Series_contact_zip/postal_code': '02118', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE85nnn/GSE85198/suppl/GSE85198_RAW.tar', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP080941'}\n",
      "141 PRJNA339577 GSE85864 {'Series_title': 'Ribosome profiling of human platelets, reticulocytes, and in vitro cell lines', 'Series_geo_accession': 'GSE85864', 'Series_status': 'Public on Sep 29 2016', 'Series_submission_date': 'Aug 19 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27681415', 'Series_summary': 'Translation is a fundamental biological process and ribosomes are essential in all  cells, but defects in ribosome biogenesis (ribosomopathies) disproportionately  cause hematopoietic dysfunction of red blood cells and platelets. Here, we  analyze translation in primary human reticulocytes and platelets by ribosome  profiling and uncover dramatic accumulation of post-termination, unrecycled  ribosomes in the 3´UTRs of mRNAs. We then demonstrate that these ribosomes  accumulate as a result of the natural loss of the ribosome recycling factor ABCE1  during terminal differentiation. We find that induction of ribosome rescue factors  PELO and HBS1L is required to maintain translational homeostasis when ABCE1  levels fall. This activation of ribosome rescue mitigates the effects of ribosome  shortage on translational output, including on hemoglobin production. Our  observations suggest that this distinctive loss of ABCE1 in anucleate blood  lineages sensitizes them to defects in ribosome homeostasis and that activation  of a ribosome rescue pathway plays a lineage-specific role in maintaining  ribosome homeostasis during blood cell development.', 'Series_overall_design': 'Analysis of ribosome profiling and RNA sequencing data from primary human platelets, primary human reticulocytes, in vitro derived platelet-like particles, and hemin-induced differentiation of K562 cells; manipulation by knockdown and overexpression of Pelota (PELO) and HBS1L in K562 cells and Meg01/platelet-like particles.', 'Series_type': 'Other', 'Series_contributor': 'Ingolia,T,Nicholas', 'Series_sample_id': 'GSM2285927', 'Series_contact_name': 'Eric,W,Mills', 'Series_contact_email': 'ewmills@partners.org', 'Series_contact_institute': 'Johns Hopkins University School of Medicine', 'Series_contact_address': '725 N. Wolfe St', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'Maryland', 'Series_contact_zip/postal_code': '21205', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE85nnn/GSE85864/suppl/GSE85864_Primary_Retic_Ribo.txt.gz', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP082436'}\n",
      "142 PRJNA339793 GSE85944 {'Series_title': 'Translation of poly(A) tails leads to precise mRNA cleavage and widespread ribosome rescue', 'Series_geo_accession': 'GSE85944', 'Series_status': 'Public on Jan 22 2017', 'Series_submission_date': 'Aug 23 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28193672', 'Series_summary': 'Translation of poly(A) tails leads to mRNA cleavage but the mechanism and global pervasiveness of this “nonstop/no-go” decay process is not understood. Here we performed ribosome profiling of short 15-18 nt mRNA footprints to identify ribosomes stalled at 3’ ends of mRNA decay intermediates. We found mRNA cleavage extending hundreds of nucleotides upstream of ribosome stalling in poly(A) and predominantly in one reading frame. These observations suggest that cleavage is closely associated with the ribosome. Surprisingly, ribosomes appeared to stall when as few as 3 consecutive ORF-internal lysine codons were positioned in the A, P, and E sites though significant mRNA degradation was not observed. Endonucleolytic cleavage was widespread, however, at sites of premature polyadenylation and rescue of the ribosomes stalled at these sites was dependent on Dom34. These results suggest this process may be critical when changes in polyadenylation occur during development, tumorigenesis, or when translation termination/recycling is impaired.', 'Series_overall_design': \"14 samples are included in the study (2 mRNA-Seq samples and 12 ribosome footprinting samples). Read sizes were examined for lengths of 15-18 nt or 25-34 nt or both in some cases, reads were aligned by 5' or 3' ends or both in some cases.\", 'Series_type': 'Other', 'Series_contributor': 'Rachel,,Green', 'Series_sample_id': 'GSM2288308', 'Series_contact_name': 'Nicholas,R,Guydosh', 'Series_contact_email': 'nicholas.guydosh@nih.gov', 'Series_contact_laboratory': 'LBG', 'Series_contact_department': 'NIDDK', 'Series_contact_institute': 'National Institutes of Health', 'Series_contact_address': '8 Center Dr, Bldg 8, 220', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '20892', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE85nnn/GSE85944/suppl/GSE85944_RAW.tar', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP082570'}\n",
      "143 PRJNA340948 GSE86214 {'Series_title': 'YTHDF3 facilitates translation and decay of the N6-methyladenosine-modified RNA', 'Series_geo_accession': 'GSE86214', 'Series_status': 'Public on Sep 15 2016', 'Series_submission_date': 'Aug 30 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28106072', 'Series_summary': 'N6-methyladenosine (m6A) is the most abundant internal modification in eukaryotic messenger RNAs (mRNAs), and plays important roles in cell differentiation and organism development. It regulates multiple steps throughout the RNA life cycle including RNA processing, translation, and metabolism, via the recognition by selective binding proteins. In cytoplasm, m6A binding protein YTHDF1 facilitates translation of m6A-modified mRNAs, and YTHDF2 accelerates the decay of m6A-modified transcripts. The biological function of YTHDF3, another cytoplasmic m6A binder of the YTH domain family, remains unknown. Here, we report that YTHDF3 promotes protein synthesis in synergy with YTHDF1, and affects methylated mRNA decay mediated by YTHDF2. Cells deficient in all of YTHDF proteins experience the most dramatic accumulation of the m6A-methylated transcripts. These results indicate that in cytoplasm, YTHDF proteins act in an integrated and cooperative network to accelerate metabolism of m6A-modified mRNAs. The combinative and dynamic nature of YTHDF proteins may collectively impact fundamental biological processes and diseases related to m6A RNA methylation.', 'Series_overall_design': 'We utilized PAR-CLIP triplicates and RIP replicates to identify binding sites and target transcripts of YTHDF3, and performed ribosome profling replicates to assess the consequences of YTHDF3 siRNA knock-down in HeLa cells.', 'Series_type': 'Other', 'Series_contributor': 'Chuan,,He', 'Series_sample_id': 'GSM2424868', 'Series_contact_name': 'Chuan,,He', 'Series_contact_email': 'chuanhe@uchicago.edu', 'Series_contact_phone': '773-702-5061', 'Series_contact_department': 'Department of Chemistry', 'Series_contact_institute': 'The University of Chicago', 'Series_contact_address': '929 E 57th St., GCIS E319', 'Series_contact_city': 'Chicago', 'Series_contact_state': 'ILLINOIS', 'Series_contact_zip/postal_code': '60637', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE86nnn/GSE86214/suppl/GSE86214_siYTHDF3_YTHDF12RIP.xlsx', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP083699'}\n",
      "144 PRJNA344393 GSE87328 {'Series_title': 'eIF1 modulates the recognition of sub-optimal translation start sites and steers gene expression control mediated by uORFs', 'Series_geo_accession': 'GSE87328', 'Series_status': 'Public on Jan 25 2017', 'Series_submission_date': 'Sep 24 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28541577', 'Series_summary': 'Alternative translation initiation mechanisms such as leaky scanning and reinitiation potentiate the polycistronic nature of transcripts. By allowing for reprogrammed translation, these mechanisms can mediate biological responses to stress stimuli. We combined proteomics with ribosome profiling and mRNA sequencing to identify the biological targets of translation control triggered by eukaryotic translation initiation factor 1 (eIF1), a protein implicated in the stringency of start codon selection. We quantified expression changes of over 4,000 proteins and 10,000 actively translated transcripts, leading to the identification of 245 transcripts undergoing translational control mediated by upstream open reading frames (uORFs) upon eIF1 deprivation. The stringency of start codon selection and preference for optimal nucleotide context were largely diminished leading to translational regulation of uORFs with sub-optimal start sites. Affected genes were implicated in energy production and sensing of metabolic stress. Interestingly, knockdown of eIF1 elicited a synergic response from eIF5 and eIF1B.', 'Series_overall_design': 'Ribosome profiling and mRNA sequening of human HCT116 cells upon eIF1 knockdown and in control conditions.', 'Series_type': 'Other', 'Series_contributor': 'Petra,,Van Damme', 'Series_sample_id': 'GSM2327830', 'Series_contact_name': 'Petra,Van,Damme', 'Series_contact_email': 'Petra.VanDamme@UGent.be', 'Series_contact_phone': '+32 9 264 9279', 'Series_contact_laboratory': 'Kris Gevaert Lab', 'Series_contact_department': 'Department of Biochemistry', 'Series_contact_institute': 'University of Gent', 'Series_contact_address': 'Albert Baertsoenkaai 3', 'Series_contact_city': 'Gent', 'Series_contact_state': 'East Flanders', 'Series_contact_zip/postal_code': '9000', 'Series_contact_country': 'Belgium', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE87nnn/GSE87328/suppl/GSE87328_RAW.tar', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP090415'}\n",
      "145 PRJNA347980 GSE87871 {'Series_title': 'Identification of unannotated small genes in Salmonella', 'Series_geo_accession': 'GSE87871', 'Series_status': 'Public on Mar 09 2017', 'Series_submission_date': 'Oct 12 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28122954', 'Series_summary': 'Bacterial genomes serve as a blueprint in all aspects of biological research, and therefore accurate annotation is of paramount importance. However, increasing evidence suggests annotated bacterial genomes have missed many, if not all, small genes encoding proteins ≤60 amino acids. To uncover unannotated small genes in Salmonella enterica, we used a genomic technique ‘ribosome profiling’ that provides a snapshot of all mRNAs being translated (translatome). For comprehensive identification of the small genes, we obtained Salmonella translatomes in four different growth conditions including Luria-Bertani medium, MOPS rich defined medium, N-minimal medium with low Mg2+ (10 μM), and N-minimal medium at pH 5.7, respectively. Ribosome profiling data were analysed in combination with in silico predicted putative open reading frames and transcriptome profiles. As a result, we uncovered 127 previously unannotated genes, the majority of which were small genes encoding proteins ≤50 amino acids. We validated expression by western blot of the identified unannotated small genes, the smallest of which is 7-amino acid long. Because some unannotated small genes identified in this study are only expressed in the infection-relevant low Mg2+ conditions, they are likely involved in cellular processes required for Salmonella virulence. Our findings suggest that currently sequenced bacterial genomes are likely under-annotated with regard to small genes and need to be revised.', 'Series_overall_design': 'Ribosome profiling and RNA-seq were performed on Salmonella enterica 14028s cells grown in four different growth conditions.', 'Series_type': 'Other', 'Series_contributor': 'Hyunwoo,,Lee', 'Series_sample_id': 'GSM2342125', 'Series_contact_name': 'Hyunwoo,,Lee', 'Series_contact_email': 'hlee31@uic.edu', 'Series_contact_department': 'Center for Biomolecular Sciences', 'Series_contact_institute': 'University of Illinois at Chicago', 'Series_contact_address': '900 S. Ashland Ave., MBRB #3020', 'Series_contact_city': 'Chicago', 'Series_contact_state': 'IL', 'Series_contact_zip/postal_code': '60607', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE87nnn/GSE87871/suppl/GSE87871_RAW.tar', 'Series_platform_id': 'GPL20046', 'Series_platform_taxid': '588858', 'Series_sample_taxid': '588858', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP091438'}\n",
      "146 PRJNA348115 GSE87892 {'Series_title': 'Ribosome profiling of dhh1∆ yeast', 'Series_geo_accession': 'GSE87892', 'Series_status': 'Public on Nov 04 2016', 'Series_submission_date': 'Oct 12 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27821408', 'Series_summary': 'The impact of RNA structures in coding sequences (CDS) within mRNAs is poorly understood. Here \\u2028we identify a novel and highly conserved mechanism of translational control involving RNA\\xa0structures within coding sequences and the DEAD-box helicase Dhh1. Using yeast genetics and \\u2028genome-wide ribosome profiling analyses we show that this mechanism, initially derived from studies \\u2028of the Brome Mosaic virus RNA genome, extends to yeast and human mRNAs highly enriched in \\u2028membrane and secreted proteins. All Dhh1-dependent mRNAs, viral and cellular, share key common \\u2028features. First, they contain long and highly structured CDSs, including a region located around 42 to \\u2028120 nucleotides after the translation initiation site, second, they are directly bound by Dhh1 with a \\u2028specific binding distribution and third, complementary experimental approaches suggest that they are \\u2028activated by Dhh1 at the translation initiation step. Our results show that ribosome translocation is not the only unwinding force of CDS and uncover a conserved layer of translational control that involve \\u2028RNA helicases and RNA folding within CDS providing novel opportunities for regulation of \\u2028membrane and secretome proteins.', 'Series_overall_design': 'Cells were grown in rich medium and sequencing was performed on ribosome-protected footprints and matched samples of randomly fragmented polyA-enriched RNA. Dataset contains three biological replicates.', 'Series_type': 'Other', 'Series_contributor': 'Danny,D,Nedialkova', 'Series_sample_id': 'GSM2342292', 'Series_contact_name': 'Danny,D.,Nedialkova', 'Series_contact_email': 'danny.nedialkova@mpi-muenster.mpg.de', 'Series_contact_department': 'Max Planck Research Group for RNA Biology', 'Series_contact_institute': 'Max Planck Institute for Molecular Biomedicine', 'Series_contact_address': 'Von-Esmarch-Str. 54', 'Series_contact_city': 'Münster', 'Series_contact_zip/postal_code': '48149', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE87nnn/GSE87892/suppl/GSE87892_Dhh1_KO_count_table.txt.gz', 'Series_platform_id': 'GPL19958', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP091454'}\n",
      "147 PRJNA348358 GSE88725 {'Series_title': 'Global analysis of translation termination in E. coli using release factor manipulations', 'Series_geo_accession': 'GSE88725', 'Series_status': 'Public on Mar 09 2017', 'Series_submission_date': 'Oct 13 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28301469', 'Series_summary': 'Terminating protein translation accurately and efficiently is critical for both protein fidelity and ribosome recycling for continued translation. The three bacterial release factors (RFs) play key roles:  RF1 and 2 recognize stop codons and terminate translation; and RF3 promotes disassociation of bound release factors. Probing release factors mutations with reporter constructs containing programmed frameshifting sequences or premature stop codons had revealed a propensity for readthrough or frameshifting at these specific sites, but their effects on translation genome-wide have not been examined. We performed ribosome profiling on a set of isogenic strains with well-characterized release factor mutations to determine how they alter translation globally.  Consistent with their known defects, strains with increasingly severe release factor defects exhibit increasingly severe accumulation of ribosomes over stop codons, indicative of an increased duration of the termination/release phase of translation. Release factor mutant strains also exhibit increased occupancy in the region following the stop codon at a significant number of genes.  Our global analysis revealed that, as expected, translation termination is generally efficient and accurate, but that at a significant number of genes (≥ 50) the ribosome signature after the stop codon is suggestive of translation past the stop codon. Even native E. coli K-12 exhibits the ribosome signature suggestive of protein extension, especially at UGA codons, which rely exclusively on the reduced function RF2 allele of the K-12 strain for termination. Deletion of RF3 increases the severity of the defect.  We unambiguously demonstrate readthrough and frameshifting protein extensions and their further accumulation in mutant strains for a few select cases. In addition to enhancing recoding, ribosome accumulation over stop codons disrupts attenuation control of biosynthetic operons, and may alter expression of some overlapping genes.  Together, these functional alterations may either augment the protein repertoire or produce deleterious proteins.', 'Series_overall_design': 'Samples of WT, ∆RF3, RF2B, and RF2B∆RF3 were analyzed by ribosome profiling and RNA-seq. Over 4 experiments there are 4 replicates of WT, 2 replicates of ∆RF3, 2 replicates of RF2B and 5 replicates of RF2B∆RF3. One experiment (both ribosome profiling and RNA-seq) was performed in minimal media with a single sample for WT and ∆RF3', 'Series_type': 'Other', 'Series_contributor': 'Carol,,Gross', 'Series_sample_id': 'GSM2344812', 'Series_contact_name': 'Natalie,,Baggett', 'Series_contact_email': 'BAGGETT.NATALIE@GMAIL.COM', 'Series_contact_phone': '415-476-1493', 'Series_contact_laboratory': 'Carol Gross', 'Series_contact_department': 'Microbiology and Immunology', 'Series_contact_institute': 'UCSF', 'Series_contact_address': '600 16th Street, S376', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE88nnn/GSE88725/suppl/GSE88725_RAW.tar', 'Series_platform_id': 'GPL14548', 'Series_platform_taxid': '562', 'Series_sample_taxid': '562', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP091515'}\n",
      "148 PRJNA348553 GSE88790 {'Series_title': 'N-terminal Proteomics Assisted Profiling of the Unexplored Translation Initiation Landscape in Arabidopsis thaliana.', 'Series_geo_accession': 'GSE88790', 'Series_status': 'Public on Nov 25 2016', 'Series_submission_date': 'Oct 14 2016', 'Series_last_update_date': 'Jan 31 2022', 'Series_pubmed_id': '35069635', 'Series_summary': 'Proteogenomics is an emerging research field yet lacking a standard method of analysis. In this article, we demonstrate the strength of proteogenomic analysis specific for N-terminal data that aims at the discovery of novel translational start sites. In summary, unidentified spectra were matched to a specific N-terminal peptide library encompassing all theoretical protein N-termini encoded in the genome. Gene prediction suggested 81 protein-coding models, of which several alternative proteoforms with unannotated protein starts. Next to the proteomic data, complementary ribosome footprinting data was generated from Arabidopsis thaliana cell cultures. Translation initiation site mapping by the ribosome footprinting data provided orthogonal evidence for 14 novel peptides identified by our proteogenomics pipeline.', 'Series_overall_design': 'Ribosome profiling sequencing of Arabidopsis thaliana wild-type ecotype Landsberg erecta cells treated with lactimidomycin or cycloheximide.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Petra,,VanDamme', 'Series_sample_id': 'GSM2346475', 'Series_contact_name': 'Petra,Van,Damme', 'Series_contact_email': 'Petra.VanDamme@UGent.be', 'Series_contact_phone': '+32 9 264 9279', 'Series_contact_laboratory': 'Kris Gevaert Lab', 'Series_contact_department': 'Department of Biochemistry', 'Series_contact_institute': 'University of Gent', 'Series_contact_address': 'Albert Baertsoenkaai 3', 'Series_contact_city': 'Gent', 'Series_contact_state': 'East Flanders', 'Series_contact_zip/postal_code': '9000', 'Series_contact_country': 'Belgium', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE88nnn/GSE88790/suppl/GSE88790_RAW.tar', 'Series_platform_id': 'GPL13222', 'Series_platform_taxid': '3702', 'Series_sample_taxid': '3702', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP091588'}\n",
      "149 PRJNA354729 GSE90469 {'Series_title': 'Ribosome profiling of G2019S LRRK2 human dopamine neurons', 'Series_geo_accession': 'GSE90469', 'Series_status': 'Public on Jul 01 2018', 'Series_submission_date': 'Nov 23 2016', 'Series_last_update_date': 'Aug 17 2020', 'Series_summary': 'G2019S mutaion of LRRK2 is known to increase mRNA translation. We perform ribosome profiling to study defective translation using human dopamine neuron models. Patient-derived human dopamine neurons with G2019S LRRK2 mutation were generated and used. Also a mutation-corrected isogenic pair line was used.', 'Series_overall_design': 'hDA neurons: 60 days old samples were used.', 'Series_type': 'Other', 'Series_contributor': 'Valina,L,Dawson', 'Series_sample_id': 'GSM4075585', 'Series_contact_name': 'Jungwoo Wren,,Kim', 'Series_contact_email': 'jwrenkim@berkeley.edu, jwk0906@gmail.com', 'Series_contact_laboratory': 'Ingolia Lab', 'Series_contact_department': 'Molecular and Cell Biology', 'Series_contact_institute': 'UC Berkeley', 'Series_contact_address': 'Barker Hall, Room 416', 'Series_contact_city': 'Berkeley', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94720', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE90nnn/GSE90469/suppl/GSE90469_iso_hDA_ribo.csv.gz', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP093833'}\n",
      "150 PRJNA356772 GSE91066 {'Series_title': 'REPARATION:  Ribosome Profiling Assisted (Re-) Annotation of Bacterial genomes', 'Series_geo_accession': 'GSE91066', 'Series_status': 'Public on Jan 30 2017', 'Series_submission_date': 'Dec 08 2016', 'Series_last_update_date': 'Nov 04 2021', 'Series_pubmed_id': '34721520', 'Series_summary': 'The delineation of genes in bacteria has remained an important challenge because prokaryotic genomes are often tightly packed frequently resulting in overlapping genes. We hereby present a de novo approach called REPARATION (RibosomeE Profiling Assisted (Re-)AnnotaTION) to delineate translated open reading frames (ORFs) in bacteria independent of (available) genome annotation. By deep sequencing of ribosome protected mRNA fragments (RPF) to map translating ribosomes across the entire genome, REPARATION takes advantage of the recently developed ribosome profiling (Ribo-seq) technique. REPARATION starts by traversing the entire genome to generate all possible ORFs and then collects their corresponding RPF signal information. Based on a growth curve model to estimate minimum ORF read density and Ribo-seq RPF coverage, thresholds indicative of translation is estimated. Finally, our algorithm applies a random forest model to build a classifier to classify putative protein coding ORFs. We evaluated the performance of REPARATION on 3 annotated bacterial species using in-house generated Ribo-seq data and matching N-terminal and shotgun proteomics data next to publically available Ribo-seq data. In all cases, about 80% of the ORFs predicted by REPARATION were previously annotated as protein coding. While 13-20% were variants of previously annotated ORFs and about 3-4% point to novel translated ORFs within intergenic or other regions previously annotated as non-coding. Without stringent length restrictions REPARATION was able to identify several small ORFs (sORFs). Multiple supportive evidence from matching MS data and sequence conservation analysis was obtained to validate predicted ORFs.', 'Series_overall_design': 'Ribosome profiling and mRNA sequencing of monosome and polysome-enriched samples of Salmonella enterica serovar Typhimurium - strain SL1344.', 'Series_type': 'Other', 'Series_contributor': 'Petra,,Van Damme', 'Series_sample_id': 'GSM2420421', 'Series_contact_name': 'Petra,Van,Damme', 'Series_contact_email': 'Petra.VanDamme@UGent.be', 'Series_contact_phone': '+32 9 264 9279', 'Series_contact_laboratory': 'Kris Gevaert Lab', 'Series_contact_department': 'Department of Biochemistry', 'Series_contact_institute': 'University of Gent', 'Series_contact_address': 'Albert Baertsoenkaai 3', 'Series_contact_city': 'Gent', 'Series_contact_state': 'East Flanders', 'Series_contact_zip/postal_code': '9000', 'Series_contact_country': 'Belgium', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE91nnn/GSE91066/suppl/GSE91066_RAW.tar', 'Series_platform_id': 'GPL20056', 'Series_platform_taxid': '216597', 'Series_sample_taxid': '216597', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP094797'}\n",
      "151 PRJNA358868 GSE92922 {'Series_title': 'Endoplasmic reticulum as a hub organizing microRNAs and a subset of endogenous siRNAs in Arabidopsis [Ribo-Seq and RNA-Seq]', 'Series_geo_accession': 'GSE92922', 'Series_status': 'Public on Dec 27 2016', 'Series_submission_date': 'Dec 27 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27938667', 'Series_summary': 'Endoplasmic reticulum (ER) was implicated as the site of microRNA (miRNA)-mediated translational repression in plants. Here, we examined the ER- and rough ER- associated small RNAome, transcriptome and translatome. We found that nearly all cellular transcripts were present on membrane-bound polysomes (MBPs), and miRNAs and a small set of endogenous siRNAs were particularly enriched on MBPs. The MBP- enriched miRNAs and siRNAs associated with, and were recruited to membranes by, their effector protein ARGONAUTE1 (AGO1). AGO1 associated with ER in a partly RNA-independent manner. Reduced membrane association of 22-nt miRNAs, which trigger the biogenesis of phased, secondary siRNAs (phasiRNAs) from their target transcripts, was accompanied by decreased production or loss of phasing of phasiRNAs. The phasiRNA precursor transcripts, previously thought to be noncoding, were associated with MBPs in a manner that supported phasiRNA production. These findings point to the ER as a hub that hosts and organizes endogenous small RNAs in plants.', 'Series_overall_design': 'Ribo-Seq and mRNA-Seq libraries from RNA extracted from membrane bound polysome fractions (One biological replicate)', 'Series_type': 'Other', 'Series_contributor': 'Lin,,Liu', 'Series_sample_id': 'GSM2440438', 'Series_contact_name': 'Chenjiang,,You', 'Series_contact_email': 'chenjiang.you@ucr.edu, youchenjiang2003@hotmail.com', 'Series_contact_institute': 'Fudan University', 'Series_contact_address': '2005 Songhu Rd', 'Series_contact_city': 'Shanghai', 'Series_contact_state': 'Shanghai', 'Series_contact_zip/postal_code': '200438', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE92nnn/GSE92922/suppl/GSE92922_Exon.RPM.txt.gz', 'Series_platform_id': 'GPL13222', 'Series_platform_taxid': '3702', 'Series_sample_taxid': '3702', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP095689'}\n",
      "152 PRJNA369542 GSE94385 {'Series_title': 'Ribosome profiling and RNA-Sequencing of wild type and Pelota knockout mouse epidermis', 'Series_geo_accession': 'GSE94385', 'Series_status': 'Public on Feb 02 2017', 'Series_submission_date': 'Feb 01 2017', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '29643507', 'Series_summary': 'mRNA quality control mechanisms ensure fidelity of protein translation. An evolutionarily conserved component of the quality control machinery, Dom34/Pelota (Pelo), rescues stalled ribosomes.  Here we show that Pelo is required for mammalian epidermal homeostasis. Conditional deletion in murine epidermal stem cells expressing Lrig1 results in hyperproliferation and abnormal differentiation, whereas deletion in other stem cells does not. Loss of Pelo results in truncated ribosome footprints and global upregulation of translation. Translational inhibition by rapamycin-mediated down regulation of mTOR rescues the epidermal phenotype. Our study reveals a novel role for the ribosome-rescue machinery in mammalian tissue homeostasis and an unanticipated specificity in its impact on different stem cell populations.', 'Series_overall_design': 'Analysis of ribosome profiling and RNA sequencing data from primary mouse epidermis; manipulation by transgenic knockout of Pelota (PELO).', 'Series_type': 'Other', 'Series_contributor': 'Fiona,M,Watt', 'Series_sample_id': 'GSM2474475', 'Series_contact_name': 'Eric,W,Mills', 'Series_contact_email': 'ewmills@partners.org', 'Series_contact_institute': 'Johns Hopkins University School of Medicine', 'Series_contact_address': '725 N. Wolfe St', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'Maryland', 'Series_contact_zip/postal_code': '21205', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE94nnn/GSE94385/suppl/GSE94385_read_me.txt', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP098700'}\n",
      "153 PRJNA369543 GSE94384 {'Series_title': 'RNA sequencing of primary human platelets and in vitro cell lines', 'Series_geo_accession': 'GSE94384', 'Series_status': 'Public on Feb 02 2017', 'Series_submission_date': 'Feb 01 2017', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28213379', 'Series_summary': 'Platelets are anucleate cytoplasmic fragments that lack genomic DNA, but continue to synthesize protein using a pool of mRNAs, ribosomes, and regulatory small RNAs inherited from the precursor megakaryocyte (MK). The regulatory processes that shape the platelet transcriptome and the full scope of platelet translation have remained elusive. Using RNA-Seq and ribosome profiling of primary human platelets, we show the platelet transcriptome encompasses a subset of transcripts detected by RNA-Seq analysis of in vitro derived MK cells and these platelet-enriched transcripts are broadly occupied by ribosomes. We use RNA sequencing of synchronized populations of in vitro derived platelet-like particles (PLPs) to show that mRNA decay strongly shapes the nascent platelet transcriptome. Our data suggests that the decay of platelet mRNAs is slowed by the natural loss of the mRNA surveillance and ribosome rescue factor Pelota (PELO).', 'Series_overall_design': 'Analysis of RNA sequencing data from primary human platelets and in vitro derived platelet-like particles; manipulation by overexpression of Pelota (PELO) in Meg01/platelet-like particles.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Nicholas,T,Ingolia', 'Series_sample_id': 'GSM2474468', 'Series_contact_name': 'Eric,W,Mills', 'Series_contact_email': 'ewmills@partners.org', 'Series_contact_institute': 'Johns Hopkins University School of Medicine', 'Series_contact_address': '725 N. Wolfe St', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'Maryland', 'Series_contact_zip/postal_code': '21205', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE94nnn/GSE94384/suppl/GSE94384_Primary_PLT_RNA.txt.gz', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP098699'}\n",
      "154 PRJNA369742 GSE94454 {'Series_title': 'Selective stalling of human translation through small molecule engagement of the ribosome nascent chain', 'Series_geo_accession': 'GSE94454', 'Series_status': 'Public on Jun 05 2017', 'Series_submission_date': 'Feb 03 2017', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28323820', 'Series_summary': 'We use ribosome profiling to demonstrate the selectivity of a small molecule, PF-06446846 that inhibits translation of its target by selectively inducing ribosome-stalling in a nascent chain sequence dependent manner.', 'Series_overall_design': 'Study 2: Huh7 cells were treated with 1.5 µM PF-06446846 or vehicle for 60 minutes and subjected to both ribosome profiling and mRNA-seq. This study included two biological replicates.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Jamie,,Cate', 'Series_sample_id': 'GSM2476022', 'Series_contact_name': 'Jamie,,Cate', 'Series_contact_institute': 'University of California', 'Series_contact_address': '748 Stanley Hall', 'Series_contact_city': 'Berkeley', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94720', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE94nnn/GSE94454/suppl/GSE94454_Z-score_analyses.xlsx', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP098789'}\n",
      "155 PRJNA375080 GSE94982 {'Series_title': 'A spatio-temporal translatome of mouse tissue development', 'Series_geo_accession': 'GSE94982', 'Series_status': 'Public on Mar 10 2020', 'Series_submission_date': 'Feb 16 2017', 'Series_last_update_date': 'Jul 02 2021', 'Series_pubmed_id': '34107020', 'Series_summary': 'Functional specification of mammalian tissues is a result of precise regulation of gene expression during development. Although previous transcriptomic and proteomic analyses have provided great biological insight into tissue specific gene expression and their physiological relevance in development, our understanding of translational regulation in developing tissues is lacking. Here, we report a spatio-temporally resolved translatome analysis of six mouse tissues at embryonic and adult stages to quantify the effects of translational regulation and identify new translational components. We quantified the spatial and temporal divergence of gene expression and showed specific changes in gene expression and pathways underlying the divergence. We further showed dynamic translational control by modulating translational efficiency, enhancing tissue specificity during development. We discovered thousands of actively translated upstream open read frames (ORFs) that exhibited spatio-temporal patterns and demonstrated their regulatory roles in translational regulation. Finally, we identified known and novel micropeptides encoded by small ORFs from long non-coding RNAs with functional relevance to tissue development. Our data and analyses facilitate a better understanding of complex translational regulation across tissue and developmental spectra and serve as a useful resource of mouse translatome.', 'Series_overall_design': 'Ribosome profiling (Ribo-seq) and RNA sequencing (RNA-seq) were performed for six embryo tissues and six adult tissues separately from wild-type C57BL/6 mice at embryonic day (E) 15.5 and postnatal day (P) 42, including ectoderm-derived brain and retina, mesoderm-derived heart and kidney, as well as endoderm-derived liver and lung. For each tissue, two biological replicates were performed.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Zhi,,Xie', 'Series_sample_id': 'GSM4371550', 'Series_contact_name': 'Hongwei,,Wang', 'Series_contact_email': 'bioccwhw@126.com', 'Series_contact_phone': '+86 20 6667 7086', 'Series_contact_laboratory': 'XieLab', 'Series_contact_department': 'Zhongshan Ophthalmic Center', 'Series_contact_institute': 'Sun Yat-Sen University', 'Series_contact_address': 'No 7 Jinsui Rd, Research Building, Rm 301', 'Series_contact_city': 'Guangzhou', 'Series_contact_state': 'Guangdong', 'Series_contact_zip/postal_code': '510060', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE94nnn/GSE94982/suppl/GSE94982_P42_Ribo-seq_exon_level_tpm.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP100063'}\n",
      "156 PRJNA375739 GSE94984 {'Series_title': 'Differentiation of ncRNAs from small mRNAs in Escherichia coli O157:H7 EDL933 (EHEC) by combined RNAseq and RIBOseq – ryhB encodes the regulatory RNA RyhB and a peptide, RyhP', 'Series_geo_accession': 'GSE94984', 'Series_status': 'Public on Feb 18 2017', 'Series_submission_date': 'Feb 16 2017', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28245801', 'Series_summary': 'While NGS allows rapid global detection of transcripts, it remains difficult to distinguish ncRNAs from short mRNAs. To detect potentially translated RNAs, we developed an improved protocol for bacterial ribosomal footprinting (RIBOseq). This allowed distinguishing ncRNA from mRNA in EHEC. A high ratio of ribosomal footprints per transcript (ribosomal coverage value, RCV) is expected to indicate a translated RNA, while a low RCV should point to a non-translated RNA. Based on their low RCV, 150 novel non-translated EHEC transcripts were identified as putative ncRNAs, representing both antisense and intergenic transcripts, 74 of which had expressed homologs in E. coli MG1655. Bioinformatics analysis predicted statistically significant target regulons for 15 of the intergenic transcripts; experimental analysis revealed 4-fold or higher differential expression of 46 novel ncRNA in different growth media. Out of 329 annotated EHEC ncRNAs, 52 showed an RCV similar to protein-coding genes, of those, 16 had RIBOseq patterns matching annotated genes in other enterobacteriaceae, and 11 seem to possess a Shine-Dalgarno sequence, suggesting that such ncRNAs may encode small proteins instead of being solely non-coding. To support that the RIBOseq signals are reflecting translation, we tested the ribosomal-footprint covered ORF of ryhB and found a phenotype for the encoded peptide in iron-limiting condition. Determination of the RCV is a useful approach for a rapid first-step differentiation between bacterial ncRNAs and small mRNAs. Further, many known ncRNAs may encode proteins as well.', 'Series_overall_design': 'Comparison of RNAseq with RIBOseq data from EHEC grown in LB medium', 'Series_type': 'Other', 'Series_contributor': 'Siegfried,,Scherer', 'Series_sample_id': 'GSM2493798', 'Series_contact_name': 'Klaus,,Neuhaus', 'Series_contact_email': 'neuhaus@wzw.tum.de', 'Series_contact_department': 'Core Facility Microbiome/NGS', 'Series_contact_institute': 'Technische Universität München', 'Series_contact_address': 'Weihenstephaner Berg 3', 'Series_contact_city': 'Freising', 'Series_contact_zip/postal_code': '85354', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE94nnn/GSE94984/suppl/GSE94984_klausneuhaus_suppl.xlsx.gz', 'Series_platform_id': 'GPL23073', 'Series_platform_taxid': '155864', 'Series_sample_taxid': '155864', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP100156'}\n",
      "157 PRJNA376419 GSE95211 {'Series_title': 'Evolutionary Convergence of Pathway-specific Enzyme Expression Stoichiometry', 'Series_geo_accession': 'GSE95211', 'Series_status': 'Public on Mar 28 2018', 'Series_submission_date': 'Feb 22 2017', 'Series_last_update_date': 'Mar 20 2019', 'Series_pubmed_id': '29606352', 'Series_summary': 'Coexpression of proteins in response to pathway-inducing signals is the founding paradigm of gene regulation. Yet, it remains unexplored whether the relative abundance of coregulated proteins requires precise tuning. Here we present large-scale analyses of protein stoichiometry and corresponding regulatory strategies for 21 pathways and 67-224 operons in divergent bacteria separated by 0.6-2 billion years. Using end-enriched RNA-sequencing (Rend-seq) with single-nucleotide resolution, we found that many bacterial gene clusters encoding conserved pathways have undergone massive divergence in transcript abundance and architectures via remodeling of internal promoters and terminators. Remarkably, these evolutionary changes are compensated post-transcriptionally to maintain preferred stoichiometry of protein synthesis rates. Even more strikingly, in eukaryotic budding yeast, functionally analogous proteins that arose independently from bacterial counterparts also evolved to convergent in-pathway expression. The broad requirement for exact protein stoichiometries despite regulatory divergence provides an unexpected principle for building biological pathways both in nature and for synthetic activities.', 'Series_overall_design': 'Three ribosome profiling datasets (two in Bacillus subtilis and one in Vibrio natriegens). Eighteen Rend-seq data sets (Esherichia coli,  Bacillus subtilis, Vibrio natriegens, and Caulobacter crescentus, including mutants and different library preparations).', 'Series_type': 'Other', 'Series_contributor': 'Gene-Wei,,Li', 'Series_sample_id': 'GSM2971255', 'Series_contact_name': 'Jean-Benoit,,Lalanne', 'Series_contact_email': 'lalannej@uw.edu', 'Series_contact_laboratory': 'Jay Shendure', 'Series_contact_department': 'Genome Sciences', 'Series_contact_institute': 'University of Washington', 'Series_contact_address': '3720 15th Ave NE', 'Series_contact_city': 'Seattle', 'Series_contact_state': 'WA', 'Series_contact_zip/postal_code': '98195', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE95nnn/GSE95211/suppl/GSE95211_RAW.tar', 'Series_platform_id': 'GPL24556', 'Series_platform_taxid': '1219067', 'Series_sample_taxid': '1219067', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP100536'}\n",
      "158 PRJNA377088 GSE95415 {'Series_title': 'Global mRNA polarization regulates translation efficiency in the intestinal epithelium [Ribo-Seq]', 'Series_geo_accession': 'GSE95415', 'Series_status': 'Public on Aug 10 2017', 'Series_submission_date': 'Feb 27 2017', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28798045', 'Series_summary': 'specific boost in their translation. This lead to increased protein production, required for efficient nutrient absorption. These findings reveal a post-transcriptional regulatory mechanism involving dynamic polarization of mRNA and polarized translation.', 'Series_overall_design': 'RNAseq and RiboSeq of the murine duodenum.', 'Series_type': 'Other', 'Series_contributor': 'Shalev,,Itzkovitz', 'Series_sample_id': 'GSM2510305', 'Series_contact_name': 'Andreas,,Moor', 'Series_contact_email': 'andreas.moor@bsse.ethz.ch', 'Series_contact_department': 'Department of Biosystems Science and Engineering', 'Series_contact_institute': 'ETH Zurich', 'Series_contact_address': 'Mattenstrasse 26', 'Series_contact_city': 'Basel', 'Series_contact_zip/postal_code': '4058', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE95nnn/GSE95415/suppl/GSE95415_Duodenum_3_refed.stats.txt.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP100770'}\n",
      "159 PRJNA379292 GSE96643 {'Series_title': 'RNA_seq and Ribo_seq analyses of control and CPT-treated MCF7 cells', 'Series_geo_accession': 'GSE96643', 'Series_status': 'Public on Apr 01 2017', 'Series_submission_date': 'Mar 15 2017', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28388414', 'Series_summary': 'We used RNA-seq and Ribo-seq analyses to examine the effect of CPT treatment of translation efficiency (TE)', 'Series_overall_design': 'We measured expression levels (RNA.seq) and ribosome densities (ribo-seq) using biological duplicates of control and CPT-treated (5 hrs) MCF7 cells', 'Series_type': 'Other', 'Series_contributor': 'R,,Agami', 'Series_sample_id': 'GSM2537146', 'Series_contact_name': 'Ran,,Elkon', 'Series_contact_department': 'Human Genetics', 'Series_contact_institute': 'Tel Aviv University', 'Series_contact_address': 'Ramat Aviv', 'Series_contact_city': 'Tel Aviv', 'Series_contact_zip/postal_code': '69494', 'Series_contact_country': 'Israel', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE96nnn/GSE96643/suppl/GSE96643_rna_rp_P.genes_cnts_and_rpkms.txt.gz', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP101952'}\n",
      "160 PRJNA379447 GSE96716 {'Series_title': 'RNA_seq and Ribo_seq analyses applied to PC9 and H1933 human cancer cell lines', 'Series_geo_accession': 'GSE96716', 'Series_status': 'Public on Jun 13 2017', 'Series_submission_date': 'Mar 16 2017', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28388414', 'Series_summary': 'We used RNA-seq and Ribo-seq analyses to examine translation efficiency (TE) in PC9 and H1933 cells', 'Series_overall_design': 'We measured expression levels (RNA.seq) and ribosome densities (ribo-seq) in PC9 and H1933 cell lines', 'Series_type': 'Other', 'Series_contributor': 'R,,Agami', 'Series_sample_id': 'GSM2538904', 'Series_contact_name': 'Ran,,Elkon', 'Series_contact_department': 'Human Genetics', 'Series_contact_institute': 'Tel Aviv University', 'Series_contact_address': 'Ramat Aviv', 'Series_contact_city': 'Tel Aviv', 'Series_contact_zip/postal_code': '69494', 'Series_contact_country': 'Israel', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE96nnn/GSE96716/suppl/GSE96716_PC9_H1933_rna_rp_fpkms.txt.gz', 'Series_platform_id': 'GPL9115', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP102021'}\n",
      "161 PRJNA379448 GSE96714 {'Series_title': 'RNA_seq and Ribo_seq analyses of control and Nutlin3a-treated MCF7 cells (20 hrs)', 'Series_geo_accession': 'GSE96714', 'Series_status': 'Public on Jun 13 2017', 'Series_submission_date': 'Mar 16 2017', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28388414', 'Series_summary': 'We used RNA-seq and Ribo-seq analyses to examine the effect of Nutlin3a (activator of p53) treatment of translation efficiency (TE)', 'Series_overall_design': 'We measured expression levels (RNA.seq) and ribosome densities (ribo-seq) in control and Nutlin3a-treated (20 hrs) MCF7 cells', 'Series_type': 'Other', 'Series_contributor': 'R,,Agami', 'Series_sample_id': 'GSM2538886', 'Series_contact_name': 'Ran,,Elkon', 'Series_contact_department': 'Human Genetics', 'Series_contact_institute': 'Tel Aviv University', 'Series_contact_address': 'Ramat Aviv', 'Series_contact_city': 'Tel Aviv', 'Series_contact_zip/postal_code': '69494', 'Series_contact_country': 'Israel', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE96nnn/GSE96714/suppl/GSE96714_MCF7.Nutlin.rna.rp.fpkm.data.txt.gz', 'Series_platform_id': 'GPL9115', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP102020'}\n",
      "162 PRJNA380795 GSE97140 {'Series_title': 'Methylation of human eukaryotic elongation factor alpha (eEF1A) by a member of a novel protein lysine methyltransferase family modulates mRNA translation', 'Series_geo_accession': 'GSE97140', 'Series_status': 'Public on May 23 2017', 'Series_submission_date': 'Mar 28 2017', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28520920', 'Series_summary': 'Many cellular proteins are post-translationally modified by methylation of lysine residues. This has been most intensively studied in the case of histone proteins, where lysine methylations in the flexible tails are determinants of chromatin state and gene expression. Lysine methylations on non-histone proteins are also frequent, but in most cases the functional significance of the methylation event, as well as the identity of the responsible lysine (K) specific methyltransferase (KMT), remain unknown. While the majority of identified human KMTs belong to the SET (Su(var)3-9, Enhancer-of-zeste, Trithorax)-domain class of methyltransferases (MTases), several recently discovered KMTs belong to a different MTase class, the seven-beta-strand (7BS) MTases. Here, we have investigated an uncharacterized human 7BS MTase currently annotated as being part of the endothelin converting enzyme 2 (ECE2). However, transcriptomics data indicate that this MTase is not part of ECE2, and should be considered a separate protein. Combining in vitro enzymology and analyses of knockout cells, we demonstrate that this MTase efficiently methylates K36 in eukaryotic translation elongation factor 1 alpha (eEF1A) in vitro and in vivo. We suggest that this novel KMT is named eEF1A-KMT4 (gene name EEF1AKMT4), in agreement with the recently established nomenclature. Furthermore, by ribosome profiling we show that the absence of K36 methylation affects translation dynamics and changes translation speed of distinct codons. Finally, we demonstrate that eEF1A-KMT4 is part of a novel family of human KMTs, defined by a shared sequence motif in the active site, and we show the importance of this motif for catalytic activity.', 'Series_overall_design': 'This study includes the re-use of samples GSM2445294 and GSM2445295 from GSE93133. These two samples have been re-accessioned here so that the complete study is available from the SRP ID.', 'Series_type': 'Other', 'Series_contributor': 'Pål,,Falnes', 'Series_sample_id': 'GSM2552201', 'Series_contact_name': 'Benedikt,Sebastian,Nilges', 'Series_contact_email': 'benedikt.nilges@mpi-muenster.mpg.de', 'Series_contact_department': 'RNA Biology Group', 'Series_contact_institute': 'Max Planck Institute for Molecular Biomedicine', 'Series_contact_address': 'Von-Esmarch-Straße 54', 'Series_contact_city': 'Muenster', 'Series_contact_zip/postal_code': '48149', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE97nnn/GSE97140/suppl/GSE97140_count_table.txt.gz', 'Series_platform_id': 'GPL15456', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP102616'}\n",
      "163 PRJNA381205 GSE97286 {'Series_title': 'Ribosome profiling in wild-type and APOBEC1-deficient BMDMs', 'Series_geo_accession': 'GSE97286', 'Series_status': 'Public on Dec 31 2018', 'Series_submission_date': 'Mar 31 2017', 'Series_last_update_date': 'May 15 2019', 'Series_summary': 'APOBEC1 wildtype (WT) and Apobec1-/- (KO) unstimulated bone marrow-derived macrophages (BMDMs) were subject to ribosome profiling (n=6 for each genotype) in order to investigate the effect of APOBEC1 on translation', 'Series_overall_design': 'Examination of effect of functional APOBEC1 on translation', 'Series_type': 'Other', 'Series_contributor': 'F,N,Papavasiliou', 'Series_sample_id': 'GSM2561027', 'Series_contact_name': 'F.,Nina,Papavasiliou', 'Series_contact_institute': 'Rockefeller University', 'Series_contact_address': '1230 York Ave.', 'Series_contact_city': 'New York', 'Series_contact_state': 'New York', 'Series_contact_zip/postal_code': '10065', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE97nnn/GSE97286/suppl/GSE97286_top25percent_resultsDESeq2_batchCorrected.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP102816'}\n",
      "164 PRJNA382534 GSE97637 {'Series_title': 'Ribosome pausing at the AT-rich codons regulates the protein expression of secondary metabolite gene clusters in the Streptomyces tsukubaensis NRRL 18488', 'Series_geo_accession': 'GSE97637', 'Series_status': 'Public on Oct 16 2019', 'Series_submission_date': 'Apr 11 2017', 'Series_last_update_date': 'May 18 2020', 'Series_pubmed_id': '32385251', 'Series_summary': 'Streptomyces tsukubaensis NRRL 18488 is the preferred strain for the production of immunosuppressant agent tacrolimus (FK506). To take full advantage of its genetic potential, systematic understanding of secondary metabolism and related regulatory mechanisms is highly demanded. Here, to this end, we complete its 7.9 Mbp linear genome sequence followed by integrating with multi-omics measurements. With accurate reannotation of FK506 gene cluster, total 2,389 transcription start sites were determined by using primary transcriptome analysis. Integrated analysis of transcriptome and translatome data revealed that secondary metabolic gene clusters, especially FK506 cluster, undergo translational control with decrease in translational efficiency according to the growth. Furthermore, we demonstrated that SD motif has little correlation with ribosome pausing but AT-rich codons delay the translational elongation. Strong ribosome pausing was observed in the rare TTA codon in FK506 cluster. This comprehensive genome-scale analysis provides insight to the translational regulation of secondary metabolism in S. tsukubaensis.', 'Series_overall_design': 'Transcription abundance and ribosomal profiling analysis of Streptomyces tsukubaensis according to the growth phase', 'Series_type': 'Other', 'Series_contributor': 'Byung-Kwan,,Cho', 'Series_sample_id': 'GSM2574064', 'Series_contact_name': 'Namil,,Lee', 'Series_contact_email': '1991namil@gmail.com', 'Series_contact_phone': '821098633995', 'Series_contact_laboratory': 'Systems and Synthetic Biology Lab', 'Series_contact_department': 'Biological science', 'Series_contact_institute': 'KAIST', 'Series_contact_address': 'Daehakro', 'Series_contact_city': 'Daejeon', 'Series_contact_zip/postal_code': '34141', 'Series_contact_country': 'South Korea', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE97nnn/GSE97637/suppl/GSE97637_tsukubaensis_TSS.gff.gz', 'Series_platform_id': 'GPL23282', 'Series_platform_taxid': '83656', 'Series_sample_taxid': '83656', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP103795'}\n",
      "165 PRJNA382718 GSE97717 {'Series_title': 'Translation initiation from conserved non-AUG codons provides additional layers of regulation and coding capacity', 'Series_geo_accession': 'GSE97717', 'Series_status': 'Public on Apr 13 2017', 'Series_submission_date': 'Apr 12 2017', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28655822', 'Series_summary': 'Neurospora crassa cpc-1 and Saccharomyces cerevisiae GCN4 are homologs specifying transcription activators that drive the transcriptional response to amino acid limitation. The cpc-1 mRNA contains two upstream open reading frames (uORFs) in its >700 nt 5’-leader and its expression is controlled at the level of translation in response to amino acid starvation. We used N. crassa cell-free extracts and obtained data indicating that cpc-1 uORF1 and uORF2 are functionally analogous to GCN4 uORF1 and uORF4 in controlling translation. We also found that the 5’ region upstream of the main coding sequence of the cpc-1 mRNA extends for more than 700 nucleotides without any in-frame stop codon. For 100 cpc-1 homologs from Pezizomycotina and from selected Basidiomycota, 5’ conserved extensions of the CPC1 reading frame are also observed. Multiple non-AUG near-cognate codons (NCCs) in the CPC1 reading frame upstream of uORF2, some deeply conserved, could potentially initiate translation. At least four NCCs initiated translation in vitro. In vivo data were consistent with initiation at NCCs to produce N-terminally extended N. crassa CPC1 isoforms. The pivotal role played by CPC1, combined with its translational regulation by uORFs and NCC utilization, underscore the emerging significance of non-canonical initiation events in controlling gene expression.', 'Series_overall_design': 'Ribosome profiling of  N. crassa wild type cells and N. crassa cells grown in the presence or absence of 3-Amino-1,2,4-triazole (3AT)', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Matthew,S,Sachs', 'Series_sample_id': 'GSM2576092', 'Series_contact_name': 'Audrey,M,Michel', 'Series_contact_email': 'A.Michel@ucc.ie', 'Series_contact_phone': '+353 876217126', 'Series_contact_laboratory': 'Baranov', 'Series_contact_department': 'School of Biochemistry and Cell Biology', 'Series_contact_institute': 'University College Cork', 'Series_contact_address': 'Western Gateway, Western Road', 'Series_contact_city': 'Cork', 'Series_contact_state': 'Cork', 'Series_contact_zip/postal_code': 'T12 XF62', 'Series_contact_country': 'Ireland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE97nnn/GSE97717/suppl/GSE97717_NCU07732-1.csv.gz', 'Series_platform_id': 'GPL16164', 'Series_platform_taxid': '5141', 'Series_sample_taxid': '5141', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP103847'}\n",
      "166 PRJNA384766 GSE98333 {'Series_title': 'Analysis of Ribosome Protected Fragments of mRNAs from yeast strains either expressing Rio1p or depleted of Rio1p', 'Series_geo_accession': 'GSE98333', 'Series_status': 'Public on Aug 16 2017', 'Series_submission_date': 'Apr 28 2017', 'Series_last_update_date': 'Jul 25 2021', 'Series_pubmed_id': '28977579', 'Series_summary': 'mRNAs bound by ribosomes from yeast cells were analysed in order to determine the exact position of ribosomes in the presence or absence of Rio1p. Beside total Ribosome Protected Fragments (RPFs), RPFs from mRNAs protected by immature pre-40S pre-ribosomes was also analysed. The analysis showed that immature 40S ribosomes can carry out translation and their premature entry into translation is hindered by Rio1p.', 'Series_overall_design': 'mRNAs fragments protected by ribosomes were subjected to NGS Ribosome Profiling experiment. Samples included (1) total ribosome-associated mRNAs from yeast cells expressing Rio1p, (2) total ribosome-associated mRNAs from yeast cells depleted of Rio1p and (3) immuno-selected mRNAs associated with immature pre-40S pre-ribosomal particles from yeast-cells depleted of Rio1p. Ribosome-protected mRNA fragments were used as templates for cDNA libraries. The resulting cDNAs were sequenced and the sequences were annealed to the yeast genome. The position of the reads were determined in relation to either the translation start site or the stop codon of the corresponding coding DNA sequences.', 'Series_type': 'Other', 'Series_contributor': 'Annie,,Mougin', 'Series_sample_id': 'GSM2592342', 'Series_contact_name': 'Beata,,Jady', 'Series_contact_email': 'jady@ibcg.biotoul.fr', 'Series_contact_institute': 'Laboratoire de Biologie Moléculaire Eucaryote', 'Series_contact_address': '118 route de Narbonne', 'Series_contact_city': 'Toulouse', 'Series_contact_zip/postal_code': '31062', 'Series_contact_country': 'France', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE98nnn/GSE98333/suppl/GSE98333_RAW.tar', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP105412'}\n",
      "167 PRJNA385767 GSE98623 {'Series_title': 'miCLIP, RNA-Seq, and Ribo-Seq in MOLM13 cells', 'Series_geo_accession': 'GSE98623', 'Series_status': 'Public on Sep 18 2017', 'Series_submission_date': 'May 08 2017', 'Series_last_update_date': 'Jul 25 2021', 'Series_pubmed_id': '28920958', 'Series_summary': 'We studied the following conditions: m6A RNA methylation in MOLM13 AML cells; METTL3 deficiency on gene expression and translational efficiency', 'Series_overall_design': 'miCLIP was performed on poly(A) mRNA from MOLM13 as previously described(see GSE63753), RNA-Seq and Ribo-Seq were performed on MOLM13 cells four days after lentiviral transduction with a control shRNA or two independent hairpins targeting METTL3 (shRNA#9 and shRNA#12).  All experiments were performed in biological triplicate.', 'Series_type': 'Other', 'Series_contributor': 'Samie,R,Jaffrey', 'Series_sample_id': 'GSM2602090', 'Series_contact_name': 'Brian,Frederick,Pickering', 'Series_contact_laboratory': 'Samie Jaffrey', 'Series_contact_department': 'Pharmacology', 'Series_contact_institute': 'Weill Cornell Medicine', 'Series_contact_address': '1300 York Ave', 'Series_contact_city': 'New York', 'Series_contact_state': 'New York', 'Series_contact_zip/postal_code': '10065', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE98nnn/GSE98623/suppl/GSE98623_rpf.counts.xls.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP106608'}\n",
      "168 PRJNA388919 GSE99585 {'Series_title': 'Leucine differentially regulates gene-specific translation in mouse skeletal muscle', 'Series_geo_accession': 'GSE99585', 'Series_status': 'Public on Feb 14 2018', 'Series_submission_date': 'Jun 01 2017', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28615380', 'Series_summary': 'Mice received an oral gavage of either leucine or vehicle, and quadriceps muscle were harvested for ribosome profiling or RNA-Seq analysis.', 'Series_overall_design': 'Gene expression analysis in the skeletal muscle of leucine treated or control mice.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Michael,T,Howard', 'Series_sample_id': 'GSM2648087', 'Series_contact_name': 'Michael,T,Howard', 'Series_contact_email': 'mhoward@genetics.utah.edu', 'Series_contact_phone': '801-585-1927', 'Series_contact_laboratory': 'Howard', 'Series_contact_department': 'Human Genetics', 'Series_contact_institute': 'University of Utah', 'Series_contact_address': '15 N 2030 E', 'Series_contact_city': 'Salt Lake City', 'Series_contact_state': 'UT', 'Series_contact_zip/postal_code': '84112', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE99nnn/GSE99585/suppl/GSE99585_RNA_RPF_RawCts.xlsx', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP108547'}\n",
      "169 PRJNA389737 GSE99827 {'Series_title': 'Exhaustive profiling in Arabidopsis reveals abundant polysome-associated 24-nt small RNAs including hitherto undescribed AGO5-associated pseudogene-derived siRNAs (psiRNAs) [sRNA]', 'Series_geo_accession': 'GSE99827', 'Series_status': 'Public on Dec 31 2017', 'Series_submission_date': 'Jun 08 2017', 'Series_last_update_date': 'Jul 25 2021', 'Series_pubmed_id': '31138671', 'Series_summary': 'The present study was primarily aimed at expanding the scope of the previous analyses, which focused largely on AGO1-miRNA interactions with total polysomes and MBPs, respectively. Our larger aim was to comprehensively analyze which, among the multiple AGO effector proteins and the numerous cellular sRNA classes of Arabidopsis, interact with the translational machinery.', 'Series_overall_design': '6 samples examined: translating-ribosome affinity immunopurification (TRAP) and duplicates of polysomal and monosomal ribosomal profiling in inflorescences, and small RNA-seq in seedlings.', 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'Olivier,,Voinnet', 'Series_sample_id': 'GSM2653809', 'Series_contact_name': 'Antonin,,Marchais', 'Series_contact_email': 'mantonin@ethz.ch', 'Series_contact_phone': '+41 44 632 64 08', 'Series_contact_laboratory': 'Group Prof. Olivier Voinnet', 'Series_contact_department': 'Department of Biology', 'Series_contact_institute': 'ETHZ', 'Series_contact_address': 'Universitatstrasse 2', 'Series_contact_city': 'Zurich', 'Series_contact_zip/postal_code': '8092', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE99nnn/GSE99827/suppl/GSE99827_RAW.tar', 'Series_platform_id': 'GPL17639', 'Series_platform_taxid': '3702', 'Series_sample_taxid': '3702', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP108862'}\n",
      "170 PRJNA390134 GSE99920 {'Series_title': 'Transcriptional and ribosome profiling of Drosophila 3rd instar larvae muscle', 'Series_geo_accession': 'GSE99920', 'Series_status': 'Public on Nov 06 2017', 'Series_submission_date': 'Jun 12 2017', 'Series_last_update_date': 'May 15 2019', 'Series_summary': 'Presynaptic efficacy can be modulated by retrograde control mechanisms, but the nature of these complex signaling systems remain obscure. Some studies have suggested that retrograde synaptic communication requires new protein synthesis in the postsynaptic compartment, yet the putative translational targets that mediate this signaling are enigmatic. To gain insight into the retrograde mechanisms that stabilize synaptic transmission at the neuromuscular junction, we have developed and optimized a tissue-specific ribosome profiling approach in Drosophila. We first demonstrate the ability of this technology to define genome-wide translational regulations that could not be inferred from transcription, and validate the superior sensitivity of ribosome profiling over conventional translational profiling. We then leverage this technology to test the relative contributions of transcriptional, translational, and post-translational mechanisms in the postsynaptic muscle that orchestrate the retrograde control of presynaptic function. Surprisingly, we find no changes in transcription or translation are necessary to enable retrograde homeostatic signaling. Rather, post-translational mechanisms appear to ultimately gate instructive retrograde communication. Finally, we find that a global increase in translation induces adaptive responses in both transcription and translation of protein chaperones and degradation factors to promote cellular proteostasis. Together, this demonstrates the power of ribosome profiling to define transcriptional, translational, and post-translational mechanisms driving retrograde signaling during adaptive plasticity.', 'Series_overall_design': 'Transcriptional (RNA-seq of total RNA) and ribosome profiling were performed in Drosophila muscle tissue. Three genotypes were analyzed: wild type (control sample), GluRIIA mutant and Tor muscle overexpression. In addition, translating ribosome affinity purification (TRAP) based translational profiling was done in wild type sample to assay the difference between ribosome profiling and TRAP translational profiling. Three replicates were performed for each analysis.', 'Series_type': 'Other', 'Series_contributor': 'Dion,K,Dickman', 'Series_sample_id': 'GSM2664273', 'Series_contact_name': 'Dion,K,Dickman', 'Series_contact_email': 'dickman@usc.edu', 'Series_contact_department': 'Neurobiology', 'Series_contact_institute': 'University of Southern California', 'Series_contact_address': '3641 Watt Way, HNB 309', 'Series_contact_city': 'Los Angeles', 'Series_contact_state': 'California', 'Series_contact_zip/postal_code': '90089-2520', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE99nnn/GSE99920/suppl/GSE99920_WT_transcriptional_profiling_R1_2_3_RPKM.xlsx', 'Series_platform_id': 'GPL19132', 'Series_platform_taxid': '7227', 'Series_sample_taxid': '7227', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP108999'}\n",
      "171 PRJNA391171 GSE100210 {'Series_title': 'RNA editing by ADAR1 leads to context-dependent transcriptome-wide changes in RNA Secondary Structure', 'Series_geo_accession': 'GSE100210', 'Series_status': 'Public on Jun 25 2017', 'Series_submission_date': 'Jun 19 2017', 'Series_last_update_date': 'Jul 25 2021', 'Series_pubmed_id': '29129909', 'Series_summary': 'Adenosine deaminase acting on RNA 1 (ADAR1) is the master RNA editor, catalyzing the deamination of adenosine to inosine. RNA editing is vital for  preventing abnormal activation of cytosolic nucleic acid sensing pathways by self-double-stranded RNAs. Here we determine by parallel analysis of RNA secondary structure sequencing (PARS-seq), the global RNA secondary structure changes in ADAR1 deficient cells. Surprisingly, ADAR1 silencing resulted in a lower global double-stranded to single-stranded RNA ratio, suggesting that A-to-I editing can stabilize a large subset of imperfect RNA duplexes. The duplexes destabilized by editing are composed of vastly complementary inverted Alus found in untranslated regions of genes with vital biological processes, including housekeeping functions and type-I interferon responses. They are predominantly cytoplasmic and generally demonstrate higher ribosomal occupancy. Our findings imply that the editing effect on RNA secondary structure is context dependent and underline the intricate regulatory role of ADAR1 on global RNA secondary structure.', 'Series_overall_design': 'Examination of control and ADAR1 KD HepG2 cells using PARSseq method in order to find changes in RNA 2D structure induced by A-to-I RNA editing. We used PARSseq for HepG2 control and ADAR KD cells in two replicates with S1/V1 nucleases treatment for each ((control+KD)*2*2= 8 PARSseq samples). We used RNAseq of HepG2 control and ADAR KD cells with additional Ribo-seq (control and ADAR KD in 2 replicates=4 Ribo-seq samples) to further confirm the changes we observed in the PARSseq experiment.', 'Series_type': 'Other', 'Series_contributor': 'Gidi,,Rechavi', 'Series_sample_id': 'GSM2676364', 'Series_contact_name': 'Oz,,Solomon', 'Series_contact_email': 'oz.solomon@live.biu.ac.il', 'Series_contact_institute': 'Technion', 'Series_contact_address': 'Technion', 'Series_contact_city': 'Haifa', 'Series_contact_zip/postal_code': '32000', 'Series_contact_country': 'Israel', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE100nnn/GSE100210/suppl/GSE100210_RPKM_table.all_samples.txt.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP109827'}\n",
      "172 PRJNA392312 GSE100626 {'Series_title': 'Transcriptome-wide analysis of roles for tRNA modifications in translational regulation', 'Series_geo_accession': 'GSE100626', 'Series_status': 'Public on Jun 29 2017', 'Series_submission_date': 'Jun 28 2017', 'Series_last_update_date': 'Oct 29 2019', 'Series_pubmed_id': '29198561', 'Series_summary': 'Covalent nucleotide modifications in noncoding RNAs such as tRNAs affect a plethora of biological processes, with new functions continuing to be discovered for even well-known tRNA modifications. To systematically compare the functions of a large set of ncRNA modifications in gene regulation, we carried out ribosome profiling and RNA-Seq in budding yeast for 57 nonessential genes involved in tRNA modification.', 'Series_overall_design': 'Yeast strains were treated with cycloheximide and proceeded to ribosome profiling. For ribosome-protected footprints (RPF), 80S monosome fractions were isolated after RNase I treatment, and 27-34 nt RPF were isolated by denaturing PAGE. For RNA-Seq, total RNA was depleted of rRNA using Ribo-Zero, followed by zinc-based fragmentation. RNA fragments and RPF are constructed into Illumina deep-sequencing libraries by RNA 3’ adaptor ligation and cDNA circularization. Barcoded libraries were sequenced on an Illumina NextSeq 500. Raw fastq reads were de-multiplexed and removed of adaptor sequence. RPF reads were mapped to S. cerevisiae rDNA and the mapping reads were discarded. The remaining RPF reads and RNA-seq reads were mapped to sacCer3 genome. Uniquely mapping reads in length of 27-34 nt (RPF) or 27 nt (RNA-seq) were quantified as RPKM.', 'Series_type': 'Other', 'Series_contributor': 'Oliver,,Rando', 'Series_sample_id': 'GSM2793661', 'Series_contact_name': 'Oliver,,Rando', 'Series_contact_email': 'oliver.rando@umassmed.edu', 'Series_contact_phone': '508-856-8879', 'Series_contact_laboratory': '940C', 'Series_contact_department': 'Biochemistry & Molecular Pharm', 'Series_contact_institute': 'UMass Medical School', 'Series_contact_address': '364 Plantation Street', 'Series_contact_city': 'Worcester', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '01605', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE100nnn/GSE100626/suppl/GSE100626_Chou_Table_S6.txt.gz', 'Series_platform_id': 'GPL19756', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP110707'}\n",
      "173 PRJNA395723 GSE101865 {'Series_title': 'Scope and mechanism of eIF4F-independent mRNA translation', 'Series_geo_accession': 'GSE101865', 'Series_status': 'Public on Jul 26 2017', 'Series_submission_date': 'Jul 25 2017', 'Series_last_update_date': 'Jul 25 2021', 'Series_pubmed_id': '29107534', 'Series_web_link': 'http://www.sciencedirect.com/science/article/pii/S1097276517307517', 'Series_summary': 'In eukaryotic cells, protein synthesis typically begins with the binding of eIF4F to the 7-methylguanylate (m7G)cap found on the 5’ end of the majority of mRNAs. Surprisingly, overall translational output remains robust under eIF4F inhibition. The sustained protein synthesis has been largely attributed to cap-independent translation mediated by internal ribosome entry sites (IRES). However, the IRES-driven translation is substrate-specific and largely incompatible with the broad spectrum of eIF4F-resistant translatomes.Here, we report that N6-methyladenosine (m6A)-mediated translation prevails on capped mRNAs and is resistant to eIF4F inactivation.Depletion of the methyltransferase METTL3 selectively inhibits translation of mRNAs bearing 5’UTR methylation, but not mRNAs with 5’ terminal oligopyrimidine (TOP) elements. Mechanistically, we identify ABCF1 as a critical mediator of m6A-promoted translation under both stress and physiological conditions. Supporting the role of ABCF1 in m6A-mediated cap-independent translation, ABCF1-sensitive transcripts largely overlap with METTL3-responsible mRNA targets. By illustrating the scope and the mechanism of translation initiation that is neither cap- nor IRES-dependent, these findingsreshape our current perceptions of cellular translational pathways', 'Series_overall_design': 'Ribo-seq and RNA-seq in Scramble, ABCF1 and METTL3 knockdown.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Shu-Bing,,Qian', 'Series_sample_id': 'GSM2717371', 'Series_contact_name': 'Yuanhui,,Mao', 'Series_contact_email': 'maoyuanhui123@gmail.com', 'Series_contact_institute': 'Zhejiang University', 'Series_contact_address': '1369 West Wenyi Road', 'Series_contact_city': 'Hangzhou', 'Series_contact_state': 'Zhejiang', 'Series_contact_zip/postal_code': '310000', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE101nnn/GSE101865/suppl/GSE101865_RAW.tar', 'Series_platform_id': 'GPL9185', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP113540'}\n",
      "174 PRJNA396456 GSE102040 {'Series_title': 'Base-resolution mapping reveals distinct classes of N1-methyladenosine methylome in nuclear- and mitochondrial-encoded transcripts', 'Series_geo_accession': 'GSE102040', 'Series_status': 'Public on Oct 18 2017', 'Series_submission_date': 'Jul 30 2017', 'Series_last_update_date': 'Jul 25 2021', 'Series_pubmed_id': '29107537', 'Series_summary': 'Gene expression can be post-transcriptionally regulated via dynamic and reversible RNA modifications. N1-methyladenosine (m1A) is a recently identified mRNA modification; however, little is known about its precise location, regulation and function. Here, we develop a single-nucleotide resolution m1A profiling method, based on m1A-induced misincorporation during reverse transcription, and report distinct classes of m1A methylome in the human transcriptome. m1A in the 5’-untranslated region, particularly those located exactly at the first nucleotide of mRNA transcripts, associates with increased translation efficiency. A different subset of m1A sites exhibit a GUUCRA tRNA-like motif, are evenly distributed in the transcriptome and are dependent on the methyltransferase complex TRMT6/61A. Additionally, we show for the first time that m1A is prevalent in the mitochondrial-encoded transcripts. Manipulation of m1A level via TRMT61B, a mitochondria-localizing m1A methyltransferase, demonstrates that m1A in mitochondrial mRNA interferes with translation. Collectively, our approaches reveal distinct classes of m1A methylome and provide a resource for functional studies of m1A-mediated epitranscriptomic regulation.', 'Series_overall_design': \"Please note that these data were produced from the X-ten sequencing platform, and the output raw data were pair-end. However, due to the particularity of our library construction (a random barcode of 10 nt were included in our 3' end cDNA adapter, which cannot be precisely located in Read 1), data of Read 1 were discarded and only the data of Read 2 were used in our analysis and, therefore, submitted as single-end files.\", 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'Chengqi,,Yi', 'Series_sample_id': 'GSM2815744', 'Series_contact_name': 'Xushen,,Xiong', 'Series_contact_email': 'xiongxs@zju.edu.cn', 'Series_contact_department': 'Medical Center', 'Series_contact_institute': 'Zhejiang University', 'Series_contact_address': 'Wenyi Road West', 'Series_contact_city': 'Hangzhou', 'Series_contact_state': 'Zhejiang', 'Series_contact_zip/postal_code': '311121', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE102nnn/GSE102040/suppl/GSE102040_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP114321'}\n",
      "175 PRJNA396698 GSE102113 {'Series_title': 'Acetylation of cytidine in messenger RNA', 'Series_geo_accession': 'GSE102113', 'Series_status': 'Public on Nov 07 2018', 'Series_submission_date': 'Aug 01 2017', 'Series_last_update_date': 'Jun 10 2022', 'Series_pubmed_id': '30449621', 'Series_summary': 'Generation of the “epitranscriptome” through post-transcriptional ribonucleoside modification embeds a layer of regulatory complexity into RNA structure and function. Here we describe N4-acetylcytidine (ac4C) as a novel mRNA modification that is catalyzed by the acetyltransferase NAT10. Transcriptome-wide mapping of ac4C revealed discretely acetylated regions that were distributed across target mRNAs with the majority of peaks occurring within coding regions. Depletion of ac4C through NAT10 ablation revealed a relationship to gene expression wherein loss of ac4C was globally associated with transcript downregulation. The presence of ac4C within coding sequences was associated with elevated ribosome density and enhanced translation, as assessed in vivo and in vitro. In addition to expanding the repertoire of mRNA modifications to include an acetylated residue, these findings highlight a role for ac4C in the control of mRNA metabolism at the level of translation.', 'Series_overall_design': 'Detection of acetylated cytide in RNA with immunoprecipitation (RIP), along with RNA-Seq, Ribo-Seq, and BRIC-Seq to analyze effects on RNA processing and stability.', 'Series_type': 'Other', 'Series_contributor': 'Shalini,,Oberdoerffer', 'Series_sample_id': 'GSM3415519', 'Series_contact_name': 'David,,Sturgill', 'Series_contact_email': 'davidsturgill@mail.nih.gov', 'Series_contact_phone': '240-760-6725', 'Series_contact_laboratory': 'LRBGE', 'Series_contact_department': 'NCI', 'Series_contact_institute': 'NIH', 'Series_contact_address': 'Building 41, Rm B622', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'MARYLAND', 'Series_contact_zip/postal_code': '20892', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE102nnn/GSE102113/suppl/GSE102113_processed_data.tar.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP114504'}\n",
      "176 PRJNA397035 GSE102216 {'Series_title': 'Ribosome profiling analysis of heat effect on rice', 'Series_geo_accession': 'GSE102216', 'Series_status': 'Public on Nov 12 2018', 'Series_submission_date': 'Aug 03 2017', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '30413617', 'Series_summary': 'To impose the heat treatment similar to the Structure-seq experiment, we applied Ribo-seq to 14-day-old rice (Oryza sativa) shoot tissue to investigate the effect of heat stress (42°C) as compared to control condition (22°C).', 'Series_overall_design': 'Two-week-old rice plants in pots were inverted and the shoots were immersed in a water bath at 22°C or 42°C for 10 min treatment, and ten shoots were harvested for each individual sample. Two biological replicates were obtained for each treatment and time-point. Ribosome profiling libraries were then made using these four sets of materials to determine the translational efficiency changes under different temperatures.', 'Series_type': 'Other', 'Series_contributor': 'Sarah,M,Assmann', 'Series_sample_id': 'GSM2730478', 'Series_contact_name': 'Zhao,,SU', 'Series_contact_laboratory': 'Sarah Assmann', 'Series_contact_department': 'Biology', 'Series_contact_institute': 'The Penn State University', 'Series_contact_address': '354 N. Frear', 'Series_contact_city': 'University Park', 'Series_contact_state': 'PA', 'Series_contact_zip/postal_code': '16802', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE102nnn/GSE102216/suppl/GSE102216_all_info.txt.gz', 'Series_platform_id': 'GPL19290', 'Series_platform_taxid': '4530', 'Series_sample_taxid': '4530', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP114761'}\n",
      "177 PRJNA397543 GSE102354 {'Series_title': 'Active ribosome profiling  from mouse brains with RiboLace', 'Series_geo_accession': 'GSE102354', 'Series_status': 'Public on Oct 09 2018', 'Series_submission_date': 'Aug 08 2017', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '30355487', 'Series_summary': 'Keywords: ribosome profiling, translation, RiboLace, Ribo-Seq, active translation', 'Series_overall_design': 'We performed active ribosome profiling with RiboLace in brain extracts from a mouse (Smn+/-; SMN2tg/0). Experiments were performed in biological duplicate.', 'Series_type': 'Other', 'Series_contributor': 'Gabriella,,Viero', 'Series_sample_id': 'GSM2735373', 'Series_contact_name': 'Toma,,Tebaldi', 'Series_contact_email': 't.tebaldi@unitn.it', 'Series_contact_department': 'Centre for Integrative Biology', 'Series_contact_institute': 'University of Trento', 'Series_contact_address': 'via Sommarive n. 9', 'Series_contact_city': 'Povo', 'Series_contact_state': 'Trento', 'Series_contact_zip/postal_code': '38123', 'Series_contact_country': 'Italy', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE102nnn/GSE102354/suppl/GSE102354_RiboLace_rpf_table.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP115013'}\n",
      "178 PRJNA398285 GSE102659 {'Series_title': 'N6-Methyladenosine Guides mRNA Alternative Translation during Integrated Stress Response', 'Series_geo_accession': 'GSE102659', 'Series_status': 'Public on Aug 15 2017', 'Series_submission_date': 'Aug 14 2017', 'Series_last_update_date': 'Jul 25 2021', 'Series_pubmed_id': '29429926', 'Series_summary': \"The integrated stress response (ISR) facilitates cellular adaptation to a variety of stress conditions via phosphorylation of the common target eIF2α. During ISR, the translation of certain stress-related mRNAs is upregulated in spite of global suppression of protein synthesis.The selective translation often relieson alternative mechanisms, such as leaky scanning or reinitiation, but the underlying mechanism remains incompletely understood. Here we report that,in response to amino acid starvation, the reinitiation of ATF4 is not only governed by eIF2α-controlled ternary complex availability, but is also subjected to regulation by mRNA methylation in the form of N6-methyladenosine (m6A). We demonstrate that m6A in the 5' untranslated region (5’ UTR) controls ribosome scanning and subsequent start codonselection. Global profiling of initiating ribosomes reveals widespread alternative translation events influenced by mRNA methylation. Consistently, Fto-transgenic mice manifest enhanced ATF4 expression, highlighting the critical role of 5’ UTR methylation in translational regulation of ISR at cellular and organismal levels.\", 'Series_overall_design': 'Ribo-seq, QTI-seq and m6A-seq in Scramble, ALKBH5 knockdown and FTO overexpression', 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'Shu-Bing,,Qian', 'Series_sample_id': 'GSM2742572', 'Series_contact_name': 'Yuanhui,,Mao', 'Series_contact_email': 'maoyuanhui123@gmail.com', 'Series_contact_institute': 'Zhejiang University', 'Series_contact_address': '1369 West Wenyi Road', 'Series_contact_city': 'Hangzhou', 'Series_contact_state': 'Zhejiang', 'Series_contact_zip/postal_code': '310000', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE102nnn/GSE102659/suppl/GSE102659_RAW.tar', 'Series_platform_id': 'GPL9185', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP115436'}\n",
      "179 PRJNA398585 GSE102720 {'Series_title': 'Ternary protein•drug•RNA interface explains RocA sequence selectivity and Aglaia resistance', 'Series_geo_accession': 'GSE102720', 'Series_status': 'Public on Dec 30 2018', 'Series_submission_date': 'Aug 16 2017', 'Series_last_update_date': 'Jul 25 2021', 'Series_pubmed_id': '30595437', 'Series_summary': 'A novel class of translation inhibitors isolated from Aglaia plants, exemplified by Rocaglamide A (RocA), shows promise as an anti-cancer therapy. RocA converts its molecular target, translation initiation factor 4A (eIF4A), a DEAD-box protein, into a purine-selective RNA-binding protein that represses protein synthesis from a subset of mRNAs. This unusual mechanism of action raises the question of how the drug induces selectivity for polypurine sequences. Furthermore, as eIF4A is found in all eukaryotes, it is unclear how Aglaia resists the effects of RocA. Here, we assembled the Aglaia transcriptome de novo and identified highly specific mutations in eIF4A that enable it to evade RocA-dependent translation inhibition. Furthermore, we determined the crystal structure of the human eIF4A1•ATP analog•RocA•polypurine RNA complex. RocA binds residues mutated in Aglaia and intercalates between consecutive purines at a sharp RNA bend, revealing the structural basis of polypurine-selective translational inhibition by RocA in human and resistance in Aglaia.', 'Series_overall_design': 'Ribosome,profiling', 'Series_type': 'Other', 'Series_contributor': 'Nicholas,T,Ingolia', 'Series_sample_id': 'GSM2743738', 'Series_contact_name': 'Shintaro,,Iwasaki', 'Series_contact_email': 'shintaro.iwasaki@riken.jp', 'Series_contact_department': 'Cluster for Pioneering Research', 'Series_contact_institute': 'RIKEN', 'Series_contact_address': 'S205 Bioscience Bldg. 2-1 Hirosawa', 'Series_contact_city': 'Wako', 'Series_contact_state': 'Saitama', 'Series_contact_zip/postal_code': '351-0198', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE102nnn/GSE102720/suppl/GSE102720_RAW.tar', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP115648'}\n",
      "180 PRJNA399079 GSE102837 {'Series_title': 'Prediction of ribosomal elongation rate from ribosome profiling data', 'Series_geo_accession': 'GSE102837', 'Series_status': 'Public on Aug 18 2018', 'Series_submission_date': 'Aug 18 2017', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '29997263', 'Series_summary': 'We leverage a model statistical framework coupled with high resoluation ribosome profiling data to produce robust estimates of per-codon elongation rates in yeast.', 'Series_overall_design': 'Codon-level Ribosome Profiling Analysis', 'Series_type': 'Other', 'Series_contributor': 'Jeff,,Coller', 'Series_sample_id': 'GSM2747049', 'Series_contact_name': 'Jeff,,Coller', 'Series_contact_email': 'jmc71@case.edu', 'Series_contact_phone': '216-368-0299', 'Series_contact_laboratory': 'Coller Lab', 'Series_contact_department': 'Center for RNA Science and Therapeutics', 'Series_contact_institute': 'Case Western Reserve University School of Medicine', 'Series_contact_address': '10900 Euclid Ave., Wood Bldg.  W113', 'Series_contact_city': 'Cleveland', 'Series_contact_state': 'Ohio', 'Series_contact_zip/postal_code': '44106', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE102nnn/GSE102837/suppl/GSE102837_codon_read_counts.tsv.gz', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP115846'}\n",
      "181 PRJNA399166 GSE102890 {'Series_title': 'Multiple RNA structures affect translation initiation and UGA redefinition efficiency during synthesis of selenoprotein p', 'Series_geo_accession': 'GSE102890', 'Series_status': 'Public on Aug 22 2017', 'Series_submission_date': 'Aug 21 2017', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '29069514', 'Series_summary': 'Mice carrying genomic deletions of SECIS1 or SECIS2 in the Selenop gene were euthanized and liver tissue was harvested and processed for RNA-Seq and ribosome profiling. From these studies, we determined the affect of SECIS1 and SECIS2 on UGA redifinition across the Selenop gene.', 'Series_overall_design': \"Gene expression analysis of liver tissue from mice carrying genomic deletions of the 3' UTR SECIS1 or SECIS2 elements of Selenop\", 'Series_type': 'Other', 'Series_contributor': 'Michael,T,Howard', 'Series_sample_id': 'GSM2747861', 'Series_contact_name': 'Michael,T,Howard', 'Series_contact_email': 'mhoward@genetics.utah.edu', 'Series_contact_phone': '801-585-1927', 'Series_contact_laboratory': 'Howard', 'Series_contact_department': 'Human Genetics', 'Series_contact_institute': 'University of Utah', 'Series_contact_address': '15 N 2030 E', 'Series_contact_city': 'Salt Lake City', 'Series_contact_state': 'UT', 'Series_contact_zip/postal_code': '84112', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE102nnn/GSE102890/suppl/GSE102890_Selenop_AsiteCt.xlsx', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP115915'}\n",
      "182 PRJNA401857 GSE103421 {'Series_title': 'A stress response that monitors and regulates mRNA structure is central to cold-shock adaptation', 'Series_geo_accession': 'GSE103421', 'Series_status': 'Public on Apr 21 2018', 'Series_submission_date': 'Sep 04 2017', 'Series_last_update_date': 'Jul 25 2021', 'Series_pubmed_id': '29628307', 'Series_summary': 'Temperature influences the structural and functional properties of cellular components, necessitating stress responses to restore homeostasis following temperature shift. The heat shock circuitry is well understood, but that controlling the E. coli cold-shock adaptation program is not.  We found that during the growth arrest phase (acclimation) that follows shift to low temperature, protein synthesis increases and ORF-wide mRNA secondary structure decreases. We identified two components of an mRNA surveillance system that enable recovery of translation during acclimation: RNase R assures appropriate mRNA degradation and the Csps dynamically adjust mRNA secondary structure to globally modulate protein expression level. Csps also remodel their own mRNAs to tune their expression to cellular need. Thus, appropriate translation is dynamically adjusted with Csp control. The universality of Csps in bacteria suggests broad utilization of this control mechanism.', 'Series_overall_design': 'Please note that the samples with _1 and _2 (in the sample titles) represent biological replicates of the same experiments.', 'Series_type': 'Other', 'Series_contributor': 'Carol,A,Gross', 'Series_sample_id': 'GSM2946871', 'Series_contact_name': 'Yan,,Zhang', 'Series_contact_email': 'yan.jenny.zhang@gmail.com', 'Series_contact_laboratory': 'Carol Gross', 'Series_contact_department': 'Microbiology and Immunology', 'Series_contact_institute': 'UC San Francisco', 'Series_contact_address': '600 16th Street', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE103nnn/GSE103421/suppl/GSE103421_RAW.tar', 'Series_platform_id': 'GPL17439', 'Series_platform_taxid': '511145', 'Series_sample_taxid': '511145', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP117035'}\n",
      "183 PRJNA403993 GSE103667 {'Series_title': 'Transcriptome-wide profiling of translation efficiency using mRNA-seq and Ribo-seq in ER stress-induced NIH3T3 cells', 'Series_geo_accession': 'GSE103667', 'Series_status': 'Public on Jan 01 2018', 'Series_submission_date': 'Sep 10 2017', 'Series_last_update_date': 'Jul 25 2021', 'Series_pubmed_id': '29576526', 'Series_summary': 'We report systematical profiling of translation efficiency in stress conditions of NIH3T3 cells.', 'Series_overall_design': 'In this study, we pharmacologically induced ER stress by treatment of thapsigargin (THAP). Ribo-seq and RNA-seq were parallelly conducted in whole cells with DMSO- or THAP-treated NIH3T3 cells in duplicate.', 'Series_type': 'Other', 'Series_contributor': 'Hojoong,,Kwak', 'Series_sample_id': 'GSM2778765', 'Series_contact_name': 'Yumi,,Woo', 'Series_contact_email': 'yw768@cornell.edu', 'Series_contact_laboratory': 'Kwak Lab', 'Series_contact_department': 'Molecular Biology and Genetics', 'Series_contact_institute': 'Cornell University', 'Series_contact_address': '526 Campus Road', 'Series_contact_city': 'Ithaca', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '14853', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE103nnn/GSE103667/suppl/GSE103667_TE.norm.txt.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP117211'}\n",
      "184 PRJNA406823 GSE103719 {'Series_title': 'Transcriptome-wide profiling of poly(A)-tail length, translation efficiency and mRNA stability using TED-seq, mRNA-seq, Ribo-seq and PRO-seq in ER stress conditions', 'Series_geo_accession': 'GSE103719', 'Series_status': 'Public on Jan 01 2018', 'Series_submission_date': 'Sep 11 2017', 'Series_last_update_date': 'Jul 25 2021', 'Series_pubmed_id': '30257221', 'Series_summary': \"We report systematical profiling of translation efficiency and mRNA stability dependent on the dynamics of poly(A)-tail length in stress conditions of human cells. In this study, we developed a new feasible method measuring poly(A)-tail length called TED-seq and applied it to investigate the change of mRNA's poly(A)-tail lengths in ER stress pharmacologically induced by thapsigargin (THAP). Combined with other global RNA analyses such as RNA-seq, Ribo-seq and PRO-seq, we observed that ER stress induced lenthening poly(A)-tail length, in particular of ER-stress-regulators, upon ER stress. More specifically, these mRNAs are translationally de-repressed and more stabilized based on increase in poly(A)-tail length. We also identified that insoluble fractions which include stress-induced RNA-granules have overall shorter length of poly(A) tail. Taken together, our data suggest that poly(A)-tail lengths are dynamically regulated and influence both translation efficiency and mRNA stability in ER stress.\", 'Series_overall_design': '2) RNA-seq (4 replicates), Ribo-seq (4 replicates) and PRO-seq (2 replicates) in whole cells with DMSO- or THAP-treated HEK293 cells', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Hojoong,,Kwak', 'Series_sample_id': 'GSM3082793', 'Series_contact_name': 'Yumi,,Woo', 'Series_contact_email': 'yw768@cornell.edu', 'Series_contact_laboratory': 'Kwak Lab', 'Series_contact_department': 'Molecular Biology and Genetics', 'Series_contact_institute': 'Cornell University', 'Series_contact_address': '526 Campus Road', 'Series_contact_city': 'Ithaca', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '14853', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE103nnn/GSE103719/suppl/GSE103719_RAW.tar', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP117282'}\n",
      "185 PRJNA407986 GSE104028 {'Series_title': 'Analysis of the translational landscape of Arabidopsis mitochondria', 'Series_geo_accession': 'GSE104028', 'Series_status': 'Public on May 31 2018', 'Series_submission_date': 'Sep 19 2017', 'Series_last_update_date': 'Jan 07 2020', 'Series_pubmed_id': '29873797', 'Series_summary': 'Messenger RNA translation is a complex process that is still poorly understood in eukaryotic organelles like mitochondria. Growing evidence indicate though that mitochondrial translation differs from its bacterial counterpart in many key aspects. In this analysis, we used the ribosome profiling technology to generate a genome-wide snapshot view of mitochondrial translation in Arabidopsis. We show that, unlike in humans, most Arabidopsis mitochondrial ribosomes footprints measure 27 and 28 bases. We also reveal that respiratory subunits encoding mRNAs show much higher ribosome association than other mitochondrial mRNAs, implying that they are translated to higher levels. Homogenous ribosome densities were generally detected within each respiratory complex except for complex V where higher ribosome coverage corroborate with higher needs in specific subunits. In complex I respiratory mutants, a slight reorganization of mitochondrial mRNAs ribosome association was detected involving an increase in ribosome densities on certain ribosomal protein encoding transcripts and a reduction in the translation of a few complex V mRNAs. Altogether, our observations reveal that plant mitochondrial translation is a dynamic process and that translational control is important for gene expression in plant mitochondria. This study paves the way for future advances in the understanding of translation in higher plant mitochondria.', 'Series_overall_design': 'In this study, 2 Arabidopsis mutants are compared to the wild type using duplicates for the Ribosome Profiling experiment. RNASEQ: was conducted to observe the editing rates of Col0 transcripts. RIBOSEQ: was conducted to observe the ribosome occupancy on each transcript of wild type and mutants.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Hakim,,Mireau', 'Series_sample_id': 'GSM2787955', 'Series_contact_name': 'Pierre,,Bertin', 'Series_contact_email': 'bertinp71@gmail.com', 'Series_contact_institute': 'Institut de Biologie Intégrative de la Cellule', 'Series_contact_address': 'Batiment 400, Université Paris Sud', 'Series_contact_city': 'ORSAY', 'Series_contact_zip/postal_code': '91400', 'Series_contact_country': 'France', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE104nnn/GSE104028/suppl/GSE104028_sites_details.txt.gz', 'Series_platform_id': 'GPL17639', 'Series_platform_taxid': '3702', 'Series_sample_taxid': '3702', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP118154'}\n",
      "186 PRJNA412874 GSE104506 {'Series_title': 'Ssd1 and Gcn2 suppress global translational efficiency in replicatively aged yeast, while their activation in young cells extends lifespan', 'Series_geo_accession': 'GSE104506', 'Series_status': 'Public on Aug 10 2018', 'Series_submission_date': 'Oct 02 2017', 'Series_last_update_date': 'Jul 25 2021', 'Series_pubmed_id': '30117416', 'Series_summary': 'Using ribosome profiling, we find globally reduced translation efficiency during mitotic / replicative aging in budding yeast. Two mechanisms contribute to this: Firstly, the mRNA binding protein Ssd1 is induced during aging, sequestering mRNAs to P-bodies and stress granules that are abundant in old cells. Indeed, overexpression of Ssd1 reduced protein synthesis in young cells and extended lifespan, while loss of Ssd1 reduced the translational deficit of old cells and shortened lifespan. Secondly, the Gcn2 kinase is activated in old cells, phosphorylating and inactivating the translational initiation factor eIF2α. Accordingly, deletion of GCN2 reduced the translational defect of old cells. Furthermore, overexpressing an uncharged tRNA to fully activate Gcn2, or overexpression of its downstream mediator, Gcn4, extended replicative lifespan in a manner that was mostly dependent on autophagy without inhibiting the TOR pathway. As such, Ssd1 induction, activation of the integrated stress response or autophagy are favorable TOR-independent therapeutic targets for lifespan extension.', 'Series_overall_design': 'Examination of transcription and translational efficiency in replicative lifespan analysis with RNA-seq and Ribo-seq with 3 replicates at 3 different timepoints', 'Series_type': 'Other', 'Series_contributor': 'Jessica,K,Tyler', 'Series_sample_id': 'GSM2802585', 'Series_contact_name': 'Bo,,Xia', 'Series_contact_email': 'bxia@houstonmethodist.org', 'Series_contact_phone': '7134419140', 'Series_contact_laboratory': 'Kaifu Chen Lab', 'Series_contact_institute': 'Houston Methodist', 'Series_contact_address': '6670 Bertner Ave, R10-414D', 'Series_contact_city': 'Houston', 'Series_contact_state': 'TX', 'Series_contact_zip/postal_code': '77030', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE104nnn/GSE104506/suppl/GSE104506_all_sample_normalized.xls.gz', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP119264'}\n",
      "187 PRJNA414959 GSE105147 {'Series_title': 'Translation control of the immune checkpoint in cancer and its therapeutic targeting', 'Series_geo_accession': 'GSE105147', 'Series_status': 'Public on Jan 07 2019', 'Series_submission_date': 'Oct 19 2017', 'Series_last_update_date': 'Jul 25 2021', 'Series_pubmed_id': '30643286', 'Series_summary': 'Cancer cells develop mechanisms to escape immunosurveillance, among which modulating the expression of immune suppressive messenger RNAs is most well-documented. However, how this is molecularly achieved remains largely unresolved. Here, we develop an in vivo mouse model of liver cancer to study oncogene cooperation in immunosurveillance. We show that MYC overexpression (MYCTg) synergizes with KRASG12D to induce an aggressive liver tumor leading to metastasis formation and reduced mouse survival compared with KRASG12D alone. Genome-wide ribosomal footprinting of MYCTg;KRASG12 tumors compared with KRASG12D revealed potential alterations in translation of mRNAs, including programmed-death-ligand 1 (PD-L1). Further analysis revealed that PD-L1 translation is repressed in KRASG12D tumors by functional, non-canonical upstream open reading frames in its 5′ untranslated region, which is bypassed in MYCTg;KRASG12D tumors to evade immune attack. We show that this mechanism of PD-L1 translational upregulation was effectively targeted by a potent, clinical compound that inhibits eIF4E phosphorylation, eFT508, which reverses the aggressive and metastatic characteristics of MYCTg;KRASG12D tumors. Together, these studies reveal how immune-checkpoint proteins are manipulated by distinct oncogenes at the level of mRNA translation, which can be exploited for new immunotherapies.', 'Series_overall_design': 'To identify the transcripts specifically translated in primary liver tumors in MYC; KRAS(G12D) and KRAS(G12D) mice, we first carefully micro-dissected primary liver tumor nodules in MYC; KRAS(G12D) and KRAS(G12D) mice as well as wildtype (WT) controls. We sequenced total mRNAs (2 replicates each) and ribosome-protected mRNA fragments (RPFs) using ribosome profiling (2 replicates each) to generate genome-wide transcriptional and translational landscapes, respectively.', 'Series_type': 'Other', 'Series_contributor': 'Davide,,Ruggero', 'Series_sample_id': 'GSM2823266', 'Series_contact_name': 'Zhen,,Shi', 'Series_contact_institute': 'Stanford', 'Series_contact_address': '279 Campus Drive', 'Series_contact_city': 'Stanford', 'Series_contact_state': 'California', 'Series_contact_zip/postal_code': '94305', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE105nnn/GSE105147/suppl/GSE105147_RAW.tar', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP120478'}\n",
      "188 PRJNA415033 GSE105172 {'Series_title': 'DHX36 resolves G-rich structures in mRNA untranslated region to allow their translation [Ribo-Seq]', 'Series_geo_accession': 'GSE105172', 'Series_status': 'Public on Jul 01 2019', 'Series_submission_date': 'Oct 19 2017', 'Series_last_update_date': 'Sep 30 2019', 'Series_pubmed_id': '31160600', 'Series_summary': 'DHX36 is a ATP-dependent, 3´-5´ RNA helicase of the DEAH family. Previous publications reported this helicase to associate with AU-rich elements and to specifically unwind G-quadruplex structures. Here, we performed Ribosome footprintin in triplicats of HEK293 wildtype cells and DHX36-KO-HEK293 cells to identify changes in ribosome occupancy on mRNAs upon presence or absence of this helicase. After sequencing and mapping to the human genome at nucleotide-resolution level we analysed the coverage of ribosomes on target mRNAs, identified in this study (see PAR-CLIP). Our analyses show that only minor changes upon DHX36 knockout can be observed.', 'Series_overall_design': 'Examination of ribosome footprints in HEK293 wildtype cells and DHX36-KO-HEK293 cells', 'Series_type': 'Other', 'Series_contributor': 'Katrin,,Paeschke', 'Series_sample_id': 'GSM2824219', 'Series_contact_name': 'Markus,,Sauer', 'Series_contact_email': 'markus.sauer@stud-mail.uni-wuerzburg.de', 'Series_contact_institute': 'University of Würzburg', 'Series_contact_address': 'Am Hubland, Biozentrum - Lehrstuhl für Biochemie', 'Series_contact_city': 'Würzburg', 'Series_contact_zip/postal_code': '97074', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE105nnn/GSE105172/suppl/GSE105172_Riboseq_gene_exp.diff.gz', 'Series_platform_id': 'GPL21290', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP120504'}\n",
      "189 PRJNA415390 GSE105782 {'Series_title': 'N6-methyladenosine mRNA marking promotes selective translation of regulons required for human erythropoiesis[Ribosome Profiling]', 'Series_geo_accession': 'GSE105782', 'Series_status': 'Public on Aug 12 2019', 'Series_submission_date': 'Oct 23 2017', 'Series_last_update_date': 'Oct 21 2019', 'Series_pubmed_id': '31601799', 'Series_summary': 'Many of the regulatory features governing erythrocyte specification, maturation, and associated disorders remain enigmatic. To identify new regulators of erythropoiesis, we performed a functional genomic screen for genes affecting expression of the erythroid marker CD235a/GYPA. Among validating hits were genes coding for the N6-methyladenosine (m6A) mRNA methyltransferase (MTase) complex, including, METTL14, METTL3, and WTAP.  We found that m6A MTase activity promotes erythroid gene expression programs through selective translation of ~300 m6A marked mRNAs, including those coding for SETD histone methyltransferases, ribosomal components, and polyA RNA binding proteins, and novel genes. Remarkably, loss of m6A marks resulted in dramatic loss of H3K4me3 marks across key erythroid-specific KLF1 transcriptional targets (e.g., Heme biosynthesis genes). Further, each m6A MTase subunit and a subset of their mRNAs targets are required for human erythroid specification in primary bone-marrow derived progenitors. Thus, m6A mRNA marks promote the translation of a network of genes required for human erythropoiesis.', 'Series_overall_design': 'Contains three replicates of ribosome protected RNA samples and total input RNA from non-targeting control sgRNA HEL cells and WTAP sgRNA KO HEL cells', 'Series_type': 'Other', 'Series_contributor': 'Patrick,J,Paddison', 'Series_sample_id': 'GSM2828176', 'Series_contact_name': 'Sonali,,Arora', 'Series_contact_email': 'sarora@fredhutch.org', 'Series_contact_institute': 'FHCRC', 'Series_contact_address': '1100 Fairview Ave N,', 'Series_contact_city': 'Seattle', 'Series_contact_state': 'WA', 'Series_contact_zip/postal_code': '98109', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE105nnn/GSE105782/suppl/GSE105782_RAW.tar', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP120964'}\n",
      "190 PRJNA415722 GSE106140 {'Series_title': 'Dynamic remodeling of translation programs drives hepatocyte proliferation during liver regeneration', 'Series_geo_accession': 'GSE106140', 'Series_status': 'Public on Sep 17 2018', 'Series_submission_date': 'Oct 25 2017', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '30250226', 'Series_summary': 'During liver regeneration, most new hepatocytes arise from pre-existing ones; yet, the underlying mechanisms that drive these cells from quiescence to proliferation remain poorly defined. By using high-resolution transcriptome profiling of polysome fractions from purified hepatocytes isolated from quiescent and toxin-injured adult mouse livers,we uncover the mRNA transcripts regulated during liver regeneration. The translational remodeling modulates protein levels of a set of splicing factors including Epithelial splicing regulatory protein 2 (ESRP2), which activates an early postnatal splicing program in regenerating hepatocytes as well as metabolic genes.', 'Series_overall_design': 'Hepatocytes were isolated in presence of Cycloheximide (CHX) from normal fed mice and DDC fed (4 week) mice. Polysomal fractions were separated over sucrose gradient and RNA was isolated from the three fractions Monosome (1 ribosome), Light polysome (2-4 ribosomes) and Heavy polysomes (>5).  Isolated RNA was subject to polyA selection and sequenced on an Illumina platform.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Waqar,,Arif', 'Series_sample_id': 'GSM2830377', 'Series_contact_name': 'Sushant,,Bangru', 'Series_contact_email': 'bangru2@illinois.edu, sushantbangru@gmail.com, sb725@duke.edu', 'Series_contact_phone': '+1-217-979-9320', 'Series_contact_laboratory': 'Dr. Auinash Kalsotra', 'Series_contact_department': 'Biochemistry', 'Series_contact_institute': 'University of Illinois, Urbana-Champaign', 'Series_contact_address': '600 S Mathews Avenue', 'Series_contact_city': 'Urbana', 'Series_contact_state': 'Illinois', 'Series_contact_zip/postal_code': '61801', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE106nnn/GSE106140/suppl/GSE106140_Transcript_cuffnorm_counts_polyseq.csv.gz', 'Series_platform_id': 'GPL21103', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP121411'}\n",
      "191 PRJNA416661 GSE106386 {'Series_title': 'Amyotrophic Lateral Sclerosis associated FUS mutation shortens mitochondria and induces neurotoxicity', 'Series_geo_accession': 'GSE106386', 'Series_status': 'Public on Nov 01 2017', 'Series_submission_date': 'Oct 31 2017', 'Series_last_update_date': 'Aug 20 2019', 'Series_pubmed_id': '30349096', 'Series_summary': 'We employ CLIP-Seq, RNA-Seq and Ribo-Seq in cultured neurons expressing R495X or wild-type FUS to identify effects of the mutation on the FUS targetome and on the neuronal transcriptome at the expression and translation level', 'Series_overall_design': 'We set up an inducible expression system based on lentiviral transduction of FLAG-tagged human R495X or wild-type FUS in neurons differentiated from mouse Embryonic Stem cells. we performed CLIP-Seq , RNA-Seq and Ribo-Seq in neurons expressing WT or R495X.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Tadashi,,Nakaya', 'Series_sample_id': 'GSM2837005', 'Series_contact_name': 'Manolis,,Maragkakis', 'Series_contact_email': 'emmanouil.maragkakis@nih.gov', 'Series_contact_institute': 'National Institute on Aging', 'Series_contact_address': '251 Bayview Blvd', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21224', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE106nnn/GSE106386/suppl/GSE106386_RAW.tar', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP123232'}\n",
      "192 PRJNA417116 GSE106529 {'Series_title': 'Translatome analysis of the ribosomal protein L10 R98S mutation reveals altered serine metabolism in acute lymphoblastic leukemia [Ribo-seq]', 'Series_geo_accession': 'GSE106529', 'Series_status': 'Public on Apr 10 2019', 'Series_submission_date': 'Nov 03 2017', 'Series_last_update_date': 'Jul 10 2019', 'Series_pubmed_id': '31186416', 'Series_summary': 'Somatic ribosomal protein defects have recently been described in cancer, yet their impact on cellular transcription and translation remain poorly understood. Here we integrated mRNA sequencing, ribosome footprinting, polysomal RNA seq and quantitative mass spectrometry datasets obtained from an isogenic mouse lymphoid cell model in order to study the T-cell acute lymphoblastic leukemia (T-ALL) associated R98S mutation in ribosomal protein L10 (RPL10 R98S). RPL10 R98S induced changes in protein levels were to a much larger extent caused by transcriptional then translational changes and RPL10 R98S cells showed a gene signature corresponding to deregulation of hematopoietic transcription factors. Phosphoserine phosphatase (PSPH), a key enzyme in serine biosynthesis, displayed elevated transcription and translation and was one of the proteins showing the strongest upregulation in RPL10 R98S cells. Increased Psph protein levels were confirmed in RPL10 R98S engineered JURKAT cells and in hematopoietic cell cultures derived from Rpl10 R98S knock-in mice. Moreover, elevated serine and glycine biosynthesis in RPL10 R98S cells was supported by metabolic flux analyses. Analysis of PSPH expression levels in T-ALL patient samples revealed that PSPH upregulation is a generalized phenomenon in this disease, associated with elevated circulating serine and glycine levels. Addition of serine and glycine enhanced survival of stromal and myeloid cells, suggesting supportive effects on the hematopoietic niche. Finally, reduction of PSPH expression levels in T-ALL cell lines suppressed their in vitro proliferation and their capacity to expand in T-ALL xenograft models. In conclusion, transcriptome, translatome and proteome analysis of the RPL10 R98S mutation identified RPL10 R98S driven induction of cellular serine biosynthesis. Whereas serine metabolism has been implicated in cancer via PHGDH amplification, this is the first report supporting dependence of ALL cells on the serine biosynthesis enzyme PSPH.', 'Series_overall_design': 'Ribosome footprinting and mRNA-Seq libraries from 3 biological replicates for each condition (RPL10 R98S, RPL10 WT). For ribosome footprinting three technical replicates were generated for each biological replicate. The sequencing files of three technical replicates were merged for downstream analyses', 'Series_type': 'Other', 'Series_contributor': 'Kim,,De Keersmaecker', 'Series_sample_id': 'GSM2840023', 'Series_contact_name': 'Kim,,De Keersmaecker', 'Series_contact_laboratory': 'Disease mechanisms in cancer', 'Series_contact_department': 'Oncology', 'Series_contact_institute': 'KU Leuven', 'Series_contact_address': 'Herestraat 49', 'Series_contact_city': 'Leuven', 'Series_contact_zip/postal_code': '3000', 'Series_contact_country': 'Belgium', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE106nnn/GSE106529/suppl/GSE106529_NormalizedCountsByDESeq2_RiboSeq.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP123624'}\n",
      "193 PRJNA419198 GSE107169 {'Series_title': 'Circular RNA profiling reveals the different distribution/characteristic and possible transport mechanism among the subcellular fractions', 'Series_geo_accession': 'GSE107169', 'Series_status': 'Public on Jan 16 2019', 'Series_submission_date': 'Nov 20 2017', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '30614753', 'Series_summary': 'We systemically investigated the circRNA profiles among the subcellular fractions of HepG2 cell, including nucleus, cytoplasm, mitochondria, ribosome, cytosol and exosome. Our studies reveals the wide distribution of circRNAs among the subcellular fractions except the mitochondria. Further comparative analysis indicate the differential subcellular distributions in expression, length, GC content, classification, alternative circularization and function of the parental genes. Through analyzing the binding motifs distribution of the transport-related RBPs among the subcellular circRNAs, we found that the different subcellular distribution characteristics of circRNAs might be resulted from the RBP-mediated selective transportation. It also implies that the circRNAs may follow the same transportation mechanism as linear RNAs: the RBPs specially recognize, bind to and transport the RNAs with the corresponding binding motifs, regardless of circular or linear RNAs. Our researches not only facilitate better understanding of the life history and molecular behavior of circRNA in cell, but also contribute to the exploration for new biological functions of circRNA.', 'Series_overall_design': 'Examined seven fractions of HepG2 (cell, nucleus, cytoplasm, ribsome, cytosol, exosome, mitochodria) and each has two repeats', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Fei,,Chen', 'Series_sample_id': 'GSM2862020', 'Series_contact_name': 'Fei,,Chen', 'Series_contact_email': 'jiaxm@big.ac.cn', 'Series_contact_institute': 'Beijing Institute of Genomics', 'Series_contact_address': 'NO.1 Beichen West Road, Chaoyang District', 'Series_contact_city': 'Beijing', 'Series_contact_zip/postal_code': '100101', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE107nnn/GSE107169/suppl/GSE107169_circRNA_expression_of_six_fractions.xlsx', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP125321'}\n",
      "194 PRJNA419768 GSE107328 {'Series_title': 'E. coli MazF does not create specialized ribosomes that translate leaderless mRNAs, but instead blocks rRNA maturation and ribosome biogenesis [Ribo-Seq]', 'Series_geo_accession': 'GSE107328', 'Series_status': 'Public on Dec 05 2017', 'Series_submission_date': 'Nov 24 2017', 'Series_last_update_date': 'Sep 05 2021', 'Series_pubmed_id': '29861158', 'Series_summary': 'The purpose of this experiment was to assess translation of mRNA after MazF expression to determine if cleavage events resulted in a loss in ribosome footprints.', 'Series_overall_design': 'A + MazF sample at 5 minutes and 25 minutes after induction and an empty vector sample at 15 minutes of induction', 'Series_type': 'Other', 'Series_contributor': 'Michael,T,Laub', 'Series_sample_id': 'GSM2864869', 'Series_contact_name': 'Peter,,Culviner', 'Series_contact_laboratory': 'Laub', 'Series_contact_department': 'Biology', 'Series_contact_institute': 'MIT', 'Series_contact_address': '31 Ames St', 'Series_contact_city': 'Cambridge', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02142', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE107nnn/GSE107328/suppl/GSE107328_ribosome_footprint_ends.csv.gz', 'Series_platform_id': 'GPL21222', 'Series_platform_taxid': '562', 'Series_sample_taxid': '562', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP125609'}\n",
      "195 PRJNA420695 GSE107588 {'Series_title': 'Ribosome profiling in transgenic and HEK293T cells', 'Series_geo_accession': 'GSE107588', 'Series_status': 'Public on Aug 01 2019', 'Series_submission_date': 'Dec 01 2017', 'Series_last_update_date': 'Aug 01 2019', 'Series_summary': 'To investigate whether natural TAG can be decoded by stably transfected pyltRNA, whole cell lysate of transgenic cells was collected 48h after the addition of NAEK and subjected to ribosome profiling, examining the physical locations of ribosomes along 15662 TAG terminated transcripts. And HEK293T cells were taken as control.', 'Series_overall_design': 'Examine the physical locations of ribosomes along 15662 TAG terminated transcripts in both transgenic and HEK293T cells', 'Series_type': 'Other', 'Series_contributor': 'Yiming,,Wu', 'Series_sample_id': 'GSM2871727', 'Series_contact_name': 'Yiming,,Wu', 'Series_contact_email': '1309600826@qq.com', 'Series_contact_laboratory': 'Demin Zhou', 'Series_contact_department': 'Chemical Bilogy', 'Series_contact_institute': 'Peking University Health Science Center', 'Series_contact_address': 'No 38 Xueyuan Road', 'Series_contact_city': 'Beijing', 'Series_contact_state': 'Beijing', 'Series_contact_zip/postal_code': '100191', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE107nnn/GSE107588/suppl/GSE107588_Ribosome_profiling_of_transgenic_and_HEK293T_cells.xlsx', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP125941'}\n",
      "196 PRJNA421178 GSE107718 {'Series_title': 'The extent of ribosome queuing in budding yeast', 'Series_geo_accession': 'GSE107718', 'Series_status': 'Public on Jan 17 2018', 'Series_submission_date': 'Dec 05 2017', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '29377894', 'Series_summary': \"Ribosome queuing is a fundamental phenomenon suggested to be related to topics such as genome evolution, synthetic biology, gene expression regulation, intracellular biophysics, and more. However, this phenomenon hasn't been quantified yet at a genomic level. Nevertheless, methodologies for studying translation (e.g. ribosome footprints) are usually calibrated to capture only single ribosome protected footprints (mRPFs) and thus limited in their ability to detect ribosome queuing. On the other hand, most of the models in the field assume and analyze a certain level of queuing.     Here we present an experimental-computational approach for studying ribosome queuing based on sequencing of RNA footprints extracted from pairs of ribosomes (dRPFs) using a modified ribosome profiling protocol. We combine our approach with traditional ribosome profiling to generate a detailed profile of ribosome traffic. The data are analyzed using computational models of translation dynamics.    The approach was implemented on the Saccharomyces cerevisiae transcriptome. Our data shows that ribosome queuing is more frequent than previously thought: the measured ratio of ribosomes within dRPFs to mRPFs is 0.2-0.35, suggesting that at least one to five translating ribosomes is in a traffic jam; these queued ribosomes cannot be captured by traditional methods.     We found that specific regions are enriched with queued ribosomes, such as the 5’-end of ORFs, and regions upstream to mRPF peaks, among others. While queuing is related to higher density of ribosomes on the transcript (characteristic of highly translated genes), we report cases where traffic jams are relatively more severe in lowly expressed genes and possibly even selected for. In addition, our analysis demonstrates that higher adaptation of the coding region to the intracellular tRNA levels is associated with lower queuing levels. Our analysis also suggests that the Saccharomyces cerevisiae transcriptome undergoes selection for eliminating traffic jams. Thus, our proposed approach is an essential tool for high resolution analysis of ribosome traffic during mRNA translation and understanding its evolution.\", 'Series_overall_design': 'We present an experimental-computational approach for studying ribosome queuing based on sequencing of RNA footprints extracted from pairs of ribosomes (dRPFs) using a modified ribosome profiling protocol. We combine our approach with traditional ribosome profiling to generate a detailed profile of ribosome traffic. The data are analyzed using computational models of translation dynamics.', 'Series_type': 'Other', 'Series_contributor': 'Tamir,,Tuller', 'Series_sample_id': 'GSM2877031', 'Series_contact_name': 'Tamir,,Tuller', 'Series_contact_email': 'tamirtul@post.tau.ac.il', 'Series_contact_phone': '+972-3-6405836', 'Series_contact_laboratory': 'Tuller Lab', 'Series_contact_department': 'Biomedical Engineering Dept.', 'Series_contact_institute': 'Tel Aviv University', 'Series_contact_address': '30 Levanon st.', 'Series_contact_city': 'Tel Aviv-Yafo', 'Series_contact_zip/postal_code': '6997801', 'Series_contact_country': 'Israel', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE107nnn/GSE107718/suppl/GSE107718_RAW.tar', 'Series_platform_id': 'GPL19756', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP126173'}\n",
      "197 PRJNA423281 GSE108334 {'Series_title': 'eIF1A residues implicated in cancer stabilize translation preinitiation complexes and favor suboptimal initiation sites in yeast', 'Series_geo_accession': 'GSE108334', 'Series_status': 'Public on Dec 21 2017', 'Series_submission_date': 'Dec 20 2017', 'Series_last_update_date': 'Mar 11 2019', 'Series_pubmed_id': '29206102', 'Series_summary': 'The translation pre-initiation complex (PIC) scans the mRNA for an AUG codon in favorable context, and AUG recognition stabilizes a closed PIC conformation. The unstructured N-terminal tail (NTT) of yeast eIF1A deploys five basic residues to contact tRNAi, mRNA, or 18S rRNA exclusively in the closed state. Interestingly, EIF1AX mutations altering the human eIF1A NTT are associated with uveal melanoma (UM). We found that substituting all five basic residues, and seven UM-associated substitutions, in yeast eIF1A suppresses initiation at near-cognate UUG codons and AUGs in poor context. Ribosome profiling of NTT substitution R13P reveals heightened discrimination against unfavorable AUG context genome-wide. Both R13P and K16D substitutions destabilize the closed complex at UUG codons in reconstituted PICs. Thus, electrostatic interactions involving the eIF1A NTT stabilize the closed conformation and promote utilization of suboptimal start codons. We predict UM-associated mutations alter human gene expression by increasing discrimination against poor initiation sites.', 'Series_overall_design': 'We examined the effect of\\xa0tif11-R13P\\xa0on global translational efficiencies (TEs) by ribosome footprint profiling of isogenic WT and\\xa0tif11-R13P\\xa0strains. The study includes 8 samples, comprised of 4 mRNA-Seq samples and 4 ribosome footprint profiling samples, derived from 2 biological replicates of\\xa0tif11?\\xa0mutant strains harboring plasmid-borne\\xa0tif11-R13P\\xa0or the WT\\xa0TIF11\\xa0allele. Additional Ribosome profiling data were used for yeast uORF identification.', 'Series_type': 'Other', 'Series_contributor': 'A,G,Hinnebusch', 'Series_sample_id': 'GSM2895491', 'Series_contact_name': 'Jon,R.,Lorsch', 'Series_contact_email': 'jon.lorsch@nih.gov', 'Series_contact_phone': '301-594-2172', 'Series_contact_laboratory': 'Section on the Mechanism and Regulation of Protein Synthesis', 'Series_contact_department': 'Eunice Kennedy Shriver National Institute of Child Health and Human Development,', 'Series_contact_institute': 'National Institutes of Health', 'Series_contact_address': 'BG 49 RM 2C08, 49 Convent Dr.', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '20892', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE108nnn/GSE108334/suppl/GSE108334_yeast_all_uorfs.bed.gz', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP127274'}\n",
      "198 PRJNA428526 GSE108778 {'Series_title': 'Pervasive, coordinated protein level changes driven by transcript isoform switching during meiosis', 'Series_geo_accession': 'GSE108778', 'Series_status': 'Public on Feb 22 2018', 'Series_submission_date': 'Jan 04 2018', 'Series_last_update_date': 'Mar 11 2019', 'Series_pubmed_id': '29474919', 'Series_summary': \"To better understand the gene regulatory mechanisms that program developmental processes, we carried out simultaneous, genome-wide measurements of mRNA, translation and protein through meiotic differentiation in budding yeast. Surprisingly, we observed that the levels of several hundred mRNAs are anti-correlated with their corresponding protein products. We show that rather than arising from canonical forms of gene regulatory control, the regulation of at least 380 such cases—or over 8% of all measured genes—involves temporally regulated switching between production of a canonical, translatable transcript and a 5' extended isoform that is not efficiently translated into protein. By this pervasive mechanism for the modulation of protein levels through a natural developmental program, a single transcription factor can coordinately activate and repress protein synthesis for distinct sets of genes. The distinction is not based on whether an mRNA is induced or not, but rather based on the type of transcript produced.\", 'Series_overall_design': 'Sequencing of mRNA molecules and ribosome footprints to identify regulated transcript isoform changes that result in apparent translational regulation. In short,two major experiments are included here, one for parallel mRNA-seq and ribosome profiling of cells progressing through meiosis. This was performed in parallel with mass spectrometry to ascertain the relationship between transcript abundance, translation, and protein abundance over time in a developmental process. The other experiment investigates the effects on transcript abundance and translation of induction of a single transcription factor (Ndt80). The results and analysis of both studies are in the publication, \"Pervasive, coordinated protein level changes driven by transcript isoform switching during meiosis\" by Ze Cheng*, George Maxwell Otto*, Emily Nicole Powers, Abdurrahman Keskin, Philipp Mertins, Steve Carr, Marko Jovanovic, and Gloria Ann Brar. In press, Cell.', 'Series_type': 'Other', 'Series_contributor': 'Ze,,Cheng', 'Series_sample_id': 'GSM2913190', 'Series_contact_name': 'Gloria,,Brar', 'Series_contact_email': 'gabrar@berkeley.edu', 'Series_contact_department': 'MCB', 'Series_contact_institute': 'UC Berkeley', 'Series_contact_address': '626 Barker Hall', 'Series_contact_city': 'Berkeley', 'Series_contact_zip/postal_code': '94720', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE108nnn/GSE108778/suppl/GSE108778_timecourse_replicate_2_totRNA.txt.gz', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP128065'}\n",
      "199 PRJNA429734 GSE109122 {'Series_title': 'Transcripts from downstream alternative transcription start sites evade uORF-mediated inhibition of gene expression during blue light exposure in Arabidopsis', 'Series_geo_accession': 'GSE109122', 'Series_status': 'Public on Jul 27 2018', 'Series_submission_date': 'Jan 12 2018', 'Series_last_update_date': 'Oct 03 2018', 'Series_pubmed_id': '29915080', 'Series_summary': 'uORF-mediated translation reguration', 'Series_overall_design': 'Ribosome profiling', 'Series_type': 'Other', 'Series_contributor': 'Minami,,Matsui', 'Series_sample_id': 'GSM2932484', 'Series_contact_name': 'tomoya,,fujita', 'Series_contact_email': 'tomoya.fujita.at.riken@gmail.com', 'Series_contact_laboratory': 'RNA Systems Biochemistry Laboratory', 'Series_contact_department': 'Cluster for Pioneering Research', 'Series_contact_institute': 'RIKEN', 'Series_contact_address': '2-1, Hirosawa', 'Series_contact_city': 'Wako', 'Series_contact_state': 'Saitama', 'Series_contact_zip/postal_code': '3510198', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE109nnn/GSE109122/suppl/GSE109122_RAW.tar', 'Series_platform_id': 'GPL21785', 'Series_platform_taxid': '3702', 'Series_sample_taxid': '3702', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP128983'}\n",
      "200 PRJNA430526 GSE109343 {'Series_title': 'Lso2 is a conserved ribosome-bound protein required for translational recovery in yeast', 'Series_geo_accession': 'GSE109343', 'Series_status': 'Public on Sep 12 2018', 'Series_submission_date': 'Jan 18 2018', 'Series_last_update_date': 'May 20 2019', 'Series_pubmed_id': '30208026', 'Series_summary': 'Ribosome binding proteins function broadly in protein synthesis, gene regulation and cellular homeostasis but the complete complement of functional ribosome-bound proteins remains unknown. Using quantitative mass spectrometry we identified Late-annotated short open reading frame 2 (Lso2) as a ribosome-associated protein that is broadly conserved in eukaryotes. Genome-wide crosslinking and immunoprecipitation of Lso2 and its human ortholog CCDC124 recovered 25S rRNA in a region near the A site that overlaps the GTPase activating center. Consistent with this location, Lso2 also crosslinked to most tRNAs. Ribosome profiling of yeast lacking LSO2 (lso2Δ) revealed global translation defects during recovery from stationary phase with translation of most genes reduced more than 4-fold. Ribosomes accumulated at start codons, were depleted from stop codons, and showed codon-specific changes in occupancy in lso2Δ. These defects, and the conservation of the specific ribosome-binding activity of Lso2/CCDC124, indicate broadly important functions in translation and physiology.', 'Series_overall_design': 'Enhanced crosslinking and immunoprecipitation (IP) followed by deep sequencing: 5 IP samples of Lso2 and CCDC124, 6 controls of size-matched input and untagged IP. Ribosome footprint profiling: 7 samples of wild-type and 7 samples of lso2 null mutants.', 'Series_type': 'Other', 'Series_contributor': 'Wendy,,Gilbert', 'Series_sample_id': 'GSM3185748', 'Series_contact_name': 'Yinuo,,Wang', 'Series_contact_laboratory': 'Dr. Wendy Gilbert', 'Series_contact_department': 'Molecular Biophysics and Biochemistry', 'Series_contact_institute': 'Yale University', 'Series_contact_address': '333 Cedar St C129', 'Series_contact_city': 'New Haven', 'Series_contact_zip/postal_code': '06510', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE109nnn/GSE109343/suppl/GSE109343_Sigma1278b.single_copy_tRNAs_rRNAs.fasta.gz', 'Series_platform_id': 'GPL19756', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP131127'}\n",
      "201 PRJNA432322 GSE109907 {'Series_title': 'Potassium tellurite exerts its toxicity on ribosome in Escherichia coli', 'Series_geo_accession': 'GSE109907', 'Series_status': 'Public on Jan 29 2021', 'Series_submission_date': 'Jan 31 2018', 'Series_last_update_date': 'Jan 29 2021', 'Series_summary': 'Conclusions: Tellurite exerts its toxicity on ribosome by disabling 70S ribosome formation, reaching to stop codon, and inducing differential gene expression.', 'Series_overall_design': 'Ribosome footprints and mRNAs from BW25113 (WT) and JW2553 (ΔEF4) strains that were treated with potassium tellurite and sequenced, in two replicates, using Illumina HiSeq 2500', 'Series_type': 'Other', 'Series_contributor': 'Chunlai,,Chen', 'Series_sample_id': 'GSM2973660', 'Series_contact_name': 'Bin,,Liu', 'Series_contact_email': 'lexylexylexy@163.com', 'Series_contact_phone': '86-18810235415', 'Series_contact_department': 'School of Life Sciences', 'Series_contact_institute': 'Tsinghua University', 'Series_contact_address': 'Hai Dian', 'Series_contact_city': 'Beijing', 'Series_contact_state': 'Beijing', 'Series_contact_zip/postal_code': '100083', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE109nnn/GSE109907/suppl/GSE109907_Processed_data_RPKMs.xlsx', 'Series_platform_id': 'GPL18133', 'Series_platform_taxid': '562', 'Series_sample_taxid': '562', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP131851'}\n",
      "202 PRJNA432508 GSE109980 {'Series_title': 'Global identification of functional microRNA-mRNA interactions in Drosophila', 'Series_geo_accession': 'GSE109980', 'Series_status': 'Public on Feb 11 2019', 'Series_submission_date': 'Feb 01 2018', 'Series_last_update_date': 'Mar 27 2020', 'Series_pubmed_id': '30967537', 'Series_summary': 'Absolute quantification of cellular miRNA levels presents the miRNA pool in Drosophila cell lines to be more diverse than previously reported. Benchmarking two CLIP approaches, we identify a similar predictive potential to unambiguously assign thousands of miRNA-mRNA pairs from AGO1 interaction data at unprecedented depth, achieving higher signal-to-noise ratios than with computational methods alone. Quantitative RNA-Seq and sub-codon resolution ribosomal footprinting data upon AGO1 depletion enabled the determination of miRNA-mediated effects on target expression and translation. We thus provide the first comprehensive resource of miRNA target sites and their quantitative functional impact in Drosophila.', 'Series_overall_design': '1) Endogenous AGO1 PARCLIP and HITSCLIP small RNA sequencing data of wt Drosophila S2 cells. 2) smallRNA-seq data of endogenously 5’phosphorylated small RNA species wt Drosophila S2 cells. 3) Stranded, single-end, rRNA-depleted RNA-seq data upon dsRNA-mediated knockdown of AGO1 and control treatments. 4) Matched ribosomal footprinting data upon dsRNA-mediated knockdown of AGO1 and control treatments.', 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'Uwe,,Ohler', 'Series_sample_id': 'GSM2975561', 'Series_contact_name': 'Hans-Hermann,,Wessels', 'Series_contact_email': 'hwessels@nygenome.org', 'Series_contact_institute': 'New York Genome Center', 'Series_contact_address': '101 Avenue of the Americas', 'Series_contact_city': 'New York', 'Series_contact_zip/postal_code': '10013', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE109nnn/GSE109980/suppl/GSE109980_XtailResults_dsAGO1low_vs_dsGFP.csv.gz', 'Series_platform_id': 'GPL19132', 'Series_platform_taxid': '7227', 'Series_sample_taxid': '7227', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP131957'}\n",
      "203 PRJNA436358 GSE111255 {'Series_title': 'DBP1 in budding yeast', 'Series_geo_accession': 'GSE111255', 'Series_status': 'Public on Mar 01 2018', 'Series_submission_date': 'Feb 28 2018', 'Series_last_update_date': 'Jul 18 2019', 'Series_pubmed_id': '31299079', 'Series_summary': 'We analyzed the effect of deleting the gene encoding putative RNA helicase DBP1 in budding yeast on translational efficiencies (TEs) genome wide in wild-type or ded1-ts (temperature-sensitive allele of DED1) strains by combining ribosome footprint profiling with RNA-seq analysis of mRNA abundance.  This study includes a total of 32 samples comprised of 16 RNA-Seq samples (mRNA) and 16 ribosome footprint profiling samples (ribo). Experiment 1 includes 8 samples, comprised of 4 RNA-Seq samples and 4 ribosome footprint profiling samples, derived from 2 biological replicates each of the dbp1Δ  and ded1-ts dbp1Δ mutant strains, cultured in synthetic complete (SC) medium, following shifts in growth temperature from 30°C to 37°C for 2h.  Experiment 2 examines the effect of overexpression of DBP1 in rescuing genome-wide translational defects of a ded1-cs mutant (cold-sensitive allele of DED1) and includes 12 samples, 6 RNA-Seq samples and 6 ribosome footprint profiling samples, derived from 2 biological replicates each of WT DED1 (dubbed DED1-CS), ded1-cs and ded1-cs overexpressing DBP1 (ded1-cs_hcDBP1) strains, cultured in SC-Leu-His medium, following shifts in growth temperature from 30°C to 15°C for 10 min. Experiment 3 examines rescue of translational defects of ded1-ts by DBP1 overexpression and includes 12 samples, 6 RNA-Seq samples and 6 ribosome footprint profiling samples, derived from 2 biological replicates each of WT DED1(DED1-TS), ded1-ts and ded1-ts overexpressing DBP1 (ded1-ts_hcDBP1) strains, cultured in SC-Leu-His medium, following shifts in growth temperature from 30°C to 37°C for 2h.', 'Series_overall_design': '32 samples', 'Series_type': 'Other', 'Series_contributor': 'Alan,G,Hinnebusch', 'Series_sample_id': 'GSM3027265', 'Series_contact_name': 'Alan,G,Hinnebusch', 'Series_contact_email': 'ahinnebusch@nih.gov', 'Series_contact_institute': 'NICHD', 'Series_contact_address': '6 CENTER DR Room 230, MSC 0609', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '20814', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE111nnn/GSE111255/suppl/GSE111255_ded1tsdbp1_37deg.csv.gz', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP133670'}\n",
      "204 PRJNA438445 GSE111866 {'Series_title': 'Ribosome profiling and transcriptome profiling of MCF-7 cells', 'Series_geo_accession': 'GSE111866', 'Series_status': 'Public on Jul 13 2018', 'Series_submission_date': 'Mar 15 2018', 'Series_last_update_date': 'Mar 27 2019', 'Series_pubmed_id': '30102689', 'Series_summary': 'Keywords: ribosome profiling, translation, MCF-7', 'Series_overall_design': 'We performed ribosome profiling (RIBO-seq) and transcriptome profiling (RNA-seq) on MCF-7 cells', 'Series_type': 'Other', 'Series_contributor': 'Gabriella,,Viero', 'Series_sample_id': 'GSM3042965', 'Series_contact_name': 'Toma,,Tebaldi', 'Series_contact_email': 't.tebaldi@unitn.it', 'Series_contact_department': 'Centre for Integrative Biology', 'Series_contact_institute': 'University of Trento', 'Series_contact_address': 'via Sommarive n. 9', 'Series_contact_city': 'Povo', 'Series_contact_state': 'Trento', 'Series_contact_zip/postal_code': '38123', 'Series_contact_country': 'Italy', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE111nnn/GSE111866/suppl/GSE111866_MCF7_count_table.txt.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP135757'}\n",
      "205 PRJNA438489 GSE111891 {'Series_title': 'Ribosome profiling experiment in control or Elp3 depleted B16 melanoma cells', 'Series_geo_accession': 'GSE111891', 'Series_status': 'Public on Jun 26 2018', 'Series_submission_date': 'Mar 15 2018', 'Series_last_update_date': 'Mar 25 2019', 'Series_pubmed_id': '29925953', 'Series_summary': 'We report the application of ribosome profiling using high-throughput sequencing and total mRNA used for normalisation upon depletion of ELP3 protein in B16 melanoma cells.', 'Series_overall_design': 'Examination of Ribosome distribution across mRNAs in control or Elp3 depleted B16 melanoma cells', 'Series_type': 'Other', 'Series_contributor': 'Pierre,,Close', 'Series_sample_id': 'GSM3043651', 'Series_contact_name': 'pierre,,close', 'Series_contact_laboratory': 'Cancer Signaling', 'Series_contact_department': 'GIGA-Institute', 'Series_contact_institute': 'University of Liege', 'Series_contact_address': \"Avenue de l'hopital\", 'Series_contact_city': 'Liegge', 'Series_contact_zip/postal_code': '4000', 'Series_contact_country': 'Belgium', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE111nnn/GSE111891/suppl/GSE111891_MergedCounts.txt.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP135781'}\n",
      "206 PRJNA438895 GSE111991 {'Series_title': 'Divergent rRNAs as regulators of gene expression at the ribosome level', 'Series_geo_accession': 'GSE111991', 'Series_status': 'Public on Nov 27 2018', 'Series_submission_date': 'Mar 19 2018', 'Series_last_update_date': 'Feb 21 2019', 'Series_pubmed_id': '30718849', 'Series_summary': 'It is generally assumed that each organism has evolved to possess a unique ribosomal RNA (rRNA) species optimal for its physiological needs. However, some organisms express divergent rRNAs, the functional roles of which remain unknown. Here, we show that ribosomes containing the most variable rRNAs, encoded by the rrnI operon (herein designated I-ribosomes), direct the preferential translation of a subset of mRNAs in Vibrio vulnificus, enabling the rapid adaptation of bacteria to temperature and nutrient shifts. In addition, genetic and functional analyses of I-ribosomes and target mRNAs suggest that both I-ribosomal subunits are required for the preferential translation of specific mRNAs, the Shine-Dalgarno sequences of which do not play a critical role in I-ribosome binding. This study identifies genome-encoded divergent rRNAs as regulators of gene expression at the ribosome level, providing an additional level of regulation of gene expression in bacteria in response to environmental changes.', 'Series_overall_design': 'Sequencing data analysis. The sequence of the reference genome was retrieved from the NCBI database. RNA-seq reads were aligned to the reference genome sequence using Bowtie2 or BWA. Coordinates of mapped reads were obtained by processing BAM formatted files using BEDOPS tools and were stored in BED formatted files. Coordinates of genome features such as protein coding genes were extracted from the genome annotation data. Boundaries of features such as transcription start sites and gene segments were prepared: a transcription start site was defined as positions from -60 to +60 (the first nucleotide of a start codon was numbered as +1) and a gene segment as positions from -60 to the end of the stop codon. Overlapping genome-mapped RNA-seq reads were assigned to these features. The normalized abundance of reads was measured as number of reads per million mapped reads (RPM).', 'Series_type': 'Other', 'Series_contributor': 'Kangseok,,Lee', 'Series_sample_id': 'GSM3048769', 'Series_contact_name': 'Seong-il,,Eyun', 'Series_contact_department': 'Department of Life Science', 'Series_contact_institute': 'Chung-Ang University', 'Series_contact_address': '84 Heukseok-ro', 'Series_contact_city': 'Seoul', 'Series_contact_zip/postal_code': '06911', 'Series_contact_country': 'South Korea', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE111nnn/GSE111991/suppl/GSE111991_new_table_S2.xlsx', 'Series_platform_id': 'GPL24744', 'Series_platform_taxid': '216895', 'Series_sample_taxid': '216895', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP152673'}\n",
      "207 PRJNA439265 GSE112085 {'Series_title': 'Codon usage optimization in pluripotent embryonic stem cells [RNA-seq]', 'Series_geo_accession': 'GSE112085', 'Series_status': 'Public on May 17 2019', 'Series_submission_date': 'Mar 20 2018', 'Series_last_update_date': 'Jun 20 2019', 'Series_pubmed_id': '31174582', 'Series_summary': 'The uneven use of synonymous codons in the transcriptome regulates the efficiency and fidelity of protein translation rates. Yet, the importance of this codon bias on regulating cell state-specific expression programs is currently debated. Here, we asked whether the gene expression program in the well-defined cell states of self-renewal and differentiation in embryonic stem cells is driven by optimized codon usage. Using ribosome and transcriptome profiling, we identified distinct codon signatures for human self-renewing and differentiating embryonic stem cells. One driver for the cell state-specific codon bias was the genomic GC-content of the differentially expressed genes and thus, determined by transcription rather than translation. However, by measuring the codon frequencies at the ribosome’s active sites interacting with transfer RNAs (tRNA), we discovered that the wobble position tRNA modification inosine strongly influenced the codon optimization in self-renewing embryonic stem cells. This effect was conserved in mice and independent of the differentiation\\xa0stimulus. In summary, we newly reveal how translational mechanisms based on RNA\\xa0modifications can shape optimized codon usage in embryonic stem cells.', 'Series_overall_design': 'Ribosome profiling and total RNA from self-renewing and differentiating human H9 cells, 4 biological replicates per condition', 'Series_type': 'Other', 'Series_contributor': 'Michaela,,Frye', 'Series_sample_id': 'GSM3057215', 'Series_contact_name': 'Sabine,,Dietmann', 'Series_contact_institute': 'Washington University School of Medicine', 'Series_contact_address': '4565 McKinley Avenue', 'Series_contact_city': 'St Louis', 'Series_contact_state': 'MO', 'Series_contact_zip/postal_code': '63110', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE112nnn/GSE112085/suppl/GSE112085_Ribo_counts_CDS.txt.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP136094'}\n",
      "208 PRJNA445183 GSE112185 {'Series_title': 'RPL12/uL11 phosphorylation regulates translation during mitosis [Ribosome profiling]', 'Series_geo_accession': 'GSE112185', 'Series_status': 'Public on May 03 2019', 'Series_submission_date': 'Mar 21 2018', 'Series_last_update_date': 'May 03 2019', 'Series_pubmed_id': '30220558', 'Series_summary': 'Emerging evidence indicates that heterogeneity in ribosome composition can give rise to specialized functions. Until now, research mainly focused on differences in core ribosomal proteins and associated factors. The impact of posttranslational modifications has not yet been studied systematically. Analyzing ribosome heterogeneity is challenging since individual proteins can be part of different subcomplexes (40S, 60S, 80S and polysomes). Here, we develop polysome proteome profiling to obtain unbiased proteomic maps across ribosomal subcomplexes. Our method combines extensive fractionation by sucrose gradient centrifugation with quantitative mass spectrometry. The high resolution of the profiles allows us to assign proteins to specific subcomplexes. Phosphoproteomics on the fractions reveals that phosphorylation of serine 38 in RPL12/uL11 -- a known mitotic CDK1 substrate -- is strongly depleted in polysomes. Follow-up experiments confirm that RPL12/uL11 phosphorylation regulates translation of specific subsets of mRNAs during mitosis. Together, our results show that posttranslational modification of ribosomal proteins can regulate translation.', 'Series_overall_design': 'Ribosome profiling of B cells with different mutant RPL12 backgrounds', 'Series_type': 'Other', 'Series_contributor': 'Matthias,,Selbach', 'Series_sample_id': 'GSM3059585', 'Series_contact_name': 'Katharina,,Schmidt-Bleek', 'Series_contact_email': 'katharina.schmidt-bleek@bih-charite.de', 'Series_contact_department': 'Julius Wolff Institute of Biomechanics and Musculoskeletal Regeneration', 'Series_contact_institute': 'Charité - Universitätsmedizin Berlin', 'Series_contact_address': 'Augustenburger Platz 1', 'Series_contact_city': 'Berlin', 'Series_contact_zip/postal_code': '13353', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE112nnn/GSE112185/suppl/GSE112185_RAW.tar', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP136215'}\n",
      "209 PRJNA445295 GSE112223 {'Series_title': 'Translation elongation rate varies among organs and decreases with age', 'Series_geo_accession': 'GSE112223', 'Series_status': 'Public on Dec 31 2018', 'Series_submission_date': 'Mar 22 2018', 'Series_last_update_date': 'Feb 01 2021', 'Series_pubmed_id': '33264395', 'Series_summary': 'There has been a surge of interest towards targeting protein synthesis to treat diseases and extend lifespan. Despite the progress, few options are available to assess translation in live animals, as their complexity limits the repertoire of experimental tools to monitor and manipulate processes within organs and individual cells. It this study, we developed a labeling-free method for measuring organ- and cell-type-specific translation elongation rates in vivo. It is based on time-resolved delivery of translation initiation and elongation inhibitors in live animals followed by ribosome profiling. It also reports translation initiation sites in an organ-specific manner. Using this method, we found that the elongation rates differ more than 50% among mouse organs and determined them to be 6.8, 5.0, and 4.3 amino acids per second for liver, kidney, and skeletal muscle, respectively. We further found that the elongation rate is reduced by 20% between young adulthood and mid-life. Thus, translation, a major metabolic process in cells, is tightly regulated at the level of elongation of nascent polypeptide chains.', 'Series_overall_design': 'Measuring translation elongation rate in three organs (heart, kidney, skeletal muscle) and two ages (15 and 72 weeks), three timepoints per each organ, 2-4 replicated per a timepoint.', 'Series_type': 'Other', 'Series_contributor': 'Vadim,N,Gladyshev', 'Series_sample_id': 'GSM4795543', 'Series_contact_name': 'Maxim,,Gerashchenko', 'Series_contact_email': 'mgerashchenko@bwh.harvard.edu', 'Series_contact_department': 'Medicine', 'Series_contact_institute': \"Brigham and Women's Hospital\", 'Series_contact_address': '77 Louis Pasteur Ave, NRB 435', 'Series_contact_city': 'Boston', 'Series_contact_state': 'Massachusetts', 'Series_contact_zip/postal_code': '02115', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE112nnn/GSE112223/suppl/GSE112223_organs_mRNAseq_count.txt.gz', 'Series_platform_id': 'GPL21103', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP136268'}\n",
      "210 PRJNA445453 GSE112276 {'Series_title': 'Transcriptome Mapping of Internal N7-methylguanosine Methylome in Mammalian mRNA', 'Series_geo_accession': 'GSE112276', 'Series_status': 'Public on Apr 26 2019', 'Series_submission_date': 'Mar 23 2018', 'Series_last_update_date': 'Mar 16 2023', 'Series_pubmed_id': '31031084', 'Series_summary': 'N7-methylguanosine (m7G) is a positively charged, essential modification at the 5′ cap of eukaryotic mRNA, regulating mRNA export, translation, and splicing. m7G also occurs internally within tRNA and rRNA, but its existence and distribution within eukaryotic mRNA remain to be investigated. Here, we show the presence of internal m7G sites within mammalian mRNA. We then performed transcriptome-wide profiling of internal m7G methylome using m7G-MeRIP sequencing (MeRIP-seq). To map this modification at base resolution, we developed a chemical-assisted sequencing approach that selectively converts internal m7G sites into abasic sites, inducing misincorporation at these sites during reverse transcription. This base-resolution m7G-seq enabled transcriptome-wide mapping of m7G in human tRNA and mRNA, revealing distribution features of the internal m7G methylome in human cells. We also identified METTL1 as a methyltransferase that installs a subset of m7G within mRNA and showed that internal m7G methylation could affect mRNA translation.', 'Series_overall_design': '[dataset 4] 38 samples. Duplicates for m7G-MeRIP-seq (anti-m7G antibody, MBL) of cell mRNA samples, including two sections of RNA input and IP in HeLa, HepG2, HEK293T, MEF and mESC wild-type cell lines; also in HeLa cell siControl vs siMETTL1, and HepG2 cell shControl vs shMETTL1.', 'Series_type': 'Other', 'Series_contributor': 'Chuan,,He', 'Series_sample_id': 'GSM3449507', 'Series_contact_name': 'Li-Sheng,,Zhang', 'Series_contact_email': 'zhangls@ust.hk', 'Series_contact_phone': '(852) 3469 2392', 'Series_contact_laboratory': 'Prof. Chuan He Lab', 'Series_contact_department': 'Department of Chemistry', 'Series_contact_institute': 'The University of Chicago', 'Series_contact_address': '929 E 57th St Rm E313', 'Series_contact_city': 'Chicago', 'Series_contact_state': 'IL', 'Series_contact_zip/postal_code': '60637', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE112nnn/GSE112276/suppl/GSE112276_mESC_HighFC.txt.gz', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP136352'}\n",
      "211 PRJNA445600 GSE112305 {'Series_title': 'Post-transcriptional regulation of the epithelial cell response to colitis', 'Series_geo_accession': 'GSE112305', 'Series_status': 'Public on Dec 31 2018', 'Series_submission_date': 'Mar 25 2018', 'Series_last_update_date': 'May 13 2019', 'Series_pubmed_id': '31061170', 'Series_summary': 'Conclusions: IMP1 acts as a post-transcriptional regulator of gut epithelial repair, in part through modulation of autophagy. This study highlights the need for examining post-transcriptional regulation as a critical mechanism in inflammatory bowel disease.', 'Series_overall_design': 'Ribosome-footprinting and RNA-seq samples from WT SW480 cells and IMP1-/- knockout cells', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'K,E,Hamilton', 'Series_sample_id': 'GSM3067423', 'Series_contact_name': 'Premal,,Shah', 'Series_contact_email': 'premal.shah@rutgers.edu', 'Series_contact_laboratory': 'Shah Lab', 'Series_contact_department': 'Genetics', 'Series_contact_institute': 'Rutgers University', 'Series_contact_address': '145 Bevier Road, Rm 326', 'Series_contact_city': 'Piscataway', 'Series_contact_state': 'NJ', 'Series_contact_zip/postal_code': '08854', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE112nnn/GSE112305/suppl/GSE112305_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP136411'}\n",
      "212 PRJNA445733 GSE112353 {'Series_title': 'Active ribosome profiling  with RiboLace and standard ribosome profiling in HEK-293 cells', 'Series_geo_accession': 'GSE112353', 'Series_status': 'Public on Oct 09 2018', 'Series_submission_date': 'Mar 26 2018', 'Series_last_update_date': 'Mar 27 2019', 'Series_pubmed_id': '30355487', 'Series_summary': 'Keywords: ribosome profiling, translation, RiboLace, Ribo-Seq, active translation', 'Series_overall_design': 'We performed active ribosome profiling with RiboLace and standard ribosome profiling in HEK-293 cells. Experiments were performed in biological duplicate.', 'Series_type': 'Other', 'Series_contributor': 'Gabriella,,Viero', 'Series_sample_id': 'GSM3068267', 'Series_contact_name': 'Toma,,Tebaldi', 'Series_contact_email': 't.tebaldi@unitn.it', 'Series_contact_department': 'Centre for Integrative Biology', 'Series_contact_institute': 'University of Trento', 'Series_contact_address': 'via Sommarive n. 9', 'Series_contact_city': 'Povo', 'Series_contact_state': 'Trento', 'Series_contact_zip/postal_code': '38123', 'Series_contact_country': 'Italy', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE112nnn/GSE112353/suppl/GSE112353_HEK_count_table.txt.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP136497'}\n",
      "213 PRJNA448005 GSE112502 {'Series_title': 'Regulatory Discrimination of mRNAs by FMRP Controls Adult Neural Stem Cell Differentiation', 'Series_geo_accession': 'GSE112502', 'Series_status': 'Public on Nov 20 2018', 'Series_submission_date': 'Mar 29 2018', 'Series_last_update_date': 'Mar 25 2019', 'Series_pubmed_id': '30373821', 'Series_summary': 'Fragile X syndrome (FXS) is caused by the loss of fragile X mental retardation protein (FMRP), a translation-inhibitor RNA binding protein. The impact of FMRP-deficiency on neural function is widespread, including its regulation of adult neural stem cell (aNSC) differentiation. To assess FMRP activity, we performed ribosome profiling of aNSCs from normal and Fmr1 knockout mice, which revealed diverse gene expression changes at the mRNA and translation levels. Many mitosis and neurogenesis genes were dysregulated primarily at the mRNA level, while numerous synaptic genes were mostly dysregulated at the translation level. Translational “buffering” was also evident, whereby changes in ribosome association with mRNA is compensated by alterations in RNA abundance. Our data revealed that FMRP-regulated neurogenesis is mediated by the transcriptional factor necdin and that FMRP determines mitochondrial mRNA expression and energy homeostasis. Thus, FMRP controls diverse transcriptional and post-transcriptional gene expression programs critical for the adult neural stem cell differentiation.', 'Series_overall_design': 'Ribosome profiling and RNA-seq of wild type (WT) and Fmr1 -/y mouse adult neural stem cells were generated from four biological replicates using Illumina Nextseq.', 'Series_type': 'Other', 'Series_contributor': 'Joel,D,Richter', 'Series_sample_id': 'GSM3071486', 'Series_contact_name': 'Botao,,Liu', 'Series_contact_laboratory': 'Joel Richter Lab', 'Series_contact_department': 'Program in Molecular Medicine', 'Series_contact_institute': 'University of Massachusetts Medical School', 'Series_contact_address': '373 Plantation Street', 'Series_contact_city': 'Worcester', 'Series_contact_state': 'Massachusetts', 'Series_contact_zip/postal_code': '01605', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE112nnn/GSE112502/suppl/GSE112502_ansc.ribo.rna.rsem.gene.summary.tpm.txt.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP136730'}\n",
      "214 PRJNA448601 GSE112643 {'Series_title': 'Effects of single amino acid deficiency on mRNA translation are markedly different for methionine versus leucine', 'Series_geo_accession': 'GSE112643', 'Series_status': 'Public on Apr 04 2018', 'Series_submission_date': 'Apr 03 2018', 'Series_last_update_date': 'Mar 27 2019', 'Series_pubmed_id': '29795412', 'Series_summary': 'Amino acid availability regulates translation through the action of the GCN2 and mTORC1 pathways. Low amino acids activate the eIF2α kinase GCN2 through binding of uncharged tRNAs to a histidyl-tRNA synthetase−related regulatory domain. Once activated GCN2 phosphorylates eIF2α, inhibiting ternary complex formation and translation initiation. Recent studies show that mTORC1 is particularly sensitive to arginine and leucine status, with a deprivation of these amino acids leading to a strong inhibition of mTORC1 that prevents the phosphorylation and inactivation of the translational repressor 4EBP1. Though amino acids are known regulators of translation, the effects that deficiencies of specific amino acids have on translation have yet to be determined. We demonstrate that deprivation of leucine or methionine results in large inhibitory effects on translation initiation and on polysome formation that are not replicated by overexpressing non-phosphorylatable 4EBP1 or a phosphomimetic eIF2α. Our results demonstrate that a lack of either leucine or methionine has a major impact on mRNA translation, though they act by quite different mechanisms. Leucine deprivation appears to primarily inhibit ribosome loading, whereas methionine deprivation appears to primarily impair start site recognition. These data point to a unique regulatory effect that methionine status has on translation initiation.', 'Series_overall_design': 'Ribosome profiling of single Amino acid starvation', 'Series_type': 'Other', 'Series_contributor': 'Martha,H,Stipanuk', 'Series_sample_id': 'GSM3074853', 'Series_contact_name': 'Yuanhui,,Mao', 'Series_contact_email': 'maoyuanhui123@gmail.com', 'Series_contact_institute': 'Zhejiang University', 'Series_contact_address': '1369 West Wenyi Road', 'Series_contact_city': 'Hangzhou', 'Series_contact_state': 'Zhejiang', 'Series_contact_zip/postal_code': '310000', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE112nnn/GSE112643/suppl/GSE112643_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP137018'}\n",
      "215 PRJNA448763 GSE112705 {'Series_title': 'Landscapes of gene translation in hepatocellular carcinoma tumors revealed by ribosome profiling', 'Series_geo_accession': 'GSE112705', 'Series_status': 'Public on Jun 14 2019', 'Series_submission_date': 'Apr 04 2018', 'Series_last_update_date': 'Jul 16 2019', 'Series_pubmed_id': '31281537', 'Series_summary': 'Despite the critical role of translation in the multi-level gene expression regulation program that controls the protein abundance, a high-resolution view on the landscapes of gene translation in cancer has been missing. We used the technique of ribosome profiling to dissect the translatomes of human liver tumors and their adjacent normal tissues. Our results, as a novel resource, revealed genome-wide abnormalities of translation efficiencies in tumors and context-dependent non-canonical ORFs.', 'Series_overall_design': 'We profiled the paired tumor and adjacent normal tissue samples from 10 patients of HCC by the procedure of ribosome profiling, which was adapted and optimized for processing of clinical solid tissue samples. Total RNA sequencing was performed in parallel for the same samples.', 'Series_type': 'Other', 'Series_contributor': 'Xuerui,,Yang', 'Series_sample_id': 'GSM3081775', 'Series_contact_name': 'Xuerui,,Yang', 'Series_contact_email': 'yangxuerui@tsinghua.edu.cn', 'Series_contact_department': 'School of Life Sciences', 'Series_contact_institute': 'Tsinghua University', 'Series_contact_address': 'Haidian District', 'Series_contact_city': 'Beijing', 'Series_contact_zip/postal_code': '100084', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE112nnn/GSE112705/suppl/GSE112705_RPF_RNA_readCounts.txt.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP137150'}\n",
      "216 PRJNA448921 GSE112766 {'Series_title': 'DDX6 decouples translational repression from RNA degradation of miRNA targets [Ribosome Profiling]', 'Series_geo_accession': 'GSE112766', 'Series_status': 'Public on Jul 25 2018', 'Series_submission_date': 'Apr 05 2018', 'Series_last_update_date': 'Sep 15 2021', 'Series_pubmed_id': '30044225', 'Series_summary': 'Translation and mRNA degradation are intimately connected, yet the mechanisms that regulate them are not fully understood. Here we examine the regulation of translation and mRNA stability in mouse embryonic stem cells (ESCs) and during differentiation. In contrast to previous reports, we found that transcriptional changes account for most of the molecular changes during ESC differentiation. Within ESCs translation level and mRNA stability are positively correlated. The RNA-binding protein DDX6 has been implicated in processes involving both translational repression and mRNA destabilization; in yeast DDX6 connects codon optimality and mRNA stability and in mammals DDX6 is involved in microRNA-mediated repression. We generated DDX6 KO ESCs and found that while there was minimal connection between codon usage and stability changes, the loss of DDX6 leads to the translational depression of microRNA targets. Surprisingly, the translational derepression of microRNA targets occurs without affecting mRNA stability. Furthermore, DDX6 KO ESCs share overlapping phenotypes and global molecular changes with ESCs that completely lack all microRNAs. Together our results demonstrate that the loss of DDX6 decouples the two forms of microRNA induced repression and emphasize that translational repression by microRNAs is underappreciated.', 'Series_overall_design': 'Ribosome profilling was performed on mouse embryonic stem cells (ESCs) and Epiblast like cells (EpiLCs).', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Robert,,Blelloch', 'Series_sample_id': 'GSM3083132', 'Series_contact_name': 'Robert,,Blelloch', 'Series_contact_email': 'blellochr@stemcell.ucsf.edu', 'Series_contact_phone': '415-476-2838', 'Series_contact_institute': 'University of California, San Francisco', 'Series_contact_address': '513 Parnassus Avenue, Campus Box #0525', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94143', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE112nnn/GSE112766/suppl/GSE112766_Ribosome_profiling_counts.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP137758'}\n",
      "217 PRJNA453686 GSE113695 {'Series_title': 'EIF1AX-A113 splice and RAS mutations cooperate to drive thyroid tumorigenesis through ATF4 and c-MYC', 'Series_geo_accession': 'GSE113695', 'Series_status': 'Public on Jan 28 2019', 'Series_submission_date': 'Apr 26 2018', 'Series_last_update_date': 'Apr 29 2019', 'Series_pubmed_id': '30305285', 'Series_summary': 'Translation initiation in higher eukaryotes is orchestrated by the tight regulation of the cap binding and the 43S pre-initiation complexes (PIC). The PIC component eukaryotic initiation factor 1A (EIF1A), encoded on human chromosomes X and Y by EIF1AX and EIF1AY, respectively, is essential for recruitment of the ternary complex and for assembling the 43S PIC, which after recruitment onto capped mRNAs scans their 5’UTR and localizes the AUG to initiate translation. Recurrent EIF1AX mutations are found in several cancers, including ~1% of papillary thyroid cancers in a mutually exclusive manner with other drivers (BRAF, RAS, and oncogenic fusions). They are enriched in advanced thyroid cancers (11%) where they display a striking co-occurrence with RAS, which we show cooperate to induce tumorigenesis in mouse models and in isogenic cell lines. The C-terminal tail EIF1AX-A113splice mutation is the most prevalent in advanced thyroid cancer, and private to this disease. We found that EIF1AX-A113spl variants stabilize the PIC and induces ATF4 expression, a sensor of cellular stress, which is co-opted to suppress EIF-2α phosphorylation, thus enabling a general increase in protein synthetic rate. RAS stabilizes c-MYC, an effect augmented by EIF1AX-A113spl. ATF4 and c-MYC induce expression of aminoacid transporters, and enhance sensitivity of mTOR to aminoacid supply. These mutually reinforcing events generate therapeutic vulnerabilities to MEK, BRD4 and mTOR kinase inhibitors. The RNA-Seq and Ribosome Profiling samples provided here were used to support the claim of increased translational efficiency of ATF4.', 'Series_overall_design': 'Comparison of Ribosome footprints in the context of EIF1AX-splice mutated (C643) and splice reverted isogenic cell line (C643-spl- rev), three replicates of ribosome-protected RNA sequencing and three replicates of RNA-seq were used. Additionally, 3 replicates of RNA-seq comparing KRAS G12R mutant CAL62 vs. CAL62-splice, an isogenic cell line generated with CRISPR-Cas9 mediated knock in of EIF1AX-splice mutation.', 'Series_type': 'Other', 'Series_contributor': 'Natalie,R,Davidson', 'Series_sample_id': 'GSM3112198', 'Series_contact_name': 'Gunnar,,Rätsch', 'Series_contact_email': 'raetsch@inf.ethz.ch', 'Series_contact_laboratory': 'Biomedical Informatics', 'Series_contact_department': 'Institute for Machine Learning', 'Series_contact_institute': 'ETH Zürich', 'Series_contact_address': 'Universitatstrasse 6', 'Series_contact_city': 'Zurich', 'Series_contact_state': '-None-', 'Series_contact_zip/postal_code': '8006', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE113nnn/GSE113695/suppl/GSE113695_biomart_silva_human_rRNA.fasta.gz', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP142722'}\n",
      "218 PRJNA453992 GSE113789 {'Series_title': 'Ribosome profiling Seq of mouse adult cortex after acute stress', 'Series_geo_accession': 'GSE113789', 'Series_status': 'Public on Jul 10 2018', 'Series_submission_date': 'Apr 27 2018', 'Series_last_update_date': 'May 23 2021', 'Series_pubmed_id': '30048615', 'Series_summary': 'N6-Methyladenosine (m6A) and N6,2′-O-dimethyladenosine (m6Am) are abundant mRNA modifications that regulate transcript processing and translation. The role of both, here termed m6A/m, in the stress response in the adult brain in vivo are currently unknown.  Here, we provide ribosome profiling data of 10-12 w male C57BL/6 mouse cortex of unstressed or acutely stressed mice (4 h after 15 min restraint stress) of 6 samples per condition to relate to m6A/m-Seq data of similiar samples.', 'Series_overall_design': '6 replicates of basal mouse cortex (\"Control\", \"C\"), 6 replicates for stressed mouse cortex (4 h after 15 min of restraint stress, \"Stress\", \"S\").', 'Series_type': 'Other', 'Series_contributor': 'Alon,,Chen', 'Series_sample_id': 'GSM3119482', 'Series_contact_name': 'Mareen,,Engel', 'Series_contact_email': 'mareen_engel@psych.mpg.de', 'Series_contact_department': 'Stress Neurobiology and Neurogenetics', 'Series_contact_institute': 'Max-Planck-Institute of Psychiatry', 'Series_contact_address': 'Kraepelinstr. 2-10', 'Series_contact_city': 'Munich', 'Series_contact_zip/postal_code': '80804', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE113nnn/GSE113789/suppl/GSE113789_processed_data_files.tar.gz', 'Series_platform_id': 'GPL21103', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP143492'}\n",
      "219 PRJNA464051 GSE114064 {'Series_title': 'Widespread Alterations in Translation Elongation in the Brain of Fmr1 Knock-Out Mice', 'Series_geo_accession': 'GSE114064', 'Series_status': 'Public on Mar 10 2019', 'Series_submission_date': 'May 04 2018', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '30893603', 'Series_summary': 'FMRP is a polysome-associated RNA-binding protein encoded by\\xa0Fmr1\\xa0and lost in Fragile X syndrome. Increasing evidence suggests that FMRP regulates\\xa0both translation initiation and elongation, but the gene-specificity of these effects is unclear. To elucidate the\\xa0effects of FMRP loss on translation, we used ribosome profiling for genome-wide measurements of ribosomal occupancy and positioning in the\\xa0cortex of\\xa0Fmr1\\xa0knock-out mice. We found a remarkably coherent reduction in ribosome footprint abundance per mRNA for previously identified, high-affinity mRNA binding partners of FMRP, and an increase for terminal oligo-pyrimidine\\xa0(TOP) motif-containing genes canonically controlled by mTOR-4EBP-eIF4E signaling.\\xa0Amino\\xa0acid motif- and gene-level analyses\\xa0both\\xa0showed a widespread reduction of translational pausing in\\xa0Fmr1\\xa0knock-out mice. Our findings are consistent with a model of FMRP-mediated regulation of both translation initiation through eIF4E and elongation that is disrupted in Fragile X syndrome.', 'Series_overall_design': 'Examination of genome-wide translational state in Fmr1-deficient and wild-type mouse cortex via next-generation sequencing-based profiling of transcription and translation', 'Series_type': 'Other', 'Series_contributor': 'Peter,A,Sims', 'Series_sample_id': 'GSM3131880', 'Series_contact_name': 'Peter,,Sims', 'Series_contact_email': 'pas2182@cumc.columbia.edu', 'Series_contact_department': 'Systems Biology', 'Series_contact_institute': 'Columbia University', 'Series_contact_address': '3960 Broadway', 'Series_contact_city': 'New York', 'Series_contact_zip/postal_code': '10032', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE114nnn/GSE114064/suppl/GSE114064_All_Gene_Counts.txt.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP144655'}\n",
      "220 PRJNA472015 GSE114636 {'Series_title': 'Comprehensive profiling of translation initiation in influenza-virus infected cells', 'Series_geo_accession': 'GSE114636', 'Series_status': 'Public on May 20 2018', 'Series_submission_date': 'May 18 2018', 'Series_last_update_date': 'Mar 27 2019', 'Series_pubmed_id': '30673779', 'Series_summary': 'Cellular stress is often accompanied by non-canonical initiation of translation at alternate start codons in mammalian cells. Here we systematically investigate the extent and impact of alternate translation initiation in the context of influenza virus infection. We use ribosome profiling with the initiation inhibitor lactidomycin to experimentally delineate translation initiation sites in a human lung epithelial cell line infected with influenza virus.', 'Series_overall_design': 'We use genome-wide methods (Ribo-Seq and RNA-Seq) to examine how translation initiation is altered in influenza virus infected lung-epithelial cells', 'Series_type': 'Other', 'Series_contributor': 'Arvind,,Subramaniam', 'Series_sample_id': 'GSM3146286', 'Series_contact_name': 'Rasi,,Subramaniam', 'Series_contact_email': 'rasi@fredhutch.org', 'Series_contact_laboratory': 'Subramaniam', 'Series_contact_institute': 'Fred Hutch', 'Series_contact_address': '1100 Fairview Ave N', 'Series_contact_city': 'Seattle', 'Series_contact_zip/postal_code': '98109', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE114nnn/GSE114636/suppl/GSE114636_called_host_tis.tsv.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP148422'}\n",
      "221 PRJNA472972 GSE114882 {'Series_title': 'Nascent Polypeptide Domain Topology and Elongation Rate Direct the Cotranslational Hierarchy of Hsp70 and TRiC/CCT', 'Series_geo_accession': 'GSE114882', 'Series_status': 'Public on Aug 22 2019', 'Series_submission_date': 'May 24 2018', 'Series_last_update_date': 'Aug 24 2019', 'Series_pubmed_id': '31400849', 'Series_web_link': 'https://doi.org/10.1016/j.molcel.2019.06.036', 'Series_summary': 'Folding newly synthesized proteins relies on the ribosome intricately coordinating mRNA translation with a network of ribosome-associated machinery. The principles that drive the coordination of this diverse machinery remain poorly understood. Here, we use selective ribosome profiling to determine how the essential chaperonin TRiC/CCT and the Hsp70 Ssb are recruited to ribosome-nascent chain complexes to mediate cotranslational protein folding. Whereas substrate localization and nascent chain sequence are the major determinants of cotranslational recruitment of Ssb, we found that temporal and structural elements drive TRiC engagement. For both chaperones, however, local slowdowns in translation enhance chaperone enrichment. This work helps define the principles that dictate the coordinated activity of ribosome-associated factors to perform their critical role in maintaining a properly folded nascent proteome.', 'Series_overall_design': 'Replicate ribosome profiling libraries were prepared of immunoprecipitated TRiC/CCT- and Ssb-bound ribosome nascent chain complexes compared to the total ribosome pool, using wild-type yeast and yeast harboring the atp2-P353A,P355A mutations. Additional ribosome profiling libraries were prepared to monitor translation kinetics in yeast not treated with cycloheximide.', 'Series_type': 'Other', 'Series_contributor': 'Judith,,Frydman', 'Series_sample_id': 'GSM3152900', 'Series_contact_name': 'Kevin,C.,Stein', 'Series_contact_email': 'kcstein@stanford.edu', 'Series_contact_laboratory': 'Frydman', 'Series_contact_department': 'Biology', 'Series_contact_institute': 'Stanford University', 'Series_contact_address': 'Clark Center E200, 318 Campus Drive', 'Series_contact_city': 'Stanford', 'Series_contact_state': 'California', 'Series_contact_zip/postal_code': '94305', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE114nnn/GSE114882/suppl/GSE114882_RAW.tar', 'Series_platform_id': 'GPL21656', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP148891'}\n",
      "222 PRJNA472988 GSE114892 {'Series_title': 'Conserved mRNA-granule component Scd6 targets Dhh1 to   repress translation initiation and activates Dcp2-mediated mRNA decay in vivo', 'Series_geo_accession': 'GSE114892', 'Series_status': 'Public on Feb 27 2019', 'Series_submission_date': 'May 24 2018', 'Series_last_update_date': 'Mar 11 2019', 'Series_pubmed_id': '30532217', 'Series_summary': 'We have identified mRNAs whose abundance or translational efficiency is regulated in nutrient-rich medium by the Scd6 or Dhh1 protein in either wild-type cells or dcp2∆ cells lacking the decapping enzyme', 'Series_overall_design': 'Ribosome profiling and RNA-seq was conducted to determine changes in mRNA abundance, translation, or translational efficiecies in comparing scd6∆ or dhh1∆ mutants to an isogenic wild-type strain; or scd6∆dcp2∆ and dhh1∆dcp2∆ strains to a dcp2∆ mutant', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Alan,G,Hinnebusch', 'Series_sample_id': 'GSM3153826', 'Series_contact_name': 'Alan,,Hinnebusch', 'Series_contact_email': 'alanh@mail.nih.gov', 'Series_contact_phone': '301-496-4480', 'Series_contact_laboratory': 'Gene Regulation and Development', 'Series_contact_department': 'NICHD', 'Series_contact_institute': 'NIH', 'Series_contact_address': '6 Center Drive, Building 6, Room 233', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '20892', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE114nnn/GSE114892/suppl/GSE114892_DESeq2_scd6delta_vs_WT.csv.gz', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP148900'}\n",
      "223 PRJNA473808 GSE115105 {'Series_title': 'RNA m6A-Ythdf1 in dendritic cells triggers anti-tumor immunity (ribo-seq and m6a-seq in GMDCs)', 'Series_geo_accession': 'GSE115105', 'Series_status': 'Public on Dec 29 2018', 'Series_submission_date': 'May 30 2018', 'Series_last_update_date': 'Mar 19 2019', 'Series_pubmed_id': '30728504', 'Series_summary': 'Emerging evidence emphasizes the important role of tumor neoantigen in generating the spontaneous antitumor immune response and predicting the clinical response to immunotherapies. Despite the presence of numerous neoantigens, complete tumor elimination rarely occurs in majority of patients due to failures in mounting a sufficient and lasting antitumor immunity. Here we show that the durable neoanitgen-specific immunity is regulated by a m6A-binding protein, Ythdf1. In contrast to wild-type mice, Ythdf1-deficient (Ythdf1-/-) mice generate more antigen-specific CD8+ T cell response for persistent tumor control. Loss of Ythdf1 in dendritic cell (DC) results in an enhanced cross-presentation of tumor antigen and cross-priming of CD8+ T cell in vivo. To confirm our observations, we performed Ribo-Seq to analyze the translational efficiency of genes in DCs and performed m6A-seq to locate the m6A sites.', 'Series_overall_design': 'BMDCs obtained from WT and YTHDF1 KO mice were used to genetarate ribo-seq (monosome footprints and polysome footprints) and m6a-seq libraries. Two replicates for each conditions.', 'Series_type': 'Other', 'Series_contributor': 'Chuan,,He', 'Series_sample_id': 'GSM3165361', 'Series_contact_name': 'Dali,,Han', 'Series_contact_email': 'handali294@gmail.com', 'Series_contact_institute': 'Beijing Institute of Genomics, and China National Center for Bioinformation, Chinese Academy of Sciences', 'Series_contact_address': 'Da-Tun Road', 'Series_contact_city': 'Beijing', 'Series_contact_zip/postal_code': '100101', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE115nnn/GSE115105/suppl/GSE115105_RAW.tar', 'Series_platform_id': 'GPL21103', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP149359'}\n",
      "224 PRJNA473812 GSE115110 {'Series_title': 'NF-Y controls fidelity of transcription initiation at gene promoters through maintenance of the nucleosome-depleted region', 'Series_geo_accession': 'GSE115110', 'Series_status': 'Public on Jul 01 2019', 'Series_submission_date': 'May 30 2018', 'Series_last_update_date': 'Jul 26 2019', 'Series_pubmed_id': '31296853', 'Series_summary': 'Faithful transcription initiation is critical for accurate gene expression, yet the mechanisms underlying specific transcription start site (TSS) selection in mammals remain unclear. Here, we show that the histone-fold domain protein NF-Y, a ubiquitously expressed transcription factor, controls the fidelity of transcription initiation at gene promoters. We report that NF-Y maintains the region upstream of TSSs in a nucleosome-depleted state while simultaneously protecting this accessible region against aberrant and/or ectopic transcription initiation. We find that loss of NF-Y binding in mammalian cells disrupts the promoter chromatin landscape, leading to nucleosomal encroachment over the canonical TSS. Importantly, this chromatin rearrangement is accompanied by upstream relocation of the transcription pre-initiation complex and ectopic transcription initiation. Further, this phenomenon generates aberrant extended transcripts that undergo translation, disrupting gene expression profiles. These results demonstrate that NF-Y is a central player in TSS selection in metazoans and highlight the deleterious consequences of inaccurate transcription initiation.', 'Series_overall_design': 'MNase-Seq, RNA-Seq, Start-Seq, Ribo-Seq, and ATAC-seq experiments on control knockdown (KD) and NF-YA KD mouse ESCs.', 'Series_type': 'Genome binding/occupancy profiling by high throughput sequencing', 'Series_contributor': 'R,,Jothi', 'Series_sample_id': 'GSM3763081', 'Series_contact_name': 'Raja,,Jothi', 'Series_contact_email': 'jothi@mail.nih.gov', 'Series_contact_laboratory': 'Systems Biology', 'Series_contact_department': 'National Institute of Environmental Health Sciences', 'Series_contact_institute': 'National Institutes of Health', 'Series_contact_address': '111 TW Alexander Drive; A314', 'Series_contact_city': 'RTP', 'Series_contact_state': 'NC', 'Series_contact_zip/postal_code': '27709', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE115nnn/GSE115110/suppl/GSE115110_NT_open_chromatin_51nt.bigWig', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP149362'}\n",
      "225 PRJNA473925 GSE115146 {'Series_title': 'A Novel Class of MicroRNA Recognition Elements That Function Only in Open Reading Frames', 'Series_geo_accession': 'GSE115146', 'Series_status': 'Public on Aug 21 2018', 'Series_submission_date': 'May 31 2018', 'Series_last_update_date': 'May 08 2019', 'Series_pubmed_id': '30297778', 'Series_summary': 'MicroRNA (miRNA) are well known to target 3’ untranslated regions (3’UTR) in mRNAs to silence gene expression at post-transcriptional levels. Multiple reports have also indicated the capability of miRNAs to target protein-coding sequences (CDS); however, miRNAs have been generally believed to function in a similar mechanism(s) regardless of the location of their action sites. We herein report a class of miRNA recognition elements (MREs) that exclusively function in CDS regions in humans. Through functional and mechanistic characterization of these “unusual” MREs, we demonstrate that CDS-targeted miRNAs require extensive base pairings in the 3’ side rather than the 5’ seed; cause gene silencing in an Argonaute-dependent, but GW182-independent manner; and repress translation by inducing transient ribosome stalling instead of mRNA destabilization. These findings reveal distinct mechanisms and functional consequences for miRNAs to target CDS versus 3’UTR and suggest that CDS-targeted miRNAs may enlist a translational quality control (QC)-related mechanism to regulate translation in mammalian cells.', 'Series_overall_design': 'Ago2 eCLIP-seq, RNA-seq, and Ribo-seq w/o miR-17/20a overexpression', 'Series_type': 'Other', 'Series_contributor': 'Xiang-Dong,,Fu', 'Series_sample_id': 'GSM3168237', 'Series_contact_name': 'Xiang-Dong,,Fu', 'Series_contact_laboratory': 'George Palade Laboratories', 'Series_contact_department': 'CMM', 'Series_contact_institute': 'UC San Diego', 'Series_contact_address': '9500 Gilman Drive, Room 231', 'Series_contact_city': 'La Jolla', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '92093-0651', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE115nnn/GSE115146/suppl/GSE115146_RAW.tar', 'Series_platform_id': 'GPL20795', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP149426'}\n",
      "226 PRJNA473987 GSE115160 {'Series_title': 'Regulation of Translation Elongation Revealed by Ribosome Profiling [Dataset_3]', 'Series_geo_accession': 'GSE115160', 'Series_status': 'Public on Jan 27 2019', 'Series_submission_date': 'May 31 2018', 'Series_last_update_date': 'Aug 21 2020', 'Series_pubmed_id': '31858614', 'Series_summary': 'Ribosomes undergo substantial conformational changes during translation elongation to accommodate incoming aminoacyl-tRNAs and translocate along the mRNA template. We used multiple elongation inhibitors and chemical probing to define ribosome conformational states corresponding to different sized ribosome-protected mRNA fragments (RPFs) generated by ribosome profiling. We show using various genetic and environmental perturbations that the previously identified 20-22 nucleotide (nt) RPFs correspond predominantly to ribosomes in a pre-accommodation state with an open 40S ribosomal A site while the classical 27-29 nt fragments correspond to ribosomes in a pre-translocation state with an occupied 40S ribosomal A site. These distinct ribosome conformational states revealed by ribosome profiling are seen in all eukaryotes tested including fungi, worms and mammals. This high-resolution ribosome profiling approach reveals the anticipated Rck2-dependent inhibition of translocation through eEF2 phosphorylation during hyperosmotic stress. These same approaches reveal a strong translation elongation arrest during oxidative stress where the ribosomes are trapped in a pre-translocation state, but in this case the translational arrest is independent of Rck2-driven eEF2 phosphorylation. These results provide new insights and approaches for defining the molecular events that impact translation elongation throughout biology.', 'Series_overall_design': '5 biological samples are included for ribosome profiling libaries prepared with CHX in the lysis buffer.', 'Series_type': 'Other', 'Series_contributor': 'Rachel,,Green', 'Series_sample_id': 'GSM3442053', 'Series_contact_name': 'Colin,,Wu', 'Series_contact_department': 'RNA Biology Laboratory', 'Series_contact_institute': 'NCI', 'Series_contact_address': '1050 Boyles St, BG 560, RM21-102C', 'Series_contact_city': 'Frederick', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21702', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE115nnn/GSE115160/suppl/GSE115160_CHX_21ntRPF_occupancy2.csv.gz', 'Series_citation': 'Colin Chih-Chien Wu, Boris Zinshteyn, Karen A. Wehner, Rachel Green. High-Resolution Ribosome Profiling Defines Discrete Ribosome Elongation States and Translational Regulation during Cellular Stress. Molecular Cell 2019. doi:10.1016/j.molcel.2018.12.009', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP149445'}\n",
      "227 PRJNA473989 GSE115161 {'Series_title': 'Regulation of Translation Elongation Revealed by Ribosome Profiling [Dataset_4]', 'Series_geo_accession': 'GSE115161', 'Series_status': 'Public on Jan 27 2019', 'Series_submission_date': 'May 31 2018', 'Series_last_update_date': 'Nov 24 2020', 'Series_pubmed_id': '31858614', 'Series_summary': 'Ribosomes undergo substantial conformational changes during translation elongation to accommodate incoming aminoacyl-tRNAs and translocate along the mRNA template. We used multiple elongation inhibitors and chemical probing to define ribosome conformational states corresponding to different sized ribosome-protected mRNA fragments (RPFs) generated by ribosome profiling. We show using various genetic and environmental perturbations that the previously identified 20-22 nucleotide (nt) RPFs correspond predominantly to ribosomes in a pre-accommodation state with an open 40S ribosomal A site while the classical 27-29 nt fragments correspond to ribosomes in a pre-translocation state with an occupied 40S ribosomal A site. These distinct ribosome conformational states revealed by ribosome profiling are seen in all eukaryotes tested including fungi, worms and mammals. This high-resolution ribosome profiling approach reveals the anticipated Rck2-dependent inhibition of translocation through eEF2 phosphorylation during hyperosmotic stress. These same approaches reveal a strong translation elongation arrest during oxidative stress where the ribosomes are trapped in a pre-translocation state, but in this case the translational arrest is independent of Rck2-driven eEF2 phosphorylation. These results provide new insights and approaches for defining the molecular events that impact translation elongation throughout biology.', 'Series_overall_design': '24 biological samples are included for ribosome footprinting samples. These include HeLa, MB-MDA-231 and yeast cells and C. elegans embryos.', 'Series_type': 'Other', 'Series_contributor': 'Rachel,,Green', 'Series_sample_id': 'GSM3442058', 'Series_contact_name': 'Colin,,Wu', 'Series_contact_department': 'RNA Biology Laboratory', 'Series_contact_institute': 'NCI', 'Series_contact_address': '1050 Boyles St, BG 560, RM21-102C', 'Series_contact_city': 'Frederick', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21702', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE115nnn/GSE115161/suppl/GSE115161_RAW.tar', 'Series_citation': 'Colin Chih-Chien Wu, Boris Zinshteyn, Karen A. Wehner, Rachel Green. High-Resolution Ribosome Profiling Defines Discrete Ribosome Elongation States and Translational Regulation during Cellular Stress. Molecular Cell 2019. doi:10.1016/j.molcel.2018.12.009', 'Series_platform_id': 'GPL18245', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP149446'}\n",
      "228 PRJNA474431 GSE115265 {'Series_title': 'Genome-wide profiling of translation in Oikopleura dioica during growth arrest and recovery', 'Series_geo_accession': 'GSE115265', 'Series_status': 'Public on Nov 08 2019', 'Series_submission_date': 'Jun 04 2018', 'Series_last_update_date': 'Dec 10 2019', 'Series_pubmed_id': '31783727', 'Series_summary': 'Under crowded, nutrient-limiting conditions,\\xa0growth in the marine chordate O. dioica\\xa0arrests until favorable conditions return. We profiled translation genome-wide using ribosome profiling in O. dioica during growth arrest and growth arrest recovery. We found that initial recovery is independent of nutrient-responsive,\\xa0trans-spliced genes, suggesting that animal density is the primary trigger for the resumption of development in this species.', 'Series_overall_design': 'Ribosome-protected RNA fragments and total RNA extracted from animals during growth arrest and half an hour after release from arrest in two biological replicates sequenced on two Illumina flow cells', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Eric,,Thompson', 'Series_sample_id': 'GSM3172874', 'Series_contact_name': 'Eric,,Thompson', 'Series_contact_email': 'Eric.Thompson@bio.uib.no', 'Series_contact_phone': '+47 55 58 43 46', 'Series_contact_institute': 'Sars International Centre for Marine Molecular Biology', 'Series_contact_address': 'Thormøhlensgt. 55', 'Series_contact_city': 'Bergen', 'Series_contact_zip/postal_code': 'N-5008', 'Series_contact_country': 'Norway', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE115nnn/GSE115265/suppl/GSE115265_count_matrix.txt.gz', 'Series_platform_id': 'GPL25139', 'Series_platform_taxid': '34765', 'Series_sample_taxid': '34765', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP149663'}\n",
      "229 PRJNA474715 GSE115366 {'Series_title': 'Global proteome remodeling during ER stress involves Hac1-driven expression of long undecoded transcript isoforms', 'Series_geo_accession': 'GSE115366', 'Series_status': 'Public on Aug 26 2018', 'Series_submission_date': 'Jun 05 2018', 'Series_last_update_date': 'Mar 11 2019', 'Series_pubmed_id': '30016623', 'Series_summary': 'Cellular stress response pathways often require transcription-based activation of gene expression to promote cellular adaptation. However, whether general mechanisms exist for stress-responsive gene down-regulation is less clear. A recently defined gene regulatory mechanism enables both up- and down-regulation of protein levels for distinct gene sets by the same transcription factor (TF) via coordinated induction of canonical mRNAs and long undecoded transcript isoforms (LUTIs). We analyzed deep, parallel gene expression datasets to determine whether this mechanism contributes to the conserved Hac1-driven branch of the unfolded protein response (UPRER). Indeed, we found Hac1-dependent protein down-regulation that accompanied the well-characterized up-regulation of ER-related proteins that typifies UPRER activation. Proteins down-regulated by Hac1-driven LUTIs include those with electron transport chain (ETC) function.  Aerobic respiration also appears dampened during the UPRER, and abrogated ETC function improves the fitness of UPRER-activated cells, suggesting functional importance of LUTI regulation during the UPRER. We conclude that the UPRER involves large-scale proteome remodeling, mediated in part by Hac1-induced LUTIs, and that this mechanism enables coordination of up- and down-regulation of gene expression during this stress response.', 'Series_overall_design': 'Sequencing of mRNA molecules and ribosome footprints to identify regulated transcript isoform changes that result in apparent translational regulation. In short, two major experiments are included here, one for parallel mRNA-seq and ribosome profiling of cells with or without UPR induction by either DTT or Tm, with or without deletion of HAC1. The other experiment investigates the examines the same type of measurements for cells that are either WT or depleted for Hac1 and treated with DTT.', 'Series_type': 'Other', 'Series_contributor': 'Kelsey,M,Van Dalfsen', 'Series_sample_id': 'GSM3176653', 'Series_contact_name': 'Gloria,,Brar', 'Series_contact_email': 'gabrar@berkeley.edu', 'Series_contact_department': 'MCB', 'Series_contact_institute': 'UC Berkeley', 'Series_contact_address': '626 Barker Hall', 'Series_contact_city': 'Berkeley', 'Series_contact_zip/postal_code': '94720', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE115nnn/GSE115366/suppl/GSE115366_Dec2016_qexpr_rpkm_MERGE.txt.gz', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP149818'}\n",
      "230 PRJNA475144 GSE115483 {'Series_title': 'Cell type specific profiling of alternative translation identifies novel protein isoforms in the mouse brain', 'Series_geo_accession': 'GSE115483', 'Series_status': 'Public on Jan 14 2019', 'Series_submission_date': 'Jun 07 2018', 'Series_last_update_date': 'Apr 15 2019', 'Series_pubmed_id': '30650354', 'Series_summary': 'Translation canonically begins at a single AUG and terminates at the stop codon, generating one protein species per transcript. However, some transcripts may use alternative initiation sites or sustain translation past their stop codon, generating multiple protein isoforms. Through other mechanisms such as alternative splicing, both neurons and glia exhibit remarkable transcriptional diversity, and these other forms of post-transcriptional regulation are impacted by neural activity and disease. Here, using ribosome footprinting, we demonstrate that alternative translation is likewise abundant in the central nervous system and modulated by stimulation and disease. First, in neuron/glia mixed cultures we identify hundreds of transcripts with alternative initiation sites and confirm the protein isoforms corresponding to a subset of these sites by mass spectrometry. Many of them modulate their alternative initiation in response to KCl stimulation, indicating activity-dependent regulation of this phenomenon. Next, we detect several transcripts undergoing stop codon readthrough thus generating novel C-terminally-extended protein isoforms in vitro. Further, by coupling Translating Ribosome Affinity Purification to ribosome footprinting to enable cell-type specific analysis in vivo, we find that several of both neuronal and astrocytic transcripts undergo readthrough in the mouse brain. Functional analyses of one of these transcripts, Aqp4, reveals readthrough confers perivascular localization, indicating readthrough can be a conserved mechanism to modulate protein function. Finally, we show that AQP4 readthrough is disrupted in multiple gliotic disease models. Our study demonstrates the extensive and regulated use of alternative translational events in the brain and indicates that some of these events alter key protein properties. [NOTE: this GEO series only contains in vivo data]', 'Series_overall_design': 'For in vivo experiments (4 samples in total): 2 replicates were prepared from P21 Snap25::eGFP-RpL10a mouse brains for profiling neurons, 2 replicates from Aldh1l1:: eGFP-RpL10a mouse brains for profiling astrocytes. All samples were uniquely indexed and pooled across two lanes.  For each sample, the uploaded fastq files are union of both lanes.', 'Series_type': 'Other', 'Series_contributor': 'Joseph,D,Dougherty', 'Series_sample_id': 'GSM3178973', 'Series_contact_name': 'Joseph,D,Dougherty', 'Series_contact_department': 'Genetics and Psychiatry', 'Series_contact_institute': 'Washington University', 'Series_contact_address': '4566 Scott Ave, box 8232', 'Series_contact_city': 'St Louis', 'Series_contact_state': 'Missouri', 'Series_contact_zip/postal_code': '63110', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE115nnn/GSE115483/suppl/GSE115483_RAW.tar', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP150017'}\n",
      "231 PRJNA475266 GSE115526 {'Series_title': 'Molecular profiling of reticular gigantocellularis neurons', 'Series_geo_accession': 'GSE115526', 'Series_status': 'Public on Nov 01 2018', 'Series_submission_date': 'Jun 08 2018', 'Series_last_update_date': 'May 21 2019', 'Series_pubmed_id': '29967172', 'Series_summary': 'we used technique that allows the molecular characterization of particular neuronal subpopulations based on their neuroanatomical projections and the locations of their cell bodies.  This ‘retro-TRAP’ (translating ribosome affinity purification from retrogradely labeled neurons) approach relies on viral injection into an anatomical area targeted by the neurons of interest, followed by selective precipitation of ribosomes from retrogradely labeled cell bodies, and subsequent RNAseq analysis.', 'Series_overall_design': 'By comparing the mRNAs enriched in the NGC neurons which are retrogradely labeled due to viral injection into central thalamus, to gene expression of non-labeled surrounding cells in NGC, and then performing a comprehensive bioinformatics analysis of these results, we were able to identify genes enriched in these cells. This procedure allowed us to highlight genes and pathways unique to these neurons with projections ascending to thalamus, as compared to other cells in reticular NucleusGigantocellularis.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Yupu,,Liang', 'Series_sample_id': 'GSM3181523', 'Series_contact_name': 'Inna,,Tabansky', 'Series_contact_email': 'tabansky@post.harvard.edu', 'Series_contact_laboratory': 'Pfaff', 'Series_contact_department': 'Neurobiology and Behavior', 'Series_contact_institute': 'The Rockefeller University', 'Series_contact_address': '1230 York Ave, Box 275', 'Series_contact_city': 'New York', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '10065', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE115nnn/GSE115526/suppl/GSE115526_counts.matrix.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP150080'}\n",
      "232 PRJNA476979 GSE116082 {'Series_title': 'The androgen receptor regulates a druggable translational regulon in advanced prostate cancer [ribosome profiling]', 'Series_geo_accession': 'GSE116082', 'Series_status': 'Public on Jul 31 2019', 'Series_submission_date': 'Jun 20 2018', 'Series_last_update_date': 'Aug 03 2019', 'Series_pubmed_id': '31366581', 'Series_summary': 'The recent advent of highly potent inhibitors of the androgen receptor and androgen biosynthesis has had the unfortunate iatrogenic effect of fueling new lethal prostate cancer phenotypes in patients. In particular, non-neuroendocrine androgen receptor-low castration resistant prostate cancer (AR low CRPC) is increasing in occurrence amongst patients and is uniformly fatal. The mechanisms that promote this phenotype remain poorly understood. Through molecular studies of murine and human models of AR low CRPC, we have identified a new functional interface between the androgen receptor and the translation initiation complex inhibitor 4EBP1. AR directly regulates 4EBP1 expression which translationally represses eIF4F complex formation and a pro-proliferation program. In the context of AR low prostate cancer, de-repression of translation initiation drives the aberrant expression of the pro-proliferation regulon which fuels uncontrolled cell growth. Genetic and pharmacologic inhibition of the downstream eIF4F translation initiation complex reverses the proliferation phenotype in vitro and in vivo. These findings reveal a new previously unrecognized druggable nexus which functionally link the processes of mRNA transcription and translation initiation in a rapidly emerging class of advanced lethal prostate cancer', 'Series_overall_design': 'Ribosome profiling of prostate tissue from castrated or intact PB-Cre4;PTENLoxP/LoxP mice.', 'Series_type': 'Other', 'Series_contributor': 'Sonali,,Arora', 'Series_sample_id': 'GSM3208719', 'Series_contact_name': 'Sonali,,Arora', 'Series_contact_email': 'sarora@fredhutch.org', 'Series_contact_institute': 'FHCRC', 'Series_contact_address': '1100 Fairview Ave N,', 'Series_contact_city': 'Seattle', 'Series_contact_state': 'WA', 'Series_contact_zip/postal_code': '98109', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE116nnn/GSE116082/suppl/GSE116082_RAW.tar', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP151006'}\n",
      "233 PRJNA477831 GSE116233 {'Series_title': 'Optimization of Ribosome Profiling Using Brain Tissue from Fragile X Mice', 'Series_geo_accession': 'GSE116233', 'Series_status': 'Public on Mar 15 2019', 'Series_submission_date': 'Jun 25 2018', 'Series_last_update_date': 'Mar 26 2019', 'Series_pubmed_id': '30590705', 'Series_summary': 'Dysregulated protein synthesis is a major underlying cause of many neurodevelopmental diseases such as Fragile X Syndrome. A very robust technique is required to capture subtle but biologically significant differences in neurological disorders. Ribosome profiling, which is based on deep sequencing of mRNA fragments protected from ribonuclease digestion by ribosomes, is a powerful tool to study translational control. However, it has been mainly applied to rapidly dividing cells where translation is robust and where large amounts of starting material are readily available. The application of ribosome profiling to low-input brain tissue where translation is modest and where gene expression changes between genotypes are expected to be small has not been carefully evaluated. Using hippocampal tissue from wide type and fragile X mental retardation 1 (Fmr1) knockout mice, we show that variable RNase digestion can lead to significant sample batch effects. We also establish GC content and ribosome footprint length as quality control metrics for ribonuclease digestion. We performed ribonuclease titration experiments for low-input samples to identify optimal conditions for this critical step that is often improperly conducted. Our data reveal that optimal RNase digestion is essential to ensure high quality and reproducibility of ribosome profiling especially for low-input brain tissue.', 'Series_overall_design': 'Ribosome profiling libraries of wild type mouse hippocampus were generated with different concentrations of ribonucleases and sequenced with Illumina Nextseq500. 5 experimental and 51 QC samples.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Joel,D,Richter', 'Series_sample_id': 'GSM3397009', 'Series_contact_name': 'Botao,,Liu', 'Series_contact_laboratory': 'Joel Richter Lab', 'Series_contact_department': 'Program in Molecular Medicine', 'Series_contact_institute': 'University of Massachusetts Medical School', 'Series_contact_address': '373 Plantation Street', 'Series_contact_city': 'Worcester', 'Series_contact_state': 'Massachusetts', 'Series_contact_zip/postal_code': '01605', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE116nnn/GSE116233/suppl/GSE116233_rnase_titration_reads_overlap_CDS_length.tar.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP151279'}\n",
      "234 PRJNA480786 GSE116981 {'Series_title': 'Cancer associated mutants of eIF1A impair Rps3/Rps10 binding and enhance scanning of cell cycle genes [Ribo-Seq]', 'Series_geo_accession': 'GSE116981', 'Series_status': 'Public on Jul 13 2018', 'Series_submission_date': 'Jul 12 2018', 'Series_last_update_date': 'Mar 21 2019', 'Series_pubmed_id': '30420357', 'Series_summary': \"The Ribo-seq analysis demonstrated that eIF1A is predominantly essential for translation of genes with long 5'UTR genes including cell proliferation and cell cycle progression genes. eIF1A depletion causes broad stimulation of initiation in 5’UTRs at near-cognate AUG codons that diminshes the translation initiation fidelity\", 'Series_overall_design': 'Ribo-seq analysis was performed either in contol or eIF1A knockdown MEFs', 'Series_type': 'Other', 'Series_contributor': 'Rivka,,Dikstein', 'Series_sample_id': 'GSM3266821', 'Series_contact_name': 'Dena,,Leshkowitz', 'Series_contact_email': 'dena.leshkowitz@weizmann.ac.il', 'Series_contact_department': 'Bioinformatics Unit, Life Sciences Core Facilities', 'Series_contact_institute': 'Weizmann Institute of Science', 'Series_contact_address': 'P.O.B. 26', 'Series_contact_city': 'Rehovot', 'Series_contact_zip/postal_code': '76100', 'Series_contact_country': 'Israel', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE116nnn/GSE116981/suppl/GSE116981_Ribo-seq_norm_counts.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP153150'}\n",
      "235 PRJNA480907 GSE117043 {'Series_title': 'Peptides encoded by 5’ untranslated regions mediate pain sensitization', 'Series_geo_accession': 'GSE117043', 'Series_status': 'Public on Feb 01 2020', 'Series_submission_date': 'Jul 12 2018', 'Series_last_update_date': 'Feb 01 2020', 'Series_summary': 'Nociceptors are neurons responsible for the detection of pain producing stimuli. Persistent changes in their activity, termed plasticity, benefit survival through injury avoidance and are regulated on a translational basis. Yet, the mRNAs whose translation facilitates plasticity are unknown. Here, we apply ribosome profiling to dorsal root ganglion and identify a small number of transcripts that are selectively translated in response to plasticity mediators. Among them are Arc and Fos, genes implicated in episodic learning in the brain. We demonstrate that the ribosomal S6 kinase 1 is responsible for their production in nociceptors. Blocking S6 driven translation also reduces pain associated behavioral responses in vivo. In addition to translation of coding regions of mRNA, we detect pervasive ribosome occupancy in 5’ untranslated regions. We find that peptides encoded by open reading frames in the 5’ untranslated regions of Calca and Egr2 increase neuronal excitability in vitro and are sufficient to induce pain-like behaviors in vivo. Together, our findings uncover new targets of translational control that drive changes in plasticity and suggest new mechanisms for targeted pain therapeutics that disrupt pain signaling.', 'Series_overall_design': 'We applied ribosome profiling to DRG neurons in culture in response to NGF/IL-6 treatment', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Zachary,T,Campbell', 'Series_sample_id': 'GSM3267698', 'Series_contact_name': 'Theodore,J,Price', 'Series_contact_email': 'Theodore.Price@utdallas.edu', 'Series_contact_institute': 'University of Texas at Dallas', 'Series_contact_address': '800 W Campbell Rd.', 'Series_contact_city': 'Richardson', 'Series_contact_state': 'TX', 'Series_contact_zip/postal_code': '75080', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE117nnn/GSE117043/suppl/GSE117043_all_sample.prot_cod.polyA_rmd.str_spc.xlsx', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP153254'}\n",
      "236 PRJNA484185 GSE117998 {'Series_title': 'Protein Syndesmos is a novel RNA binding protein that regulates primary cilia formation [Ribo-seq]', 'Series_geo_accession': 'GSE117998', 'Series_status': 'Public on Sep 26 2018', 'Series_submission_date': 'Aug 01 2018', 'Series_last_update_date': 'Mar 27 2019', 'Series_pubmed_id': '30260431', 'Series_summary': 'We show here that NUDT16L1/TIRR/Syndesmos (SDOS) interacts with another important cancer-linked protein, the chaperone TRAP1, associates with actively translating polyribosomes and represses translation. Moreover, we demonstrate that SDOS binds directly RNA in living cells. Combining individual gene expression profiling, nucleotide cross-linking and immunoprecipitation (iCLIP), and ribosome profiling, we discover several crucial pathways regulated post-transcriptionally by SDOS.', 'Series_overall_design': 'Evaluation of differentially translated transcripts in SDOS-eGFP over eGFP expressing cells', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Danilo,S,Matassa', 'Series_sample_id': 'GSM3316961', 'Series_contact_name': 'Danilo,,Swann Matassa', 'Series_contact_email': 'daniloswann.matassa@unina.it', 'Series_contact_department': 'Molecular Medicine and Medical Biotechnology', 'Series_contact_institute': 'University of Naples “Federico II”', 'Series_contact_address': 'Via Sergio Pansini 5', 'Series_contact_city': 'Napoli', 'Series_contact_zip/postal_code': '80131', 'Series_contact_country': 'Italy', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE117nnn/GSE117998/suppl/GSE117998_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP156197'}\n",
      "237 PRJNA486172 GSE118597 {'Series_title': 'Transcriptome and translatome of Streptomyces avermitilis MA-4680', 'Series_geo_accession': 'GSE118597', 'Series_status': 'Public on Mar 03 2020', 'Series_submission_date': 'Aug 15 2018', 'Series_last_update_date': 'Jan 31 2022', 'Series_pubmed_id': '35062881', 'Series_summary': 'The gram-positive bacterium, Streptomyces avermitilis holds industrial importance, which produces widely used anthelmintic agent, avermectin. Furthermore, S. avermitilis is generally considered as a prominent heterologous gene expression host for diverse secondary metabolites biosynthesis. However, despite of its industrial importance, it largely remains unknown how its genome is organized and regulated for timely gene expression. Here, we determined 1,601 transcription units (TU) encoded in its genome using the integrated analysis of high-throughput sequencing data including dRNA-Seq, Term-Seq, RNA-Seq, and Ribo-Seq. In addition to TU cataloguing, these information-rich results also revealed the presence of diverse regulatory elements for the transcriptional and translational control of individual TU, such as promoters, 5¢-UTRs, terminators, 3¢-UTRs, and riboswitches. The conserved promoter sequences for transcription initiation were identified from 2,361 transcription start sites as 5¢-TANNNT and 5¢-TGAC for -10 and -35 elements, respectively. Interestingly, the -35 element and spacer length between them were critical for transcriptional regulation of functionally distinct genes. Total 2,017 transcription termination sites were detected from Term-Seq analysis, revealing that stem structure formation is a prerequisite for transcription termination and that Rho-independent termination prevails in S. avermitilis. Lastly, the TU architecture suggests the presence of novel small RNAs and cis-regulatory elements in the genome. Our findings will serve as invaluable resources for comprehensive understanding on regulatory features of S. avermitilis. Moreover, it is anticipated the elevation of its potential as the heterologous expression host for diverse secondary metabolite biosynthesis.', 'Series_overall_design': \"Profiles of 5' termini of primary transcripts, 3' termini of whole transcripts, whole transcripts and ribosome protected RNA fragements of Streptomyces avermitilis were generated by deep sequencing using Illumina Hi-Seq 2500\", 'Series_type': 'Other', 'Series_contributor': 'Byung-Kwan,,Cho', 'Series_sample_id': 'GSM3334109', 'Series_contact_name': 'Yongjae,,Lee', 'Series_contact_email': 'ahrk7247@kaist.ac.kr', 'Series_contact_institute': 'KAIST', 'Series_contact_address': '291 Daehak-ro', 'Series_contact_city': 'Daejeon', 'Series_contact_zip/postal_code': 'ASI|KR|KS015|DAEJEON', 'Series_contact_country': 'South Korea', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE118nnn/GSE118597/suppl/GSE118597_Term.txt.gz', 'Series_platform_id': 'GPL25453', 'Series_platform_taxid': '33903', 'Series_sample_taxid': '33903', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP158023'}\n",
      "238 PRJNA488955 GSE119365 {'Series_title': 'Iron and Heme Coordinate Erythropoiesis through HRI-Mediated Regulation of Protein Translation and Gene Expression', 'Series_geo_accession': 'GSE119365', 'Series_status': 'Public on Sep 03 2018', 'Series_submission_date': 'Sep 02 2018', 'Series_last_update_date': 'Jun 18 2019', 'Series_pubmed_id': '31033440', 'Series_summary': 'Iron and heme play central roles in red blood cell production. However, the mechanisms by which iron and heme levels coordinate erythropoiesis remain incompletely understood. HRI is a heme-regulated kinase that controls translation by phosphorylating eIF2a. Here, we investigate the global impact of iron, heme and HRI on protein translation in vivo in murine primary erythroblasts using ribosome profiling. By defining the underlying changes in translation during iron and HRI deficiencies, we validate known regulators of this process, including Atf4, and identify novel pathways such as co-regulation of ribosomal protein mRNA translation. Surprisingly, we found that heme and HRI pathways, but not iron-regulated pathways, mediate the major protein translational and transcriptional responses to iron deficiency in erythroblasts in vivo and thereby identify previously unappreciated regulators of erythropoiesis. Our genome-wide study uncovers the major impact of the HRI-mediated integrated stress response for the adaptation to iron deficiency anemia.', 'Series_overall_design': 'Primary basophilic erythroblasts from E14.5 fetal livers (FLs) of Wild-type (Wt) and Hri-/- mice maintained under +Fe or -Fe conditions were sorted. FLs from embryos of the same mother were pooled before sorting. Two and three biological replicas of each condition were collected for preparations of Ribo-seq and mRNA-seq libraries, respectively.', 'Series_type': 'Other', 'Series_contributor': 'Jane-Jane,,Chen', 'Series_sample_id': 'GSM3371797', 'Series_contact_name': 'Shuping,,Zhang', 'Series_contact_institute': 'Massachusetts Institute of Technology', 'Series_contact_address': '77 Massachusetts Avenue', 'Series_contact_city': 'Cambridge', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02139', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE119nnn/GSE119365/suppl/GSE119365_RAW.tar', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP159428'}\n",
      "239 PRJNA489275 GSE119454 {'Series_title': 'Assembly and functionality of the ribosome with tethered subunits', 'Series_geo_accession': 'GSE119454', 'Series_status': 'Public on Feb 25 2019', 'Series_submission_date': 'Sep 04 2018', 'Series_last_update_date': 'Mar 06 2019', 'Series_pubmed_id': '30804338', 'Series_summary': 'Ribo-T is an engineered ribosome whose small and large subunits are tethered together by insertion of circularly permutated 23S rRNA into 16S rRNA [Orelle, C., Carlson, E. D., Szal, T., Florin, T., Jewett, M. C., Mankin, A.S. Protein synthesis by ribosomes with tethered subunits. Nature 524, 119-124 (2015)]. Whether the growth defects are related to problems with Ribo-T functionality is unclear. The translation in Ribo-T cells was examined by ribosome profiling (Ribo-seq) and RNA sequencing (RNA-seq). Besides the control cells carrying dissociable WT ribosome (SQ171),  RIBO-T cells are the original SQ171 strain transformed with pRibo-T plasmid and cured of the plasmid carrying wt rRNA that was evolved to have improved growth characteristics. The Ribo-T strain carries two mutations: a nonsense mutation in the Leu22 codon of the ybeX gene and a missense mutation in codon 549 of the rpsA gene encoding ribosomal protein S1. Analysis of Ribo-T performance by ribosome profiling showed a notably higher ribosome occupancy of the start codons and somewhat increased occupancy at the stop codons of many genes, suggesting that subunit tethering specifically impairs the initiation and termination stages of translation.', 'Series_overall_design': 'Ribosome profiling and RNA-seq was performed on Escherichia coli Ribo-T (a derivative of the strain SQ171, 2 biological replicates) and compared to a SQ171 control cells carrying WT ribosomes (2 biological replicates).', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Alexander,S,Mankin', 'Series_sample_id': 'GSM3553916', 'Series_contact_name': 'Adam,John,Hockenberry', 'Series_contact_email': 'adam.hockenberry@utexas.edu', 'Series_contact_laboratory': 'Institute for Cellular and Molecular Biology', 'Series_contact_department': 'Department of Integrative Biology', 'Series_contact_institute': 'The University of Texas at Austin', 'Series_contact_address': '2500 Speedway, A4800, MBB 3.232', 'Series_contact_city': 'Austin', 'Series_contact_state': 'TX', 'Series_contact_zip/postal_code': '78712', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE119nnn/GSE119454/suppl/GSE119454_adapters.fasta.gz', 'Series_platform_id': 'GPL21222', 'Series_platform_taxid': '562', 'Series_sample_taxid': '562', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP159555'}\n",
      "240 PRJNA489606 GSE119567 {'Series_title': 'Nutrient sensing by the intestinal epithelium orchestrates mucosal antimicrobial defense via translational control of Hes1', 'Series_geo_accession': 'GSE119567', 'Series_status': 'Public on Mar 25 2019', 'Series_submission_date': 'Sep 06 2018', 'Series_last_update_date': 'Jun 25 2019', 'Series_pubmed_id': '31053533', 'Series_summary': 'To investigate whether intestinal Hes1 expression was translationally regulated during nutrient deprivation, we performed ribosome profiling of ileum from fed and fasted mice.', 'Series_overall_design': 'Examination of mRNA translation in ileum of mice fed or fasted for 48 hrs', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Rongyao,,Huang', 'Series_sample_id': 'GSM3377786', 'Series_contact_name': 'Shaonan,,Liang', 'Series_contact_email': 'lsn15@mails.tsinghua.edu.cn', 'Series_contact_phone': '15701209375', 'Series_contact_laboratory': 'Xiaoyu Hu Lab', 'Series_contact_department': 'School of Medicine', 'Series_contact_institute': 'Institute for Immunology', 'Series_contact_address': 'Room D311, Medical Science Building, Tsinghua University', 'Series_contact_city': 'Beijing', 'Series_contact_state': 'Beijing', 'Series_contact_zip/postal_code': '100084', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE119nnn/GSE119567/suppl/GSE119567_RAW.tar', 'Series_platform_id': 'GPL21103', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP159824'}\n",
      "241 PRJNA490651 GSE119903 {'Series_title': 'Genome-wide transcriptome analysis of brain region (ventral part of telenchephalon, hypothalamus and pituitary) in medaka [Ribo-Seq]', 'Series_geo_accession': 'GSE119903', 'Series_status': 'Public on Apr 08 2019', 'Series_submission_date': 'Sep 13 2018', 'Series_last_update_date': 'Jul 08 2019', 'Series_pubmed_id': '30962562', 'Series_summary': 'To examine the protein-coding potential of transcripts, we performed ribosomal profiling analysis. We compared the RNA-seq and ribosomal protected fragments profiles (Ribo-seq) in the ventral part of telenchephalon, hypothalamus and pituitary by deep sequencing.', 'Series_overall_design': 'Compasison of RNA-seq and Ribo-seq profiles in the ventral part of telenchephalon, hypothalamus and pituitary in medaka.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Takashi,,Yoshimura', 'Series_sample_id': 'GSM3386946', 'Series_contact_name': 'Takashi,,Yoshimura', 'Series_contact_email': 'takashiy@agr.nagoya-u.ac.jp', 'Series_contact_institute': 'Nagoya University', 'Series_contact_address': 'Furo-cho, Chikusa-ku', 'Series_contact_city': 'Nagoya', 'Series_contact_zip/postal_code': '4648601', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE119nnn/GSE119903/suppl/GSE119903_FPKM_RNAseq_Riboseq.xlsx', 'Series_platform_id': 'GPL23308', 'Series_platform_taxid': '8090', 'Series_sample_taxid': '8090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP161676'}\n",
      "242 PRJNA491747 GSE120159 {'Series_title': 'Targeting the eIF4A dependent translation of KRAS and other oncoproteins in pancreatic cancer', 'Series_geo_accession': 'GSE120159', 'Series_status': 'Public on Mar 03 2021', 'Series_submission_date': 'Sep 18 2018', 'Series_last_update_date': 'Mar 03 2021', 'Series_summary': 'New and effective therapeutics are urgently needed for the treatment of pancreatic ductal adenocarcinoma (PDAC). The eIF4A/DDX2 RNA helicase drives translation of mRNAs with highly structured 5’UTRs. The natural compound silvestrol and synthetic analogues are potent and selective inhibitors of eIF4A1/2 that show promising activity in models of hematological malignancies. Here, we show silvestrol analogues have nanomolar activity against PDAC cell lines and organoids in vitro. Moreover, we see single agent activity in the KRAS/p53 mouse PDAC model and also against PDAC xenografts and primary, patient derived PDAC tumors. These therapeutic effects occur at non-toxic dose levels. Transcriptome-wide ribosome profiling, analyses of protein and gene expression, and translation reporter studies reveal that eIF4A inhibitors block an oncogenic translation program in PDAC cells that includes G-quadruplex containing mRNAs such as KRAS, MYC, YAP1, MET, SMAD3, TGFβ and others. Together, our data indicate that pharmacological inhibition of eIF4A disrupts oncoprotein production and shows efficacy across several PDAC models.', 'Series_overall_design': 'We performed ribosome footprinting and RNA sequencing on six samples of PANC1 cells including three untreated samples (DMSO) and three samples that were collected 45 minutes after treatment with CR31B (25 nM). DMSO treated samples are indicated as CT and CR31B treated samples are indicate as S.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Hans-Guido,,Wendel', 'Series_sample_id': 'GSM3395021', 'Series_contact_name': 'Kamini,,Singh', 'Series_contact_email': 'kamini.singh@einsteinmed.edu', 'Series_contact_phone': '7184302466', 'Series_contact_laboratory': 'Kamini Singh', 'Series_contact_department': 'Molecular Pharmacology', 'Series_contact_institute': 'Albert Einstein College of Medicine', 'Series_contact_address': '1300 Morris Park Ave, Golding 203', 'Series_contact_city': 'Bronx', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '10461', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE120nnn/GSE120159/suppl/GSE120159_CS_master_motif.txt.gz', 'Series_citation': 'https://doi.org/10.1158/0008-5472.CAN-20-2929', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP162072'}\n",
      "243 PRJNA493293 GSE120533 {'Series_title': 'In vivo RNAi mediated knockdown of translation initiation factor eIF3m is associated with alterations in ribosome biogenesis [RNAseq/Riboseq]', 'Series_geo_accession': 'GSE120533', 'Series_status': 'Public on Jul 29 2020', 'Series_submission_date': 'Sep 26 2018', 'Series_last_update_date': 'Jul 31 2020', 'Series_pubmed_id': '31855834', 'Series_summary': 'Translation is a fundamental biological process, which defines every aspect of cell physiology. Dysregulation of protein synthesis is associated with the diseases, such as ribosomopathies, diabetes, and cancer. To examine translation dysregulation in vivo, we employed RNA interference to knockdown m-subunit of the translation initiation factor eIF3 exclusively in mouse liver. We observed an increase in the number of monosomes and multiple signs of impaired translation. We further characterized the early cellular response using transcriptome sequencing, ribosome profiling, whole proteome, and phosphoproteome analyses. The major response of hepatocytes to eIF3m deficiency was associated with the changes in mRNA expression, with almost no effect on translation efficiency for particular mRNAs. The transcription changes fell into two main categories: ribosome biogenesis (increased transcription of ribosomal proteins, dephosphorylation of eIF2α, and inhibition of rRNA processing) and cell metabolism (lipid, amino acid, nucleic acid, and drug metabolism). Overall, this work uncovers a new mode of regulation of protein synthesis mediated by the inhibition of translation initiation. It also highlights advantages of RNAi-based in vivo approach for studying regulatory network associated with essential factors involved in mammalian translation.', 'Series_overall_design': 'Please note that GSM3327719-GSM3327724 sample raw data (in GSE118394) have been re-analyzed in the currect study and therefore, duplicated sample records (GSM3402655-GSM3402660) have been created for the convenient retrieval of the complete raw data from SRA.', 'Series_type': 'Other', 'Series_contributor': 'Victor,,Koteliansky', 'Series_sample_id': 'GSM3402674', 'Series_contact_name': 'Charles,Arthur,Whittaker', 'Series_contact_email': 'charliew@mit.edu', 'Series_contact_institute': 'Koch Institute', 'Series_contact_address': '77 Mass Ave 76-189', 'Series_contact_city': 'Cambridge', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02152', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE120nnn/GSE120533/suppl/GSE120533_esmekal_counts.txt.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP162708'}\n",
      "244 PRJNA494404 GSE120762 {'Series_title': 'The Translation of Non-Canonical Open Reading Frames Controls Mucosal Immunity', 'Series_geo_accession': 'GSE120762', 'Series_status': 'Public on Dec 14 2018', 'Series_submission_date': 'Oct 02 2018', 'Series_last_update_date': 'Mar 19 2019', 'Series_pubmed_id': '30542152', 'Series_summary': 'We describe the identification of novel non-canonical ORFs in murine macrophages', 'Series_overall_design': 'Using RNAseq, RiboTag RNA seq and Ribosome Profiling in Murine macrophages untreated and stimulated with LPS', 'Series_type': 'Other', 'Series_contributor': 'Lina,,Kroehling', 'Series_sample_id': 'GSM3414922', 'Series_contact_name': 'Xaralabos,,Varelas', 'Series_contact_institute': 'Boston Unviersity', 'Series_contact_address': '91 E Concord St', 'Series_contact_city': 'Boston', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02118', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE120nnn/GSE120762/suppl/GSE120762_Supplementary_Table_1.xlsx', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP163147'}\n",
      "245 PRJNA497255 GSE121391 {'Series_title': 'Genome-wide analysis of eIF4G-RNA interaction in CDK12-depleted cells (Ribo-Seq)', 'Series_geo_accession': 'GSE121391', 'Series_status': 'Public on Jun 12 2019', 'Series_submission_date': 'Oct 17 2018', 'Series_last_update_date': 'Mar 28 2022', 'Series_summary': 'Methods: U2OS mRNA profiles of control siRNA(siControl) and siCDK12 were generated by deep sequencing, in triplicate, using Illumina. The sequence reads that passed quality filters were analyzed at the transcript isoform level. Total RNA-Seq was used for normalzation of RIP-Seq signals.Results: We report the application of RNA-immunoprecipitation-based sequencing for high-throughput profiling of mRNA abundence in mammalian cells. We generated genome-wide views of translation rate in CDK12-depleted U2OS cells in comparion to mTOR inhibitor, rapamycin treatment. We find that significant overlap and high corelation of change in eIF4G-associated mRNAs between two condition, rapamycin treatment and siCDK12 transfected cells. This study provides a role of CDK12 in translation control of  mammalian cell.', 'Series_overall_design': 'Ribosome footprinting followed by sequencing (Ribo-Seq) in 2 different treatments in U2OS cells.', 'Series_type': 'Other', 'Series_contributor': 'Kathrine,,Jones', 'Series_sample_id': 'GSM3434473', 'Series_contact_name': 'April,Elizabeth,Williams', 'Series_contact_email': 'apriljack06@gmail.com, awilliams@salk.edu', 'Series_contact_phone': '7345461645', 'Series_contact_department': 'IGC', 'Series_contact_institute': 'Salk Institute for Biological Studies', 'Series_contact_address': '10010 N Torrey Pines Rd', 'Series_contact_city': 'San Diego', 'Series_contact_state': 'California', 'Series_contact_zip/postal_code': '92037', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE121nnn/GSE121391/suppl/GSE121391_xtail_siCtrl_siCDK12_rep2.xlsx', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP165989'}\n",
      "246 PRJNA499089 GSE121933 {'Series_title': 'RNA m6A-Ythdf1 in dendritic cells triggers anti-tumor immunity (MeRIP-Seq, RIP-Seq and Ribo-Seq in Flt3L-DCs)', 'Series_geo_accession': 'GSE121933', 'Series_status': 'Public on Dec 29 2018', 'Series_submission_date': 'Oct 29 2018', 'Series_last_update_date': 'Mar 01 2019', 'Series_pubmed_id': '30728504', 'Series_summary': 'Emerging evidence emphasizes the important role of tumor neoantigen in generating the spontaneous antitumor immune response and predicting the clinical response to immunotherapies. Despite the presence of numerous neoantigens, complete tumor elimination rarely occurs in majority of patients due to failures in mounting a sufficient and lasting antitumor immunity. Here we show that the durable neoanitgen-specific immunity is regulated by a m6A-binding protein, Ythdf1. In contrast to wild-type mice, Ythdf1-deficient (Ythdf1-/-) mice generate more antigen-specific CD8+ T cell response for persistent tumor control. Loss of Ythdf1 in dendritic cell (DC) results in an enhanced cross-presentation of tumor antigen and cross-priming of CD8+ T cell in vivo. To confirm our observations, we performed Ribo-Seq to analyze the translational efficiency of genes in DCs and performed m6A-seq to locate the m6A sites.', 'Series_overall_design': 'Flt3L-DCs obtained from WT and YTHDF1 KO mice were used to generate MeRIP-Seq, RIP-Seq and Ribo-Seq libraries. Two replicates for each conditions.', 'Series_type': 'Other', 'Series_contributor': 'Chuan,,He', 'Series_sample_id': 'GSM3450277', 'Series_contact_name': 'Dali,,Han', 'Series_contact_email': 'handali294@gmail.com', 'Series_contact_institute': 'Beijing Institute of Genomics, and China National Center for Bioinformation, Chinese Academy of Sciences', 'Series_contact_address': 'Da-Tun Road', 'Series_contact_city': 'Beijing', 'Series_contact_zip/postal_code': '100101', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE121nnn/GSE121933/suppl/GSE121933_RAW.tar', 'Series_platform_id': 'GPL21273', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP167150'}\n",
      "247 PRJNA501871 GSE121973 {'Series_title': 'Profiling the endothelial translatome in vivo using ‘AngioTag’ zebrafish', 'Series_geo_accession': 'GSE121973', 'Series_status': 'Public on Mar 29 2019', 'Series_submission_date': 'Oct 30 2018', 'Series_last_update_date': 'Mar 29 2019', 'Series_summary': 'We profiled the gene expression patterns of undisturbed endothelial cells in living animals using a novel ‘AngioTag’ zebrafish transgenic line that permits isolation of actively translating mRNAs from endothelial cells in their native environment. This transgenic line uses the endothelial cell-specific kdrl promoter to drive expression of an epitope tagged Rpl10a 60S ribosomal subunit protein, allowing for Translating Ribosome Affinity Purification (TRAP) of actively translating endothelial cell mRNAs.  We also collected the whole embryo translatome using the TRAP protocol and a ubiquitously expressed tagged Rpl10a, and the endothelial transcritome by collecting the GFP+ endothelial cells that had the kdrl promoter driving GFP.', 'Series_overall_design': 'Examination of 24hpf zebrafish whole embryo translatome, 24hpf zebrafish endothelial translatome, and 24hpf zebrafish endothelial transciptome', 'Series_type': 'Other', 'Series_contributor': 'Derek,E,Gildea', 'Series_sample_id': 'GSM3452133', 'Series_contact_name': 'Derek,,Gildea', 'Series_contact_email': 'dgildea@mail.nih.gov', 'Series_contact_institute': 'NHGRI/NIH', 'Series_contact_address': '50 South Drive Bldg 50 Rm 5228', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '20892-8004', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE121nnn/GSE121973/suppl/GSE121973_raw_counts.txt.gz', 'Series_platform_id': 'GPL18413', 'Series_platform_taxid': '7955', 'Series_sample_taxid': '7955', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP167240'}\n",
      "248 PRJNA501872 GSE121952 {'Series_title': 'Active ribosome profiling in SETD2 or METTL14 knockdown HepG2 cells', 'Series_geo_accession': 'GSE121952', 'Series_status': 'Public on Feb 21 2019', 'Series_submission_date': 'Oct 29 2018', 'Series_last_update_date': 'Mar 21 2019', 'Series_pubmed_id': '30867593', 'Series_summary': 'SETD2 is the specific methyltransferase of H3K36me3, while METTL14 is a critical subunit of the m6A methyltransferase complex. To evaluate the effect of SETD2 and METTL14 on translation, we conducted robosome profiling in SETD2 and METTL14 knockdown and control HepG2 cells. Our RNA ribosome profiling revealed that depletion of SETD2 and METTL14 resulted in a global reduction in RNA translation and the changes of translation efficiency were correlated between SETD2 and METTL14 knockdown cells.', 'Series_overall_design': 'We performed active ribosome profiling in HepG2 cells with stably expressed SETD2, METTL14 or control shRNA.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Jianjun,,Chen', 'Series_sample_id': 'GSM3450424', 'Series_contact_name': 'Jianjun,,Chen', 'Series_contact_email': 'jianchen@coh.org', 'Series_contact_phone': '6262185591', 'Series_contact_laboratory': 'Chen Lab', 'Series_contact_department': 'Systems Biology', 'Series_contact_institute': 'Beckman Research Institute of City of Hope', 'Series_contact_address': '1218 S. 5th Ave., Room 2062', 'Series_contact_city': 'Monrovia', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '91016', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE121nnn/GSE121952/suppl/GSE121952_HepG2_shSETD2_1_RiboDiff_Differential_Translation_Efficiency.txt.gz', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP167238'}\n",
      "249 PRJNA501880 GSE121981 {'Series_title': 'Cellular response to small molecules that selectively stall protein synthesis by the ribosome', 'Series_geo_accession': 'GSE121981', 'Series_status': 'Public on Mar 06 2019', 'Series_submission_date': 'Oct 30 2018', 'Series_last_update_date': 'Apr 23 2019', 'Series_pubmed_id': '30875366', 'Series_summary': 'We use ribosome profiling to compare the selectivity of two small molecules, PF-06446846 and PF-06378503 that inhibits translation of their target by selectively inducing ribosome-stalling in a nascent chain sequence dependent manner.', 'Series_overall_design': 'Huh7 cells were treated with 1.5 µM PF-06446846, 1.5µM PF-06378503 or vehicle for 60 minutes, and subjected to ribosome profiling. This study included three biological replicates', 'Series_type': 'Other', 'Series_contributor': 'Jamie,H,Cate', 'Series_sample_id': 'GSM3452618', 'Series_contact_name': 'Jamie,HD,Cate', 'Series_contact_email': 'jcate@lbl.gov', 'Series_contact_institute': 'University of California Berkeley', 'Series_contact_address': '748 Stanley Hall', 'Series_contact_city': 'Berkeley', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94720', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE121nnn/GSE121981/suppl/GSE121981_846_vs_Veh_DEseq_poststall.csv.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP167245'}\n",
      "250 PRJNA503203 GSE122039 {'Series_title': 'Translatome and transcriptome analysis of TMA20 (MCT-1) and TMA64 (eIF2D) knockout yeast strains', 'Series_geo_accession': 'GSE122039', 'Series_status': 'Public on Nov 01 2018', 'Series_submission_date': 'Oct 31 2018', 'Series_last_update_date': 'Mar 13 2019', 'Series_pubmed_id': '30815525', 'Series_summary': 'We report ribosome profiling and RNA-Seq data of S.cerevisiae strains with TMA20, TMA64 single- and TMA20/TMA64 double-gene knockouts, grown in the rich YPD or minimal SD medium.', 'Series_overall_design': 'Yeast cells of 4 different S. cerevisiae strains, namely:  Δtma20, Δtma64, ΔΔtma20tma64 knockout strains and the wild-type BY4741, were maintained in YPD or SD media. Two biological replicates were performed for wild-type strain, maintained in YPD media. Ribosomal profiling experiments of the wild-type and knockout strains grown in different media were performed. The сDNA libraries were obtained from total 9 RNA samples and 9 ribosome-bound RNA samples.', 'Series_type': 'Other', 'Series_contributor': 'Ivan,V,Kulakovskiy', 'Series_sample_id': 'GSM3453503', 'Series_contact_name': 'Ivan,V,Kulakovskiy', 'Series_contact_institute': 'Institute of Protein Research', 'Series_contact_address': 'Institutskaya 4', 'Series_contact_city': 'Pushchino', 'Series_contact_zip/postal_code': '142290', 'Series_contact_country': 'Russia', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE122nnn/GSE122039/suppl/GSE122039_genecounts.tsv.gz', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP167363'}\n",
      "251 PRJNA503400 GSE122071 {'Series_title': 'Cap-specific terminal N6-methylation of RNA by an RNA polymerase II-associated methyltransferase.', 'Series_geo_accession': 'GSE122071', 'Series_status': 'Public on Nov 02 2018', 'Series_submission_date': 'Nov 01 2018', 'Series_last_update_date': 'Mar 21 2019', 'Series_pubmed_id': '30467178', 'Series_summary': \"N6-methyladenosine (m6A), a major modification of messenger RNAs (mRNAs), plays critical roles in RNA metabolism and function. In addition to the internal m6A, N6, 2'-O-dimethyladenosine (m6Am) is present at the transcription start nucleotide of capped mRNAs in vertebrates. However, its biogenesis and functional role remain elusive. Using a reverse genetics approach, we identified PCIF1, a factor that interacts with the serine-5-phosphorylated carboxyl-terminal domain of RNA polymerase II, as a cap-specific adenosine methyltransferase (CAPAM) responsible for N6-methylation of m6Am. The crystal structure of CAPAM in complex with substrates revealed the molecular basis of cap-specific m6A formation. A transcriptome-wide analysis revealed that N6-methylation of m6Am promotes the translation of capped mRNAs. Thus, a cap-specific m6A writer promotes translation of mRNAs starting from m6Am.\", 'Series_overall_design': 'Ribosome profiling and RNA-seq', 'Series_type': 'Other', 'Series_contributor': 'Tsutomu,,Suzuki', 'Series_sample_id': 'GSM3454284', 'Series_contact_name': 'Shintaro,,Iwasaki', 'Series_contact_email': 'shintaro.iwasaki@riken.jp', 'Series_contact_laboratory': 'RNA Systems Biochemistry Laboratory', 'Series_contact_department': 'Center for Pioneering Research', 'Series_contact_institute': 'RIKEN', 'Series_contact_address': '2-1, Hirosawa', 'Series_contact_city': 'Wako', 'Series_contact_state': 'Saitama', 'Series_contact_zip/postal_code': '3510198', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE122nnn/GSE122071/suppl/GSE122071_RAW.tar', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP167457'}\n",
      "252 PRJNA505780 GSE122615 {'Series_title': 'Double Stranded DNA Breaks and Genome Editing Trigger Ribosome Remodeling and Translational Shutdown', 'Series_geo_accession': 'GSE122615', 'Series_status': 'Public on Nov 15 2021', 'Series_submission_date': 'Nov 16 2018', 'Series_last_update_date': 'Feb 16 2022', 'Series_pubmed_id': '34914197', 'Series_summary': 'DNA damage activates a robust transcriptional stress response, but much less is known about how DNA damage impacts translation. The advent of genome editing with Cas9 has intensified interest in understanding cellular responses to DNA damage. Here, we find that DNA double-strand breaks (DSBs), including those induced by Cas9, trigger the loss of ribosomal protein RPS27A from ribosomes via p53-independent proteasomal degradation. Comparisons of Cas9 and dCas9 ribosome profiling and mRNA-seq experiments reveal a global translational response to DSBs that precedes changes in transcript abundance. Our results demonstrate that even a single DSB can lead to altered translational output and ribosome remodeling, suggesting caution in interpreting cellular phenotypes measured immediately after genome editing.', 'Series_overall_design': 'mRNA-Seq and ribosome profiling on HEK293 cells after inducing double-strand breaks by Cas9 RNP nucleofection, 36 hours and 72 hours after', 'Series_type': 'Other', 'Series_contributor': 'Jacob,E,Corn', 'Series_sample_id': 'GSM3476302', 'Series_contact_name': 'Nicholas,T,Ingolia', 'Series_contact_email': 'nick@ingolia.org', 'Series_contact_phone': '410 246 3025', 'Series_contact_laboratory': 'Ingolia', 'Series_contact_department': 'Embryology', 'Series_contact_institute': 'Carnegie Institution', 'Series_contact_address': '3520 San Martin Drive', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21218', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE122nnn/GSE122615/suppl/GSE122615_RAW.tar', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP169066'}\n",
      "253 PRJNA506724 GSE122870 {'Series_title': 'Translation complex profiling for bacteria [Ribo-seq]', 'Series_geo_accession': 'GSE122870', 'Series_status': 'Public on Sep 16 2019', 'Series_submission_date': 'Nov 23 2018', 'Series_last_update_date': 'Dec 05 2019', 'Series_pubmed_id': '31586395', 'Series_summary': 'Ribosomes were arrested during initiation, elongation and termination complex formation using formaldehyde crosslinking. The complex protected footprints were isolated after MNase treatment and sequenced.', 'Series_overall_design': 'Studying translation initiation in Wt and assembly defecient strains', 'Series_type': 'Other', 'Series_contributor': 'Himanshu,,Sharma', 'Series_sample_id': 'GSM3487546', 'Series_contact_name': 'B,,Anand', 'Series_contact_email': 'banand@iitg.ac.in', 'Series_contact_laboratory': 'B.Anand', 'Series_contact_department': 'Biosciences and Bioengineering', 'Series_contact_institute': 'Indian institute of technology guwahati', 'Series_contact_address': 'North guwahati', 'Series_contact_city': 'Guwahati', 'Series_contact_state': 'Assam', 'Series_contact_zip/postal_code': '781039', 'Series_contact_country': 'India', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE122nnn/GSE122870/suppl/GSE122870_RAW.tar', 'Series_platform_id': 'GPL21117', 'Series_platform_taxid': '511145', 'Series_sample_taxid': '511145', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP170541'}\n",
      "254 PRJNA507085 GSE122971 {'Series_title': 'Multifaceted stoichiometry control of bacterial operons revealed by data-independent acquisition mass spectrometry', 'Series_geo_accession': 'GSE122971', 'Series_status': 'Public on Jul 01 2019', 'Series_submission_date': 'Nov 27 2018', 'Series_last_update_date': 'Jul 02 2019', 'Series_summary': 'More than half of the protein-coding genes in bacteria are organized in polycistronic operons composed of two or more genes. Whether the operon organization maintains the stoichiometric expression of the genes within an operon remain under debate. In this study, we performed a label-free data-independent acquisition hyper reaction monitoring mass-spectrometry (HRM-MS) experiment to quantify the Escherichia coli proteome in exponential phase and quantified 93.6% of the cytosolic proteins, covering 67.9% and 56.0% of the translating polycistronic operons in BW25113 and MG1655 strains, respectively. We found the shorter operons tend to be more tightly controlled for stoichiometry than longer operons, and those operons for metabolic pathways are less controlled for stoichiometry compared with operons for protein complexes, illustrating the multifaceted nature of the operon-wise regulation: the operon-wise unified transcriptional level and gene-specific translational level. This multi-level regulation benefits the host by optimizing the efficiency of the productivity of metabolic pathways and maintenance of different types of protein complexes.', 'Series_overall_design': 'RNA-seq and Ribo-seq of E. coli strain BW25113 in M9 minimal medium', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Gong,,Zhang', 'Series_sample_id': 'GSM3489379', 'Series_contact_name': 'zhang,,hong', 'Series_contact_email': 'alexnoname@outlook.com', 'Series_contact_phone': '13631392383', 'Series_contact_institute': 'zhanghong', 'Series_contact_address': 'Huangpu Boulavard West Jinan uinversity', 'Series_contact_city': 'guangzhou', 'Series_contact_state': 'guangdong province', 'Series_contact_zip/postal_code': '510632', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE122nnn/GSE122971/suppl/GSE122971_RAW.tar', 'Series_platform_id': 'GPL25852', 'Series_platform_taxid': '679895', 'Series_sample_taxid': '679895', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP170846'}\n",
      "255 PRJNA509230 GSE123564 {'Series_title': 'Mammalian Hbs1L deficiency causes Pelota depletion and is associated with a unique phenotype', 'Series_geo_accession': 'GSE123564', 'Series_status': 'Public on Jan 15 2019', 'Series_submission_date': 'Dec 10 2018', 'Series_last_update_date': 'Feb 01 2021', 'Series_pubmed_id': '30707697', 'Series_summary': 'Hbs1 has been established as a central component of the cell’s translational quality control pathways in both yeast and prokaryotic models; however, the functional characteristics of its human ortholog (Hbs1L) have not been well-defined.  We recently reported a novel human phenotype resulting from a mutation in the critical coding region of the HBS1L gene characterized by facial dysmorphism, severe growth restriction, axial hypotonia, global developmental delay and retinal pigmentary deposits.  Here we further characterize downstream effects of the human HBS1L mutation. HBS1L has three transcripts in humans, and RT-PCR demonstrated reduced mRNA levels corresponding with transcripts V1 and V2 whereas V3 expression was unchanged. Western blot analyses revealed HBS1L protein was absent in proband samples. Additionally, polysome profiling revealed an abnormal aggregation of 80S monosomes in patient cells under baseline conditions.  RNASeq data corresponded to the patient phenotype, with biological process analysis showing skeletal system development and sensory development genes being most altered between patient and controls.  Ribosomal sequencing demonstrated an increased translation efficiency of ribosomal RNA in Hbs1L-deficient fibroblasts, suggesting that there may be a compensatory increase in ribosome translation to accommodate the increased 80S monosome peak. Furthermore, lack of Hbs1L caused depletion of Pelota protein in both patient cells and mouse tissues, while PELO mRNA levels were unaffected. Inhibition of proteasomal function partially restored Pelota expression in human Hbs1L-deficient cells. We also describe a mouse model harboring a knockdown mutation in the murine Hbs1l gene that shared several of the phenotypic elements observed in the Hbs1L-deficient human including facial dysmorphism, growth restriction and retinal deposits. The Hbs1lKO mice similarly demonstrate diminished Pelota levels that were rescued by proteasome inhibition.', 'Series_overall_design': '12 physical samples of cell cultures (patient and 2 healthy individuals as controls, 2-6 replicates per individual). Ribo-seq and mRNA-seq libraries prepared from the same samples (matched within sample)', 'Series_type': 'Other', 'Series_contributor': 'Pankaj,B,Agrawal', 'Series_sample_id': 'GSM3507230', 'Series_contact_name': 'Pankaj,,Agrawal', 'Series_contact_email': 'pagrawal@enders.tch.harvard.edu', 'Series_contact_institute': \"Boston Children's Hospital\", 'Series_contact_address': '300 Longwood ave', 'Series_contact_city': 'Boston', 'Series_contact_zip/postal_code': '02115', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE123nnn/GSE123564/suppl/GSE123564_patient_vs_healthy_mRNAseq.csv.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP173240'}\n",
      "256 PRJNA510416 GSE123981 {'Series_title': 'Age-resolved ribosome profiling and RNA-Seq data of mice liver and kidney samples', 'Series_geo_accession': 'GSE123981', 'Series_status': 'Public on Jun 01 2020', 'Series_submission_date': 'Dec 17 2018', 'Series_last_update_date': 'Sep 01 2020', 'Series_pubmed_id': '32576685', 'Series_summary': 'We report ribosome profiling and RNA-Seq data of liver and kidney samples taken from C57BL/6 mice of different ages', 'Series_overall_design': 'Ribosome profiling and transcriptome sequencing of liver and kidney samples of C57BL/6 mice of different ages. Liver samples were collected from mice of six (1, 3, 11, 20, 26 and 32 months) different ages, whereas kidney samples were collected from the same mice of three (3, 20 and 32 months) ages. Samples from three animals were taken for each age, except for 32 months old mice (2 animals).', 'Series_type': 'Other', 'Series_contributor': 'Vadim,N,Gladyshev', 'Series_sample_id': 'GSM3518186', 'Series_contact_name': 'Aleksandra,S,Anisimova', 'Series_contact_email': 'aleksandra.anisimova@univie.ac.at, alessandrick@gmail.com', 'Series_contact_laboratory': 'Karagöz Lab', 'Series_contact_institute': 'Max Perutz Labs, Vienna BioCenter, University of Vienna, Medical University of Vienna', 'Series_contact_address': 'Dr. Bohr Gasse 9', 'Series_contact_city': 'Vienna', 'Series_contact_zip/postal_code': '1030', 'Series_contact_country': 'Austria', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE123nnn/GSE123981/suppl/GSE123981_genecounts.tsv.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP173659'}\n",
      "257 PRJNA510796 GSE124115 {'Series_title': 'Translational Regulation of Metabolic Dynamics during Effector-Triggered Immunity', 'Series_geo_accession': 'GSE124115', 'Series_status': 'Public on Dec 16 2020', 'Series_submission_date': 'Dec 19 2018', 'Series_last_update_date': 'Dec 16 2020', 'Series_summary': 'It is not fully known whether translational regulation also occurs in later stage immune responses, such as effector-triggered immunity (ETI), which often leads to strong metabolic dynamics. In this study, we performed a genome-wide ribosome profiling in Arabidopsis upon ETI activation and discovered that specific groups of genes were translationally regulated, especially metabolic genes in aromatic amino acid, phenylpropanoid, camalexin, and sphingolipid metabolism. The involvement of these components in the induction of ETI was confirmed by genetic analysis, amino acid profiling and exogeneous application of phenylalanine or an inhibitor of aromatic amino acid biosynthesis. Our findings provide new insight into the diverse translational regulation in the plant immune responses and demonstrate that translational coordination of metabolic gene expression is an important strategy for ETI activation.', 'Series_overall_design': 'Arabidopsis transgenic lines carrying the 35S:uORFsTBF1-LUC construct in WT Col-0 (C) background or rps2 (R)\\xa0background were grown on soil at 22 °C under a 12/12-h light/dark cycle with 55% humidity. Leaf tissue (0.3 g) of 4 week old transgenic plants was collected 8 hours post inoculation with either Psm ES4326/AvrRpt2 (OD600nm = 0.01) (A)\\xa0or 10 mM MgCl2 as Mock (M). For WT plants, 3 replicates of Mock and AvrRpt2 were collected for RNA-seq (RS) and Ribo-seq (RF). For rps2, 2 replicates were collected for Mock and AvrRpt2 RS and RF.', 'Series_type': 'Other', 'Series_contributor': 'Xinnian,,Dong', 'Series_sample_id': 'GSM3521122', 'Series_contact_name': 'Xinnian,,Dong', 'Series_contact_laboratory': 'Xinnian Dong', 'Series_contact_department': 'Biology', 'Series_contact_institute': 'Duke University', 'Series_contact_address': '124 Science Drive', 'Series_contact_city': 'Durham', 'Series_contact_state': 'NC', 'Series_contact_zip/postal_code': '27708', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE124nnn/GSE124115/suppl/GSE124115_rps2_RPKM.txt.gz', 'Series_platform_id': 'GPL13222', 'Series_platform_taxid': '3702', 'Series_sample_taxid': '3702', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP173921'}\n",
      "258 PRJNA511455 GSE124279 {'Series_title': 'The RNA-binding protein HuR is a negative regulator in adipogenesis [Ribo-seq_RNA-seq_matched]', 'Series_geo_accession': 'GSE124279', 'Series_status': 'Public on Sep 09 2019', 'Series_submission_date': 'Dec 21 2018', 'Series_last_update_date': 'Jan 21 2020', 'Series_pubmed_id': '31924774', 'Series_summary': 'Human antigen R (HuR) protein, a RNA binding protein (RBP), has been reported to  regulate essential steps in RNA metabolism and immune response in a variety of cell types,  but its function in metabolism remains unclear. This study identifies HuR as a major  repressor during adipogenesis. Knockdown and overexpression of HuR in primary adipocyte  culture enhances and inhibits adipogenesis in vitro, respectively. Fat-specific knockout of  HuR significantly enhances adipogenic gene program in all three major adipose tissues  including epidydimal, inguinal white and brown adipose tissue, accompanied with systemic  glucose intolerance and insulin resistance. Conversely, transgenic overexpression of HuR in  adipose tissue prevents the HFD induced obesity by repressing adipogenesis.  Mechanistically, HuR may inhibit adipogenesis by recognizing and modulating the stability of  hundreds of adipocyte transcripts, including the mRNA of Insig1, a negative regulator during  adipogenesis. Taken together, our work establishes HuR as a novel posttranscriptional  regulator of adipogenesis and provides a new insight into how RNA processing contributes  to adipocyte development.', 'Series_overall_design': '8 samples were included. Duplicate of epididymal white fat from control and huR knockout animals were harvested for matched RNA-seq and RIBO-seq.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Yen,C,Lim', 'Series_sample_id': 'GSM3526850', 'Series_contact_name': 'Lei,,Sun', 'Series_contact_email': 'sun.lei@duke-nus.edu.sg', 'Series_contact_institute': 'Duke-NUS', 'Series_contact_address': '8 College Road', 'Series_contact_city': 'Singapore', 'Series_contact_zip/postal_code': '169857', 'Series_contact_country': 'Singapore', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE124nnn/GSE124279/suppl/GSE124279_HuR_master_Background.xlsx', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP174218'}\n",
      "259 PRJNA512491 GSE124558 {'Series_title': 'Differential roles of human PUS10 in miRNA processing and tRNA pseudouridylation', 'Series_geo_accession': 'GSE124558', 'Series_status': 'Public on Nov 21 2019', 'Series_submission_date': 'Jan 02 2019', 'Series_last_update_date': 'Feb 20 2020', 'Series_pubmed_id': '31819270', 'Series_summary': 'Pseudouridine synthases (PUSs) are responsible for the installation of pseudouridine (Ψ) modification in RNA. However, the activity and function of the PUS enzymes remain largely unexplored. Here we focus on human PUS10 and find that it co-expresses with the microprocessor (DROSHA–DGCR8 complex). Depletion of PUS10 results in a marked reduction of the expression level of a large number of mature miRNAs and concomitant accumulation of unprocessed primary microRNAs (pri-miRNAs) in multiple human cells. Mechanistically, PUS10 directly binds to pri-miRNAs and interacts with the microprocessor to promote miRNA biogenesis. Unexpectedly, this process is independent of the catalytic activity of PUS10. Additionally, we develop a sequencing method to profile Ψ in the tRNAome and report PUS10-dependent Ψ sites in tRNA. Collectively, our findings reveal differential functions of PUS10 in nuclear miRNA processing and in cytoplasmic tRNA pseudouridylation.', 'Series_overall_design': 'DM-Ψ-seq,  PolyA RNA-seq,  rRNA(-) nuclear RNA-seq, Ribo-seq of PUS10 knockout and wild-type HEK293T cells and PAR-CLIP-seq against PUS10 in HEK293T cells.', 'Series_type': 'Other', 'Series_contributor': 'Chengqi,,Yi', 'Series_sample_id': 'GSM3984257', 'Series_contact_name': 'Jinghui,,Song', 'Series_contact_email': 'sophiaatzz@163.com', 'Series_contact_institute': 'Peking university', 'Series_contact_address': '5 Yiheyuan Road, Haidian District', 'Series_contact_city': 'Beijing', 'Series_contact_zip/postal_code': '100871', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE124nnn/GSE124558/suppl/GSE124558_PolyA_RNA_seq_summary.txt.gz', 'Series_platform_id': 'GPL20795', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP175017'}\n",
      "260 PRJNA512837 GSE124609 {'Series_title': 'Cellular proteostasis collapse in mammalian senescence', 'Series_geo_accession': 'GSE124609', 'Series_status': 'Public on Dec 09 2020', 'Series_submission_date': 'Jan 03 2019', 'Series_last_update_date': 'Dec 10 2020', 'Series_pubmed_id': '33257563', 'Series_summary': 'Proteostasis collapse, the diminished ability to maintain protein homeostasis, has been established as a hallmark of nematode aging. However, whether proteostasis collapse occurs in humans has remained unclear. Here, we demonstrate that proteostasis decline is intrinsic to human senescence. Using transcriptome-wide characterization of gene expression, splicing, and translation, we found a significant deterioration in the transcriptional activation of the heat shock response in stressed senescent cells. Furthermore, phosphorylated HSF1 nuclear localization and distribution were impaired in senescence. Interestingly, alternative splicing regulation was also dampened. Surprisingly, we found a decoupling between different unfolded protein response (UPR) branches in stressed senescent cells. While young cells initiated UPR-related translational and transcriptional regulatory responses, senescent cells showed enhanced translational regulation and endoplasmic reticulum (ER) stress sensing; however, they were unable to trigger UPR-related transcriptional responses. This was accompanied by diminished ATF6 nuclear localization in stressed senescent cells. Finally, we found that proteasome function was impaired following heat stress in senescent cells, and did not recover upon return to normal temperature. Together, our data unraveled a deterioration in the ability to mount dynamic stress transcriptional programs upon human senescence with broad implications on proteostasis control and connected proteostasis decline to human aging.', 'Series_overall_design': 'RNA-Seq and Ribo-Seq for young and senescent WI-38 cells with and without heat shock', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Niv,,Sabath', 'Series_sample_id': 'GSM3537597', 'Series_contact_name': 'Reut,,Shalgi', 'Series_contact_email': 'reutshalgi@technion.ac.il', 'Series_contact_department': 'Faculty of Medicine', 'Series_contact_institute': 'Technion', 'Series_contact_address': '1 Efron st.', 'Series_contact_city': 'Haifa', 'Series_contact_zip/postal_code': '31096', 'Series_contact_country': 'Israel', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE124nnn/GSE124609/suppl/GSE124609_Ribo_Seq_rsem.genes.tpm.txt.gz', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP175158'}\n",
      "261 PRJNA513069 GSE124693 {'Series_title': 'Coordination of hunger and sleep through circadian regulation of transcription in energy-sensing neurons', 'Series_geo_accession': 'GSE124693', 'Series_status': 'Public on Mar 04 2019', 'Series_submission_date': 'Jan 04 2019', 'Series_last_update_date': 'Jul 26 2022', 'Series_summary': 'Alignment of fasting and feeding with the sleep/wake cycle is coordinated by hypothalamic neurons, though the underlying molecular programs remain incompletely understood. Here we demonstrate that the clock transcription pathway maximizes eating during wakefulness and glucose production during sleep through transcription pathway maximizes eating during autonomous circadian regulation of NPY/AgRP neurons. Tandem profiling of whole cell and ribosome-bound mRNAs in morning and evening under dynamic fasting and fed conditions identified temporal control of activity-dependent gene repertoires in AgRP neurons central to synaptogenesis, bioenergetics, and neurotransmitter and peptidergic signaling. Synaptic and circadian pathways were specific to whole cell RNA analyses, while bioenergetic pathways were selectively enriched in the ribosome-bound transcriptome. Finally, we demonstrate that the AgRP clock mediates the transcriptional food acquisition with sleep/wake state. response to leptin. Our results reveal that time-of-day restriction in transcriptional control of energy-sensing neurons underlies the alignment of hunger and day restriction in transcriptional control of energy-sensing neurons underlies the alignment of hunger and food acquisition with sleep/wake state.', 'Series_overall_design': 'Examination of AGRP neurons including BMAL1-KO, under 3 different nutrient states, leptin adiministration, and time of day collection points.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Joseph,,Bass', 'Series_sample_id': 'GSM3543543', 'Series_contact_name': 'Nathan,,Waldeck', 'Series_contact_email': 'nathanwaldeck2022@u.northwestern.edu', 'Series_contact_institute': 'Northwestern University', 'Series_contact_address': '303 E Superior St, Lurie 7-220', 'Series_contact_city': 'Chicago', 'Series_contact_state': 'IL', 'Series_contact_zip/postal_code': '60611', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE124nnn/GSE124693/suppl/GSE124693_agrp_normalized_counts.txt.gz', 'Series_citation': 'Jonathan Cedernaes, Wenyu Huang, Kathryn Moynihan Ramsey, Nathan Waldeck, Lei Cheng, Biliana Marcheva, Chiaki Omura, Yumiko Kobayashi, Clara Bien Peek, Daniel C. Levine, Ravindra Dhir, Raj Awatramani, Christopher A. Bradfield, Xiaozhong A. Wang, Joseph S. Takahashi, Mohamad Mokadem, Rexford S. Ahima, and Joseph Bass. Transcriptional Basis for Rhythmic Control of Hunger and Metabolism within the AgRP Neuron. Cell Metabolism 2019; doi:10.1016/j.cmet.2019.01.023', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP175343'}\n",
      "262 PRJNA513305 GSE124737 {'Series_title': 'Ribosome profiling in transgenic and HEK293T cells treated with NAEK', 'Series_geo_accession': 'GSE124737', 'Series_status': 'Public on Jan 07 2020', 'Series_submission_date': 'Jan 07 2019', 'Series_last_update_date': 'Jan 07 2020', 'Series_summary': 'To investigate whether natural TAG was decoded by stably transfected Mb pyltRNA in transgenic cells, their cell lysate was subjected to ribosome profiling, examining the physical locations of ribosomes. HEK293T cells were taken as control.', 'Series_overall_design': 'Examine the physical locations of ribosomes in both transgenic and HEK293T cells', 'Series_type': 'Other', 'Series_contributor': 'Y,,Wu', 'Series_sample_id': 'GSM3544718', 'Series_contact_name': 'Yiming,,Wu', 'Series_contact_email': '1309600826@qq.com', 'Series_contact_laboratory': 'Demin Zhou', 'Series_contact_department': 'Chemical Bilogy', 'Series_contact_institute': 'Peking University Health Science Center', 'Series_contact_address': 'No 38 Xueyuan Road', 'Series_contact_city': 'Beijing', 'Series_contact_state': 'Beijing', 'Series_contact_zip/postal_code': '100191', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE124nnn/GSE124737/suppl/GSE124737_Ribosome_profiling_of_transgenic_and_HEK293T_cells.xlsx', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP176433'}\n",
      "263 PRJNA514710 GSE124962 {'Series_title': 'The landscape of mRNA translation in tomato roots', 'Series_geo_accession': 'GSE124962', 'Series_status': 'Public on Jul 05 2019', 'Series_submission_date': 'Jan 11 2019', 'Series_last_update_date': 'Jul 05 2019', 'Series_pubmed_id': '31248964', 'Series_summary': 'We used Ribo-seq (Ribosome profiling) combining with RNA-seq to explore the translational landscape of tomato roots. We generated three biological replicates of RNA-seq and Ribo-seq data for tomato roots. We next used the RNA-seq result for de novo transcriptome assembly and Ribo-seq to identify novel translated open reading frames (ORFs). Our data revealed more than three hundreds of novel translated ORFs on previously unannotated transcripts. Most of the newly identified ORFs are small and difficult to detect with in silico methods. We also identified over thirteen hundreds of upstream ORFs on annotated genes. This data could facilitate gene annotation. Besides, this data also demonstrated that uORFs, miRNAs and antisense RNAs are regulating the expression of associated genes. This study uncovered mechanisms of translational regulation and gene annotation in tomato.', 'Series_overall_design': '3 biological replicates of 4-day old tomato roots (about 2~3 cm from root tips) collected and processed for Ribo-seq and RNA-seq sample preparation. The Ribo-seq library was sequenced with illumina single-end 50 sequencing and the RNA-seq library was sequenced with illumina paired-end 100 sequencing.', 'Series_type': 'Other', 'Series_contributor': 'Hsin-Yen,,Wu', 'Series_sample_id': 'GSM3560216', 'Series_contact_name': 'Polly,,Hsu', 'Series_contact_email': 'pollyhsu@msu.edu', 'Series_contact_department': 'Biochemistry and Molecular Biology', 'Series_contact_institute': 'Michigan State University', 'Series_contact_address': '603 Wilson Road', 'Series_contact_city': 'East Lansing', 'Series_contact_state': 'MI', 'Series_contact_zip/postal_code': '48824', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE124nnn/GSE124962/suppl/GSE124962_translated_ORFs_filtered_sorted.bed.gz', 'Series_platform_id': 'GPL25655', 'Series_platform_taxid': '4081', 'Series_sample_taxid': '4081', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP178496'}\n",
      "264 PRJNA515289 GSE125114 {'Series_title': 'DDX3 depletion represses translation of mRNAs with complex 5′ UTRs', 'Series_geo_accession': 'GSE125114', 'Series_status': 'Public on Mar 25 2019', 'Series_submission_date': 'Jan 15 2019', 'Series_last_update_date': 'May 19 2021', 'Series_pubmed_id': '33905506', 'Series_summary': 'We describe the subset of transcripts that require DDX3 for efficient translation.', 'Series_overall_design': 'Ribosome profiling upon DDX3 knockdown and DDX3 PAR-CLIP was performed from HEK 293 cells.', 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'Stephen,N,Floor', 'Series_sample_id': 'GSM4769586', 'Series_contact_name': 'Emanuel,,Wyler', 'Series_contact_email': 'emanuel.wyler@mdc-berlin.de', 'Series_contact_phone': '+49 30 9406 3009', 'Series_contact_laboratory': 'RNA Biology and Posttranscriptional Regulation', 'Series_contact_department': 'Berlin Institute for Medical Systems Biology', 'Series_contact_institute': 'Max Delbrück Center for Molecular Medicine', 'Series_contact_address': 'Robert Roessle Str 10', 'Series_contact_city': 'Berlin', 'Series_contact_zip/postal_code': '13125', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE125nnn/GSE125114/suppl/GSE125114_RAW.tar', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP179636'}\n",
      "265 PRJNA515538 GSE125218 {'Series_title': 'Accurate annotation of human protein-coding small open reading frames', 'Series_geo_accession': 'GSE125218', 'Series_status': 'Public on Jul 03 2019', 'Series_submission_date': 'Jan 16 2019', 'Series_last_update_date': 'Jun 26 2024', 'Series_pubmed_id': '38842510', 'Series_web_link': 'https://www.nature.com/articles/s41589-019-0425-0', 'Series_summary': 'Protein-coding small\\xa0open reading frames (smORFs) are emerging as an important class of genes, however, the coding capacity of smORFs in the human genome is unclear. By integrating de novo transcriptome assembly and Ribo-Seq, we confidently annotate thousands of novel translated smORFs in three human cell lines. We find that smORF translation prediction is noisier than for annotated coding sequences, underscoring the importance of analyzing multiple experiments and footprinting conditions. These smORFs are located within non-coding and antisense transcripts, the UTRs of mRNAs, and unannotated transcripts. Analysis of RNA levels and translation efficiency during cellular stress identifies regulated\\xa0smORFs and provides an approach for identifying smORFs for further investigation. Sequence conservation and signatures of positive selection\\xa0indicate that encoded microproteins are likely functional. Additionally, proteomics data from enriched human leukocyte antigen complexes validates the translation of hundreds of smORFs and positions them as a source of novel antigens. Thus, smORFs represent a significant number of important, yet unexplored human genes.', 'Series_overall_design': 'Annotation of protein-coding smORFs in 3 human cell lines using de novo transcriptome assembly and Ribo-Seq.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Maxim,N,Shokhirev', 'Series_sample_id': 'GSM3566429', 'Series_contact_name': 'Thomas,Farid,Martinez', 'Series_contact_email': 'tmartinez@salk.edu', 'Series_contact_laboratory': 'Saghatelian (PBL-A)', 'Series_contact_department': 'Peptide Biology Laboratories', 'Series_contact_institute': 'Salk Institute for Biological Studies', 'Series_contact_address': '10010 N Torrey Pines Rf', 'Series_contact_city': 'La Jolla', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '92130', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE125nnn/GSE125218/suppl/GSE125218_K562_Cufflinks_transcript_assembly.gtf.gz', 'Series_citation': 'Thomas F. Martinez, Qian Chu, Cynthia Donaldson, Dan Tan, Maxim N. Shokhirev, Alan Saghatelian. An Improved Human smORF Annnotation Workflow Combining De Novo Transcriptome Assembly and Ribo-Seq. bioRxiv 523860. doi:10.1101/523860', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP179798'}\n",
      "266 PRJNA517401 GSE125725 {'Series_title': 'RNA-seq and ribosome profiling of heat shocked wild type and Mettl14 -/- mouse embryonic stem cells', 'Series_geo_accession': 'GSE125725', 'Series_status': 'Public on Jul 15 2019', 'Series_submission_date': 'Jan 28 2019', 'Series_last_update_date': 'Jul 22 2019', 'Series_pubmed_id': '31292544', 'Series_summary': 'N6-methyladenosine (m6A) is a dynamic and reversible nucleotide modification in mRNA. m6A alters mRNA fate, but it is unclear why the effects of m6A can vary in different cellular contexts. Here we show that methylated mRNAs are catalysts for liquid-liquid phase separation of the YTHDF family of m6A-binding proteins. RNAs that contain multiple, but not single, m6A residues recruit multiple YTHDF proteins, causing them to undergo a proximity-induced phase separation. YTHDF proteins show liquid-like properties upon binding polymethylated mRNAs in cells, and partition into endogenous phase-separated compartments, such as P-bodies, neuronal RNA granules, and stress granules. The complexes of YTHDF proteins and polymethylated mRNAs are targeted to different compartments depending on the cell context, leading to different effects on m6A mRNAs. These studies reveal a role for nucleotide modifications in regulating phase separation and indicate that the cellular properties of m6A-modified mRNAs can be explained by liquid-liquid phase separation principles.', 'Series_overall_design': 'RNA-seq was performed on wild type and Mettl14 -/- mouse embryonic stem cells. Four replicates were sequenced under non-stressed conditions, after heat shock at 42°C for 30 min, and 1 h after recovery from heat shock stress.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Samie,R,Jaffrey', 'Series_sample_id': 'GSM3581014', 'Series_contact_name': 'Ryan,J,Ries', 'Series_contact_email': 'rjr2007@med.cornell.edu', 'Series_contact_institute': 'Weill Cornell Medical College', 'Series_contact_address': '1300 York Avenue', 'Series_contact_city': 'New York', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '10021', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE125nnn/GSE125725/suppl/GSE125725_mES_WT_RNAseq_HeatShockvNoStress.csv.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP182641'}\n",
      "267 PRJNA517458 GSE125757 {'Series_title': 'p53-inducible long non-coding RNAs encode functional peptides in hepatocellular carcinoma cells [Ribo-seq]', 'Series_geo_accession': 'GSE125757', 'Series_status': 'Public on Jul 18 2020', 'Series_submission_date': 'Jan 28 2019', 'Series_last_update_date': 'Jul 18 2020', 'Series_summary': 'To globally analyse the lncRNAs with potential coding ability upon DNA damage, we performed Ribo-seq using ADR-treated HepG2 cells. The HepG2 cells were treated with 500 ng/mL ADR for 24 h, and cycloheximide (CHX) was added to inhibit the translational elongation of ribosomes.Furthermore,we identified the candidate ORFs with the selected RPF reads using the RiboCode software.', 'Series_overall_design': 'Ribo-seq profiles of HepG2 p53 wild-type (WT) treated with ADR were generated by deep sequencing, in\\xa0two independent\\xa0experiments, using  Xten Illumina sequencing platform. The ribo-seq data of the two biological replicates were integrated.Then,for the analysis of Ribo-seq data, sequenced reads that were trimmed for the adaptor sequence were aligned to the UCSC hg19 reference genome by STAR (version 2.4.2a) with parameters provided by the RiboCode (version 1.2.10) manual.', 'Series_type': 'Other', 'Series_contributor': 'Jian-Hua,,Yang', 'Series_sample_id': 'GSM3581511', 'Series_contact_name': 'Jian-Hua,,Yang', 'Series_contact_email': 'yangjh7@mail.sysu.edu.cn', 'Series_contact_phone': '86-20-84112517', 'Series_contact_department': 'School of Life Sciences', 'Series_contact_institute': 'Sun Yat-sen University', 'Series_contact_address': 'No. 135, Xingang Xi Road', 'Series_contact_city': 'Guangzhou', 'Series_contact_zip/postal_code': '510275', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE125nnn/GSE125757/suppl/GSE125757_HepG2_WT_ADRplus_RiboCode_ORF.bed12.gz', 'Series_platform_id': 'GPL20795', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP182676'}\n",
      "268 PRJNA517673 GSE125832 {'Series_title': 'Ribosome profiling analysis of lncRNA TINCR silenced primary melanoma cell line WM902B', 'Series_geo_accession': 'GSE125832', 'Series_status': 'Public on Nov 25 2020', 'Series_submission_date': 'Jan 29 2019', 'Series_last_update_date': 'Feb 26 2021', 'Series_pubmed_id': '33586907', 'Series_summary': 'Results: we found that TINCR silencing induces activation of ATF4 translation', 'Series_overall_design': 'Analysis of differential translation in shScrambled and shTINCR expressing WM902B cells', 'Series_type': 'Other', 'Series_contributor': 'Luisa,,Lanfrancone', 'Series_sample_id': 'GSM3582931', 'Series_contact_name': 'Marine,,Melixetian', 'Series_contact_email': 'marine.melixetian@ieo.eu', 'Series_contact_institute': 'European Institute of Oncology', 'Series_contact_address': 'via Adamello 16', 'Series_contact_city': 'Milano', 'Series_contact_zip/postal_code': '20141', 'Series_contact_country': 'Italy', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE125nnn/GSE125832/suppl/GSE125832_WM902B_shTINCR_shSCR_diffTE_genes.txt.gz', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP182817'}\n",
      "269 PRJNA521364 GSE126234 {'Series_title': 'Suppressor mutations in ribosomal proteins and FliY restore Bacillus subtilis swarming motility in the absence of EF-P [Ribosome Profiling]', 'Series_geo_accession': 'GSE126234', 'Series_status': 'Public on May 20 2019', 'Series_submission_date': 'Feb 07 2019', 'Series_last_update_date': 'Jul 29 2019', 'Series_pubmed_id': '31237868', 'Series_summary': 'Translation elongation factor P (EF-P) alleviates ribosome pausing at a subset of motifs encoding consecutive proline residues and is required for growth in many organisms. Here we show that Bacillus subtilis EF-P also alleviated ribosome pausing at sequences encoding tandem prolines, and ribosomes paused within several essential genes without a corresponding growth defect in an efp mutant. The B. subtilis efp mutant is instead impaired for flagellar biosynthesis which results in the abrogation of a form of motility called swarming. We isolate swarming suppressors of efp and identify mutations in 8 genes that suppressed the efp mutant swarming defect, many of which encode conserved ribosomal proteins or ribosome-associated factors. One mutation abolished a translational pause site within the flagellar C-ring component FliY to increase flagellar number and restore swarming motility in the absence of EF-P. Our data support a model wherein EF-P-alleviation of ribosome pausing may be particularly important for macromolecular assemblies like the flagellum that require precise protein stoichiometries.', 'Series_overall_design': '4 Bacillus subtilis strains with 3 biological replicates each were analyzed. DK1042 is the wild type and parent strain.', 'Series_type': 'Other', 'Series_contributor': 'Daniel,B,Kearns', 'Series_sample_id': 'GSM3594065', 'Series_contact_name': 'Daniel,B.,Kearns', 'Series_contact_email': 'dbkearns@indiana.edu', 'Series_contact_laboratory': 'Kearns Lab', 'Series_contact_department': 'Biology', 'Series_contact_institute': 'Indiana University', 'Series_contact_address': '1001 E 3rd St', 'Series_contact_city': 'Bloomington', 'Series_contact_state': 'IN', 'Series_contact_zip/postal_code': '47405-7005', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE126nnn/GSE126234/suppl/GSE126234_RAW.tar', 'Series_platform_id': 'GPL24109', 'Series_platform_taxid': '1423', 'Series_sample_taxid': '1423', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP184511'}\n",
      "270 PRJNA521496 GSE126298 {'Series_title': 'Selective translation mechanism through two paralogs of eukaryotic initiation factor 4A', 'Series_geo_accession': 'GSE126298', 'Series_status': 'Public on Oct 26 2019', 'Series_submission_date': 'Feb 08 2019', 'Series_last_update_date': 'Oct 28 2019', 'Series_summary': 'Eukaryotic initiation factor (eIF) 4A is a DEAD-box RNA-binding protein that plays a pivotal role in translation initiation. Although mammals have two eIF4A paralogs, eIF4A1 and eIF4A2, their redundancy has been long assumed because of high homology and their functional difference has been poorly understood. Here we show that eIF4A paralogs employ differential translation programs. Ribosome profiling of eIF4A1- and eIF4A2-knockout HEK293T cell lines revealed that translation efficiency changes were divergent across transcriptome and distinct group of mRNAs were regulated by each paralog. Our analysis indicates that eIF4A1 and eIF4A2 facilitate translation of specific mRNAs.', 'Series_overall_design': 'Ribosome profiling and RNA-seq', 'Series_type': 'Other', 'Series_contributor': 'Shintaro,,Iwasaki', 'Series_sample_id': 'GSM3595803', 'Series_contact_name': 'Shintaro,,Iwasaki', 'Series_contact_email': 'shintaro.iwasaki@riken.jp', 'Series_contact_laboratory': 'RNA Systems Biochemistry Laboratory', 'Series_contact_department': 'Center for Pioneering Research', 'Series_contact_institute': 'RIKEN', 'Series_contact_address': '2-1, Hirosawa', 'Series_contact_city': 'Wako', 'Series_contact_state': 'Saitama', 'Series_contact_zip/postal_code': '3510198', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE126nnn/GSE126298/suppl/GSE126298_RAW.tar', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP184677'}\n",
      "271 PRJNA521751 GSE126385 {'Series_title': 'Divergent effects of eRF3 and Upf1 on the expression of uORF carrying mRNAs and ribosome protein genes', 'Series_geo_accession': 'GSE126385', 'Series_status': 'Public on Oct 27 2019', 'Series_submission_date': 'Feb 11 2019', 'Series_last_update_date': 'Oct 27 2019', 'Series_pubmed_id': '31619139', 'Series_summary': \"In addition to its role in translation termination, eRF3 has been implicated in the nonsense-mediated mRNA decay (NMD) pathway through its interaction with Upf1. NMD is a RNA quality control mechanism, which detects and degrades aberrant mRNAs as well as some normal transcripts including those that harbor upstream open reading frames in their 5' leader sequence and long 3′ UTR. In this study, we used RNA-sequencing and ribosome profiling to perform a genome wide analysis of the effect of either eRF3 or Upf1 depletion in human cells. Our bioinformatics analyses allow to delineate the features of the transcripts controlled by eRF3 and Upf1 and to compare the effect of each of these factors on gene expression. We show that eRF3 and Upf1 have very different impact on the human transcriptome, only ~250 transcripts being targeted by both factors. We also show that in contrast to Upf1, eRF3a depletion globally derepressed the expression of mRNAs containing translated uORFs. Finally, we find that eRF3a and Upf1 have opposite effects on ribosome protein gene expression. Together, our results provide important elements for understanding the impact of translation termination and NMD on the human transcriptome and reveal novel determinants of ribosome biogenesis regulation.\", 'Series_overall_design': 'Examination of the translational consequences of the inactivation of the eRF3 and Upf1 in HCT-116', 'Series_type': 'Other', 'Series_contributor': 'Olivier,,Jean-Jean', 'Series_sample_id': 'GSM3597573', 'Series_contact_name': 'Olivier,,Namy', 'Series_contact_email': 'olivier.namy@i2bc.paris-saclay.fr', 'Series_contact_laboratory': 'Genomic, structure and Translation', 'Series_contact_department': 'I2BC', 'Series_contact_institute': 'CNRS, Université Paris-Sud', 'Series_contact_address': 'Rue Gregor Mendel, bat 400', 'Series_contact_city': 'Orsay', 'Series_contact_zip/postal_code': '91400', 'Series_contact_country': 'France', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE126nnn/GSE126385/suppl/GSE126385_RNASEQ_raw_trcounts.xlsx', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP185496'}\n",
      "272 PRJNA523167 GSE126736 {'Series_title': 'ribosome profiling of cell line models of human breast cancer', 'Series_geo_accession': 'GSE126736', 'Series_status': 'Public on Dec 05 2019', 'Series_submission_date': 'Feb 19 2019', 'Series_last_update_date': 'Jan 13 2020', 'Series_pubmed_id': '31911279', 'Series_summary': \"We performed ribosome profiling in cell line models of human breast cancer. We profiled 2 representative estrogen receptor-positive (T47D and ZR75-1) and 2 representative triple negative (SUM159PT and MDA-MB-231) cell lines. Translational profiles of a nonmalignant (Human Mammary Epithelial Cells, HMECs) and a nontumorigenic (MCF10A cells) counterparts were also analyzed. We analyzed and compared translational efficiencies, variances of the translational efficiencies, differential gene and transcript expression between malignant and non-malignant cell lines. We performed discriminative motif analysis of the 5'untranslated regions of the transcripts that are commonly transcribed in nonmalignant and malignant cells but preferentially translated in malignant cells and identified common motifs. Some of these motifs are highly similar to the RNA binding motifs of a small list of RNA interacting proteins. This approach led us to identify SRSF1 as an RNA binding protein commonly expressed among all analyzed cell lines that has a pervasive impact on the translational landscape of a representative estrogen receptor-positive and -negative cell line\", 'Series_overall_design': 'Ribosome profiling of 6 different cell lines in at least 2 biological replicates', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'William,E,Grizzle', 'Series_sample_id': 'GSM3611955', 'Series_contact_name': 'Christos,,Vaklavas', 'Series_contact_email': 'cvaklavas@uabmc.edu', 'Series_contact_phone': '2059345677', 'Series_contact_department': 'Medicine', 'Series_contact_institute': 'University of Alabama at Birmingham', 'Series_contact_address': 'NP2540M 1720 2nd Ave S', 'Series_contact_city': 'Birmingham', 'Series_contact_state': 'Alabama', 'Series_contact_zip/postal_code': '35294', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE126nnn/GSE126736/suppl/GSE126736_translational_efficiency_ratio_transcripts.csv.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP186252'}\n",
      "273 PRJNA525090 GSE127713 {'Series_title': 'Cellular gene expression during Hepatitis C Virus replication revealed by Ribosome profiling', 'Series_geo_accession': 'GSE127713', 'Series_status': 'Public on Mar 02 2019', 'Series_submission_date': 'Mar 01 2019', 'Series_last_update_date': 'Mar 02 2021', 'Series_pubmed_id': '30875926', 'Series_summary': 'Conclusion: After establishing HCV replication, cellular gene expression is reprogrammed towards stress response and HCC development. Downregulation of mitochondrial respiratory chain genes indicates how a virus induces cancer cell-like metabolic reprogramming (\"Warburg effect\"). Thus, HCV escapes stress response pathways but induces selective gene expression changes which likely are beneficial for chronic infection and cancerogenesis.', 'Series_overall_design': 'Transcriptome and translatome changes in Huh-7.5 hepatoma cells replicating HCV for 6 days in comparison to non-infected cells were analyzed, in triplicates, by NGS.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Michael,,Niepmann', 'Series_sample_id': 'GSM3637575', 'Series_contact_name': 'Audrey,M,Michel', 'Series_contact_email': 'A.Michel@ucc.ie', 'Series_contact_phone': '+353 876217126', 'Series_contact_laboratory': 'Baranov', 'Series_contact_department': 'School of Biochemistry and Cell Biology', 'Series_contact_institute': 'University College Cork', 'Series_contact_address': 'Western Gateway, Western Road', 'Series_contact_city': 'Cork', 'Series_contact_state': 'Cork', 'Series_contact_zip/postal_code': 'T12 XF62', 'Series_contact_country': 'Ireland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE127nnn/GSE127713/suppl/GSE127713_Gerresheim_et_al_2019_IJMS_Riboseq_HCV_vs_mock.xlsx', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP187206'}\n",
      "274 PRJNA525595 GSE127827 {'Series_title': 'Selective translation by alternative bacterial ribosomes', 'Series_geo_accession': 'GSE127827', 'Series_status': 'Public on Apr 11 2019', 'Series_submission_date': 'Mar 05 2019', 'Series_last_update_date': 'Aug 31 2020', 'Series_pubmed_id': '32723820', 'Series_summary': \"Mycobacteria encode several additional alternative ribosomal proteins (AltRP) and the funtion of AltRP invovling in translation is little known. In this study, we investigate the different translation landscape of AltRibo and CanRibo and found AltRibo have a relative 5' reads accumlation and preferential codon usage.\", 'Series_overall_design': 'Please note that processed data was generated from multiple samples/replicates as indicated in the corresponding sample description field.', 'Series_type': 'Other', 'Series_contributor': 'Zhiyu,,Xu', 'Series_sample_id': 'GSM3639499', 'Series_contact_name': 'Babak,,Javid', 'Series_contact_email': 'bjavid@gmail.com', 'Series_contact_phone': '+86-10-62771552', 'Series_contact_laboratory': 'Javid Lab', 'Series_contact_department': 'School of medicine', 'Series_contact_institute': 'Tsinghua University', 'Series_contact_address': 'Biotechnology Building Room 4302, Tsinghua University, Haidian District', 'Series_contact_city': 'Beijing', 'Series_contact_zip/postal_code': '100084', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE127nnn/GSE127827/suppl/GSE127827_StrainB_CanRibo_TotalRibo.edgeR.table.txt.gz', 'Series_platform_id': 'GPL26255', 'Series_platform_taxid': '1772', 'Series_sample_taxid': '1772', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP187517'}\n",
      "275 PRJNA526538 GSE128136 {'Series_title': 'Full-length transcriptome reconstruction reveals large diversity of RNA isoforms and open reading frames in rat hippocampus', 'Series_geo_accession': 'GSE128136', 'Series_status': 'Public on Oct 14 2019', 'Series_submission_date': 'Mar 11 2019', 'Series_last_update_date': 'Nov 04 2019', 'Series_pubmed_id': '31676752', 'Series_summary': 'We developed a hybrid-sequencing workflow, combining next-generation and third-generation sequencing, to reconstruct full-length transcriptomes. Integrating with polysome profiling and ribosome footprinting data, we predicted isoform–specific translational status and reconstructed ORFeome. Moreover, we identified isoforms with specific subcellular localization pattern in neurons.', 'Series_overall_design': 'At least two replicates were performed on the adult rat hippocampus.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Wei,,Chen', 'Series_sample_id': 'GSM3665494', 'Series_contact_name': 'Xi,,Wang', 'Series_contact_institute': 'Nanjing Medical University', 'Series_contact_address': 'Longmian Avenue 101, Jiangning District', 'Series_contact_city': 'Nanjing', 'Series_contact_zip/postal_code': '21116', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE128nnn/GSE128136/suppl/GSE128136_soma_neur.isoform.TPM.tsv.gz', 'Series_platform_id': 'GPL22396', 'Series_platform_taxid': '10116', 'Series_sample_taxid': '10116', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP188139'}\n",
      "276 PRJNA527039 GSE128298 {'Series_title': 'Translatome and transcriptome analysis of resting B and T cells from mice', 'Series_geo_accession': 'GSE128298', 'Series_status': 'Public on Aug 07 2019', 'Series_submission_date': 'Mar 14 2019', 'Series_last_update_date': 'Aug 09 2019', 'Series_pubmed_id': '31318442', 'Series_summary': 'We isolated ex vivo resting state B and T lymphocytes from B6 mouse spleen using magnetic separation (Miltenyi). After 2 hours of rest in RPMI mediaum, these cells were either used to isolate total RNA for RNAseq library preparation or treated with cycloheximide and proceeded with RNase digestion and library preparation for ribosome profiling (Riboseq).', 'Series_overall_design': 'RNAseq and Riboseq for resting state total B and T cells.', 'Series_type': 'Other', 'Series_contributor': 'Gagan,,Dev', 'Series_sample_id': 'GSM3670849', 'Series_contact_name': 'G Aneeshkumar,,Arimbasseri', 'Series_contact_email': 'aneesh@nii.ac.in', 'Series_contact_phone': '+91 9971986746', 'Series_contact_laboratory': 'Molecular Genetics Laboratory', 'Series_contact_department': 'Molecular Genetics Laboratory', 'Series_contact_institute': 'National Institute of Immunology', 'Series_contact_address': 'Aruna Asaf Ali Marg', 'Series_contact_city': 'New Delhi', 'Series_contact_state': 'Delhi', 'Series_contact_zip/postal_code': '110067', 'Series_contact_country': 'India', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE128nnn/GSE128298/suppl/GSE128298_RAW.tar', 'Series_platform_id': 'GPL24247', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP188420'}\n",
      "277 PRJNA527113 GSE128320 {'Series_title': 'The ribosome profiling strategy for monitoring translation in vivo by deep sequencing of ribosome-protected mRNA fragments in HuH-7 cells', 'Series_geo_accession': 'GSE128320', 'Series_status': 'Public on Apr 30 2020', 'Series_submission_date': 'Mar 14 2019', 'Series_last_update_date': 'Apr 30 2020', 'Series_pubmed_id': '31240521', 'Series_summary': 'To globally analyse the lncRNAs with potential coding ability of HCC cells, we performed Ribo-seq using HuH-7 cells. The cycloheximide (CHX) was added to inhibit the translational elongation of ribosomes of the HuH-7 cells.Furthermore,we identified the candidate ORFs with the selected RPF reads using the RiboCode software.', 'Series_overall_design': 'Ribo-seq profiles of HuH-7 cells were generated by deep sequencing, in\\xa0two independent\\xa0experiments, using  Xten Illumina sequencing platform. The ribo-seq data of the two biological replicates were integrated.Then,for the analysis of Ribo-seq data, sequenced reads that were trimmed for the adaptor sequence were aligned to the UCSC hg19 reference genome by STAR (version 2.4.2a) with parameters provided by the RiboCode (version 1.2.10) manual.', 'Series_type': 'Other', 'Series_contributor': 'Liang-Hu,,Qu', 'Series_sample_id': 'GSM3671371', 'Series_contact_name': 'Jian-Hua,,Yang', 'Series_contact_email': 'yangjh7@mail.sysu.edu.cn', 'Series_contact_phone': '86-20-84112517', 'Series_contact_department': 'School of Life Sciences', 'Series_contact_institute': 'Sun Yat-sen University', 'Series_contact_address': 'No. 135, Xingang Xi Road', 'Series_contact_city': 'Guangzhou', 'Series_contact_zip/postal_code': '510275', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE128nnn/GSE128320/suppl/GSE128320_Huh7_Ribo-seq_modified.bed12.gz', 'Series_platform_id': 'GPL20795', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP188452'}\n",
      "278 PRJNA527127 GSE128322 {'Series_title': 'Ribosome profiling of HSV-1 infected cells', 'Series_geo_accession': 'GSE128322', 'Series_status': 'Public on Apr 07 2020', 'Series_submission_date': 'Mar 14 2019', 'Series_last_update_date': 'May 07 2020', 'Series_pubmed_id': '32341360', 'Series_summary': 'Primary human foreskin fibroblasts (HFF) were infected with wild-type simplex virus 1 (HSV-1) strain 17 at a multiplicity of infection (MOI) of 10. Ribosome profiling was performed at various times during infection with minor modification to the protocol described in Stern-Ginossar N et al., Science 2012. Translation start site profiling was performed by culturing cells in presence of Harringtonin or Lactimidomycin for 30 min prior to cell harvest.', 'Series_overall_design': 'Ribosome profiling was performed a 0, 1, 2, 4, 6 and 8 h post infection. Translation start site profiling was performed using Harringtonin (2µ/ml for for 30 min) or Lactimidomycin (50µM for 30 min) prior to cell lysis. Two biological replicates were analysed.', 'Series_type': 'Other', 'Series_contributor': 'Lars,,Dölken', 'Series_sample_id': 'GSM3671393', 'Series_contact_name': 'Florian,,Erhard', 'Series_contact_email': 'Florian.Erhard@uni-wuerzburg.de', 'Series_contact_department': 'Institut für Virologie und Immunbiologie', 'Series_contact_institute': 'Julius-Maximilians-Universität Würzburg', 'Series_contact_address': 'Versbacher Str. 7', 'Series_contact_city': 'Würzburg', 'Series_contact_zip/postal_code': '97078', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE128nnn/GSE128322/suppl/GSE128322_Human.B.tsv.gz', 'Series_platform_id': 'GPL26303', 'Series_platform_taxid': '10299', 'Series_sample_taxid': '10299', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP188457'}\n",
      "279 PRJNA527204 GSE128344 {'Series_title': 'A large panel of isogenic APP and PSEN1 mutant human iPSC neurons reveals shared endosomal abnormalities mediated by APP b-CTFs, not Ab [ribosome profiling]', 'Series_geo_accession': 'GSE128344', 'Series_status': 'Public on Aug 08 2019', 'Series_submission_date': 'Mar 15 2019', 'Series_last_update_date': 'Mar 06 2021', 'Series_pubmed_id': '31416668', 'Series_summary': \"Human iPS cells with different mutations linked to Alzheimer's Disease were differentiated into neurons and subjected to ribosome profiling to identify Alzheimer's Disease associated changes\", 'Series_overall_design': 'Ribosome profiling analysis of wildtype and three different mutated human iPS cells differentiated into neurons before', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Marc,,Tessier-Lavigne', 'Series_sample_id': 'GSM3671691', 'Series_contact_name': 'Claudia,,Scheckel', 'Series_contact_email': 'claudia.scheckel@gmail.com', 'Series_contact_department': 'Institute of Neuropathology', 'Series_contact_institute': 'University Hospital Zurich', 'Series_contact_address': 'Schmelzbergstrasse 12', 'Series_contact_city': 'Zurich', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '8091', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE128nnn/GSE128344/suppl/GSE128344_RP_raw.txt.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP188505'}\n",
      "280 PRJNA528026 GSE128538 {'Series_title': 'Ribosome profiling in RPEI cells upon CDK1-inhibition', 'Series_geo_accession': 'GSE128538', 'Series_status': 'Public on Feb 06 2020', 'Series_submission_date': 'Mar 19 2019', 'Series_last_update_date': 'Feb 12 2020', 'Series_pubmed_id': '32040547', 'Series_summary': 'Protein synthesis is the second most energy-intensive process in the cell. Therefore, cells tightly control mRNA translation rates in response to external stimuli (growth factors, differentiation signals) and the internal resources available (nutrients, energy). We set out to unravel new molecular mechanisms of global translation control required for cellular homeostasis. Using a siRNA screening approach to knock-down all human kinases and phosphatases, we analyzed the impact of protein phosphorylation on translation regulation in unstressed, proliferating cells. A visual screen was conducted to assess stress granule formation (as a consequence of translational shutdown) in the absence of 711 kinases and 256 phosphatases. Several candidates identified in the screen are linked, either directly or indirectly, to cell cycle regulation. We further identified a cyclin-dependent kinase (CDK), whose genetic downregulation or chemical inhibition results in global translation suppression. In order to assess CDK downstream signaling, we performed phosphoproteomics analysis and found the phosphorylation status of many ribosomal proteins to be altered upon CDK inhibition. Indeed, we could show that S6-kinase 1 partially contributes to CDK-dependent translational control. Ribosome profiling further revealed that the translation of 5’TOP mRNAs, including many mRNAs encoding for ribosomal proteins, is especially sensitive to CDK inhibition. While the translation regulatory function of this kinase is independent of the cell cycle phase, its activity is ideally suited to couple the overall proliferation rate of cells to the global rate of protein synthesis. Our results identify a pathway that enables cells to directly adapt protein homeostasis to proliferation and growth signals.', 'Series_overall_design': 'RO-3306 or DMSO treated RPEI cells were subjected to ribosome profiling in three biological replicates. As a spike-in to detect global changes in translation efficiency, S. cerevisiae lysate was added to the RPEI lysates. To determine ribosome densities, both ribosome protected fragments and alkaline-fragmented input material were analyzed, amounting to 12 samples in total.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Georg,,Stoecklin', 'Series_sample_id': 'GSM3679628', 'Series_contact_name': 'Johanna,Daniela,Schott', 'Series_contact_email': 'johanna.schott@medma.uni-heidelberg.de', 'Series_contact_laboratory': 'Stoecklin lab', 'Series_contact_department': 'Mannheim Institute for Innate Immunoscience', 'Series_contact_institute': 'Heidelberg University', 'Series_contact_address': 'Ludolf-Krehl-Str. 13-17', 'Series_contact_city': 'Mannheim', 'Series_contact_zip/postal_code': '68167', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE128nnn/GSE128538/suppl/GSE128538_RiboSeq_CDKi_yeast.csv.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP188848'}\n",
      "281 PRJNA528192 GSE128578 {'Series_title': 'Ribosomal protein S7 ubiquitination during ER stress in yeast is associated with selective mRNA translation and stress outcome.', 'Series_geo_accession': 'GSE128578', 'Series_status': 'Public on Nov 12 2020', 'Series_submission_date': 'Mar 20 2019', 'Series_last_update_date': 'Feb 11 2021', 'Series_pubmed_id': '33184379', 'Series_summary': 'eIF2α phosphorylation-mediated translational regulation is crucial for global translation repression by various stresses, including the unfolded protein response (UPR). However, translational control during UPR has not been demonstrated in yeast. This study investigated ribosome ubiquitination-mediated translational controls during UPR. Tunicamycin-induced ER stress enhanced the levels of ubiquitination of the ribosomal proteins uS10, uS3 and eS7. Not4-mediated monoubiquitination of eS7A was required for resistance to tunicamycin, whereas E3 ligase Hel2-mediated ubiquitination of uS10 was not. Ribosome profiling showed that the monoubiquitination of eS7A was crucial for translational regulation, including the upregulation of the spliced form of HAC1 (HAC1i) mRNA and the downregulation of Histidine triad NucleoTide-binding 1 (HNT1) mRNA. Downregulation of the deubiquitinating enzyme complex Upb3-Bre5 increased the levels of ubiquitinated eS7A during UPR in an Ire1-independent manner. These findings suggest that the monoubiquitination of ribosomal protein eS7A plays a crucial role in translational controls during the ER stress response in yeast.', 'Series_overall_design': 'eS7-WT and eS7-4KR cells were harvested at 0 and 4 hour after Tunicamycin treatment.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Toshifumi,,Inada', 'Series_sample_id': 'GSM3680564', 'Series_contact_name': 'Yoshitaka,,Matsuo', 'Series_contact_department': 'IMSUT', 'Series_contact_institute': 'The University of Tokyo,', 'Series_contact_address': '4-6-1, Shirokanedai, Minato-Ku', 'Series_contact_city': 'Tokyo', 'Series_contact_zip/postal_code': '108-8639', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE128nnn/GSE128578/suppl/GSE128578_RAW.tar', 'Series_platform_id': 'GPL21656', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP188933'}\n",
      "282 PRJNA528488 GSE128680 {'Series_title': 'Multiscale gene expression profiling of 1 cm root tips of control and submerged Oryza sativa, Medicago truncatula, Solanum lycopersicum and Solanum pennellii seedlings', 'Series_geo_accession': 'GSE128680', 'Series_status': 'Public on Sep 19 2019', 'Series_submission_date': 'Mar 21 2019', 'Series_last_update_date': 'Jan 15 2020', 'Series_pubmed_id': '31604238', 'Series_summary': 'Climate change has increased the frequency and intensity of floods that impact global agricultural productivity. To better understand the response mechanisms and evolutionary history of gene family member regulation across angiosperm phyla, we studied the rapid submergence response of rice, the legume Medicago truncatula, and two Solanum species, domesticated tomato (S. lycopersicum cv. M82) and its dryland-adapted wild relative S. pennellii. Response to hypoxic conditions was measured by analyzing transcriptional and post-translational regulation in root tips of each species. This was achieved by the use of Nuclei Tagged in specific Cell Types (INTACT) and Translating Ribosome Affinity Purification to obtain chromatin and sub-populations of gene transcripts. (1) Chromatin accessibility was evaluated by coupling INTACT with ATAC-seq (assay for Transposon-Accessible Chromatin). (2) INTACT was used to capture nuclear RNA (nRNA). (3) Polyadenylated mRNA (polyA RNA) was obtained by standard oligo(dT) selection. (4)  Ribosome-associated polyA mRNA (polyA RNA) was obtained by use of Translating Ribosome Affinity Purification (TRAP).  Ribosome footprinting (Ribo-seq) was accomplished by using TRAP to capture ribosome protected fragments after RNAseI digestion. Samples evaluated include the apical root tip (four species) and shoot region (Solanum species only) under control conditions and after 2 h of submergence', 'Series_overall_design': '[2] The processed data generated from multiple samples is linked as Series supplementary files and is inidicated in the corresponding sample description field.', 'Series_type': 'Other', 'Series_contributor': 'Roger,B,Deal', 'Series_sample_id': 'GSM3682444', 'Series_contact_name': 'Roger,B,Deal', 'Series_contact_email': 'roger.deal@emory.edu', 'Series_contact_phone': '404-727-8087', 'Series_contact_department': 'Biology', 'Series_contact_institute': 'Emory University', 'Series_contact_address': '1510 Clifton Rd NE', 'Series_contact_city': 'Atlanta', 'Series_contact_state': 'GA', 'Series_contact_zip/postal_code': '30322', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE128nnn/GSE128680/suppl/GSE128680_bHLH-SP-WG.bed.gz', 'Series_platform_id': 'GPL26333', 'Series_platform_taxid': '39947', 'Series_sample_taxid': '39947', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP189094'}\n",
      "283 PRJNA530200 GSE129128 {'Series_title': 'Cue2 and Slh1 define parallel pathways to rescue stalled ribosomes', 'Series_geo_accession': 'GSE129128', 'Series_status': 'Public on Jun 25 2019', 'Series_submission_date': 'Apr 01 2019', 'Series_last_update_date': 'Jul 09 2019', 'Series_pubmed_id': '31219035', 'Series_summary': 'Ribosome stalling at problematic sequences in mRNAs leads to collisions that trigger a collection of quality control events including ribosome rescue, targeting the nascent polypeptide for decay (Ribosome-mediated Quality Control or RQC), and targeting of the mRNA for decay (No Go Decay or NGD). Using a reverse genetic screen in yeast, we identify Cue2 as the endonuclease that is recruited to stalled ribosomes to promote NGD. Following Cue2-mediated cleavage, ribosomes upstream of the cleavage site translate to the end of the truncated mRNA and are rescued by the Dom34:Hbs1 complex. We also show that the putative helicase Slh1 (part of the RQC Trigger or RQT complex) removes collided ribosomes behind the lead stalled ribosome and thereby reduces endonucleolytic cleavage by Cue2. The synergistic activities of Cue2 and Slh1 define two parallel pathways that allow cells to recognize and respond to ribosomes trapped on problematic mRNAs.', 'Series_overall_design': '7 samples for ribosome profiling experiments using various yeast strains and 3 samples for in vitro ribosome footprinting experiments using isloated trisomes from sucrose gradients.', 'Series_type': 'Other', 'Series_contributor': 'Rachel,,Green', 'Series_sample_id': 'GSM3893150', 'Series_contact_name': 'Colin,,Wu', 'Series_contact_department': 'RNA Biology Laboratory', 'Series_contact_institute': 'NCI', 'Series_contact_address': '1050 Boyles St, BG 560, RM21-102C', 'Series_contact_city': 'Frederick', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21702', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE129nnn/GSE129128/suppl/GSE129128_GFP_traces_GSM3893141-GSM3893150.csv.gz', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP190019'}\n",
      "284 PRJNA530508 GSE129194 {'Series_title': 'm6A in mRNA coding regions promotes translation via the RNA helicase-containing YTHDC2', 'Series_geo_accession': 'GSE129194', 'Series_status': 'Public on Jul 01 2019', 'Series_submission_date': 'Apr 02 2019', 'Series_last_update_date': 'Dec 03 2019', 'Series_pubmed_id': '31767846', 'Series_summary': 'Dynamic mRNA modification in the form of N6-methyladenosine (m6A) adds considerable richness and sophistication to gene regulation. The m6A mark is asymmetrically distributed along mature mRNAs, with a strong preference around the stop codon and 3’UTR. Nevertheless, approximately 35% of m6A residues are located within the coding region (CDS). It has been suggested that methylation in CDS slows down translation elongation by interfering the decoding process. However, neither the decoding feature of endogenous mRNAs nor the physiological significance of CDS m6A has been clearly defined. By integrating Ribo-seq and m6A-seq data sets, we found that CDS m6A methylation leads to ribosome pausing in a codon-specific manner. Unexpectedly, removing CDS m6A modification from these transcripts results in a further decrease of translation. A systemic analysis of RNA structural datasets revealed that CDS m6A methylation positively regulates translation by resolving mRNA secondary structures. We further demonstrate that the elongation-promoting effect of CDS methylation requires the RNA helicase-containing m6A reader YTHDC2. Our findings established the physiological significance of CDS methylation and uncovered non-overlapping function of m6A reader proteins.', 'Series_overall_design': 'Using Ribo-seq to investigate the effect of m6A/m6A reader on translation efficiency', 'Series_type': 'Other', 'Series_contributor': 'Shu-Bing,,Qian', 'Series_sample_id': 'GSM3702267', 'Series_contact_name': 'Yuanhui,,Mao', 'Series_contact_email': 'maoyuanhui123@gmail.com', 'Series_contact_institute': 'Zhejiang University', 'Series_contact_address': '1369 West Wenyi Road', 'Series_contact_city': 'Hangzhou', 'Series_contact_state': 'Zhejiang', 'Series_contact_zip/postal_code': '310000', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE129nnn/GSE129194/suppl/GSE129194_RAW.tar', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP190145'}\n",
      "285 PRJNA530968 GSE129305 {'Series_title': 'Splicing modulators elicit global translational repression by condensate-prone proteins translated from introns', 'Series_geo_accession': 'GSE129305', 'Series_status': 'Public on Mar 29 2021', 'Series_submission_date': 'Apr 03 2019', 'Series_last_update_date': 'Dec 06 2021', 'Series_pubmed_id': '34520743', 'Series_summary': 'Chemical splicing modulators that bind to the spliceosome have provided an attractive venue for cancer treatment. Splicing modulators induce accumulation and subsequent translation of a subset of intron-retained mRNAs. Yet, the biological effect of proteins containing translated intron sequences remains unclear. Here we identified a number of truncated proteins generated upon treatment with the splicing modulator spliceostatin A (SSA) using genome-wide ribosome profiling and bio-orthogonal non-canonical amino-acid tagging (BONCAT) mass spectrometry. A subset of these truncated proteins has intrinsically disordered regions, forms insoluble cellular condensates, and triggers the proteotoxic stress response through JNK phosphorylation, thereby inhibiting the mTORC1 pathway. In turn, this reduces global translation. These findings indicate that creating an overburden of condensate-prone proteins derived from introns represses translation and prevents further production of harmful truncated proteins. This mechanism appears to contribute to the antiproliferative and proapoptotic activity of splicing modulators.', 'Series_overall_design': 'Broadly, the objective of the study is to understand how cells cope with the aberrant splicing. We sought for the fates of each transcripts upon splicing modulation by a small molecule compound, spliceostatin A (SSA). Also, SSA is antitumor compound but the reason for the selective killing of cancer cells in vitro and in vivo has been unrevealed. Our study provides insight over it as well. we treated the HeLa S3 cells with either 100ng/mL SSA or two replicates with methanol (MeOH) (solvent for SSA as negative control) for 6 hours. Same cell lysate was simultaneously used to prepare RNA-seq and Ribosome profiling libraries.', 'Series_type': 'Non-coding RNA profiling by array', 'Series_contributor': 'Minoru,,Yoshida', 'Series_sample_id': 'GSM3704310', 'Series_contact_name': 'Shintaro,,Iwasaki', 'Series_contact_email': 'shintaro.iwasaki@riken.jp', 'Series_contact_phone': '+81-48-467-3613', 'Series_contact_laboratory': 'RNA Systems Biochemistry Laboratory', 'Series_contact_institute': 'RIKEN', 'Series_contact_address': 'Bioscience Bldg. 2-1, Hirosawa', 'Series_contact_city': 'Wako', 'Series_contact_state': 'Saitama', 'Series_contact_zip/postal_code': '351-0198', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE129nnn/GSE129305/suppl/GSE129305_RAW.tar', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP190851'}\n",
      "286 PRJNA532532 GSE129701 {'Series_title': 'Super-enhancer redistribution as a mechanism of broad gene dysregulation in repeatedly drug-treated cancer cells [dataset 2]', 'Series_geo_accession': 'GSE129701', 'Series_status': 'Public on Apr 01 2020', 'Series_submission_date': 'Apr 12 2019', 'Series_last_update_date': 'Mar 08 2021', 'Series_pubmed_id': '32320655', 'Series_summary': 'We provide a series of chromatin and transcriptomic profiles to establish the short-term and long-term effects of cisplatin treatment in ovarian cancer A2780 cells.', 'Series_overall_design': 'All ChIP-Seq, GRO-seq, RNA-seq, and ribosome-profiling experiments were designed to understand the effects of cisplatin on 6 different layers of gene-expression regulation in cancer cells.', 'Series_type': 'Other', 'Series_contributor': 'Ivan,,Garcia-Bassets', 'Series_sample_id': 'GSM3720438', 'Series_contact_name': 'Qi,,Ma', 'Series_contact_email': 'q1ma@ucsd.edu', 'Series_contact_institute': 'UCSD', 'Series_contact_address': '9500 Gilman Dr,', 'Series_contact_city': 'San Diego', 'Series_contact_zip/postal_code': '92093', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE129nnn/GSE129701/suppl/GSE129701_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP192310'}\n",
      "287 PRJNA532643 GSE129757 {'Series_title': 'Whole transcriptome profiles and ribosome profiling of thapsigargin- and tunicamycin-treated LN308 cells', 'Series_geo_accession': 'GSE129757', 'Series_status': 'Public on Apr 30 2020', 'Series_submission_date': 'Apr 12 2019', 'Series_last_update_date': 'May 02 2020', 'Series_summary': 'Accumulation of unfolded proteins in the endoplasmic reticulum triggers the unfolded protein response (UPR), an adaptive signal transduction pathway aimed at reinstating cellular homeostasis, or, if that fails, at triggering of apoptosis. To gain a comprehensive and systems-wide understanding of the UPR, we have pursued a multi-omics approach that, upon chemical induction of the UPR with two different compounds, monitors in parallel several parameters in the astrocytoma cell line LN-308. Changes to the cellular transcriptome (by high throughput sequencing) and altered translation status (by ribosome profiling) were measured after 2h and 6h of treatment.', 'Series_overall_design': 'Whole transcriptome profiles and ribosome profiling libraries were generated for thapsigargin- and tunicamycin-treated LN308 cell lines by next generation sequencing (Illumina Next Seq 500). Samples were taken after 0, 2 and 6 hours of pharmacological treatment.', 'Series_type': 'Other', 'Series_contributor': 'Jan,,Medenbach', 'Series_sample_id': 'GSM3721670', 'Series_contact_name': 'Grischa,,Toedt', 'Series_contact_department': 'Computational Biology', 'Series_contact_institute': 'EMBL', 'Series_contact_address': 'Meyerhofstr. 1', 'Series_contact_city': 'Heidelberg', 'Series_contact_zip/postal_code': '69115', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE129nnn/GSE129757/suppl/GSE129757_Table_S3_-_Ribosome_Profiling.xlsx', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP192442'}\n",
      "288 PRJNA533074 GSE129869 {'Series_title': 'Ribo-seq of Toxoplasma-infected host cells', 'Series_geo_accession': 'GSE129869', 'Series_status': 'Public on May 17 2019', 'Series_submission_date': 'Apr 16 2019', 'Series_last_update_date': 'Aug 27 2019', 'Series_pubmed_id': '31167946', 'Series_summary': 'Toxoplasma gondii is a ubiquitous obligate intracellular parasite that infects the nucleated cells of warm-blooded animals. From within the parasitophorous vacuole in which they reside, Toxoplasma tachyzoites secrete an arsenal of effector proteins that can reprogram host gene expression to facilitate parasite survival and replication. Gaining a better understanding of how host gene expression is altered upon infection is central for understanding parasite strategies for host invasion and for developing new parasite therapies. Here, we applied ribosome profiling coupled with mRNA measurements to concurrently study gene expression in the parasite and in host human foreskin fibroblasts. By examining the parasite transcriptome and translatome, we identified potential upstream open reading frames that may permit the stress-induced preferential translation of parasite mRNAs. We also determined that tachyzoites reduce host death-associated pathways and increase survival, proliferation, and motility in both quiescent and proliferative host cell models of infection. Additionally, proliferative cells alter their gene expression in ways consistent with massive transcriptional rewiring while quiescent cells were best characterized by re-entry into the cell cycle. We also identified a translational control regimen consistent with mTOR activation in quiescent cells, and to a lesser degree in proliferative cells. This study illustrates the utility of the method for dissection of gene expression programs simultaneously in parasite and host.', 'Series_overall_design': 'Ribosome profiling and RNA-seq of Toxoplasma-infected confluent and subconfluent human foreskin fibroblasts in triplicate', 'Series_type': 'Other', 'Series_contributor': 'William,J,Sullivan Jr', 'Series_sample_id': 'GSM3723773', 'Series_contact_name': 'Michael,James,Holmes', 'Series_contact_email': 'holmesmi@iu.edu', 'Series_contact_department': 'Pharmacology & Toxicology', 'Series_contact_institute': 'Indiana University School of Medicine', 'Series_contact_address': '635 Barnhill Drive', 'Series_contact_city': 'Indianapolis', 'Series_contact_state': 'IN', 'Series_contact_zip/postal_code': '46202', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE129nnn/GSE129869/suppl/GSE129869_RAW.tar', 'Series_platform_id': 'GPL26509', 'Series_platform_taxid': '383379', 'Series_sample_taxid': '383379', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP192707'}\n",
      "289 PRJNA533132 GSE129912 {'Series_title': 'Monosome profiling in starved HEK293 cells', 'Series_geo_accession': 'GSE129912', 'Series_status': 'Public on Aug 04 2022', 'Series_submission_date': 'Apr 16 2019', 'Series_last_update_date': 'Aug 05 2022', 'Series_summary': 'Ribosome profiling from monosome and polysome fractions of serum or amino acid starved HEK293 cells', 'Series_overall_design': 'Human embryonic kindney cells were subjected to 2h of serum or 1h of amino acid starvation. Ribonucleoprotein complexes were separated on polysome gradients. Ribosome protected fragments were purified from pooled monosome and polysome fractions.', 'Series_type': 'Other', 'Series_contributor': 'Utz,,Fischer', 'Series_sample_id': 'GSM3725005', 'Series_contact_name': 'Florian,,Erhard', 'Series_contact_email': 'Florian.Erhard@uni-wuerzburg.de', 'Series_contact_department': 'Institut für Virologie und Immunbiologie', 'Series_contact_institute': 'Julius-Maximilians-Universität Würzburg', 'Series_contact_address': 'Versbacher Str. 7', 'Series_contact_city': 'Würzburg', 'Series_contact_zip/postal_code': '97078', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE129nnn/GSE129912/suppl/GSE129912_price.orfs.tsv.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP192770'}\n",
      "290 PRJNA533159 GSE129924 {'Series_title': 'Free circular introns with an unusual branchpoint in neuronal projections', 'Series_geo_accession': 'GSE129924', 'Series_status': 'Public on Apr 18 2019', 'Series_submission_date': 'Apr 16 2019', 'Series_last_update_date': 'Nov 08 2019', 'Series_pubmed_id': '31697236', 'Series_summary': 'In eukaryotic cells, most introns are degraded soon after splicing in the nucleus but some persist either due to lack of splicing (detained/retained introns) or because they contain important functional elements, for example, sno/scaRNAs. Few introns are detectable outside the nucleus. To hunt for interesting new phenomena in cytoplasmic introns and splicing, we conducted a multimodal study of total RNA within projections (axons, dendrites, glial projections) of rat hippocampal neurons and discovered a class of free circular introns enriched in distal projections.', 'Series_overall_design': 'A variety of sequencing datasets from rat primary hippocampal neurons are provided: From physically separated cellular projections and whole-cell lysates, (a) rRNA-depleted total RNA-seq (5 biological replicates, each) and (b) polyA site sequencing or PAS-seq (3 biological replicates, each); from fractionated cytoplasmic lysate (c) ribosome profiling (3 biological replicates) and (d) polyA+ RNA-seq (3 biological replicates).', 'Series_type': 'Other', 'Series_contributor': 'Melissa,J,Moore', 'Series_sample_id': 'GSM3728213', 'Series_contact_name': 'Harleen,,Saini', 'Series_contact_email': 'harleen_saini@hms.harvard.edu', 'Series_contact_laboratory': 'Danesh Moazed', 'Series_contact_institute': 'Harvard Medical School', 'Series_contact_address': '240 Longwood Avenue', 'Series_contact_city': 'Boston', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02115', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE129nnn/GSE129924/suppl/GSE129924_RAW.tar', 'Series_platform_id': 'GPL22396', 'Series_platform_taxid': '10116', 'Series_sample_taxid': '10116', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP192787'}\n",
      "291 PRJNA534111 GSE130155 {'Series_title': 'Effect of adat2 silencing in Neurospora crassa on translation kinetics and transcriptome', 'Series_geo_accession': 'GSE130155', 'Series_status': 'Public on May 22 2020', 'Series_submission_date': 'Apr 22 2019', 'Series_last_update_date': 'Jun 17 2020', 'Series_pubmed_id': '32479508', 'Series_summary': 'The silencing of tRNA-specific adenosine deaminase adat2 in Neurospora crassa abolished most of the I34 modification, resulting in major tRNA profile changes. The adat2 silencing caused genome-wide codon usage-biased ribosome pausing on mRNAs corresponding to changes in the tRNA profile, which reprograms translation elongation kinetics on ADAT-related codons. The ribosome profiling in this study was used to measure the effect of defective I34 modification of the eight tRNAs on translation kinetics during elongation stage. The accompanying mRNA-seq was used to normalize the ribosome profiling data using the same library construction protocol (reference to ARTseq™ Ribosome Profiling Kit (Catalog Number: RPYSC12116)) under the same growth condition. The independent mRNA-seq was used to measure the effect of I34 abolishment on transcriptome change using standard Illumina protocols. Meanwhile, to isolate the role of codon usage in translation elongation and protein level changes when adat2 is silenced, the independent mRNA-seq was also used to exclude the effect of mRNA level changes.', 'Series_overall_design': 'The transcription profile and ribosome protected fragment (RPF) profiles of wild-type strain and adat2-silenced mutant of Neurospora crassa were generated by deep sequencing using Illumina sequencing platform.', 'Series_type': 'Other', 'Series_contributor': 'Yi,,Liu', 'Series_sample_id': 'GSM3905562', 'Series_contact_name': 'xueliang,,lyu', 'Series_contact_email': 'lvxueliang0715@aliyun.com', 'Series_contact_laboratory': 'Yi Liu', 'Series_contact_department': 'Physiology', 'Series_contact_institute': 'UT Southwestern Medical Center', 'Series_contact_address': '6001 Forest Park Road', 'Series_contact_city': 'Dallas', 'Series_contact_state': 'Texas', 'Series_contact_zip/postal_code': '75235', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE130nnn/GSE130155/suppl/GSE130155_RAW.tar', 'Series_platform_id': 'GPL26551', 'Series_platform_taxid': '5141', 'Series_sample_taxid': '5141', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP193378'}\n",
      "292 PRJNA540298 GSE130465 {'Series_title': \"Multiplexed functional genomic analysis of somatic 5' untranslated region mutations across the spectrum of human prostate cancer (Ribosome Profiling)\", 'Series_geo_accession': 'GSE130465', 'Series_status': 'Public on May 17 2021', 'Series_submission_date': 'Apr 29 2019', 'Series_last_update_date': 'Jul 27 2021', 'Series_pubmed_id': '34244513', 'Series_summary': 'The functional consequences of cancer patient-derived genetic variants within the 5’ untranslated regions (UTRs) on a genome-wide scale and their effects on mRNA transcript and translation levels are poorly understood. To systematically interrogate the mutational landscape of 5’ UTRs across cancer patients with localized to metastatic disease, we analyzed the genomes of 226 prostate cancer patients and observed thousands of mutations, many of which impact known cis-regulatory elements. We developed a high-throughput multi-layer massively parallel sequencing-based method called PLUMAGE (Pooled full-length UTR Multiplex Assay on Gene Expression) to simultaneously quantify the effects of 545 5’ UTR somatic mutations across recurrently mutated or cancer-related genes from our patient cohort. Our method enabled unprecedented insights into how 5’ UTR mutations can control multiple levels of gene expression simultaneously. In particular, we identified 190 mutations that significantly altered 5’ UTR function, either by creating new DNA binding elements, disrupting known translation regulatory motifs, or simultaneously impacting both transcript levels and mRNA translation. Furthermore, we also determined that 5’ UTR mutations to the MAP kinase signaling pathway are significantly associated with early metastasis. This study is the first to comprehensively interrogate the functional 5’ UTR mutational landscape of a human cancer revealing the importance of untranslated regions in regulating multiple levels of oncogenic gene expression and provides a high-throughput functional genomics solution applicable to many genetically driven diseases.', 'Series_overall_design': 'To investigate the functional impact of 5’ UTR mutations in advanced prostate cancer, we will conduct whole exome sequencing utilizing new 5’ UTR exon capture technology on a cohort of end stage castration resistant prostate cancer patient derived xenografts (PDXs). Derived from rapid autopsy specimens, these PDX models have never been cultured, faithfully recapitulate human disease, and represent a nearly inexhaustible supply of tissues for biological replicates. A number of 5’ UTR somatic mutations, insertions, and deletions were identified. To directly correlate these findings with the translation of specific transcripts genome-wide, we will perform ribosome profiling on biological replicates of the same PDXs. Normal human prostate tissue will also be profiled as a comparison to expression in prostate cancer. This will be the very first functional analysis of the 5’ UTR regulome in a human cancer.', 'Series_type': 'Other', 'Series_contributor': 'Andrew,C,Hsieh', 'Series_sample_id': 'GSM5321289', 'Series_contact_name': 'Sonali,,Arora', 'Series_contact_email': 'sarora@fredhutch.org', 'Series_contact_institute': 'FHCRC', 'Series_contact_address': '1100 Fairview Ave N,', 'Series_contact_city': 'Seattle', 'Series_contact_state': 'WA', 'Series_contact_zip/postal_code': '98109', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE130nnn/GSE130465/suppl/GSE130465_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP194188'}\n",
      "293 PRJNA541324 GSE130781 {'Series_title': 'Transcriptome (RNA-Seq), translatome (Ribo-Seq), and YB-1/YB-3-bound mRNAs (RIP-Seq) of HEK293T, HEK293TdeltaYB-1, and HEK293T+HA-YB-1 cells', 'Series_geo_accession': 'GSE130781', 'Series_status': 'Public on Jan 08 2020', 'Series_submission_date': 'May 06 2019', 'Series_last_update_date': 'Apr 08 2020', 'Series_pubmed_id': '31944153', 'Series_summary': 'We present results of RNA-Seq, Ribo-Seq, and RIP-Seq (YB-1, YB-3) experiments performed in HEK293T cells, as well as in HEK293T cells with YB-1 knockout and overexpression. The data shows YB-1 function as a global translation inhibitor and YB-3 ability to substitute YB-1 in its function in YB-1 knockout mutant.', 'Series_overall_design': 'Cells were cultivated in Dulbecco’s Modified Eagle’s Medium (DMEM) supplemented with 10% fetal calf serum, 2 mM glutamine, 100 U/ml penicillin, and 100 µg streptomycin. The cells were incubated at 37°C in a humidified atmosphere containing 5% CO2 and passaged by standard methods. YB-1 knockout was performed with CRISPR/Cas9. YB-1 overexpression was achieved by transit transfection of pcDNA3.1 HA-YB1 plasmid. RIP-Seq was performed using (4202, CST; A303-230A, Bethyl) antibodies for YB-1 and (A303-070A, Bethyl) antibodies for YB-3. Libraries for Ribo-Seq were prepared with TruSeq Ribo Profile Kit (Illumina). The same kit was used for size-matched RNA-Seq (used as the control for Ribo-Seq). RIP-Seq and its control RNA-Seq were produced with NEBNext Ultra Directional RNA Library Prep Kit for Illumina (NEB). Ribosomal RNA depletion was performed with Ribo-Zero rRNA Removal Kit (Human/Mouse/Rat)(Illumina) for Ribo-Seq and with NEBNext Poly(A) mRNA Magnetic Isolation Module (NEB) or NEBNext rRNA Depletion Kit (Human/Mouse/Rat) (NEB) for RIP-Seq. Libraries were sequenced on Illumina HiSeq 2000 and Illumina NextSeq 500.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Irina,A,Eliseeva', 'Series_sample_id': 'GSM3753541', 'Series_contact_name': 'Ivan,V,Kulakovskiy', 'Series_contact_institute': 'Institute of Protein Research', 'Series_contact_address': 'Institutskaya 4', 'Series_contact_city': 'Pushchino', 'Series_contact_zip/postal_code': '142290', 'Series_contact_country': 'Russia', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE130nnn/GSE130781/suppl/GSE130781_RAW.tar', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP195632'}\n",
      "294 PRJNA542601 GSE131074 {'Series_title': 'Ribosome profiling of eIF3e-KD MCF-10a cells', 'Series_geo_accession': 'GSE131074', 'Series_status': 'Public on Jun 01 2020', 'Series_submission_date': 'May 13 2019', 'Series_last_update_date': 'Aug 31 2020', 'Series_pubmed_id': '32589965', 'Series_summary': 'eIF3 is a multi-subunit complex thought to execute numerous functions in canonical translation initiation, including mRNA recruitment to the 40S ribosome, scanning for the start codon, and inhibition of 60S subunit joining 1–3. eIF3 was also found to interact with 40S and 60S ribosomal proteins and translation elongation factors 4, but a direct involvement in translation elongation has never been demonstrated. Using selective ribosome profiling, we made the unexpected observation that eIF3 remains bound to post-initiation 80S ribosomes, followed by release after translation of ~50 codons. Furthermore, eIF3 deficiency reduces early ribosomal elongation speed, particularly on mRNAs encoding proteins associated with membrane-associated functions, resulting in defective synthesis of their encoded proteins and abnormal mitochondrial and lysosomal physiology.  Accordingly, heterozygous eIF3e+/- knockout mice accumulate giant mitochondria in skeletal muscle and show a progressive decline in muscle strength with age. Hence, in addition to its canonical role in translation initiation, eIF3 interacts with 80S ribosomes to enhance, at the level of early elongation, the synthesis of proteins with membrane-associated functions, an activity that is critical for normal muscle health.', 'Series_overall_design': 'Using si-RNA to do eIF3e-KD then do the ribosome profiling samples with total RNA, 3 replicates.', 'Series_type': 'Other', 'Series_contributor': 'Xudong,,Xing', 'Series_sample_id': 'GSM3940762', 'Series_contact_name': 'Xuerui,,Yang', 'Series_contact_email': 'yangxuerui@tsinghua.edu.cn', 'Series_contact_department': 'School of Life Sciences', 'Series_contact_institute': 'Tsinghua University', 'Series_contact_address': 'Haidian District', 'Series_contact_city': 'Beijing', 'Series_contact_zip/postal_code': '100084', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE131nnn/GSE131074/suppl/GSE131074_mRNA_counts.xlsx', 'Series_platform_id': 'GPL20795', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP198221'}\n",
      "295 PRJNA542722 GSE131154 {'Series_title': 'Fidelity of translation initiation is required for coordinated respiratory complex assembly', 'Series_geo_accession': 'GSE131154', 'Series_status': 'Public on Jan 13 2020', 'Series_submission_date': 'May 13 2019', 'Series_last_update_date': 'Jan 13 2020', 'Series_pubmed_id': '31903419', 'Series_summary': 'Mammalian mitochondrial ribosomes are unique molecular machines that translate 11 leaderless mRNAs. To date it is not clear how mitoribosomes recognize and initiate translation in the absence of untranslated regions in the mitochondrial mRNAs. Translation initiation in mitochondria shares similarities with prokaryotic systems, such as the formation of a ternary complex of fMet-tRNAMet, mRNA and the 28S subunit, but differs in the requirements for initiation factors. Mitochondria have two initiation factors, MTIF2 that closes the decoding centre and stabilizes the binding of the fMet-tRNAMet to the leaderless mRNAs, and MTIF3 whose role is not clear. We knocked out Mtif3 in mice and show that this protein is essential for embryo development and heart- and skeletal muscle-specific loss of MTIF3 causes premature death. We identify increased but uncoordinated mitochondrial protein synthesis in mice lacking MTIF3 that results in loss of specific respiratory complexes. Therefore, we show that coordinated assembly of OXPHOS complexes requires stoichiometric levels of nuclear and mitochondrially-encoded protein subunits in vivo. Our ribosome profiling and transcriptomic analyses show that MTIF3 is required for recognition and regulation of translation initiation of mitochondrial mRNAs, but not dissociation of the ribosome subunits.', 'Series_overall_design': 'Total RNA was isolated from heart tissue from 25 week old control (Mtif3loxP/loxP) and Mtif3 knockout mice (Mtif3loxP/loxP, +/Ckmm), and TruSeq libraries produced, sequenced and analysed for differential expression. Sucrose gradient fractions of mitochondrial RNA corresponding to small and large subunit, as well as assembled ribosomes, was isolated from heart tissue from 25 week old control (Mtif3loxP/loxP) and Mtif3 knockout mice (Mtif3loxP/loxP, +/Ckmm), and ribosome profiling libraries produced and sequenced.', 'Series_type': 'Other', 'Series_contributor': 'Aleksandra,,Filipovska', 'Series_sample_id': 'GSM3764763', 'Series_contact_name': 'Stefan,J,Siira', 'Series_contact_laboratory': 'Mitochondrial Medicine and Biology', 'Series_contact_department': 'Precision Health', 'Series_contact_institute': 'The Kids Research Institute Australia', 'Series_contact_address': '15 Hospital Ave, Nedlands', 'Series_contact_city': 'Perth', 'Series_contact_state': 'Western Australia', 'Series_contact_zip/postal_code': '6009', 'Series_contact_country': 'Australia', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE131nnn/GSE131154/suppl/GSE131154_RAW.tar', 'Series_platform_id': 'GPL16417', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP198313'}\n",
      "296 PRJNA543493 GSE131408 {'Series_title': 'The Ccr4-Not complex monitors the translating ribosome for codon optimality', 'Series_geo_accession': 'GSE131408', 'Series_status': 'Public on Apr 16 2020', 'Series_submission_date': 'May 17 2019', 'Series_last_update_date': 'Jun 10 2020', 'Series_pubmed_id': '32299921', 'Series_summary': 'Control of messenger RNA (mRNA) decay rate is intimately connected to translation elongation, but the spatial coordination of these events is poorly understood. The Ccr4-Not complex initiates mRNA decay through deadenylation and activation of decapping. We used a combination of cryo–electron microscopy, ribosome profiling, and mRNA stability assays to examine the recruitment of Ccr4-Not to the ribosome via specific interaction of the Not5 subunit with the ribosomal E-site in Saccharomyces cerevisiae. This interaction occurred when the ribosome lacked accommodated A-site transfer RNA, indicative of low codon optimality. Loss of the interaction resulted in the inability of the mRNA degradation machinery to sense codon optimality. Our findings elucidate a physical link between the Ccr4-Not complex and the ribosome and provide mechanistic insight into the coupling of decoding efficiency with mRNA stability.', 'Series_overall_design': '2 replicates in 2 different strains', 'Series_type': 'Other', 'Series_contributor': 'Roland,,Beckmann', 'Series_sample_id': 'GSM3775677', 'Series_contact_name': 'Yoshitaka,,Matsuo', 'Series_contact_department': 'IMSUT', 'Series_contact_institute': 'The University of Tokyo,', 'Series_contact_address': '4-6-1, Shirokanedai, Minato-Ku', 'Series_contact_city': 'Tokyo', 'Series_contact_zip/postal_code': '108-8639', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE131nnn/GSE131408/suppl/GSE131408_RAW.tar', 'Series_platform_id': 'GPL21656', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP198768'}\n",
      "297 PRJNA544411 GSE131650 {'Series_title': 'Pervasive functional translation of noncanonical open reading frames', 'Series_geo_accession': 'GSE131650', 'Series_status': 'Public on Mar 14 2020', 'Series_submission_date': 'May 22 2019', 'Series_last_update_date': 'Mar 15 2020', 'Series_pubmed_id': '32139545', 'Series_summary': 'Ribosome profiling has revealed pervasive but largely uncharacterized translation outside of canonical coding sequences (CDSs). Here, we exploit a systematic CRISPR-based screening strategy to identify hundreds of non-canonical CDSs that are essential for cellular growth and whose disruption elicit specific, robust transcriptomic and phenotypic changes in human cells. Functional characterization of the encoded microproteins reveals distinct cellular localizations, specific protein binding partners, and hundreds that are presented by the HLA system. Interestingly, we find multiple microproteins encoded in upstream open reading frames, which form stable complexes with the main, canonical protein encoded on the same mRNA, thus revealing the diverse use of functional bicistronic operons in mammals. Together, our results point to a family of functional human microproteins that play critical and diverse cellular roles.', 'Series_overall_design': 'Paired RNA-seq and ribosome profiling sequencing data for human induced pluripotent stem cells (iPSCs) and iPSC-derived cardiomyocytes,  either with or without the translational inhibitor harringtonine. Three replicates are availble for the no drug treatment samples.', 'Series_type': 'Other', 'Series_contributor': 'Jonathan,S,Weissman', 'Series_sample_id': 'GSM3791736', 'Series_contact_name': 'Jin,,Chen', 'Series_contact_email': 'jin.chen@utsouthwestern.edu', 'Series_contact_laboratory': 'Jin Chen', 'Series_contact_institute': 'UT Southwestern Medical Center', 'Series_contact_address': '5323 Harry Hines Blvd', 'Series_contact_city': 'Dallas', 'Series_contact_state': 'TX', 'Series_contact_zip/postal_code': '75390', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE131nnn/GSE131650/suppl/GSE131650_orfratings.bed.gz', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP199285'}\n",
      "298 PRJNA545050 GSE131809 {'Series_title': 'Control of human hemoglobin switching by LIN28B-mediated regulation of BCL11A translation (Ribo-seq, RNA-seq)', 'Series_geo_accession': 'GSE131809', 'Series_status': 'Public on Nov 16 2019', 'Series_submission_date': 'May 28 2019', 'Series_last_update_date': 'Feb 15 2020', 'Series_pubmed_id': '31959994', 'Series_summary': 'While BCL11A protein is not well synthesized at these earlier stages of development, its mRNA curiously continues to be associated with ribosomes. Through unbiased proteomic analyses in erythroid cells, we demonstrate that the RNA-binding protein LIN28B, which is developmentally expressed in a reciprocal pattern to BCL11A, directly interacts with ribosomes. We show that the observed suppression of BCL11A protein translation is mediated by LIN28B through a direct interaction with BCL11A mRNA and independent of its role in let-7 microRNA biogenesis. Finally, we show that BCL11A is the major functional target in LIN28B-mediated fetal hemoglobin induction. Our results reveal a previously unappreciated regulatory mechanism underlying human hemoglobin switching and illuminate opportunities for developing improved treatments for sickle cell disease and Beta-thalassemia', 'Series_overall_design': 'Four replicates of ribosome profiling in adult and newborn erythroid cells with matching RNA-seq per sample.', 'Series_type': 'Other', 'Series_contributor': 'Caleb,,Lareau', 'Series_sample_id': 'GSM3820039', 'Series_contact_name': 'Caleb,,Lareau', 'Series_contact_email': 'lareauc@mskcc.org', 'Series_contact_institute': 'Memorial Sloan Kettering', 'Series_contact_address': '417 E 68th St, Zuckerman - ZRC 1132', 'Series_contact_city': 'New York', 'Series_contact_state': 'New York', 'Series_contact_zip/postal_code': '10065', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE131nnn/GSE131809/suppl/GSE131809_RBF_RNAcounts_ADCB.tsv.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP199612'}\n",
      "299 PRJNA545625 GSE132007 {'Series_title': 'Suppression of ribosomal pausing by eIF5A is necessary to maintain the fidelity of start codon selection (Ribosome profiling)', 'Series_geo_accession': 'GSE132007', 'Series_status': 'Public on Dec 03 2019', 'Series_submission_date': 'May 31 2019', 'Series_last_update_date': 'Dec 31 2019', 'Series_pubmed_id': '31801078', 'Series_summary': \"Sequences within 5' untranslated regions (UTRs) dictate the site and efficiency of translation initiation. In this study, an unbiased screen designed to interrogate the 5' UTR-mediated regulation of the growth-promoting gene MYC unexpectedly revealed the ribosomal pause-relief factor eIF5A as a regulator of translation initiation codon selection. Depletion of eIF5A enhanced upstream translation within 5' UTRs across yeast and human transcriptomes, including on the MYC transcript where this resulted in increased production of an N-terminally extended protein. Furthermore, ribosome profiling experiments established that the function of eIF5A as a suppressor of ribosomal pausing at sites of suboptimal peptide bond formation is conserved in human cells. We present evidence that proximal ribosomal pausing on a transcript triggers enhanced usage of upstream suboptimal or non-canonical initiation codons. Thus, we propose that eIF5A functions not only to maintain efficient translation elongation in eukaryotic cells, but also to maintain the fidelity of translation initiation.\", 'Series_overall_design': 'Ribosome profiling (and complementary RNA-sequencing) was performed in HCT116 cells that received, via lentivirus, Cas9 and sgRNAs targeting either EIF5A, DOHH or non-targeting control sgRNAs. Two sgRNAs were used for each condition, and thus 6 biological samples were used for these experiments.', 'Series_type': 'Other', 'Series_contributor': 'Joshua,,Mendell', 'Series_sample_id': 'GSM3834401', 'Series_contact_name': 'Joshua,,Mendell', 'Series_contact_email': 'joshua.mendell@utsouthwestern.edu', 'Series_contact_department': 'Molecular Biology', 'Series_contact_institute': 'UT Southwestern Medical Center', 'Series_contact_address': '6000 Harry Hines Blvd., NA6.200A', 'Series_contact_city': 'Dallas', 'Series_contact_state': 'TX', 'Series_contact_zip/postal_code': '75390-9148', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE132nnn/GSE132007/suppl/GSE132007_RPF_processed_HM62-63.1_.xlsx', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP199985'}\n",
      "300 PRJNA548813 GSE132725 {'Series_title': 'Transcriptome and ribosome profiling of XRN1-null HEK293T human cell line generated by CRISPR/Cas9 gene editing', 'Series_geo_accession': 'GSE132725', 'Series_status': 'Public on Jul 25 2019', 'Series_submission_date': 'Jun 13 2019', 'Series_last_update_date': 'Jul 25 2019', 'Series_pubmed_id': '31340047', 'Series_summary': 'XRN1 is the major cytoplasmic exoribonuclease in eukaryotes, which degrades deadenylated and decapped mRNAs in the last step of the 5′–3′ mRNA decay pathway. Metazoan XRN1 interacts with decapping factors coupling the final stages of decay. Ribosome profiling revealed that XRN1 loss impacts not only on mRNA levels but also on the translational efficiency of many cellular transcripts likely as a consequence of incomplete decay. We compared differentially expressed genes derived from the analysis of the transcriptome and the translatome of HEK293T wild-type (WT) and XRN1-null cells by RNA-Seq and Ribo-Seq, respectively. Two biological replicates of each cell line were analyzed. The RNA-Seq analysis indicated that a substantial fraction of the total 10,105 genes showed significant differences between the two cell lines. 2,906 genes were significantly upregulated whilst 2,900 genes were downregulated using a fold change >0 on log2 scale with an FDR<0.005 to determine abundance. Significantly fewer genes showed differences between the WT and the XRN1-null cells at the level of translation with 433 genes showing an increase in abundance of ribosomal footprints whilst 560 genes were decreased as indicated by the Ribo-Seq analysis. Comparative analysis of translational efficiency (TE) in WT and XRN1-null cells on a genome-wide scale as a ratio of ribosomal occupancy to mRNA abundance identified an increase in TE for 102 genes but a decrease for 598 genes with most of the genes showing no evidence of change in the TE.', 'Series_overall_design': 'Two cell types with two biological replicates analysed using two methods.', 'Series_type': 'Other', 'Series_contributor': 'Eugene,,Valkov', 'Series_sample_id': 'GSM3890771', 'Series_contact_name': 'Eugene,,Valkov', 'Series_contact_email': 'eugene.valkov@tuebingen.mpg.de', 'Series_contact_phone': '497071601357', 'Series_contact_laboratory': 'Abt. II', 'Series_contact_department': 'Biochemistry', 'Series_contact_institute': 'Max Planck Institute for Developmental Biology', 'Series_contact_address': 'Max-Planck-Ring 5', 'Series_contact_city': 'Tuebingen', 'Series_contact_state': 'Baden-Wuerttemberg', 'Series_contact_zip/postal_code': '72076', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE132nnn/GSE132725/suppl/GSE132725_TE_XRN1.txt.gz', 'Series_platform_id': 'GPL21290', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP201395'}\n",
      "301 PRJNA549955 GSE133111 {'Series_title': 'Ribosome profiling of A549 cells depleted of RPLP1 and RPLP2 and infected with DENV.', 'Series_geo_accession': 'GSE133111', 'Series_status': 'Public on Jun 21 2019', 'Series_submission_date': 'Jun 20 2019', 'Series_last_update_date': 'Jun 21 2019', 'Series_summary': 'We used ribosome profiling to evaluate viral and cellular translation in RPLP1/2-depleted cells. This revealed that ribosomes pause in the sequence coding for the N-terminus of the envelope protein, immediately downstream of sequences encoding two adjacent transmembrane domains (TMDs). RPLP1/2 function to enhance ribosome elongation at this position and increase viral protein stability, possibly by improving co-translational folding of DENV proteins. We also analyzed the effects of RPLP1/2 depletion on cellular translation. We find that RPLP1/2 moderately affects ribosome density for a small subset of cellular mRNAs. However, meta-analysis of ribosome positions on all cellular mRNAs revealed slightly increased accumulation of ribosomes downstream of start codons in RPLP1/2-depleted cells, suggesting that RPLP1/2 enhance elongation efficiency. Importantly, we found that ribosome density on mRNAs encoding multiple TMDs was disproportionately affected by RPLP1/2 knockdown, suggesting a role for RPLP1/2 in transmembrane protein biogenesis. Together, our findings reveal insights into the function of RPLP1/2 in DENV and cellular translation.', 'Series_overall_design': \"A549 cells were plated at 1.5 x 106 cells per 10 cm dish. Three 10cm dishes were transfected with NSC siRNA whereas three other dishes were transfected with either siP1_1, siP2_1 or siP2_4 siRNAs as described in the transfections section. 48 h later cells were infected as described previously with DENV-2 (NGC strain) at MOI of 10 in a total volume of 10 ml, rocked every 15 minutes for 1 h and the infection was allowed to proceed for more 1.5 h (2.5 h total time). Cells were then flash frozen in liquid nitrogen without cycloheximide pretreatment and cold lysis buffer containing CHX was used to lyse the cells on ice. The RIBOseq strategy was adapted from Ingolia and colleagues with a few modifications described next. After nuclease digestion, samples were run in a polysome gradient and the ribosome fractions were collected. As described by Reid and colleagues, fractions were extracted using Trizol LS (Thermo Fisher Scientific), and rRNAs were removed using the Ribo-Zero gold rRNA removal kit (Illumina, San Diego CA) according to the manufacturer's protocol. For adapter ligation and library building we used NEBNext Small RNA Library Prep Set (Illumina).\", 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Shelton,S,Bradrick', 'Series_sample_id': 'GSM3900219', 'Series_contact_name': 'Rafael,,Kroon Campos', 'Series_contact_email': 'rkkroonc@utmb.edu', 'Series_contact_institute': 'UTMB', 'Series_contact_address': '11th street and mechanic', 'Series_contact_city': 'Galveston', 'Series_contact_state': 'TX', 'Series_contact_zip/postal_code': '77550', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE133nnn/GSE133111/suppl/GSE133111_TPM_all_genes.xlsx', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP202033'}\n",
      "302 PRJNA553142 GSE133925 {'Series_title': 'CircMYBL2 Regulates FLT3-ITD FLT3AML Translation in AML', 'Series_geo_accession': 'GSE133925', 'Series_status': 'Public on Jul 09 2019', 'Series_submission_date': 'Jul 08 2019', 'Series_last_update_date': 'Jan 21 2020', 'Series_pubmed_id': '31387917', 'Series_summary': 'CircMYBL2 is more highly expressed in AML patients with FLT3-ITD mutations than in those without the FLT3-ITD mutation. We found that circMYBL2 knockdown specifically inhibits proliferation and promotes the differentiation of FLT3-ITD AML cells in vitro and in vivo. We used the ribosome profiling and RNA-seq libraries sequenced with Illumina HiSeq 2500 to identify the mRNA that circMYBL2 targeted. Interestingly, we found that circMYBL2 significantly influences the protein level of mutant FLT3 kinase, which contributes to the activation of FLT3-ITD-dependent signaling pathways. Mechanistically, circMYBL2 enhanced the translational efficiency of FLT3 kinase by increasing the binding of PTBP1 to FLT3 mRNA. Moreover, circMYBL2 knockdown impaired the cytoactivity of inhibitor-resistant FLT3-ITD-positive cells, with a significant decrease in FLT3 kinase expression, followed by the inactivation of its downstream pathways. In summary, we are the first to reveal a circRNA that specifically influences FLT3-ITD AML and regulates FLT3 kinase levels through translational regulation, suggesting that circMYBL2 may be a potential therapeutic target for FLT3-ITD AML.', 'Series_overall_design': 'Sh-NC and sh-circMYBL2 MOLM-13 cells used', 'Series_type': 'Other', 'Series_contributor': 'Wen-Tao,,Wang', 'Series_sample_id': 'GSM3930178', 'Series_contact_name': 'Wen-Tao,,Wang', 'Series_contact_email': 'wangwt8@mail.sysu.edu.cn', 'Series_contact_phone': '86-20-84111736', 'Series_contact_laboratory': 'Key Laboratory of Gene Engineering of the Ministry of Education', 'Series_contact_department': 'School of Life Sciences', 'Series_contact_institute': 'sun yat sen university', 'Series_contact_address': 'No. 135, Xingang Xi Road', 'Series_contact_city': 'Guangzhou', 'Series_contact_state': 'Guangdong', 'Series_contact_zip/postal_code': '510275', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE133nnn/GSE133925/suppl/GSE133925_circshMY_rna_fpkm_ann.csv.gz', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP213599'}\n",
      "303 PRJNA553494 GSE134020 {'Series_title': 'Mammalian RNA decay pathways are highly specialized and widely linked to translation', 'Series_geo_accession': 'GSE134020', 'Series_status': 'Public on Jan 29 2020', 'Series_submission_date': 'Jul 09 2019', 'Series_last_update_date': 'Mar 30 2020', 'Series_pubmed_id': '32048998', 'Series_summary': 'RNA decay is crucial for RNA turnover and surveillance, and misregulated in many diseases. This complex system is challenging to study, particularly in mammals, where it remains unclear whether decay pathways perform specialized or redundant roles. Cytoplasmic pathways, and links to translation, are particularly enigmatic. By directly profiling targets of decay factors (XRN1, SKIV2L and MTR4) and normal/aberrant translation events in mouse embryonic stem cells, we uncovered extensive specialization between decay pathways and crosstalk with translation. XRN1 (5’-3’) mediated cytoplasmic bulk mRNA turnover whereas SKIV2L (3’-5’) was universally recruited by ribosomes, tackling aberrant translation and sometimes modulating mRNA abundance. Further exploring translation surveillance, we identified AVEN and FOCAD as SKIV2L interactors. AVEN prevented ribosome stalls at structured regions, which otherwise required SKIV2L for clearance. This pathway was crucial for histone translation, uORF regulation and counteracting spurious non-coding RNA translation. In summary, we identified key targets, components and functions of mammalian RNA decay pathways, and uncovered extensive coupling to translation.', 'Series_overall_design': 'mESCs with endogenously 3xFLAG-Avi-tagged decay factors (either MTR4, SKIV2L, XRN1 or AVEN) were used for crosslinking and analysis of cDNAs (CRAC), to identify transcriptome-wide direct RNA binding sites. Various WT or knockout cell lines were also profiled by RNA-seq, and some CRAC experiments were repeated in knockout backgrounds.', 'Series_type': 'Other', 'Series_contributor': 'Alaaddin,B,Arpat', 'Series_sample_id': 'GSM4148298', 'Series_contact_name': 'Alex,Charles,Tuck', 'Series_contact_institute': 'Friedrich Miescher Institute for Biomedical Research', 'Series_contact_address': 'Maulbeerstrasse 66', 'Series_contact_city': 'Basel', 'Series_contact_zip/postal_code': '4058', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE134nnn/GSE134020/suppl/GSE134020_half_lives_231019.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP213860'}\n",
      "304 PRJNA554579 GSE134256 {'Series_title': 'Experimental evolution reveals the genetic basis and systems biology of superoxide stress tolerance', 'Series_geo_accession': 'GSE134256', 'Series_status': 'Public on Jul 07 2022', 'Series_submission_date': 'Jul 15 2019', 'Series_last_update_date': 'Nov 13 2023', 'Series_pubmed_id': '37713311', 'Series_summary': 'E. coli frequently encounters oxidative stress both in its natural environment or in industrial biotechnology. Elucidating the mechanisms behind tolerance to oxidative stress would be beneficial for understanding pathogenesis as well as improving production strain fitness. We make use of adaptive laboratory evolution to develop two strains of E. coli which exhibit 500% increased tolerance to paraquat stress compared to wild type. Evolved strains tolerate oxidative stress by reduction of flux through TCA, dyregulation of iron-uptake genes, and up-regulation of cell motility or iron-sulfur cluster repair genes.', 'Series_overall_design': 'PQ1 and PQ2 are end-point strains from an adaptive laboratory evolution (ALE) which took place over 33 days and a total of 8x10^11 cumulative cell divisions (CCD). GLU4 is the wild type strain which comes from LaCroix et. al. 2015 (doi: 10.1128/AEM.02246-14 ). The MP strain was isolated from the midpoint of the evolution of PQ2 at 6x10^11 CCD. Genome mutations for these strains will be available at ALEdb (Phaneuf et. al. 2019 - https://doi.org/10.1093/nar/gky983)', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Bernhard,O,Palsson', 'Series_sample_id': 'GSM3940516', 'Series_contact_name': 'Wun Kiat Justin,,Tan', 'Series_contact_email': 'justin.tanwk@gmail.com', 'Series_contact_institute': 'UCSD', 'Series_contact_address': '9500 Gilman Drive #0412, PFBH 414', 'Series_contact_city': 'La Jolla', 'Series_contact_state': 'California', 'Series_contact_zip/postal_code': '92093', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE134nnn/GSE134256/suppl/GSE134256_RAW.tar', 'Series_platform_id': 'GPL21433', 'Series_platform_taxid': '562', 'Series_sample_taxid': '562', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP214584'}\n",
      "305 PRJNA554781 GSE134324 {'Series_title': 'High throughput ribosome profiling of E. coli expressing a library of heterologous proteins', 'Series_geo_accession': 'GSE134324', 'Series_status': 'Public on Jul 07 2022', 'Series_submission_date': 'Jul 16 2019', 'Series_last_update_date': 'Jul 07 2022', 'Series_summary': 'Heterologous gene expression to expand the native genetic capability of E. coli is the backbone of protein expression and metabolic engineering. The goal of this study was to determine how the identity of the heterologous gene expressed affected the host cell transcriptome. We generated a library of E. coli expressing 46 heterologous genes through an identical rhamnose inducible expression system and perform high throughput ribosome profiling.', 'Series_overall_design': 'Ribosome profiling of 46 heterologous gene expressing lines of E. coli using the same expression system', 'Series_type': 'Other', 'Series_contributor': 'Bernhard,O,Palsson', 'Series_sample_id': 'GSM3942293', 'Series_contact_name': 'Wun Kiat Justin,,Tan', 'Series_contact_email': 'justin.tanwk@gmail.com', 'Series_contact_institute': 'UCSD', 'Series_contact_address': '9500 Gilman Drive #0412, PFBH 414', 'Series_contact_city': 'La Jolla', 'Series_contact_state': 'California', 'Series_contact_zip/postal_code': '92093', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE134nnn/GSE134324/suppl/GSE134324_20180723_riboseq_rawcounts_final_cleaned.csv.gz', 'Series_platform_id': 'GPL21222', 'Series_platform_taxid': '562', 'Series_sample_taxid': '562', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP214776'}\n",
      "306 PRJNA555528 GSE134541 {'Series_title': 'Transcriptome-wide sites of collided ribosomes reveal sequence determinants of translational pausing in vivo', 'Series_geo_accession': 'GSE134541', 'Series_status': 'Public on Jun 18 2020', 'Series_submission_date': 'Jul 19 2019', 'Series_last_update_date': 'Sep 17 2020', 'Series_pubmed_id': '32703885', 'Series_summary': 'Translation initiation is considered overall rate-limiting for protein biosynthesis, whereas the impact of non-uniform ribosomal elongation rates is largely unknown. Using a modified ribosome profiling protocol based on footprints from two closely packed ribosomes (disomes), we have mapped ribosomal collisions transcriptome-wide in mouse liver. We uncover that the stacking of an elongating onto a paused ribosome occurs frequently and scales with translation rate, trapping ~10% of translating ribosomes in the disome state. A distinct class of pause sites, independent of translation rate, is indicative of deterministic pausing signals. We find pause sites associated with specific codons, amino acids, and peptide motifs, and with structural features of the nascent polypeptide, suggestive of programmed pausing as a widespread mechanism associated with protein folding. Evolutionary conservation at disome sites and experiments indicate functional relevance of translational pausing. Collectively, our disome profiling approach allows novel and unexpected insights into gene regulation occurring at the step of translation elongation.', 'Series_overall_design': 'Six biological samples (3 timepoints x 2 replicates) from GSE67305 were used. Briefly, mice were entrained under 12hours light:dark conditions for 2 weeks and also collected under 12hours light:dark. Mice were sacrificed at 0, 2, and 12h during the 24 hours daily cycle. Two replicates per time point, each replicate is a pool of 2 livers (2 mice).', 'Series_type': 'Other', 'Series_contributor': 'David,,Gatfield', 'Series_sample_id': 'GSM3955380', 'Series_contact_name': 'David,,Gatfield', 'Series_contact_email': 'david.gatfield@unil.ch', 'Series_contact_phone': '+41 21 692 39 94', 'Series_contact_department': 'CIG', 'Series_contact_institute': 'University of Lausanne', 'Series_contact_address': 'UNIL CIG', 'Series_contact_city': 'Lasuanne', 'Series_contact_zip/postal_code': '1015', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE134nnn/GSE134541/suppl/GSE134541_RAW.tar', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP215428'}\n",
      "307 PRJNA556267 GSE134752 {'Series_title': 'Mechanism of virus attenuation by codon pair deoptimization', 'Series_geo_accession': 'GSE134752', 'Series_status': 'Public on Jan 01 2020', 'Series_submission_date': 'Jul 23 2019', 'Series_last_update_date': 'May 19 2021', 'Series_pubmed_id': '32348767', 'Series_summary': 'Determination of translation efficiency of recoded neuramindase mRNA transcripts in transiently transfected HEK 293T cells.', 'Series_overall_design': 'Expression plasmids containing recoded IAV neuramindase were transfected into HEK 293T cells and translational dynamics of neuraminidase mRNA was determined by ribosome footprint-profiling experiments', 'Series_type': 'Other', 'Series_contributor': 'Dusan,,Kunec', 'Series_sample_id': 'GSM3966956', 'Series_contact_name': 'Emanuel,,Wyler', 'Series_contact_email': 'emanuel.wyler@mdc-berlin.de', 'Series_contact_phone': '+49 30 9406 3009', 'Series_contact_laboratory': 'RNA Biology and Posttranscriptional Regulation', 'Series_contact_department': 'Berlin Institute for Medical Systems Biology', 'Series_contact_institute': 'Max Delbrück Center for Molecular Medicine', 'Series_contact_address': 'Robert Roessle Str 10', 'Series_contact_city': 'Berlin', 'Series_contact_zip/postal_code': '13125', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE134nnn/GSE134752/suppl/GSE134752_RPlength30_fpkm_CDS.txt.gz', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP216083'}\n",
      "308 PRJNA558435 GSE135336 {'Series_title': 'Paralog-specific phenotypes of ribosomal protein mutants identify translational control mechanisms in the cell cycle and replicative longevity', 'Series_geo_accession': 'GSE135336', 'Series_status': 'Public on Nov 20 2019', 'Series_submission_date': 'Aug 02 2019', 'Series_last_update_date': 'Jun 08 2020', 'Series_pubmed_id': '32432546', 'Series_summary': 'The translational efficiency of mRNAs of cells progressing synchronously through the mitotic cell cycle has been interrogated by ribosome profiling.', 'Series_overall_design': 'Ribosome Profiling (RP) and Transcriptional Profiling (TP) samples corresponding to a cell-size series, for the following Saccharomyces cerevisiae ribosome protein mutants: RPL22A, RPL22B, RPL34A, and RPL34B', 'Series_type': 'Other', 'Series_contributor': 'Michael,,Polymenis', 'Series_sample_id': 'GSM4006643', 'Series_contact_name': 'Rodolfo,,Aramayo', 'Series_contact_email': 'raramayo@tamu.edu', 'Series_contact_laboratory': 'AramayoLab', 'Series_contact_department': 'Biology', 'Series_contact_institute': 'Texas A&M University', 'Series_contact_address': 'Room 412, Biological Sciences Building West (BSBW)', 'Series_contact_city': 'College Station', 'Series_contact_state': 'Texas', 'Series_contact_zip/postal_code': '77843-3258', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE135nnn/GSE135336/suppl/GSE135336_RAW.tar', 'Series_platform_id': 'GPL21656', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA558435'}\n",
      "309 PRJNA559859 GSE135722 {'Series_title': 'A p53-dependent translational program directs tissue-selective phenotypes in a model of ribosomopathies', 'Series_geo_accession': 'GSE135722', 'Series_status': 'Public on May 09 2021', 'Series_submission_date': 'Aug 12 2019', 'Series_last_update_date': 'Aug 08 2021', 'Series_pubmed_id': '34242585', 'Series_summary': 'In ribosomopathies, perturbed expression of ribosome components leads to tissue-specific phenotypes. What accounts for such tissue-selective manifestations as a result of mutations in the ribosome, a ubiquitous cellular machine, has remained a mystery. Combining mouse genetics and in vivo ribosome profiling, we observe limb patterning phenotypes in ribosomal protein (RP) haploinsufficient embryos and uncover selective translational changes of transcripts controlling limb development. Surprisingly, both loss of p53, which is activated by RP haploinsufficiency, and augmented protein synthesis rescue these phenotypes. These findings are explained by the identification that p53 functions as a master regulator of protein synthesis, at least in part, through transcriptional activation of 4E-BP1. 4E-BP1, a key translational regulator, in turn, facilitates selective changes in the translatome downstream of p53, and thereby explains how RP haploinsufficiency may elicit specificity to gene expression. These results provide an integrative model to understand how in vivo tissue-specific phenotypes emerge in ribosomopathies.', 'Series_overall_design': 'Ribosome profiling from E10.5 mouse forelimbs (N=3 of each genotype): Rps6lox/+;Trp53+/+, Prx1Cre;Rps6lox/+;Trp53+/+, Rps6lox/+;Trp53-/-, Prx1Cre;Rps6lox/+;Trp53-/-', 'Series_type': 'Other', 'Series_contributor': 'Maria,,Barna', 'Series_sample_id': 'GSM4026945', 'Series_contact_name': 'Gerald,,Tiu', 'Series_contact_institute': 'Stanford University', 'Series_contact_address': '279 Campus Dr., Beckman Center B309', 'Series_contact_city': 'Stanford', 'Series_contact_state': 'California', 'Series_contact_zip/postal_code': '94305', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE135nnn/GSE135722/suppl/GSE135722_ribo_cds_rna_total_counts.csv.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP218125'}\n",
      "310 PRJNA561206 GSE136091 {'Series_title': 'Translation elongation factor 2 depletion by siRNA in mouse liver leads to mTOR-independent translational upregulation of ribosomal protein genes', 'Series_geo_accession': 'GSE136091', 'Series_status': 'Public on May 31 2020', 'Series_submission_date': 'Aug 20 2019', 'Series_last_update_date': 'Oct 02 2020', 'Series_pubmed_id': '32968084', 'Series_summary': 'Due to breakthroughs in RNAi and genome editing methods in the past decade, it is now easier than ever to study fine details of protein synthesis in animal models. However, most of our understanding of translation comes from unicellular organisms and cultured mammalian cells. In this study, we demonstrate the feasibility of perturbing protein synthesis in a mouse liver by targeting translation elongation factor 2 (eEF2) with RNAi. We were able to achieve over 90% knockdown efficacy and maintain it for 2 weeks effectively slowing down the rate of translation elongation. As the total protein yield declined, both proteomics and ribosome profiling assays showed robust translational upregulation of ribosomal proteins relative to other proteins. Although all these genes bear the TOP regulatory motif, the branch of the mTOR pathway responsible for translation regulation was not activated. Paradoxically, coordinated translational upregulation of ribosomal proteins only occurred in the liver but not in murine cell culture. Thus, the upregulation of ribosomal transcripts likely occurred via passive mTOR-independent mechanisms. Impaired elongation sequesters ribosomes on mRNA and creates a shortage of free ribosomes. This leads to preferential translation of transcripts with high initiation rates such as ribosomal proteins. Furthermore, severe eEF2 shortage reduces the negative impact of positively charged amino acids frequent in ribosomal proteins on ribosome progression.', 'Series_overall_design': '8 cell culture transcriptomes (4 control and 4 eef2 knockdowns); transcriptomes from 12 mouse livers (control, 7, 9 and 12 days; three animals in each group); 6 ribo-seq libraries prepared from the same tissues (matched within animal)', 'Series_type': 'Other', 'Series_contributor': 'Viktor,,Kotelianskii', 'Series_sample_id': 'GSM4040978', 'Series_contact_name': 'Maxim,,Gerashchenko', 'Series_contact_email': 'mgerashchenko@bwh.harvard.edu', 'Series_contact_department': 'Medicine', 'Series_contact_institute': \"Brigham and Women's Hospital\", 'Series_contact_address': '77 Louis Pasteur Ave, NRB 435', 'Series_contact_city': 'Boston', 'Series_contact_state': 'Massachusetts', 'Series_contact_zip/postal_code': '02115', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE136nnn/GSE136091/suppl/GSE136091_readme.xls', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP218948'}\n",
      "311 PRJNA563844 GSE136838 {'Series_title': 'Genome-wide transcriptomic and translatomic profiling in halofuginone and vehicle treated mouse fibroblasts', 'Series_geo_accession': 'GSE136838', 'Series_status': 'Public on Sep 09 2021', 'Series_submission_date': 'Sep 04 2019', 'Series_last_update_date': 'Sep 09 2021', 'Series_pubmed_id': '32611237', 'Series_summary': 'Conclusions: Our study represents the first detailed analysis of fibroblast translatome in comparison with transcriptome under EPRS inhibitory condition (halofuginone treatment), with biological triplicates, generated by RNA-seq and polysome-seq technology. Our results show that the next generation sequencing offers a comprehensive qualitative and quantitative evaluation of mRNA content and its translation state within cells treated with translation inhibitory drug halofuginone (EPRS inhibitor). Based on our results, we conclude that Pro-rich motif bearing genes are preferential targets of EPRS, and mild inhibition of EPRS leads to inefficient translation of these PRR genes.', 'Series_overall_design': 'We treated NIH/3T3 mouse fibroblasts with halofuginone (EPRS-specific inhibitor; 100 nM) or vehicle (PBS) in biological triplicates. For polysome-Seq, all the fractions were pooled into 3 samples, polysome free (free mRNP, 40S, 60S subunit), light polysomes (monosome, disome, trisome, etc.), and heavy polysomes (>5 ribosomes). Total RNA was extracted from the same volume of 3 samples and subjected to next generation RNA deep sequencing for translatome profiling as polysome-seq. For transcriptome profiling, total RNA extracted from the same treated cells was subjected to RNA-Seq. All the extracted RNA samples were treated with DNase I to remove potential genomic DNA followed by purification using phenol:chloroform:isoamyl alcohol for RNA-Seq. Raw reads were generated from the Illumina HiSeq2500 sequencer in the Genomic Research Center from University of Rochester Medical Center.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Peng,,Yao', 'Series_sample_id': 'GSM4059082', 'Series_contact_name': 'Peng,,Yao', 'Series_contact_email': 'yp5200qq@gmail.com', 'Series_contact_phone': '2166507876', 'Series_contact_laboratory': 'Peng Yao lab', 'Series_contact_department': 'Medicine', 'Series_contact_institute': 'Aab Cardiovascular Research Institute', 'Series_contact_address': '601 Elmwood Avenue', 'Series_contact_city': 'Rochester', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '14534', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE136nnn/GSE136838/suppl/GSE136838_deSeq2_counts.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP220286'}\n",
      "312 PRJNA575265 GSE138278 {'Series_title': 'Ribosome profiling of Streptomyces griseus NBRC13350 and Streptomyces venezuelae ATCC15439', 'Series_geo_accession': 'GSE138278', 'Series_status': 'Public on Mar 03 2020', 'Series_submission_date': 'Oct 01 2019', 'Series_last_update_date': 'May 18 2020', 'Series_pubmed_id': '32385251', 'Series_summary': 'We performed ribosome profiling which is the deep-sequencing of mRNA fragments protected by translating ribosome for two Streptomyces species through different growth phases to provide the translatome data', 'Series_overall_design': 'Ribosome profiling data of Streptomyces griseus NBRC13350 and Streptomyces venezuelae ATCC15439 at four different growth phases in duplicates using Illumina HiSeq', 'Series_type': 'Other', 'Series_contributor': 'Byung-Kwan,,Cho', 'Series_sample_id': 'GSM4104321', 'Series_contact_name': 'Soonkyu,,Hwang', 'Series_contact_email': 'geager@kaist.ac.kr', 'Series_contact_department': 'Biological Sciences', 'Series_contact_institute': 'KAIST', 'Series_contact_address': '291 Daehakro, Yuseong-gu', 'Series_contact_city': 'Daejeon', 'Series_contact_zip/postal_code': '34141', 'Series_contact_country': 'South Korea', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE138nnn/GSE138278/suppl/GSE138278_Sven_Ribo_gene_expression.txt.gz', 'Series_platform_id': 'GPL27560', 'Series_platform_taxid': '54571', 'Series_sample_taxid': '54571', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP223903'}\n",
      "313 PRJNA576649 GSE138638 {'Series_title': 'Stop Codon Context Influences Genome-Wide Stimulation of Termination Codon Readthrough by Aminoglycosides [Dataset 1]', 'Series_geo_accession': 'GSE138638', 'Series_status': 'Public on Jan 14 2020', 'Series_submission_date': 'Oct 09 2019', 'Series_last_update_date': 'Mar 30 2020', 'Series_pubmed_id': '31971508', 'Series_summary': 'Stop codon readthrough (SCR) occurs when the ribosome miscodes at a stop codon. Such readthrough events can be therapeutically desirable when a premature termination codon (PTC) is found in a critical gene. To study SCR in vivo in a genome-wide manner, we treated mammalian cells with aminoglycosides and performed ribosome profiling. We find that in addition to stimulating readthrough of PTCs, aminoglycosides stimulate readthrough of normal termination codons (NTCs) genome-wide. Stop codon identity, the nucleotide following the stop codon, and the surrounding mRNA sequence context all influence the likelihood of SCR. In comparison to NTCs, downstream stop codons in 3′UTRs are recognized less efficiently by ribosomes, suggesting that targeting of critical stop codons for readthrough may be achievable without general disruption of translation termination. Finally, we find that G418 treatment globally alters gene expression with substantial effects on translation of histone genes, selenoprotein genes, and S-adenosylmethionine decarboxylase (AMD1).', 'Series_overall_design': '18 ribosome profiling and 8 paired-end RNA-seq samples are included from HEK293T cells', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Rachel,,Green', 'Series_sample_id': 'GSM4114792', 'Series_contact_name': 'Rachel,,Green', 'Series_contact_email': 'ragreen@jhmi.edu', 'Series_contact_laboratory': 'Rachel Green', 'Series_contact_department': 'Molecular Biology and Genetics', 'Series_contact_institute': 'Johns Hopkins University School of Medicine', 'Series_contact_address': '725 N Wolfe St', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21205', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE138nnn/GSE138638/suppl/GSE138638_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP224952'}\n",
      "314 PRJNA576650 GSE138639 {'Series_title': 'Stop Codon Context Influences Genome-Wide Stimulation of Termination Codon Readthrough by Aminoglycosides [Dataset 2]', 'Series_geo_accession': 'GSE138639', 'Series_status': 'Public on Jan 14 2020', 'Series_submission_date': 'Oct 09 2019', 'Series_last_update_date': 'Mar 30 2020', 'Series_pubmed_id': '31971508', 'Series_summary': 'Stop codon readthrough (SCR) occurs when the ribosome miscodes at a stop codon. Such readthrough events can be therapeutically desirable when a premature termination codon (PTC) is found in a critical gene. To study SCR in vivo in a genome-wide manner, we treated mammalian cells with aminoglycosides and performed ribosome profiling. We find that in addition to stimulating readthrough of PTCs, aminoglycosides stimulate readthrough of normal termination codons (NTCs) genome-wide. Stop codon identity, the nucleotide following the stop codon, and the surrounding mRNA sequence context all influence the likelihood of SCR. In comparison to NTCs, downstream stop codons in 3′UTRs are recognized less efficiently by ribosomes, suggesting that targeting of critical stop codons for readthrough may be achievable without general disruption of translation termination. Finally, we find that G418 treatment globally alters gene expression with substantial effects on translation of histone genes, selenoprotein genes, and S-adenosylmethionine decarboxylase (AMD1).', 'Series_overall_design': '4 ribosome profiling samples are included from HEK293T cells', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Rachel,,Green', 'Series_sample_id': 'GSM4114796', 'Series_contact_name': 'Rachel,,Green', 'Series_contact_email': 'ragreen@jhmi.edu', 'Series_contact_laboratory': 'Rachel Green', 'Series_contact_department': 'Molecular Biology and Genetics', 'Series_contact_institute': 'Johns Hopkins University School of Medicine', 'Series_contact_address': '725 N Wolfe St', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21205', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE138nnn/GSE138639/suppl/GSE138639_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP224953'}\n",
      "315 PRJNA576651 GSE138640 {'Series_title': 'Stop Codon Context Influences Genome-Wide Stimulation of Termination Codon Readthrough by Aminoglycosides [Dataset 3]', 'Series_geo_accession': 'GSE138640', 'Series_status': 'Public on Jan 14 2020', 'Series_submission_date': 'Oct 09 2019', 'Series_last_update_date': 'Mar 30 2020', 'Series_pubmed_id': '31971508', 'Series_summary': 'Stop codon readthrough (SCR) occurs when the ribosome miscodes at a stop codon. Such readthrough events can be therapeutically desirable when a premature termination codon (PTC) is found in a critical gene. To study SCR in vivo in a genome-wide manner, we treated mammalian cells with aminoglycosides and performed ribosome profiling. We find that in addition to stimulating readthrough of PTCs, aminoglycosides stimulate readthrough of normal termination codons (NTCs) genome-wide. Stop codon identity, the nucleotide following the stop codon, and the surrounding mRNA sequence context all influence the likelihood of SCR. In comparison to NTCs, downstream stop codons in 3′UTRs are recognized less efficiently by ribosomes, suggesting that targeting of critical stop codons for readthrough may be achievable without general disruption of translation termination. Finally, we find that G418 treatment globally alters gene expression with substantial effects on translation of histone genes, selenoprotein genes, and S-adenosylmethionine decarboxylase (AMD1).', 'Series_overall_design': '3 ribosome profiling samples are included from HEK293T cells', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Rachel,,Green', 'Series_sample_id': 'GSM4114799', 'Series_contact_name': 'Rachel,,Green', 'Series_contact_email': 'ragreen@jhmi.edu', 'Series_contact_laboratory': 'Rachel Green', 'Series_contact_department': 'Molecular Biology and Genetics', 'Series_contact_institute': 'Johns Hopkins University School of Medicine', 'Series_contact_address': '725 N Wolfe St', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21205', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE138nnn/GSE138640/suppl/GSE138640_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP224954'}\n",
      "316 PRJNA576652 GSE138641 {'Series_title': 'Stop Codon Context Influences Genome-Wide Stimulation of Termination Codon Readthrough by Aminoglycosides [Dataset 4]', 'Series_geo_accession': 'GSE138641', 'Series_status': 'Public on Jan 14 2020', 'Series_submission_date': 'Oct 09 2019', 'Series_last_update_date': 'Mar 30 2020', 'Series_pubmed_id': '31971508', 'Series_summary': 'Stop codon readthrough (SCR) occurs when the ribosome miscodes at a stop codon. Such readthrough events can be therapeutically desirable when a premature termination codon (PTC) is found in a critical gene. To study SCR in vivo in a genome-wide manner, we treated mammalian cells with aminoglycosides and performed ribosome profiling. We find that in addition to stimulating readthrough of PTCs, aminoglycosides stimulate readthrough of normal termination codons (NTCs) genome-wide. Stop codon identity, the nucleotide following the stop codon, and the surrounding mRNA sequence context all influence the likelihood of SCR. In comparison to NTCs, downstream stop codons in 3′UTRs are recognized less efficiently by ribosomes, suggesting that targeting of critical stop codons for readthrough may be achievable without general disruption of translation termination. Finally, we find that G418 treatment globally alters gene expression with substantial effects on translation of histone genes, selenoprotein genes, and S-adenosylmethionine decarboxylase (AMD1).', 'Series_overall_design': '2 ribosome profiling samples are included from HEK293T cells', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Rachel,,Green', 'Series_sample_id': 'GSM4114801', 'Series_contact_name': 'Rachel,,Green', 'Series_contact_email': 'ragreen@jhmi.edu', 'Series_contact_laboratory': 'Rachel Green', 'Series_contact_department': 'Molecular Biology and Genetics', 'Series_contact_institute': 'Johns Hopkins University School of Medicine', 'Series_contact_address': '725 N Wolfe St', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21205', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE138nnn/GSE138641/suppl/GSE138641_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP224955'}\n",
      "317 PRJNA576653 GSE138642 {'Series_title': 'Stop Codon Context Influences Genome-Wide Stimulation of Termination Codon Readthrough by Aminoglycosides [Dataset 5]', 'Series_geo_accession': 'GSE138642', 'Series_status': 'Public on Jan 14 2020', 'Series_submission_date': 'Oct 09 2019', 'Series_last_update_date': 'Mar 30 2020', 'Series_pubmed_id': '31971508', 'Series_summary': 'Stop codon readthrough (SCR) occurs when the ribosome miscodes at a stop codon. Such readthrough events can be therapeutically desirable when a premature termination codon (PTC) is found in a critical gene. To study SCR in vivo in a genome-wide manner, we treated mammalian cells with aminoglycosides and performed ribosome profiling. We find that in addition to stimulating readthrough of PTCs, aminoglycosides stimulate readthrough of normal termination codons (NTCs) genome-wide. Stop codon identity, the nucleotide following the stop codon, and the surrounding mRNA sequence context all influence the likelihood of SCR. In comparison to NTCs, downstream stop codons in 3′UTRs are recognized less efficiently by ribosomes, suggesting that targeting of critical stop codons for readthrough may be achievable without general disruption of translation termination. Finally, we find that G418 treatment globally alters gene expression with substantial effects on translation of histone genes, selenoprotein genes, and S-adenosylmethionine decarboxylase (AMD1).', 'Series_overall_design': '2 ribosome profiling and 4 RNA-seq samples are included from Calu-6 cells', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Rachel,,Green', 'Series_sample_id': 'GSM4114807', 'Series_contact_name': 'Rachel,,Green', 'Series_contact_email': 'ragreen@jhmi.edu', 'Series_contact_laboratory': 'Rachel Green', 'Series_contact_department': 'Molecular Biology and Genetics', 'Series_contact_institute': 'Johns Hopkins University School of Medicine', 'Series_contact_address': '725 N Wolfe St', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21205', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE138nnn/GSE138642/suppl/GSE138642_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP224956'}\n",
      "318 PRJNA577942 GSE138990 {'Series_title': 'Ribosome profiling in archaea reveals leaderless translation, novel translational initiation sites, and ribosome pausing at single codon resolution', 'Series_geo_accession': 'GSE138990', 'Series_status': 'Public on Apr 27 2020', 'Series_submission_date': 'Oct 16 2019', 'Series_last_update_date': 'Jul 27 2020', 'Series_pubmed_id': '32382758', 'Series_summary': 'Members of the third domain of life, the Archaea, are ubiquitous in all environments on Earth but remain understudied in many aspects including regulatory processes of the central dogma. Archaea present unique biology because they utilize a mosaic of molecular features from both Bacteria and Eukarya, along with unique features. The advent of a high-throughput view of the translation landscape via ribosome profiling in the Bacteria and the Eukarya has illuminated the complexity and previously underappreciated regulation of translation (i.e. translation efficiency, ribosome surveillance, etc.) that potentially has large scale effects on cellular functionality. Here, we developed ribosome profiling in a model archaeon, Haloferax volcanii and elucidated the translational landscape for the first time in the third domain of life. We coupled ribosome profiling with translation inhibitors to: (a) determine the size of the archaeal ribosome footprint, (b) systematically assign translation states of the ribosome to footprint lengths in a mostly leaderless transcriptome, (c) experimentally induce ribosome pauses and clarify the pausing landscape comprehensively, (d) identify putative novel proteins, including small open reading frames (smORFs), and (e) provide evidence that many genes initiate on putative alternative translation start sites (paTSS) around and within open reading frames (ORFs), demonstrating how a microorganism with a gene dense genome can produce proteins with distinct functions using the same gene.', 'Series_overall_design': 'Adapter to trim: CTGTAGGCACCATCAATAGATCGGAAGAGCACACGTCTGAACTCCAGTCA', 'Series_type': 'Other', 'Series_contributor': 'Jocelyne,,DiRuggiero', 'Series_sample_id': 'GSM4125810', 'Series_contact_name': 'Diego,Rivera,Gelsinger', 'Series_contact_email': 'dgelsin1@jhu.edu', 'Series_contact_phone': '3233144077', 'Series_contact_laboratory': 'DiRuggiero', 'Series_contact_department': 'Biology', 'Series_contact_institute': 'The Johns Hopkins Univeristy', 'Series_contact_address': '910 Light Street, Apartment 2', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21230', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE138nnn/GSE138990/suppl/GSE138990_RAW.tar', 'Series_platform_id': 'GPL27628', 'Series_platform_taxid': '309800', 'Series_sample_taxid': '309800', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP225954'}\n",
      "319 PRJNA578122 GSE139036 {'Series_title': 'Disome and trisome profiling reveal targets of ribosome quality control', 'Series_geo_accession': 'GSE139036', 'Series_status': 'Public on May 26 2020', 'Series_submission_date': 'Oct 17 2019', 'Series_last_update_date': 'Aug 01 2021', 'Series_pubmed_id': '32615089', 'Series_summary': 'The ribosome-associated protein quality control (RQC) system that resolves stalled translation events is activated when ribosomes collide and form disome, trisome or higher order complexes. However, it is unclear whether this system distinguishes collision complexes formed on defective mRNAs from those with functional roles on endogenous transcripts. Here, we performed disome and trisome footprint profiling in yeast and found collisions were enriched on diverse sequence motifs known to slow translation. When 60S recycling was inhibited, disomes accumulated at stop codons and could move into the 3’UTR to reinitiate translation. The ubiquitin ligase and RQC factor Hel2/ZNF598 generally recognized collisions but did not trigger degradation of endogenous transcripts. However, loss of Hel2 triggered the integrated stress response, via phosphorylation of eIF2alpha, thus linking these pathways. Our results suggest that Hel2 has a role in sensing ribosome collisions on endogenous mRNAs and such events may be important for cellular homeostasis.', 'Series_overall_design': \"31 samples are included in the study (4 mRNA-Seq, 4 monosome footprints, 21 disome footprints, and 2 trisome footprints). Read sizes were examined according to lengths relevant for each type of analysis and aligned by 5' or 3' ends, as noted. Samples also differ according to media conditions (3-AT, anisomycin, etc.) and genotype. Some samples serve as biological replicates, as noted.\", 'Series_type': 'Other', 'Series_contributor': 'Nicholas,R,Guydosh', 'Series_sample_id': 'GSM4445998', 'Series_contact_name': 'Nicholas,R,Guydosh', 'Series_contact_email': 'nicholas.guydosh@nih.gov', 'Series_contact_laboratory': 'LBG', 'Series_contact_department': 'NIDDK', 'Series_contact_institute': 'National Institutes of Health', 'Series_contact_address': '8 Center Dr, Bldg 8, 220', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '20892', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE139nnn/GSE139036/suppl/GSE139036_end3SM015Fd23Fd_plus.wig.gz', 'Series_platform_id': 'GPL27812', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP226076'}\n",
      "320 PRJNA578373 GSE139097 {'Series_title': 'Binding to the Ribosome by Eukaryotic Initiation Factor 4B Drives Yeast Translational Control in Response to Urea', 'Series_geo_accession': 'GSE139097', 'Series_status': 'Public on Apr 15 2020', 'Series_submission_date': 'Oct 18 2019', 'Series_last_update_date': 'Jan 21 2022', 'Series_pubmed_id': '35047555', 'Series_summary': 'The yeast eukaryotic initiation factor 4B binds the 40S subunit in translation preinitiation complexes (PICs) and promotes mRNA binding. Here we have compared the effects of disrupting eIF4B RNA- and ribosome-binding domains under ~1400 growth conditions. The RNA-binding RRM was dispensable for stress responses, but ribosome binding stimulated by the NTD promoted growth in response to a number of stressors through changes in translation. In particular, the NTD confers a strong growth advantage in the presence of the denaturing reagent, urea, and a number of osmolytes that require robust cellular integrity for survival. Ribosome profiling of cells with and without the NTD of eIF4B reveals changes in translation of mRNAs containing longer than average and highly structured 5-prime untranslated regions both normally, and in response to urea exposure. Because these changes require 40S binding, our results suggest eIF4B regulates mRNA recruitment as a part of the scanning PIC, rather than by activating isolated mRNPs prior to ribosome binding. This analysis indicates the cellular response to urea in yeast includes a translational component, driven by increased translation of mRNAs encoding proteins associated with the cellular periphery, and highlights the importance of the general translation factor eIF4B in adapting to external conditions.', 'Series_overall_design': 'Two biological replicates of each Saccharomyces culture of WT without urea, mutant without urea, WT with urea, and mutant with urea were harvested. Riboseq and RNAseq libraries were derived from each set of cells for a total of 16 libraries for illumina sequencing.', 'Series_type': 'Other', 'Series_contributor': 'Sarah,E,Walker', 'Series_sample_id': 'GSM4130832', 'Series_contact_name': 'Sarah,E.,Walker', 'Series_contact_department': 'Biological Sciences', 'Series_contact_institute': 'University at Buffalo, SUNY', 'Series_contact_address': '534 Cooke Hall, North Campus', 'Series_contact_city': 'Buffalo', 'Series_contact_state': 'New York', 'Series_contact_zip/postal_code': '14260', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE139nnn/GSE139097/suppl/GSE139097_RNAseq_Riboseq_TE.csv.gz', 'Series_platform_id': 'GPL21656', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP226217'}\n",
      "321 PRJNA578863 GSE139238 {'Series_title': 'High-Throughput Translational Profiling with riboPLATE-seq', 'Series_geo_accession': 'GSE139238', 'Series_status': 'Public on Oct 23 2019', 'Series_submission_date': 'Oct 22 2019', 'Series_last_update_date': 'Apr 19 2022', 'Series_pubmed_id': '35383235', 'Series_summary': 'Protein synthesis is dysregulated in many diseases, but we lack a systems-level picture of how signaling molecules and RNA binding proteins interact with the translational machinery, largely due to technological limitations. Here we present riboPLATE-seq, a scalable method for generating paired libraries of ribosome-associated and total mRNA. As an extension of the PLATE-seq protocol, riboPLATE-seq utilizes barcoded primers for pooled library preparation, but additionally leverages rRNA immunoprecipitation on whole polysomes to measure ribosome association (RA). We demonstrate the performance of riboPLATE-seq and its utility in detecting translational alterations induced by inhibition of protein kinases.', 'Series_overall_design': '96 paired riboPLATE- and PLATE-seq libraries, corresponding to independent TS543 glioma neurosphere samples seeded on a 96-well plate, plus 13 RNA-seq and 10 ribosome profiling libraries. 6 replicates per drug treatment (PP242, MNK-i1, BKM120, PP242+BKM120, PP242+MNK-i1, BKM120+MNK-i1, AZD8055, 4EGi-1) and 48 DMSO-treated controls, plus three replicates each of ribosome profiling and RNA sequencing for PP242 or DMSO treatment at 30 minutes, two further replicates at 6 hours PP242 or DMSO treatment, and one additional unpaired 6-hour DMSO-treated RNA-seq library.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Peter,,Sims', 'Series_sample_id': 'GSM5699627', 'Series_contact_name': 'Jordan,Benjamin,Metz', 'Series_contact_email': 'jm3423@cumc.columbia.edu', 'Series_contact_laboratory': 'Sims', 'Series_contact_department': 'Systems Biology', 'Series_contact_institute': 'Columbia University', 'Series_contact_address': '3960 Broadway Room 203A', 'Series_contact_city': 'New York', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '10032', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE139nnn/GSE139238/suppl/GSE139238_RPF.counts.txt.gz', 'Series_platform_id': 'GPL21697', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP226588'}\n",
      "322 PRJNA579539 GSE139391 {'Series_title': 'Selective 40S footprinting reveals that scanning ribosomes remain cap-tethered in human cells', 'Series_geo_accession': 'GSE139391', 'Series_status': 'Public on Jun 25 2020', 'Series_submission_date': 'Oct 25 2019', 'Series_last_update_date': 'May 12 2023', 'Series_pubmed_id': '36869665', 'Series_summary': 'Translation regulation occurs largely during initiation. Currently, translation initiation can be studied in vitro, but these systems lack features present in vivo and on endogenous mRNAs. Here we develop selective 40S footprinting for visualizing initiating 40S ribosomes on endogenous mRNAs in vivo. It pinpoints where on an mRNA initiation factors join the ribosome to act, and where they leave. We discover that in human cells most scanning ribosomes remain attached to the 5’ cap. Consequently, only one ribosome scans a 5’UTR at a time, and 5’UTR length affects translation efficiency. We discover that eIF3B, eIF4G1 and eIF4E remain on translating 80S ribosomes with a decay half-length of ~12 codons. Hence ribosomes retain these initiation factors while translating short upstream Open Reading Frames (uORFs), providing an explanation for how ribosomes can re-initiate translation after uORFs in humans. This method will be of use for studying translation initiation mechanisms in vivo.', 'Series_overall_design': 'Selective and total 40S and 80S footprinting was carried out on HeLa and NIH3T3 cells. Selective datasets were produced for factors eIF3B, eIF4E, eIF2S1 and eIF4G1.', 'Series_type': 'Other', 'Series_contributor': 'Aurelio,,Teleman', 'Series_sample_id': 'GSM4477464', 'Series_contact_name': 'Jonathan,,Bohlen', 'Series_contact_laboratory': 'AG Teleman', 'Series_contact_department': 'B140', 'Series_contact_institute': 'DKFZ', 'Series_contact_address': 'Im Neuenheimer Feld 581', 'Series_contact_city': 'Heidelberg', 'Series_contact_zip/postal_code': '69123', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE139nnn/GSE139391/suppl/GSE139391_Master_Data_Table.xlsx', 'Series_platform_id': 'GPL21697', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP226993'}\n",
      "323 PRJNA579549 GSE139399 {'Series_title': 'Ribosome profiling study of Miwi-null testes', 'Series_geo_accession': 'GSE139399', 'Series_status': 'Public on Dec 15 2019', 'Series_submission_date': 'Oct 25 2019', 'Series_last_update_date': 'Dec 16 2019', 'Series_pubmed_id': '31835033', 'Series_web_link': 'https://www.cell.com/cell/fulltext/S0092-8674(19)31278-4', 'Series_summary': 'piRNAs are a novel class of small noncoding RNAs that were recently identified in animal germ cells. This class of small noncoding RNAs is specifically associated with an evolutionarily conserved PIWI family proteins, which belong to the germline-specific members of the Argonaute protein family and are indispensable to germline development in animals. The PIWI/piRNA pathway has been deemed as an innate immune system that prevents mobile genetic elements from destabilizing DNA and that protects genome integrity in animal germ cells. By ribosome profiling using Miwi-null testes, we identified a group of ~600 mRNAs as likely direct piRNA targets for translational regulation in mouse spermatids. These results suggest that the mouse PIWI (MIWI)/piRNA is responsible for activating translation of a subset of spermiogenic mRNAs to coordinate with morphological transformation into elongated spermatids.', 'Series_overall_design': '8 biological samples are included in the study. These include RNA-seq and ribosome profiling in 24 dpp wild-type and Miwi-null testes.', 'Series_type': 'Other', 'Series_contributor': 'Zong-Gui,,Chen', 'Series_sample_id': 'GSM4140502', 'Series_contact_name': 'Zonggui,,Chen', 'Series_contact_email': 'chenzonggui@outlook.com', 'Series_contact_laboratory': 'Fuchou Tang', 'Series_contact_department': 'Academy for Advanced Interdisciplinary Studies', 'Series_contact_institute': 'Peking University', 'Series_contact_address': 'No.5 Yiheyuan Road Haidian District', 'Series_contact_city': 'Beijing', 'Series_contact_state': 'Beijing', 'Series_contact_zip/postal_code': '100871', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE139nnn/GSE139399/suppl/GSE139399_different_translation_efficiency_rep2.tsv.gz', 'Series_platform_id': 'GPL21273', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP227006'}\n",
      "324 PRJNA587444 GSE139880 {'Series_title': 'Recruitment of mRNAs to P granules by gelation with intrinsically-disordered proteins (RNAseq and ribosome profiling)', 'Series_geo_accession': 'GSE139880', 'Series_status': 'Public on Dec 01 2019', 'Series_submission_date': 'Nov 04 2019', 'Series_last_update_date': 'Feb 18 2020', 'Series_pubmed_id': '31975687', 'Series_summary': 'RNA granules are cellular condensates that contain RNAs and proteins. The mechanism that drive the recruitment of many mRNAs to RNA granules are not fully understood.  Here we characterize the assembly and transcriptome of the germ (P) granules of C. elegans. We find that mRNAs are recruited into P granules by condensation with the intrinsically-disordered protein MEG-3. MEG-3 binds ~500 mRNAs in C. elegans embryos in a sequence-independent manner that favors mRNAs with low ribosome coverage. Translational stress causes additional mRNAs to localize to P granules and translational activation correlates with P granule exit for two mRNAs coding for germ cell fate regulators. MEG-3/RNA condensates assembled in vitro are gel-like and trap mRNAs in a non-dynamic state. Our observations reveal similarities between P granules and stress granules and identify gelation with intrinsically-disordered proteins as a sequence-independent mechanism to enrich low-translation mRNAs in RNA granules', 'Series_overall_design': '12 embryonic samples are included in this submission. It contains 6 RNAseq (3 wild type and 3 meg-3meg-4 mutant) and 6 corresponding ribosome profiling (3 wild type and 3 meg-3meg-4 mutant).', 'Series_type': 'Other', 'Series_contributor': 'Geraldine,,Seydoux', 'Series_sample_id': 'GSM4148088', 'Series_contact_name': 'Geraldine,,Seydoux', 'Series_contact_email': 'gseydoux@jhmi.edu', 'Series_contact_laboratory': 'Dr. Geraldine Seydoux', 'Series_contact_department': 'MBG', 'Series_contact_institute': 'Johns Hopkins University', 'Series_contact_address': '725 N. wolfe St. PCTB 706', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21205', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE139nnn/GSE139880/suppl/GSE139880_Ribosom_occupancy_for_wild-type.xlsx', 'Series_platform_id': 'GPL18245', 'Series_platform_taxid': '6239', 'Series_sample_taxid': '6239', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP228538'}\n",
      "325 PRJNA587699 GSE139939 {'Series_title': 'Genome-scale transcription–translation mapping reveals novel features of Zymomonas mobilis promoters and transcription units', 'Series_geo_accession': 'GSE139939', 'Series_status': 'Public on Jul 09 2020', 'Series_submission_date': 'Nov 05 2019', 'Series_last_update_date': 'Oct 08 2020', 'Series_pubmed_id': '32694125', 'Series_summary': 'We collected sample-matched multiomics data including RNA-seq, transcription start site sequencing (TSS-seq), termination sequencing (term-seq), ribosome profiling, and label-free shotgun proteomic mass spectrometry across different growth conditions to improve annotation and assign functional sites in the Zymomonas mobilis subsp. mobilils ZM4 genome. Proteomics and ribosome profiling informed revisions of protein-coding genes, which included 44 start codon changes and 42 added proteins. We developed statistical methods for annotating transcript 5′ and 3′ ends enabling identification of 3940 TSSs and their corresponding promoters and 2091 transcription termination sites, which were distinguished from RNA processing sites by lack of an adjacent RNA 5′ end.', 'Series_overall_design': '3 or 4 biological replicate samples of cells grown aerobically or anaerobically in rich media and anaerobically in minimal media. Samples were collected at growing phase and stationary phase time points.', 'Series_type': 'Other', 'Series_contributor': 'Robert,,Landick', 'Series_sample_id': 'GSM4150249', 'Series_contact_name': 'Robert,,Landick', 'Series_contact_email': 'landick@bact.wisc.edu', 'Series_contact_institute': 'University of Wisconsin', 'Series_contact_address': '1550 Linden Dr', 'Series_contact_city': 'Madison', 'Series_contact_state': 'WI', 'Series_contact_zip/postal_code': '53706', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE139nnn/GSE139939/suppl/GSE139939_compiledDESeq2.results.txt.gz', 'Series_platform_id': 'GPL27722', 'Series_platform_taxid': '264203', 'Series_sample_taxid': '264203', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP228586'}\n",
      "326 PRJNA588223 GSE140084 {'Series_title': 'Master stress response gene ATF4 & other oncogenes are translated by DENR-dependent reinitiation', 'Series_geo_accession': 'GSE140084', 'Series_status': 'Public on Jul 18 2020', 'Series_submission_date': 'Nov 07 2019', 'Series_last_update_date': 'Sep 24 2020', 'Series_pubmed_id': '32938922', 'Series_summary': 'Translation efficiency varies 1000-fold between different mRNAs, thereby strongly impacting protein expression. Translation of the master stress response gene ATF4 increases in response to stress, but the molecular mechanisms are not well understood. We discover here that translation initiation factors DENR, MCTS1 and eIF2D are absolutely required to induce ATF4 translation upon stress, by promoting translation reinitiation on the ATF4 5’UTR. Hence DENR and MCTS1 are important players in the cellular Integrated Stress Response. We find DENR and MCTS1 promote reinitiation after long uORFs with specific penultimate codons, due to the tRNA that remains attached to 40S ribosomes after translation termination. This provides a model for how DENR and MCTS1 promote translation reinitiation. Since cancer cells are exposed to many stresses, they require ATF4 for survival and proliferation. We find a strong correlation between DENR•MCTS1 expression and ATF4 activity across cancers. Additional oncogenes including a-Raf, c-Raf and Cdk4 have long uORFs and are translated in a DENR•MCTS1 dependent manner. This explains in part why DENR and MCTS1 are oncogenes.', 'Series_overall_design': 'We created a DENR KO Hela Cell line and carried out ribosome footprinting to determine translational targets of the DENR-MCTS1 complex.', 'Series_type': 'Other', 'Series_contributor': 'Aurelio,,Teleman', 'Series_sample_id': 'GSM4478781', 'Series_contact_name': 'Jonathan,,Bohlen', 'Series_contact_laboratory': 'AG Teleman', 'Series_contact_department': 'B140', 'Series_contact_institute': 'DKFZ', 'Series_contact_address': 'Im Neuenheimer Feld 581', 'Series_contact_city': 'Heidelberg', 'Series_contact_zip/postal_code': '69123', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE140nnn/GSE140084/suppl/GSE140084_Master_Data_Table.xlsx', 'Series_platform_id': 'GPL21697', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP229088'}\n",
      "327 PRJNA589056 GSE140255 {'Series_title': 'Analysis of lacking Nα-terminal acetylation of proteins by NatA and NatB in Saccharomyces cerevisiae on mRNA levels and protein synthesis', 'Series_geo_accession': 'GSE140255', 'Series_status': 'Public on Jul 27 2020', 'Series_submission_date': 'Nov 12 2019', 'Series_last_update_date': 'Jul 29 2020', 'Series_summary': 'N-terminal (Nt)-acetylation is a highly prevalent co-translational protein modification in eukaryotes, catalyzed by at least five Nt-acetyltransferases (Nat) with differing specificities. Nt-acetylation has been implicated in protein quality control but its broad biological significance remains elusive. We investigated the roles of the two major Nats of S. cerevisiae, NatA and NatB, by performing transcriptome and translatome profiling by using mRNA sequencing and ribosome profiling. The results are combined with global proteome, aggregome and stabilome datasets. To analyze previously proposed physiolocal roles of NatA and NatB in yeast cells on protein stability and interaction specific growth conditions such as different growth media and heat stress are used in addition to physiological growth.', 'Series_overall_design': 'mRNA sequencing and ribosome profiling of generated yeast strains lacking the N-terminal acetyltransferase complex catalytic subunit Naa10 (for NatA) or Naa20 (for NatB) in wild-type background or hmlΔ background under different growth conditions (as indicated), each sample is prepared in replicates and wild-type yeast strain is used as reference (also in replicates)', 'Series_type': 'Other', 'Series_contributor': 'Bernd,,Bukau', 'Series_sample_id': 'GSM4158427', 'Series_contact_name': 'Ulrike,Anne,Friedrich', 'Series_contact_laboratory': 'Bernd Bukau', 'Series_contact_department': 'ZMBH', 'Series_contact_institute': 'Heidelberg University', 'Series_contact_address': 'Im Neuenheimer Feld 282', 'Series_contact_city': 'Heidelberg', 'Series_contact_zip/postal_code': 'D-69120', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE140nnn/GSE140255/suppl/GSE140255_RAW.tar', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP229607'}\n",
      "328 PRJNA589466 GSE140366 {'Series_title': 'Robust partitioning of microRNA targets from downstream regulatory changes [Ribo-seq]', 'Series_geo_accession': 'GSE140366', 'Series_status': 'Public on Aug 07 2020', 'Series_submission_date': 'Nov 13 2019', 'Series_last_update_date': 'Aug 24 2020', 'Series_pubmed_id': '32821933', 'Series_summary': 'MicroRNA’s function is defined by the targets it regulates. Despite miRNAs’ crucial role in various cellular processes and disease states, the identification of their targets remains challenging. Existing methods suffer from high false-positive rates and are inadequate in differentiating direct targets from downstream regulatory changes. Here, we introduce a simple approach involving combined analysis of RNA-seq and Precise RunOn-seq (PRO-seq) (CARP) to effectively measure post-transcriptional regulation. We apply this approach to seven different miRNAs and robustly distinguish their direct targets from downstream changes. We validate the efficacy of our approach using argonaute-CLIP-seq. We show using CARP and ribosome-profiling analyses that accelerated mRNA decay is the major mode of target regulation by miRNAs. Additionally, our comparative analyses reveal that CARP aids in discovery of complex regulatory mechanisms and in capturing false-negatives of existing approaches. Furthermore, post-transcriptional changes measured using CARP uncover miRNA-dependent regulation of coding sites. Lastly, we demonstrate that PRO-seq provides mechanistic explanation of indirect targeting, unraveling miRNA regulatory networks. Finally, we believe that CARP is particularly suitable to study in vivo functions of miRNAs in primary cells.', 'Series_overall_design': 'Ribosome profiling of HEK293 cells with and without specific miRNAs by high-throughput sequencing, in duplicate, using Illumina NextSeq500', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Andrew,W,Grimson', 'Series_sample_id': 'GSM4160143', 'Series_contact_name': 'Andrew,,Grimson', 'Series_contact_department': 'Molecular Biology and Genetics', 'Series_contact_institute': 'Cornell University', 'Series_contact_address': '445 Biotech Bldg', 'Series_contact_city': 'Ithaca', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '14850', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE140nnn/GSE140366/suppl/GSE140366_codingRegion.read.count.tsv.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP229759'}\n",
      "329 PRJNA590189 GSE140565 {'Series_title': 'Ribosome profiling and RNA-seq of Fragile X mouse brain cortex', 'Series_geo_accession': 'GSE140565', 'Series_status': 'Public on Jun 01 2020', 'Series_submission_date': 'Nov 18 2019', 'Series_last_update_date': 'Jun 03 2020', 'Series_summary': 'Fragile X syndrome (FXS) is caused by inactivation of FMR1 gene and loss of its encoded product the RNA binding protein FMRP, which generally represses translation of its target transcripts in the brain. In mouse models of FXS (i.e., Fmr1 knockout animals; Fmr1 KO), deletion of Cpeb1, which encodes a translational activator, mitigates nearly all pathophysiologies associated with the disorder. Here we reveal unexpected wide-spread dys-regulation of RNA abundance in Fmr1 KO brain cortex and its rescue to normal levels in Fmr1/Cpeb1 double KO mice. Alteration and restoration of RNA levels are the dominant molecular events that drive the observed dys-regulation and rescue of translation as measured by whole transcriptome ribosome occupany in the brain. The RNAs down-regulated and rescued in these animal models are highly enriched for FMRP binding targets and have an optimal codon bias that would predict their stability in wild type and possible instability in FMRP knock-out brain. These results leads to a further study to profile RNA metabolism rates in Fragile X neurons.', 'Series_overall_design': 'Please note that differential expression analyses between samples were generated by DEseq2 (RNADEseq_* and RPFDEseq_*) or Xtail (ROXtail_*) as indicated in the corresponding sample description field.', 'Series_type': 'Other', 'Series_contributor': 'Joel,D,Richter', 'Series_sample_id': 'GSM4174248', 'Series_contact_name': 'Huan,Raya,Shu', 'Series_contact_email': 'dr.huan.shu@gmail.com', 'Series_contact_laboratory': 'Richter Lab', 'Series_contact_department': 'Program in Molecular Medicine', 'Series_contact_institute': 'University of Massachusetts Medical School', 'Series_contact_address': '373 Plantation St', 'Series_contact_city': 'Worcester', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '01605', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE140nnn/GSE140565/suppl/GSE140565_RPFDEseq_FKxWT.txt.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP230417'}\n",
      "330 PRJNA590355 GSE140639 {'Series_title': 'Genome-wide survey of ribosome collision [anisomycin]', 'Series_geo_accession': 'GSE140639', 'Series_status': 'Public on May 05 2020', 'Series_submission_date': 'Nov 19 2019', 'Series_last_update_date': 'May 29 2020', 'Series_pubmed_id': '32375038', 'Series_summary': 'In protein synthesis, ribosome movement is not always smooth, rather often impeded by numerous reasons. Although the deceleration of ribosome defines the fates of the mRNAs and the synthesizing proteins, fundamental questions remain to be addressed including where ribosomes pause in mRNAs, what kind of RNA/amino acid context causes the pausing, and how physiologically significant the slowdown of protein synthesis is. Here we surveyed the position of ribosome collisions, caused by ribosome pausing, at a genome-wide level using the modified ribosome profiling in human and zebrafish. The collided ribosomes, i.e. disome, emerge at various sites; the proline-proline-lysine motif, stop codons, and 3′ UTR. The number of ribosomes in a collision is not limited to two, rather four to five, forming a queue of ribosomes. Especially, XBP1, a key modulator of unfolded protein response, shows striking queues of collided ribosomes thus acts as a substrate for ribosome-associated quality control (RQC) to avoid the accumulation of undesired proteins in the absence of stress. Our results provide an insight into the causes and the consequences of ribosome slowdowns by dissecting the specific architecture of ribosomes.', 'Series_overall_design': 'Ribosome profiling', 'Series_type': 'Other', 'Series_contributor': 'Shintaro,,Iwasaki', 'Series_sample_id': 'GSM4176098', 'Series_contact_name': 'Peixun,,Han', 'Series_contact_institute': 'Riken', 'Series_contact_address': 'Hirosawa Wako', 'Series_contact_city': 'Saitama', 'Series_contact_zip/postal_code': '351-0198', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE140nnn/GSE140639/suppl/GSE140639_Disome_Tri_ans_merge.1.txt.gz', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP230602'}\n",
      "331 PRJNA591625 GSE140997 {'Series_title': 'Sequence- and structure-specific cytosine-5 methylation in mRNAs by NSUN6', 'Series_geo_accession': 'GSE140997', 'Series_status': 'Public on Nov 20 2020', 'Series_submission_date': 'Nov 25 2019', 'Series_last_update_date': 'Feb 19 2021', 'Series_pubmed_id': '33330931', 'Series_summary': 'Methylation is the most common internal modification in mRNA. While the highly abundant N6-methyladonsine (m6A) modification affects most aspects of mRNA function, the precise functions of the rarer 5-methylcytosine (m5C) remains largely unknown. Here, we map m5C in the human transcriptome using methylation-dependent individual-nucleotide resolution cross-linking and immunoprecipitation (miCLIP) combined with RNA bisulfite sequencing. We identify NSUN6 as a methyltransferase with strong substrate specificity towards mRNA. NSUN6 primarily targeted three prime untranslated regions (3’UTR) at the consensus sequence motif CTCCA. Knockout and rescue experiments revealed that only mRNA methylation sites containing the consensus motif depended on the presence of NSUN6. Furthermore, ribosome profiling demonstrated that NSUN6-specific consensus motifs marked translation termination. However, even though NSUN6-methylated mRNAs were reduced in NSUN6 knockout cells, NSUN6 was dispensable for mouse embryonic development. Thus, our study identifies NSUN6 as methyltransferase targeting mRNA in a sequence- and structure-specific manner.', 'Series_overall_design': 'Identification of NSUN6 methylation targets by miCLIP in human HEK293 cells.', 'Series_type': 'Other', 'Series_contributor': 'Michaela,,Frye', 'Series_sample_id': 'GSM4195001', 'Series_contact_name': 'Susanne,,Bornelöv', 'Series_contact_department': 'WT-MRC Cambridge Stem Cell Institute', 'Series_contact_institute': 'University of Cambridge', 'Series_contact_address': 'Tennis Court Road', 'Series_contact_city': 'Cambridge', 'Series_contact_zip/postal_code': 'CB2 1QR', 'Series_contact_country': 'United Kingdom', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE140nnn/GSE140997/suppl/GSE140997_Ribo_counts_CDS.txt.gz', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP233222'}\n",
      "332 PRJNA591627 GSE140995 {'Series_title': 'Multiple links between 5-methylcytosine content of mRNA and translation', 'Series_geo_accession': 'GSE140995', 'Series_status': 'Public on Mar 09 2020', 'Series_submission_date': 'Nov 25 2019', 'Series_last_update_date': 'Apr 27 2020', 'Series_pubmed_id': '32293435', 'Series_summary': '5-methylcytosine (m5C) is a prevalent base modification in tRNA and rRNA but it also occurs more broadly in the transcriptome, including in mRNA. In pursuit of potential links of m5C with mRNA translation, we performed polysome profiling of human HeLa cell lysates and subjected RNA from resultant fractions to efficient bisulfite conversion followed by RNA sequencing (bsRNA-seq). Bioinformatic filters for rigorous site calling were devised to reduce technical noise. We obtained ~1,000 candidate m5C sites in the wider transcriptome, most of which were found in exonic regions of mRNA. Multiple novel sites were validated by amplicon-specific bsRNA-seq in independent samples of either human HeLa, LNCap and PrEC cells. Furthermore, RNAi-mediated depletion of either the NSUN2 or TRDMT1 m5C:RNA methyltransferases showed a clear dependence on NSUN2 for the majority of tested sites in mRNA and noncoding RNA. Candidate m5C sites in mRNAs are enriched in 5’UTRs and near start codons, and commonly embedded in a local context reminiscent of the NSUN2-dependent m5C sites found in the variable loop of tRNA. Analysing mRNA sites across the polysome profile revealed that non-conversion levels, at bulk and for many individual sites, were inversely correlated with ribosome association. Altogether, these findings emphasise the major role of NSUN2 in making this mark transcriptome-wide and further substantiate a functional interdependence of cytosine methylation level with mRNA translation', 'Series_overall_design': 'Bisulfite RNA sequencing of HeLa cell polysome profiling-derived samples representing different translational states; in biological triplicates.', 'Series_type': 'Other', 'Series_contributor': 'Thomas,,Preiss', 'Series_sample_id': 'GSM4192083', 'Series_contact_name': 'Thomas,,Preiss', 'Series_contact_email': 'thomas.preiss@anu.edu.au', 'Series_contact_laboratory': 'RNA biology', 'Series_contact_department': 'Genome Science', 'Series_contact_institute': 'Australian National University, John Curtin School of Medical Research', 'Series_contact_address': '31 Garran Road', 'Series_contact_city': 'Canberra', 'Series_contact_state': 'Acton', 'Series_contact_zip/postal_code': '2601', 'Series_contact_country': 'Australia', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE140nnn/GSE140995/suppl/GSE140995_transcriptome_fraction_sites.xlsx', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP233220'}\n",
      "333 PRJNA591767 GSE141013 {'Series_title': 'Integrated analyses of early responses to radiation in glioblastoma identify new alterations in RNA processing and candidate targets to improve treatment outcomes', 'Series_geo_accession': 'GSE141013', 'Series_status': 'Public on Nov 26 2019', 'Series_submission_date': 'Nov 26 2019', 'Series_last_update_date': 'Jun 08 2020', 'Series_pubmed_id': '32488114', 'Series_summary': 'High-dose radiation is the main component of glioblastoma therapy. Unfortunately, radio-resistance is a common problem and a major contributor to tumor relapse. Understanding the molecular mechanisms driving response to radiation is critical for identifying regulatory routes that could be targeted to improve treatment response. We conducted an integrated analysis in the U251 and U343 glioblastoma cell lines to map early alterations in the expression of genes at three levels: transcription, splicing, and translation in response to ionizing radiation.', 'Series_overall_design': 'Expression and Ribosome profilling of U251 and U343 glioblastoma cell lines at three time points T0, T1, and T24. U251 RNA-seq: T0 (2 replicates); T1 (3 replicates); T24 (3 replicates). U343 RNA-seq: T0 (3 replicates); T1( 3 replicates); T24 (3 replicates). U251 Ribo-seq: T0 (2 replicates); T1 (3 replicates); T24 (3 replicates). U343 Ribo-seq: T0 (3 replicates); T1( 3 replicates); T24 (3 replicates).', 'Series_type': 'Other', 'Series_contributor': 'Saket,,Choudhary', 'Series_sample_id': 'GSM4192441', 'Series_contact_name': 'David,,Kadosh', 'Series_contact_email': 'kadosh@uthscsa.edu', 'Series_contact_phone': '(210) 567-3976', 'Series_contact_department': 'Department of Microbiology, Immunology & Molecular Genetics', 'Series_contact_institute': 'University of Texas Health Science Center at San Antonio', 'Series_contact_address': '7703 Floyd Curl Dr., MC: 7758', 'Series_contact_city': 'San Antonio', 'Series_contact_state': 'Texas', 'Series_contact_zip/postal_code': '78229', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE141nnn/GSE141013/suppl/GSE141013_RAW.tar', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP233276'}\n",
      "334 PRJNA593553 GSE141459 {'Series_title': 'ZAK activation on colliding ribosomes triggers SAPK signaling pathways to regulate cell fate', 'Series_geo_accession': 'GSE141459', 'Series_status': 'Public on Aug 18 2020', 'Series_submission_date': 'Dec 04 2019', 'Series_last_update_date': 'Aug 18 2020', 'Series_pubmed_id': '32610081', 'Series_summary': 'Problems arising during translation of mRNAs lead to ribosome stalling and collisions with trailing ribosomes. These collisions are known to trigger a series of events including degradation of the stalled nascent polypeptide, decay of the problematic mRNAs, and ribosome rescue. However, the systemic cellular response of ribosome collision has not been explored. Here, we uncover a novel function for ribosome collisions in signal transduction. Using multiple translation elongation inhibitors and cellular stress conditions, we show that ribosome collisions activate both the SAPK (Stress Activated Protein Kinase) and GCN2-mediated cellular stress response pathways that lead to apoptosis and the integrated stress response (ISR), respectively. We further show that the MAPKKK ZAK functions as the sentinel for ribosome collisions, and ZAK is required for activation of both SAPKs p38/JNK and GCN2 signaling pathways. Selective ribosome profiling and biochemistry demonstrate that ZAK preferentially associates with the minimal unit of colliding ribosomes, the disome, inducing ZAK phosphorylation. While ZAK associates with elongating ribosomes under non-stressed conditions, activation of ZAK is specifically trigger by colliding ribosomes. Together, these results provide new insights into how perturbation of translational homeostasis, as read-out by colliding ribosomes, regulates cell fate.', 'Series_overall_design': '4 samples for ribosome profiling experiments using HeLa cells.', 'Series_type': 'Other', 'Series_contributor': 'Rachel,,Green', 'Series_sample_id': 'GSM4485599', 'Series_contact_name': 'Colin,,Wu', 'Series_contact_department': 'RNA Biology Laboratory', 'Series_contact_institute': 'NCI', 'Series_contact_address': '1050 Boyles St, BG 560, RM21-102C', 'Series_contact_city': 'Frederick', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21702', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE141nnn/GSE141459/suppl/GSE141459_HelaUnt_UV_20to23A_occupancy.csv.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP234715'}\n",
      "335 PRJNA594022 GSE141599 {'Series_title': 'Autism-misregulated eIF4G microexons control synaptic translation and higher-order cognitive functions [Ribo-Seq]', 'Series_geo_accession': 'GSE141599', 'Series_status': 'Public on Jan 30 2020', 'Series_submission_date': 'Dec 06 2019', 'Series_last_update_date': 'Apr 30 2020', 'Series_pubmed_id': '31999954', 'Series_summary': 'Microexons represent the most highly conserved class of alternative splicing, yet their functions are poorly understood. Here, we focus on closely related neuronal microexons overlapping prion-like domains in the translation initiation factors, eIF4G1 and eIF4G3, the splicing of which is activity-dependent and frequently disrupted in autism. CRISPR-Cas9 deletion of these microexons selectively up-regulates synaptic proteins that control neuronal activity and plasticity and further triggers a gene expression program mirroring that of activated neurons. Mice lacking the Eif4g1 microexon display social behavior, learning and memory deficits, as well as alterations in hippocampal synaptic plasticity. The eIF4G microexons appear to reduce synaptic protein synthesis by causing ribosome stalling, through a mechanism whereby they promote the coalescence of cytoplasmic granule components associated with translation repression, including the Fragile X mental retardation protein, FMRP. The results thus reveal an autism-disrupted mechanism by which alternative splicing specializes translation to control higher-order cognitive functioning.', 'Series_overall_design': 'Ribo-seq and control RNA-seq in wt and Eif4g1/Eif4g3 double-knockout in vitro-differentiated neurons', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Benjamin,J,Blencowe', 'Series_sample_id': 'GSM4209400', 'Series_contact_name': 'Ulrich,,Braunschweig', 'Series_contact_laboratory': 'Benjamin J. Blencowe', 'Series_contact_department': 'Donnelly Centre', 'Series_contact_institute': 'University of Toronto', 'Series_contact_address': '160 College Street', 'Series_contact_city': 'Toronto', 'Series_contact_state': 'Ontario', 'Series_contact_zip/postal_code': 'M5S 3E1', 'Series_contact_country': 'Canada', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE141nnn/GSE141599/suppl/GSE141599_kallisto_readcount_prot_trans_tsl1-3_v2.tab.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP234998'}\n",
      "336 PRJNA594648 GSE141765 {'Series_title': 'Translational regulation in response to boron in Arabidopsis thaliana', 'Series_geo_accession': 'GSE141765', 'Series_status': 'Public on Feb 13 2022', 'Series_submission_date': 'Dec 10 2019', 'Series_last_update_date': 'Feb 13 2022', 'Series_pubmed_id': '33772920', 'Series_summary': \"We previously reported that ribosome stalling at AUG-stop sequences in the 5'-UTR plays a critical role in regulating the expression of Arabidopsis thaliana NIP5;1, which encodes a boron uptake transporter, in response to boron conditions in media. Here, we conducted ribosome profiling analysis to reveal the genome-wide regulation of translation in response to boron conditions in A. thaliana. We identified 460 translationally regulated genes. Transcripts with reduced translation efficiency were rich in upstream open reading frames (uORFs), highlighting the importance of uORF-mediated translational regulation. We found that 148 uORF instances had greater ribosome density under high boron conditions. Moreover, translationally downregulated transcripts were rich in minimum uORFs (AUG-stops), suggesting that AUG-stops play a global role in the boron response. Boron increased the ribosome occupancy of stop codons, indicating that this element is involved in global translational termination processes.\", 'Series_overall_design': 'Whole shoot tissues of 14-day-old Arabidopsis thaliana grown under three different boric acid concentrations (–B, 0.3 µM; Ctrl, 30 µM; ++B, 500 µM). 2 biological replicates.', 'Series_type': 'Other', 'Series_contributor': 'Toru,,Fujiwara', 'Series_sample_id': 'GSM4212890', 'Series_contact_name': 'Naoyuki,,Sotta', 'Series_contact_email': 'nsotta@gmail.com', 'Series_contact_phone': '+81-3-5841-5104', 'Series_contact_laboratory': 'Laboratory of Plant Nutrition and Fertilizers', 'Series_contact_department': 'Department of Applied Biological Chemistry', 'Series_contact_institute': 'Graduate School of Agricultural and Life Sciences, The University of Tokyo', 'Series_contact_address': '1-1-1 Yayoi, Bunkyo-ku', 'Series_contact_city': 'Tokyo', 'Series_contact_zip/postal_code': '113-8657', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE141nnn/GSE141765/suppl/GSE141765_sumcount.RNA.CDS.csv.gz', 'Series_platform_id': 'GPL17639', 'Series_platform_taxid': '3702', 'Series_sample_taxid': '3702', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP235569'}\n",
      "337 PRJNA594945 GSE141842 {'Series_title': 'Change in translation efficiency in mouse eyes at P0.5 by RNG140 knockout', 'Series_geo_accession': 'GSE141842', 'Series_status': 'Public on Jul 31 2020', 'Series_submission_date': 'Dec 11 2019', 'Series_last_update_date': 'Oct 30 2020', 'Series_pubmed_id': '32839273', 'Series_summary': 'RNG140 is an RNA-binding protein that increases in expression during eye lens differentiation and is involved in lens formation. However, the relevance of RNG140-mediated translational regulation to differentiation is not well understood. RNG140-mediated translational regulation operates in the mouse eye, where RNG140 knockout increased the translation of long mRNAs. mRNAs involved in lens differentiation, such as crystallin mRNAs, are short, and can escape translational inhibition by RNG140 and be translated in differentiating lenses.', 'Series_overall_design': 'Ribosome profiling and RNA-seq in the eye from P0.5 RNG140 knockout mice and control mice', 'Series_type': 'Other', 'Series_contributor': 'Nobuyuki,,Shiina', 'Series_sample_id': 'GSM4214059', 'Series_contact_name': 'Kaori,,Nakazawa', 'Series_contact_email': 'katayama@nibb.ac.jp', 'Series_contact_institute': 'National Institute for Basic Biology', 'Series_contact_address': 'Nishigonaka 38, Myodaiji', 'Series_contact_city': 'Okazaki', 'Series_contact_zip/postal_code': '444-8585', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE141nnn/GSE141842/suppl/GSE141842_RAW.tar', 'Series_platform_id': 'GPL21103', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP236878'}\n",
      "338 PRJNA594947 GSE141840 {'Series_title': 'Change in translation efficiency by RNG140-GFP overexpression in CHO cells', 'Series_geo_accession': 'GSE141840', 'Series_status': 'Public on Jul 31 2020', 'Series_submission_date': 'Dec 11 2019', 'Series_last_update_date': 'Oct 30 2020', 'Series_pubmed_id': '32839273', 'Series_summary': 'RNG140 is an RNA binding protein involved in the regulation of various cell differentiation and functional processes. Although RNG140 has been shown to inhibit translation in rabbit reticulocyte lysates, it is not known what kind of mRNAs were regulated their translation by RNG140. Comprehensive ribosome profiling revealed that overexpression of RNG140 in cultured CHO cells reduces translation of long mRNAs, including those associated with cell proliferation.', 'Series_overall_design': 'Ribosome profiling and RNA-seq in RNG140-GFP-expressing CHO cells and control CHO cells', 'Series_type': 'Other', 'Series_contributor': 'Nobuyuki,,Shiina', 'Series_sample_id': 'GSM4214039', 'Series_contact_name': 'Kaori,,Nakazawa', 'Series_contact_email': 'katayama@nibb.ac.jp', 'Series_contact_institute': 'National Institute for Basic Biology', 'Series_contact_address': 'Nishigonaka 38, Myodaiji', 'Series_contact_city': 'Okazaki', 'Series_contact_zip/postal_code': '444-8585', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE141nnn/GSE141840/suppl/GSE141840_RAW.tar', 'Series_platform_id': 'GPL26969', 'Series_platform_taxid': '10029', 'Series_sample_taxid': '10029', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP236875'}\n",
      "339 PRJNA598359 GSE142800 {'Series_title': 'RNA-Seq and Ribo-Seq of P19 WT and SRSF7-GFP overexpressing cells', 'Series_geo_accession': 'GSE142800', 'Series_status': 'Public on Mar 02 2020', 'Series_submission_date': 'Dec 31 2019', 'Series_last_update_date': 'Mar 30 2020', 'Series_pubmed_id': '32123389', 'Series_summary': 'We used P19 cells that overexpress GFP-tagged SRSF7 by 3.4-fold.', 'Series_overall_design': '1-2 biological replicates were subjected to RNA-Seq and Ribo-Seq.', 'Series_type': 'Other', 'Series_contributor': 'Michaela,,Müller-McNicoll', 'Series_sample_id': 'GSM4240284', 'Series_contact_name': 'Michaela,,Müller-McNicoll', 'Series_contact_email': 'mueller-mcnicoll@bio.uni-frankfurt.de', 'Series_contact_phone': '+49 69 798 42079', 'Series_contact_laboratory': 'RNA Regulation Group', 'Series_contact_department': 'Institute for Cell Biology and Neuroscience', 'Series_contact_institute': 'Goethe University Frankfurt', 'Series_contact_address': 'Max-von-Laue-Str. 13', 'Series_contact_city': 'Frankfurt am Main', 'Series_contact_zip/postal_code': '60438', 'Series_contact_country': 'Germany', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP239192'}\n",
      "340 PRJNA598453 GSE142817 {'Series_title': 'A novel mechanism of bacteria to adapt to nutrient deprivation\\xa0via\\xa0the ribosome silencing factor RsfS (reference number: NMICROBIOL-19112987)', 'Series_geo_accession': 'GSE142817', 'Series_status': 'Public on Aug 22 2022', 'Series_submission_date': 'Jan 01 2020', 'Series_last_update_date': 'Aug 22 2022', 'Series_summary': 'The factor RsfS is present in almost all bacteria and organelles. RsfS is important under conditions of nutrient downshifts leading to decreasing growth rates. A common feature of these conditions is a strong reduction of the translation capacity revealed by polysomal fraction decrease and a vast appearance of vacant 70S ribosomes, which might cause a devastating energy drain by stimulating the activity of GTPase proteins e.g. EF-G with idle ribosomes. Here we show that RsfS blocks association of free ribosomal subunits and induces dissociation exclusively of vacant ribosomes rather than of programmed ribosomes carrying tRNAs. As a consequence, RsfS reduces the abundance of the cellular translational apparatus, and the EF-G dependent burst of GTP cleavage in the presence of vacant 70S ribosomes as well as with 30S and 50S subunits is decreased. These observations hint to a regulatory role of RsfS by utilizing cellular resource in an economic way under conditions, where the amino acids are limited and energy has to be saved.', 'Series_overall_design': 'Ribo-seq of E. coli strain BW25113 and BW25113_ΔRsfS in M9 minimal medium.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Gong,,Zhang', 'Series_sample_id': 'GSM4240578', 'Series_contact_name': 'zhang,,hong', 'Series_contact_email': 'alexnoname@outlook.com', 'Series_contact_phone': '13631392383', 'Series_contact_institute': 'zhanghong', 'Series_contact_address': 'Huangpu Boulavard West Jinan uinversity', 'Series_contact_city': 'guangzhou', 'Series_contact_state': 'guangdong province', 'Series_contact_zip/postal_code': '510632', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE142nnn/GSE142817/suppl/GSE142817_r4_2f.txt.gz', 'Series_platform_id': 'GPL25852', 'Series_platform_taxid': '679895', 'Series_sample_taxid': '679895', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP239234'}\n",
      "341 PRJNA599943 GSE143301 {'Series_title': 'Ribosome profiling under ABCE1 depletion to study aberrant translation termination', 'Series_geo_accession': 'GSE143301', 'Series_status': 'Public on Jun 30 2020', 'Series_submission_date': 'Jan 08 2020', 'Series_last_update_date': 'Oct 15 2020', 'Series_pubmed_id': '32807779', 'Series_summary': 'We report the effect of siRNA-mediated ABCE1 KD in HeLa cells, using a modified ribosome profiling protocol adapted from Ingolia et al., 2011 in order to specifically investigate ABCE1 role on translation termination', 'Series_overall_design': 'Control and ABCE1 KD conditions, three replicates for condition. totalRNA samples in parallel for input control.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'O,,Muhlemann', 'Series_sample_id': 'GSM4407916', 'Series_contact_name': 'Rene,,Dreos', 'Series_contact_email': 'rene.dreos@unil.ch', 'Series_contact_phone': '0044 (0)1603 450793', 'Series_contact_laboratory': 'Gatfield Lab', 'Series_contact_department': 'CIG', 'Series_contact_institute': 'UNIL', 'Series_contact_address': 'Ecublens', 'Series_contact_city': 'Lausanne', 'Series_contact_state': 'VD', 'Series_contact_zip/postal_code': '1015', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE143nnn/GSE143301/suppl/GSE143301_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP240357'}\n",
      "342 PRJNA600028 GSE143331 {'Series_title': 'FMRP Control of Ribosome Translocation Promotes Chromatin Modifications and Alternative Splicing of Neuronal Genes Linked to Autism (Hrt timecourse ribosome profiling)', 'Series_geo_accession': 'GSE143331', 'Series_status': 'Public on Mar 31 2020', 'Series_submission_date': 'Jan 08 2020', 'Series_last_update_date': 'Apr 27 2020', 'Series_pubmed_id': '32234480', 'Series_summary': 'Silencing of FMR1 and loss of its gene product FMRP results in Fragile X Syndrome. FMRP binds brain mRNAs and inhibits polypeptide elongation. Using ribosome profiling of the hippocampus, we find that ribosome footprint levels in Fmr1-deficient tissue mostly reflect changes in RNA abundance. Profiling over a time course of ribosome runoff in wildtype tissue reveals a wide range of ribosome translocation rates; on many mRNAs, the ribosomes are stalled. Sucrose gradient ultracentrifugation of hippocampal slices after ribosome runoff reveals that FMRP co-sediments with stalled ribosomes; and its loss results in decline of ribosome stalling on specific mRNAs. One such mRNA encodes SETD2, a lysine methyltransferase that catalyzes H3K36me3. ChIP-Seq demonstrates that loss of FMRP alters the deployment of this epigenetic mark on chromatin. H3K36me3 is associated with alternative pre-RNA processing, which we find occurs in an FMRP-dependent manner on transcripts linked to neural function and autism spectrum disorders.', 'Series_overall_design': 'Ribosome profiling of Transverse hippocampal or hippocampal-cortical brain slices treated with homoharringtonine for 0,5,10,30 or 60 minutes respectively, were acutely prepared from P28-35 C57BL/6N wild-type or Fmr1 KO male mice littermates using Illumina Nextseq.', 'Series_type': 'Other', 'Series_contributor': 'Joel,D,Richter', 'Series_sample_id': 'GSM4257250', 'Series_contact_name': 'Sneha,,Shah', 'Series_contact_email': 'sneha.19.6@gmail.com', 'Series_contact_laboratory': 'Joel Richter', 'Series_contact_department': 'Molecular Medicine', 'Series_contact_institute': 'UMASS CHAN MEDICAL SCHOOL', 'Series_contact_address': '373 Biotech II, Suite 204, Plantation Street', 'Series_contact_city': 'Worcester', 'Series_contact_state': 'Massachusetts', 'Series_contact_zip/postal_code': '01605', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE143nnn/GSE143331/suppl/GSE143331_slice.hrt.ribo.cds.wo.ends.rsem.gene.summary.tpm.txt.gz', 'Series_platform_id': 'GPL9185', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP240704'}\n",
      "343 PRJNA600029 GSE143330 {'Series_title': 'FMRP Control of Ribosome Translocation Promotes Chromatin Modifications and Alternative Splicing of Neuronal Genes Linked to Autism (Steady-state ribosome profiling)', 'Series_geo_accession': 'GSE143330', 'Series_status': 'Public on Mar 31 2020', 'Series_submission_date': 'Jan 08 2020', 'Series_last_update_date': 'Apr 27 2020', 'Series_pubmed_id': '32234480', 'Series_summary': 'Silencing of FMR1 and loss of its gene product FMRP results in Fragile X Syndrome. FMRP binds brain mRNAs and inhibits polypeptide elongation. Using ribosome profiling of the hippocampus, we find that ribosome footprint levels in Fmr1-deficient tissue mostly reflect changes in RNA abundance. Profiling over a time course of ribosome runoff in wildtype tissue reveals a wide range of ribosome translocation rates; on many mRNAs, the ribosomes are stalled. Sucrose gradient ultracentrifugation of hippocampal slices after ribosome runoff reveals that FMRP co-sediments with stalled ribosomes; and its loss results in decline of ribosome stalling on specific mRNAs. One such mRNA encodes SETD2, a lysine methyltransferase that catalyzes H3K36me3. ChIP-Seq demonstrates that loss of FMRP alters the deployment of this epigenetic mark on chromatin. H3K36me3 is associated with alternative pre-RNA processing, which we find occurs in an FMRP-dependent manner on transcripts linked to neural function and autism spectrum disorders.', 'Series_overall_design': 'Ribosome profiling of Transverse hippocampal or hippocampal-cortical brain slices  were acutely prepared from P28-35 C57BL/6N wild-type or Fmr1 KO male mice littermates using Illumina Nextseq.', 'Series_type': 'Other', 'Series_contributor': 'Joel,D,Richter', 'Series_sample_id': 'GSM4278106', 'Series_contact_name': 'Sneha,,Shah', 'Series_contact_email': 'sneha.19.6@gmail.com', 'Series_contact_laboratory': 'Joel Richter', 'Series_contact_department': 'Molecular Medicine', 'Series_contact_institute': 'UMASS CHAN MEDICAL SCHOOL', 'Series_contact_address': '373 Biotech II, Suite 204, Plantation Street', 'Series_contact_city': 'Worcester', 'Series_contact_state': 'Massachusetts', 'Series_contact_zip/postal_code': '01605', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE143nnn/GSE143330/suppl/GSE143330_steadystate.rpf.rna.rsem.gene.summary.tpm.txt.gz', 'Series_platform_id': 'GPL9185', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP240703'}\n",
      "344 PRJNA600267 GSE143390 {'Series_title': 'Virus-induced codon-specific reprogramming to favor viral RNA translation', 'Series_geo_accession': 'GSE143390', 'Series_status': 'Public on Jun 17 2022', 'Series_submission_date': 'Jan 09 2020', 'Series_last_update_date': 'Aug 24 2022', 'Series_pubmed_id': '35953468', 'Series_summary': 'How viruses, such as the emerging mosquito-borne Chikungunya virus (CHIKV), express their genomes at high levels despite an enrichment in suboptimal codons remains a puzzling question. By integrating subcellular fractionation and transcriptome-wide analyses of translation in CHIKV-infected human cells, we demonstrate an unanticipated virus-induced reprogramming of the host translation machinery to favor translation of viral RNA over cellular genes featuring optimal codon usage. This reprogramming was specifically apparent at the endoplasmic reticulum (ER), the preferred translation compartment of CHIKV RNA, and it is mediated by the wobble uridine 34 tRNA modification enzyme KIAA1456 whose expression is enhanced upon viral infection. Since KIAA1456 itself is encoded by a CHIKV-like codon usage, infection triggers a positive feed-back loop that ensures efficient virus protein production. Our findings demonstrate an unprecedented interplay of viruses with the host tRNA epitranscriptome to favor viral protein expression.', 'Series_overall_design': 'HEK 293T cells were subjected to subcellular fractionation after being infected with Chikungunya virus for 12 h (MOI=40), while control cells were mock infected for 12 h. Both subcellular compartments were then subjected to RNA-seq and Ribo-seq to identify differentially translated mRNAs. All experiments were performed at least in triplicates.', 'Series_type': 'Other', 'Series_contributor': 'Juana,,Díez', 'Series_sample_id': 'GSM4258244', 'Series_contact_name': 'Juana,,Díez', 'Series_contact_email': 'juana.diez@upf.edu', 'Series_contact_phone': '+34 933160862', 'Series_contact_laboratory': 'Molecular Virology Group', 'Series_contact_department': 'Department of Experimental and Health Sciences', 'Series_contact_institute': 'Universitat Pompeu Fabra', 'Series_contact_address': 'Carrer del Dr. Aiguader 88', 'Series_contact_city': 'Barcelona', 'Series_contact_state': 'Barcelona', 'Series_contact_zip/postal_code': '08003', 'Series_contact_country': 'Spain', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE143nnn/GSE143390/suppl/GSE143390_raw_counts_mRNA.tsv.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP240955'}\n",
      "345 PRJNA600412 GSE143415 {'Series_title': \"Co-existence of compulsory and plastic rRNA 2'-O-Methylation sites in human breast tumours\", 'Series_geo_accession': 'GSE143415', 'Series_status': 'Public on Dec 22 2020', 'Series_submission_date': 'Jan 10 2020', 'Series_last_update_date': 'Aug 03 2021', 'Series_pubmed_id': '34316693', 'Series_summary': 'Emerging evidence suggests that changes in ribosomal RNA 2’O-ribose methylation (rRNA 2’OMe) in the human ribosome play a key role in regulating translation thereby contributing in setting particular phenotypes such as hallmarks of cancer cells. Here, by analysing the rRNA 2’OMe in a large series of 195 human primary breast tumours using the RiboMETH-seq approach  we have identified the positions within the rRNA molecules which can tolerate the absence of 2’OMe in humans. The frequency of rRNA 2’OMe at some positions is highly stable between individuals (termed “stable”) whereas it can vary substantially at other positions (termed “variant”). Differential association of rRNA 2’OMe with breast cancer features, evolution and structure/function characteristics suggest the stable positions showing a low inter-variability might be required to maintain the core structure and thus the function of the ribosome – representing compulsory rRNA 2’OMe positions. In contrast, the variant positions displaying high inter-variability might be the source of regulatory events – representing plastic rRNA 2’OMe positions. Therefore our data issued from rRNA 2’OMe profiling of a large-scale human sample series bring the first compelling evidence to resolve two divergent long lasting visions of the role of rRNA 2’O-Me and sustain the emerging role of the ribosome plasticity in regulating translation in cancer.', 'Series_overall_design': \"rRNA 2'OMe profiling in a series of 195 primary breast tumours\", 'Series_type': 'Methylation profiling by high throughput sequencing', 'Series_contributor': 'Laurie,,Tonon', 'Series_sample_id': 'GSM4259012', 'Series_contact_name': 'Virginie,,MARCEL', 'Series_contact_email': 'virginie.marcel@lyon.unicancer.fr', 'Series_contact_institute': 'Cancer Research Center of Lyon', 'Series_contact_address': '28 rue Laennec', 'Series_contact_city': 'Lyon', 'Series_contact_zip/postal_code': '69373 cedex 08', 'Series_contact_country': 'France', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE143nnn/GSE143415/suppl/GSE143415_RAW.tar', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP241165'}\n",
      "346 PRJNA601135 GSE143623 {'Series_title': 'Deregulation of ribosomal protein expression and translation promotes breast cancer metastasis [ribo-Seq BRx142]', 'Series_geo_accession': 'GSE143623', 'Series_status': 'Public on Feb 07 2020', 'Series_submission_date': 'Jan 14 2020', 'Series_last_update_date': 'Feb 10 2020', 'Series_pubmed_id': '32029688', 'Series_summary': 'We conducted an in vivo genome-wide CRISPR activation screen to identify genes that accelerate distal metastasis by breast cancer patient-derived circulating tumor cells (CTCs) following direct intravascular inoculation in mice. Regulators of translation and ribosomal proteins were prominent among these, and expression of RPL15, a component of the large ribosome subunit, was sufficient to increase metastatic growth in multiple organs. RPL15 overexpression selectively increases translation of other ribosomal proteins and cell cycle regulators. Unsupervised analysis of single-cell RNA sequencing of freshly-isolated CTCs from breast cancer patients identifies a subset with strong ribosomal and protein translation signatures, correlated with increased proliferative markers, epithelial markers and poor clinical outcome. Thus, ribosome protein expression identifies an aggressive subset of CTCs, whose therapeutic targeting may suppress metastatic progression.', 'Series_overall_design': 'We performed ribosomal profiling in control and RPL15 overexpressing circulating tumor cell-derived cell lines to determine the global ribosome occupancy of mRNA transcripts, identify changes in translational preference, and calculate the translational efficiency for any given transcript (i.e., the ratio of ribosome-bound mRNA to all polyA-mRNA). One control specimen and one RPL15 overexpressing specimen was profiled in each of two batches.', 'Series_type': 'Other', 'Series_contributor': 'Douglas,S,Micalizzi', 'Series_sample_id': 'GSM4263929', 'Series_contact_name': 'Ben,S.,Wittner', 'Series_contact_email': 'wittner.ben@mgh.harvard.edu', 'Series_contact_laboratory': 'Lawrence', 'Series_contact_department': 'Center for Cancer Research', 'Series_contact_institute': 'Massachusetts General Hospital', 'Series_contact_address': '149 13th Street', 'Series_contact_city': 'Boston', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02129', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE143nnn/GSE143623/suppl/GSE143623_riboSeq_readCounts.xlsx', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP241899'}\n",
      "347 PRJNA601197 GSE143659 {'Series_title': 'p70 S6 Kinase 1-dependent alterations in cortical mRNA translation in fragile X syndrome mice', 'Series_geo_accession': 'GSE143659', 'Series_status': 'Public on Oct 31 2020', 'Series_submission_date': 'Jan 14 2020', 'Series_last_update_date': 'Oct 31 2020', 'Series_summary': 'Fragile X syndrome (FXS) is caused by loss of the fragile X mental retardation protein (FMRP). The mechanism by which FMRP regulates messenger RNA (mRNA) translation remains disputed. We observed reduced ribosome footprint abundance in the majority of differentially translated genes in cortices of FXS mice, which was correlated with an increased rate of ribosome translocation that was normalized by inhibition of p70 S6 Kinase 1 (S6K1). We also show that alterations in translation efficiencies across mRNAs in FXS mouse cortices exhibit a positive to negative gradation with coding sequence length, which is prevented by the genetic reduction of S6K1. Our findings reveal the identity of dysregulated mRNAs and a molecular mechanism by which reduction of S6K1 prevents altered translation in FXS.', 'Series_overall_design': 'Examination of genome-wide translational state in wild-type, Fmr1-deficient, Rps6kb1-deficient, and double knockout (Fmr1 and Rps6kb1 deficient) mouse cortices via next-generation sequencing-based profiling of transcription and translation.', 'Series_type': 'Other', 'Series_contributor': 'Eric,,Klann', 'Series_sample_id': 'GSM4271568', 'Series_contact_name': 'Eric,,Klann', 'Series_contact_institute': 'New York University', 'Series_contact_address': '4 Washington Pl', 'Series_contact_city': 'New York', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '10003', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE143nnn/GSE143659/suppl/GSE143659_te_fxsVwt_all.csv.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP241934'}\n",
      "348 PRJNA602407 GSE143975 {'Series_title': 'A PRC2-independent function for EZH2 in regulating rRNA 2′-O methylation and IRES-dependent translation', 'Series_geo_accession': 'GSE143975', 'Series_status': 'Public on Dec 10 2020', 'Series_submission_date': 'Jan 21 2020', 'Series_last_update_date': 'Apr 04 2021', 'Series_pubmed_id': '33795875', 'Series_summary': 'Overactivated ribosome biogenesis is a common feature of cancer. However, the underlying molecular mechanism remains elusive. Herein, we report enhancer of zeste homolog 2 (EZH2), a histone methyltransferase, could interact with fibrillarin (FBL) directly in the nucleolus. While loss of EZH2 does not impact FBL-mediated histone H2AQ104 methylation (H2AQ104me) and 18S rRNA processing, EZH2 is proved to be involved in rRNA 2′-O methylation in a manner dependent of its interaction with FBL. Mechanistically, EZH2 strengthens the fibrillarin (FBL)-NOP56 interaction by binding to both proteins and thus facilitates the assembly of box C/D small nucleolar ribonucleoprotein (box C/D snoRNP). Strikingly, EZH2 deficiency alters translational efficiency and reduces internal ribosome entry site (IRES) activity of key oncogenes in prostate cancer. In summary, our findings reveal a novel non-methyltransferase role of EZH2 that mediates FBL functions, which may provide more options for the development of EZH2-targeting curative strategies in cancer.', 'Series_overall_design': 'Examination of transcription and translational efficiency in replicative  analysis with RNA-seq and Ribo-seq with 2 replicates at shCTRL, shEZH2 and shFBL', 'Series_type': 'Other', 'Series_contributor': 'Qi,,Cao', 'Series_sample_id': 'GSM4826367', 'Series_contact_name': 'Yanqiang,,Li', 'Series_contact_laboratory': 'Kaifu Chen Lab', 'Series_contact_department': 'Cardiology Department', 'Series_contact_institute': \"Boston Children's Hospital\", 'Series_contact_address': '300 Longwood Ave.', 'Series_contact_city': 'Boston', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02115', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE143nnn/GSE143975/suppl/GSE143975_Table2.Ribo_seq_raw_counts.txt.gz', 'Series_platform_id': 'GPL23227', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP243638'}\n",
      "349 PRJNA602676 GSE144065 {'Series_title': 'The zinc finger domains in U2AF26 and U2AF35 have diverse functionalities including a role in controlling translation', 'Series_geo_accession': 'GSE144065', 'Series_status': 'Public on May 01 2020', 'Series_submission_date': 'Jan 22 2020', 'Series_last_update_date': 'May 17 2023', 'Series_pubmed_id': '32116123', 'Series_summary': 'Recent work has associated point mutations in both zinc fingers (ZnF) of the spliceosome component U2AF35 with malignant transformation. However, surprisingly little is known about the functionality of the U2AF35 ZnF domains in general. Here we have analyzed key functionalities of the ZnF domains of mammalian U2AF35 and its paralog U2AF26. Both ZnFs are required for splicing regulation, whereas only ZnF2 controls protein stability and contributes to the interaction with U2AF65. These features are confirmed in a naturally occurring splice variant of U2AF26 lacking ZnF2, that is strongly induced upon activation of primary mouse T cells and localized in the cytoplasm. Using Ribo-Seq in a model T cell line we provide evidence for a role of U2AF26 in activating cytoplasmic steps in gene expression, notably translation. Consistently, an MS2 tethering assay shows that cytoplasmic U2AF26/35 increase translation when localized to the 5’UTR of a model mRNA. This regulation is partially dependent on ZnF1 thus providing a connection between a core splicing factor, the ZnF domains and the regulation of translation. Altogether, our work reveals unexpected functions of U2AF26/35 and their ZnF domains, thereby contributing to a better understanding of their role and regulation in mammalian cells.', 'Series_overall_design': 'EL4 cells, either control or stabely overexpressing a U2af26 variant localizing to the cytoplasm, were investigated via ribosome profiling. In parallel total RNA was sequenced to normalize for changes in gene expression levels.', 'Series_type': 'Other', 'Series_contributor': 'Florian,,Heyd', 'Series_sample_id': 'GSM4279089', 'Series_contact_name': 'Marco,,Preussner', 'Series_contact_email': 'mpreussner@zedat.fu-berlin.de', 'Series_contact_laboratory': 'Heyd', 'Series_contact_institute': 'FU berlin', 'Series_contact_address': 'Takustrasse 6', 'Series_contact_city': 'Berlin', 'Series_contact_zip/postal_code': '14195', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE144nnn/GSE144065/suppl/GSE144065_RAW.tar', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP243905'}\n",
      "350 PRJNA602876 GSE144140 {'Series_title': 'Use of NanoLuc-Based Reporters that Monitor Nonsense Suppression and Inhibition of Nonsense-Mediated mRNA Decay to Identify Potent New Agents That Partially Reverse the Effects of Nonsense Mutations', 'Series_geo_accession': 'GSE144140', 'Series_status': 'Public on Dec 01 2020', 'Series_submission_date': 'Jan 23 2020', 'Series_last_update_date': 'Jul 19 2021', 'Series_pubmed_id': '34272367', 'Series_summary': 'Premature termination codons (PTCs) lead to loss of protein function by 1) ending mRNA translation before a full-length protein is made, and 2) by triggering nonsense-mediated mRNA decay (NMD), a pathway that targets a PTC-containing mRNA for degradation. Nonsense suppression therapy aims to restore protein function by suppressing termination at PTCs (called readthrough, or RT). However, the efficiency of current readthrough agents is generally quite low. We reasoned that by stimulating both readthrough and increasing mRNA levels, greater protein function can be restored than with readthrough alone. In this study, we developed a series of novel NanoLuc reporters designed to identify and differentiate compounds that induce readthrough, enhance mRNA abundance (by NMD inhibition or other mechanisms), or target both mechanisms simultaneously (referred to as dual RT/NMD reporters). We show that treatments that promote readthrough and NMD inhibition led to synergistic increases in NanoLuc activity of the RT/NMD reporter over either condition alone. We then used a dual RT/NMD reporter to carry out a high throughput screen of a library of 771,345 compounds. We identified 180 compounds with the ability to increase NanoLuc activity in our dual RT/NMD reporter assay. To confirm that such synergy can also amplify protein function in a disease model, we treated cells expressing a CFTR-Y122X mini-intron construct with one hit from this screen, SRI-37240. We found that this compound produced a much larger increase in CFTR activity compared to either the RT compound G418 +/- the NMD inhibitor amlexanox, and the activity obtained could be further enhanced with CFTR modulators. These results confirm the utility of these new reporters to identify many potent new molecules that counteract the effects of nonsense mutations.', 'Series_overall_design': '9 ribosome profiling samples are included from HEK293T cells', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Rachel,,Green', 'Series_sample_id': 'GSM4282040', 'Series_contact_name': 'Rachel,,Green', 'Series_contact_email': 'ragreen@jhmi.edu', 'Series_contact_laboratory': 'Rachel Green', 'Series_contact_department': 'Molecular Biology and Genetics', 'Series_contact_institute': 'Johns Hopkins University School of Medicine', 'Series_contact_address': '725 N Wolfe St', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21205', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE144nnn/GSE144140/suppl/GSE144140_RAW.tar', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP244347'}\n",
      "351 PRJNA602917 GSE144163 {'Series_title': 'Ribosome profiling of ABCE1-knockout cells', 'Series_geo_accession': 'GSE144163', 'Series_status': 'Public on Jul 14 2020', 'Series_submission_date': 'Jan 23 2020', 'Series_last_update_date': 'Jul 17 2020', 'Series_pubmed_id': '32668236', 'Series_summary': 'Nonsense-mediated decay (NMD) is a pathway that degrades messenger RNAs containing premature termination codons. Here, a genome-wide screen for NMD factors uncovered an unexpected mechanism that broadly governs 3ʹ untranslated region (UTR)-directed regulation. The screen revealed that NMD requires lysosomal acidification, which allows transferrin-mediated iron uptake, which in turn is necessary for iron-sulfur (Fe-S) cluster biogenesis. This pathway converges on the Fe-S cluster-containing ribosome recycling factor ABCE1, whose impaired function results in the movement of ribosomes into 3ʹ UTRs where they displace exon junction complexes, thereby abrogating NMD. Importantly, these effects extend beyond NMD substrates, with ABCE1 activity required to maintain the accessibility of 3ʹ UTRs to diverse regulators, including microRNAs and RNA binding proteins. Due to the sensitivity of the Fe-S cluster of ABCE1 to iron availability and reactive oxygen species, these findings reveal an unanticipated vulnerability of 3ʹ UTR-directed regulation to lysosomal dysfunction, iron deficiency, and oxidative stress.', 'Series_overall_design': 'Method: Ribosome profiling was performed in HCT116 cells that received, via lentivirus, Cas9 and sgRNAs targeting either ABCE1 or non-targeting control sgRNAs. Two sgRNAs were used for each condition, and thus 4 biological samples were used for these experiments.', 'Series_type': 'Other', 'Series_contributor': 'Joshua,T,Mendell', 'Series_sample_id': 'GSM4282355', 'Series_contact_name': 'Joshua,,Mendell', 'Series_contact_email': 'joshua.mendell@utsouthwestern.edu', 'Series_contact_department': 'Molecular Biology', 'Series_contact_institute': 'UT Southwestern Medical Center', 'Series_contact_address': '6000 Harry Hines Blvd., NA6.200A', 'Series_contact_city': 'Dallas', 'Series_contact_state': 'TX', 'Series_contact_zip/postal_code': '75390-9148', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE144nnn/GSE144163/suppl/GSE144163_RibosomeProtectedFragment_Counts.xlsx', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP244436'}\n",
      "352 PRJNA603124 GSE144250 {'Series_title': 'The Ccr4-Not complex monitors the translating ribosome for codon optimality', 'Series_geo_accession': 'GSE144250', 'Series_status': 'Public on Apr 16 2020', 'Series_submission_date': 'Jan 24 2020', 'Series_last_update_date': 'Jun 10 2020', 'Series_pubmed_id': '32299921', 'Series_summary': 'Control of messenger RNA (mRNA) decay rate is intimately connected to translation elongation, but the spatial coordination of these events is poorly understood. The Ccr4-Not complex initiates mRNA decay through deadenylation and activation of decapping. We used a combination of cryo–electron microscopy, ribosome profiling, and mRNA stability assays to examine the recruitment of Ccr4-Not to the ribosome via specific interaction of the Not5 subunit with the ribosomal E-site in Saccharomyces cerevisiae. This interaction occurred when the ribosome lacked accommodated A-site transfer RNA, indicative of low codon optimality. Loss of the interaction resulted in the inability of the mRNA degradation machinery to sense codon optimality. Our findings elucidate a physical link between the Ccr4-Not complex and the ribosome and provide mechanistic insight into the coupling of decoding efficiency with mRNA stability.', 'Series_overall_design': '2 replicates in 2 different strains.', 'Series_type': 'Other', 'Series_contributor': 'Roland,,Beckmann', 'Series_sample_id': 'GSM4284449', 'Series_contact_name': 'Yoshitaka,,Matsuo', 'Series_contact_department': 'IMSUT', 'Series_contact_institute': 'The University of Tokyo,', 'Series_contact_address': '4-6-1, Shirokanedai, Minato-Ku', 'Series_contact_city': 'Tokyo', 'Series_contact_zip/postal_code': '108-8639', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE144nnn/GSE144250/suppl/GSE144250_RAW.tar', 'Series_platform_id': 'GPL21656', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP244728'}\n",
      "353 PRJNA603355 GSE144313 {'Series_title': 'RNA-seq and Ribo-seq of WT and METTL5 KO mECS.', 'Series_geo_accession': 'GSE144313', 'Series_status': 'Public on Apr 15 2020', 'Series_submission_date': 'Jan 27 2020', 'Series_last_update_date': 'Jul 15 2020', 'Series_pubmed_id': '32217665', 'Series_summary': 'Covalent chemical modifications of cellular RNAs directly impact all biological processes. However, our mechanistic understanding of the enzymes catalysing these modifications, their substrates and biological functions remains vague.\\xa0Here, we undertook a systematic screen to uncover new RNA methyltransferases. We demonstrate that the methyltransferase-like 5 (METTL5) protein catalyses m6A in\\xa018S\\xa0rRNA at position A1832. We report that absence of\\xa0Mettl5\\xa0in\\xa0mouse embryonic stem cells (mESCs) results in a changes gene expression, decrease in translation rate, spontaneous loss of pluripotency and compromised differentiation potential. Mice lacking METTL5 recapitulate symptoms of patients with METTL5 mutations, thereby providing a new mouse disease model. Overall, our work\\xa0\\xa0highlights the importance of m6A in rRNA in stemness, differentiation, development and diseases.', 'Series_overall_design': 'RNA and Riob-seq of WT and METTL5 KO mESC in biological replicates.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Robert,,Schneider', 'Series_sample_id': 'GSM4285687', 'Series_contact_name': 'Oliver,,Rando', 'Series_contact_email': 'oliver.rando@umassmed.edu', 'Series_contact_department': 'BMB', 'Series_contact_institute': 'University of Massachusetts', 'Series_contact_address': '364 Plantation St', 'Series_contact_city': 'Worcester', 'Series_contact_state': 'Massachusetts', 'Series_contact_zip/postal_code': '01605', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE144nnn/GSE144313/suppl/GSE144313_RAW.tar', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP245406'}\n",
      "354 PRJNA603987 GSE144539 {'Series_title': 'Ribosome profiling and RNA-seq of Tsc2+/- hippocampal slices', 'Series_geo_accession': 'GSE144539', 'Series_status': 'Public on Oct 05 2020', 'Series_submission_date': 'Jan 30 2020', 'Series_last_update_date': 'Jan 04 2021', 'Series_pubmed_id': '33054857', 'Series_summary': 'Tuberous sclerosis (TSC) is caused by loss of function in Tsc2 and haploinsufficiency of the protein product tuberin.', 'Series_overall_design': 'Ribosome profiling and RNA-seq of hippocampal slices from WT and littermate Tsc2+/- mice before and after DHPG-mGluR stimulation', 'Series_type': 'Other', 'Series_contributor': 'Joel,D,Richter', 'Series_sample_id': 'GSM4812194', 'Series_contact_name': 'Annie,,Hien', 'Series_contact_email': 'annie.hien@umassmed.edu', 'Series_contact_laboratory': 'Richter Lab', 'Series_contact_department': 'Molecular Medicine', 'Series_contact_institute': 'UMass Medical School Worcester', 'Series_contact_address': '373 Plantation Street', 'Series_contact_city': 'Worcester', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '01605', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE144nnn/GSE144539/suppl/GSE144539_ca1.rep1-3.rna.rsem.gene.summary.counts.txt.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP246149'}\n",
      "355 PRJNA604580 GSE144682 {'Series_title': 'The role of long noncoding RNAs during pancreas development', 'Series_geo_accession': 'GSE144682', 'Series_status': 'Public on Aug 03 2020', 'Series_submission_date': 'Feb 03 2020', 'Series_last_update_date': 'Aug 24 2020', 'Series_pubmed_id': '32744504', 'Series_summary': \"LncRNAs are developmentally regulated and highly cell type-specific non-coding RNAs that have emerged as important regulators of cell fate commitment and maintenance. In this study, we dissected the role of lncRNAs in human pancreas development by classifying lncRNAs based on their dynamic regulation, subcellular localization, and engagement with ribosomes during the stepwise differentiation of human embryonic stem cells (hESCs) towards pancreatic fate. We then deleted 10 candidate lncRNAs in hESCs and characterized the knockout phenotypes of pancreatic developmental intermediates, prioritizing dynamically regulated lncRNAs with validated translation potential and proximity to developmental TFs. This small-scale loss-of-function screen revealed that most lncRNAs are dispensable for pancreatic development and the regulation of nearby genes. We identify LINC00261 as the first translated lncRNA involved in human endocrine cell development, and show that it regulates gene expression in pancreatic progenitor cells in trans rather than cis. Through systematic dissection of LINC00261's coding and noncoding functions, we can exclude a role for the produced micropeptides in this process. Instead, we posit that, over the course of pancreatic differentiation, the biological activity of multiple lncRNAs is controlled by a regulatory, translation machinery dependent mechanism that employs ribosome engagement and short open reading frame translation to dose lncRNA activity in the nucleus. It is conceivable that inadequate nuclear LINC00261 dosage during human pancreas development could predispose individuals to developing diabetes.\", 'Series_overall_design': 'RNA-seq and Ribo-seq analysis of H1 embryonic stem cells differentiated towards the pancreatic endocrine cell fate.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Maike,,Sander', 'Series_sample_id': 'GSM4293642', 'Series_contact_name': 'Maike,,Sander', 'Series_contact_email': 'masander@ucsd.edu', 'Series_contact_institute': 'UC San Diego', 'Series_contact_address': '2880 Torrey Pines Scenic Dr.', 'Series_contact_city': 'La Jolla', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '92037', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE144nnn/GSE144682/suppl/GSE144682_pp2_fractionation_rpkm.txt.gz', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP246901'}\n",
      "356 PRJNA605114 GSE144841 {'Series_title': '4EHP and GIGYF1/2 induce translation-coupled messenger RNA decay', 'Series_geo_accession': 'GSE144841', 'Series_status': 'Public on Oct 23 2020', 'Series_submission_date': 'Feb 06 2020', 'Series_last_update_date': 'Jan 22 2021', 'Series_pubmed_id': '33053355', 'Series_summary': 'Although canonical mRNA degradation is generally recognized to be translation dependent, our understanding of the molecular events that coordinate ribosome movement with the decay machinery is still limited. Here, we show that the 4EHP–GIGYF1/2 complex triggers co-translational mRNA decay as a result of altered ribosome activity during elongation. Human cells lacking 4EHP and GIGYF1 and 2 proteins accumulate transcripts known to be degraded in a translation dependent manner or with prominent ribosome pausing. These include mRNAs encoding secretory and membrane-bound proteins or specific tubulin isotypes, among others. 4EHP–GIGYF1/2 fails to reduce target mRNA levels in the absence of ribosome stalling or upon disruption of its interaction with the cap structure, DDX6 and a GYF domain-associated protein. Our studies reveal how a repressor complex linked to neurological disorders minimizes the protein output of a subset of mRNAs.', 'Series_overall_design': 'Three cell types with two biological replicates analysed using RNA-Seq and Ribo-Seq', 'Series_type': 'Other', 'Series_contributor': 'Cátia,,Igreja', 'Series_sample_id': 'GSM4299136', 'Series_contact_name': 'Ramona,,Weber', 'Series_contact_email': 'ramona.weber@uzh.ch', 'Series_contact_department': 'Institute for Regenerative Medicine', 'Series_contact_institute': 'University of Zurich', 'Series_contact_address': 'Wagistrasse 12', 'Series_contact_city': 'Schlieren', 'Series_contact_zip/postal_code': '8952', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE144nnn/GSE144841/suppl/GSE144841_readcounts_Ribo.txt.gz', 'Series_platform_id': 'GPL21290', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP247494'}\n",
      "357 PRJNA605444 GSE143393 {'Series_title': 'Male-specific lymphomagenesis by serial inverse dysregulation of the RNA helicases DDX3X and DDX3Y.', 'Series_geo_accession': 'GSE143393', 'Series_status': 'Public on Sep 01 2021', 'Series_submission_date': 'Jan 09 2020', 'Series_last_update_date': 'Dec 02 2021', 'Series_pubmed_id': '34437837', 'Series_summary': 'DDX3X is a ubiquitously expressed RNA helicase involved in multiple stages of RNA biogenesis. DDX3X is frequently mutated in Burkitt lymphoma but the functional basis for this is unknown. Here, we show that loss-of-function DDX3X mutations are also commonly found in MYC-translocated diffuse large B cell lymphoma and reveal functional co-operation between mutant DDX3X and MYC. We show that DDX3X promotes translation of mRNAs encoding components of the core translational machinery, thereby driving global protein synthesis. Loss-of-function DDX3X mutations moderate MYC-driven global protein synthesis, thereby buffering MYC-induced proteotoxic stress during early lymphomagenesis. Established lymphoma cells subsequently restore full protein synthetic capacity by ectopic expression of DDX3Y, a Y-chromosome homologue that is normally expressed exclusively in testis. These findings highlight the vulnerability of MYC-driven lymphoma to proteotoxic stress and identify an unexpected male-specific mechanism of carcinogenesis, namely the commandeering of a testis-specific Y-chromosome gene to drive full malignant transformation.', 'Series_overall_design': 'Ribo-Seq and RNA-Seq sequencing data in 2 lymphoma cell lines Mutu and U2392 following DDX3X shRNA knockdown at 2 time points: 24h and 48h; RNA-Seq sequencing data comparing DDX3X R475C knockin versus control edited cell line clones', 'Series_type': 'Other', 'Series_contributor': 'Daniel,,Hodson', 'Series_sample_id': 'GSM4258320', 'Series_contact_name': 'Daniel,J,Hodson', 'Series_contact_department': 'Department of Haematology', 'Series_contact_institute': 'Wellcome Trust - Medical Research Council Cambridge Stem Cell Institute, University of Cambridge', 'Series_contact_address': 'The Jeffrey Cheah Biomedical Centre, Cambridge Biomedical Campus', 'Series_contact_city': 'Cambridge', 'Series_contact_zip/postal_code': 'CB2 0AW', 'Series_contact_country': 'United Kingdom', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE143nnn/GSE143393/suppl/GSE143393_CDScounts.csv.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP247659'}\n",
      "358 PRJNA605655 GSE144997 {'Series_title': 'Determining ribosome-associated mRNA in motor neurons in adult GarsC201R/+ mice [16NGS-002-burgess]', 'Series_geo_accession': 'GSE144997', 'Series_status': 'Public on Feb 11 2020', 'Series_submission_date': 'Feb 09 2020', 'Series_last_update_date': 'Feb 13 2020', 'Series_summary': 'How mutations in broadly expressed housekeeping genes lead to neurodegeneration in specific cell types remains unclear. Mutations in ubiquitously expressed tRNA synthetase genes cause axonal peripheral neuropathy, accounting for at least six forms of Charcot-Marie-Tooth disease. Genetic evidence in mouse and Drosophila models suggests a neomorphic gain-of-function mechanism. Here, we use in vivo, cell-type-specific transcriptional and translational profiling of affected peripheral neurons to show that mutant tRNA synthetases impair translation and activate the integrated stress response (ISR) through the sensor kinase, GCN2. The chronic activation of the ISR contributes to the pathophysiology, and genetic deletion of Gcn2 alleviates the peripheral neuropathy. The activation of GCN2 by tRNA synthetase mutations indicates their neomorphic activity is still related to translation and suggests inhibiting GCN2 or the ISR as a therapeutic strategy.', 'Series_overall_design': 'RiboTagging was performed on spinal cords from HARPL22;ChAT-Cre;Gars+/+ or GarsC201R/+ and corresponding negative controls (no Cre) and differential expression analyis was run.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Robert,W,Burgess', 'Series_sample_id': 'GSM4304061', 'Series_contact_name': 'Abigail,,Tadenev', 'Series_contact_laboratory': 'Dr. Robert Burgess', 'Series_contact_institute': 'The Jackson Laboratory', 'Series_contact_address': '600 Main St', 'Series_contact_city': 'Bar Harbor', 'Series_contact_state': 'ME', 'Series_contact_zip/postal_code': '04609', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE144nnn/GSE144997/suppl/GSE144997_RAW.tar', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP247865'}\n",
      "359 PRJNA605656 GSE144996 {'Series_title': 'Determining ribosome-associated mRNA in motor neurons after nerve crush [15NGS-001-rwb]', 'Series_geo_accession': 'GSE144996', 'Series_status': 'Public on Feb 11 2020', 'Series_submission_date': 'Feb 09 2020', 'Series_last_update_date': 'Feb 13 2020', 'Series_summary': 'How mutations in broadly expressed housekeeping genes lead to neurodegeneration in specific cell types remains unclear. Mutations in ubiquitously expressed tRNA synthetase genes cause axonal peripheral neuropathy, accounting for at least six forms of Charcot-Marie-Tooth disease. Genetic evidence in mouse and Drosophila models suggests a neomorphic gain-of-function mechanism. Here, we use in vivo, cell-type-specific transcriptional and translational profiling of affected peripheral neurons to show that mutant tRNA synthetases impair translation and activate the integrated stress response (ISR) through the sensor kinase, GCN2. The chronic activation of the ISR contributes to the pathophysiology, and genetic deletion of Gcn2 alleviates the peripheral neuropathy. The activation of GCN2 by tRNA synthetase mutations indicates their neomorphic activity is still related to translation and suggests inhibiting GCN2 or the ISR as a therapeutic strategy.', 'Series_overall_design': 'RiboTagging was performed on spinal cords from HARPL22;ChAT-Cre;Gars+/+ crushed or uncrushed and corresponding negative controls (no Cre) and differential expression analyis was run.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Robert,W,Burgess', 'Series_sample_id': 'GSM4304037', 'Series_contact_name': 'Abigail,,Tadenev', 'Series_contact_laboratory': 'Dr. Robert Burgess', 'Series_contact_institute': 'The Jackson Laboratory', 'Series_contact_address': '600 Main St', 'Series_contact_city': 'Bar Harbor', 'Series_contact_state': 'ME', 'Series_contact_zip/postal_code': '04609', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE144nnn/GSE144996/suppl/GSE144996_RAW.tar', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP247864'}\n",
      "360 PRJNA605660 GSE145002 {'Series_title': 'Determining ribosome-associated mRNA in motor neurons in pre-disease onset GarsC201R mice', 'Series_geo_accession': 'GSE145002', 'Series_status': 'Public on Feb 11 2020', 'Series_submission_date': 'Feb 09 2020', 'Series_last_update_date': 'Feb 13 2020', 'Series_summary': 'How mutations in broadly expressed housekeeping genes lead to neurodegeneration in specific cell types remains unclear. Mutations in ubiquitously expressed tRNA synthetase genes cause axonal peripheral neuropathy, accounting for at least six forms of Charcot-Marie-Tooth disease. Genetic evidence in mouse and Drosophila models suggests a neomorphic gain-of-function mechanism. Here, we use in vivo, cell-type-specific transcriptional and translational profiling of affected peripheral neurons to show that mutant tRNA synthetases impair translation and activate the integrated stress response (ISR) through the sensor kinase, GCN2. The chronic activation of the ISR contributes to the pathophysiology, and genetic deletion of Gcn2 alleviates the peripheral neuropathy. The activation of GCN2 by tRNA synthetase mutations indicates their neomorphic activity is still related to translation and suggests inhibiting GCN2 or the ISR as a therapeutic strategy.', 'Series_overall_design': 'RiboTagging was performed on spinal cords from 2 week old HARPL22;ChAT-Cre;Gars+/+ or GarsC201R/+ and corresponding negative controls (no Cre) and differential expression analyis was run.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Robert,W,Burgess', 'Series_sample_id': 'GSM4304138', 'Series_contact_name': 'Abigail,,Tadenev', 'Series_contact_laboratory': 'Dr. Robert Burgess', 'Series_contact_institute': 'The Jackson Laboratory', 'Series_contact_address': '600 Main St', 'Series_contact_city': 'Bar Harbor', 'Series_contact_state': 'ME', 'Series_contact_zip/postal_code': '04609', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE145nnn/GSE145002/suppl/GSE145002_RAW.tar', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP247870'}\n",
      "361 PRJNA605662 GSE145000 {'Series_title': 'Determining ribosome-associated mRNA in motor neurons in adult GarsP278KY/+ mice [17-burgess-004]', 'Series_geo_accession': 'GSE145000', 'Series_status': 'Public on Feb 11 2020', 'Series_submission_date': 'Feb 09 2020', 'Series_last_update_date': 'Feb 13 2020', 'Series_summary': 'How mutations in broadly expressed housekeeping genes lead to neurodegeneration in specific cell types remains unclear. Mutations in ubiquitously expressed tRNA synthetase genes cause axonal peripheral neuropathy, accounting for at least six forms of Charcot-Marie-Tooth disease. Genetic evidence in mouse and Drosophila models suggests a neomorphic gain-of-function mechanism. Here, we use in vivo, cell-type-specific transcriptional and translational profiling of affected peripheral neurons to show that mutant tRNA synthetases impair translation and activate the integrated stress response (ISR) through the sensor kinase, GCN2. The chronic activation of the ISR contributes to the pathophysiology, and genetic deletion of Gcn2 alleviates the peripheral neuropathy. The activation of GCN2 by tRNA synthetase mutations indicates their neomorphic activity is still related to translation and suggests inhibiting GCN2 or the ISR as a therapeutic strategy.', 'Series_overall_design': 'RiboTagging was performed on spinal cords from HARPL22;ChAT-Cre;Gars+/+ or GarsP278KY/+ and corresponding negative controls (no Cre) and differential expression analyis was run.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Robert,W,Burgess', 'Series_sample_id': 'GSM4304103', 'Series_contact_name': 'Abigail,,Tadenev', 'Series_contact_laboratory': 'Dr. Robert Burgess', 'Series_contact_institute': 'The Jackson Laboratory', 'Series_contact_address': '600 Main St', 'Series_contact_city': 'Bar Harbor', 'Series_contact_state': 'ME', 'Series_contact_zip/postal_code': '04609', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE145nnn/GSE145000/suppl/GSE145000_RAW.tar', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP247868'}\n",
      "362 PRJNA605796 GSE145046 {'Series_title': 'Decoding mRNA translatability and stability from 5’ UTR', 'Series_geo_accession': 'GSE145046', 'Series_status': 'Public on Jun 01 2020', 'Series_submission_date': 'Feb 10 2020', 'Series_last_update_date': 'Aug 11 2020', 'Series_pubmed_id': '32719458', 'Series_summary': 'Precise control of protein synthesis by engineering sequence elements in 5’ untranslated region (5’UTR) remains a fundamental challenge. To accelerate our understanding of cis-regulatory code embedded in 5’UTR, we devised massively parallel reporter assays from a synthetic mRNA library composed of over one million 5’UTR variants. A completely randomized 10-nucleotide sequence preceding an upstream open reading frame (uORF) and downstream GFP leads to a broad range of mRNA translatability and stability in mammalian cells. While efficient translation protects mRNA from degradation, uORF translation triggers mRNA decay in a UPF1-dependent manner. We also identified translational inhibitory elements in 5’UTR with G-quadruplex as a mark for mRNA decay in the P-body. Unexpectedly, an unstructured A-rich element in 5’UTR, while enabling cap-independent translation, destabilizes mRNAs in the absence of translation. Our results not only expose diverse sequence features of 5’UTR in controlling mRNA translatability, but also reveal ribosome-dependent and -independent mRNA surveillance pathways.', 'Series_overall_design': 'To accelerate our understanding of cis-regulatory code embedded in 5’UTR of mRNA, we devised a high-throughput uORF reporter assay from a synthetic mRNA library composed of over one million 5’UTR variants. This reporter assay is compatible with fluorescence-activated cell sorting and polysome profiling.', 'Series_type': 'Other', 'Series_contributor': 'Shu-Bing,,Qian', 'Series_sample_id': 'GSM4546416', 'Series_contact_name': 'Yuanhui,,Mao', 'Series_contact_email': 'maoyuanhui123@gmail.com', 'Series_contact_institute': 'Zhejiang University', 'Series_contact_address': '1369 West Wenyi Road', 'Series_contact_city': 'Hangzhou', 'Series_contact_state': 'Zhejiang', 'Series_contact_zip/postal_code': '310000', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE145nnn/GSE145046/suppl/GSE145046_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP247978'}\n",
      "363 PRJNA608705 GSE145904 {'Series_title': '40S ribosome profiling reveals distinct roles for Tma20/Tma22 (MCT-1/DENR) and Tma64 (eIF2D) in 40S subunit recycling', 'Series_geo_accession': 'GSE145904', 'Series_status': 'Public on Mar 01 2021', 'Series_submission_date': 'Feb 25 2020', 'Series_last_update_date': 'May 28 2021', 'Series_pubmed_id': '34016977', 'Series_summary': 'The recycling of ribosomes at stop codons for use in further rounds of translation is critical for efficient protein synthesis. Removal of the 60S subunit is catalyzed by the ATPase Rli1 (ABCE1) while removal of the 40S is thought to require Tma64 (eIF2D), Tma20 (MCT-1), and Tma22 (DENR). However, it remains unclear how these Tma proteins cause 40S removal and control reinitiation of downstream translation. Here we developed an enhanced 40S ribosome footprinting strategy to directly observe intermediate steps of ribosome recycling in cells. Deletion of the genes encoding these Tma proteins resulted in broad accumulation of unrecycled 40S subunits at stop codons, directly establishing their role in 40S recycling. Furthermore, the Tma20/Tma22 heterodimer was responsible for a majority of 40S recycling events while Tma64 played a minor role. Introduction of an autism-associated mutation into TMA22 resulted in a loss of 40S recycling activity, linking ribosome recycling and neurological disease.', 'Series_overall_design': '**Please note that wig files composed of combination of two biological replicates (e.g. 26-45end5_DY134Ffx182Ffx_*.wig) are linked to the sample with the lower number (e.g. DY134Ffx sample for 26-45end5_DY134Ffx182Ffx_*.wig) as indicated in the corresponding sample description field.', 'Series_type': 'Other', 'Series_contributor': 'Nicholas,R,Guydosh', 'Series_sample_id': 'GSM4339114', 'Series_contact_name': 'Nicholas,R,Guydosh', 'Series_contact_email': 'nicholas.guydosh@nih.gov', 'Series_contact_laboratory': 'LBG', 'Series_contact_department': 'NIDDK', 'Series_contact_institute': 'National Institutes of Health', 'Series_contact_address': '8 Center Dr, Bldg 8, 220', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '20892', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE145nnn/GSE145904/suppl/GSE145904_RAW.tar', 'Series_platform_id': 'GPL23014', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP250700'}\n",
      "364 PRJNA608916 GSE145986 {'Series_title': 'Microbes modulate sympathetic neurons via a gut-brain circuit', 'Series_geo_accession': 'GSE145986', 'Series_status': 'Public on Mar 20 2020', 'Series_submission_date': 'Feb 26 2020', 'Series_last_update_date': 'Aug 30 2020', 'Series_pubmed_id': '32641826', 'Series_summary': 'Gut-brain connections monitor the intestinal tissue and its microbial and dietary content1, regulating both intestinal physiological functions such as nutrient absorption and motility2,3, and brain–wired feeding behaviour2. It is therefore plausible that circuits exist to detect gut microbes and relay this information to central nervous system (CNS) areas that, in turn, regulate gut physiology4. We characterized the influence of the microbiota on enteric–associated neurons (EAN) by combining gnotobiotic mouse models with transcriptomics, circuit–tracing methods, and functional manipulation. We found that the gut microbiome modulates gut–extrinsic sympathetic neurons; while microbiota depletion led to increased cFos expression, colonization of germ-free mice with short-chain fatty acid–producing bacteria suppressed cFos expression in the gut sympathetic ganglia. Chemogenetic manipulations, translational profiling, and anterograde tracing identified a subset of distal intestine-projecting vagal neurons positioned to play an afferent role in microbiota–mediated modulation of gut sympathetic neurons. Retrograde polysynaptic neuronal tracing from the intestinal wall identified brainstem sensory nuclei activated during microbial depletion, as well as efferent sympathetic premotor glutamatergic neurons that regulate gastrointestinal transit. These results reveal microbiota–dependent control of gut extrinsic sympathetic activation through a gut-brain circuit.', 'Series_overall_design': '16S RNA profiles, from cecal samples of C57BL6/J mice given streptomycin or PBS, were prepared for deep sequencing on an Illumina MiSeq.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Daniel,,Mucida', 'Series_sample_id': 'GSM4340774', 'Series_contact_name': 'Paul,Andrew,Muller', 'Series_contact_laboratory': 'Laboratory of Mucosal Immunology', 'Series_contact_institute': 'The Rockefeller University', 'Series_contact_address': '1230 York Avenue', 'Series_contact_city': 'New York', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '10065', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE145nnn/GSE145986/suppl/GSE145986_RAW.tar', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP250834'}\n",
      "365 PRJNA609216 GSE146059 {'Series_title': 'Germline inherited small RNAs facilitate the clearance of untranslated maternal mRNAs in C. elegans embryos (Ribo-Seq)', 'Series_geo_accession': 'GSE146059', 'Series_status': 'Public on Jan 13 2021', 'Series_submission_date': 'Feb 27 2020', 'Series_last_update_date': 'Apr 06 2021', 'Series_pubmed_id': '33664268', 'Series_summary': 'Inheritance and clearance of maternal mRNAs are two of the most critical events required for animal early embryonic development. However, the mechanisms regulating this process are still largely unknown. Here, we show that together with maternal mRNAs, C. elegans embryos inherit a complementary pool of small non-coding RNAs that facilitate the cleavage and removal of hundreds of maternal mRNAs. These antisense small RNAs are loaded into the maternal catalytically-active Argonaute CSR-1 and cleave complementary mRNAs no longer engaged in translation in somatic blastomeres. Induced depletion of CSR-1 specifically during embryonic development leads to embryonic lethality in a slicer-dependent manner and impairs the degradation of CSR-1 embryonic mRNA targets. Given the conservation of Argonaute catalytic activity, we propose that a similar mechanism operates to clear maternal mRNAs during the maternal-to-zygotic transition across species.', 'Series_overall_design': \"7 samples were analyzed by Ribo-seq, including replicates (sample name contains 'rep').\", 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Germano,,Cecere', 'Series_sample_id': 'GSM4363859', 'Series_contact_name': 'Germano,,Cecere', 'Series_contact_email': 'germano.cecere@pasteur.fr', 'Series_contact_phone': '0033140613225', 'Series_contact_laboratory': 'Mechanisms of Epigenetic Inheritance', 'Series_contact_department': 'Development and stem cell biology', 'Series_contact_institute': 'Institut Pasteur', 'Series_contact_address': 'Institut Pasteur, 28 Rue Du Docteur Roux, Batiment Monod, 4eme Etage', 'Series_contact_city': 'Paris', 'Series_contact_zip/postal_code': '75724', 'Series_contact_country': 'France', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE146nnn/GSE146059/suppl/GSE146059_RAW.tar', 'Series_platform_id': 'GPL19757', 'Series_platform_taxid': '6239', 'Series_sample_taxid': '6239', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP251101'}\n",
      "366 PRJNA611609 GSE146674 {'Series_title': 'Mutant huntingtin stalls ribosomes and represses protein synthesis in a cellular model of Huntington disease [Ribo-seq]', 'Series_geo_accession': 'GSE146674', 'Series_status': 'Public on Jul 22 2020', 'Series_submission_date': 'Mar 09 2020', 'Series_last_update_date': 'Apr 06 2021', 'Series_pubmed_id': '33674575', 'Series_summary': 'The polyglutamine expansion of huntingtin (mHtt) causes Huntington disease (HD) and neurodegeneration, but the mechanisms remain unclear. Here, we found that mHtt promotes ribosome stalling and suppresses protein synthesis. Depletion of mHtt enhances protein synthesis and increases the speed of ribosome translocation, while mHtt directly inhibits protein synthesis in vitro. We found interactions of ribosomal proteins and translating ribosomes with mHtt. High-resolution global ribosome footprint profiling (Ribo-Seq) and mRNA-Seq indicated a widespread shift in ribosome occupancy toward the 5’ and 3’ end and unique single-codon pauses on selected mRNA targets in HD cells, compared to controls. Thus, mHtt impedes ribosomal translocation during translation by promoting ribosome stalling, a novel mechanistic defect that can be exploited for HD therapeutics.', 'Series_overall_design': 'For ribosome footprinting, three independent rounds of cultured stratial cells (i.e. control, HD-heterozygous, and HD-homozygous cells) were used. To resolve ribosome-associated footprints, the lysates were treated with RNase T1 and RNase A for 30 min, separated on 10-50% sucrose gradients and RNA from the 80S monosome fractions were purified.', 'Series_type': 'Other', 'Series_contributor': 'Audrey,,Michel', 'Series_sample_id': 'GSM4403974', 'Series_contact_name': 'Srinivasa,,Subramaniam', 'Series_contact_email': 'ssubrama@scripps.edu', 'Series_contact_institute': 'The Scripps Research Institute', 'Series_contact_address': '130 Scripps Way', 'Series_contact_city': 'Jupiter', 'Series_contact_state': 'FL', 'Series_contact_zip/postal_code': '33458', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE146nnn/GSE146674/suppl/GSE146674_Subramaniam_Riboseq_genecounts.csv.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP252106'}\n",
      "367 PRJNA612239 GSE146849 {'Series_title': 'Chromatin state maintains human myeloid leukemia by regulating m6A demethylase ALKBH5 expression [Ribo-seq]', 'Series_geo_accession': 'GSE146849', 'Series_status': 'Public on May 27 2020', 'Series_submission_date': 'Mar 11 2020', 'Series_last_update_date': 'May 27 2020', 'Series_pubmed_id': '32402251', 'Series_summary': 'ALKBH5 is the RNA N(6)-methyladenosine (m6A) demethylase. To understhand the function and mechnism of ALKBH5 in human acute myeloid leukemia, we compared the translational efficiency in wild-type and ALKBH5-knock-down THP1 cells.', 'Series_overall_design': 'Ribo-seq in wild-type or ALKBH5 knock-down THP1 cells', 'Series_type': 'Other', 'Series_contributor': 'Haojian,,Zhang', 'Series_sample_id': 'GSM4407821', 'Series_contact_name': 'Jiazhen,,Wang', 'Series_contact_email': 'jiazhen_wang@hactcm.edu.cn', 'Series_contact_institute': 'Henan University of Chinese Medicine', 'Series_contact_address': '156 Jinshui East Road', 'Series_contact_city': 'Zhengzhou', 'Series_contact_zip/postal_code': '450046', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE146nnn/GSE146849/suppl/GSE146849_riborex_res_deseq2_ann.xlsx', 'Series_platform_id': 'GPL20795', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP252557'}\n",
      "368 PRJNA613736 GSE147324 {'Series_title': 'Nuclease-mediated depletion biases in ribosome footprint profiling libraries', 'Series_geo_accession': 'GSE147324', 'Series_status': 'Public on Mar 27 2020', 'Series_submission_date': 'Mar 20 2020', 'Series_last_update_date': 'Jun 26 2020', 'Series_pubmed_id': '32503920', 'Series_summary': 'Ribosome footprint profiling is a high throughput sequencing based technique that provides detailed and global views of translation in living cells. An essential part of this technology is removal of unwanted, normally abundant, ribosomal RNA sequences in order to save on sequencing costs. The most effective commercial solution (Ribo-Zero) has been discontinued and a number of new, experimentally distinct commercial applications have emerged on the market. Here we evaluated several commercially available alternatives designed for RNA-seq of human samples and find them unsuitable for ribosome footprint profiling. We instead recommend the use of custom-designed biotinylated oligos, which were widely used in early ribosome profiling studies. Importantly, we warn that depletion solutions based on targeted nuclease cleavage significantly perturb the high-resolution information that can be derived from the data, and thus do not recommend their use for any applications that require precise determination of the ends of RNA fragments.', 'Series_overall_design': 'Ribosome profiling libraries were genereated in several cell lines using different rRNA depletion techniques', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Rachel,,Green', 'Series_sample_id': 'GSM4426015', 'Series_contact_name': 'Boris,,Zinshteyn', 'Series_contact_email': 'zinshteyn.boris@gmail.com', 'Series_contact_laboratory': 'Green', 'Series_contact_department': 'Molecular Biology and Genetics', 'Series_contact_institute': 'Johns Hopkins University School of Medicine', 'Series_contact_address': '725 N Wolfe St, PCTB 714', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'Maryland', 'Series_contact_zip/postal_code': '21205', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE147nnn/GSE147324/suppl/GSE147324_set4_CDS_rpms.tsv.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP253534'}\n",
      "369 PRJNA622261 GSE147840 {'Series_title': 'N1-methyladenosine methylations in tRNA drive liver tumorigenesis by PPARδ-mediated cholesterol synthesis', 'Series_geo_accession': 'GSE147840', 'Series_status': 'Public on Mar 01 2021', 'Series_submission_date': 'Mar 31 2020', 'Series_last_update_date': 'Nov 30 2021', 'Series_pubmed_id': '34728628', 'Series_summary': 'Increased protein translation plays a critical role in cancer development and treatment1,2. However, the molecular mechanism that is involved in this process remains poorly understood. N1-methyladenosine (m1A) methylation in RNA accounts for regulating mRNA translation in a post-transcriptional manner3,4. Here we show that m1A methylation levels are remarkably elevated in hepatocellular carcinoma (HCC) patient tumor tissues, especially in patients with microscopic vascular invasion (MVI). Moreover, m1A methylation signals are increased in liver cancer stem cells (CSCs) and are negatively correlated with HCC patient survival. Consistently, TRMT6 and TRMT61A, forming m1A methyltransferase complex, are highly expressed in advanced HCC tumors and are negatively correlated with HCC survival. TRMT6/TRMT61A-mediated m1A methylations are required for self-renewal of liver CSCs and tumorigenesis. Mechanistically, TRMT6/TRMT61A-dependent m1A in tRNA boost PPARδ expression, which triggers cholesterol synthesis to activate Hedgehog signaling, driving self-renewal of liver CSCs and tumorigenesis. For potential therapeutic benefit, we further identify a specific inhibitor against TRMT6/TRMT61A complex that exerts effective therapeutic effect on liver cancer with high m1A methylations. Our findings provide novel insights into the function and molecular mechanism of m1A modifications underlying liver tumorigenesis and drug target, which will serve as a new biomarker for HCC and pave a new way to develop more effective therapeutic strategies for HCC patients.', 'Series_overall_design': 'We find that the overall m1A methylation signal as well as the relevant m1A-methyltransferase TRMT6/61A are of abberrently high level in HCC tumor samples. By performing m1A sequencing for small RNA populations, we discovered several tRNAs with elevated m1A58 methylation stoicheometry. Then we evaluate the influenences on gene expression via Ribo-seq, and proved that TRMT6/TRMT61A-dependent m1A in tRNA boost expression of some metabolic pathway genes, which trigger cholesterol synthesis to activate Hedgehog signaling, driving self-renewal of liver CSCs and tumorigenesis.', 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'Zusen,,Fan', 'Series_sample_id': 'GSM4446984', 'Series_contact_name': 'Xushen,,Xiong', 'Series_contact_email': 'xiongxs@zju.edu.cn', 'Series_contact_department': 'Medical Center', 'Series_contact_institute': 'Zhejiang University', 'Series_contact_address': 'Wenyi Road West', 'Series_contact_city': 'Hangzhou', 'Series_contact_state': 'Zhejiang', 'Series_contact_zip/postal_code': '311121', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE147nnn/GSE147840/suppl/GSE147840_PLC_stem.TE.xls.gz', 'Series_platform_id': 'GPL23227', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP254791'}\n",
      "370 PRJNA623564 GSE148237 {'Series_title': 'Maternal Immune Activation Disrupts Proteostasis in the Fetal Brain', 'Series_geo_accession': 'GSE148237', 'Series_status': 'Public on Nov 09 2020', 'Series_submission_date': 'Apr 07 2020', 'Series_last_update_date': 'Nov 11 2020', 'Series_summary': 'Maternal infection and inflammation during pregnancy is associated with neurodevelopmental disorders in offspring, but little is understood about the molecular mechanisms underlying this epidemiologic phenomenon. We leveraged single-cell RNA sequencing to profile transcriptional changes in the rodent fetal brain in response to maternal immune activation (MIA) and identified perturbations in cellular pathways associated with mRNA translation, ribosome biogenesis, and stress signaling. We found that MIA specifically activated the integrated stress response (ISR) in male, but not female, MIA offspring, thereby reducing global mRNA translation and altering nascent proteome synthesis. Moreover, genetic blockade of ISR activation rescued the social behavior phenotype in MIA male offspring. Our data suggest that therapeutic targeting of the ISR may be beneficial in reducing maternal inflammation-associated neurodevelopmental disorders.', 'Series_overall_design': 'RNA-seq and ribosome profiling of one timepoint (E18) with replicates of the following conditions: PBS control male, PBS control female, maternal immune activation male, maternal immune activation female.', 'Series_type': 'Other', 'Series_contributor': 'Michael,E,Greenberg', 'Series_sample_id': 'GSM4721939', 'Series_contact_name': 'Brian,,Kalish', 'Series_contact_email': 'brian.kalish@sickkids.ca', 'Series_contact_phone': '6305329007', 'Series_contact_laboratory': 'Greenberg Lab', 'Series_contact_department': 'Neurobiology', 'Series_contact_institute': 'Harvard Medical School', 'Series_contact_address': '220 Longwood Ave', 'Series_contact_city': 'Boston', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02115', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE148nnn/GSE148237/suppl/GSE148237_file.tar.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP255607'}\n",
      "371 PRJNA623612 GSE148258 {'Series_title': 'Vigilin/HDLBP promotes translation of endoplasmic reticulum-targeted mRNAs (WT vs HDLBP Ribo-seq + RNA-seq)', 'Series_geo_accession': 'GSE148258', 'Series_status': 'Public on Feb 04 2022', 'Series_submission_date': 'Apr 07 2020', 'Series_last_update_date': 'May 26 2022', 'Series_pubmed_id': '35585045', 'Series_summary': 'The biological role of RNA-binding proteins (RBPs) in the secretory pathway and their contribution to the recognition and co-translational targeting of ER-localized mRNAs is not well established. In this work we used biochemical, transcriptomic and proteomic approaches to delineate the role of human HDLBP/vigilin. PAR-CLIP analysis revealed that HDLBP directly and specifically interacted with more than 80% of all expressed ER-localized mRNAs. Interestingly, the binding to the coding sequence was most prominent for ER-localized mRNAs, while cytosolic mRNAs showed higher binding in the 3’UTR. HDLBP crosslinked strongly to long CU-rich motifs that resided more frequently in coding sequences of ER-localized but not in cytosolic mRNAs. This indicated that the primary sequence composition determines the basis for HDLBP binding specificity and its multivalent interactions with ER-bound mRNAs. PAR-CLIP analysis also revealed direct interactions of HDLBP with the RNA components of the translational apparatus,\\xa0while in vivo proximity proteomics detected proteins involved in translation and components of the signal recognition particle (SRP). Functional studies using CRISPR-Cas9 HDLBP knockout cell lines in combination with ribosome profiling, pSILAC, and luciferase assays showed decreased translation efficiency of HDLBP target mRNAs, impaired protein synthesis and secretion in the knockout conditions. Finally, HDLBP absence resulted in decrease of in vivo lung tumor formation. These results highlight a general function for HDLBP in the translation of ER -localized mRNAs via the secretory pathway and discover its relevance for cell proliferation and tumor progression.', 'Series_overall_design': 'Translational efficiency in WT vs HDLBP KO HEK293T cells were analyzed by ribosome profiling (ribo-seq) and matching RNA-seq datasets.', 'Series_type': 'Other', 'Series_contributor': 'Markus,,Landthaler', 'Series_sample_id': 'GSM4458898', 'Series_contact_name': 'Katharina,,Schmidt-Bleek', 'Series_contact_email': 'katharina.schmidt-bleek@bih-charite.de', 'Series_contact_department': 'Julius Wolff Institute of Biomechanics and Musculoskeletal Regeneration', 'Series_contact_institute': 'Charité - Universitätsmedizin Berlin', 'Series_contact_address': 'Augustenburger Platz 1', 'Series_contact_city': 'Berlin', 'Series_contact_zip/postal_code': '13353', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE148nnn/GSE148258/suppl/GSE148258_processed_data_riboseq_te.txt.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP255633'}\n",
      "372 PRJNA624906 GSE148536 {'Series_title': 'Ythdf m6A readers compensate each other in a context dependent manner [Ribo-seq]', 'Series_geo_accession': 'GSE148536', 'Series_status': 'Public on Jun 04 2020', 'Series_submission_date': 'Apr 12 2020', 'Series_last_update_date': 'Jun 04 2020', 'Series_summary': 'YTH domain family proteins Ythdf1, Ythdf2, Ythdf3 are three readers with highly similar structure, which were shown to have different roles in the cell. However, their similarity and the fact that they tend to bind the same targets suggest that in some cases they may have an overlapping role. In this paper, we systematically knocked-out (KO) each of the three readers and the three together (triple-KO), and estimated the effect in-vitro in mouse embryonic stem cells (mESCs), and in-vivo, in viability and gametogenesis. We show that in mESCs there is a compensation between the three readers, with a dramatic hyper-pluripotent phenotype only in the triple-KO. However, in-vivo, Ythdf2 has a dominant role that cannot be compensated by Ythdf1 or Ythdf3, both in viability of the mouse pups and in gametogenesis. We suggest that compensation is dosage-dependent, and in-vivo we cannot detect it due to difference in the readers’ expression, both in level and in spatial location. In addition, we found that Ythdf1 and Ythdf3 specifically down-regulate 2-cell genes, that Ythdf2-KO in gametogenesis affects microtubuline related genes, and that Mettl3-KO severity is increased as the deletion occurs earlier in gametogenesis.', 'Series_overall_design': 'Mouse embryonic stem cells were knocked-out to each of the following proteins: Mettl3, Ythdf1, Ythdf2, Ythdf3, and Ythdf1/2/3 triple-KO. Ribosome footprint (ribo-seq) was generated in each of these KOs, and in WT control.', 'Series_type': 'Other', 'Series_contributor': 'Jacob,,Hanna', 'Series_sample_id': 'GSM4473280', 'Series_contact_name': 'Noa,,Novershtern', 'Series_contact_email': 'noa.novershtern@weizmann.ac.il', 'Series_contact_department': 'Molecular Genetics', 'Series_contact_institute': 'Weizmann Institute of Science', 'Series_contact_address': 'Weizmann Institute', 'Series_contact_city': 'Rehovot', 'Series_contact_zip/postal_code': '7610001', 'Series_contact_country': 'Israel', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE148nnn/GSE148536/suppl/GSE148536_RAW.tar', 'Series_platform_id': 'GPL21626', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP256256'}\n",
      "373 PRJNA625261 GSE148636 {'Series_title': 'Dual targeting of DDX3 and eIF4A by the translation inhibitor rocaglamide A', 'Series_geo_accession': 'GSE148636', 'Series_status': 'Public on Dec 07 2020', 'Series_submission_date': 'Apr 14 2020', 'Series_last_update_date': 'Mar 08 2021', 'Series_pubmed_id': '33296667', 'Series_summary': 'Small molecule compounds that sense the nucleic acid sequences, promise the attractive venue for drug development. Such an unusual effect has been observed in the natural product Rocaglamide A (RocA) from Aglaia plant, proving to exhibit anti-tumor effects by clamping eukaryotic initiation factor (eIF) 4A onto mRNA polypurine sequences. Although eIF4A has been speculated the unique target of RocA, the insensitization of eIF4A in human cells only partially rescued the translation repression from RocA, suggesting another alternative target of this compound. Here, we revealed that DDX3 is an alternative target of RocA. Developing a RocA derivative with an O-nitrobenzoxadiazole unit (RocA-O-NBD), which can covalently bind to proximate proteins and provide fluorescence to them, we identified DDX3 bound to RocA. As observed in eIF4A, RocA locked the DDX3 protein onto polypurine sequences of RNA in an ATP-independent manner. De novo assembled Aglaia plant transcriptome uncovered the natural amino acid substitutions in Aglaia DDX3 to protect itself from RocA toxicity. Because of the dominant negative effect of RocA, we also proved the protein abundance of eIF4A and DDX3 in cancer cells determines their sensitivity to RocA. Overall, this study discovered DDX3 as another target of RocA and suggests the probability to predict tumor toxicity of RocA by the target abundance.', 'Series_overall_design': 'Ribosome profiling', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Shintaro,,Iwasaki', 'Series_sample_id': 'GSM4475951', 'Series_contact_name': 'Mingming,,Chen', 'Series_contact_email': 'mingming.chen@riken.jp', 'Series_contact_phone': '07031428686', 'Series_contact_laboratory': 'RNA systems biochemistry lab', 'Series_contact_department': 'CPR', 'Series_contact_institute': 'RIKEN', 'Series_contact_address': 'Hirisawa 2-2', 'Series_contact_city': 'Wakoshi', 'Series_contact_state': 'Saitama', 'Series_contact_zip/postal_code': '351-0106', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE148nnn/GSE148636/suppl/GSE148636_RAW.tar', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP256332'}\n",
      "374 PRJNA626635 GSE148965 {'Series_title': 'Regulation of translation by site-specific ribosomal RNA methylation [RNA-Seq]', 'Series_geo_accession': 'GSE148965', 'Series_status': 'Public on Aug 16 2021', 'Series_submission_date': 'Apr 20 2020', 'Series_last_update_date': 'Mar 08 2022', 'Series_pubmed_id': '34759377', 'Series_summary': 'We report that 2’-O-methylation levels at subset of positions in human ribosomal RNA are variable between cell types and conditions, and that the degree of methylation at distinct sites is responsive to key cellular pathways. MYC overexpression results in increased methylation at a particular rRNA site (18S:C174). We find that this methylation is important for modulating translation of distinct mRNAs, leading to phenotypic changes including modulation of cell proliferation rate.', 'Series_overall_design': 'Please note that each processed data was generated from all replicates and is linked to the corresponding *_1 sample records.', 'Series_type': 'Other', 'Series_contributor': 'Anders,H,Lund', 'Series_sample_id': 'GSM5291038', 'Series_contact_name': 'Anders,Henrik,Lund', 'Series_contact_email': 'anders.lund@bric.ku.dk', 'Series_contact_institute': 'University of Copenhagen,BRIC', 'Series_contact_address': 'Ole Maaløes Vej 5', 'Series_contact_city': 'Copenhagen', 'Series_contact_zip/postal_code': '2200', 'Series_contact_country': 'Denmark', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE148nnn/GSE148965/suppl/GSE148965_mean_nooffset_CPM_HelaS3_WT_total.bigWig', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP257547'}\n",
      "375 PRJNA627130 GSE149016 {'Series_title': 'Selectivity of mRNA degradation by autophagy in yeast', 'Series_geo_accession': 'GSE149016', 'Series_status': 'Public on Mar 08 2021', 'Series_submission_date': 'Apr 21 2020', 'Series_last_update_date': 'Nov 09 2021', 'Series_pubmed_id': '33875662', 'Series_summary': 'Synthesis and degradation of cellular constituents must be balanced to maintain cellular homeostasis, especially during adaptation to environmental stress. The role of autophagy in the degradation of proteins and organelles is well-characterized. However, autophagy-mediated RNA degradation in response to stress and the potential preference of specific RNAs to undergo autophagy-mediated degradation have not been examined. In this study, we demonstrate selective mRNA degradation by rapamycin-induced autophagy in yeast. Profiling of mRNAs from the vacuole reveals that subsets of mRNAs, such as those encoding amino acid biosynthesis and ribosomal proteins, are preferentially delivered to the vacuole by autophagy for degradation. We also reveal that autophagy-mediated mRNA degradation is tightly coupled with translation by ribosomes. Genome-wide ribosome profiling suggested a high correspondence between ribosome association and targeting to the vacuole. We propose that autophagy-mediated mRNA degradation is a unique and previously-unappreciated function of autophagy that affords post-transcriptional gene regulation.', 'Series_overall_design': 'Ribosome profiling and RNA-seq', 'Series_type': 'Other', 'Series_contributor': 'Yoshinori,,Ohsumi', 'Series_sample_id': 'GSM4487484', 'Series_contact_name': 'Shiho,,Makino', 'Series_contact_email': 'makino.s.ac@m.titech.ac.jp', 'Series_contact_institute': 'Tokyo Institute of Technology', 'Series_contact_address': '4259-S2-12 Nagatsuta-cho, Midori-ku', 'Series_contact_city': 'Yokohama', 'Series_contact_zip/postal_code': '226-8503', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE149nnn/GSE149016/suppl/GSE149016_RAW.tar', 'Series_platform_id': 'GPL21656', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP257699'}\n",
      "376 PRJNA627881 GSE149279 {'Series_title': '4EHP and GIGYF1/2 induce translation-coupled messenger RNA decay [Wildtype and ZNF598-null]', 'Series_geo_accession': 'GSE149279', 'Series_status': 'Public on Oct 06 2020', 'Series_submission_date': 'Apr 24 2020', 'Series_last_update_date': 'Jan 15 2021', 'Series_pubmed_id': '33053355', 'Series_summary': 'Although canonical mRNA degradation is generally recognized to be translation dependent, our understanding of the molecular events that coordinate ribosome movement with the decay machinery is still limited. Here, we show that the 4EHP–GIGYF1/2 complex triggers co-translational mRNA decay as a result of altered ribosome activity during elongation. Human cells lacking 4EHP and GIGYF1 and 2 proteins accumulate transcripts known to be degraded in a translation dependent manner or with prominent ribosome pausing. These include mRNAs encoding secretory and membrane-bound proteins or specific tubulin isotypes, among others. 4EHP–GIGYF1/2 fails to reduce target mRNA levels in the absence of ribosome stalling or upon disruption of its interaction with the cap structure, DDX6 and a GYF domain-associated protein. Our studies reveal how a repressor complex linked to neurological disorders minimizes the protein output of a subset of mRNAs.', 'Series_overall_design': 'Two cell types with two biological replicates analysed using RNA-Seq and Ribo-Seq', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Cátia,,Igreja', 'Series_sample_id': 'GSM4495448', 'Series_contact_name': 'Ramona,,Weber', 'Series_contact_email': 'ramona.weber@uzh.ch', 'Series_contact_department': 'Institute for Regenerative Medicine', 'Series_contact_institute': 'University of Zurich', 'Series_contact_address': 'Wagistrasse 12', 'Series_contact_city': 'Schlieren', 'Series_contact_zip/postal_code': '8952', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE149nnn/GSE149279/suppl/GSE149279_readcounts_Ribo.txt.gz', 'Series_platform_id': 'GPL21290', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP258289'}\n",
      "377 PRJNA630259 GSE149844 {'Series_title': 'Global Analysis of Translational Reprogramming during Flagellar Regeneration Reveals the Role of Sphingolipid Metabolism in Ciliogenesis and Ciliopathies', 'Series_geo_accession': 'GSE149844', 'Series_status': 'Public on Apr 28 2022', 'Series_submission_date': 'May 04 2020', 'Series_last_update_date': 'May 01 2022', 'Series_summary': 'Protein synthesis is essential for cilium assembly, but little is known about translational reprogramming that occurs during ciliogenesis. Using ribosome footprinting, we perform global translatome profiling on Chlamydomonas during flagellar regeneration.', 'Series_overall_design': 'Chlamydomonas reinhardtii (strain CC-125) that were grown for 15, 30, 60, and 90 min after deflagellation to generate libraries for ribosome footprint sequencing (Ribo-seq) and RNA sequencing (RNA-seq) . A reference sample was collected from the same culture before deflagellation (Non-Treated). We also performed RNA sequencing for C.elegans mutants : sptl-1/sptl-2/sptl-3. Each strain was analyzed in biological duplicate.', 'Series_type': 'Other', 'Series_contributor': 'Jingying,,Huang', 'Series_sample_id': 'GSM4514176', 'Series_contact_name': 'Jingying,,Huang', 'Series_contact_email': 'huang-jy18@mails.tsinghua.edu.cn', 'Series_contact_department': 'School of Life Sciences', 'Series_contact_institute': 'Tsinghua University', 'Series_contact_address': 'Haidian  District', 'Series_contact_city': 'Beijing', 'Series_contact_zip/postal_code': '100084', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE149nnn/GSE149844/suppl/GSE149844_sptl_3_Expressed_gene_count_DEseq_normalized.txt.gz', 'Series_platform_id': 'GPL15922', 'Series_platform_taxid': '6239', 'Series_sample_taxid': '6239', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP260039'}\n",
      "378 PRJNA630973 GSE150034 {'Series_title': 'Genome-wide effects of the antimicrobial peptide apidaecin on translation termination', 'Series_geo_accession': 'GSE150034', 'Series_status': 'Public on Oct 02 2020', 'Series_submission_date': 'May 07 2020', 'Series_last_update_date': 'Oct 13 2020', 'Series_pubmed_id': '33031031', 'Series_summary': 'The proline-rich antimicrobial peptide apidaecin (Api) inhibits bacterial protein synthesis in a distinctive way. In vitro biochemical and structural studies showed that Api binds in the nascent peptide exit tunnel of a ribosome that has completed translation of a gene and traps the release factors on the ribosome. By analyzing the distribution of ribosomes on mRNAs in Api-treated cells using Ribo-seq, we uncovered a range of effects that stem from the unique mechanism of Api action. Exposure of bacterial cells to Api results in arrest of ribosomes at stop codons, likely in a post-release state due to direct Api action, as well as in a pre-release state due to the depletion of available release factors. In addition, Api action leads to a pronounced queuing of translating ribosomes behind those arrested at stop codons. One of the major consequences of Api action is a dramatically increased level of stop codon bypass by ribosomes paused in a pre-release state, leading to an accumulation of proteins with C-terminal extensions, as confirmed by whole-cell proteomics analysis. Stop codon bypass occurs either in 0-frame, by misincorporation of a near-cognate aminoacyl-tRNA at the stop codon, or via frameshifting, allowing for a fraction of the translating ribosomes to reach the 3’ ends of mRNA transcripts. The pervasive stalling of pre-release ribosomes at the stop codons and at the RNA transcripts ends triggers activation of the cellular ribosome rescue systems which, likely due to Api action as well, remain ineffective. Understanding the unique mode of Api action as translation termination inhibitor in the cell can advance the development of treatments for infection and genetic diseases as well as new research tools for genome exploration.', 'Series_overall_design': 'Ribosome profiling and RNA-seq were performed on Escherichia coli dosed with apidaecin and compared to non-treated cells (2 biological replicates each). Additionally, the lysate of one Api replicate and one control replicate were treated with the antibiotic puromycin in an attempt to clear ribosomes from the mRNAs.', 'Series_type': 'Other', 'Series_contributor': 'Nora,,Vázquez-Laslop', 'Series_sample_id': 'GSM4521036', 'Series_contact_name': 'Kyle,,Mangano', 'Series_contact_institute': 'University of Illinois at Chicago', 'Series_contact_address': '900 S Ashland Ave, MBRB Rm 3056', 'Series_contact_city': 'Chicago', 'Series_contact_state': 'Illinois', 'Series_contact_zip/postal_code': '60607', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE150nnn/GSE150034/suppl/GSE150034_RAW.tar', 'Series_platform_id': 'GPL21222', 'Series_platform_taxid': '562', 'Series_sample_taxid': '562', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP260436'}\n",
      "379 PRJNA631930 GSE150375 {'Series_title': 'Translation initiation site profiling reveals widespread synthesis of non-AUG-initiated protein isoforms in yeast', 'Series_geo_accession': 'GSE150375', 'Series_status': 'Public on Jul 24 2020', 'Series_submission_date': 'May 12 2020', 'Series_last_update_date': 'Oct 23 2020', 'Series_pubmed_id': '32710835', 'Series_summary': 'Genomic analyses in budding yeast have helped to define the foundational principles of eukaryotic gene expression, but have systematically excluded specific classes of potential coding regions, including those with non-AUG start codons. Without methods to define coding regions empirically, the prevalence of these non-canonical coding regions has been impossible to assess. Here, we applied an experimental approach to globally annotate translation initiation sites in yeast and identified a class of 149 genes that encode N-terminally extended alternate protein isoforms that result from translation initiation at non-AUG codons upstream of the annotated AUG start codon. These alternate isoforms are produced in concert with canonical isoforms and are translated with a high degree of specificity, resulting from initiation at only a small subset of possible start codons in 5’ leader regions. Their translation is enriched during meiosis, and is induced by low eIF5A levels, which are observed in this context. These findings reveal widespread production of non-canonical protein isoforms and, more generally, show unexpected complexity to the rules by which the budding yeast genome is decoded.', 'Series_overall_design': 'We performed ribosome profiling following pre-treatment of budding yeast cells with 3 micromolar of the translation initiation inhibitor, lactimidomycin (LTM).', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Gloria,A,Brar', 'Series_sample_id': 'GSM4547713', 'Series_contact_name': 'Gloria,,Brar', 'Series_contact_email': 'gabrar@berkeley.edu', 'Series_contact_department': 'MCB', 'Series_contact_institute': 'UC Berkeley', 'Series_contact_address': '626 Barker Hall', 'Series_contact_city': 'Berkeley', 'Series_contact_zip/postal_code': '94720', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE150nnn/GSE150375/suppl/GSE150375_sk1_revised_100331_cleanest.gtf.gz', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP261277'}\n",
      "380 PRJNA632042 GSE150439 {'Series_title': 'Effect of QTRT2 KO  on mitochondrial translation', 'Series_geo_accession': 'GSE150439', 'Series_status': 'Public on Aug 28 2020', 'Series_submission_date': 'May 12 2020', 'Series_last_update_date': 'Sep 08 2020', 'Series_pubmed_id': '32859890', 'Series_summary': 'Mitochondria are organelles that generate most of the energy in eukaryotic cells in the form of ATP via oxidative phosphorylation in eukaryote. Twenty-two species of mitochondrial (mt-)tRNAs encoded in mtDNA are required to translate essential subunits of the respiratory chain complexes involved in oxidative phosphorylation. mt-tRNAs contain post-transcriptional modifications introduced by nuclear-encoded tRNA-modifying enzymes. These modifications are required for deciphering genetic code accurately, as well as stabilizing tRNA. Loss of tRNA modifications frequently results in severe pathological consequences. We performed a comprehensive analysis of post-transcriptional modifications of all human mt-tRNAs, including 14 previously-uncharacterized species, and revised the modification status of some of the previously studied species. In total, we found 17 kinds of RNA modifications at 137 positions (8.7% in 1,575 nucleobases) in 22 species of human mt-tRNAs. An up-to-date list of 34 genes responsible for human mt-tRNA modifications are provided. We here demonstrated that both QTRT1 and QTRT2 are required for biogenesis of queuosine (Q) at position 34 of four mt-tRNAs. Our results provide insight into the molecular mechanisms underlying the mitochondrial decoding system, and could help to elucidate the molecular pathogenesis of human mitochondrial diseases caused by aberrant tRNA modifications.', 'Series_overall_design': 'Ribosome profiling and RNA-seq', 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'Tsutomu,,Suzuki', 'Series_sample_id': 'GSM4550618', 'Series_contact_name': 'Shintaro,,Iwasaki', 'Series_contact_email': 'shintaro.iwasaki@riken.jp', 'Series_contact_laboratory': 'Iwasaki RNA systems biochemistry lab', 'Series_contact_department': 'Cluster for Pioneering Research', 'Series_contact_institute': 'RIKEN', 'Series_contact_address': '2-1', 'Series_contact_city': 'Hirosawa Wako city, Saitama prefecture, Japan', 'Series_contact_zip/postal_code': '351-0198', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE150nnn/GSE150439/suppl/GSE150439_RAW.tar', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP261344'}\n",
      "381 PRJNA635625 GSE151370 {'Series_title': 'Elucidating the role of Zuo1 protein on the translational control of gene expression in Saccharomyces cerevisiae', 'Series_geo_accession': 'GSE151370', 'Series_status': 'Public on Jun 01 2023', 'Series_submission_date': 'May 28 2020', 'Series_last_update_date': 'Jun 01 2023', 'Series_summary': 'The ribosome associated complex (RAC) is a ribosome bound protein chaperone complex reported to surveil the translation of proteins with positively charged regions. It has been posited that RAC might be able to directly regulate translation by coupling co-translational folding with the peptide-elongation cycle. To identify the targets of RAC in cells and test the hypothesis that the complex modulates translation, we performed ribosome profiling on wild- type yeast cells and cells lacking a key component of the RAC that binds near the ribosome active site (zuo1Δ). Ribosome profiling is a sequencing-based technique that allows us to take a nucleotide resolution snapshot of where every ribosome sits on every mRNA in a cell at a given point in time. This powerful approach can provide information about the contributions of individual proteins to the translational landscape of a cell. We identified >300 targets for the RAC, and unexpectedly observed that the ribosome frameshifts on ~10% of these targets in zuo1Δ cells. The maintenance of ribosome reading frame is essential for cell health because frameshifts can result in the production of non-functional truncated and extended protein products. These studies have the potential to uncover RAC as a critical determinant of translational fidelity in eukaryotic cells.', 'Series_overall_design': 'mRNA-seq and Ribosome profiling of exponentially growing (OD600 = 0.6) Saccharomyces cerevisiae cells were performed by deep sequencing , in two biological replicates, using Illumina HiSeq-4000 platform.', 'Series_type': 'Other', 'Series_contributor': 'Kristin,S,Koutmou', 'Series_sample_id': 'GSM4576307', 'Series_contact_name': 'mehmet,,tardu', 'Series_contact_email': 'mtardu@umich.edu', 'Series_contact_laboratory': 'Koutmou Lab', 'Series_contact_department': 'Chemistry', 'Series_contact_institute': 'University of Michigan', 'Series_contact_address': '930 N University Ave', 'Series_contact_city': 'Ann Arbor', 'Series_contact_state': 'MI', 'Series_contact_zip/postal_code': '48109', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE151nnn/GSE151370/suppl/GSE151370_read_counts.xlsx', 'Series_platform_id': 'GPL21656', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP265101'}\n",
      "382 PRJNA636959 GSE151687 {'Series_title': 'Authentication of Hippuristanol-eIF4A1 Target Engagement Facilitates Identification of eIF4A1 Helicase Dependencies within 5’ Leader Regions', 'Series_geo_accession': 'GSE151687', 'Series_status': 'Public on Aug 05 2020', 'Series_submission_date': 'Jun 03 2020', 'Series_last_update_date': 'Oct 06 2020', 'Series_pubmed_id': '32766783', 'Series_summary': 'Hippuristanol (Hipp) is a natural product that selectively inhibits protein synthesis by targeting eukaryotic initiation factor (eIF) 4A, a DEAD-box RNA helicase required for ribosome recruitmentt o mRNA templates. Using a CRISPR/Cas9-based variomics screen, we identify functional eIF4A1 Hipp-resistant alleles, which in turn allow us to link the translation-inhibitory and cytotoxic properties of Hipp to eIF4A1 target-engagement. Genome-wide translational profiling in the absence or presence of Hipp (~EC50) were undertaken and our validation studies provided insight into structure-activity relationships of eIF4A-dependent mRNAs.', 'Series_overall_design': 'Ribosome Profiling and RNA-Sequencing of DMSO and Hippuristanol treated HAP1 cells.', 'Series_type': 'Other', 'Series_contributor': 'Jerry,,Pelletier', 'Series_sample_id': 'GSM4588302', 'Series_contact_name': 'Jerry,,Pelletier', 'Series_contact_email': 'jerry.pelletier@mcgill.ca', 'Series_contact_laboratory': 'Room 810B', 'Series_contact_department': 'Biochemistry', 'Series_contact_institute': 'McGill university', 'Series_contact_address': 'McIntyre Medical Sciences Building 3655 promenade Sir-William-Osler', 'Series_contact_city': 'Montreal', 'Series_contact_state': 'Quebec', 'Series_contact_zip/postal_code': 'H3G 1Y6', 'Series_contact_country': 'Canada', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE151nnn/GSE151687/suppl/GSE151687_raw_counts.csv.gz', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP265763'}\n",
      "383 PRJNA636967 GSE151688 {'Series_title': 'RRF plays critical roles in ribosome homeostasis in E. coli but has little effect on translational coupling', 'Series_geo_accession': 'GSE151688', 'Series_status': 'Public on Oct 06 2020', 'Series_submission_date': 'Jun 03 2020', 'Series_last_update_date': 'Oct 08 2020', 'Series_pubmed_id': '32965213', 'Series_summary': 'We used ribosome profiling to characterize the biological role of ribosome recycling factor (RRF) in Escherichia coli. As expected, RRF depletion leads to enrichment of post-termination 70S complexes in 3′-UTRs. We also observe that elongating ribosomes are unable to complete translation because they are blocked by non-recycled ribosomes at stop codons. Previous studies have suggested a role for recycling in translational coupling within operons; if a ribosome remains bound to an mRNA after termination, it may re-initiate downstream. We found, however, that RRF depletion did not significantly affect coupling efficiency in reporter assays or in ribosome density genome-wide. These findings argue that re-initiation is not a major mechanism of translational coupling in E. coli. Finally, RRF depletion has dramatic effects on the activity of ribosome rescue factors tmRNA and ArfA. Our results provide a global view of the effects of the loss of ribosome recycling on protein synthesis in E. coli.', 'Series_overall_design': 'Ribosome profiling of RRF knock-down condition derived from E. coli MG1655', 'Series_type': 'Other', 'Series_contributor': 'Allen,R,Buskirk', 'Series_sample_id': 'GSM4588320', 'Series_contact_name': 'Allen,R,Buskirk', 'Series_contact_email': 'buskirk@jhmi.edu', 'Series_contact_department': 'Molecular Biology and Genetics', 'Series_contact_institute': 'Johns Hopkins University School of Medicine', 'Series_contact_address': '725 N. Wolfe St', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21205', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE151nnn/GSE151688/suppl/GSE151688_RAW.tar', 'Series_platform_id': 'GPL18956', 'Series_platform_taxid': '511145', 'Series_sample_taxid': '511145', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP265779'}\n",
      "384 PRJNA637044 GSE151718 {'Series_title': 'Translatome analysis of Mycobacterium tuberculosis', 'Series_geo_accession': 'GSE151718', 'Series_status': 'Public on Jun 02 2021', 'Series_submission_date': 'Jun 03 2020', 'Series_last_update_date': 'Jan 25 2023', 'Series_pubmed_id': '36624167', 'Series_summary': 'We present the data obtained from high-resolution ribosome profiling analysis of Mycobacterium tuberculosis grown under standard conditions (log phase) and cells subjected to oxidative stress (50 µM cumene hydroperoxide) and pH stress (pH 4.5). Our data shows pervasive ribosome pausing in the M. tb translatome. A large number of genes show very high pause immediately downstream to the translation start site. Moreover, serines and alanines in the E site of the ribosome exhibit highest pause scores.', 'Series_overall_design': 'Ribosome profiling analysis of M. tb grown to log phase, subjected to oxidative stress (50µM cumene hydroperoxide) and pH stress (pH 4.5).', 'Series_type': 'Other', 'Series_contributor': 'Somdeb,,Chattopadhyay', 'Series_sample_id': 'GSM4589057', 'Series_contact_name': 'G Aneeshkumar,,Arimbasseri', 'Series_contact_email': 'aneesh@nii.ac.in', 'Series_contact_phone': '+91 9971986746', 'Series_contact_laboratory': 'Molecular Genetics Laboratory', 'Series_contact_department': 'Molecular Genetics Laboratory', 'Series_contact_institute': 'National Institute of Immunology', 'Series_contact_address': 'Aruna Asaf Ali Marg', 'Series_contact_city': 'New Delhi', 'Series_contact_state': 'Delhi', 'Series_contact_zip/postal_code': '110067', 'Series_contact_country': 'India', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE151nnn/GSE151718/suppl/GSE151718_RAW.tar', 'Series_platform_id': 'GPL22688', 'Series_platform_taxid': '1773', 'Series_sample_taxid': '1773', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP265811'}\n",
      "385 PRJNA637076 GSE151742 {'Series_title': 'Expression of the neuronal tRNA n-Tr20 regulates synaptic transmission and seizure susceptibility', 'Series_geo_accession': 'GSE151742', 'Series_status': 'Public on Jun 04 2020', 'Series_submission_date': 'Jun 03 2020', 'Series_last_update_date': 'Sep 03 2020', 'Series_pubmed_id': '32853550', 'Series_summary': 'Loss of the nuclear-encoded brain-specific arginine UCU tRNA, n-Tr20, gene increases seizure threshold in mice, and alters inhibitory neurotransmission in the hippocampus. We investigated the molecular impact of loss of n-Tr20 expression on the trancriptome and translatome in the forebrain and hippocampus of multiple n-Tr20 mutant and control strains. Loss of n-Tr20 altered translation initiation by activating the integrated stress response and suppressing mTOR signaling, the latter of which contributes to the enhanced GABAergic transmission.', 'Series_overall_design': '33 samples are analyzed in this study from n-Tr20 tRNA knockouts and transgenes in the context of C57BL/6J (B6J) and C57BL6/NJ (B6N) forebrain and hippocampus. The design includes 4 replicates each of forebrain RNA-seq from B6N, B6N-nTr20-/-, B6J-nTr20-/-, B6J, and B6J with the B6N n-Tr20 transgene (B6J-Tg(nTr20N/N) or B6J-Tgwt) mice; one sample was sequenced twice to account for batch effects. Additionally, this study includes hippocampus RNA-seq and ribosome profiling with 3 replicates each from B6N and B6N-nTr20-/- hippocampi.', 'Series_type': 'Other', 'Series_contributor': 'Susan,L,Ackerman', 'Series_sample_id': 'GSM4589689', 'Series_contact_name': 'Scott,,Adamson', 'Series_contact_email': 'scott.adamson@jax.org', 'Series_contact_laboratory': 'Chuang', 'Series_contact_institute': 'The Jackson Laboratory', 'Series_contact_address': '10 Discovery Drive', 'Series_contact_city': 'Farmington', 'Series_contact_state': 'CT', 'Series_contact_zip/postal_code': '06032', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE151nnn/GSE151742/suppl/GSE151742_RAW.tar', 'Series_platform_id': 'GPL21103', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP265841'}\n",
      "386 PRJNA638018 GSE151986 {'Series_title': 'Low-input RNase footprinting to profile RNA translation', 'Series_geo_accession': 'GSE151986', 'Series_status': 'Public on Jan 20 2022', 'Series_submission_date': 'Jun 08 2020', 'Series_last_update_date': 'Jan 22 2022', 'Series_summary': 'We treated HEK293T cells with different doses of RNase and sequence RNA fragements after the treatments. Furthermore, we developed a low-input RNase footprinting approach to quantify genome-wide RNA translation using 50,000 and 1,000 HEK293T and K562 cells. The ribosome profiling and RNA-seq data were used as the control.', 'Series_overall_design': '1. HEK293T cells were treated with 3 different doses of RNase (i.e. low, medium, high) and RNA fragments were isolated for sequencing. 2. Low-input RNase footprinting for 50,000 and 1,000 HEK293T and K562 cells. 3. Ribosome profiling and RNA-seq data for HEK293T and K562 cells were used for comparisons.', 'Series_type': 'Other', 'Series_contributor': 'Zhe,,Ji', 'Series_sample_id': 'GSM4594589', 'Series_contact_name': 'Zhe,,Ji', 'Series_contact_email': 'zhe.ji@northwestern.edu', 'Series_contact_institute': 'Northwestern University', 'Series_contact_address': '303 E Superior St, Lurie 6-107', 'Series_contact_city': 'Chicago', 'Series_contact_state': 'ILLINOIS', 'Series_contact_zip/postal_code': '60611', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE151nnn/GSE151986/suppl/GSE151986_RAW.tar', 'Series_platform_id': 'GPL21290', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP266410'}\n",
      "387 PRJNA639716 GSE152556 {'Series_title': 'Ribosome profiling analysis of aging human skeletal muscle (Ribo-Seq)', 'Series_geo_accession': 'GSE152556', 'Series_status': 'Public on Mar 08 2021', 'Series_submission_date': 'Jun 16 2020', 'Series_last_update_date': 'Mar 08 2021', 'Series_pubmed_id': '33472542', 'Series_summary': 'With advancing age, human muscle loses strength and function, but the molecular causes of these losses are unknown. Skeletal muscle shows an age-dependent decline in the levels of different proteins, but whether such decline is associated with reduced translation has not been studied. To address this gap of knowledge, we used the technique of ribosome profiling to study translation in muscle from middle-aged and old individuals. Using ribosome occupancy as a measure of translation status, several mRNAs showed differential translation with age. Older age was associated with lower translation of myosin and titin isoforms and more broadly with the translation of proteins involved in oxidative phosphorylation encoded by the mitochondrial genome. Based on our findings, we propose that mitochondrial proteins are less translated in old skeletal muscle.', 'Series_overall_design': 'A 6-mm Bergström biopsy of 200-400 mg of vastus lateralis skeletal muscle from each of three middle-aged and two old individuals were analyzed  by ribosome profiling.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Luigi,,Ferrucci', 'Series_sample_id': 'GSM4618181', 'Series_contact_name': 'Supriyo,,De', 'Series_contact_laboratory': 'Computational Biology & Genomics Core', 'Series_contact_department': 'Laboratory of Genetics and Genomics', 'Series_contact_institute': 'NIA-IRP, NIH', 'Series_contact_address': '251 Bayview Blvd', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'Maryland', 'Series_contact_zip/postal_code': '21224', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE152nnn/GSE152556/suppl/GSE152556_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP267470'}\n",
      "388 PRJNA640006 GSE152664 {'Series_title': 'Genetic circuit characterization by inferring RNA polymerase movement and ribosome usage', 'Series_geo_accession': 'GSE152664', 'Series_status': 'Public on Jun 18 2020', 'Series_submission_date': 'Jun 17 2020', 'Series_last_update_date': 'Oct 27 2020', 'Series_pubmed_id': '33020480', 'Series_summary': 'When performing a computation, genetic circuits change states through a symphony of genetic parts that turn regulator expression on and off. Debugging is frustrated by an inability to measure part function and identify the origins of failures. Here, we take “snapshots” of a large genetic circuit in different states. RNA-seq is used to visualize circuit function as a changing pattern of RNA polymerase (RNAP) flux along the DNA. Together with ribosome profiling, all 54 genetic parts are parameterized. The circuit behaves as designed; however, it is riddled with errors, including cryptic sense/antisense promoters and translation, attenuation, incorrect start codons and a failed gate. While serendipitously not impacting function, they reduce prediction accuracy and could lead to failures when used in other designs. Finally, the cellular power (RNAPs and ribosomes) required to maintain a circuit state is calculated. This work demonstrates the use of a small number of measurements to fully parameterize a regulatory circuit and quantify its impact on host.', 'Series_overall_design': 'Transcriptome and translatome of E.coli DH10b cells carrying a genetic circuit 0x41 at different induction states were generated', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Christopher,A,Voigt', 'Series_sample_id': 'GSM4745079', 'Series_contact_name': 'Jing,,Zhang', 'Series_contact_email': 'jillwill00@gmail.com', 'Series_contact_institute': 'Boston University', 'Series_contact_address': '610 Commonwealth Avenuce', 'Series_contact_city': 'Boston', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02215', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE152nnn/GSE152664/suppl/GSE152664_RAW.tar', 'Series_platform_id': 'GPL14548', 'Series_platform_taxid': '562', 'Series_sample_taxid': '562', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP267640'}\n",
      "389 PRJNA641624 GSE153142 {'Series_title': 'A systematic survey of PRMT interactomes reveals key roles of arginine methylation in the global control of RNA splicing and translation [Ribo-Seq]', 'Series_geo_accession': 'GSE153142', 'Series_status': 'Public on Jun 30 2022', 'Series_submission_date': 'Jun 24 2020', 'Series_last_update_date': 'Sep 30 2022', 'Series_summary': 'We used Ribo-seq to determine how PRMT inhibition affects translation of different mRNAs, This analysis can generate a “snapshot” of all mRNAs that are occupied by active ribosomes (i.e., undergoing active translation) in a cell at a particular condition.', 'Series_overall_design': 'Ribo-seq was used to measure the distribution of ribosome protected RNA fragments in control and PRMT inhibition samples.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Zefeng,,Wang', 'Series_sample_id': 'GSM4634839', 'Series_contact_name': 'Zefeng,,Wang', 'Series_contact_email': 'wangzefeng@picb.ac.cn', 'Series_contact_department': 'CAG-MPG Partner Institution for Computational Biology', 'Series_contact_institute': 'Chinese Academy of Sciences', 'Series_contact_address': '320 YueYang Road', 'Series_contact_city': 'ShangHai', 'Series_contact_state': '-', 'Series_contact_zip/postal_code': '200032', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE153nnn/GSE153142/suppl/GSE153142_PRMT_inhibitor_RiboSeq_TPM.txt.gz', 'Series_citation': 'Huan-Huan Wei, Xiao-Juan Fan, Yue Hu, Xiao-Xu Tian, Meng Guo, Miao-Wei Mao, Zhao-Yuan Fang, Ping Wu, Shuai-Xin Gao, Chao Peng, Yun Yang, and Zefeng Wang. A systematic survey of PRMT interactomes reveals the key roles of arginine methylation in the global control of RNA splicing and translation. Science Bulletin 2021; Volume 66, Issue 13, Pages 1342-1357. doi:10.1016/j.scib.2021.01.004', 'Series_platform_id': 'GPL20795', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP268623'}\n",
      "390 PRJNA643292 GSE153597 {'Series_title': 'GIGYF2 and 4EHP Inhibit Translation Initiation of Defective Messenger RNAs to Assist Ribosome-Associated Quality Control', 'Series_geo_accession': 'GSE153597', 'Series_status': 'Public on Jul 01 2020', 'Series_submission_date': 'Jun 30 2020', 'Series_last_update_date': 'Aug 04 2020', 'Series_pubmed_id': '32726578', 'Series_summary': 'Ribosome-associated Quality Control (RQC) pathways protect cells from toxicity caused by incomplete protein products resulting from translation of damaged or problematic mRNAs. Extensive work in yeast has identified highly conserved mechanisms that lead to the degradation of the faulty mRNA and partially synthesized polypeptide. Here, we used CRISPR-Cas9-based screening to search for additional RQC strategies in mammals. We found that failed translation leads to specific inhibition of translation initiation on that message. This negative feedback loop is mediated by two translation inhibitors, GIGYF2 and 4EHP. Both model substrates and growth-based assays established that inhibition of additional rounds of translation acts in concert with known RQC pathways to prevent buildup of toxic proteins. Inability to block translation of faulty mRNAs, and subsequent accumulation of partially synthesized polypeptides, could explain the neurodevelopmental and neuropsychiatric disorders observed in mice and humans with compromised GIGYF2 function.', 'Series_overall_design': 'This datasets consists of paired ribosome profiling and RNA-seq datasets collected in replicate after the CRISPRi-mediated knockdown of NEMF, ZNF598, EIF4E2, GIGYF2, and non-targeting negative controls.', 'Series_type': 'Other', 'Series_contributor': 'Kamena,,Kostova', 'Series_sample_id': 'GSM4647810', 'Series_contact_name': 'Joseph,,Replogle', 'Series_contact_email': 'joseph.replogle@ucsf.edu, joseph@wi.mit.edu', 'Series_contact_laboratory': 'Jonathan Weissman Lab, Whitehead Institute Suite 661B', 'Series_contact_institute': 'UCSF / Whitehead Institute / MIT', 'Series_contact_address': '455 Main St', 'Series_contact_city': 'Cambridge', 'Series_contact_state': 'Massachussetts', 'Series_contact_zip/postal_code': '02142', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE153nnn/GSE153597/suppl/GSE153597_RAW.tar', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP269441'}\n",
      "391 PRJNA644594 GSE153945 {'Series_title': 'Persistent activation of mRNA translation by transient Hsp90 inhibition', 'Series_geo_accession': 'GSE153945', 'Series_status': 'Public on Aug 11 2020', 'Series_submission_date': 'Jul 07 2020', 'Series_last_update_date': 'May 11 2023', 'Series_pubmed_id': '32783929', 'Series_summary': 'The heat shock protein 90 (Hsp90) chaperone functions as a protein-folding buffer and plays a unique role promoting the evolution of new heritable traits. To better understand how Hsp90 can affect mRNA translation we screened more than 1600 factors involved in mRNA regulation for physical interactions with Hsp90 in human cells. The mRNA binding protein CPEB2 strongly binds Hsp90 via its prion domain. In a yeast model, transient inhibition of Hsp90 resulted in persistent activation of a CPEB translation reporter even in the absence of exogenous CPEB that persisted for 30 generations after the inhibitor was removed. Ribosomal profiling revealed that some endogenous yeast mRNAs, including HAC1, show a persistent change in translation efficiency following transient Hsp90 inhibition. Thus, transient loss of Hsp90 function can promote a non-genetic inheritance of a translational state affecting specific mRNAs, introducing a new mechanism by which Hsp90 can promote phenotypic variation.', 'Series_overall_design': 'Ribosome-profiling data examining heat shock and Hsp90 inhibition in yeast.', 'Series_type': 'Other', 'Series_contributor': 'Susan,,Lindquist', 'Series_sample_id': 'GSM4658970', 'Series_contact_name': 'Timothy,J,Eisen', 'Series_contact_email': 'timeisen@berkeley.edu', 'Series_contact_laboratory': 'John Kuriyan', 'Series_contact_department': 'QB3', 'Series_contact_institute': 'UC Berkeley', 'Series_contact_address': 'Stanley Hall', 'Series_contact_city': 'Berkeley', 'Series_contact_state': 'CALIFORNIA', 'Series_contact_zip/postal_code': '94720', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE153nnn/GSE153945/suppl/GSE153945_RAW.tar', 'Series_citation': 'Tsvetkov, P., Eisen, T. J., Heinrich, S. U., Brune, Z., Hallacli, E.,  Newby, G. A., Kayatekin, C., Pincus, D., & Lindquist, S. (2020). Persistent Activation of mRNA Translation by Transient Hsp90 Inhibition. Cell Reports, Activation of mRNA Translation by Transient Hsp90 Inhibition. Cell Reports,  32(6), 108001. https://doi.org/10.1016/j.celrep.2020.108001.', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP270667'}\n",
      "392 PRJNA645135 GSE154106 {'Series_title': 'Ribosome profiling of SMN-primed ribosomes and active ribosome profiling from brains of a mouse model of severe SMA', 'Series_geo_accession': 'GSE154106', 'Series_status': 'Public on Aug 11 2020', 'Series_submission_date': 'Jul 09 2020', 'Series_last_update_date': 'Nov 10 2020', 'Series_pubmed_id': '32958857', 'Series_summary': 'Keywords: motor neuron disease, spinal muscular atrophy, SMA, SMN, SMN1, survival of motor neuron, SMN-primed ribosome profiling, translation, RiboLace, Ribo-Seq, active translation, ribosome heterogeneity', 'Series_overall_design': 'We performed ribosome profiling of SMN-primed ribosomes in brain extracts from control mice at P5 (Smn+/-; SMN2tg/0). Experiments were performed in biological triplicate. In parallel, samples from immunoprecipitation with IgG were used as controls for possible unspecific isolation of ribosome protected fragments. Additionally, we performed active ribosome profiling with RiboLace (PMID: 30355487) and ribosome profiling in SMA early symptomatic mouse brains (Smn-/-; SMN2tg/0). Experiments were performed in biological duplicate. Ribosome profiling of age-matched controls were deposited under accessions GSE102354 (RiboLace) and GSE102318 (ribosome profiling).', 'Series_type': 'Other', 'Series_contributor': 'Gabriella,,Viero', 'Series_sample_id': 'GSM4663998', 'Series_contact_name': 'Toma,,Tebaldi', 'Series_contact_email': 'toma.tebaldi@yale.edu, toma.tebaldi@unitn.it', 'Series_contact_laboratory': 'Laboratory of RNA and Disease Data Science (RDDS)', 'Series_contact_department': 'CIBIO & Department of Internal Medicine, Yale Cancer Center', 'Series_contact_institute': 'Yale University & University of Trento', 'Series_contact_address': '300 George Street', 'Series_contact_city': 'New Haven, US & Trento, Italy', 'Series_contact_state': 'CT', 'Series_contact_zip/postal_code': '06511', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE154nnn/GSE154106/suppl/GSE154106_SMN_SMA_rpf_table.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP271117'}\n",
      "393 PRJNA646470 GSE154488 {'Series_title': 'Global Translational Landscape of the Candida albicans Morphological Transition', 'Series_geo_accession': 'GSE154488', 'Series_status': 'Public on Dec 10 2020', 'Series_submission_date': 'Jul 15 2020', 'Series_last_update_date': 'Feb 16 2021', 'Series_pubmed_id': '33585865', 'Series_web_link': 'https://gsajournals.figshare.com/articles/dataset/Supplemental_Material_for_Mundodi_et_al_2020/13173215/1', 'Series_summary': 'Candida albicans is a major human fungal pathogen that represents the fourth leading cause of hospital-acquired bloodstream infections in the U.S. and is associated with high mortality and/or morbidity rates in a wide variety of immunocompromised individuals, including cancer and AIDS patients. While the C. albicans morphological transition from yeast to filamentous cells is required for virulence, considerably little is known about translational mechanisms important for controlling this transition as well as other virulence-related processes in C. albicans and other human fungal pathogens. Using ribosome profiling, we report the first global translational profile associated with C. albicans morphogenesis. Strikingly, many genes involved in pathogenesis, filamentation, response to stress and cell wall organization show reduced translational efficiency (TE). Several of these genes are known to be strongly induced at the transcriptional level, suggesting that a translational fine-tuning mechanism is in place. Using a recently developed ORF-calling method, we have identified a significant number of potential uORFs in genes associated with pathogenesis and at least 57 potential novel ORFs, several of which appear to show altered TE in response to filamentation. Finally, using a novel method for global analysis of ribosome pausing from Ribo-seq data, we demonstrate an enrichment of ribosome pausing sites in C. albicans genes associated with protein synthesis and cell wall functions. Altogether, our results suggest that the C. albicans morphological transition is associated with widespread global translational alterations that do not simply reflect transcriptional changes and affect the expression of many genes associated with virulence-related processes and pathogenesis.', 'Series_overall_design': 'Examination of gene expression by RNA-seq and Ribo-seq in 1 strain of Candida albicans grown under 2 conditions  (YEPD + serum at 37°C (filament-inducing) and YEPD at 30°C (non filament-inducing)). There are three biological replicates for each strain grown under each condition (12 samples total).', 'Series_type': 'Other', 'Series_contributor': 'Saket,,Choudhary', 'Series_sample_id': 'GSM4672131', 'Series_contact_name': 'David,,Kadosh', 'Series_contact_email': 'kadosh@uthscsa.edu', 'Series_contact_phone': '(210) 567-3976', 'Series_contact_department': 'Department of Microbiology, Immunology & Molecular Genetics', 'Series_contact_institute': 'University of Texas Health Science Center at San Antonio', 'Series_contact_address': '7703 Floyd Curl Dr., MC: 7758', 'Series_contact_city': 'San Antonio', 'Series_contact_state': 'Texas', 'Series_contact_zip/postal_code': '78229', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE154nnn/GSE154488/suppl/GSE154488_ribo_rna_counts.tsv.gz', 'Series_platform_id': 'GPL27827', 'Series_platform_taxid': '5476', 'Series_sample_taxid': '5476', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP272074'}\n",
      "394 PRJNA646476 GSE154491 {'Series_title': 'Identification of translated small open reading frames in lymphocytes', 'Series_geo_accession': 'GSE154491', 'Series_status': 'Public on Apr 28 2021', 'Series_submission_date': 'Jul 15 2020', 'Series_last_update_date': 'May 15 2021', 'Series_pubmed_id': '33970232', 'Series_summary': 'Open reading frames (ORFs) are the genomic DNA sequences that have the potential to be translated. Genome annotation pipelines dismiss translation products of small ORFs (smORFs) of less than 100 codons (<300 nucleotides) as being unlikely to have a biological function. In this study, we systematically characterized smORFs in mouse B and T cells under different conditions and predicted a total of 5744 unique actively translated smORFs. We then extended our analysis to ORFs of 101-200 codons in length and predicted 945 of such longer translation products. Additionally, our results have suggested the existence of candidate secreted micropeptides. Furthermore, verifying their existence and identifying their functions will be essential and potentially lead to useful applications.', 'Series_overall_design': 'Peripheral lymph nodes were isolated from 5 biological replicates of control C57BL/6 mice (Zfp36l1 floxed/floxed without Cre). Each sample was split to generate Ribosome profiling  and total RNA-seq libraries. Ribosome profiling data is submitted here; the corresponding total RNA-seq datasets are available in GSE146073.', 'Series_type': 'Other', 'Series_contributor': 'Martin,,Turner', 'Series_sample_id': 'GSM4672169', 'Series_contact_name': 'Louise,,Matheson', 'Series_contact_email': 'louise.matheson@babraham.ac.uk', 'Series_contact_department': 'Laboratory of Lymphocyte Signalling and Development', 'Series_contact_institute': 'The Babraham Institute', 'Series_contact_address': 'Babraham Research Campus', 'Series_contact_city': 'Cambridge', 'Series_contact_zip/postal_code': 'CB22 3AT', 'Series_contact_country': 'United Kingdom', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE154nnn/GSE154491/suppl/GSE154491_Ribo_CDS_counts.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP272078'}\n",
      "395 PRJNA648109 GSE154992 {'Series_title': 'Combined analysis of ribo-seq and rna-seq reveals that Mettl3 regulates embryonic cerebral cortex development in mice by controlling the proliferation and differentiation of neural progenitor cells', 'Series_geo_accession': 'GSE154992', 'Series_status': 'Public on Jan 04 2021', 'Series_submission_date': 'Jul 23 2020', 'Series_last_update_date': 'Jul 27 2021', 'Series_pubmed_id': '34262022', 'Series_summary': 'N6-methyladenosine (m6A) is the most important internal messenger RNA (mRNA) modification in mammals. The role of Mettl3, the core component of methyltransferase, and Fto, the demethylase, has not been revealed in the embryonic development of the cerebral cortex in mice. We constructed the conditional knockout of Mettl3 and Fto in the cerebral cortex of mice. We found that Mettl3 deletion causes cell cycle extension of progenitors and massive proliferation of intermediate progenitors at E15.5, which lead to a fold structure of cortex, while Fto deletion hardly altered cell numbers or morphology. Meanwhile, both transcriptome sequence (RNA-Seq) and ribosome profiling (Ribo-Seq) of Mettl3-deleted cortex mRNAs in E15.5 revealed that the differential genes were enriched in cell differentiation, neurogenesis, neuron differentiation, cell proliferation, and cell cycle. Moreover, deleted Mettl3 but not Fto significantly upregulated Ythdf1 and Ythdf2, which caused a substantial change in gene translation efficiency. Our findings indicate cortex m6A represents a novel layer of complexity in gene expression regulation and provide a new mechanism of precisely programmed RNA maturation.', 'Series_overall_design': 'RNA-SEQ and Ribo-SEQ of the cerebral cortex of Emx1-Cre: Mettl3flox/flox and Emx1-Cre: Ftoflox/flox mice at E15.5', 'Series_type': 'Other', 'Series_contributor': 'Tao,,Sun', 'Series_sample_id': 'GSM4692804', 'Series_contact_name': 'kunzhao,,du', 'Series_contact_email': 'Dukunzhao@stu.hqu.edu.cn', 'Series_contact_institute': 'huaqiao university', 'Series_contact_address': 'jimei', 'Series_contact_city': 'xiamen', 'Series_contact_zip/postal_code': '361021', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE154nnn/GSE154992/suppl/GSE154992_Ribo-seq_read_count.txt.gz', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP273291'}\n",
      "396 PRJNA648316 GSE155036 {'Series_title': 'MazF toxin causes alterations in Staphylococcus aureus transcriptome, translatome and proteome that underlie bacterial dormancy', 'Series_geo_accession': 'GSE155036', 'Series_status': 'Public on Dec 28 2020', 'Series_submission_date': 'Jul 24 2020', 'Series_last_update_date': 'Mar 10 2021', 'Series_pubmed_id': '33544858', 'Series_summary': 'Bacterial antibiotic resistance is as a serious health problem. Antibiotic resistance appears either because of mutations or as a result of a bacteria dormant state without heritable genetic change. This non-growing state allows bacteria to survive antibiotic treatment. The mechanisms of entrance to the bacterial dormant state are unknown. It has been suggested that toxin-antitoxin systems (TASs) are possible controlling factors for cell dormancy. In Staphylococcus aureus, the role of TASs genome-wide and their link to the dormancy induction mechanisms has not been investigated in detail. In this study, we analyzed the role of MazF toxin on transcriptome, translatome and proteome of S. aureus using RNA-Seq, Ribo-Seq and quantitative proteomics. We characterized the correlation between transcription, translation, and protein levels, and demonstrated that the MazF endonuclease decreases translation directly by cleaving mRNA, and indirectly, by decreasing translation factors and by promoting ribosome hibernation. Thus, MazF represses transcription and translation of many genes rather than a particular set of genes. Nevertheless, several specific pathways affected by MazF were identified: we demonstrated that cell wall thickness is increased and cell division is decreased upon MazF induction. MazF cleaves mRNA in vivo, creating stop-less transcripts and stalled ribosomes. These stalled ribosomes are rescued by SsrA-system, which is activated upon MazF induction. Finally, we described the overall impact of MazF on S. aureus metabolism, and propose one of the mechanisms by which MazF may induce bacterial dormancy.', 'Series_overall_design': 'Samples in dupicates RNA-Seq and Ribo-Seq of WT with pRAB11 vector, ΔmazEF with pRAB11, and ΔmazEF with pRAB11 vector expressing inducible MazF. All samples submited to RNA-Seq and Ribo-Seq.', 'Series_type': 'Other', 'Series_contributor': 'Olesya,,Panasenko', 'Series_sample_id': 'GSM4694012', 'Series_contact_name': 'Fedor,,Bezrukov', 'Series_contact_email': 'fedor.bezrukov@gmail.com', 'Series_contact_department': 'Department of Physics and Astronomy', 'Series_contact_institute': 'The University of Manchester', 'Series_contact_address': 'Brunswick Street, Schuster laboratory', 'Series_contact_city': 'Manchester', 'Series_contact_zip/postal_code': 'M13 9PL', 'Series_contact_country': 'United Kingdom', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE155nnn/GSE155036/suppl/GSE155036_rnaseq_rawcounts.csv.gz', 'Series_platform_id': 'GPL26273', 'Series_platform_taxid': '1280', 'Series_sample_taxid': '1280', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP273404'}\n",
      "397 PRJNA648380 GSE155073 {'Series_title': 'Translation and codon usage regulate Argonaute slicer activity to trigger germline small RNA biogenesis [Ribo-Seq]', 'Series_geo_accession': 'GSE155073', 'Series_status': 'Public on Apr 22 2021', 'Series_submission_date': 'Jul 24 2020', 'Series_last_update_date': 'Jul 08 2021', 'Series_pubmed_id': '34108460', 'Series_summary': 'In Caenorhabditis elegans germline, thousands of mRNAs are concomitantly expressed with antisense 22G-RNAs, which are loaded into the Argonaute CSR-1. Despite their essential functions for animal fertility and embryonic development, how CSR-1 22G-RNAs are produced remain unknown. Here, we show that CSR-1 slicer activity is primarily involved in triggering the synthesis of small RNAs on the coding sequences of germline mRNAs and post-transcriptionally regulates a fraction of targets. CSR-1-cleaved mRNAs prime the RNA-dependent RNA polymerase, EGO-1 to synthesize 22G-RNAs concurrently with ribosome translation in the cytoplasm, in contrast to other 22G-RNAs mostly synthesized in germ-granules. Moreover, differences in germline mRNA codon optimality and ribosome occupancy antagonistically affect the abundance of CSR-1 22G-RNAs and the stability of their mRNA targets. We propose that codon usage differences encoded into mRNA sequences might be a conserved strategy adopted in eukaryotes to regulate small RNA biogenesis and Argonaute targeting.', 'Series_overall_design': \"14 samples were analyzed by Ribo-seq, including replicates (sample name contains 'rep').\", 'Series_type': 'Other', 'Series_contributor': 'Germano,,Cecere', 'Series_sample_id': 'GSM4694776', 'Series_contact_name': 'Germano,,Cecere', 'Series_contact_email': 'germano.cecere@pasteur.fr', 'Series_contact_phone': '0033140613225', 'Series_contact_laboratory': 'Mechanisms of Epigenetic Inheritance', 'Series_contact_department': 'Development and stem cell biology', 'Series_contact_institute': 'Institut Pasteur', 'Series_contact_address': 'Institut Pasteur, 28 Rue Du Docteur Roux, Batiment Monod, 4eme Etage', 'Series_contact_city': 'Paris', 'Series_contact_zip/postal_code': '75724', 'Series_contact_country': 'France', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE155nnn/GSE155073/suppl/GSE155073_RAW.tar', 'Series_platform_id': 'GPL19757', 'Series_platform_taxid': '6239', 'Series_sample_taxid': '6239', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP273441'}\n",
      "398 PRJNA648398 GSE155087 {'Series_title': 'Multiomics analysis couples mRNA turnover and translational control of glutamine metabolism to the differentiation of the activated CD4+ T cell', 'Series_geo_accession': 'GSE155087', 'Series_status': 'Public on Apr 28 2021', 'Series_submission_date': 'Jul 24 2020', 'Series_last_update_date': 'Nov 18 2022', 'Series_pubmed_id': '36385275', 'Series_summary': 'The ZFP36 family of RNA-binding proteins acts post-transcriptionally to repress translation and promote RNA decay. Studies of genes and pathways regulated by the ZFP36 family in CD4+ T cells have focussed largely on cytokines, but their impact on metabolic reprogramming and differentiation is unclear. Using CD4+ T cells lacking Zfp36 and Zfp36l1, we combined the quantification of mRNA transcription, stability, abundance and translation with crosslinking immunoprecipitation and metabolic profiling to determine how they regulate T cell metabolism and differentiation. Our results suggest that ZFP36 and ZFP36L1 act directly to limit the expression of genes driving anabolic processes by two distinct routes: by targeting transcription factors and by targeting transcripts encoding rate-limiting enzymes. These enzymes span numerous metabolic pathways including glycolysis, one-carbon metabolism and glutaminolysis. Direct binding and repression of transcripts encoding glutamine transporter SLC38A2 correlated with increased cellular glutamine content in ZFP36/ZFP36L1-deficient T cells. Increased conversion of glutamine to α-ketoglutarate in these cells was consistent with direct binding of ZFP36/ZFP36L1 to Gls (encoding glutaminase) and Glud1 (encoding glutamate dehydrogenase). We propose that ZFP36 and ZFP36L1 as well as glutamine and α-ketoglutarate are limiting factors for the acquisition of the cytotoxic CD4+ T cell fate. Our data implicate ZFP36 and ZFP36L1 in limiting glutamine anaplerosis and differentiation of activated CD4+ T cells, likely mediated by direct binding to transcripts of critical genes that drive these processes.', 'Series_overall_design': 'mRNA-seq (3 biological replicates), nascent, 4sU-labelled mRNA-seq (2 biological replcates, separated into labelled [nascent RNA], unlabelled [pre-existing RNA] and total fractions) and Ribo-seq (5 biological replicates) were performed in activated CD4+ T cells from control and Zfp36/Zfp36l1 double conditional knockout mice. To identify direct targets of ZFP36-family proteins, iCLIP was performed on activated CD4+ T cells from WT C57BL/6 mice (3 biological replicates), using two independent antibodies that bind to ZFP36L1.', 'Series_type': 'Other', 'Series_contributor': 'Martin,,Turner', 'Series_sample_id': 'GSM4694996', 'Series_contact_name': 'Louise,,Matheson', 'Series_contact_email': 'louise.matheson@babraham.ac.uk', 'Series_contact_department': 'Laboratory of Lymphocyte Signalling and Development', 'Series_contact_institute': 'The Babraham Institute', 'Series_contact_address': 'Babraham Research Campus', 'Series_contact_city': 'Cambridge', 'Series_contact_zip/postal_code': 'CB22 3AT', 'Series_contact_country': 'United Kingdom', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE155nnn/GSE155087/suppl/GSE155087_mRNA_raw_counts.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP273454'}\n",
      "399 PRJNA649003 GSE155233 {'Series_title': 'Ribo-Seq timecourse in RAW264.7 cells upon LPS treatment', 'Series_geo_accession': 'GSE155233', 'Series_status': 'Public on Jun 23 2021', 'Series_submission_date': 'Jul 27 2020', 'Series_last_update_date': 'Sep 08 2021', 'Series_pubmed_id': '34480152', 'Series_summary': 'mRNAs are generally assumed to be loaded instantly with ribosomes upon entry into the cytoplasm. To measure ribosome density on nascent mRNA, we developed nascent Ribo-Seq (nRibo-Seq) by combining Ribo-Seq with progressive 4-thiouridine labelling. In mouse macrophages, we experimentally determined, for the first time, the lag between the appearance of nascent RNA and its association with ribosomes, which was calculated to be 20 - 22 min for bulk mRNA, and approximated the time it takes for mRNAs to be fully loaded with ribosomes to be 41 - 44 min. Notably, ribosomal loading time is adapted to gene function as rapid loading was observed with highly regulated genes. The lag and ribosomal loading time correlate positively with ORF size and half-life, and negatively with tRNA adaptation indices. Similar results were obtained in mESCs, where the lag between IN and FP was even more pronounced with 35 - 38 min. We validated our measurements after stimulation of macrophages with lipopolysaccharide, where the lag between cytoplasmic and translated mRNA leads to a corresponding uncoupling between input and ribosome-protected fragments. Uncoupling is stronger for mRNAs with long ORFs or half-lives, a finding we also confirmed at the level of protein production by nascent chain proteomics. As a consequence of the lag in ribosome loading, ribosome density measurements are distorted when performed under conditions where mRNA levels are far from steady state expression, and transcriptional changes affect ribosome density in a passive way. This study uncovers an unexpected and considerable lag in ribosome loading, and provides guidelines for the interpretation of Ribo-Seq data taking passive effects on ribosome density into account.', 'Series_overall_design': 'Limited RNaseI digest or alkaline fragmentation of cytoplasmic input mRNA to asses ribosome density in RAW264.7 cells upon LPS treatment', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Georg,,Stoecklin', 'Series_sample_id': 'GSM4697844', 'Series_contact_name': 'Johanna,Daniela,Schott', 'Series_contact_email': 'johanna.schott@medma.uni-heidelberg.de', 'Series_contact_laboratory': 'Stoecklin lab', 'Series_contact_department': 'Mannheim Institute for Innate Immunoscience', 'Series_contact_institute': 'Heidelberg University', 'Series_contact_address': 'Ludolf-Krehl-Str. 13-17', 'Series_contact_city': 'Mannheim', 'Series_contact_zip/postal_code': '68167', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE155nnn/GSE155233/suppl/GSE155233_RiboSeq_LPS.csv.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP273780'}\n",
      "400 PRJNA649449 GSE155350 {'Series_title': 'Coupled protein synthesis and ribosome-guided piRNA processing on mRNAs', 'Series_geo_accession': 'GSE155350', 'Series_status': 'Public on Sep 07 2021', 'Series_submission_date': 'Jul 29 2020', 'Series_last_update_date': 'Nov 03 2021', 'Series_pubmed_id': '34645830', 'Series_summary': \"piRNAs can derive from transposable elements (TEs), long non-coding RNAs, or the 3'UTR of mRNAs, the last being the least characterized out of the three piRNA classes. In this study, we systematically defined the regulation, biogenesis and function of the 3'UTR piRNAs that are expressed throughout mouse spermatogenesis. These piRNAs are regulated by A-MYB transcription factors, and their precursors are bifunctional mRNAs which are processed into piRNAs after pioneer rounds of translation. This dataset includes high-throughput sequencing data that have been used in this study.\", 'Series_overall_design': 'High-throughput sequencing data including RNAseq, small RNa-seq, Ribo-seq and Degradome sequencing of mouse and rooster testis. All sequencing data were sequenced on Illumina platform.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Jihong,,Zheng', 'Series_sample_id': 'GSM5640071', 'Series_contact_name': 'Xin,,Li', 'Series_contact_email': 'Xin_Li@urmc.rochester.edu', 'Series_contact_phone': '(585) 275-6576', 'Series_contact_laboratory': 'Li Lab', 'Series_contact_department': 'Biochemistry and Biophysics', 'Series_contact_institute': 'University of Rochester', 'Series_contact_address': '601 Elmwood Ave', 'Series_contact_city': 'Rochester', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '14623', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE155nnn/GSE155350/suppl/GSE155350_RAW.tar', 'Series_platform_id': 'GPL16133', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP274105'}\n",
      "401 PRJNA649751 GSE155432 {'Series_title': 'Extension of mRNA poly(A) tails and 3′UTRs during neuronal differentiation exhibits variable association with post-transcriptional dynamics', 'Series_geo_accession': 'GSE155432', 'Series_status': 'Public on Jun 09 2023', 'Series_submission_date': 'Jul 30 2020', 'Series_last_update_date': 'Sep 10 2023', 'Series_pubmed_id': '37293985', 'Series_summary': 'Differentiation of neural progenitor cells into mature neuronal phenotypes relies on extensive temporospatial coordination of mRNA expression to support the development of functional brain circuitry. Cleavage and polyadenylation of mRNA has tremendous regulatory capacity through the alteration of mRNA stability and modulation of microRNA (miRNA) function, however the extent of utilization in neuronal development is currently unclear. Here, we employed poly(A) tail sequencing, mRNA sequencing, ribosome profiling and small RNA sequencing to explore the functional relationship between mRNA abundance, translation, poly(A) tail length, alternative polyadenylation (APA) and miRNA expression in an in vitro model of neuronal differentiation. Differential analysis revealed a strong bias towards poly(A) tail and 3´UTR lengthening during differentiation, both of which were associated with changes in mRNA abundance but not translation. Globally, changes in miRNA expression were predominantly associated with mRNA abundance and translation, however several miRNA-mRNA pairings with potential to regulate poly(A) tail length were identified. Furthermore, 3´UTR lengthening was observed to significantly increase the inclusion of non-conserved miRNA binding sites, potentially enhancing the regulatory capacity of these molecules in mature neuronal cells. Together, our findings suggest poly(A) tail length and APA function as part of a rich post-transcriptional regulatory matrix during neuronal differentiation.', 'Series_overall_design': 'Examination of changes in mRNA expression, translation, poly(A) tail length, alternative polyadenylation and miRNA expression during neuronal differentiation of SH-SY5Y cells.', 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'Murray,,Cairns', 'Series_sample_id': 'GSM5711746', 'Series_contact_name': 'Dylan,,Kiltschewskij', 'Series_contact_email': 'c3146470@uon.edu.au', 'Series_contact_laboratory': 'Molecular Neurobiology', 'Series_contact_department': 'School of Biomedical Sciences and Pharmacy', 'Series_contact_institute': 'The University of Newcastle', 'Series_contact_address': 'Room MS616, Level 6, Medical Sciences Building, University Drive', 'Series_contact_city': 'Callaghan', 'Series_contact_state': 'NSW', 'Series_contact_zip/postal_code': '2308', 'Series_contact_country': 'Australia', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE155nnn/GSE155432/suppl/GSE155432_ribo.cpm.txt.gz', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP274313'}\n",
      "402 PRJNA649776 GSE155447 {'Series_title': 'RNA-seq and Ribo-seq analysis of the effects of METTL3/14 depletion on gene expression during the interferon response in Huh7 cells', 'Series_geo_accession': 'GSE155447', 'Series_status': 'Public on Feb 19 2021', 'Series_submission_date': 'Jul 30 2020', 'Series_last_update_date': 'Apr 06 2021', 'Series_pubmed_id': '33657363', 'Series_summary': \"The RNA modification N6-methyladenosine (m6A) regulates gene expression through various transcript-specific effects. The overall goal of these experiments was to determine the effects of the m6A methyltransferase complex proteins METTL3 and METTL14 on the expression of interferon stimulated genes. First, we depleted METTL3 and METTL14 in Huh7 cells using siRNAs and treated with Mock or IFN-β to analyze the regulatory effects of METTL3/14 on interferon-stimulated genes' transcript abundance. We then used Ribo-seq following METTL3/14 siRNA depletion and IFN treatment to quantify its effect on the translation of ISGs and other genes. Overall, we found that METTL3/14 exerts transcript-specific regulatory effects on a subset of interferon-stimulated genes. Overall, METTL3/14 appeared to enhance the translation of a subset of interferon-stimulated genes. We characterized the molecular effects of METTL3/14 and m6A on certain ISGs and determined that m6A augments the antiviral effects of the interferon response.\", 'Series_overall_design': 'siRNA transfections were performed for 36 hours in Huh7 cells prior to IFN-β treatment (50 units/mL) for 8 hours, after which cells were harvested and RNA-seq or Ribo-seq protocols were performed. For RNA-seq, 3 replicates of each condition were performed and for Ribo-seq, 4 replicates for each condition were performed.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Stacy,,Horner', 'Series_sample_id': 'GSM4704577', 'Series_contact_name': 'Christopher,E,Mason', 'Series_contact_email': 'chm2042@med.cornell.edu', 'Series_contact_laboratory': 'Christopher Mason Lab', 'Series_contact_department': 'Physiology and Biophysics', 'Series_contact_institute': 'Weill Cornell Medical College', 'Series_contact_address': '1305 York Ave', 'Series_contact_city': 'New York', 'Series_contact_zip/postal_code': '10065', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE155nnn/GSE155447/suppl/GSE155447_RNAseq_CTRL_MOCK_vs_M314_MOCK_countMatrix.txt.gz', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP274360'}\n",
      "403 PRJNA655417 GSE155727 {'Series_title': 'Transcriptome-wide analysis of mRNA translation and associated post-transcriptional regulatory mechanisms following neuronal membrane depolarisation', 'Series_geo_accession': 'GSE155727', 'Series_status': 'Public on Sep 25 2020', 'Series_submission_date': 'Aug 05 2020', 'Series_last_update_date': 'Nov 02 2020', 'Series_pubmed_id': '32992958', 'Series_summary': 'Here, we employed ribosome profiling to explore global changes in mRNA translational status in neuronally differentiated neuroblast cultures at several time-points after whole cell depolarisation, and compared this to steady-state mRNA levels. Immediately after depolarisation, genes with known function at the synapse were subjected to a surge of translational activity without corresponding changes in mRNA levels. At later time-points, however, these changes became synchronised, suggesting there are different layers of post-transcriptional regulation which are temporally separated but become coordinated over time. Sequencing of small RNAs additionally revealed that changes in microRNA (miRNA) and tRNA-derived small RNA fragments (tRFs) coincided with this early post-transcriptional disparity, however these molecules exhibited strongest association with the remodelling of mRNA steady-state levels. Transcripts displaying high levels of translational dynamics relative to underlying mRNA levels were characterised by more intrinsic features, including mRNA length, GC content, secondary structure and motifs for RNA binding proteins. These data suggest that activity-associated neuronal translation is initially regulated independently of mRNA levels, wherein transcript features contribute to modulation of mRNA translatability, while miRNA and tRFs fine-tune mRNA levels.', 'Series_overall_design': 'Examination of changes in mRNA translation, expression and small RNA expression in neuronally differentiated SH-SY5Y cells.', 'Series_type': 'Other', 'Series_contributor': 'Murray,,Cairns', 'Series_sample_id': 'GSM4711320', 'Series_contact_name': 'Dylan,,Kiltschewskij', 'Series_contact_email': 'c3146470@uon.edu.au', 'Series_contact_laboratory': 'Molecular Neurobiology', 'Series_contact_department': 'School of Biomedical Sciences and Pharmacy', 'Series_contact_institute': 'The University of Newcastle', 'Series_contact_address': 'Room MS616, Level 6, Medical Sciences Building, University Drive', 'Series_contact_city': 'Callaghan', 'Series_contact_state': 'NSW', 'Series_contact_zip/postal_code': '2308', 'Series_contact_country': 'Australia', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE155nnn/GSE155727/suppl/GSE155727_cpm.trna.r.0h.2h.txt.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP276055'}\n",
      "404 PRJNA655835 GSE155854 {'Series_title': 'DAP5 enables main ORF translation on mRNAs with structured and uORF-containing 5’ leaders', 'Series_geo_accession': 'GSE155854', 'Series_status': 'Public on Nov 07 2022', 'Series_submission_date': 'Aug 07 2020', 'Series_last_update_date': 'Jan 04 2023', 'Series_pubmed_id': '36473845', 'Series_summary': 'Half of mammalian transcripts contain short upstream open reading frames (uORFs) that potentially regulate translation of the downstream coding sequence (CDS). The molecular mechanisms governing these events remain poorly understood. Here, we find that the non-canonical initiation factor Death-associated protein 5 (DAP5 or eIF4G2) is required for translation initiation on select transcripts. Using ribosome profiling and luciferase-based reporters coupled with mutational analysis we show that DAP5-dependent translation occurs on messenger RNAs (mRNAs) with long, structure-prone 5′ leader sequences and persistent uORF translation. These mRNAs preferentially code for signalling factors such as kinases and phosphatases. We also report that cap/eIF4F- and eIF4A-dependent recruitment of DAP5 to the mRNA facilitates main CDS, but not uORF, translation suggesting a role for DAP5 in translation re-initiation. Our study reveals important mechanistic insights into how a non-canonical translation initiation factor involved in stem cell fate shapes the synthesis of specific signalling factors.', 'Series_overall_design': 'Two cell types with two biological replicates analysed using RNA-Seq and Ribo-Seq', 'Series_type': 'Other', 'Series_contributor': 'Cátia,,Igreja', 'Series_sample_id': 'GSM4713006', 'Series_contact_name': 'Ramona,,Weber', 'Series_contact_email': 'ramona.weber@uzh.ch', 'Series_contact_department': 'Institute for Regenerative Medicine', 'Series_contact_institute': 'University of Zurich', 'Series_contact_address': 'Wagistrasse 12', 'Series_contact_city': 'Schlieren', 'Series_contact_zip/postal_code': '8952', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE155nnn/GSE155854/suppl/GSE155854_readcounts_Ribo.txt.gz', 'Series_platform_id': 'GPL21290', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP276619'}\n",
      "405 PRJNA655955 GSE155899 {'Series_title': 'The translational landscape of phenylephrine-treated neonatal rat cardiomyocytes [Ribo-seq]', 'Series_geo_accession': 'GSE155899', 'Series_status': 'Public on Mar 11 2021', 'Series_submission_date': 'Aug 07 2020', 'Series_last_update_date': 'Jun 10 2021', 'Series_pubmed_id': '33677093', 'Series_summary': 'In this study, we generated nucleotide-resolution translatome as well as transcriptome data of isolated primary cardiomyocytes undergoing hypertrophy. More than 10,000 open reading frames (ORF) were detected from the deep sequencing of ribosome protected fragments orchestrating the shift of translatome in hypertrophied cardiomyocytes.We further identified more than 100 potential micropeptides encoded by uncharacterized small open reading frames (sORFs) in the long noncoding RNA genes. Over two-thirds of these micropeptide candidates were experimentally validated in a random test with three micropeptides showing regulatory function in cardiomyocyte hypertrophy via modulating the activities of Oxidative phosphorylation, Calcium signaling pathway, and MAPK signaling pathway.Our study provided a genome-wide overview of the translational controls of cardiomyocyte hypertrophy and demonstrated an unrecognized role of micropeptides in cardiomyocyte biology.', 'Series_overall_design': 'Ribo-seq data presented here.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Zhan-Peng,,Huang', 'Series_sample_id': 'GSM4715428', 'Series_contact_name': 'Youchen,,Yan', 'Series_contact_email': 'yanych8@mail2.sysu.edu.cn', 'Series_contact_institute': 'The First Affiliated Hospital, Sun Yat-sen University', 'Series_contact_address': 'No.58 Zhongshan Road II', 'Series_contact_city': 'Guangzhou', 'Series_contact_state': 'Guangdong', 'Series_contact_zip/postal_code': '510080', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE155nnn/GSE155899/suppl/GSE155899_rat_ORF_raw.counts_matrix.txt.gz', 'Series_platform_id': 'GPL24688', 'Series_platform_taxid': '10116', 'Series_sample_taxid': '10116', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP276711'}\n",
      "406 PRJNA656190 GSE155917 {'Series_title': 'Ribo-seq (Ribosome profiling) analyses in human multiple myeloma cells', 'Series_geo_accession': 'GSE155917', 'Series_status': 'Public on Nov 30 2021', 'Series_submission_date': 'Aug 08 2020', 'Series_last_update_date': 'Nov 30 2021', 'Series_summary': 'Detection of protein translation status at the gene level. Ribo-seq experiment of human multiple myeloma cells including NAT10 overexpression and controls were conducted.', 'Series_overall_design': 'To identify the protein translation level in the  human multiple meloma cells', 'Series_type': 'Other', 'Series_contributor': 'Rongfang,,Wei', 'Series_sample_id': 'GSM4715689', 'Series_contact_name': 'Rong,Fang,Wei', 'Series_contact_email': 'wrf1026@sina.com', 'Series_contact_phone': '17386405180', 'Series_contact_institute': 'Nanjing Unveristy of Chinese Medicine', 'Series_contact_address': '138 Xianlin Road', 'Series_contact_city': 'Nanjing', 'Series_contact_state': 'Jiangsu', 'Series_contact_zip/postal_code': '210023', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE155nnn/GSE155917/suppl/GSE155917_fpkm_TE_ann.txt.gz', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP276951'}\n",
      "407 PRJNA658654 GSE156671 {'Series_title': 'Genome-wide structure changes during neuronal differentiation drive gene regulatory networks', 'Series_geo_accession': 'GSE156671', 'Series_status': 'Public on Aug 23 2020', 'Series_submission_date': 'Aug 22 2020', 'Series_last_update_date': 'Aug 25 2020', 'Series_summary': 'RNAs structures play central roles in various cellular processes, and can be remodeled upon binding to cellular factors such as RNA binding proteins (RBP). However, how widespread this remodeling is and whether structural changes can serve as additional regulatory steps for other regulators remains under-explored. Here, we assayed RNA structural dynamics, gene expression, translation and decay during different stages of human neuronal differentiation to understand the impact of RNA structures in stem cell regulation. RBP binding resulted in wide-spread RNA structure changes during early neuronal differentiation.  Structure changes act as a common mechanism for RBPs to modulate the binding of additional RBPs and miRNAs to coordinate gene expression temporally.  In particular, we showed that PUM2 binding on LIN28A resulted in RNA structure changes that enabled miR-30 binding, leading to downregulation of LIN28A. Compensatory mutagenesis experiments that lock specific structural states confirmed the importance of PUM2-induced structure changes in miR-30 regulation. This systematic study broadens our understanding of RNA structural dynamics during human neuronal differentiation and illustrates the wide-spread and complex role of RBPs in regulating RNA structures for gene regulation during human development.', 'Series_overall_design': 'High throughput RNA structure probing experiments in vivo using icSHAPE, as well as RNA sequencing, ribosome profiling and RNA decay experiments in hESCs (Day 0, D0), neuronal precursor cells (Day 7, D7), early neurons (Day 8, D8) and neurons (Day 14, D14).', 'Series_type': 'Other', 'Series_contributor': 'Yue,,Wang', 'Series_sample_id': 'GSM4736566', 'Series_contact_name': 'Yue,,Wan', 'Series_contact_email': 'wany@gis.a-star.edu.sg', 'Series_contact_institute': 'Genome Institute of Singapore', 'Series_contact_address': '60 Biopolis Street', 'Series_contact_city': 'Singapore', 'Series_contact_zip/postal_code': '138672', 'Series_contact_country': 'Singapore', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE156nnn/GSE156671/suppl/GSE156671_RAW.tar', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP278460'}\n",
      "408 PRJNA658965 GSE156749 {'Series_title': 'Upregulation of 5’-terminal oligopyrimidine mRNA translation upon loss of the tumor suppressor ARF', 'Series_geo_accession': 'GSE156749', 'Series_status': 'Public on Dec 20 2020', 'Series_submission_date': 'Aug 24 2020', 'Series_last_update_date': 'Dec 22 2020', 'Series_pubmed_id': '33335292', 'Series_summary': 'Tumor cells require nominal increases in protein synthesis in order to maintain high proliferation rates. As such, tumor cells must acquire enhanced ribosome production. How many of the mutations in tumor cells ultimately achieve this aberrant production is largely unknown. The gene encoding ARF is the most commonly deleted gene in human cancer. ARF plays a significant role in regulating ribosomal RNA synthesis and processing, ribosome export into the cytoplasm, and global protein synthesis. Utilizing ribosome profiling, we show that 5’-terminal oligopyrimidine mRNA translation is upregulated following loss of ARF. Genes with increased translational efficiency following loss of ARF include many ribosomal proteins and translation factors.  Knockout of p53 largely phenocopies ARF loss, with increased protein synthesis and expression of 5’-TOP encoded proteins. The 5’-TOP regulators eIF4G1 and LARP1 are upregulated in ARF and p53 null cells.', 'Series_overall_design': 'RNAseq and Ribosome profiling for three conditions with two replicates for each.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Kyle,A,Cottrell', 'Series_sample_id': 'GSM4743456', 'Series_contact_name': 'Kyle,Aaron,Cottrell', 'Series_contact_email': 'cottrellka@wustl.edu', 'Series_contact_institute': 'Washington University in St. Louis', 'Series_contact_address': '660 S Euclid', 'Series_contact_city': 'St. Louis', 'Series_contact_state': 'MO', 'Series_contact_zip/postal_code': '63110', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE156nnn/GSE156749/suppl/GSE156749_RAW.tar', 'Series_platform_id': 'GPL21493', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP278615'}\n",
      "409 PRJNA659038 GSE156796 {'Series_title': 'Determine the effects of METTL16 on global translation efficiency via ribosome profiling (Ribo-seq)', 'Series_geo_accession': 'GSE156796', 'Series_status': 'Public on Oct 28 2021', 'Series_submission_date': 'Aug 24 2020', 'Series_last_update_date': 'May 15 2023', 'Series_pubmed_id': '35145225', 'Series_web_link': 'https://pubmed.ncbi.nlm.nih.gov/35145225/', 'Series_summary': 'METTL16 is a member of methyltransferase like (METTL) family. Unlike well-studied METTL3 and METTL14, we found a much higher percentage of METTL16 is localized in the cytosol. The subcellular distribution holds the ability to potentiate translation efficiency. Via Far-western blotting and Co-Immunoprecipitation (Co-IP) assays, we have identified the direct interactions between METTL16 and eukaryotic initiation factor 3 (eIF3) a and b. Via cross-linking immunoprecipitation and qPCR (CLIP-qPCR), we have discovered the direct associations between METTL16 and rRNAs. The METTL16-eIF3a/b and METTL16-rRNAs interactions induce the binding between eIF3 and 18S rRNA, promote the formation of 43S preinitiation complex, and eventually expedite translation initiation, the rate-limiting step of translation. To determine the exact effects of METTL16 on translation efficiency, we performed ribosome profiling (Ribo-seq) with HEK293T upon CRISPR-Cas9-induced METTL16 knockout. To guarantee the repeatability and avoid any potential off-target effects, we included 3 distint sgRNAs against METTL16.', 'Series_overall_design': 'The Cas9 single clone of HEK293T cells was infected with lentivirus sgRNAs against METTL16 and the cells were collected at 70-80% confluency after selection with blasticidin and puromycin for at least 1 week. The samples were subjected to ribosome profiling and both the input RNA and footprint RNAs were used for sequencing. Three sgRNAs against METTL16 groups and one sgNS group were included.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Jianjun,,Chen', 'Series_sample_id': 'GSM4744700', 'Series_contact_name': 'Jianjun,,Chen', 'Series_contact_email': 'jianchen@coh.org', 'Series_contact_phone': '6262185591', 'Series_contact_laboratory': 'Chen Lab', 'Series_contact_department': 'Systems Biology', 'Series_contact_institute': 'Beckman Research Institute of City of Hope', 'Series_contact_address': '1218 S. 5th Ave., Room 2062', 'Series_contact_city': 'Monrovia', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '91016', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE156nnn/GSE156796/suppl/GSE156796_ribo.tpm.txt.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP278667'}\n",
      "410 PRJNA660002 GSE157063 {'Series_title': 'Ribosome profiling in DDX3X degron cell lines and covering variants', 'Series_geo_accession': 'GSE157063', 'Series_status': 'Public on Sep 30 2020', 'Series_submission_date': 'Aug 28 2020', 'Series_last_update_date': 'Jul 24 2021', 'Series_pubmed_id': '33905506', 'Series_summary': 'We describe the subset of transcripts that require DDX3 for efficient translation.', 'Series_overall_design': 'Ribosome profiling upon auxin-induced degradation of DDX3X, and covered with either wild-type or mutant DDX3X or wild-type DDX3Y was performed from HCT116 cells.', 'Series_type': 'Other', 'Series_contributor': 'Stephen,,Floor', 'Series_sample_id': 'GSM4751872', 'Series_contact_name': 'Srivats,,Venkataramanan', 'Series_contact_email': 'srivats88@gmail.com', 'Series_contact_phone': '4154763275', 'Series_contact_department': 'Cell and Tissue Biology', 'Series_contact_institute': 'University of California, San Francisco', 'Series_contact_address': '513 Parnassus Ave, HSW 740', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'California', 'Series_contact_zip/postal_code': '94143', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE157nnn/GSE157063/suppl/GSE157063_RAW.tar', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP279211'}\n",
      "411 PRJNA661275 GSE157423 {'Series_title': 'Ribosome profiling analysis of Neuro2a cells with Ebp1 knockdown', 'Series_geo_accession': 'GSE157423', 'Series_status': 'Public on Mar 23 2021', 'Series_submission_date': 'Sep 03 2020', 'Series_last_update_date': 'Jan 06 2023', 'Series_pubmed_id': '36482253', 'Series_summary': 'We perform Ribosome Profiling (Riboseq) analysis of mouse Neuro2a neuronal cultures in Ebp1-siRNA knockdown vs. scrambled-siRNA control conditions in biological triplicate to assess the translation-specific function of Ebp1', 'Series_overall_design': 'Each biological triplicate analyzed for both ribosome-protected mRNA fragments after RNAse digestion (Riboseq), and total steady-state mRNA (RNASeq), followed by high-throughput sequencing.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Matthew,L,Kraushar', 'Series_sample_id': 'GSM4764684', 'Series_contact_name': 'Matthew,Lee,Kraushar', 'Series_contact_email': 'matthew.kraushar@molgen.mpg.de', 'Series_contact_department': 'Department of Computational Molecular Biology', 'Series_contact_institute': 'Max Planck Institute for Molecular Genetics', 'Series_contact_address': 'Ihnestrasse 63-73', 'Series_contact_city': 'Berlin', 'Series_contact_state': 'Berlin', 'Series_contact_zip/postal_code': '14195', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE157nnn/GSE157423/suppl/GSE157423_counts_colrnm.tsv.gz', 'Series_platform_id': 'GPL21103', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP279969'}\n",
      "412 PRJNA661558 GSE157519 {'Series_title': 'Adaptive translational pausing is a hallmark of the cellular response to severe environmental stress', 'Series_geo_accession': 'GSE157519', 'Series_status': 'Public on Sep 01 2021', 'Series_submission_date': 'Sep 04 2020', 'Series_last_update_date': 'Dec 27 2021', 'Series_pubmed_id': '34686314', 'Series_summary': 'Adaptation and survival in response to diverse stressors involve reprogramming of mRNA translation. Although the cellular response to mild stress has been intensively studied, the response to severe stress remains unclear. We show here that under severe stress conditions, cells induce transient hibernation-like mechanisms in anticipation of recovery, which we have termed the Adaptive Pausing Response (APR). The APR is a coordinated cellular response that limits ATP supply and consumption processes via severe mitochondrial fragmentation and inhibits mRNA translation initiation by induction of ribosome pausing at translation initiation codons, respectively. During recovery from severe stress, cells exhibited adaptive ISR signaling, which permitted cell cycle progression, resumption of growth, and reversal of mitochondria fragmentation. We propose that the APR preserves vital elements of cellular function in response to severe environmental stress, and likely plays a major role in homeostasis and pathogenesis.', 'Series_overall_design': 'Cells were treated with the appropriate stressor. Ribo-seq and corresponding RNA-seq experiments were performed.', 'Series_type': 'Other', 'Series_contributor': 'Maria,,Hatzoglou', 'Series_sample_id': 'GSM4769319', 'Series_contact_name': 'Yuanhui,,Mao', 'Series_contact_email': 'maoyuanhui123@gmail.com', 'Series_contact_institute': 'Zhejiang University', 'Series_contact_address': '1369 West Wenyi Road', 'Series_contact_city': 'Hangzhou', 'Series_contact_state': 'Zhejiang', 'Series_contact_zip/postal_code': '310000', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE157nnn/GSE157519/suppl/GSE157519_RAW.tar', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP280144'}\n",
      "413 PRJNA663319 GSE157902 {'Series_title': 'GTPBP1 resolves paused ribosomes to maintain neuronal homeostasis', 'Series_geo_accession': 'GSE157902', 'Series_status': 'Public on Oct 26 2020', 'Series_submission_date': 'Sep 14 2020', 'Series_last_update_date': 'Apr 21 2021', 'Series_pubmed_id': '33186095', 'Series_summary': 'Ribosome-associated quality control pathways respond to defects in translational elongation to recycle arrested ribosomes and degrade aberrant polypeptides and mRNAs. Loss of an individual tRNA gene leads to ribosomal pausing that is resolved by the translational GTPase GTPBP2, and in its absence causes neuron death. Here we show that loss of the homologous protein GTPBP1 during tRNA deficiency in the mouse brain also leads to codon-specific ribosome pausing and neurodegeneration, suggesting that these non-redundant translational GTPases function in the same pathway to mitigate ribosome pausing. Ribosome stalling in the mutant brain led to activation of the integrated stress response (ISR) mediated by GCN2 and decreased mTORC1 signaling. However, in contrast to the ISR, which enhanced neuron survival, reduced mTORC1 signaling increased neuronal death. Our data demonstrate that GTPBP1 functions as an important quality control mechanism during translation elongation and suggest that translational signaling pathways intricately interact to regulate neuronal homeostasis during defective translation elongation.', 'Series_overall_design': '18 samples are in analyzed in this study in the context of 3 week old C57BL/6J (B6J) mouse cerebellum. These include 3 replicates of the following genotypes Gtpbp1 -/-, Gtpbp2 -/- (mediated through a previously characterized nmf205 point mutation), and wildtype mice. RNA-seq and ribosome footprint profiling datasets were generated for each sample.', 'Series_type': 'Other', 'Series_contributor': 'Susan,L,Ackerman', 'Series_sample_id': 'GSM4780432', 'Series_contact_name': 'Scott,,Adamson', 'Series_contact_email': 'scott.adamson@jax.org', 'Series_contact_laboratory': 'Chuang', 'Series_contact_institute': 'The Jackson Laboratory', 'Series_contact_address': '10 Discovery Drive', 'Series_contact_city': 'Farmington', 'Series_contact_state': 'CT', 'Series_contact_zip/postal_code': '06032', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE157nnn/GSE157902/suppl/GSE157902_RAW.tar', 'Series_platform_id': 'GPL21103', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP282337'}\n",
      "414 PRJNA663848 GSE158044 {'Series_title': 'Activation of the antiviral factor RNase L triggers translation of non-coding mRNA sequences', 'Series_geo_accession': 'GSE158044', 'Series_status': 'Public on Jan 18 2021', 'Series_submission_date': 'Sep 16 2020', 'Series_last_update_date': 'Apr 19 2021', 'Series_pubmed_id': '33556964', 'Series_summary': 'Ribonuclease L (RNase L) is activated as part of the innate immune response and plays an important role in the clearance of viral infections. When activated, it endonucleolytically cleaves both viral and host RNAs, leading to a global reduction in protein synthesis. However, it remains unknown how widespread RNA decay, and consequent changes in the translatome, promote the elimination of viruses. To study how this altered transcriptome is translated, we assayed the global distribution of ribosomes in RNase L activated human cells with ribosome profiling. We found that RNase L activation leads to a substantial increase in the fraction of translating ribosomes in ORFs internal to coding sequences (iORFs) and ORFs within 5’ and 3’ UTRs (uORFs and dORFs). Translation of these alternative ORFs was dependent on RNase L’s cleavage activity, suggesting that mRNA decay fragments are translated to produce short peptides that may be important for antiviral activity.', 'Series_overall_design': '30 Ribsome profiling datasets on 2-5A or poly I:C treated or untreated A549 lung carcinoma cells.', 'Series_type': 'Other', 'Series_contributor': 'Nicholas,R,Guydosh', 'Series_sample_id': 'GSM4787185', 'Series_contact_name': 'Nicholas,R,Guydosh', 'Series_contact_email': 'nicholas.guydosh@nih.gov', 'Series_contact_laboratory': 'LBG', 'Series_contact_department': 'NIDDK', 'Series_contact_institute': 'National Institutes of Health', 'Series_contact_address': '8 Center Dr, Bldg 8, 220', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '20892', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE158nnn/GSE158044/suppl/GSE158044_RAW.tar', 'Series_platform_id': 'GPL21290', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP282584'}\n",
      "415 PRJNA664109 GSE158136 {'Series_title': 'Ribo-seq data from NIH 3T3 cells lacking eIF4G2', 'Series_geo_accession': 'GSE158136', 'Series_status': 'Public on Oct 19 2020', 'Series_submission_date': 'Sep 17 2020', 'Series_last_update_date': 'Jan 26 2022', 'Series_pubmed_id': '35018467', 'Series_summary': 'Eukaryotic translation initiation factor (eIF) 4G2 is a long-known homologue of the canonical eIF4G1. However, unlike the latter, it does not bind eIF4E or PABP, thus it remains unclear what and when brings eIF4G2 onto a ribosome. Here we report results of  ribosome profiling performed in NIH 3T3 eIF4G2 (DAP5) -/- cells.', 'Series_overall_design': 'Sequencing of ribosome-protected mRNA fragments and total RNA in\\xa0wild-type NIH 3T3 and eIF4G2(-/-) NIH 3T3 cells', 'Series_type': 'Other', 'Series_contributor': 'Daria,S,Nogina', 'Series_sample_id': 'GSM4793046', 'Series_contact_name': 'Ivan,V,Kulakovskiy', 'Series_contact_institute': 'Institute of Protein Research', 'Series_contact_address': 'Institutskaya 4', 'Series_contact_city': 'Pushchino', 'Series_contact_zip/postal_code': '142290', 'Series_contact_country': 'Russia', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE158nnn/GSE158136/suppl/GSE158136_readcounts.tsv.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP282901'}\n",
      "416 PRJNA664116 GSE158141 {'Series_title': 'Ribosome Profiling with CNOT1 depletion', 'Series_geo_accession': 'GSE158141', 'Series_status': 'Public on Aug 23 2021', 'Series_submission_date': 'Sep 17 2020', 'Series_last_update_date': 'Jan 09 2022', 'Series_pubmed_id': '34615539', 'Series_summary': 'Ribosome profiling in HEK293 cells with and without the depletion of CNOT1. This allows the determination of translational efficiency and nucleotide resolution of ribosome occupancy.', 'Series_overall_design': \"The control ribosome profiling data has also previously been used in Wilczynska et al. eIF4A2 drives repression of translation at initiation by Ccr4-Not through purine-rich motifs in the 5'UTR. Genome Biol 2019 Dec 2;20(1):262. PMID: 31791371 (GSE134517)\", 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Sarah,L,Gillen', 'Series_sample_id': 'GSM4793184', 'Series_contact_name': 'Sarah,Louise,Gillen', 'Series_contact_institute': 'Beatson Institute for Cancer Research', 'Series_contact_address': 'Beatson Institute, Garscube Estate, Switchback Road, Bearsden', 'Series_contact_city': 'Glasgow', 'Series_contact_zip/postal_code': 'G611BD', 'Series_contact_country': 'United Kingdom', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE158nnn/GSE158141/suppl/GSE158141_RibosomeProfiling_DESeq2results.csv.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP282906'}\n",
      "417 PRJNA665035 GSE158374 {'Series_title': 'Genes with 5′ terminal oligopyrimidine tracts preferentially escape global suppression of translation by the SARS-CoV-2 NSP1 protein', 'Series_geo_accession': 'GSE158374', 'Series_status': 'Public on Oct 01 2020', 'Series_submission_date': 'Sep 22 2020', 'Series_last_update_date': 'Sep 08 2021', 'Series_pubmed_id': '34127534', 'Series_summary': 'Viruses rely on the host translation machinery to synthesize their own proteins. Consequently, they have evolved varied mechanisms to co-opt host translation for their survival. SARS-CoV-2 relies on a non-structural protein, NSP1, for shutting down host translation. Despite this, it is currently unknown how viral proteins and host factors critical for viral replication can escape a global shutdown of host translation. Here, using a novel FACS-based assay called MeTAFlow, we report a dose-dependent reduction in both nascent protein synthesis and mRNA abundance in cells expressing NSP1. We perform RNA-Seq and matched ribosome profiling experiments to identify gene-specific changes both at the mRNA expression and translation level. We discover a functionally-coherent subset of human genes preferentially translated in the context of NSP1 expression. These genes include the translation machinery components, RNA binding proteins, and others important for viral pathogenicity. Importantly, we also uncover potential mechanisms of preferential translation through the presence of shared sites for specific RNA binding proteins and a remarkable enrichment for 5′ terminal oligo-pyrimidine tracts. Collectively, the present study suggests fine tuning of host gene expression and translation by NSP1 despite its global repressive effect on host protein synthesis.', 'Series_overall_design': 'Effects of NSP1 expression in translation and transcription in HEK293 cells.', 'Series_type': 'Other', 'Series_contributor': 'Can,,Cenik', 'Series_sample_id': 'GSM5342786', 'Series_contact_name': 'Hakan,,Ozadam', 'Series_contact_laboratory': 'Cenik Lab', 'Series_contact_department': 'Department of Molecular Biosciences', 'Series_contact_institute': 'University of Texas at Austin', 'Series_contact_address': 'Department of Molecular Biosciences Moffett Molecular Biology Building 2.122 2500 Speedway', 'Series_contact_city': 'Austin', 'Series_contact_state': 'TX', 'Series_contact_zip/postal_code': '78712', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE158nnn/GSE158374/suppl/GSE158374_manuscript.HEK293.ribo.hdf5', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP284977'}\n",
      "418 PRJNA666167 GSE158669 {'Series_title': 'BCDIN3D RNA methyltransferase stimulates Aldolase C expression and glycolysis through let-7 microRNA in breast cancer cells', 'Series_geo_accession': 'GSE158669', 'Series_status': 'Public on Apr 14 2021', 'Series_submission_date': 'Sep 28 2020', 'Series_last_update_date': 'Apr 14 2021', 'Series_pubmed_id': '33664453', 'Series_summary': 'BCDIN3D is an RNA modifying enzyme that methylates specific precursor microRNAs and tRNAHis. In addition to breast cancer, BCDIN3D may also be linked to metabolism, as its gene locus is associated with obesity and T2D. In order to uncover metabolic pathways regulated by BCDIN3D in cancer, we performed an unbiased analysis of the metabolome, transcriptome, and proteome of breast cancer cells depleted for BCDIN3D. Intersection of these analyses showed that BCDIN3D-depleted cells have increased levels of Fructose 1,6 Bisphosphate (F1,6-BP), the last six-carbon glycolytic intermediate accompanied by reduced glycolytic capacity.', 'Series_overall_design': 'We performed combined RNA-Seq, miRNA-seq, and ribosome profiling in shBCDIN3D MDA-MB-231 breast cancer cells', 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'Blerta,,Xhemalçe', 'Series_sample_id': 'GSM4805353', 'Series_contact_name': 'Blerta,,Xhemalce', 'Series_contact_institute': 'University of Texas at Austin', 'Series_contact_address': '2500 Speedway', 'Series_contact_city': 'Austin', 'Series_contact_state': 'Texas', 'Series_contact_zip/postal_code': '78712', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE158nnn/GSE158669/suppl/GSE158669_sRNA-counts.txt.gz', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP285644'}\n",
      "419 PRJNA666888 GSE158881 {'Series_title': 'Ribosome profiling and RNA-seq of wild-type hippocampal slices', 'Series_geo_accession': 'GSE158881', 'Series_status': 'Public on Oct 02 2020', 'Series_submission_date': 'Oct 01 2020', 'Series_last_update_date': 'Jan 04 2021', 'Series_pubmed_id': '33054857', 'Series_summary': 'Metabotropic glutamate receptor-mediated long-term depression (mGluR-LTD) is a form of synaptic plasticity that is dysregulated in multiple mouse models of autism, including fragile X syndrome and tuberous sclerosis.', 'Series_overall_design': 'Ribosome profiling and RNA-seq of hippocampal slices from wild-type mice before and after DHPG-mGluR stimulation', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Joel,D,Richter', 'Series_sample_id': 'GSM4813857', 'Series_contact_name': 'Annie,,Hien', 'Series_contact_email': 'annie.hien@umassmed.edu', 'Series_contact_laboratory': 'Richter Lab', 'Series_contact_department': 'Molecular Medicine', 'Series_contact_institute': 'UMass Medical School Worcester', 'Series_contact_address': '373 Plantation Street', 'Series_contact_city': 'Worcester', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '01605', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE158nnn/GSE158881/suppl/GSE158881_ca1.wt.dhpg.rep1-8.rna.rsem.gene.summary.counts.txt.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP286016'}\n",
      "420 PRJNA667051 GSE158930 {'Series_title': 'Translational landscape of SARS-CoV-2 and infected cells', 'Series_geo_accession': 'GSE158930', 'Series_status': 'Public on Nov 23 2020', 'Series_submission_date': 'Oct 02 2020', 'Series_last_update_date': 'Jul 07 2022', 'Series_pubmed_id': '35604092', 'Series_summary': 'Conclusions: Our study represents the first detailed analysis of viral and host translational landscape in infected cells and demonstrate that translation of host mRNAs involved in innate immunity is specifically blocked. The optimized data analysis workflows reported here should provide a framework for translational profiling studies in other settings.', 'Series_overall_design': 'Vero E6 and primary human bronchial epithelial cells were infected with SARS-CoV-2 at varying multiplicities of infection and followed throughout early and late phases of infection.  Parallel samples were processed for RNA-seq and Ribo-seq. Sequencing reads were cleaned off of adapters and rRNAs. Reads were first mapped to the SARS-CoV-2 genome and then to the African green monkey and human genomes using STAR. Mapped reads were further annotated and processed using publicly available software and custom scripts deposited at Github. DE gene expression analysis was performed usign EdgeR. Gene set enrichment analysis was done using TcGSA.', 'Series_type': 'Other', 'Series_contributor': 'Sebla,B,Kutluay', 'Series_sample_id': 'GSM5494638', 'Series_contact_name': 'Sebla,Bulent,Kutluay', 'Series_contact_email': 'kutluay@wustl.edu', 'Series_contact_phone': '3147478908', 'Series_contact_institute': 'Washington University School of Medicine', 'Series_contact_address': '660 S. Euclid Avenue, Campus Box 8230', 'Series_contact_city': 'Saint Louis', 'Series_contact_state': 'MO', 'Series_contact_zip/postal_code': '63110', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE158nnn/GSE158930/suppl/GSE158930_RAW.tar', 'Series_platform_id': 'GPL28836', 'Series_platform_taxid': '60711', 'Series_sample_taxid': '60711', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP286182'}\n",
      "421 PRJNA667952 GSE159210 {'Series_title': 'Relaxed initiation pausing of ribosomes drives oncogenic translation', 'Series_geo_accession': 'GSE159210', 'Series_status': 'Public on Dec 29 2020', 'Series_submission_date': 'Oct 07 2020', 'Series_last_update_date': 'Mar 30 2021', 'Series_pubmed_id': '33597240', 'Series_summary': 'Using quantitative profiling of initiating ribosomes, we found that ribosomal pausing at the start codon serves as a “brake” to restrain the translational output. In response to oncogenic RAS signaling, the initiation pausing relaxes and contributes to the increased translational flux. Intriguingly, mRNA m6A modification in the vicinity of start codons influences the behavior of initiating ribosomes. Under oncogenic RAS signaling, the reduced mRNA methylation leads to relaxed initiation pausing, thereby promoting malignant transformation and tumor growth. Restored initiation pausing by inhibiting m6A demethylases suppresses RAS-mediated oncogenic translation and subsequent tumorigenesis. Our findings unveil a new paradigm of translational control that is co-opted by RAS mutant cancer cells to drive malignant phenotypes.', 'Series_overall_design': 'We took advantage of a human TtH cell line stably expressing a 4-hydroxytamoxifen (4-OHT)-inducible ER:HRASG12V fusion protein. QTI-seq, Ribo-seq, RNA-seq, m6A-seq and corresponding experiments were performed on the ER:HRASG12V cells with or without 4-OHT treatment and METTL3 or FTO knockdown.', 'Series_type': 'Other', 'Series_contributor': 'Shu-Bing,,Qian', 'Series_sample_id': 'GSM4822852', 'Series_contact_name': 'Yuanhui,,Mao', 'Series_contact_email': 'maoyuanhui123@gmail.com', 'Series_contact_institute': 'Zhejiang University', 'Series_contact_address': '1369 West Wenyi Road', 'Series_contact_city': 'Hangzhou', 'Series_contact_state': 'Zhejiang', 'Series_contact_zip/postal_code': '310000', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE159nnn/GSE159210/suppl/GSE159210_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP286677'}\n",
      "422 PRJNA668327 GSE159300 {'Series_title': 'Translation-dependent and independent mRNA decay occur through mutually exclusive pathways that are defined by ribosome density during T Cell activation [Ribo-Seq]', 'Series_geo_accession': 'GSE159300', 'Series_status': 'Public on Mar 01 2024', 'Series_submission_date': 'Oct 09 2020', 'Series_last_update_date': 'May 24 2024', 'Series_pubmed_id': '38508694', 'Series_summary': 'Post-transcriptional control is crucial for regulating protein expression, both basally and in response to extracellular cues. Proper signal transduction requires tight control of both response induction and termination. One way protein expression might be attenuated is by targeting mRNAs to translation-dependent degradation (TDD), thus making any increase in protein expression self-limiting. However, the extent to which TDD is a general mechanism for limiting protein expression is currently unknown. Here we describe a comprehensive analysis of basal and signal-induced TDD in mouse primary CD4 T cells. Our data indicate that most cellular transcripts are decayed to some extent in a translation-dependent manner, both in resting and activated cells. Analysis of transcript features revealed that 3’UTR length and ribosome density are major determinants of the magnitude of TDD as well as GC content and amino acid identity. Consistently, upon T cell activation, all transcripts that undergo changes in ribosome density display a corresponding change in their level of TDD. Surprisingly, the amplitude of translation-independent mRNA decay appears as a mirror image of TDD. Moreover, translation-independent decay also responds to changes in ribosome density upon T cell activation but in the opposite direction to those observed for TDD. Our data demonstrate a strong interconnection between mRNA translation and decay in mammalian cells. Furthermore they indicate that ribosome loading is a major determinant of the pathway by which transcripts are degraded within cells', 'Series_overall_design': 'Primary macrophages and T-cell from mice were activated or not and incubated with an transcription (5,6-Dichloro-1-β-D-ribofuranosylbenzimidazole or Triptolide) and translation (Cycloheximide or Harringtonine) during 0, 1 or 3h to monitor translation dependent and indenpendent decay of mRNA', 'Series_type': 'Other', 'Series_contributor': 'Emiliano,P,Ricci', 'Series_sample_id': 'GSM4826322', 'Series_contact_name': 'Emmanuel,,Labaronne', 'Series_contact_email': 'elabaronne@gmail.com', 'Series_contact_laboratory': 'RNA Metabolism in Immunity and Infection (RMI2)', 'Series_contact_institute': 'LBMC', 'Series_contact_address': \"46 allée d'Italie\", 'Series_contact_city': 'Lyon', 'Series_contact_zip/postal_code': '69007', 'Series_contact_country': 'France', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE159nnn/GSE159300/suppl/GSE159300_RiboProf_counts_stats.df.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP286875'}\n",
      "423 PRJNA668372 GSE159329 {'Series_title': 'Sensing of individual stalled 80S ribosomes by Fap1 for non-functional rRNA turnover. Li et al', 'Series_geo_accession': 'GSE159329', 'Series_status': 'Public on Sep 15 2022', 'Series_submission_date': 'Oct 09 2020', 'Series_last_update_date': 'Sep 29 2022', 'Series_pubmed_id': '36113412', 'Series_summary': 'Cells can respond to stalled ribosomes by sensing ribosome collisions and employing quality control pathways. How ribosome stalling is resolved without collisions, however, has remained elusive. Here, focusing on non-colliding stalling exhibited by decoding-defective ribosomes, we identified Fap1 as a stalling sensor triggering 18S non-functional rRNA decay via poly-ubiquitination of uS3. Ribosome profiling revealed an enrichment of Fap1 at the translation initiation site but also association with elongating individual ribosomes. Cryo-EM structures of Fap1-bound ribosomes elucidated Fap1 probing the mRNA simultaneously at both the entry and exit channels suggesting a mRNA stasis sensing activity, and Fap1 sterically hinders formation of canonical collided di-ribosomes. Our findings indicate that individual stalled ribosomes are the potential signal for ribosome dysfunction, leading to accelerated turnover of the ribosome itself.', 'Series_overall_design': '20 ribosome profiling (ribo-seq) samples are included in the study. Ribo-seq samples were obtained from (1) total cell lysates (input) collected prior to purification, (2) MS2-purified ribosomes carrying exogenously encoded wild type or mutant rRNA, or (3) co-purified ribosomes from pull-downs of proteins of interest (Mag2, Fap1, or Yil161w). Input samples serve as controls that are compared against purified ribosomes. Each sample has a biological replicate.', 'Series_type': 'Other', 'Series_contributor': 'Toshifumi,,Inada', 'Series_sample_id': 'GSM4826759', 'Series_contact_name': 'Sihan,,Li', 'Series_contact_email': 'li-sihan@g.ecc.u-tokyo.ac.jp', 'Series_contact_department': 'The Institute of Medical Science', 'Series_contact_institute': 'The University of Tokyo', 'Series_contact_address': 'Shirokanedai 4-6-1, Minato-ku', 'Series_contact_city': 'Tokyo', 'Series_contact_zip/postal_code': '108-8639', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE159nnn/GSE159329/suppl/GSE159329_Riboseq_position.txt.gz', 'Series_platform_id': 'GPL26171', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP286916'}\n",
      "424 PRJNA668384 GSE159336 {'Series_title': 'Host-informed molecular engineering improves antigen quality and enables integrated manufacturing of a trivalent subunit vaccine [Ribo-seq]', 'Series_geo_accession': 'GSE159336', 'Series_status': 'Public on May 05 2021', 'Series_submission_date': 'Oct 09 2020', 'Series_last_update_date': 'May 05 2021', 'Series_pubmed_id': '33933073', 'Series_summary': 'We analyzed total RNA and ribosome footprints collected from a K. phaffii strain secreting the VP8-P[6] serotype of the rotavirus antigen and compared against a strain secreting the VP8-P[4] serotype for reference.', 'Series_overall_design': 'K. phaffii strains expressing either VP8-P[4] or VP8-P[6] were analyzed for total mRNA expression and ribosome protected mRNA quantification. All strains were sampled following biomass accumulation in glycerol-containing medium for 24 hr followed by induction of protein expression by growth in methanol-containing medium for an additional 16 hr.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'J,C,Love', 'Series_sample_id': 'GSM4826877', 'Series_contact_name': 'Charles,Arthur,Whittaker', 'Series_contact_email': 'charliew@mit.edu', 'Series_contact_institute': 'Koch Institute', 'Series_contact_address': '77 Mass Ave 76-189', 'Series_contact_city': 'Cambridge', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02152', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE159nnn/GSE159336/suppl/GSE159336_jrbrady_RNARibo_l2tpm.txt.gz', 'Series_platform_id': 'GPL23539', 'Series_platform_taxid': '460519', 'Series_sample_taxid': '460519', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP286925'}\n",
      "425 PRJNA673613 GSE160564 {'Series_title': 'Ribosome profiling of wild-type and NSUN2 knockout flies', 'Series_geo_accession': 'GSE160564', 'Series_status': 'Public on Oct 30 2023', 'Series_submission_date': 'Oct 31 2020', 'Series_last_update_date': 'Oct 30 2023', 'Series_summary': 'We performed ribosome profiling for wild-type and NSUN2 knockout flies.', 'Series_overall_design': 'Ovaries of wild-type and NSUN2 knockout flies were collected.', 'Series_type': 'Other', 'Series_contributor': 'Wanying,,Chen', 'Series_sample_id': 'GSM4875438', 'Series_contact_name': 'Rui,,Zhang', 'Series_contact_email': 'zhangrui3@mail.sysu.edu.cn', 'Series_contact_institute': 'Sun Yat-sen University', 'Series_contact_address': 'No. 135, Xingang Xi Road', 'Series_contact_city': 'Guangzhou', 'Series_contact_zip/postal_code': '510275', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE160nnn/GSE160564/suppl/GSE160564_RAW.tar', 'Series_platform_id': 'GPL21306', 'Series_platform_taxid': '7227', 'Series_sample_taxid': '7227', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP290539'}\n",
      "426 PRJNA674837 GSE160917 {'Series_title': 'Ribosome profiling and mRNA-seq of muscle skeletal muscle and DROSHA KO 293T cells', 'Series_geo_accession': 'GSE160917', 'Series_status': 'Public on Dec 31 2020', 'Series_submission_date': 'Nov 05 2020', 'Series_last_update_date': 'Jan 02 2021', 'Series_summary': 'We conducted ribosome profiling and mRNA-seq using mouse skeletal muscle tissues and 293T cells (NTC control and DROSHA KO) with Hiseq 2500', 'Series_overall_design': 'Examination of translation status of mouse skeletal muscle and the differences between NTC and DROSHA KO in 293T cells', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Tadashi,,Yamamoto', 'Series_sample_id': 'GSM4886547', 'Series_contact_name': 'Hiroaki,,Sako', 'Series_contact_institute': 'Waseda University', 'Series_contact_address': '2-579-15', 'Series_contact_city': 'Tokorozawa', 'Series_contact_zip/postal_code': '359-1192', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE160nnn/GSE160917/suppl/GSE160917_RAW.tar', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP291315'}\n",
      "427 PRJNA676161 GSE161301 {'Series_title': 'RNA-Seq and Ribo-Seq analysis of contol and  METTL3 inducible shRNA LNCaP lines with and without doxycycline induction', 'Series_geo_accession': 'GSE161301', 'Series_status': 'Public on Jun 07 2021', 'Series_submission_date': 'Nov 11 2020', 'Series_last_update_date': 'Sep 06 2021', 'Series_pubmed_id': '34088870', 'Series_summary': 'Genomic and transcriptomic alterations are insufficient to explain the variance in protein expression seen in cancer. Recent evidence has highlighted the role of N6-methyladenosine (m6A) in the regulation of mRNA expression, stability and translation, supporting a potential role for post-transcriptional regulation mediated by m6A in cancer. Here we explore prostate cancer as an exemplar cancer and generate the first prostate m6A maps, and further examined how low levels of N6-adenosine-methyltransferase (METTL3) associates with advanced prostate cancer and results in altered expression at the level of transcription, translation, and protein. In particular extracellular matrix proteins have a high number of m6A sites and show significant changes in expression with METTL3 knock-down. We also discovered the upregulation of a hepatocyte nuclear factor-driven gene signature that is associated with therapy resistance in prostate cancer. Significantly, METTL3 knock-down rendered the cells resistant to androgen receptor antagonists, implicating changes in m6A as a mechanism for therapy resistance in metastatic prostate cancer.', 'Series_overall_design': 'RNA-Seq and Ribo-Seq analysis of contol and  METTL3 inducible shRNA LNCaP lines with and without doxycycline induction.', 'Series_type': 'Other', 'Series_contributor': 'Mark,A,Rubin', 'Series_sample_id': 'GSM4904521', 'Series_contact_name': 'Kellie,,Cotter', 'Series_contact_department': 'Dept for BioMedical Research', 'Series_contact_institute': 'University of Bern', 'Series_contact_address': 'Murtenstrasse 35', 'Series_contact_city': 'Bern', 'Series_contact_state': 'Bern', 'Series_contact_zip/postal_code': '3008', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE161nnn/GSE161301/suppl/GSE161301_rna-seq_STAR_18sample_normCounts.txt.gz', 'Series_platform_id': 'GPL21697', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP292228'}\n",
      "428 PRJNA677870 GSE161319 {'Series_title': 'N7-methylguanosine tRNA modification enhances oncogenic mRNA translation and promotes intrahepatic cholangiocarcinoma progression', 'Series_geo_accession': 'GSE161319', 'Series_status': 'Public on Aug 07 2021', 'Series_submission_date': 'Nov 12 2020', 'Series_last_update_date': 'Aug 09 2021', 'Series_pubmed_id': '34352206', 'Series_summary': 'The cancer cells selectively promote translation of specific oncogenic transcripts to facilitate cancer survival and progression, while the underlying mechanisms are poorly understood. N7-methylguanosine (m7G) tRNA modification and its methyltransferase complex METTL1/WDR4 are significantly up-regulated in intrahepatic cholangiocarcinoma (ICC) and associated with poor prognosis. We developed tRNA reduction and cleavage sequencing (TRAC-Seq) to reveal the m7G tRNA methylome inICC cell line and ribosome nascent-chain complex-bound mRNAs sequencing(RNC-seq) and ribosome profiling(Ribo-seq) to study the differential translated genes and reveal the ribosome pausing. A subset of 22 tRNAs is modified at a ‘RAGGU’ motif within the variable loop. We observe increased ribosome occupancy at the corresponding codons in the Mettl1 knockdown ICC cell line implying widespread effects on tRNA function, ribosome pausing, and mRNA translation. Translation of cell cycle genes and EGFR signaling pathway genes is particularly affected. Our study uncovers the important physiological function and mechanism of METTL1-mediated m7G tRNA modification in the regulation of cancer progression.', 'Series_overall_design': 'TRAC-Seq was developed to identify the tRNA m7G methylome in the intrahepatic cholangiocarcinoma cell line. RNC-seq and Ribo-seq were used to study the differential translated genes in the Mettl1 knockdown and control cells.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Ming,,Kuang', 'Series_sample_id': 'GSM5386907', 'Series_contact_name': 'Haining,,Liu', 'Series_contact_email': 'liuhn6@mail2.sysu.edu.cn', 'Series_contact_phone': '17820523342', 'Series_contact_institute': 'The first affiliated hospital of sun yat-sen university', 'Series_contact_address': 'Zhongshan road 2', 'Series_contact_city': 'Guangzhou', 'Series_contact_zip/postal_code': '510000', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE161nnn/GSE161319/suppl/GSE161319_RiboSeq_reads.txt.gz', 'Series_platform_id': 'GPL30209', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP292287'}\n",
      "429 PRJNA680784 GSE162169 {'Series_title': 'Spurious regulatory connections dictate the expression-fitness landscape of translation factors', 'Series_geo_accession': 'GSE162169', 'Series_status': 'Public on Nov 28 2020', 'Series_submission_date': 'Nov 25 2020', 'Series_last_update_date': 'May 09 2021', 'Series_pubmed_id': '33900014', 'Series_summary': 'During steady-state cell growth, individual enzymatic fluxes can be directly inferred from growth rate by mass conservation, but the inverse problem remains unsolved. Perturbing the flux and expression of a single enzyme could have pleiotropic effects that may or may not dominate the impact on cell fitness. Here, we quantitatively dissect the molecular and global responses to varied expression of translation termination factors (peptide release factors, RFs) in the bacterium Bacillus subtilis. While endogenous RF expression maximizes proliferation, deviations in expression lead to unexpected distal regulatory responses that dictate fitness reduction. Molecularly, RF depletion causes expression imbalance at specific operons, which activates master regulators and detrimentally overrides the transcriptome. Through these spurious connections, RF abundances are thus entrenched by focal points within the regulatory network, in one case located at a single stop codon. Such regulatory entrenchment suggests that predictive bottom-up models of expression-fitness landscapes will require near-exhaustive characterization', 'Series_overall_design': '26 competition experiment amplicon libraries from strain pools, 12 matched Rend-seq and ribosome profiling datasets, one individual Rend-seq dataset, 56 RNA-seq datasets. All data from Bacillus subtilis samples.', 'Series_type': 'Other', 'Series_contributor': 'Gene-Wei,,Li', 'Series_sample_id': 'GSM4944142', 'Series_contact_name': 'Jean-Benoit,,Lalanne', 'Series_contact_email': 'lalannej@uw.edu', 'Series_contact_laboratory': 'Jay Shendure', 'Series_contact_department': 'Genome Sciences', 'Series_contact_institute': 'University of Washington', 'Series_contact_address': '3720 15th Ave NE', 'Series_contact_city': 'Seattle', 'Series_contact_state': 'WA', 'Series_contact_zip/postal_code': '98195', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE162nnn/GSE162169/suppl/GSE162169_RAW.tar', 'Series_platform_id': 'GPL24109', 'Series_platform_taxid': '1423', 'Series_sample_taxid': '1423', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP294161'}\n",
      "430 PRJNA680923 GSE162197 {'Series_title': 'OXPHOS deficiency activates global adaptation pathways to maintain mitochondrial membrane potential', 'Series_geo_accession': 'GSE162197', 'Series_status': 'Public on Nov 27 2020', 'Series_submission_date': 'Nov 26 2020', 'Series_last_update_date': 'Mar 08 2021', 'Series_pubmed_id': '33655635', 'Series_summary': 'Reduction of mitochondrial membrane potential is a hallmark of mitochondrial dysfunction. It activates adaptive responses in organisms from yeast to human to rewire metabolism, remove depolarized mitochondria, and degrade unimported precursor proteins. It remains unclear how cells maintain mitochondrial membrane potential, which is critical for maintaining iron-sulfur cluster (ISC) synthesis, an indispensable function of mitochondria. Here we show that yeast oxidative phosphorylation mutants deficient in complex III, IV, V, and mtDNA respectively, have graded reduction of mitochondrial membrane potential and proliferation rates. Extensive omics analyses of these mutants show that accompanying mitochondrial membrane potential reduction, these mutants progressively activate adaptive responses, including transcriptional downregulation of ATP synthase inhibitor Inh1 and OXPHOS subunits, Puf3-mediated upregulation of import receptor Mia40 and global mitochondrial biogenesis, Snf1/AMPK-mediated upregulation of glycolysis and repression of ribosome biogenesis, and transcriptional upregulation of cytoplasmic chaperones. These adaptations disinhibit mitochondrial ATP hydrolysis, remodel mitochondrial proteome, and optimize ATP supply to mitochondria to convergently maintain mitochondrial membrane potential, ISC biosynthesis, and cell proliferation.', 'Series_overall_design': 'Three biological replicates were prepared for mRNA profiles, and two biological replicates were prepared for ribosome profiling.', 'Series_type': 'Other', 'Series_contributor': 'Hui,,Jiang', 'Series_sample_id': 'GSM4946385', 'Series_contact_name': 'Hui,,Jiang', 'Series_contact_email': 'jianghui@nibs.ac.cn', 'Series_contact_phone': '+86-80726688-8621', 'Series_contact_laboratory': 'Hui Jiang', 'Series_contact_institute': 'National Institute of Biological Sciences', 'Series_contact_address': '7 Science Park Road ZGC Life Science Park', 'Series_contact_city': 'Beijing', 'Series_contact_zip/postal_code': '102206', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE162nnn/GSE162197/suppl/GSE162197_RAW.tar', 'Series_platform_id': 'GPL25517', 'Series_platform_taxid': '580240', 'Series_sample_taxid': '580240', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP294317'}\n",
      "431 PRJNA682206 GSE162556 {'Series_title': 'Mutations in distinct translation-dependent quality control pathways lead to convergent molecular pathogenesis and\\xa0neurodevelopmental defects.', 'Series_geo_accession': 'GSE162556', 'Series_status': 'Public on Mar 15 2021', 'Series_submission_date': 'Dec 02 2020', 'Series_last_update_date': 'May 06 2021', 'Series_pubmed_id': '33899734', 'Series_summary': 'Translation-dependent quality control pathways govern protein synthesis and proteostasis by degrading potentially toxic peptide products derived from aberrant mRNAs. However, little is known about the in vivo defects that arise in the absence of these pathways and how translation is altered in the absence of these pathways. Here, we demonstrate that the ribosome rescue factors Pelo/Hbs1l are both critical for embryonic and neuronal development but dispensable for neuronal survival in the adult brain. Loss of Pelo/Hbs1l in cerebellar progenitors impaired differentiation of these cells. Our analysis of Pelo/Hbs1l deficient fibroblasts revealed translational reprogramming of multiple pathways including mTORC1 signaling likely to restore cellular homeostasis. Interestingly, similar affects on the translatome, signaling pathways and neurogenesis were observed when we conditionally deleted Upf2 to inhibit NMD. These data reveal that translation-dependent quality control pathways, which set out to mitigate errors in translation and to clear defective peptide products and aberrant mRNAs can trigger similar cellular responses and neurodevelopmental abnormalities.', 'Series_overall_design': '41 samples are analyzed in this study which include RNA-seq and Ribosome footprint profiling data from mouse tissues with knockouts of different translation-dependent quality control genes. Profiling datasets include 3 replicates of Hbs1l global knockout from whole mouse embryos with paired wildtype controls. Ribosome profiling datasets also include 3 replicates of Hbs1l conditional knockout in cerebellum with matched wildtype controls. There are also RNA-seq and ribosome footprint profiling for Cre-induced knockouts from mouse embryonic fibroblasts of Pelo, Hbs1l, and Upf2, as well as Cre control samples. There are 3 replicates each with the exception of a Pelo RNA-seq sample which was excluded due to being a biological outlier. The matched Cre controls for profiling datasets are included with knockouts in each batch.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Susan,L,Ackerman', 'Series_sample_id': 'GSM4954383', 'Series_contact_name': 'Scott,,Adamson', 'Series_contact_email': 'scott.adamson@jax.org', 'Series_contact_laboratory': 'Chuang', 'Series_contact_institute': 'The Jackson Laboratory', 'Series_contact_address': '10 Discovery Drive', 'Series_contact_city': 'Farmington', 'Series_contact_state': 'CT', 'Series_contact_zip/postal_code': '06032', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE162nnn/GSE162556/suppl/GSE162556_RAW.tar', 'Series_platform_id': 'GPL21103', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP295558'}\n",
      "432 PRJNA683007 GSE162780 {'Series_title': 'Transcriptome-wide investigation of stop codon readthrough in Saccharomyces cerevisiae', 'Series_geo_accession': 'GSE162780', 'Series_status': 'Public on Dec 08 2020', 'Series_submission_date': 'Dec 07 2020', 'Series_last_update_date': 'May 12 2021', 'Series_pubmed_id': '33878104', 'Series_summary': 'Translation of mRNA into a polypeptide is terminated when the release factor eRF1 recognizes a UAA, UAG, or UGA stop codon in the ribosomal A site and stimulates nascent peptide release. However, stop codon readthrough can occur when a near-cognate tRNA outcompetes eRF1 in decoding the stop codon, resulting in the continuation of the elongation phase of protein synthesis. At the end of a conventional mRNA coding region readthrough allows translation into the mRNA 3’-UTR. Previous studies with reporter systems have shown that the efficiency of termination or readthrough is modulated by cis-acting elements other than stop codon identity, including two nucleotides 5’ of the stop codon, six nucleotides 3’ of the stop codon in the ribosomal mRNA channel, and stem-loop structures in the mRNA 3’-UTR. It is unknown whether these elements are important at a genome-wide level and whether other mRNA features proximal to the stop codon significantly affect termination and readthrough efficiencies in vivo. Accordingly, we carried out ribosome profiling analyses of yeast cells expressing wild-type or temperature-sensitive eRF1 and developed bioinformatics strategies to calculate readthrough efficiency, and to identify mRNA and peptide features which influence that efficiency. We found that the stop codon (nt +1 to +3), the nucleotide after it (nt +4), the codon in the P site (nt -3 to -1), and 3’-UTR length are the most influential features in the control of readthrough efficiency, while nts +5 to +9 and mRNA secondary structure in the 3’-UTR had milder effects. Additionally, we found low readthrough genes to have shorter 3’-UTRs compared to high readthrough genes in cells with thermally inactivated eRF1, while this trend was reversed in wild-type cells. Together, our results demonstrated the general roles of known regulatory elements in genome-wide regulation and identified several new mRNA or peptide features important for translation termination and readthrough.', 'Series_overall_design': \"Translation profiles of the WT and sup45-2 strains were generated by ribosome profiling in duplicates using Illumina NextSeq 500. Readthrough efficiency of each gene was defined as the rato of ribosome density of the extension (region between the canonical stop and the first in-frame downstream stop in the 3'-UTR) to ribosome density of the coding region. For each sample, random forest algorithm was used to identify significant predictors of readthrough efficiency..\", 'Series_type': 'Other', 'Series_contributor': 'Allan,,Jacobson', 'Series_sample_id': 'GSM4959800', 'Series_contact_name': 'Kotchaphorn,,Mangkalaphiban', 'Series_contact_email': 'kotchaphorn.mangkalaphiban@umassmed.edu', 'Series_contact_laboratory': 'Jacobson Lab', 'Series_contact_institute': 'University of Massachusetts Medical School', 'Series_contact_address': '368 Plantation St.', 'Series_contact_city': 'Worcester', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '01605', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE162nnn/GSE162780/suppl/GSE162780_sup45_wt_37C_readthrough.txt.gz', 'Series_platform_id': 'GPL27477', 'Series_platform_taxid': '580240', 'Series_sample_taxid': '580240', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP296783'}\n",
      "433 PRJNA689830 GSE164308 {'Series_title': 'Landscape of transcriptional and translational changes over 22 years of bacterial adaptation', 'Series_geo_accession': 'GSE164308', 'Series_status': 'Public on Jan 12 2021', 'Series_submission_date': 'Jan 05 2021', 'Series_last_update_date': 'Nov 30 2022', 'Series_pubmed_id': '36214449', 'Series_summary': 'We use RNA-seq and Ribo-seq to analyze expression changes in the Long Term Evolution Experiment. By comparing expression levels between the ancestral strains and each of the evolved lines, we find that expression changes are largely parallel after 22 years of evolution, that these changes occur in specific pathways, and that known mutations may be responsible for some of the observed expression changes.', 'Series_overall_design': '2 biological replicates each of RNA-seq and Ribo-seq of 13 E. coli cell lines. AraM and AraP refer to the 6 evolved lines of the ancestral REL606 and REL607 strains, respectively. Each file contains the data for the indicated biological replicate (rep1 or rep2), the indicated sequencing method (RNA-seq or ribo-seq), and the indicated cell lines (Ara-1-6 or Ara+1-6 and the corresponding ancestor). Code for processing the data can be found at https://github.com/shahlab/LTEE-gene-expression.', 'Series_type': 'Other', 'Series_contributor': 'Premal,,Shah', 'Series_sample_id': 'GSM5006213', 'Series_contact_name': 'Premal,,Shah', 'Series_contact_department': 'Genetics', 'Series_contact_institute': 'Rutgers University', 'Series_contact_address': '145 Bevier Rd', 'Series_contact_city': 'Piscataway', 'Series_contact_state': 'NJ', 'Series_contact_zip/postal_code': '08854', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE164nnn/GSE164308/suppl/GSE164308_table_s1_read_counts.csv.gz', 'Series_platform_id': 'GPL21222', 'Series_platform_taxid': '562', 'Series_sample_taxid': '562', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP300526'}\n",
      "434 PRJNA692108 GSE164806 {'Series_title': 'Effect of N1-acetylspermidine treatment on the translatome of hair follicle stem cell organoid cultures', 'Series_geo_accession': 'GSE164806', 'Series_status': 'Public on Apr 23 2021', 'Series_submission_date': 'Jan 14 2021', 'Series_last_update_date': 'Jun 28 2021', 'Series_pubmed_id': '33973637', 'Series_summary': \"Stem cell differentiation has been shown to involve an increase in mRNA translation. Amongst others, the rate of translation is influenced by availability of the natural polyamines putrescine, spermidine, and spermine that are essential for cell growth and modulate stem cell maintenance. However, the link between polyamines and translation in cell fate decisions remains obscure. Here, we used hair follicle stem cell (HFSC) organoids to study the role of polyamines as key regulators of mRNA translation in stem cell fate decisions. HFSCs showed lower translation and reduced polyamine levels than progenitor cells. Surprisingly, we found that reduced translation achieved by changed polyamine availability and stemness do not correlate in the organoids. We identified N1-acetylspermidine as regulator of cell fate decisions. To investigate the effect on translation upon N1-AcSpd supplementation, we performed ribosome foot pronting. We confirmed the increase in proliferation identified by 3'RNA-sequencing on the translatome level. N1-AcSpd treatment partially aligns the translatome of progenitor cells to that of HFSCs. Overall, this study delineates the diverse routes of polyamine metabolism-mediated regulation of stem cell fate decisions.\", 'Series_overall_design': 'RNA-Seq / Ribo-Seq of sorted HFSC organoids (alpha6+CD34- and alpha6+CD34+ cells), 8 samples, 2 biological replicates, untreated and treated with N1-acetylspermidine for 72h, samples for Ribo-Seq and RNA-Seq were obtained from the same lysis', 'Series_type': 'Other', 'Series_contributor': 'Martin,,Denzel', 'Series_sample_id': 'GSM5019847', 'Series_contact_name': 'Franziska,,Metge', 'Series_contact_email': 'franziska.metge@age.mpg.de', 'Series_contact_department': 'Bioinformatics Corefacility', 'Series_contact_institute': 'Max Planck Institute for Biology of Ageing', 'Series_contact_address': 'Joseph-Stelzmann-Str 9b', 'Series_contact_city': 'Cologne', 'Series_contact_zip/postal_code': '50931', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE164nnn/GSE164806/suppl/GSE164806_rnaSeq_counts.tsv.gz', 'Series_platform_id': 'GPL24247', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP301728'}\n",
      "435 PRJNA694918 GSE165536 {'Series_title': 'Dynamic changes in tRNA modifications and abundance during T-cell activation [RNA/Ribo-seq]', 'Series_geo_accession': 'GSE165536', 'Series_status': 'Public on Oct 13 2021', 'Series_submission_date': 'Jan 26 2021', 'Series_last_update_date': 'Dec 28 2021', 'Series_pubmed_id': '34642250', 'Series_summary': 'The tRNA pool determines the efficiency, throughput, and accuracy of translation. Previous studies have identified dynamic changes in the tRNA supply and mRNA demand during cancerous proliferation. Yet, dynamic changes may occur also during physiologically normal proliferation, and these are less characterized. We examined the tRNA and mRNA pools of T-cells during their vigorous proliferation and differentiation upon triggering of the T cell antigen receptor. We observe a global signature of switch in demand for codon at the early proliferation phase of the response, accompanied by corresponding changes in tRNA expression levels. In the later phase, upon differentiation of the T cells, the response of the tRNA pool is relaxed back to basal level, potentially restraining excessive proliferation. Sequencing of tRNAs allowed us to also evaluate their diverse base-modifications. We found that two types of tRNA modifications, Wybutosine and ms2t6A, are reduced dramatically during T-cell activation. These modifications occur in the anti-codon loops of two tRNAs that decode “slippery codons”, that are prone to ribosomal frameshifting. Attenuation of these frameshift-protective modifications is expected to increase proteome-wide frameshifting during T-cell proliferation. Indeed, human cell lines deleted of a Wybutosine writer showed increased ribosomal frameshifting, as detected with a reporter that consists of a critical frameshifting site taken from the HIV gag-pol slippery codon motif. These results may explain HIV’s specificity to proliferating T-Cells since it requires ribosomal frameshift exactly on this codon for infection. The changes in tRNA expression and modifications uncover a new layer of translation regulation during T-cell proliferation and exposes a potential trade-off between cellular growth and translation fidelity.', 'Series_overall_design': 'tRNA-sequencing of naïve and in-vitro activated T-cells', 'Series_type': 'Other', 'Series_contributor': 'Yitzhak,,Pilpel', 'Series_sample_id': 'GSM5035556', 'Series_contact_name': 'Roni,,Rak', 'Series_contact_email': 'ronirak@gmail.com', 'Series_contact_phone': '+972547762552', 'Series_contact_laboratory': 'Yitzhak Pilpel', 'Series_contact_department': 'Molecular genetics', 'Series_contact_institute': 'Weizmann institute', 'Series_contact_address': 'Hertzel', 'Series_contact_city': 'Rehovot', 'Series_contact_zip/postal_code': '4725000', 'Series_contact_country': 'Israel', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE165nnn/GSE165536/suppl/GSE165536_RawCount_all.csv.gz', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP303322'}\n",
      "436 PRJNA695008 GSE165592 {'Series_title': 'Loss of N1-Methylation of G37 in tRNA Induces  Ribosome Stalling and Reprograms Gene Expression', 'Series_geo_accession': 'GSE165592', 'Series_status': 'Public on Aug 19 2021', 'Series_submission_date': 'Jan 26 2021', 'Series_last_update_date': 'Aug 31 2021', 'Series_pubmed_id': '34382933', 'Series_summary': 'N1-methylation of G37 is required for a subset of tRNAs to maintain the translational reading-frame. While loss of m1G37 increases ribosomal +1 frameshifting, whether it incurs additional translational defects is unknown. Here we address this question by applying ribosome profiling to gain a genome-wide view of the effects of m1G37 deficiency on protein synthesis. Using E. coli as a model, we show that m1G37 deficiency induces ribosome stalling at codons that are normally translated by m1G37-containing tRNAs. Stalling occurs during decoding of affected codons at the ribosomal A site, indicating a distinct mechanism than that of +1 frameshifting, which occurs after the affected codons leave the A site. Enzyme- and cell-based assays show that m1G37 deficiency reduces tRNA aminoacylation and in some cases peptide-bond formation. We observe changes of gene expression in m1G37 deficiency similar to those in the stringent response that is typically induced by deficiency of amino acids. This work demonstrates a previously unrecognized function of m1G37 that emphasizes its role throughout the entire elongation cycle of protein synthesis, providing new insight into its essentiality for bacterial growth and survival.', 'Series_overall_design': 'Ribosome profiling and RNA-seq E. coli cells to study the effects of loss of the tRNA methylase TrmD either by 1) a degron strategy using ClpXP or 2) a knockout strain where Trm5 is expressed initially and then repressed', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Allen,R,Buskirk', 'Series_sample_id': 'GSM5044543', 'Series_contact_name': 'Allen,R,Buskirk', 'Series_contact_email': 'buskirk@jhmi.edu', 'Series_contact_department': 'Molecular Biology and Genetics', 'Series_contact_institute': 'Johns Hopkins University School of Medicine', 'Series_contact_address': '725 N. Wolfe St', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21205', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE165nnn/GSE165592/suppl/GSE165592_RAW.tar', 'Series_platform_id': 'GPL18956', 'Series_platform_taxid': '511145', 'Series_sample_taxid': '511145', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP303381'}\n",
      "437 PRJNA697913 GSE165782 {'Series_title': 'Ultrasensitive Ribo-seq reveals translational landscapes during mammalian oocyte-to-embryo transition and pre-implantation development', 'Series_geo_accession': 'GSE165782', 'Series_status': 'Public on Apr 19 2022', 'Series_submission_date': 'Jan 29 2021', 'Series_last_update_date': 'Jul 17 2024', 'Series_pubmed_id': '35697785', 'Series_summary': 'Note: RNA-seq data for PN5 zygotes, early two-cell, late two-cell, four-cell and eight-cell stages and ICM were obtained from our previous publication (Zhang et al., Nature, 2016: Allelic reprogramming of the histone modification H3K4me3 in early mammalian development, GSE71434).', 'Series_overall_design': 'By employing Ribo-seq and RNA-seq, we systematically examined the dynamics of translatome and transcriptome of WT oocytes, WT early embryos, Cnot6l/Btg4 KO oocytes and CHX/DRB treated early embryos.', 'Series_type': 'Other', 'Series_contributor': 'Wei,,Xie', 'Series_sample_id': 'GSM6033317', 'Series_contact_name': 'Zhuqing,,Xiong', 'Series_contact_email': 'lexizqxiong@gmail.com', 'Series_contact_institute': 'School of Life Science, Tsinghua Univers', 'Series_contact_address': 'Haidian District', 'Series_contact_city': 'Beijing', 'Series_contact_state': 'FOREIGN', 'Series_contact_zip/postal_code': '100084', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE165nnn/GSE165782/suppl/GSE165782_RAW.tar', 'Series_platform_id': 'GPL29177', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP303825'}\n",
      "438 PRJNA700841 GSE166408 {'Series_title': 'Testing conditions for ribosome profiling in Drosophila melanogaster', 'Series_geo_accession': 'GSE166408', 'Series_status': 'Public on Jan 25 2022', 'Series_submission_date': 'Feb 09 2021', 'Series_last_update_date': 'Feb 14 2022', 'Series_pubmed_id': '35145996', 'Series_summary': 'Comparison of experimental conditions for ribosome profiling.', 'Series_overall_design': 'Ribo-seq of Drosophila melanogaster S2 cells from 4 different conditions: untreated or treated with cycloheximide, each undergoing two different RNase I footprinting conditions.', 'Series_type': 'Other', 'Series_contributor': 'Isabel,,Birds', 'Series_sample_id': 'GSM5070736', 'Series_contact_name': 'Julie,,Aspden', 'Series_contact_department': 'FBS', 'Series_contact_institute': 'University of Leeds', 'Series_contact_address': 'Woodhouse', 'Series_contact_city': 'Leeds', 'Series_contact_zip/postal_code': 'LS2 9JT', 'Series_contact_country': 'United Kingdom', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166408/suppl/GSE166408_RAW.tar', 'Series_platform_id': 'GPL19132', 'Series_platform_taxid': '7227', 'Series_sample_taxid': '7227', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP305462'}\n",
      "439 PRJNA701262 GSE166544 {'Series_title': 'The molecular basis of coupling between poly(A)-tail length and translational efficiency', 'Series_geo_accession': 'GSE166544', 'Series_status': 'Public on Jul 07 2021', 'Series_submission_date': 'Feb 10 2021', 'Series_last_update_date': 'Jul 08 2021', 'Series_pubmed_id': '34213414', 'Series_summary': 'In animal oocytes and early embryos, mRNA poly(A)-tail length strongly influences translational efficiency (TE), but later in development this coupling between tail length and TE disappears. Here, we elucidate how this coupling is first established and why it disappears. Overexpressing cytoplasmic poly(A)-binding protein (PABPC) in frog oocytes specifically improved translation of short-tailed mRNAs, thereby diminishing coupling between tail length and TE. Thus, coupling requires limiting PABPC, implying that in coupled systems longer-tail mRNAs better compete for limiting PABPC. In addition to expressing excess PABPC, post-embryonic cells had two other properties that prevented strong coupling: terminal-uridylation-dependent destabilization of mRNAs lacking bound PABPC, and a regulatory regime wherein PABPC contributes minimally to TE. Thus, these results revealed three fundamental mechanistic requirements for coupling and defined the context-dependent functions for PABPC, in which this protein promotes TE but not mRNA stability in coupled systems and mRNA stability but not TE in uncoupled systems.', 'Series_overall_design': 'RNA-seq, Ribo-seq TAIL-seq, PAL-seq', 'Series_type': 'Other', 'Series_contributor': 'David,,Bartel', 'Series_sample_id': 'GSM5074415', 'Series_contact_name': 'David,,Bartel', 'Series_contact_institute': 'Whitehead Institute', 'Series_contact_address': '455 Main Street', 'Series_contact_city': 'Cambridge', 'Series_contact_zip/postal_code': '02142', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166544/suppl/GSE166544_RAW.tar', 'Series_platform_id': 'GPL18936', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP305686'}\n",
      "440 PRJNA701894 GSE166742 {'Series_title': 'eIF1-eIF4G1 inhibitors uncover alternative translation activation of stress-response genes via enhanced ribosome loading and 5’UTR translation [Ribo-Seq and Ti-Seq]', 'Series_geo_accession': 'GSE166742', 'Series_status': 'Public on Jul 07 2022', 'Series_submission_date': 'Feb 15 2021', 'Series_last_update_date': 'Aug 17 2022', 'Series_pubmed_id': '35857873', 'Series_summary': 'The Ribo-seq and TI-seq analysis following i14G1s (eI1-eIF4G1 inhibitors) treatments uncover opposing roles of eIF1-eIF4G1 and eIF4E-eIF4G1 in scanning-dependent and independent translation. Furthermore, i14G1s inhibition of eIF4G1-eIF1 resulted in translation activation of ER/UPR stress-response genes via enhanced ribosome loading, elevated 5’UTR translation at near cognate AUGs, and unexpected concomitant upregulation of coding-region translation.', 'Series_overall_design': 'Ribo-seq and TI-seq analysis was performed either in HEK293T cells treated with either DMSO (vehicle control), i14G1-10  or i14G1-12.', 'Series_type': 'Other', 'Series_contributor': 'Rivka,,Dikstein', 'Series_sample_id': 'GSM5082082', 'Series_contact_name': 'Dena,,Leshkowitz', 'Series_contact_email': 'dena.leshkowitz@weizmann.ac.il', 'Series_contact_department': 'Bioinformatics Unit, Life Sciences Core Facilities', 'Series_contact_institute': 'Weizmann Institute of Science', 'Series_contact_address': 'P.O.B. 26', 'Series_contact_city': 'Rehovot', 'Series_contact_zip/postal_code': '76100', 'Series_contact_country': 'Israel', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166742/suppl/GSE166742_Ribo-seq_norm_counts.txt.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP306425'}\n",
      "441 PRJNA701895 GSE166743 {'Series_title': 'eIF1-eIF4G1 inhibitors uncover alternative translation activation of stress-response genes via enhanced ribosome loading and 5’UTR translation [MARS-seq]', 'Series_geo_accession': 'GSE166743', 'Series_status': 'Public on Jul 07 2022', 'Series_submission_date': 'Feb 15 2021', 'Series_last_update_date': 'Jul 07 2022', 'Series_summary': 'The Ribo-seq and TI-seq analysis following i14G1s (eI1-eIF4G1 inhibitors) treatments uncover opposing roles of eIF1-eIF4G1 and eIF4E-eIF4G1 in scanning-dependent and independent translation. Furthermore, i14G1s inhibition of eIF4G1-eIF1 resulted in translation activation of ER/UPR stress-response genes via enhanced ribosome loading, elevated 5’UTR translation at near cognate AUGs, and unexpected concomitant upregulation of coding-region translation.', 'Series_overall_design': 'Ribo-seq and TI-seq analysis was performed either in HEK293T cells treated with either DMSO (vehicle control), i14G1-10  or i14G1-12.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Rivka,,Dikstein', 'Series_sample_id': 'GSM5082094', 'Series_contact_name': 'Dena,,Leshkowitz', 'Series_contact_email': 'dena.leshkowitz@weizmann.ac.il', 'Series_contact_department': 'Bioinformatics Unit, Life Sciences Core Facilities', 'Series_contact_institute': 'Weizmann Institute of Science', 'Series_contact_address': 'P.O.B. 26', 'Series_contact_city': 'Rehovot', 'Series_contact_zip/postal_code': '76100', 'Series_contact_country': 'Israel', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166743/suppl/GSE166743_mars-seq_norm_counts.txt.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP306426'}\n",
      "442 PRJNA701951 GSE166783 {'Series_title': 'Ribo-Seq experiments in HCT116 WT and TP53-/- cells upon neocarzinostatin treatment', 'Series_geo_accession': 'GSE166783', 'Series_status': 'Public on Apr 13 2022', 'Series_submission_date': 'Feb 15 2021', 'Series_last_update_date': 'Apr 13 2022', 'Series_pubmed_id': '35388737', 'Series_summary': 'The transcription factor p53 exerts its tumor suppressive effects through transcriptional activation of numerous target genes controlling cell cycle arrest, apoptosis, cellular senescence and DNA repair. In addition, there is evidence that p53 influences the translation of specific mRNAs, including translational attenuation of ribosomal protein synthesis and translational activation of MDM2. A challenge in the analysis of translational control is that changes in mRNA abundance exert a kinetic (passive) effect on ribosome densities. In order to separate these passive effects from active regulation of translation efficiency in response to p53 activation, we conducted a comprehensive analysis of translational regulation by comparative analysis of mRNA levels and ribosome densities upon DNA damage induced by neocarzinostatin in wild-type and TP53-/- HCT116 colorectal carcinoma cells. Thereby, we identified a specific group of mRNAs whose translation is actively up-regulated in response to p53 activation, the majority of which are transcribed from p53 target genes including MDM2, SESN1 and CDKN1A. By subsequent polysome profile analysis, we could demonstrate that a p53-dependent switch in transcript isoform expression is responsible for translational activation of SESN1, whereas translational activation of CDKN1A occurs through a trans-acting mechanism that affects multiple transcript isoforms.', 'Series_overall_design': 'RNaseI digested ribosome footprint of cytoplasmic mRNAs or monosome fraction after sucrose gradient ultracentrifugation, or alkaline fragmentated total mRNA in HCT116 WT and TP53-/- cells upon neocarzinostatin treatment', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Georg,,Stoecklin', 'Series_sample_id': 'GSM5082468', 'Series_contact_name': 'Johanna,Daniela,Schott', 'Series_contact_email': 'johanna.schott@medma.uni-heidelberg.de', 'Series_contact_laboratory': 'Stoecklin lab', 'Series_contact_department': 'Mannheim Institute for Innate Immunoscience', 'Series_contact_institute': 'Heidelberg University', 'Series_contact_address': 'Ludolf-Krehl-Str. 13-17', 'Series_contact_city': 'Mannheim', 'Series_contact_zip/postal_code': '68167', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166783/suppl/GSE166783_Ribo_Seq_HCT116.csv.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP306464'}\n",
      "443 PRJNA702160 GSE166874 {'Series_title': 'Ribosome profiling reveals sequence-independent post-initiation pausing as a signature of translation', 'Series_geo_accession': 'GSE166874', 'Series_status': 'Public on Feb 17 2021', 'Series_submission_date': 'Feb 16 2021', 'Series_last_update_date': 'Feb 22 2021', 'Series_pubmed_id': '24903108', 'Series_summary': 'The journey of a newly synthesized polypeptide starts in the peptidyltransferase center of the ribosome, from where it traverses the exit tunnel. The interior of the ribosome exit tunnel is neither straight nor smooth. How the ribosome dynamics\\xa0in vivo\\xa0is influenced by the exit tunnel is poorly understood. Genome-wide ribosome profiling in mammalian cells reveals elevated ribosome density at the start codon and surprisingly the downstream 5th codon position as well. We found that the highly focused ribosomal pausing shortly after initiation is attributed to the geometry of the exit tunnel, as deletion of the loop region from ribosome protein L4 diminishes translational pausing at the 5th codon position. Unexpectedly, the ribosome variant undergoes translational abandonment shortly after initiation, suggesting that there exists an obligatory step between initiation and elongation commitment. We propose that the post-initiation pausing of ribosomes represents an inherent signature of the translation machinery to ensure productive translation.', 'Series_overall_design': 'We performed Ribo-seq in wildtype HEK293 cells and the cells expressed with mutated RPL4.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Shu-Bing,,Qian', 'Series_sample_id': 'GSM5087367', 'Series_contact_name': 'Yuanhui,,Mao', 'Series_contact_email': 'maoyuanhui123@gmail.com', 'Series_contact_institute': 'Zhejiang University', 'Series_contact_address': '1369 West Wenyi Road', 'Series_contact_city': 'Hangzhou', 'Series_contact_state': 'Zhejiang', 'Series_contact_zip/postal_code': '310000', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166874/suppl/GSE166874_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP306724'}\n",
      "444 PRJNA702531 GSE166987 {'Series_title': 'Reprogramming of translation in yeast cells impaired for ribosome recycling favors short, efficiently translated mRNAs', 'Series_geo_accession': 'GSE166987', 'Series_status': 'Public on Mar 31 2021', 'Series_submission_date': 'Feb 17 2021', 'Series_last_update_date': 'Mar 31 2021', 'Series_pubmed_id': '33764298', 'Series_summary': 'Yeast strains lacking orthologs of mammalian eIF2D (Tma64), and either MCT-1 (Tma20) or DENR (Tma22) are broadly impaired for 40S recycling; however, it was unknown whether this defect leads to reduced 43S PIC levels with an impact on translation of particular mRNAs. To test this, we by combined ribosome footprint profiling with RNA-seq analysis of mRNA abundance.', 'Series_overall_design': 'This study includes a total of 36 samples comprised of 18 RNA-seq samples (mRNA) and 18 ribosome footprint profiling samples (ribo). Experiment 1 includes 12 samples, comprised of 6 RNA-seq samples and 6 ribosome footprint profiling samples, derived from 2 biological replicates each of the BY4741 (WT1) and yeast tma64∆/tma20∆ double mutant H4520 (tma∆∆) which were grown at 30ºC in synthetic complete (SC) medium to log-phase (A600 = 0.5–0.6) and BY4741 grown in SC media lacking isoleucine and valine (SC-Ile/Val) to log-phase (A600 = 0.5–0.6) and treated with the antimetabolite Sulfometuron methyl (SM) at 1 µg/mL for 25 min to induce eIF2α phosphorylation. Experiment 2 examines the effect of the yeast homolog of eIF2D (Tma64) to promote translation initiation on the subset of mRNAs whose translation efficiencies (TEs) are reduced in the tma∆∆ mutant by complementing the function of eIF2 in delivering Met-tRNAi to the 40S subunit. This experiment includes 8 samples, 4 RNA-seq samples and 4 ribosome footprint profiling samples, derived from 2 biological replicates each of WT2 and the tma64∆ mutant 4051, cultured in SC medium at 30ºC. Experiment 3 examines effect of depleting 40S subunit protein on translation efficiencies of strong and weak mRNA and includes 8 samples, 4 RNA-seq samples and 4 ribosome footprint profiling samples, derived from 2 biological replicates each of WT3 strain H2994 and the rps26∆∆ mutant JVY09 were cultured in SC medium (SC medium with 2% galactose+2% raffinose) at 30ºC to log-phase were shifted from galactose- to glucose-containing media for 3h. Experiment 4 examines effect of the transcriptional changes evoked by Gcn4 induction responsible for the translational reprogramming produced by SM treatment of WT cells and includes 8 samples, 4 RNA-seq samples and 4 ribosome footprint profiling samples, derived from 2 biological replicates each of the gcn4∆ mutant 249, cultured in SC with or without SM treatment.', 'Series_type': 'Other', 'Series_contributor': 'Alan,G,Hinnebusch', 'Series_sample_id': 'GSM5090404', 'Series_contact_name': 'Swati,,Gaikwad', 'Series_contact_laboratory': 'Hinnebusch, Alan', 'Series_contact_department': 'Section on Nutrient Control Of Gene Expression', 'Series_contact_institute': 'National Institutes of Health', 'Series_contact_address': 'Building 6, Room 233', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '20892-2716', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166987/suppl/GSE166987_DESeq2_tma_vs_WT1.xlsx', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP306894'}\n",
      "445 PRJNA703954 GSE167223 {'Series_title': 'Long-RNA sequencing and ribosome profiling of inflamed beta-cells reveal an extensive translatome landscape', 'Series_geo_accession': 'GSE167223', 'Series_status': 'Public on Feb 23 2021', 'Series_submission_date': 'Feb 22 2021', 'Series_last_update_date': 'Dec 14 2021', 'Series_pubmed_id': '34554924', 'Series_summary': 'Using a transcriptome-wide profiling approach to map translation initiation start sites in human beta-cells under standard and inflammatory conditions, we identifies non-canonical start sites and translation initiation within lncRNA.', 'Series_overall_design': 'EndoC-BH1 cells have been treated 24h with proinflammatory cytokines 1000 U/ml IFNγ and 2 ng/ml IL1β for 24 h. RNA was isolated by TRizol and sequenced by nanopore sequencing to etablish the de novo transcritpome. Upon incubation, cells were treated with 2ug/ml harringtonine for 30 minutes and Cycloheximide (100ug/ml) for ribosome footprint purification.', 'Series_type': 'Other', 'Series_contributor': 'Arnaud,,Zaldumbide', 'Series_sample_id': 'GSM5099841', 'Series_contact_name': 'Hailiang,,Mei', 'Series_contact_email': 'sasc@lumc.nl', 'Series_contact_department': 'Sequencing Analysis Support Core', 'Series_contact_institute': 'Leiden University Medical Center', 'Series_contact_address': 'Einthovenweg 20', 'Series_contact_city': 'Leiden', 'Series_contact_zip/postal_code': '2333 ZC', 'Series_contact_country': 'Netherlands', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE167nnn/GSE167223/suppl/GSE167223_nanopore_filtered.tsv.gz', 'Series_platform_id': 'GPL24106', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP307513'}\n",
      "446 PRJNA704316 GSE167324 {'Series_title': 'Analysis of the eIF4E3-dependent translatome after Torin1-induced stress in N2a cells ctrl and eIF4E3 KO', 'Series_geo_accession': 'GSE167324', 'Series_status': 'Public on Feb 24 2021', 'Series_submission_date': 'Feb 23 2021', 'Series_last_update_date': 'Jun 03 2021', 'Series_pubmed_id': '33893802', 'Series_summary': 'The eIF4E are a family of initiation factors that bind the mRNA 5’ cap, regulating the proteome and the cellular phenotype. eIF4E1 mediates global translation and its activity is controlled via the PI3K/AKT/mTOR pathway. mTOR down-regulation results in eIF4E1 sequestration into an inactive complex with the 4E binding proteins (4EBPs). The second member, eIF4E2, regulates the translatome during hypoxia. However, the exact function of the third member, eIF4E3, has remained elusive. We have dissected its function using a range of techniques. Starting from the observation that it does not interact with 4EBP1, we demonstrate that eIF4E3 recruitment into an eIF4F complex occurs when Torin1 inhibits the mTOR pathway. Ribo-seq studies demonstrate that this complex (eIF4FS) is translationally active only during stress and that it selects specific mRNA populations based on 5’ TL (UTR) length. The interactome reveals that it associates with cellular proteins beyond the cognate initiation factors, suggesting that it may have “moon-lighting” functions. Finally, we provide evidence that cellular metabolism is altered in an eIF4E3 KO background but only upon Torin1 treatment. We propose that eIF4E3 acts as a second branch of the integrated stress response, re-programming the translatome to promote “stress resistance” and adaptation.', 'Series_overall_design': '3 biological replicates for each of the four conditions tested: N2a WT cells +/- Torin1, N2a KO cells +- Torin1', 'Series_type': 'Other', 'Series_contributor': 'Joseph,A,Curran', 'Series_sample_id': 'GSM5101943', 'Series_contact_name': 'Benjamin,,Weiss', 'Series_contact_institute': 'Weizmann Institute of Science', 'Series_contact_address': '234 Herzl Street', 'Series_contact_city': 'Rehovot', 'Series_contact_zip/postal_code': '761001', 'Series_contact_country': 'Israel', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE167nnn/GSE167324/suppl/GSE167324_TE_ctrl_vehicle_vs_KO_vehicle.txt.gz', 'Series_platform_id': 'GPL21103', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP307718'}\n",
      "447 PRJNA704627 GSE167421 {'Series_title': 'Structural basis of ribosomal frameshifting during translation of the SARS-CoV-2 RNA genome', 'Series_geo_accession': 'GSE167421', 'Series_status': 'Public on May 17 2021', 'Series_submission_date': 'Feb 24 2021', 'Series_last_update_date': 'Aug 16 2021', 'Series_pubmed_id': '34029205', 'Series_web_link': 'https://science.sciencemag.org/content/early/2021/05/12/science.abf3546', 'Series_summary': 'Programmed ribosomal frameshifting is the key event during translation of the SARS-CoV-2 RNA genome allowing synthesis of the viral RNA-dependent RNA polymerase and downstream viral proteins. Here we present the cryo-EM structure of the mammalian ribosome in the process of translating viral RNA paused in a conformation primed for frameshifting. We observe that the viral RNA adopts a pseudoknot structure lodged at the mRNA entry channel of the ribosome to generate tension in the mRNA that leads to frameshifting. The nascent viral polyprotein that is being synthesized by the ribosome paused at the frameshifting site forms distinct interactions with the ribosomal polypeptide exit tunnel. We use biochemical experiments to validate our structural observations and to reveal mechanistic and regulatory features that influence the frameshifting efficiency. Finally, a compound previously shown to reduce frameshifting is able to inhibit SARS-CoV-2 replication in infected cells, establishing coronavirus frameshifting as target for antiviral intervention.', 'Series_overall_design': 'Ribosome profiling of SARS-CoV-2 in-vitro translation', 'Series_type': 'Other', 'Series_contributor': 'Nenad,,Ban', 'Series_sample_id': 'GSM5105029', 'Series_contact_name': 'Rene,,Dreos', 'Series_contact_email': 'rene.dreos@unil.ch', 'Series_contact_phone': '0044 (0)1603 450793', 'Series_contact_laboratory': 'Gatfield Lab', 'Series_contact_department': 'CIG', 'Series_contact_institute': 'UNIL', 'Series_contact_address': 'Ecublens', 'Series_contact_city': 'Lausanne', 'Series_contact_state': 'VD', 'Series_contact_zip/postal_code': '1015', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE167nnn/GSE167421/suppl/GSE167421_RAW.tar', 'Series_platform_id': 'GPL29767', 'Series_platform_taxid': '2697049', 'Series_sample_taxid': '2697049', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP307946'}\n",
      "448 PRJNA704712 GSE167484 {'Series_title': 'Innate, translation-dependent silencing of an invasive transposon in Arabidopsis', 'Series_geo_accession': 'GSE167484', 'Series_status': 'Public on Aug 08 2021', 'Series_submission_date': 'Feb 24 2021', 'Series_last_update_date': 'Mar 30 2022', 'Series_pubmed_id': '34931432', 'Series_summary': 'Co-evolution between hosts\\' and parasites\\' genomes shapes diverse pathways of acquired immunity based on silencing small (s)RNAs. In plants, sRNAs cause heterochromatinization, sequence-degeneration and, ultimately, loss-of-autonomy of most transposable elements (TEs). Recognition of newly-invasive plant TEs, by contrast, involves an innate antiviral-like silencing response. To investigate this response\\'s activation, we studied the single-copy element EVADÉ (EVD), one of few representatives of the large Ty1/Copia family able to proliferate in Arabidopsis when epigenetically-reactivated. In Ty1/Copia-elements, a short subgenomic mRNA (shGAG) provides the necessary excess of structural GAG protein over the catalytic components encoded by the full-length genomic flGAG-POL. We show here that the predominant cytosolic distribution of shGAG strongly favors its translation over mostly-nuclear flGAG-POL. During this process, an unusually intense ribosomal stalling event coincides with mRNA breakage yielding unconventional 5\\'OH RNA fragments that evade RNA-quality-control. The starting-point of sRNA production by RNA-DEPENDENT-RNA-POLYMERASE-6 (RDR6), exclusively on shGAG, occurs precisely at this breakage point. This hitherto-unrecognized \"translation-dependent silencing\" (TdS) is independent of codon-usage or GC-content and is not observed on TE remnants populating the Arabidopsis genome, consistent with their poor association, if any, with polysomes. We propose that TdS forms a primal defense against EVD de novo invasions that underlies its associated sRNA pattern.', 'Series_overall_design': 'Ribo-seq and sRNA-seq were performed on RNA from inflorescence tissue of EVD and GFP-EVD-GUS overexpressing plants. NanoPare and SMART-seq2 was perfomed on wild-type (Col-0) and ddm1-2 inflorescence tissue.', 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'Olivier,,Voinnet', 'Series_sample_id': 'GSM5213541', 'Series_contact_name': 'Stefan,,Oberlin', 'Series_contact_email': 'stefan.oberlin@ucsf.edu', 'Series_contact_department': 'Department of Biology', 'Series_contact_institute': 'ETH Zürich', 'Series_contact_address': 'Universitätstrasse 2', 'Series_contact_city': 'Zürich', 'Series_contact_zip/postal_code': '8092', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE167nnn/GSE167484/suppl/GSE167484_normalized_counts_sRNA.csv.gz', 'Series_platform_id': 'GPL17639', 'Series_platform_taxid': '3702', 'Series_sample_taxid': '3702', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP308027'}\n",
      "449 PRJNA704908 GSE167535 {'Series_title': 'Valine tRNA biogenesis and bioavailability regulates mitochondrial complex I levels in acute leukemia [Ribo-Seq]', 'Series_geo_accession': 'GSE167535', 'Series_status': 'Public on Oct 27 2021', 'Series_submission_date': 'Feb 25 2021', 'Series_last_update_date': 'Dec 28 2021', 'Series_pubmed_id': '34937946', 'Series_summary': 'We report the effects on gene expression at the translational level on dietary modulation of valine in in vivo model of T cell acute lymphoblastic leukemia', 'Series_overall_design': 'Examination of changes in the ribosome occupancy as a measure of translational effeciency following dietary modulation of valine in in vivo model of T cell acute lymphoblastic leukemia', 'Series_type': 'Other', 'Series_contributor': 'Iannis,,Aifantis', 'Series_sample_id': 'GSM5106833', 'Series_contact_name': 'Andreas,,Kloetgen', 'Series_contact_department': 'Department for Computational Biology of Infection Research', 'Series_contact_institute': 'Helmholtz Centre for Infection Research', 'Series_contact_address': 'Inhoffenstr 7', 'Series_contact_city': 'Braunschweig', 'Series_contact_state': 'NI', 'Series_contact_zip/postal_code': '38124', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE167nnn/GSE167535/suppl/GSE167535_TALL_riboseq_FPKM.tsv.gz', 'Series_platform_id': 'GPL21103', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP308208'}\n",
      "450 PRJNA705181 GSE167704 {'Series_title': 'Ribosome profiling experiments of LRRK2 mouse models', 'Series_geo_accession': 'GSE167704', 'Series_status': 'Public on Feb 27 2021', 'Series_submission_date': 'Feb 26 2021', 'Series_last_update_date': 'Dec 10 2021', 'Series_pubmed_id': '34759048', 'Series_summary': 'G2019S mutaion of LRRK2 is known to increase mRNA translation. We perform ribosome profiling to study defective translation in mammalian LRRK2 models. We used G2019S LRRK2 transgenic mice, G2019S/D1994A LRRK2 transgenic mice, LRRK2 knockout mice.', 'Series_overall_design': 'Mouse: 3 months old. Biological replicates were generated for mouse experiments.', 'Series_type': 'Other', 'Series_contributor': 'Valina,L,Dawson', 'Series_sample_id': 'GSM5111968', 'Series_contact_name': 'Jungwoo Wren,,Kim', 'Series_contact_email': 'jwrenkim@berkeley.edu, jwk0906@gmail.com', 'Series_contact_laboratory': 'Ingolia Lab', 'Series_contact_department': 'Molecular and Cell Biology', 'Series_contact_institute': 'UC Berkeley', 'Series_contact_address': 'Barker Hall, Room 416', 'Series_contact_city': 'Berkeley', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94720', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE167nnn/GSE167704/suppl/GSE167704_LRRK2_KO.txt.gz', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP308389'}\n",
      "451 PRJNA705358 GSE167865 {'Series_title': 'One-shot analysis of translated mammalian lncRNAs with AHARIBO', 'Series_geo_accession': 'GSE167865', 'Series_status': 'Public on Feb 28 2021', 'Series_submission_date': 'Feb 27 2021', 'Series_last_update_date': 'May 31 2021', 'Series_pubmed_id': '33594971', 'Series_summary': 'A vast portion of the mammalian genome is transcribed as long non-coding RNAs (lncRNAs) acting in the cytoplasm with largely unknown functions. Surprisingly, lncRNAs have been shown to interact with ribosomes, encode uncharacterized proteins, or act as ribosome sponges. These functions still remain mostly undetected and understudied owing to the lack of efficient tools for genome-wide simultaneous identification of ribosome-associated lncRNAs and peptide-producing lncRNAs. Here we present AHARIBO, a method for the detection of lncRNAs either untranslated, but associated with ribosomes, or encoding small peptides. Using AHARIBO in mouse embryonic stem cells during neuronal differentiation, we isolated ribosome-protected RNA fragments, translated RNAs and corresponding de novo synthesized polypeptides. Besides identifying mRNAs under active translation and associated ribosomes, we found and distinguished lncRNAs acting as ribosome sponges or encoding micropeptides, laying the ground for a better functional understanding of hundreds lncRNAs.', 'Series_overall_design': 'We performed standard and AHARIBO (AHA-mediated RIBOsome isolation) RIBO-Seq on mouse embryonic stem cells. AHARIBO RIBO-Seq is based on the isolation of ribosomes trapped with their nascent peptides, by incorporating the non-canonical amino acid L-azidohomoalanine (AHA) followed by ribosome profiling. Experiment were performed in biological triplicate (standard RIBO-Seq) and duplicate (AHARIBO RIBO-Seq).', 'Series_type': 'Other', 'Series_contributor': 'Massimiliano,,Clamer', 'Series_sample_id': 'GSM5114107', 'Series_contact_name': 'Fabio,,Lauria', 'Series_contact_email': 'fabio.lauria@ibf.cnr.it, fab.lauria.88@gmail.com', 'Series_contact_institute': 'Institute of Biophysics (Italian National Research Council)', 'Series_contact_address': 'Via alla Cascata 56', 'Series_contact_city': 'Povo (TN)', 'Series_contact_zip/postal_code': '38123', 'Series_contact_country': 'Italy', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE167nnn/GSE167865/suppl/GSE167865_reads_per_transcript_table.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP308484'}\n",
      "452 PRJNA706704 GSE168265 {'Series_title': 'Comparative Ribo-Seq of Staphylococcus aureus harboring A2058-unmodified ribosomes and m6A2058-modified ribosomes', 'Series_geo_accession': 'GSE168265', 'Series_status': 'Public on Jan 16 2024', 'Series_submission_date': 'Mar 04 2021', 'Series_last_update_date': 'Feb 14 2024', 'Series_pubmed_id': '38252661', 'Series_summary': 'total mRNA abundance and ribosome density of each ORFs were calculated.', 'Series_overall_design': 'Investigating the translational efficiency and fidelity of unmodified- and m6A2058-modified ribosomes in a methicilin-resistant Staphylococcus aureus (MRSA).', 'Series_type': 'Other', 'Series_contributor': 'M,F,Yap', 'Series_sample_id': 'GSM5135231', 'Series_contact_name': 'Mee-Ngan,F.,Yap', 'Series_contact_email': 'frances.yap@northwestern.edu', 'Series_contact_phone': '312-503-3793', 'Series_contact_laboratory': 'Searle 3-430', 'Series_contact_department': 'Microbiology-Immunology', 'Series_contact_institute': 'Northwestern University Feinberg School of Medicine', 'Series_contact_address': '320 E Superior St', 'Series_contact_city': 'Chicago', 'Series_contact_state': 'IL', 'Series_contact_zip/postal_code': '60611', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE168nnn/GSE168265/suppl/GSE168265_RAW.tar', 'Series_platform_id': 'GPL17452', 'Series_platform_taxid': '1280', 'Series_sample_taxid': '1280', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP309329'}\n",
      "453 PRJNA707431 GSE168445 {'Series_title': 'Transcriptome and translatome of ribosomal protein-deficient human cells', 'Series_geo_accession': 'GSE168445', 'Series_status': 'Public on Jan 27 2022', 'Series_submission_date': 'Mar 08 2021', 'Series_last_update_date': 'Jul 14 2022', 'Series_pubmed_id': '35137207', 'Series_summary': 'Human ribosomes are made of around 80 ribosomal proteins (RPs) and four ribosomal RNAs. To explore gene expression regulation by RPs at the transcriptional and translational levels, parallel RNA-seq and Ribo-seq were conducted in A549 cells after knockdown of individual RP.', 'Series_overall_design': '75 human RPs were knocked-down by specific siRNAs one-by-one in A549 cells, and the RP-deficient cells were subjected to parallel RNA-seq and Ribo-seq. Gene expression in RP-deficient cells was compared with that in the control cells treated by non-targeting siRNAs. In total, we prepared 105 Ribo-seq libraries, with 105 paired RNA-seq libraries, for 78 human RPs as well as 8 control samples, where biological replicates were performed for 16 RPs and for 3 control samples.', 'Series_type': 'Other', 'Series_contributor': 'Zhi,,Xie', 'Series_sample_id': 'GSM5140438', 'Series_contact_name': 'Yizhao,,Luan', 'Series_contact_email': 'luanyz_leo@163.com', 'Series_contact_department': 'Zhongshan Ophthalmic Center', 'Series_contact_institute': 'Sun Yat-sen university', 'Series_contact_address': 'Xianlie South Rd. 54, Yuexiu Dist', 'Series_contact_city': 'Guangzhou', 'Series_contact_state': 'Guangdong', 'Series_contact_zip/postal_code': '510060', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE168nnn/GSE168445/suppl/GSE168445_Riboseq.combined.cds.counts.txt.gz', 'Series_citation': 'Yizhao Luan, Nan Tang, Jiaqi Yang, Shuting Liu, Chichi Cheng, Yan Wang, Yanan Guo, Hongwei Wang, Mengqing Xiang, Rong Ju, and Zhi Xie. Decoding regulatory specificity of human ribosomal proteins on global gene expression. bioRxiv 2021.03.27.437302; doi:10.1101/2021.03.27.437302', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP309704'}\n",
      "454 PRJNA707537 GSE168476 {'Series_title': 'A Rapid Translational Immune Response Program in CD8 Memory T Lymphocytes', 'Series_geo_accession': 'GSE168476', 'Series_status': 'Public on Mar 01 2022', 'Series_submission_date': 'Mar 08 2021', 'Series_last_update_date': 'Mar 02 2022', 'Series_summary': 'Abstract: Immune cell activation is a very rapid and concerted cellular response that requires coordination between cellular processes that occur in different compartments and on very different time scales. Here, we use ribosome profiling and deep RNA sequencing to define the acute mRNA translation changes that follow initial signaling events and enable subsequent events of T cell activation and expansion.  Briefly, early events in the activation of antigen experienced CD 8 T cells are insensitive to transcriptional blockade with actinomycin, and instead depend on the translation of pre-existing mRNAs and blocked by cycloheximide. Ribosome profiling identifies ~92 mRNAs that are recruited into ribosomes following CD8 T cell stimulation. These mRNAs typically have structured GC and pyrimidine rich 5’UTRs and they encode key regulators of T cell activation and proliferation such as Notch1, Ifngr1, Il2rb, serine metabolism enzymes Psat1 and Shmt2, and translation factors eEF1a1 and eEF2. The increased production of receptors of IL2 and IFNγ precedes the activation of gene expression and augments cellular signals and T cell activation. Together, we identify the early RNA translation program that acts in a feed-forward manner to enable the rapid and dramatic process of CD8 T cell expansion and activation.', 'Series_overall_design': 'mRNA and Ribosome footprints of CD8 memory T cells before and after  two hours after CD3/CD28 stimulation', 'Series_type': 'Other', 'Series_contributor': 'Hans-Guido,,Wendel', 'Series_sample_id': 'GSM5141315', 'Series_contact_name': 'Gunnar,,Rätsch', 'Series_contact_email': 'raetsch@inf.ethz.ch', 'Series_contact_laboratory': 'Biomedical Informatics', 'Series_contact_department': 'Institute for Machine Learning', 'Series_contact_institute': 'ETH Zürich', 'Series_contact_address': 'Universitatstrasse 6', 'Series_contact_city': 'Zurich', 'Series_contact_state': '-None-', 'Series_contact_zip/postal_code': '8006', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE168nnn/GSE168476/suppl/GSE168476_cds_xpressFP_xpressRNA.tsv.gz', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP309764'}\n",
      "455 PRJNA708348 GSE168595 {'Series_title': 'Codon usage and protein length-dependent feedback from translation elongation to translation initiation and kinetics', 'Series_geo_accession': 'GSE168595', 'Series_status': 'Public on Aug 10 2021', 'Series_submission_date': 'Mar 09 2021', 'Series_last_update_date': 'Sep 29 2021', 'Series_pubmed_id': '34417614', 'Series_summary': 'In yeast and mammals, activated GCN2 can phosphorylate its substrate eIF2α, which is a part of the eIF2-GTP-Met-tRNAiMet ternary complex. The eIF2α phosphorylation blocks the ternary complex formation and therefore inhibits translation initiation. Meanwhile, GCN2 activation associates with ribosomes and some translation elongation factors such as eEF1A. In Neurospora crassa, the homolog of GCN2 is CPC-3. Ribosome profiling and accompanying RNA-seq experiments in this project were used to explore the effects of CPC-3 on translation kinetics. Here we show that poor codon usage of mRNAs with long CDS preferentially causes CPC-3 activation, which in turn suppresses the translation initiation and elongation in both codon usage and CDS length dependent manner.', 'Series_overall_design': 'The ribosome protected fragment (RPF) profiles and transcription profiles of Δcpc-3 and wild-type strains of Neurospora crassa were generated by high-throughput sequencing', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Yi,,Liu', 'Series_sample_id': 'GSM5150339', 'Series_contact_name': 'xueliang,,lyu', 'Series_contact_email': 'lvxueliang0715@aliyun.com', 'Series_contact_laboratory': 'Yi Liu', 'Series_contact_department': 'Physiology', 'Series_contact_institute': 'UT Southwestern Medical Center', 'Series_contact_address': '6001 Forest Park Road', 'Series_contact_city': 'Dallas', 'Series_contact_state': 'Texas', 'Series_contact_zip/postal_code': '75235', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE168nnn/GSE168595/suppl/GSE168595_RAW.tar', 'Series_platform_id': 'GPL29831', 'Series_platform_taxid': '5141', 'Series_sample_taxid': '5141', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP310013'}\n",
      "456 PRJNA714824 GSE168977 {'Series_title': 'Phosphorylation of Ribosomal Protein S6 differentially affects mRNA translation based on ORF length', 'Series_geo_accession': 'GSE168977', 'Series_status': 'Public on Dec 22 2021', 'Series_submission_date': 'Mar 16 2021', 'Series_last_update_date': 'Jul 28 2022', 'Series_pubmed_id': '35869369', 'Series_summary': 'Phosphorylation of Ribosomal Protein S6 (RPS6) was the first post-translational modification of the ribosome to be identified and is a commonly-used readout for mTORC1 activity. Although the cellular and organismal functions of RPS6 phosphorylation are known, its molecular consequences on translation are less well understood. Here we use selective ribosome footprinting to analyze the location of ribosomes containing phosphorylated RPS6 on endogenous mRNAs in cells. We find that RPS6 becomes progressively dephosphorylated on ribosomes as they translate an mRNA. As a consequence, average RPS6 phosphorylation is higher on mRNAs with short coding sequences (CDSs) compared to mRNAs with long CDSs. Loss of RPS6 phosphorylation causes a correspondingly larger drop in translation efficiency of mRNAs with short CDSs than long CDSs. Interestingly, mRNAs with 5’ TOP motifs are translated well also in the absence of RPS6 phosphorylation despite short CDS lengths, suggesting they are translated via a different mode. In sum this provides a dynamic view of RPS6 phosphorylation on ribosomes as they translate mRNAs and the functional consequence on translation.', 'Series_overall_design': 'We develop modification selective ribosome footprinting to determine the rate of RPS6 phosphorylation of translating ribosomes across the translatome. This allows us to gain mechanistic understanding of the role of RPS6 phosphorylation in ribosome function.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Mykola,,Roiuk', 'Series_sample_id': 'GSM5173418', 'Series_contact_name': 'Jonathan,,Bohlen', 'Series_contact_laboratory': 'AG Teleman', 'Series_contact_department': 'B140', 'Series_contact_institute': 'DKFZ', 'Series_contact_address': 'Im Neuenheimer Feld 581', 'Series_contact_city': 'Heidelberg', 'Series_contact_zip/postal_code': '69123', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE168nnn/GSE168977/suppl/GSE168977_Processed_Data_File_MEF.xlsx', 'Series_platform_id': 'GPL21697', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP310845'}\n",
      "457 PRJNA715464 GSE169168 {'Series_title': 'Ribosome profiling RNA sequencing in zebrafish embryos', 'Series_geo_accession': 'GSE169168', 'Series_status': 'Public on Jun 03 2022', 'Series_submission_date': 'Mar 18 2021', 'Series_last_update_date': 'Jun 05 2022', 'Series_web_link': 'https://doi.org/10.1038/s41422-022-00655-5', 'Series_summary': 'We show that liquid-liquid phase separation (LLPS)-mediated zebrafish Ddx3xb condensation facilitates MZT through promoting maternal mRNAs translation. Ddx3xb forms condensates via its N-terminal intrinsically disordered regions (IDRs), and the gradually increased ATP concentration after fertilization promotes its aggregation capability. Ddx3xb-deficiency decelerates the decay of maternal mRNAs and impedes zygotic genome activation, further leading to developmental defect. The phenotype in Ddx3xb-deficiency embryos can be efficiently rescued by condensed Ddx3xb, but not the Ddx3xb without LLPS ability. Mechanistically, the condensation of Ddx3xb is vital for its RNA helicase activity, which is critical for involved in promoting translation efficiency of maternal mRNAs through opening 5’ UTR structures during the MZT process. Our study demonstrates that the condensation of Ddx3xb promotes maternal mRNAs translation via its RNA unwinding activity and further facilitates MZT, highlighting the critical role of protein phase separation in translational control and animal early development.', 'Series_overall_design': 'Please note that each processed data was generated from both replicates and is linked to the corresponding rep1 sample records.', 'Series_type': 'Other', 'Series_contributor': 'Y,,Yang', 'Series_sample_id': 'GSM5503732', 'Series_contact_name': 'Bao-Fa,,Sun', 'Series_contact_email': 'sunbf@big.ac.cn', 'Series_contact_institute': 'Beijing Institute of Genomics (BIG) of Chinese Academy of Sciences (CAS)', 'Series_contact_address': 'Da-Tun Road', 'Series_contact_city': 'Beijing', 'Series_contact_zip/postal_code': '100101', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE169nnn/GSE169168/suppl/GSE169168_RAW.tar', 'Series_platform_id': 'GPL24995', 'Series_platform_taxid': '7955', 'Series_sample_taxid': '7955', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP311222'}\n",
      "458 PRJNA716630 GSE169457 {'Series_title': 'Ribosome profiling analysis of mouse brain neocortex during development', 'Series_geo_accession': 'GSE169457', 'Series_status': 'Public on Sep 06 2022', 'Series_submission_date': 'Mar 23 2021', 'Series_last_update_date': 'Jan 06 2023', 'Series_pubmed_id': '36482253', 'Series_summary': 'We perform Ribosome Profiling (Ribo-seq) analysis of mouse brain neocortex during development embyonic days 12.5, 14, 15.5, 17, and postnatal day 0 in biological duplicate', 'Series_overall_design': 'Each biological replicate analyzed for ribosome-protected mRNA fragments after RNAse digestion (Ribosome Profiling, Ribo-seq)', 'Series_type': 'Other', 'Series_contributor': 'Kraushar,,Matthew', 'Series_sample_id': 'GSM5203567', 'Series_contact_name': 'Matthew,Lee,Kraushar', 'Series_contact_email': 'matthew.kraushar@molgen.mpg.de', 'Series_contact_department': 'Department of Computational Molecular Biology', 'Series_contact_institute': 'Max Planck Institute for Molecular Genetics', 'Series_contact_address': 'Ihnestrasse 63-73', 'Series_contact_city': 'Berlin', 'Series_contact_state': 'Berlin', 'Series_contact_zip/postal_code': '14195', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE169nnn/GSE169457/suppl/GSE169457_countdata.tsv.gz', 'Series_platform_id': 'GPL21103', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP311832'}\n",
      "459 PRJNA717060 GSE169589 {'Series_title': 'N7-methylguanosine tRNA modification promotes tumorigenesis and chemoresistance through WNT/β-Catenin pathway in nasopharyngeal carcinoma', 'Series_geo_accession': 'GSE169589', 'Series_status': 'Public on Mar 13 2024', 'Series_submission_date': 'Mar 24 2021', 'Series_last_update_date': 'Mar 13 2024', 'Series_summary': 'Treatment selections are very limited for patients with advanced nasopharyngeal carcinoma (NPC) experiencing disease progression. Uncovering the potential mechanism underlying NPC progression is crucial for identify novel treatments. Here we show that N7-methylguanosine (m7G) tRNA modification enzyme METTL1 and its partner WDR4 are significantly elevated in NPC and associated with poor prognosis. Loss-of-function and gain-of-function assays demonstrated that METTL1/WDR4 mediated m7G tRNA modification promotes NPC growth and metastasis in vitro and in vivo. Mechanistically, m7G tRNA modification selectively regulates the translation of transcripts with higher percentage of m7G tRNA decoded codons. Moreover, further analysis revealed that METTL1-mediated m7G tRNA modification activates WNT/β-Catenin signaling pathway to promote NPC cell epithelial-mesenchymal transition (EMT) and chemoresistance to cisplatin and docetaxel in vitro and in vivo. Our work uncovers a novel layer of mRNA translation regulation mechanism at codon recognition step mediated by tRNA modification and reveals the critical function of tRNA modification in cancer progression.', 'Series_overall_design': 'TRAC-Seq were developed to identify the tRNA m7G methylome in nasopharyngeal carcinoma cell line. Polyribosome-bound-RNA-seq and Ribo-seq was used to study the differential translated genes in the Mettl1 knockout and control cells.', 'Series_type': 'Other', 'Series_contributor': 'Shuibin,,Lin', 'Series_sample_id': 'GSM5210398', 'Series_contact_name': 'Shuibin,,Lin', 'Series_contact_email': 'linshb6@mail.sysu.edu.cn', 'Series_contact_department': 'Clincial for Translational Medicine', 'Series_contact_institute': 'The First Affliliated Hospital, Sun Yat-sen University', 'Series_contact_address': 'Zhongshan 2nd road, Guangzhou', 'Series_contact_city': 'Guangzhou', 'Series_contact_state': 'China', 'Series_contact_zip/postal_code': '510080', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE169nnn/GSE169589/suppl/GSE169589_TRAC-seq_expression.txt.gz', 'Series_platform_id': 'GPL23227', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP312034'}\n",
      "460 PRJNA717063 GSE169590 {'Series_title': 'N7-methylguanosine tRNA modification promotes esophageal squamous cell carcinoma tumorigenesis via the RPTOR/ULK1/autophagy axis', 'Series_geo_accession': 'GSE169590', 'Series_status': 'Public on Feb 16 2022', 'Series_submission_date': 'Mar 24 2021', 'Series_last_update_date': 'Apr 06 2022', 'Series_pubmed_id': '35304469', 'Series_summary': 'The cancer cells selectively promote the translation of oncogenic mRNAs to facilitate cancer progression, while the molecular mechanisms remain unclear. The tRNA N7-methylguanosine (m7G) modification is not essential for yeast growth, but in mammals mis-regulations of tRNA m7G modification cause stem cell defect and developmental disorders. Here we found that tRNA m7G methyltransferase complex components METTL1 and WDR4 are elevated in esophageal squamous cell carcinoma (ESCC) tissues and associated with poor ESCC prognosis. Functionally, depletion of METTL1 or WDR4 suppresses proliferation, migration, and invasion of ESCC cells. In addition, forced expression of METTL1 or WDR4 promotes ESCC progression depending on the tRNA m7G methyltransferase activity. Mechanistically, METTL1 knockdown leads to reduced tRNA m7G modification and decreased expression of m7G-modified tRNAs. Depletion of METTL1 selectively reduces the translation of a subset of oncogenic transcripts, including the genes related to mTOR signaling and autophagy in m7G related codon dependent manner. Rescue assays revealed the essential function of RPTOR/ULK1/autophagy axis in METTL1 driven ESCC progression. Furthermore, ESCC initiation and progression models using Mettl1 conditional knockout and knockin mice uncovered the strong physiological function of Mettl1 in promoting in vivo ESCC tumorigenesis. Our study uncovered a new layer of translation regulation mechanism mediated by tRNA m7G modification, provided strong evidence to support the important physiological function of mis-regulated tRNA modification in cancer, and suggested that targeting METTL1 and its downstream signaling axis could be a promising strategy for ESCC treatment.', 'Series_overall_design': 'TRAC-Seq were developed to identify the tRNA m7G methylome in ESCC cell line. Polyribosome-bound-RNA-seq was used to study the differentially translated genes in the METTL1 knockdown and control cells. Ribo-seq was performed to study the differentially translated genes and ribosome processivity in the METTL1 knockdown and control cells. RPTOR-rescued polyribosome-bound-RNA-seq was used to distinguish the m7G-dependent and RPTOR-dependent mRNA translation in the K150 cells.', 'Series_type': 'Other', 'Series_contributor': 'Shuibin,,Lin', 'Series_sample_id': 'GSM5903025', 'Series_contact_name': 'Shuibin,,Lin', 'Series_contact_email': 'linshb6@mail.sysu.edu.cn', 'Series_contact_department': 'Clincial for Translational Medicine', 'Series_contact_institute': 'The First Affliliated Hospital, Sun Yat-sen University', 'Series_contact_address': 'Zhongshan 2nd road, Guangzhou', 'Series_contact_city': 'Guangzhou', 'Series_contact_state': 'China', 'Series_contact_zip/postal_code': '510080', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE169nnn/GSE169590/suppl/GSE169590_tRAN_counts_for_TRAC-seq.txt.gz', 'Series_platform_id': 'GPL23227', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP312040'}\n",
      "461 PRJNA717246 GSE169632 {'Series_title': 'Profiling and functional characterization of maternal mRNA translation in mouse maternal-to-zygotic transition', 'Series_geo_accession': 'GSE169632', 'Series_status': 'Public on Nov 28 2021', 'Series_submission_date': 'Mar 25 2021', 'Series_last_update_date': 'Nov 29 2022', 'Series_pubmed_id': '36333315', 'Series_summary': 'Translational regulation plays an important role in gene expression and function. Although the transcriptional dynamics of mouse pre-implantation embryos has been well characterized, the global mRNA translation landscape and the master regulators of zygotic genome activation (ZGA) remain unknown. Here, by developing and applying a low-input ribosome profiling (LiRibo-seq) technique, we profiled the mRNA translation landscape in mouse pre-implantation embryos and revealed the translational dynamics during mouse pre-implantation development. We identified a dramatic translational transition from MII oocytes to zygotes, and demonstrated that active translation of maternal mRNAs is essential for maternal-to-zygotic transition (MZT). We further showed that two maternal factors, Smarcd2 and Cyclin T2, whose translation is activated in zygotes, are required for chromatin reprogramming and ZGA, respectively. Our study thus not only filled in a knowledge gap on translational regulation during mammalian pre-implantation development, but also revealed new insights into the critical function of maternal mRNA translation in MZT.', 'Series_overall_design': 'In this study, we modified RiboLace sequencing method to achieve mRNA translation profiling using as few as 100-250 oocytes or embryos. Using this low-input ribosome sequencing (LiRibo-seq) method, we have generated mRNA translation profiling of mouse MII oocytes and preimplantation embryos from 1-cell to blastocyst stage, revealing the dynamics of mRNA translation during the natural developmental process. We constructed a total of 14 LiRibo-seq datasets, including: two 5k ESC libraries, two MII oocyte libraries, two 1-cell embryo libraries, two 2-cell embryo libraries, two 4-cell embryo libraries, two morula libraries, two blastocyst embryo libraries. 12 total RNA-seq datasets were generated including two MII oocyte libraries, two 1-cell embryo libraries, two 2-cell embryo libraries, two 4-cell embryo libraries, two morula libraries, two blastocyst embryo libraries. 16 polyA RNA-seq datasets were generated including two MII oocyte libraries, two 1-cell embryo libraries, two DMSO and CHX treated zygote libraries, two DMSO and AMA treated zygote libraries, two siRNA control and Ccnt2 KD 2-cell embryo libraries. 8 ATAC-seq datasets were generated including two DMSO and CHX zygotes libraries, two siRNA control and Smarcd2 KD 1-cell embryo libraries.', 'Series_type': 'Other', 'Series_contributor': 'Yi,,Zhang', 'Series_sample_id': 'GSM5585731', 'Series_contact_name': 'Yi,,Zhang', 'Series_contact_email': 'yzhang@genetics.med.harvard.edu', 'Series_contact_department': 'PCMM', 'Series_contact_institute': \"Boston Children's Hospital / Harvard Medical School\", 'Series_contact_address': '200 Longwood Avenue WAB 149', 'Series_contact_city': 'Boston', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02115', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE169nnn/GSE169632/suppl/GSE169632_RAW.tar', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP312148'}\n",
      "462 PRJNA719381 GSE171392 {'Series_title': 'Translational Autoregulation of the S. cerevisiae High-affinity Polyamine Transporter Hol1', 'Series_geo_accession': 'GSE171392', 'Series_status': 'Public on Aug 09 2021', 'Series_submission_date': 'Apr 02 2021', 'Series_last_update_date': 'Sep 06 2021', 'Series_pubmed_id': '34375581', 'Series_summary': 'Study is aimed to understand translation regulation of mRNA in low and high polyamines in S. cerevisiae. mRNA seq and Ribosome profiling were performed in yeast cells, in duplicates, under low (0 spermidine) and high (10 µM) spermidine. Translation efficiency was calculated (low vs high SPD) and GO analysis reveals 23 transporters as the only class undergo TE downregulation in high SPD condition. Further, we investigated role of these 23 transporters in polyamine transport and identified HOL1 as the high affinity polyamine transporter. This study further show that HOL1 is under polyamine control of translation regulation through conserved fungal uORF MLLLPS*.', 'Series_overall_design': 'To understand mRNA translation regulation in low and high polyamines, ribosome profiling and total RNAseq were performed in duplicates under two conditions. In total, 8 datasets are generated, 4 of Riboseq and 4 of RNAseq.', 'Series_type': 'Other', 'Series_contributor': 'Thomas,E,Dever', 'Series_sample_id': 'GSM5224820', 'Series_contact_name': 'Thomas,E,Dever', 'Series_contact_email': 'thomas.dever@nih.gov', 'Series_contact_phone': '301-496-4519', 'Series_contact_laboratory': 'Section on Protein Biosynthesis', 'Series_contact_department': 'NICHD', 'Series_contact_institute': 'National Institutes of Health', 'Series_contact_address': 'BG 6 RM 228 6 CENTER DR', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '20892', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE171nnn/GSE171392/suppl/GSE171392_Reads_TE_spd0_spd10uM.xlsx', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP313248'}\n",
      "463 PRJNA720671 GSE171726 {'Series_title': 'Critical contribution of 3’ non-seed base pairing to the in vivo function of the evolutionarily conserved let-7a microRNA [Ribo-seq]', 'Series_geo_accession': 'GSE171726', 'Series_status': 'Public on Apr 09 2021', 'Series_submission_date': 'Apr 08 2021', 'Series_last_update_date': 'Jun 21 2022', 'Series_pubmed_id': '35476978', 'Series_summary': \"The C.elegans translatomes of single-nt let-7a 3' non-seed mutants (U13A / U18A). In order to identified gene perturbance caused by disrupting 3' non-seed pairing of let-7a, we examined the global gene expression at tranlational levels of L4 whole animals by ribosome profiling. We found that let-7a U13A mutation results in overexpression135/351 Note for let-7a U13A mutation, lin-41(ma480) was used in the background to filter out downstream regulation related to the lin-41 pathway.\", 'Series_overall_design': 'Ribosome profiling (Ribo-seq) of L4 larvae of DG3913[lin-41(tn1541)], VT3878[lin-41(tn1541ma480);let-7(ma432)], VT1367(maIs105), VT3797[ma105,let-7(ma435)] at 20 °C.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Victor,,Ambros', 'Series_sample_id': 'GSM5231638', 'Series_contact_name': 'Ye,,Duan', 'Series_contact_email': 'Ye.Duan@umassmed.edu', 'Series_contact_phone': '5088565723', 'Series_contact_laboratory': 'Victor Ambros', 'Series_contact_department': 'Program of Molecular Medicine', 'Series_contact_institute': 'Umass Medical School', 'Series_contact_address': '373 Plantation Street, Biotech Two, Suite 306', 'Series_contact_city': 'Worcester', 'Series_contact_state': 'MASSACHUSETTS', 'Series_contact_zip/postal_code': '01605', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE171nnn/GSE171726/suppl/GSE171726_Ribo.raw.reads.VT1367_VT3797.csv.gz', 'Series_platform_id': 'GPL19757', 'Series_platform_taxid': '6239', 'Series_sample_taxid': '6239', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP314065'}\n",
      "464 PRJNA722238 GSE172146 {'Series_title': 'Aberrant translation regulation by METTL1/WDR4-mediated tRNA N7-methylguanosine modification drives HNSCC progression', 'Series_geo_accession': 'GSE172146', 'Series_status': 'Public on Mar 23 2022', 'Series_submission_date': 'Apr 15 2021', 'Series_last_update_date': 'Mar 24 2022', 'Series_pubmed_id': '35179319', 'Series_summary': 'The cancer cells selectively promote the translation of specific oncogenic transcripts to stimulate cancer progression. Although growing evidence has revealed that tRNA modifications and related genes participate in this process, their roles in head and neck squamous cell carcinoma (HNSCC) remain largely uncharacterized. Here we found that tRNA m7G methyltransferase complex components METTL1/WDR4 were both upregulated in HNSCC and associated with poor prognosis. Functionally, METTL1/WDR4 promoted HNSCC progression and metastasis in cell-based and transgenic mouse models. Mechanistically, ablation of METTL1 reduced m7G levels of 16 tRNAs, causing translational inhibition of a subset of oncogenic transcripts, including the genes related to PI3K/AKT/mTOR signaling pathway. In addition, chemical modulators of PI3K/AKT/mTOR signaling pathway can reverse the effects of Mettl1 in HNSCC. Furthermore, single-cell RNA sequencing results revealed that depletion of Mettl1 in tumor cells altered the immune landscape and cell-cell interaction between tumor and stromal compartment. In summary, this study uncovered the physiological function and mechanism of mis-translation regulation mediated by tRNA m7G modification in HNSCC, and suggested that targeting METTL1 could be a promising treatment strategy for HNSCC patients.', 'Series_overall_design': 'TRAC-Seq were developed to identify the tRNA m7G methylome in HNSCC cell line.  Polyribosome-bound-RNA-seq  was used to study the differentially translated genes in the METTL1 knockout and control cells. Ribo-seq was performed to study the differentially translated genes and ribosome processivity in the METTL1 knockout and control cells. scRNA-seq of murine HNSCC from 4NQO-induced Mettl1cKO and wild type transgenic mice.', 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'Demeng,,Chen', 'Series_sample_id': 'GSM5242709', 'Series_contact_name': 'Jie,,Chen', 'Series_contact_email': 'chenj655@mail2.sysu.edu.cn', 'Series_contact_phone': '15243695906', 'Series_contact_institute': 'The First Affiliated Hospital of Sun Yat-sen University', 'Series_contact_address': 'No. 58, Zhongshan 2nd Road, Yuexiu District, Guangzhou', 'Series_contact_city': 'Guangzhou', 'Series_contact_zip/postal_code': '510080', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE172nnn/GSE172146/suppl/GSE172146_tRNA_expression_for_TRAC-seq.txt.gz', 'Series_platform_id': 'GPL24247', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP314965'}\n",
      "465 PRJNA723012 GSE172354 {'Series_title': 'Ribosome Profiling in Isoleucine tRNA Isoswitch Cells', 'Series_geo_accession': 'GSE172354', 'Series_status': 'Public on Apr 01 2023', 'Series_submission_date': 'Apr 19 2021', 'Series_last_update_date': 'Jul 01 2023', 'Series_pubmed_id': '36510010', 'Series_summary': 'Ribosome Profiling was used to evaluated whether changes in abundance of tRNAIleUAU and tRNAIleGAU would affect codon dependent translation and quantify dwell times over isoleucyl codons', 'Series_overall_design': 'Examination of ribosomal occupancy of isoleucyl codon enriched transcripts under Isoleucine tRNA modulation (increased tRNAIleUAU expression and decreased tRNAIleGAU expression via CRISPRi)', 'Series_type': 'Other', 'Series_contributor': 'Sohail,,Tavazoie', 'Series_sample_id': 'GSM5253080', 'Series_contact_name': 'Hani,,Goodarzi', 'Series_contact_department': 'Biochemistry and Biophysics', 'Series_contact_institute': 'UCSF', 'Series_contact_address': '600 16th St, GH S312D', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE172nnn/GSE172354/suppl/GSE172354_RAW.tar', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP315393'}\n",
      "466 PRJNA725118 GSE173283 {'Series_title': 'The human mitochondrial translatome promotes mitonuclear balance', 'Series_geo_accession': 'GSE173283', 'Series_status': 'Public on May 30 2021', 'Series_submission_date': 'Apr 26 2021', 'Series_last_update_date': 'Aug 24 2022', 'Series_pubmed_id': '35945592', 'Series_summary': 'Oxidative phosphorylation (OXPHOS) complexes consist of nuclear- and mitochondrial- encoded subunits, forcing cross-compartment synchronization of gene expression to achieve mitonuclear balance. To characterize how human cells coordinate OXPHOS gene expression, we establish a mitoribosome profiling approach that resolves features of the mitochondrial translatome, including the synthesis of a four amino acid mitopeptide. Strikingly, average mitochondrial and cytoplasmic synthesis rates correspond precisely across OXPHOS complexes. Coordinated mitochondrial and cytosolic synthesis does not require rapid feedback between the two translation systems. By contrast, we find that the Leigh syndrome gene, LRPPRC, maintains correlated mitochondrial and cytosolic translatomes. Our results lead to a model of human mitonuclear balance that requires tightly balanced cross-compartment protein synthesis, representing a vulnerability for cellular proteostasis.', 'Series_overall_design': 'Mitochondrial ribosome profiling in 5 human cells types, as well as cytosolic ribosome profiling and RNA-seq in select cell types. Drug and genetic perturbations also included', 'Series_type': 'Other', 'Series_contributor': 'Stirling,L,Churchman', 'Series_sample_id': 'GSM5264889', 'Series_contact_name': 'Mary,T,Couvillion', 'Series_contact_email': 'mtcouvi@gmail.com', 'Series_contact_laboratory': 'Stirling Churchman', 'Series_contact_department': 'Genetics', 'Series_contact_institute': 'Harvard Medical School', 'Series_contact_address': '77 Avenue Louis Pasteur', 'Series_contact_city': 'Boston', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02115', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE173nnn/GSE173283/suppl/GSE173283_RAW.tar', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP316314'}\n",
      "467 PRJNA725264 GSE173373 {'Series_title': \"Nucleolin HITS-CLIP, RNA-Seq and Ribo-Seq upon inhibition of 5'-tRFCys\", 'Series_geo_accession': 'GSE173373', 'Series_status': 'Public on Jun 03 2022', 'Series_submission_date': 'Apr 26 2021', 'Series_last_update_date': 'Jun 03 2022', 'Series_pubmed_id': '35654044', 'Series_summary': \"Expression of 5'-tRFCys is significantly upregulated during breast cancer metastasis and suppression of 5'-tRFCys significantly reduced metastatic lung colonization of breast cancer. We identified Nucleolin (Ncl) as the direct binding partner of 5'-tRFCys. To identify the molecular mechanism underlying 5'-tRFCys promotion of breast cancer, we performed Ncl HITS-CLIP, RNA-Seq, Ribosome profiling (Ribo-Seq) upon inhibition of 5'-tRFCys. These analyses reveal that 5'-tRFCys enhances Ncl binding to a subset of transcripts, resulting in increased transcript stability of those transcripts and expression.\", 'Series_overall_design': \"To identify tRNA fragments (tRFs) that regulate metastasis, tRFs were profiled from non-metastatic 67NR, poorly-metastatic 4TO7 or highly-metastatic 4T1 breast cancer cells using small RNA-Seq (sRNA-Seq). To identify the direct targets of 5'-tRFCys, Ncl HITS-CLIP was performed upon 5'-tRFCys suppression. To determine the impact of 5'-tRFCys on its target transcripts, 4T1 cells were transfected with a scrambled control or 5'-tRFCys antisense LNA oligo and RNA-Seq as well as Ribo-Seq were performed.\", 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'Sohail,,Tavazoie', 'Series_sample_id': 'GSM5266675', 'Series_contact_name': 'Xuhang,,Liu', 'Series_contact_laboratory': 'Laboratory of Systems Cancer Biology', 'Series_contact_institute': 'The Rockefeller University', 'Series_contact_address': '1230 York Ave', 'Series_contact_city': 'New York', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '10065', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE173nnn/GSE173373/suppl/GSE173373_srna-seq_5trh_67nr_4to7_deseq2.txt.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP316437'}\n",
      "468 PRJNA726548 GSE173654 {'Series_title': 'Translation variation across genetic backgrounds reveals a post-transcriptional buffering signature in yeast', 'Series_geo_accession': 'GSE173654', 'Series_status': 'Public on Feb 01 2024', 'Series_submission_date': 'Apr 30 2021', 'Series_last_update_date': 'Apr 02 2024', 'Series_pubmed_id': '38261993', 'Series_summary': 'Gene expression is known to vary among individuals, and this variability can impact the phenotypic diversity observed in natural populations. While the transcriptome and proteome have been extensively studied, little is known about the translation process itself. Here, we therefore performed ribosome and transcriptomic profiling on a genetically and ecologically diverse subset of natural isolates of the Saccharomyces cerevisiae yeast. Interestingly, we found that the Euclidean distances between each profile and the expression fold changes in each pairwise isolate comparison were approximately 10% higher at the transcriptomic level. This observation clearly indicates that the transcriptional variation observed in the different isolates is buffered through a phenomenon known as post-transcriptional buffering at the translation level. Furthermore, this phenomenon seemed to have a specific signature by preferentially affecting essential genes as well as genes involved in complex-forming proteins, and low transcribed genes. We also explored the translation of the S. cerevisiae pangenome and found that the accessory genes related to introgression events displayed similar transcription and translation levels as the core genome. By contrast, genes acquired through horizontal gene transfer events tended to be less efficiently translated. Together, our results highlight both the extant and signature of the post- transcriptional buffering.', 'Series_overall_design': 'Ribosome profiling and RNA-seq of eight natural Saccharomyces cerevisiae isolates', 'Series_type': 'Other', 'Series_contributor': 'Joseph,,Schacherer', 'Series_sample_id': 'GSM7882982', 'Series_contact_name': 'Shintaro,,Iwasaki', 'Series_contact_email': 'shintaro.iwasaki@riken.jp', 'Series_contact_laboratory': 'RNA Systems Biochemistry Laboratory', 'Series_contact_department': 'Center for Pioneering Research', 'Series_contact_institute': 'RIKEN', 'Series_contact_address': '2-1, Hirosawa', 'Series_contact_city': 'Wako', 'Series_contact_state': 'Saitama', 'Series_contact_zip/postal_code': '3510198', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE173nnn/GSE173654/suppl/GSE173654_RAW.tar', 'Series_platform_id': 'GPL26171', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP318024'}\n",
      "469 PRJNA729461 GSE174329 {'Series_title': 'Combinatorial analyis of translation dynamics reveals eIF2 dependence of translation initiation at near-cognate codons', 'Series_geo_accession': 'GSE174329', 'Series_status': 'Public on Jul 01 2021', 'Series_submission_date': 'May 12 2021', 'Series_last_update_date': 'Jul 27 2021', 'Series_pubmed_id': '34226921', 'Series_summary': 'Although ribosome-profiling and translation initiation sequencing (TI-seq) analyses have identified many noncanonical initiation codons, the precise detection of translation initiation sites (TISs) remains a challenge, mainly because of experimental artifacts of such analyses. Here we describe a new method, TISCA (TIS detection by translation Complex Analysis), for the accurate identification of TISs. TISCA proved to be more reliable for TIS detection compared with existing tools, and it identified a substantial number of near-cognate codons in Kozak-like sequence contexts. Analysis of proteomics data revealed the presence of methionine at the NH2-terminus of most proteins derived from near-cognate initiation codons. Although eukaryotic initiation factor 2 (eIF2), eIF2A, and eIF2D have previously been shown to contribute to translation initiation at near-cognate codons, we found that most noncanonical initiation events are dependent on eIF2, consistent with the initial amino acid being methionine. Comprehensive identification of TISs by TISCA should facilitate characterization of the mechanism of noncanonical initiation.', 'Series_overall_design': 'TISCA (TIS detection by translation Complex Analysis), for the accurate identification of TISs', 'Series_type': 'Other', 'Series_contributor': 'K,,Nakayama', 'Series_sample_id': 'GSM5291939', 'Series_contact_name': 'Kazuya,,Ichihara', 'Series_contact_email': 'kazuya1@bioreg.kyushu-u.ac.jp', 'Series_contact_institute': 'Kyushu University', 'Series_contact_address': '3-1-1, Maidashi', 'Series_contact_city': 'Higashi-ku, Fukuoka', 'Series_contact_zip/postal_code': '812-8582', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE174nnn/GSE174329/suppl/GSE174329_Counts_Table.csv.gz', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP319472'}\n",
      "470 PRJNA729866 GSE174419 {'Series_title': 'Mapping translation in wild-type and METTL5 knockout (KO) HepG2 cells', 'Series_geo_accession': 'GSE174419', 'Series_status': 'Public on May 19 2021', 'Series_submission_date': 'May 13 2021', 'Series_last_update_date': 'Feb 26 2022', 'Series_pubmed_id': '35033535', 'Series_summary': 'We performed ribosome profiling on wild-type and METTL5 KO HepG2 cells to investigate translation changes due to loss of METTL5.', 'Series_overall_design': 'Comparing sequencing of ribosome-protected-fragments (RPFs, normalized to input RNA-seq) from wild type and METTL5 KO HepG2.', 'Series_type': 'Other', 'Series_contributor': 'Chuan,,He', 'Series_sample_id': 'GSM5310251', 'Series_contact_name': 'Caraline,,Sepich', 'Series_contact_email': 'csepich@uchicago.edu', 'Series_contact_laboratory': 'GCIS E307 (He lab)', 'Series_contact_department': 'Chemistry', 'Series_contact_institute': 'University of Chicago', 'Series_contact_address': '929 E 57th Street', 'Series_contact_city': 'Chicago', 'Series_contact_state': 'IL', 'Series_contact_zip/postal_code': '60637', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE174nnn/GSE174419/suppl/GSE174419_Sepich-Poore_METTL5_HepG2_riboseq_input_merged_featurecount_revised_2021.txt.gz', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP319885'}\n",
      "471 PRJNA731067 GSE174679 {'Series_title': 'Adaptation to chronic ER stress enforces pancreatic beta-cell plasticity', 'Series_geo_accession': 'GSE174679', 'Series_status': 'Public on May 20 2021', 'Series_submission_date': 'May 19 2021', 'Series_last_update_date': 'Aug 24 2022', 'Series_pubmed_id': '35941159', 'Series_summary': 'The next-generation-sequencing (NGS) of ribosome footprintings and poly(A) enriched RNA-seq have been proformed to examine transcriptional and translational changes in an mouse insulinoma cell MIN6 under acute and chronic endoplasmic reticular stresses.', 'Series_overall_design': 'Mouse insulinoma cell MIN6 treated with 200uM of Cyclopiazonic acid for 1 (CPA1) and 18 hours (CPA18). An 18 hour DMSO treatment served as the control (CPA0). Ribo-seq and poly(A) enriched RNA-seq (mRNA-seq) have been proformed. Three independent biological replications were proformed.', 'Series_type': 'Other', 'Series_contributor': 'Maria,,Hatzoglou', 'Series_sample_id': 'GSM5322383', 'Series_contact_name': 'Chien-Wen,,Chen', 'Series_contact_email': 'cxc1036@case.edu', 'Series_contact_laboratory': 'Maria Hatzoglou', 'Series_contact_department': 'Genetics and Genome Sciences', 'Series_contact_institute': 'Case Western Reserve University', 'Series_contact_address': '2109 Adelbert Road', 'Series_contact_city': 'Cleveland', 'Series_contact_state': 'OH', 'Series_contact_zip/postal_code': '44106', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE174nnn/GSE174679/suppl/GSE174679_MIN6-Batch3-RPKM.txt.gz', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP320479'}\n",
      "472 PRJNA731473 GSE174754 {'Series_title': 'Phospho-RNA sequencing with CircAID-p-seq', 'Series_geo_accession': 'GSE174754', 'Series_status': 'Public on Mar 09 2022', 'Series_submission_date': 'May 20 2021', 'Series_last_update_date': 'Mar 09 2022', 'Series_pubmed_id': '34850942', 'Series_summary': 'Accurate positional information concerning ribosomes and RNA binding proteins with respect to their transcripts is important to understand the global regulatory network underlying protein and RNA fate in living cells. Most footprinting approaches generate RNA fragments bearing a phosphate or cyclic phosphate groups at their 3′ end. Unfortunately, all current protocols for library preparation rely only on the presence of a 3′ hydroxyl group. Here, we developed circAID-p-seq, a PCR-free library preparation for 3′ phospho-RNA sequencing. We applied circAID-p-seq to ribosome profiling, which produces fragments protected by ribosomes after endonuclease digestion. CircAID-p-seq, combined with the dedicated computational pipeline circAidMe, facilitates accurate, fast, highly efficient and low-cost sequencing of phospho-RNA fragments from eukaryotic cells and tissues. While assessing circAID-p-seq to portray ribosomes engaged with transcripts, we provide a versatile tool to unravel any 3′-phospho RNA molecules.', 'Series_overall_design': \"we design and validate a new library preparation for sequencing of 3'P or 2'/3' cP smallRNA fragments, using Oxford Nanopore platform\", 'Series_type': 'Other', 'Series_contributor': 'Massimiliano,,Clamer', 'Series_sample_id': 'GSM5325973', 'Series_contact_name': 'Alessia,,Del Piano', 'Series_contact_email': 'alessiadelpiano7@gmail.com, adelpiano@immaginabiotech.com, alessia.delpiano@unitn.it', 'Series_contact_phone': '0461312018', 'Series_contact_institute': 'IMMAGINA BioTECHNOLOGY', 'Series_contact_address': 'Via Sommarive 18', 'Series_contact_city': 'Trento', 'Series_contact_state': 'Trento', 'Series_contact_zip/postal_code': '38123', 'Series_contact_country': 'Italy', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE174nnn/GSE174754/suppl/GSE174754_LIVER.xlsx', 'Series_platform_id': 'GPL24973', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP320668'}\n",
      "473 PRJNA731488 GSE174764 {'Series_title': 'eIF2B-capturing viral protein NSs suppresses the integrated stress response', 'Series_geo_accession': 'GSE174764', 'Series_status': 'Public on Nov 11 2021', 'Series_submission_date': 'May 20 2021', 'Series_last_update_date': 'Feb 10 2022', 'Series_pubmed_id': '34876589', 'Series_summary': 'Various stressors such as viral infection lead to the suppression of cap-dependent translation and the activation of the integrated stress response (ISR), since the stress-induced phosphorylated eukaryotic translation initiation factor 2 [eIF2(αP)] tightly binds to eIF2B to prevent it from exchanging guanine nucleotide molecules on its substrate, unphosphorylated eIF2. Sandfly fever Sicilian virus (SFSV) evades this cap-dependent translation suppression through the interaction between its nonstructural protein NSs and host eIF2B. However, its precise mechanism has remained unclear. Here, our cryo-electron microscopy (cryo-EM) analysis revealed that SFSV NSs binds to the α-subunit of eIF2B in a competitive manner with eIF2(αP). Together with SFSV NSs, eIF2B retains nucleotide exchange activity even in the presence of eIF2(αP), in line with the cryo-EM structures of the eIF2B•SFSV NSs•unphosphorylated eIF2 complex. A genome-wide ribosome profiling analysis clarified that SFSV NSs expressed in cultured human cells attenuates the ISR triggered by thapsigargin, an endoplasmic reticulum stress inducer. Furthermore, SFSV NSs introduced in rat hippocampal neurons and human induced-pluripotent stem (iPS) cell-derived motor neurons exhibited neuroprotective effects against the ISR-inducing stress. Since ISR inhibition is beneficial in various neurological disease models, SFSV NSs is promising as a therapeutic ISR inhibitor.', 'Series_overall_design': 'Ribosome profiling and RNA-seq of SFSV NSs-expressing cells upon endoplasmic reticulum stress', 'Series_type': 'Other', 'Series_contributor': 'Takuhiro,,Ito', 'Series_sample_id': 'GSM5326945', 'Series_contact_name': 'Shintaro,,Iwasaki', 'Series_contact_email': 'shintaro.iwasaki@riken.jp', 'Series_contact_laboratory': 'RNA Systems Biochemistry Laboratory', 'Series_contact_department': 'Center for Pioneering Research', 'Series_contact_institute': 'RIKEN', 'Series_contact_address': '2-1, Hirosawa', 'Series_contact_city': 'Wako', 'Series_contact_state': 'Saitama', 'Series_contact_zip/postal_code': '3510198', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE174nnn/GSE174764/suppl/GSE174764_RAW.tar', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP320683'}\n",
      "474 PRJNA732080 GSE175363 {'Series_title': 'Intergenerational hormesis is regulated by heritable 18S rRNA methylation  18S rRNA methylation is heritably transmitted to regulate intergenerational hormesis', 'Series_geo_accession': 'GSE175363', 'Series_status': 'Public on May 01 2022', 'Series_submission_date': 'May 22 2021', 'Series_last_update_date': 'May 01 2022', 'Series_summary': 'Heritable non-genetic information can regulate a variety of complex phenotypes. However, what specific non-genetic cues are transmitted from parents to their descendants are unknown. Here, we perform metabolic methyl-labelling experiments to track the heritable transmission of methylation from ancestors to their descendants in the nematode Caenorhabditis elegans. We find that methylation is transmitted to descendants in proteins, RNA, DNA and lipids. We further find that in response to parental starvation, naïve progeny display reduced fertility, increased heat stress resistance, and extended longevity. This intergenerational hormesis is accompanied by a heritable increase in N6’-dimethyl adenosine (m6,2A) on the 18S ribosomal RNA at adenosines 1735 and 1736. We identified DIMT-1 as the m6,2A methyltransferase in C. elegans and find that dimt-1 is required for the intergenerational hormesis phenotypes. Together this study provides the first labeling and tracking of heritable non-genetic material across generations and demonstrates the importance of rRNA methylation for regulating the heritable response to starvation', 'Series_overall_design': 'Matched mRNAseq and Ribo-seq (ribosome profiling). Two experiments: Fed vs. Starved (6 replicates each); Empty vector vs two targeted knockdowns (4 replicates each)', 'Series_type': 'Other', 'Series_contributor': 'Eric,L,Greer', 'Series_sample_id': 'GSM5331581', 'Series_contact_name': 'Maxim,,Gerashchenko', 'Series_contact_email': 'mgerashchenko@bwh.harvard.edu', 'Series_contact_department': 'Medicine', 'Series_contact_institute': \"Brigham and Women's Hospital\", 'Series_contact_address': '77 Louis Pasteur Ave, NRB 435', 'Series_contact_city': 'Boston', 'Series_contact_state': 'Massachusetts', 'Series_contact_zip/postal_code': '02115', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE175nnn/GSE175363/suppl/GSE175363_ribocounts.csv.gz', 'Series_platform_id': 'GPL22765', 'Series_platform_taxid': '6239', 'Series_sample_taxid': '6239', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP320955'}\n",
      "475 PRJNA734003 GSE175827 {'Series_title': 'Function and Mechanism of METTL5-mediated 18S rRNA m6A1832 modification in hepatocellular carcinoma', 'Series_geo_accession': 'GSE175827', 'Series_status': 'Public on Jun 01 2021', 'Series_submission_date': 'May 31 2021', 'Series_last_update_date': 'Jun 01 2021', 'Series_summary': 'Tumor recurrence is main pattern of treatment failure for early-stage hepatocellular carcinoma (HCC). However, the molecular mechanisms underlying disease recurrence remain poorly understood. Here, we showed that 18S rRNA N6-methyladenosine (m6A1832) modification and its methyltransferase complex METTL5/TRMT112 were upregulated in HCC and correlated with poor prognosis. Loss-of-function and gain-of-function assays demonstrated that METTL5/TRMT112 mediated 18S rRNA m6A1832 modification promotes HCC tumorigenesis in vitro and in vivo. Mechanistically, 18S rRNA m6A1832 modification selectively regulated the translation of mRNAs with long 5’UTR and short 3’UTR through affecting the assembly of 80S subunit at translation initiation and its dissociation at translation termination which was executed by weakening the interaction of ABCE1 with eRF1 and eRF3. Moreover, METTL5-mediated 18S rRNA m6A1832 modification regulated β-oxidation of long-chain fatty acid through ACSL4 to promote HCC progression. Our work uncovered a novel layer of mRNA translation regulation mechanism at ribosome 80S subunit assembly and dissociation step mediated by 18S rRNA m6A1832 modification and revealed a new crosslink between RNA epigenetic modification and fatty acid metabolism in HCC.', 'Series_overall_design': 'Ribo-seq was used to study the differential translated genes in the METTL5 knockout and control cells.', 'Series_type': 'Other', 'Series_contributor': 'Shuibin,,Lin', 'Series_sample_id': 'GSM5348510', 'Series_contact_name': 'Shuibin,,Lin', 'Series_contact_email': 'linshb6@mail.sysu.edu.cn', 'Series_contact_department': 'Clincial for Translational Medicine', 'Series_contact_institute': 'The First Affliliated Hospital, Sun Yat-sen University', 'Series_contact_address': 'Zhongshan 2nd road, Guangzhou', 'Series_contact_city': 'Guangzhou', 'Series_contact_state': 'China', 'Series_contact_zip/postal_code': '510080', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE175nnn/GSE175827/suppl/GSE175827_Ribo-seq_reads.txt.gz', 'Series_platform_id': 'GPL20795', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP321996'}\n",
      "476 PRJNA734658 GSE176058 {'Series_title': 'Bi-directional ribosome scanning controls the stringency of start codon selection', 'Series_geo_accession': 'GSE176058', 'Series_status': 'Public on Nov 24 2021', 'Series_submission_date': 'Jun 02 2021', 'Series_last_update_date': 'Nov 25 2021', 'Series_pubmed_id': '34782646', 'Series_summary': 'The fidelity of start codon recognition by ribosomes is paramount during protein synthesis. The textbook knowledge of eukaryotic translation initiation depicts 5’→3’ unidirectional migration of the pre-initiation complex (PIC) along the 5’UTR. In probing translation initiation from ultra-short 5’UTR, we report that an AUG triplet near the 5’ end can be selected via PIC backsliding. The bi-directional ribosome scanning is supported by competitive selection of closely spaced AUG codons and recognition of two initiation sites flanking an internal ribosome entry site. Transcriptome-wide PIC profiling reveals footprints with an oscillation pattern near the 5’ end and start codons. Depleting the RNA helicase eIF4A leads to reduced PIC oscillations and impaired selection of 5’ end start codons. Enhancing the ATPase activity of eIF4A promotes nonlinear PIC scanning and stimulates upstream translation initiation. The helicase-mediated PIC conformational switch may provide an operational mechanism that unifies ribosome recruitment, scanning, and start codon selection.', 'Series_overall_design': 'Ribo-seq, 40S-seq and eIF3-seq of MEF and HEK293 cells with or without eIF4A1', 'Series_type': 'Other', 'Series_contributor': 'Shu-Bing,,Qian', 'Series_sample_id': 'GSM5597397', 'Series_contact_name': 'Yuanhui,,Mao', 'Series_contact_email': 'maoyuanhui123@gmail.com', 'Series_contact_institute': 'Zhejiang University', 'Series_contact_address': '1369 West Wenyi Road', 'Series_contact_city': 'Hangzhou', 'Series_contact_state': 'Zhejiang', 'Series_contact_zip/postal_code': '310000', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE176nnn/GSE176058/suppl/GSE176058_RAW.tar', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP322410'}\n",
      "477 PRJNA734953 GSE176130 {'Series_title': 'Charged tRNA profiling of mcf10a, HCC1806 cells, polysome profiling, RNA-Seq and Ribo-Seq in 4T07 cells upon LARS knockdown by shRNA, and RNA-Seq of ribosome-associated RNAs in Lars depleted PyMT tumors', 'Series_geo_accession': 'GSE176130', 'Series_status': 'Public on Jan 05 2022', 'Series_submission_date': 'Jun 03 2021', 'Series_last_update_date': 'Jan 05 2022', 'Series_summary': 'In this study, we identify leucyl-tRNA synthetase (LARS) as a breast tumor suppressor. To identify the mechanism underlying LARS-mediated breast tumor suppression, we first conducted charged tRNA profiling to assess charged and total tRNA-Leu expression in cell lines with high and low LARS expression. Expression of select charged and total tRNA-Leu isoacceptors were reduced in cell lines with lower LARS expression. We then conducted RNA-Seq and translation profiling including ribosome profiling (Ribo-Seq), polysome profiling in LARS-depleted cell lines. To test our hypothesis in vivo, we also performed RNA-Seq of ribosome-associated RNAs in Lars depleted PyMT tumors within RiboTag mice. The analyses implicate LARS as a regulator of leucine-rich translation.', 'Series_overall_design': 'To identify changes in tRNA-Leu expression, charged tRNA profiling was conducted in LARS-high non-transformed mcf10a cells compared to LARS-low HCC1806 breast cancer cells using charged tRNA profiling. To identify the impact of LARS on protein translation, RNA-Seq, Ribo-Seq and Polysome profiling were conducted in LARS-depleted 4T07 breast cancer cells. To assess the effects of LARS on translation in vivo, RNA-Seq was conducted on ribosome-associated mRNAs isolated from RiboTag mice harboring PyMT tumors, with and without monoallelic LARS deletion in the mammary tumor compartment.', 'Series_type': 'Other', 'Series_contributor': 'Sohail,,Tavazoie', 'Series_sample_id': 'GSM5356693', 'Series_contact_name': 'Maria,,Passarelli', 'Series_contact_laboratory': 'Laboratory of Systems Cancer Biology', 'Series_contact_institute': 'Rockefeller University', 'Series_contact_address': 'Rockefeller University, 1230 York Ave', 'Series_contact_city': 'New York', 'Series_contact_state': 'New York', 'Series_contact_zip/postal_code': '10065', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE176nnn/GSE176130/suppl/GSE176130_tRNAprof_mcf_HCC_tot_deseq2_result.txt.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP322534'}\n",
      "478 PRJNA740301 GSE178760 {'Series_title': 'Compromised nonsense-mediated RNA decay results in truncated RNA-binding protein production upon DUX4 expression', 'Series_geo_accession': 'GSE178760', 'Series_status': 'Public on Jun 29 2021', 'Series_submission_date': 'Jun 23 2021', 'Series_last_update_date': 'Jun 22 2023', 'Series_pubmed_id': '37314931', 'Series_web_link': 'https://doi.org/10.1016/j.celrep.2023.112642', 'Series_summary': 'Nonsense-mediated RNA decay (NMD) degrades transcripts carrying premature termination codons. NMD is thought to prevent the synthesis of toxic truncated proteins. However, whether loss of NMD results in widespread production of truncated proteins is unclear. A human genetic disease, facioscapulohumeral muscular dystrophy (FSHD), features acute inhibition of NMD upon expression of the disease-causing transcription factor, DUX4. Using a cell-based model of FSHD, we show production of truncated proteins from physiological NMD targets and find that RNA-binding proteins are enriched for aberrant truncations. The NMD isoform of one RNA-binding protein, SRSF3, is translated to produce a stable truncated protein, which is detected in FSHD patient-derived myotubes. Ectopic expression of truncated SRSF3 confers toxicity, and its downregulation is cytoprotective. Our results delineate the genome-scale impact of NMD loss. This widespread production of potentially deleterious truncated proteins has implications for FSHD biology as well as other genetic diseases where NMD is therapeutically modulated.', 'Series_overall_design': 'Ribosome profiling (Ribo-seq) of human myoblasts harboring an inducible DUX4 transgene was performed on ribosome-protected RNA samples collected from 3 separate biological replicates treated for 0, 4, 8, or 14 h with doxycycline to induce DUX4 expression.', 'Series_type': 'Other', 'Series_contributor': 'Sujatha,,Jagannathan', 'Series_sample_id': 'GSM5397176', 'Series_contact_name': 'Amy,E.,Campbell', 'Series_contact_email': 'amy.e.campbell@cuanschutz.edu', 'Series_contact_phone': '303-724-1651', 'Series_contact_institute': 'University of Colorado Anschutz Medical Campus', 'Series_contact_address': '12801 E 17th Ave, Room L18-10403G', 'Series_contact_city': 'Aurora', 'Series_contact_state': 'CO', 'Series_contact_zip/postal_code': '80045', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE178nnn/GSE178760/suppl/GSE178760_Ribo_te-table.csv.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP325334'}\n",
      "479 PRJNA743042 GSE179274 {'Series_title': 'AAV-delivered suppressor tRNA overcomes a nonsense mutation in mice (ribosome profiling)', 'Series_geo_accession': 'GSE179274', 'Series_status': 'Public on Nov 30 2021', 'Series_submission_date': 'Jul 01 2021', 'Series_last_update_date': 'Apr 22 2022', 'Series_pubmed_id': '35322228', 'Series_summary': 'In this study, we utilized a mouse model of mucopolysaccharidosis type I (MPS-I) carrying a homozygous nonsense mutation in the Idua gene (Idua-W401X, TGG→TAG) to develop rAAV-delivered sup-tRNA therapeutics that can overcome a pathogenic UAG premature termination codon (PTC).', 'Series_overall_design': 'Liver ribosome profilings in sup-tRNA treated and untreated homozygous Idua-W401X mice and patient fibroblasts.', 'Series_type': 'Other', 'Series_contributor': 'Yue,,Zhang', 'Series_sample_id': 'GSM5412825', 'Series_contact_name': 'Yue,,Zhang', 'Series_contact_email': '6leensky@gmail.com', 'Series_contact_institute': 'Umass medical school', 'Series_contact_address': 'GeneTherapyCenter, 368 Plantation st', 'Series_contact_city': 'Worcester', 'Series_contact_state': 'Massachusetts', 'Series_contact_zip/postal_code': '01605', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE179nnn/GSE179274/suppl/GSE179274_Ribo_PFB_rep2_countTable.tar.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP326500'}\n",
      "480 PRJNA744147 GSE179525 {'Series_title': 'Monitoring the 5′UTR landscape reveals isoform switches to drive translational efficiencies in cancer', 'Series_geo_accession': 'GSE179525', 'Series_status': 'Public on Dec 08 2022', 'Series_submission_date': 'Jul 06 2021', 'Series_last_update_date': 'Mar 10 2023', 'Series_pubmed_id': '36550360', 'Series_summary': 'Transcriptional and translational control are key determinants of gene expression, however, to what extent these two processes can be collectively coordinated is still poorly understood. Here, we use Nanopore long-read sequencing and cap analysis of gene expression (CAGE-seq) to document the landscape of 5′ and 3′ untranslated region (UTR) isoforms and transcription start sites of epidermal stem cells, wild-type keratinocytes and squamous cell carcinomas. Focusing on squamous cell carcinomas, we show that a small cohort of genes with alternative 5′UTR isoforms exhibit overall increased translational efficiencies and are enriched in ribosomal proteins and splicing factors. By combining polysome fractionations and CAGE-seq, we further characterize two of these UTR isoform genes with identical coding sequences and demonstrate that the underlying transcription start site heterogeneity frequently results in 5′ terminal oligopyrimidine (TOP) and pyrimidine-rich translational element (PRTE) motif switches to drive mTORC1-dependent translation of the mRNA. Genome-wide, we show that highly translated squamous cell carcinoma transcripts switch towards increased use of 5′TOP and PRTE motifs, have generally shorter 5′UTRs and expose decreased RNA secondary structures. Notably, we found that the two 5′TOP motif-containing, but not the TOP-less, RPL21 transcript isoforms strongly correlated with overall survival in human head and neck squamous cell carcinoma patients. Our findings warrant isoform-specific analyses in human cancer datasets and suggest that switching between 5′UTR isoforms is an elegant and simple way to alter protein synthesis rates, set their sensitivity to the mTORC1-dependent nutrient-sensing pathway and direct the translational potential of an mRNA by the precise 5′UTR sequence.', 'Series_overall_design': 'RNA from mouse P56-60 epidermis, cultured WT and HRASG12V; TGFBR-2 KO keratinocytes was isolated and used for NANOPORE long-read RNA sequencing. Cultured WT and HRASG12V; TGFBR-2 KO keratinocytes were used for ribosome profiling and short-read RNA sequencing.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Ataman,,Sendoel', 'Series_sample_id': 'GSM5419827', 'Series_contact_name': 'Ramona,,Weber', 'Series_contact_email': 'ramona.weber@uzh.ch', 'Series_contact_department': 'Institute for Regenerative Medicine', 'Series_contact_institute': 'University of Zurich', 'Series_contact_address': 'Wagistrasse 12', 'Series_contact_city': 'Schlieren', 'Series_contact_zip/postal_code': '8952', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE179nnn/GSE179525/suppl/GSE179525_RAW.tar', 'Series_platform_id': 'GPL26624', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP327136'}\n",
      "481 PRJNA744888 GSE179744 {'Series_title': \"Novel RNA-binding Proteins Bind U1 snRNA to Promote Splicing of Weak 5' Splice Sites\", 'Series_geo_accession': 'GSE179744', 'Series_status': 'Public on May 04 2023', 'Series_submission_date': 'Jul 08 2021', 'Series_last_update_date': 'May 09 2023', 'Series_pubmed_id': '37137667', 'Series_summary': \"Removal of introns during pre-mRNA splicing, which is central to gene expression, initiates by base pairing of U1 snRNA with a 5' splice site (5'SS). In mammals, many introns contain weak 5'SSs that are not efficiently recognized by the canonical U1 snRNP, suggesting alternative mechanisms exist. Here, we develop a cross-linking immunoprecipitation coupled to a high-throughput sequencing method, BCLIP-seq, to identify NRDE2 (Nuclear RNAi defective-2) and CCDC174 (Coiled-Coil Domain-Containing 174) as novel RNA-binding proteins in mouse ES cells that associate with U1 snRNA and unspliced 5'SSs. Both proteins bind directly to U1 snRNA independently of canonical U1 snRNP specific proteins, and they are required for the selection and effective processing of weak 5'SSs. Our results reveal that mammalian cells use non-canonical splicing factors bound directly to U1 snRNA to effectively select suboptimal 5'SS sequences in hundreds of genes, promoting proper splice site choice and accurate pre-mRNA splicing.\", 'Series_overall_design': 'All BCLIP-seq, RNA-seq and Ribo-seq experiments were performed in two independent replicates for each sample. BCLIP-seq was done with proteins endogenously tagged with a composite 3xFLAG+AviTag (3A tag). Nrde2 and Mtrex knockout were induced by treating the conditional knockout mES cells with 100 nM 4-hydroxytamoxifen for 4 and 3 days, respectively. Depletion of all the proteins fused to FKBP12F36V degron was achieved by treating the cells with 500 nM dTAG-13 compound for 24 hours. The indicated RNA-seq samples were treated for 4 hours with 0.1 mg/ml cycloheximide (CHX).', 'Series_type': 'Other', 'Series_contributor': 'Marc,,Bühler', 'Series_sample_id': 'GSM5467943', 'Series_contact_name': 'Fabio,,Mohn', 'Series_contact_email': 'fabio.mohn@fmi.ch', 'Series_contact_laboratory': 'Buehler Lab', 'Series_contact_institute': 'Friedrich Miescher Institute for Biomedical Research', 'Series_contact_address': 'Maulbeerstrasse 66', 'Series_contact_city': 'Basel', 'Series_contact_zip/postal_code': '4058', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE179nnn/GSE179744/suppl/GSE179744_Riboseq_cpm_per_gene.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP327516'}\n",
      "482 PRJNA747022 GSE180208 {'Series_title': 'Ribosome impairment regulates intestinal stem cell identity via ZAKɑ activation', 'Series_geo_accession': 'GSE180208', 'Series_status': 'Public on Jun 08 2022', 'Series_submission_date': 'Jul 15 2021', 'Series_last_update_date': 'Aug 16 2022', 'Series_pubmed_id': '35918345', 'Series_summary': 'We show that ribosome collisions, induced by Rptor deletion or aminoacid starvation, causes a ZAK alpha mediated identity switch in intestinal stem cells by activating a more fetal-like stem cell signature. This switch also causes changes in metabolic profiles of the cells, which is heavily regulated by RNA translation efficiency of different group of metabolic genes. These observations originate from performing in vitro Riboseq and RNAseq in mouse intestinal organoids, and in vivo Riboseq in Lgr5 expressing intestinal stem cells from WT and Rptor.fl/fl mice.', 'Series_overall_design': 'In vitro Ribo-seq and RNAseq in WT-stem cell enriched, WT-stem cell depleted and Rptor fl/fl mouse intestinal organoids. In vitro RNAseq in WT mouse intestinal organoids. In vivo Ribo-seq in Lgr5 expressing intestinal stem cells from WT and Rptor fl/fl mouse.', 'Series_type': 'Other', 'Series_contributor': 'Joana,,Silva', 'Series_sample_id': 'GSM5456067', 'Series_contact_name': 'William,,Faller', 'Series_contact_email': 'w.faller@nki.nl', 'Series_contact_phone': '+31 (0)20 512 2018', 'Series_contact_laboratory': 'Faller Lab', 'Series_contact_department': 'Oncogenomics', 'Series_contact_institute': 'Netherlands Cancer Institute', 'Series_contact_address': 'Plesmanlaan 121', 'Series_contact_city': 'Amsterdam', 'Series_contact_zip/postal_code': '1066CX', 'Series_contact_country': 'Netherlands', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE180nnn/GSE180208/suppl/GSE180208_RptorKO_vs_SCe_Diff_TE.RiboDiff.tsv.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP328552'}\n",
      "483 PRJNA747247 GSE180240 {'Series_title': 'Developmental Dynamics of RNA Translation in the Human Brain', 'Series_geo_accession': 'GSE180240', 'Series_status': 'Public on Aug 09 2022', 'Series_submission_date': 'Jul 16 2021', 'Series_last_update_date': 'Nov 29 2022', 'Series_pubmed_id': '36171426', 'Series_summary': 'The precise regulation of gene expression is fundamental to neurodevelopment, plasticity, and cognitive function. While several studies have profiled transcription in the developing human brain, there is a gap in our understanding of accompanying translational regulation. We performed ribosome profiling on 73 human prenatal and adult cortex samples. We characterized the translational regulation of annotated open reading frames (ORFs) and identified thousands of previously unknown translation events, including small ORFs that give rise to human- and/or brain-specific microproteins, many of which we independently verified using proteomics. Ribosome profiling in stem cell-derived human neuronal cultures corroborated these findings and revealed that several neuronal activity-induced non-coding RNAs encode previously undescribed microproteins. Physicochemical analysis of brain microproteins identified a class of proteins that contain arginine-glycine-glycine (RGG) repeats and thus may be regulators of RNA metabolism.  This resource expands the known translational landscape of the human brain and illuminates previously unknown brain-specific protein products.', 'Series_overall_design': 'Ribosome profiling of cultured human embryonic stem cell derived neurons', 'Series_type': 'Other', 'Series_contributor': 'Michael,E,Greenberg', 'Series_sample_id': 'GSM6131563', 'Series_contact_name': 'Erin,Elizabeth,Duffy', 'Series_contact_email': 'erin_duffylacy@hms.harvard.edu', 'Series_contact_phone': '(617)432-7795', 'Series_contact_laboratory': 'Michael E. Greenberg', 'Series_contact_department': 'Neurobiology', 'Series_contact_institute': 'Harvard Medical School', 'Series_contact_address': '200 Longwood Ave, Goldenson 405', 'Series_contact_city': 'Boston', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02115', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE180nnn/GSE180240/suppl/GSE180240_RAW.tar', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP328685'}\n",
      "484 PRJNA748058 GSE180400 {'Series_title': 'Balancing of mitochondrial translation through METTL8-mediated m3C modification of mitochondrial tRNAs', 'Series_geo_accession': 'GSE180400', 'Series_status': 'Public on Jan 02 2023', 'Series_submission_date': 'Jul 19 2021', 'Series_last_update_date': 'Jan 03 2023', 'Series_pubmed_id': '34774131', 'Series_summary': 'Mitochondria contain a specific translation machinery for the synthesis of respiratory chain components encoded on the mitochondrial genome. Mitochondrial tRNAs (mt-tRNAs) are also generated from the mitochondrial genome and, similar to their cytoplasmic counterparts, are modified at various positions. Here, we find that the RNA methyltransferase METTL8, is a mitochondrial protein that facilitates m3C methylation at position C32 of mt-tRNASer(UCN) and mt-tRNAThr. METTL8 knock out cells show reduced and over expressing cells enhanced respiratory chain activity. In pancreatic cancer, METTL8 levels are high, which correlates with patient survival. Indeed, METTL8 up regulation stimulates respiratory chain activity in these cells. Ribosome occupancy analysis using ribosome profiling revealed ribosome stalling on mt-tRNASer(UCN) and mt-tRNAThr codons and mass spectrometry analysis of native ribosomal subcomplexes unraveled reduced respiratory chain incorporation of the mitochondria encoded proteins ND6 and ND1. A well-balanced translation of mt-tRNASer(UCN) and mt-tRNAThr codons through METTL8-mediated C32 methylation might therefore provide optimal respiratory chain compositions and function.', 'Series_overall_design': '4 mitoribosome profiling datasets of HEK293 cells. One control dataset, two biological replicates containing METTL8 knockouts, and one with a inducible METTL8 overexpression', 'Series_type': 'Other', 'Series_contributor': 'Gunter,,Meister', 'Series_sample_id': 'GSM5462323', 'Series_contact_name': 'James,,Marks', 'Series_contact_email': 'James.marks@vanderbilt.edu', 'Series_contact_phone': '502-931-1657', 'Series_contact_institute': 'NIAMS/NIH', 'Series_contact_address': '638A Robinson Research Building', 'Series_contact_city': 'Nashville', 'Series_contact_state': 'Tennessee', 'Series_contact_zip/postal_code': '37232', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE180nnn/GSE180400/suppl/GSE180400_RAW.tar', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP328974'}\n",
      "485 PRJNA748710 GSE180565 {'Series_title': 'Dissecting long non-coding RNAs derived from microRNA genes in hematopoiesis (Illumina short-read sequencing)', 'Series_geo_accession': 'GSE180565', 'Series_status': 'Public on Oct 18 2023', 'Series_submission_date': 'Jul 21 2021', 'Series_last_update_date': 'Dec 08 2023', 'Series_pubmed_id': '37849233', 'Series_summary': \"Genome-wide transcription produces a complex population of transcripts and a wide variety of functional lncRNAs occasionally evolve from them. Here, we identify and characterize miRNA gene originated lncRNAs (molncRNAs) by employing Pacific Biosciences long-read sequencing. Combining with RNA-sequencing in differentiating hematopoietic erythrocytes, we construct a global molncRNA landscape during human erythropoiesis. Among them, miR-301b and miR-130b originated lncRNA, molnc-301b, orchestrates erythropoiesis independent of miRNAs. Mechanistically, molnc-301b attenuates the transcription of a series of erythropoietic and translation-associated genes by antagonizing SMARCA5's chromatin binding. Furthermore, molnc-301b perturbs translation of key erythroid mRNAs selectively, as GATA1 and FOS. Our findings identify a novel class of lncRNA and reveal the essential role of molnc-301b in erythroid differentiation. Of note, molnc-301b orchestrates erythropoiesis via jointly controlling the transcription and translation of key erythroid regulators. In addition, we finally establish functional molncRNA screen platform by using large-scale pooled CRISPR-based single-cell RNA-seq, and uncover 36 functional molncRNAs in erythrocytes. These results thus unearth a previously unannotated class of lncRNAs and provide a useful platform for the discovery and functional study of molncRNAs.\", 'Series_overall_design': 'smallRNA-seq, Poly(A)+ RNA-seq, ChIRP-seq, Ribo-seq and RNA-seq datasets.', 'Series_type': 'Other', 'Series_contributor': 'Jia,,Yu', 'Series_sample_id': 'GSM7785533', 'Series_contact_name': 'Yue,,Huo', 'Series_contact_email': 'yuehuo_hy@outlook.com', 'Series_contact_laboratory': 'Key Laboratory of RNA and Hematopoietic Regulation', 'Series_contact_department': 'State Key Laboratory of Medical Molecular Biology, Department of Biochemistry and Molecular Biology, Institute of Basic Medical Sciences', 'Series_contact_institute': 'Chinese Academy of Medical Sciences, School of Basic Medicine Peking Union Medical College', 'Series_contact_address': 'Dongcheng District, Dongdan 3 Tiao 5 Hao', 'Series_contact_city': 'Beijing', 'Series_contact_state': 'Beijing', 'Series_contact_zip/postal_code': '100005', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE180nnn/GSE180565/suppl/GSE180565_molnc301b_CHIRP-seq_peaks.txt.gz', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP329295'}\n",
      "486 PRJNA749053 GSE180669 {'Series_title': 'Ribosome profiling in DDX3X degron cell lines and complementation with sex-linked paralogs', 'Series_geo_accession': 'GSE180669', 'Series_status': 'Public on Nov 24 2021', 'Series_submission_date': 'Jul 22 2021', 'Series_last_update_date': 'Nov 25 2021', 'Series_pubmed_id': '34535544', 'Series_summary': 'We describe the subset of transcripts that require DDX3 for efficient translation and identify similarities and differences between X- and Y- linked paralogs of DDX3.', 'Series_overall_design': 'Ribosome profiling upon auxin-induced degradation of DDX3X, and covered with either DDX3X or DDX3Y was performed from HCT116 cells.', 'Series_type': 'Other', 'Series_contributor': 'Stephen,,Floor', 'Series_sample_id': 'GSM5467449', 'Series_contact_name': 'Srivats,,Venkataramanan', 'Series_contact_email': 'srivats88@gmail.com', 'Series_contact_phone': '4154763275', 'Series_contact_department': 'Cell and Tissue Biology', 'Series_contact_institute': 'University of California, San Francisco', 'Series_contact_address': '513 Parnassus Ave, HSW 740', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'California', 'Series_contact_zip/postal_code': '94143', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE180nnn/GSE180669/suppl/GSE180669_results_cds_25min_df_lentiCTRL_iaadmso.csv.gz', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP329509'}\n",
      "487 PRJNA749295 GSE180733 {'Series_title': 'Myocardial cellular gene expression during pressure overload: evidence for transient partial endothelial to mesenchymal transition [Ribo-seq]', 'Series_geo_accession': 'GSE180733', 'Series_status': 'Public on Feb 08 2022', 'Series_submission_date': 'Jul 23 2021', 'Series_last_update_date': 'Mar 24 2022', 'Series_pubmed_id': '35281736', 'Series_summary': 'Pathological processes underlying pressure overload triggered heart failure were mainly analyzed from a cardiomyocyte centric view. To identify new cellular mechanisms, we isolated cardiomyocytes, endothelial cells and fibroblasts as most abundant cardiac cell types from mice in the subacute and chronic stages of pressure overload (by transverse aortic constriction, TAC) and performed RNA-sequencing. We detected highly cell-type specific transcriptional responses with characteristic time courses and active intercellular communication, especially early after TAC. We focused on endothelial cells, and single-cell sequencing in this population showed enhanced expression of collagens and inflammatory mediators at this stage. Importantly, these endothelial transcriptional changes were transduced to the translational level as shown by Ribo-Tag sequencing and verification of collagen protein production by cardiac endothelial cells. In conclusion, we provide a resource of cardiac cellular gene expression in pressure overload and reveal the induction of collagens by endothelial cells as potential therapeutic target.', 'Series_overall_design': 'Ribo-tag mice were crossed with Cdh5-CreERT2 mice expressing the Cre recombinase under the control of the Cadherin 5 (Cdh5) promoter (Cdh5-CreERT2:Ribo-tag). At 9 weeks of age, male mice underwent transverse aortic constriction or sham operation', 'Series_type': 'Other', 'Series_contributor': 'Etienne,,Boileau', 'Series_sample_id': 'GSM5469206', 'Series_contact_name': 'Joerg,,Heineke', 'Series_contact_email': 'Joerg.Heineke@medma.uni-heidelberg.de', 'Series_contact_department': '– Department of Cardiovascular Research (HKF)', 'Series_contact_institute': 'Medizinische Fakultät Mannheim', 'Series_contact_address': 'Ludolf-Krehl-Str. 7-11', 'Series_contact_city': 'Mannheim', 'Series_contact_state': 'Baden Wuttemberg', 'Series_contact_zip/postal_code': '68167', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE180nnn/GSE180733/suppl/GSE180733_01EC_RiboT_Matrix_DE_Expression_Deseq2.xlsx', 'Series_platform_id': 'GPL21626', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP329630'}\n",
      "488 PRJNA750456 GSE181040 {'Series_title': 'Dynamic O-GlcNAcylation of eIF3a controls translation reinitiation during integrated stress response', 'Series_geo_accession': 'GSE181040', 'Series_status': 'Public on Nov 01 2021', 'Series_submission_date': 'Jul 28 2021', 'Series_last_update_date': 'Dec 30 2021', 'Series_pubmed_id': '34887587', 'Series_summary': 'In eukaryotic cells, many mRNAs possess upstream open reading frames (uORFs) in addition to the main coding region. After uORF translation, the ribosome could either recycle at the stop codon or resume scanning for downstream start codons in a process known as reinitiation. Accumulating evidence suggests that some initiation factors, including eIF3, linger on the early elongating ribosome, forming an eIF3-80S complex. Very little is known how the eIF3 is carried along with the 80S during elongation and its implications in subsequent translation reinitiation. Here we report that eIF3a undergoes dynamic O-GlcNAc modification in response to nutrient starvation. Stress-induced de-O-GlcNAcylation promotes eIF3 retention on the elongating ribosome and facilitates reinitiation at downstream start codons. Eliminating the modification site from eIF3a via genome editing induces ATF4 reinitiation under the nutrient rich condition. Our findings illustrate a mechanism in balancing ribosome recycling and reinitiation, thereby linking integrated stress response and translational reprogramming.', 'Series_overall_design': 'Ribo-seq and eIF3-seq of WT and OTG knockout MEF cells with or without starvation, and with or without ThiametG treatment.', 'Series_type': 'Other', 'Series_contributor': 'Shu-Bing,,Qian', 'Series_sample_id': 'GSM5482360', 'Series_contact_name': 'Yuanhui,,Mao', 'Series_contact_email': 'maoyuanhui123@gmail.com', 'Series_contact_institute': 'Zhejiang University', 'Series_contact_address': '1369 West Wenyi Road', 'Series_contact_city': 'Hangzhou', 'Series_contact_state': 'Zhejiang', 'Series_contact_zip/postal_code': '310000', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE181nnn/GSE181040/suppl/GSE181040_RAW.tar', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP330265'}\n",
      "489 PRJNA750636 GSE181077 {'Series_title': 'Integrated transcriptomic and translatomic exploration of the effect of betaine on lipid metabolic dysregulation induced by a high-fat diet', 'Series_geo_accession': 'GSE181077', 'Series_status': 'Public on Jul 31 2021', 'Series_submission_date': 'Jul 29 2021', 'Series_last_update_date': 'Jan 06 2022', 'Series_pubmed_id': '34708066', 'Series_summary': 'An excessive high-fat/energy diet is a major cause of obesity and associated complications, such as non-alcoholic fatty liver disease (NAFLD). Betaine has been shown to effectively improve hepatic lipid metabolism. However, the mechanistic basis for this improvement is largely unknown. Herein, integration of transcriptomics sequencing (RNA-seq) and ribosome footprints profiling (Ribo-seq) was used to investigate the means by which betaine alleviates hepatic lipid metabolic disorders induced by a high-fat diet. For the transcriptome, gene set enrichment analysis demonstrated betaine to reduce liver steatosis by up-regulation of fatty acid beta oxidation, lipid oxidation, and fatty acid catabolic processes. For the translatome, 574 differentially expressed genes were identified, 17 of which were associated with the NAFLD pathway. By combined analysis of RNA-seq and Ribo-seq, we found that betaine had the greatest effect on NAFLD at the translational level. Further, betaine decreased translational efficiency (TE) for IDI1, CYP5A1, TM7SF2, and APOA4, which are related to lipid biosynthesis. In summary, this study demonstrates betaine to alleviate lipid metabolic dysfunction at the translational level. The powerful multi-omics data integration approach used herein provides for a new understanding of the means by which to treat NAFLD.', 'Series_overall_design': 'High-throughput sequencing of liver tissue from high-fat and betaine treated high-fat mice.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Yixing,,Li', 'Series_sample_id': 'GSM5484318', 'Series_contact_name': 'Tengda,,Huang', 'Series_contact_email': '2022324065113@stu.scu.edu.cn', 'Series_contact_phone': '+8615208155933', 'Series_contact_department': 'West China Hospital', 'Series_contact_institute': 'Sichuan University', 'Series_contact_address': 'gaopeng road', 'Series_contact_city': 'chengdu', 'Series_contact_state': 'Sichuan', 'Series_contact_zip/postal_code': '610065', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE181nnn/GSE181077/suppl/GSE181077_RAW.tar', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP330402'}\n",
      "490 PRJNA752584 GSE181565 {'Series_title': 'Circadian Clock-Controlled Translation of Specific mRNAs in Neurospora crassa Requires Rhythmic eIF2α Activity and P-body Sequestration [Ribo-seq]', 'Series_geo_accession': 'GSE181565', 'Series_status': 'Public on Sep 23 2022', 'Series_submission_date': 'Aug 06 2021', 'Series_last_update_date': 'Oct 26 2023', 'Series_pubmed_id': '37841830', 'Series_summary': \"We report the genome-wide impact of rhythmic P-eIF2α levels in vivo by performing ribosome profiling and RNA-seq in Neurospora crassa samples grown over a circadian time course. We identified candidate genes which showed rhyhmic ribosome occupancy in WT cells but lost rhythms in mRNA levels and ribosome occupancy in the clock mutant Dfrq, eIF2α  kinase knockout Dcpc-3, and constitutively active kinase mutant cpc-3c cells. Select mRNAs harbor a P-body localization motif identified by MEME and MEME Suite's GOMo function. P-body localization of the target mRNAs provides an attractive mechanism for explaining translational rhythms from otherwise arrhythmic mRNAs.\", 'Series_overall_design': 'Ribosome profiling (ribo-seq) (done in parallel with RNA-seq) of Neurospora crassa WT, clock mutant Dfrq, eIF2α  kinase knockout Dcpc-3, and constitutively active kinase mutant cpc-3c mycelia, grown over a circadian time course. Each time point has a duplicate.', 'Series_type': 'Other', 'Series_contributor': 'Deborah,,Bell-Pedersen', 'Series_sample_id': 'GSM5506344', 'Series_contact_name': 'Deborah,,Bell-Pedersen', 'Series_contact_email': 'dpedersen@bio.tamu.edu', 'Series_contact_department': 'Biology', 'Series_contact_institute': 'Texas A&M University', 'Series_contact_address': '3258 TAMU', 'Series_contact_city': 'College Station', 'Series_contact_state': 'Texas', 'Series_contact_zip/postal_code': '77840', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE181nnn/GSE181565/suppl/GSE181565_RAW.tar', 'Series_platform_id': 'GPL30082', 'Series_platform_taxid': '5141', 'Series_sample_taxid': '5141', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP331348'}\n",
      "491 PRJNA753487 GSE181795 {'Series_title': 'An oncomicropeptide Regulates translation in leukemia', 'Series_geo_accession': 'GSE181795', 'Series_status': 'Public on Aug 12 2021', 'Series_submission_date': 'Aug 10 2021', 'Series_last_update_date': 'Dec 20 2021', 'Series_pubmed_id': '34555354', 'Series_summary': 'We identified an oncomicropeptide that was enriched on ribosomes and regulated translation. We further performed the ribosome profiling and RNA-seq libraries sequenced to globally identify the mRNAs that the peptie targeted. Interestingly, the ribo-seq data has shown that a large number of mRNAs were significantly downregulated in the peptide-knockdown samples. Finally, we discovered that high expression of the peptide supported a specific translation program to facilitate cancer progression.', 'Series_overall_design': 'Sh-NC and sh-Speptide MOLM-13 cells used', 'Series_type': 'Other', 'Series_contributor': 'Wen-Tao,,Wang', 'Series_sample_id': 'GSM5511485', 'Series_contact_name': 'Wen-Tao,,Wang', 'Series_contact_email': 'wangwt8@mail.sysu.edu.cn', 'Series_contact_phone': '86-20-84111736', 'Series_contact_laboratory': 'Key Laboratory of Gene Engineering of the Ministry of Education', 'Series_contact_department': 'School of Life Sciences', 'Series_contact_institute': 'sun yat sen university', 'Series_contact_address': 'No. 135, Xingang Xi Road', 'Series_contact_city': 'Guangzhou', 'Series_contact_state': 'Guangdong', 'Series_contact_zip/postal_code': '510275', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE181nnn/GSE181795/suppl/GSE181795_Ribo-seq_NC-vs-AP.genes.annot.txt.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP331914'}\n",
      "492 PRJNA753526 GSE181824 {'Series_title': 'Transcription-wide distribution of dihydrouridine (D) into mRNAs reveals its requirement for meiotic chromosome segregation [Ribo/RNA-seq]', 'Series_geo_accession': 'GSE181824', 'Series_status': 'Public on Oct 26 2021', 'Series_submission_date': 'Aug 10 2021', 'Series_last_update_date': 'Jan 25 2022', 'Series_pubmed_id': '34798057', 'Series_summary': 'Here we have developed Rho-seq, an integrated pipeline detecting a range of modifications through differential modification-dependent Rhodamine labeling. Using Rho-seq, we report that the reduction of uridine to dihydrouridine by the Dus reductase family targets tRNAs in E. coli but expands to mRNAs is yeast. The modified mRNAs are enriched for cytoskeleton related encoded protein. We show that the a-tubulin encoding mRNA nda2 undergoes dihydrouridination, which affects its protein expression level. The absence of the modification onto the nda2 mRNA strongly impacts meiosis by inducing a metaphase delay or by completely preventing the formation of spindles during meiosis I and meiosis II, eventually resulting in low gamete viability. Collectively these data show that the codon specific reduction of uridine within mRNA is required for proper meiotic chromosome segregation and gamete viability.', 'Series_overall_design': 'Ribo-seq and total RNA-seq for matching samples of wild-type and delta4dus S. pombe cells', 'Series_type': 'Other', 'Series_contributor': 'Damien,,Hermand', 'Series_sample_id': 'GSM5512596', 'Series_contact_name': 'Carlo,,Yague-Sanz', 'Series_contact_email': 'yague.carlo@gmail.com', 'Series_contact_laboratory': 'GEMO lab', 'Series_contact_department': 'Medecine', 'Series_contact_institute': 'Namur University', 'Series_contact_address': '61 rue de Bruxelles', 'Series_contact_city': 'Namur', 'Series_contact_zip/postal_code': '5000', 'Series_contact_country': 'Belgium', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE181nnn/GSE181824/suppl/GSE181824_rna.csv.gz', 'Series_platform_id': 'GPL17225', 'Series_platform_taxid': '4896', 'Series_sample_taxid': '4896', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP331950'}\n",
      "493 PRJNA754447 GSE182100 {'Series_title': 'Global and gene-specific translational regulation in Escherichia coli across different conditions', 'Series_geo_accession': 'GSE182100', 'Series_status': 'Public on Oct 13 2022', 'Series_submission_date': 'Aug 13 2021', 'Series_last_update_date': 'Jan 13 2023', 'Series_pubmed_id': '36264977', 'Series_summary': 'How well mRNA transcript levels represent protein abundances has been a controversial issue. Particularly across different environments, correlations between mRNA and protein exhibit remarkable variability from gene to gene. Translational regulation is likely to be one of the key factors contributing to mismatches between mRNA level and protein abundance. Here, we quantified genome-wide transcriptome and relative translation efficiency (RTE) under 12 different conditions in Escherichia coli. By quantifying the mRNA-RTE correlation both across genes and across conditions, we uncovered a diversity of gene-specific translational regulations, cooperating with transcriptional regulations, in response to carbon (C), nitrogen (N), and phosphate (P) limitations. Intriguingly, we found that many genes regulating translation are themselves subject to translational regulation, suggesting possible direct feedbacks. Furthermore, we observed that genes with similar expression patterns share similar codon usage. Utilizing a random forest model, we confirmed that codon usage contributes to condition-dependent translational regulation. These findings broaden the understanding of translational regulation under environmental changes, and provide novel strategies for the control of translation in synthetic biology.', 'Series_overall_design': 'Ribosome profiling and RNA-seq of E. coli cells under 12 different conditions', 'Series_type': 'Other', 'Series_contributor': 'Zhiyuan,,Li', 'Series_sample_id': 'GSM5518523', 'Series_contact_name': 'Di,,Zhang', 'Series_contact_email': 'phydzhang@pku.edu.cn', 'Series_contact_institute': 'Peking University', 'Series_contact_address': 'No.5 YiHeYuan Road, Haidian District', 'Series_contact_city': 'Beijing', 'Series_contact_zip/postal_code': '100871', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE182nnn/GSE182100/suppl/GSE182100_RAW.tar', 'Series_platform_id': 'GPL18133', 'Series_platform_taxid': '562', 'Series_sample_taxid': '562', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP332494'}\n",
      "494 PRJNA755296 GSE182200 {'Series_title': 'Next Generation Sequencing identifies differentially translated genes regulated by PM2.5 exposure in BEAS-2B cells [Ribo-seq]', 'Series_geo_accession': 'GSE182200', 'Series_status': 'Public on Jul 27 2022', 'Series_submission_date': 'Aug 16 2021', 'Series_last_update_date': 'Jul 27 2022', 'Series_pubmed_id': '35869499', 'Series_summary': 'We report the application of Ribosome sequencing for high-throughput profiling of tanslation efficiency in BEAS-2B cells after exposure to PM2.5.', 'Series_overall_design': 'Comparison of tanslation efficiency between  BEAS-2B cells exposed to PM2.5  and control cells.', 'Series_type': 'Other', 'Series_contributor': 'Qianqian,,Chen', 'Series_sample_id': 'GSM5520355', 'Series_contact_name': 'Rihong,,Zhai', 'Series_contact_email': 'rzhai@szu.edu.cn', 'Series_contact_phone': '+86 755 26919842', 'Series_contact_institute': 'Shenzhen University School of Medicine', 'Series_contact_address': '1066 Xueyuan Ave.', 'Series_contact_city': 'Shenzhen', 'Series_contact_state': 'Guangdong', 'Series_contact_zip/postal_code': '518060', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE182nnn/GSE182200/suppl/GSE182200_Translatome_profile_PM2.5_RZ.xlsx', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP332718'}\n",
      "495 PRJNA756018 GSE182371 {'Series_title': 'smORF atlas (part 1-RiboSeq)', 'Series_geo_accession': 'GSE182371', 'Series_status': 'Public on Jul 18 2022', 'Series_submission_date': 'Aug 18 2021', 'Series_last_update_date': 'Jul 19 2022', 'Series_pubmed_id': '35841888', 'Series_summary': 'Small open reading frames (smORFs) can have important regulatory roles and give rise to stable proteins, yet their discovery based on sequence-based predictions and proteomics has been challenging. Ribosome profiling (Ribo-seq) data can provide valuable experimental evidence of RNA translation. Stringent analysis of P-sites, representing 1.3 billion high-confidence ribosome locations, revealed 5308 uORFs, 1652 smORFs in lincRNAs and 807 dORFs that are translated in humans. We here provide a comprehensive database of Ribo-seq smORFs for a more complete understanding of the translated human genome.', 'Series_overall_design': 'To identify smORFs on a genome-wide level, we have collected, generated and processed Ribo-seq data from seven primary human cell types and two human tissues.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Sebastian,,Schafer', 'Series_sample_id': 'GSM5527751', 'Series_contact_name': 'Benjamin,,Ng', 'Series_contact_email': 'gmsbnwk@nus.edu.sg', 'Series_contact_institute': 'Duke-National University of Singapore Medical School', 'Series_contact_address': '8 College Road', 'Series_contact_city': 'Singapore', 'Series_contact_state': 'N/A', 'Series_contact_zip/postal_code': '169857', 'Series_contact_country': 'Singapore', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE182nnn/GSE182371/suppl/GSE182371_counts_ribo_final.txt.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP333128'}\n",
      "496 PRJNA757962 GSE182810 {'Series_title': 'Oligodendrocyte differentiation alters tRNA modifications and codon-dependent mRNA decay [RiboSeq]', 'Series_geo_accession': 'GSE182810', 'Series_status': 'Public on Aug 08 2022', 'Series_submission_date': 'Aug 26 2021', 'Series_last_update_date': 'Aug 31 2022', 'Series_pubmed_id': '36008413', 'Series_summary': 'Oligodendrocytes are specialized cells that confer neuronal myelination. Leukodystrophies associated with oligodendrocyte and hypomyelination are known to result when a number of tRNA metabolism genes are mutated. Thus, for unknown reasons, oligodendrocytes may be hypersensitive to perturbations in tRNA biology. In this study, we survey the tRNA transcriptome in the murine oligodendrocytes cell lineage in an effort to understanding the molecular underpinning for human disease. We find that specific tRNAs are hypomodified in oligodendrocytes within or near the anticodon. This hypomodified state may be the result of differential expression of key modification enzymes during oligodendrocyte differentiation. Moreover, we observe a concomitant relationship between tRNA hypomodification and tRNA decoding potential; observing oligodendrocyte specific alterations in codon optimality-mediated mRNA decay and ribosome transit. Our results reveal that oligodendrocytes naturally maintain a delicate, hypersensitized tRNA/mRNA axis. We suggest this axis underlies disease etiology when further insult to tRNA metabolism is introduced.', 'Series_overall_design': 'Ribosome profiling and associated RNA-Seq data (QuantM-seq) of murine Oligodendrocyte Progenitor Cells (OPCs) (2 replicates each) and Oligodendrocytes (OLs) (2 replicates each)', 'Series_type': 'Other', 'Series_contributor': 'Jeff,,Coller', 'Series_sample_id': 'GSM5537331', 'Series_contact_name': 'Jeff,,Coller', 'Series_contact_institute': 'Johns Hopkins University', 'Series_contact_address': '725 N. Wolfe St PCTB 505', 'Series_contact_city': 'Baltimore', 'Series_contact_zip/postal_code': '21205', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE182nnn/GSE182810/suppl/GSE182810_OPC_OL_footprints_and_abundance_fpkm.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP334193'}\n",
      "497 PRJNA759858 GSE183264 {'Series_title': 'Improved Super-Resolution Ribosome Profiling Reveals Prevalent Translation of Upstream ORFs and Small ORFs in Arabidopsis', 'Series_geo_accession': 'GSE183264', 'Series_status': 'Public on Sep 20 2023', 'Series_submission_date': 'Sep 02 2021', 'Series_last_update_date': 'Jul 26 2024', 'Series_pubmed_id': '38000896', 'Series_web_link': 'https://doi.org/10.1093/plcell/koad290', 'Series_summary': 'We used Ribo-seq (Ribosome profiling) combining with RNA-seq to explore the translational landscape of Arabidopsis Col-0 seedling. We generated 6 biological replicates of RNA-seq and Ribo-seq data for Arabidopsis Col-0 seedling. 3 of the replicates were collected after 20 minutes of 0.1% DMSO treatment and the other 3 samples were collected after 60 minutes of DMSO treatmeant. The resulting RNA-seq and Ribo-seq files were used to discover translated up-stream ORFs (uORFs) and analyze the translation efficiency of uORF-containing genes in Arabidopsis.', 'Series_overall_design': \"UPDATE: [07-26-2024] The titles of Samples GSM5554178-GSM5554183 were corrected from 'root' to 'seedling'.\", 'Series_type': 'Other', 'Series_contributor': 'Hsin-Yen,L,Wu', 'Series_sample_id': 'GSM5554183', 'Series_contact_name': 'Polly,,Hsu', 'Series_contact_email': 'pollyhsu@msu.edu', 'Series_contact_department': 'Biochemistry and Molecular Biology', 'Series_contact_institute': 'Michigan State University', 'Series_contact_address': '603 Wilson Road', 'Series_contact_city': 'East Lansing', 'Series_contact_state': 'MI', 'Series_contact_zip/postal_code': '48824', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE183nnn/GSE183264/suppl/GSE183264_Ribo.genes.results.gz', 'Series_platform_id': 'GPL21785', 'Series_platform_taxid': '3702', 'Series_sample_taxid': '3702', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP335382'}\n",
      "498 PRJNA760922 GSE183470 {'Series_title': 'Controlling tissue patterning by translational regulation of signaling transcripts through the core translation factor eIF3c [I]', 'Series_geo_accession': 'GSE183470', 'Series_status': 'Public on Sep 30 2021', 'Series_submission_date': 'Sep 05 2021', 'Series_last_update_date': 'Sep 30 2021', 'Series_summary': 'Although gene expression is tightly regulated during embryonic development, the impact of translational control has received less experimental attention. Here, we find that eukaryotic translation initiation factor-3 (eIF3) is required for Shh-mediated tissue patterning.  Analysis of loss-of-function  eIF3 subunit c (Eif3c) mice reveal a unique sensitivity to the Shh receptor Patched 1 (Ptch1) dosage. Genome-wide in vivo enhanced cross-linking immunoprecipitation sequence (eCLIP-seq) shows unexpected specificity for eIF3 binding to a pyrimidine rich motif present in subsets of 5’-UTRs and a corresponding change in the translation of these transcripts by ribosome profiling in Eif3c loss-of-function embryos. We further find that while Eif3c loss-of-function embryos do not show a global decrease in protein synthesis, translation of Ptch1 through this pyrimidine rich motif is specifically sensitive to eIF3 amount. Altogether, this work uncovers hidden specificity of housekeeping translation initiation machinery for the translation of key developmental signaling transcripts.', 'Series_overall_design': 'Ribosome profiling (Ribo-seq)/RNA-seq', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Maria,,Barna', 'Series_sample_id': 'GSM5558153', 'Series_contact_name': 'Gun Woo,,Byeon', 'Series_contact_institute': 'Stanford University', 'Series_contact_address': '279 Campus Dr., Beckman Center B309', 'Series_contact_city': 'Stanford', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94305', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE183nnn/GSE183470/suppl/GSE183470_ribo_counts.txt.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP335745'}\n",
      "499 PRJNA763584 GSE184209 {'Series_title': 'TRL-1 undergoes phase separation to regulate the reproduction and longevity tradeoff', 'Series_geo_accession': 'GSE184209', 'Series_status': 'Public on Mar 22 2022', 'Series_submission_date': 'Sep 15 2021', 'Series_last_update_date': 'Jun 22 2022', 'Series_pubmed_id': '35482917', 'Series_summary': \"The antagonistic pleiotropy theory of aging proposes that genes enhancing fitness in early life limit lifespan, but the molecular evidence remains underexplored. By profiling translatome changes in Caenorhabditis elegans during starvation recovery, we demonstrate that an open reading frame (ORF) trl-1 'hidden' within an annotated pseudogene significantly translates upon refeeding. trl-1 mutant animals increase brood sizes but shorten lifespan and specifically impair the germline deficiency-induced longevity. The loss of trl-1 abnormally upregulates the translation of vitellogenin that produces copious yolk to provision eggs, whereas vitellogenin overexpression is known to reduce lifespan. We show that TRL-1 protein undergoes liquid-liquid phase separation, through which TRL-1 granules recruit vitellogenin mRNA and inhibit its translation. These results indicate that trl-1 functions as an antagonistic pleiotropic gene to regulate the reproduction-longevity tradeoff by optimizing nutrient production for the next generation.\", 'Series_overall_design': 'Ribo-Seq and RNA-Seq of starved six days C. elegans larvae refed with bacterial food for 0, 10, 30, and 60 minutes (min) in wide-type(N2) animals , trl-1(cas1077) animals and trl-1(cas1078) animals. A reference sample was collected from the same culture condition from the non-starved (NS) animals, each strain was analyzed in biological duplicate.', 'Series_type': 'Other', 'Series_contributor': 'Jingying,,Huang', 'Series_sample_id': 'GSM5580336', 'Series_contact_name': 'Dou,,Wu', 'Series_contact_email': 'wd16@mails.tsinghua.edu.cn', 'Series_contact_institute': 'Tsinghua university', 'Series_contact_address': 'Haidian street', 'Series_contact_city': 'Beijing', 'Series_contact_zip/postal_code': '100084', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE184nnn/GSE184209/suppl/GSE184209_RAW.tar', 'Series_platform_id': 'GPL13657', 'Series_platform_taxid': '6239', 'Series_sample_taxid': '6239', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP337255'}\n",
      "500 PRJNA763797 GSE184247 {'Series_title': 'eIF4A1 enhances LARP1-mediated translational repression during mTORC1 inhibition', 'Series_geo_accession': 'GSE184247', 'Series_status': 'Public on Feb 13 2024', 'Series_submission_date': 'Sep 16 2021', 'Series_last_update_date': 'May 23 2024', 'Series_pubmed_id': '38773334', 'Series_summary': 'Eukaryotic translation initiation factor (eIF) 4A — a DEAD-box RNA-binding protein — plays an essential role in translation initiation. Recent reports have suggested helicase-dependent and helicase-independent functions for eIF4A, but the multifaceted roles of eIF4A have not been fully explored. Here, we show that eIF4A1 enhances translational repression during the inhibition of mechanistic target of rapamycin complex 1 (mTORC1), an essential kinase complex controlling cell proliferation. RNA pulldown followed by sequencing revealed that eIF4A1 preferentially binds to mRNAs containing terminal oligopyrimidine (TOP) motifs (TOP mRNAs), whose translation is rapidly repressed upon mTORC1 inhibition. This selective interaction depends on a La-related RNA-binding protein, LARP1. Ribosome profiling revealed that deletion of EIF4A1 attenuated the translational repression of TOP mRNAs upon mTORC1 inactivation. Moreover, eIF4A1 increases the affinity between TOP mRNAs and LARP1 and thus ensures stronger translational repression upon mTORC1 inhibition. Our data show the multimodality of eIF4A1 in modulating protein synthesis through an inhibitory binding partner and provide a unique example of the repressive role of a universal translational activator.', 'Series_overall_design': 'Ribosome profiling of EIF4A1 KO and EIF4A2 KO cells with PP242 treatment, RNA-Seq of EIF4A1 KO cells, RNA pulldown-seq of SBP-EIF4A1- and SBP-EIF4A2- expressing cells, and RIP-seq of SBP-EIF4A1-expressing cells upon LARP1 knockdown.', 'Series_type': 'Other', 'Series_contributor': 'Shintaro,,Iwasaki', 'Series_sample_id': 'GSM7792344', 'Series_contact_name': 'Shintaro,,Iwasaki', 'Series_contact_email': 'shintaro.iwasaki@riken.jp', 'Series_contact_laboratory': 'RNA Systems Biochemistry Laboratory', 'Series_contact_department': 'Center for Pioneering Research', 'Series_contact_institute': 'RIKEN', 'Series_contact_address': '2-1, Hirosawa', 'Series_contact_city': 'Wako', 'Series_contact_state': 'Saitama', 'Series_contact_zip/postal_code': '3510198', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE184nnn/GSE184247/suppl/GSE184247_RAW.tar', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP337382'}\n",
      "501 PRJNA764085 GSE184331 {'Series_title': 'Nascent polypeptide within the exit tunnel stabilizes the ribosome to counteract risky translation', 'Series_geo_accession': 'GSE184331', 'Series_status': 'Public on Dec 08 2021', 'Series_submission_date': 'Sep 17 2021', 'Series_last_update_date': 'Dec 09 2021', 'Series_pubmed_id': '34672004', 'Series_summary': 'Continuous translation elongation, irrespective of amino acid sequences, is a prerequisite for living organisms to produce their proteomes. However, the risk of elongation abortion is concealed within nascent polypeptide products. For example, negatively charged sequences with occasional intermittent prolines, termed intrinsic ribosome destabilization (IRD) sequences, destabilizes the translating ribosomal complex. Thus, some nascent chain sequences lead to premature translation cessation. Here, we show that most potential IRD sequences in the middle of open reading frames remain cryptic by two mechanisms: the nascent polypeptide itself that spans the exit tunnel and its bulky amino acid residues that occupy the tunnel entrance region. Thus, nascent polypeptide products have a built-in ability to ensure elongation continuity by serving as a bridge and thus by protecting the large and small ribosomal subunits from dissociation.', 'Series_overall_design': 'Ribosome profiling', 'Series_type': 'Other', 'Series_contributor': 'Hideki,,Taguchi', 'Series_sample_id': 'GSM5584181', 'Series_contact_name': 'Yuhei,,Chadani', 'Series_contact_laboratory': 'Hideki Taguchi', 'Series_contact_department': 'Institute of Innovative Research', 'Series_contact_institute': 'Tokyo Institute of Technology', 'Series_contact_address': '4259-S2-19', 'Series_contact_city': 'Nagatsuta-cho,Midori-ku, Yokohama-shi', 'Series_contact_state': 'Kanagawa', 'Series_contact_zip/postal_code': '226-8501', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE184nnn/GSE184331/suppl/GSE184331_RAW.tar', 'Series_platform_id': 'GPL21433', 'Series_platform_taxid': '562', 'Series_sample_taxid': '562', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP337604'}\n",
      "502 PRJNA765951 GSE184725 {'Series_title': 'Translational regulation in response to sulfur in Sorghum bicolor', 'Series_geo_accession': 'GSE184725', 'Series_status': 'Public on Nov 16 2022', 'Series_submission_date': 'Sep 24 2021', 'Series_last_update_date': 'Nov 16 2022', 'Series_pubmed_id': '35166349', 'Series_summary': 'Sorghum is a C4 cereal important not only as food, but also as forage and a bioenergy resource. Its resistance to harsh environments has made it an agriculturally important research subject. Recent accumulation of genomic and transcriptomic information has facilitated genetic studies. Yet genome-wide translational profiles in sorghum are still missing, although increasing evidence has demonstrated that translation is an important regulatory step, and the transcriptome does not necessarily reflect the profile of functional protein production in some organisms. Deep sequencing of ribosome-protected mRNA fragments (ribosome profiling, or Ribo-seq) has enabled genome-wide analysis of translation. In this study, we took advantage of Ribo-seq and identified actively translated reading frames throughout the genome. We detected translation of 7,304 main ORFs annotated in the sorghum reference genome version 3.1 and revealed a number of unannotated translational events. A comparison of the transcriptome and translatome between sorghums grown under normal and sulfur-deficient conditions revealed that gene expression is modulated independently at transcript levels and translation levels. Our study revealed the translational landscape of sorghum’s response to sulfur and provides datasets that could serve as a fundamental resource to extend research on sorghum, including translational studies.', 'Series_overall_design': 'Whole shoot tissues of 13-day-after-sowing Sorghum bicolor grown under Control and sulfur deficiency conditions (Ctrl, 500 µM; Sminus, 0 µM). 3 biological replicates.', 'Series_type': 'Other', 'Series_contributor': 'Toru,,Fujiwara', 'Series_sample_id': 'GSM5596419', 'Series_contact_name': 'Naoyuki,,Sotta', 'Series_contact_email': 'nsotta@gmail.com', 'Series_contact_phone': '+81-3-5841-5104', 'Series_contact_laboratory': 'Laboratory of Plant Nutrition and Fertilizers', 'Series_contact_department': 'Department of Applied Biological Chemistry', 'Series_contact_institute': 'Graduate School of Agricultural and Life Sciences, The University of Tokyo', 'Series_contact_address': '1-1-1 Yayoi, Bunkyo-ku', 'Series_contact_city': 'Tokyo', 'Series_contact_zip/postal_code': '113-8657', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE184nnn/GSE184725/suppl/GSE184725_sumcount_Psite_CDS.csv.gz', 'Series_platform_id': 'GPL20690', 'Series_platform_taxid': '4558', 'Series_sample_taxid': '4558', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP338570'}\n",
      "503 PRJNA767014 GSE184909 {'Series_title': 'tRNA-m1A modification promotes T cell expansion via efficient MYC protein synthesis', 'Series_geo_accession': 'GSE184909', 'Series_status': 'Public on Jul 23 2022', 'Series_submission_date': 'Sep 28 2021', 'Series_last_update_date': 'Jan 07 2023', 'Series_web_link': 'https://www.nature.com/articles/s41590-022-01301-3', 'Series_summary': '(3) m1A58 modifications in tRNA support rapid and adequate synthesis of MYC and other key functional proteins, guide the exit of naïve T cells from quiescence state into a proliferative state, and promote rapid T cell expansion after activation.', 'Series_overall_design': 'For RiboTag-seq, 500 million naïve CD4+ T cells were isolated from WT (RiboTagflox/floxCd4Cre) and Trmt61a-KO (Trmt61aflox/floxRiboTagflox/floxCd4Cre) mice (activated with plate-bound anti-CD3 plus anti-CD28 in 96-well plate for 6 hours).', 'Series_type': 'Other', 'Series_contributor': 'Xiaoting,,Zhang', 'Series_sample_id': 'GSM6380474', 'Series_contact_name': 'Xiaoyu,,Li', 'Series_contact_email': 'xiaoyu_li@zju.edu.cn', 'Series_contact_laboratory': 'Li Lab', 'Series_contact_department': 'SCHOOL OF BASIC MEDICAL SCIENCES', 'Series_contact_institute': 'Zhejiang University', 'Series_contact_address': '866 Yuhangtang Road', 'Series_contact_city': 'Hangzhou', 'Series_contact_state': 'Zhejiang', 'Series_contact_zip/postal_code': '310058', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE184nnn/GSE184909/suppl/GSE184909_m1Aseq-tRNA-WT-CKO.csv.gz', 'Series_platform_id': 'GPL24247', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP339104'}\n",
      "504 PRJNA768399 GSE185248 {'Series_title': 'Translatomics coupled with transcriptomics reveals a novel mechanism of oral carcinogenesis [Ribo-seq]', 'Series_geo_accession': 'GSE185248', 'Series_status': 'Public on Oct 01 2023', 'Series_submission_date': 'Oct 04 2021', 'Series_last_update_date': 'Oct 02 2023', 'Series_summary': 'Oral and oropharyngeal squamous cell carcinoma (OSCC and OPSCC) represents an increasing problem in the global public health. OPSCC is the most frequent malignancy in oral cavity, with an overall increasing incidence and mortality. In view of its unclear exact mechanism and lack of optimized prevention and treatment methods, new biomarkers are needed to characterize the diversity of cancer and to explore effective treatment targets. This study combined translatomics and transcriptomics to analyze the differential genes in human oral cancer and adjacent tissues, in order to provide a new perspective for the prevention and treatment of OPSCC.', 'Series_overall_design': 'Ribosome profiling sequencing of oral mucosa of human that divived into tumor tissue and paracancerous tissue of oral cavity carcinomas', 'Series_type': 'Other', 'Series_contributor': 'Can,,Wei', 'Series_sample_id': 'GSM5608768', 'Series_contact_name': 'Can,,Wei', 'Series_contact_email': 'canwei528@126.com', 'Series_contact_department': 'Department of Pathophysiology', 'Series_contact_institute': 'Harbin Medical University', 'Series_contact_address': 'Baojian Road', 'Series_contact_city': 'Harbin', 'Series_contact_zip/postal_code': '150081', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE185nnn/GSE185248/suppl/GSE185248_Raw_gene_counts_matrix.txt.gz', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP339889'}\n",
      "505 PRJNA768456 GSE185286 {'Series_title': 'RNA Sequencing and Ribosome profiling of GIR2 and PUB1 knockout yeast strains.', 'Series_geo_accession': 'GSE185286', 'Series_status': 'Public on Oct 11 2021', 'Series_submission_date': 'Oct 04 2021', 'Series_last_update_date': 'Dec 15 2021', 'Series_pubmed_id': '34900236', 'Series_summary': 'We present ribosome profiling and RNA-Seq data for the wild type BY4742 S. cerevisiae strain, and strains with the complete individual deletions of the GIR2 and PUB1 genes.', 'Series_overall_design': 'Yeast cells of 3 different S. cerevisiae strains, namely: wt, gir2Δ, pub1Δ were maintained in YPD media. The libraries were sequenced, resulting in 14 RNA-Seq and 12 Ribo-Seq data sets.', 'Series_type': 'Other', 'Series_contributor': 'Sergey,E,Dmitriev', 'Series_sample_id': 'GSM5610148', 'Series_contact_name': 'Sergey,E.,Dmitriev', 'Series_contact_email': 'sergey.dmitriev@belozersky.msu.ru', 'Series_contact_phone': '+79032220066', 'Series_contact_institute': 'Lomonosov Moscow State University', 'Series_contact_address': 'Leninskie Gory, 1', 'Series_contact_city': 'Moscow', 'Series_contact_zip/postal_code': '119991', 'Series_contact_country': 'Russia', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE185nnn/GSE185286/suppl/GSE185286_gene_counts_s2.csv.gz', 'Series_platform_id': 'GPL26302', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP339974'}\n",
      "506 PRJNA769126 GSE185458 {'Series_title': 'RNA Sequencing and Ribosome profiling of TMA46, STM1 and YGR054W knockout yeast strains', 'Series_geo_accession': 'GSE185458', 'Series_status': 'Public on Oct 08 2021', 'Series_submission_date': 'Oct 06 2021', 'Series_last_update_date': 'Dec 15 2021', 'Series_pubmed_id': '34900236', 'Series_summary': 'We present ribosome profiling and RNA-Seq data for the wild type BY4742 S. cerevisiae strain, and strains with the complete individual deletions of the TMA46, STM1 and YGR054W genes.', 'Series_overall_design': 'Yeast cells of 4 different S. cerevisiae strains, namely: wt, tma46Δ, eif2aΔ, stm1Δ were maintained in YPD media. The libraries were sequenced, resulting in 16 RNA-Seq and 16 Ribo-Seq data sets.', 'Series_type': 'Other', 'Series_contributor': 'Sergey,E,Dmitriev', 'Series_sample_id': 'GSM5615657', 'Series_contact_name': 'Sergey,E.,Dmitriev', 'Series_contact_email': 'sergey.dmitriev@belozersky.msu.ru', 'Series_contact_phone': '+79032220066', 'Series_contact_institute': 'Lomonosov Moscow State University', 'Series_contact_address': 'Leninskie Gory, 1', 'Series_contact_city': 'Moscow', 'Series_contact_zip/postal_code': '119991', 'Series_contact_country': 'Russia', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE185nnn/GSE185458/suppl/GSE185458_gene_counts_s1.csv.gz', 'Series_platform_id': 'GPL19756', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP340281'}\n",
      "507 PRJNA770523 GSE185695 {'Series_title': 'LncRNA INHEG promotes glioma stem cell maintenance and tumorigenicity through regulating rRNA 2’-O methylation', 'Series_geo_accession': 'GSE185695', 'Series_status': 'Public on Sep 28 2023', 'Series_submission_date': 'Oct 12 2021', 'Series_last_update_date': 'Dec 08 2023', 'Series_pubmed_id': '37980347', 'Series_summary': 'Glioblastoma (GBM) ranks among the most lethal of human cancers, containing glioma stem cells (GSCs) that display therapeutic resistance. Here, we report that the lncRNA INHEG is highly expressed in GSCs compared to differentiated glioma cells (DGCs) and promotes GSC self-renewal through control of rRNA 2’-O-methylation. INHEG induces the interaction between a novel SUMO E3 ligase TAF15 and NOP58, a core component of snoRNP that guides rRNA methylation, to regulate NOP58 sumoylation and accelerate the C/D box snoRNP assembly. INHEG activation enhances rRNA 2’-O-methylation, thereby increasing the translation of oncogenic proteins including EGFR and IGF1R in glioma cells. Taken together, this study identifies a lncRNA that connects snoRNP-guided rRNA 2’-O-methylation to upregulated protein translation in GSCs, supporting a new axis for potential therapeutic targeting of gliomas.', 'Series_overall_design': \"We report a lncRNA that connects snoRNP-guided rRNA 2’-O-methylation to upregulated protein translation in GSCs. To investigate the role of rRNA methylation in the tumor hierarchy, we measured the rRNA 2'-O-Me levels in two matched patient-derived GSCs (3565, MGG4) and DGCs with RiboMethSeq. To interrogate the signaling pathways modulated by methyl transferase NOP58, we performed transcriptome sequencing of 3565 and MGG4 cells with NOP58 knockout compared to non-targeting control.  Furthermore, we conducted ribosome profiling (Ribo-seq) and RNA-seq  to identify the mRNA that lncRNA INHEG regulates.\", 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Runsheng,,Chen', 'Series_sample_id': 'GSM6701593', 'Series_contact_name': 'Lihui,,Liu', 'Series_contact_email': 'lihui.mm@ibp.ac.cn', 'Series_contact_laboratory': 'RunSheng Chen', 'Series_contact_department': 'Institute of Biophysics', 'Series_contact_institute': 'Chinese Academy of Sciences', 'Series_contact_address': '15 Datun Road, Chaoyang District', 'Series_contact_city': 'Beijing', 'Series_contact_zip/postal_code': '100101', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE185nnn/GSE185695/suppl/GSE185695_NOP58RNAseq.count.txt.gz', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP340943'}\n",
      "508 PRJNA770650 GSE185732 {'Series_title': 'Single cell quantification of ribosome occupancy in early mouse development', 'Series_geo_accession': 'GSE185732', 'Series_status': 'Public on Feb 06 2023', 'Series_submission_date': 'Oct 12 2021', 'Series_last_update_date': 'Sep 11 2023', 'Series_pubmed_id': '37344592', 'Series_summary': 'Technological limitations precluded transcriptome-wide analyses of translation at single cell resolution. To solve this challenge, we developed a novel microfluidic isotachophoresis approach, named RIBOsome profiling via IsoTachoPhoresis (Ribo-ITP), and characterized translation in single oocytes and embryos during early mouse development. We identified differential translation efficiency as a key regulatory mechanism of genes involved in centrosome organization and N6-methyladenosine modification of RNAs. Our high coverage measurements enabled the first analysis of allele-specific ribosome engagement in early development and led to the discovery of stage-specific differential engagement of zygotic RNAs with ribosomes and reduced translation efficiency of transcripts with allelic-biased expression. Finally, by integrating our measurements with proteomics data, we discovered that ribosome occupancy in germinal vesicle stage oocytes is the predominant determinant of protein abundance in the zygote. The novel Ribo-ITP approach will enable numerous applications by providing high coverage and high resolution ribosome occupancy measurements from ultra-low input samples including single cells.', 'Series_overall_design': 'Ribosome profiling by isotachophoresis (Ribo-ITP) of single K562 cells', 'Series_type': 'Other', 'Series_contributor': 'Can,,Cenik', 'Series_sample_id': 'GSM7182544', 'Series_contact_name': 'Can,,Cenik', 'Series_contact_email': 'ccenik@austin.utexas.edu', 'Series_contact_phone': '512-232-2874', 'Series_contact_laboratory': 'Cenik', 'Series_contact_department': 'Molecular Biosciences', 'Series_contact_institute': 'The University of Texas at Austin', 'Series_contact_address': 'Moffett Molecular Biology Building 2.122 2500 Speedway', 'Series_contact_city': 'Austin', 'Series_contact_state': 'TX', 'Series_contact_zip/postal_code': '78712', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE185nnn/GSE185732/suppl/GSE185732_riboITP_RiboFlow_stats.csv.gz', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP341033'}\n",
      "509 PRJNA770675 GSE185757 {'Series_title': 'The context of the ribosome binding site in mRNAs defines specificity of action of kasugamycin, an inhibitor of translation initiation', 'Series_geo_accession': 'GSE185757', 'Series_status': 'Public on Feb 02 2022', 'Series_submission_date': 'Oct 12 2021', 'Series_last_update_date': 'Feb 03 2022', 'Series_pubmed_id': '35064089', 'Series_summary': 'Kasugamycin (KSG) is an aminoglycoside antibiotic widely used in agriculture and exhibiting considerable medical potential. Previous studies suggested that KSG interferes with translation by blocking binding of canonical mRNA and initiator tRNA to the small ribosomal subunit thereby preventing initiation of protein synthesis. Here, by using genome-wide approaches, we show that KSG can interfere with translation even after the formation of the 70S initiation complex on mRNA, as the extent of KSG-mediated translation inhibition correlates with increased occupancy of start codons by 70S ribosomes. We also show that KSG inhibits protein synthesis in a gene- and context-specific manner as even saturating concentrations of KSG do not completely abolish translation of all Escherichia coli genes. Differential action of KSG significantly depends on the nature of the mRNA residue immediately preceding the start codon, with guanine in this position being the most conducive to inhibition by the drug.  In addition, the activity of KSG is attenuated by translational coupling as genes whose start codons overlap with the coding regions or the stop codons of the upstream cistrons tend to be less susceptible to drug-mediated inhibition. Altogether, our findings reveal KSG as the first example of a small ribosomal subunit-targeting antibiotic with a well-pronounced context specificity of action.', 'Series_overall_design': 'Ribosome profiling (Ribo-seq) and mRNA-seq data collected from E. coli ΔgcvB cells grown in MOPS EZ rich defined media at mid-log phase, and the cells treated with 1 mg/ml an 10 mg/ml kasugamycin (KSG) for 2.5 min at 37˚C.', 'Series_type': 'Other', 'Series_contributor': 'Alexander,S,Mankin', 'Series_sample_id': 'GSM5623262', 'Series_contact_name': 'Yan,,Zhang', 'Series_contact_email': 'yan.jenny.zhang@gmail.com', 'Series_contact_laboratory': 'Carol Gross', 'Series_contact_department': 'Microbiology and Immunology', 'Series_contact_institute': 'UC San Francisco', 'Series_contact_address': '600 16th Street', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE185nnn/GSE185757/suppl/GSE185757_RAW.tar', 'Series_platform_id': 'GPL21433', 'Series_platform_taxid': '562', 'Series_sample_taxid': '562', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP341055'}\n",
      "510 PRJNA774874 GSE186634 {'Series_title': 'Concerted modification of nucleotides at functional centers of the ribosome revealed by single-molecule RNA modification profiling', 'Series_geo_accession': 'GSE186634', 'Series_status': 'Public on Dec 03 2021', 'Series_submission_date': 'Oct 26 2021', 'Series_last_update_date': 'Apr 29 2022', 'Series_pubmed_id': '35384842', 'Series_summary': 'Nucleotides in RNA and DNA are chemically modified by numerous enzymes that alter their function. Eukaryotic ribosomal RNA (rRNA) is modified at more than 100 locations, particularly at highly conserved and functionally important nucleotides. During ribosome biogenesis, modifications are added at various stages of assembly. The existence of differently modified classes of ribosomes in normal cells is unknown because no method exists to simultaneously evaluate the modification status at all sites, within a single rRNA molecule. Using a combination of yeast genetics and nanopore direct RNA sequencing, we developed a reliable method to track the modification status of single rRNA molecules at 37 sites in 18S rRNA and 73 sites in 25S rRNA. We use our method to characterize patterns of modification heterogeneity and identify concerted modification of nucleotides found near functional centers of the ribosome. Distinct undermodified subpopulations of rRNAs accumulate upon loss of Dbp3 or Prp43 RNA helicases, suggesting overlapping roles in ribosome biogenesis. Modification profiles are surprisingly resistant to change in response to many genetic and environmental conditions that affect translation, ribosome biogenesis, and pre-mRNA splicing. The ability to capture single-molecule RNA modification profiles provides new insights into the roles of nucleotide modifications in RNA function.', 'Series_overall_design': 'We sequenced 33 samples of 18S and 25S ribosomal rRNA. 3 replicates of wild type yeast and 3 replicates of wild type shifted to cold. We sequenced 2 replicates of snr80 KO, Prp43 mutant, Dbp3 KO and Pxr1 KO.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Manuel,,Ares Jr.', 'Series_sample_id': 'GSM5658073', 'Series_contact_name': 'Manuel,,Ares', 'Series_contact_email': 'ares@ucsc.edu', 'Series_contact_institute': 'University of California, Santa Cruz', 'Series_contact_address': '1156 High St', 'Series_contact_city': 'Santa Cruz', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '95064', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186634/suppl/GSE186634_RAW.tar', 'Series_platform_id': 'GPL25739', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP343289'}\n",
      "511 PRJNA776169 GSE186795 {'Series_title': 'Ribosome-bound Upf1 forms distinct 80S complexes and conducts mRNA surveillance', 'Series_geo_accession': 'GSE186795', 'Series_status': 'Public on Oct 21 2022', 'Series_submission_date': 'Oct 28 2021', 'Series_last_update_date': 'Oct 21 2022', 'Series_pubmed_id': '36192133', 'Series_web_link': 'https://rnajournal.cshlp.org/content/early/2022/10/03/rna.079416.122/suppl/DC1', 'Series_summary': 'Upf1, Upf2, and Upf3 are the central regulators of nonsense-mediated mRNA decay (NMD), the eukaryotic mRNA quality control pathway generally triggered when a premature termination codon is recognized by the ribosome. The NMD-related functions of the Upf proteins likely commence while these factors are ribosome-associated, but little is known of the timing of their ribosome binding, their specificity for ribosomes translating NMD substrates, or the nature and role of any ribosome:Upf complexes. Here, we have elucidated details of the ribosome-associated steps of NMD. By combining yeast genetics with selective ribosome profiling and co-sedimentation analyses of polysomes with wild-type and mutant Upf proteins, our approaches have identified distinct states of ribosome:Upf association. All three Upf factors manifest progressive polysome association as mRNA translation proceeds, but these events appear to be preceded by formation of a Upf1:80S complex as mRNAs initiate translation. This complex is likely executing an early mRNA surveillance function.', 'Series_overall_design': \"We have employed selective profiling of ribosomes bound by FLAG-tagged NMD factors Upf1, Upf2, or Upf3, and of ribosomes from strains harboring NMD-altering mutations. Ribosome profiling libraries were prepared from both total and immunopurified ribosomes in duplicate or triplicate.  RNAseq libraries were prepared from cell lysates of the same strains. High throughput sequencing was performed using Illumina NextSeq500 or HiSeq4000.  The size and transcript location of ribosome protected fragments with respect to  transcript coding regions, 5' and 3'UTRs, and start or stop codons from both NMD and non-NMD substrates was determined.\", 'Series_type': 'Other', 'Series_contributor': 'Allan,,Jacobson', 'Series_sample_id': 'GSM5661190', 'Series_contact_name': 'Robin,,Ganesan', 'Series_contact_laboratory': 'Allan Jacobson', 'Series_contact_department': 'Microbiology and Physiological Systems', 'Series_contact_institute': 'UMass Chan Medical School', 'Series_contact_address': '368 Plantation Street', 'Series_contact_city': 'Worcester', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '01655', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186795/suppl/GSE186795_Riboseq_Total80S_RSEM_expected_count_isoforms.results.csv.gz', 'Series_platform_id': 'GPL21656', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP343728'}\n",
      "512 PRJNA779283 GSE188529 {'Series_title': 'Transcriptome and translatome profiling of warm temperature (35°C -> 38.5°C) treated mouse embryonic fibroblast cells', 'Series_geo_accession': 'GSE188529', 'Series_status': 'Public on Mar 06 2023', 'Series_submission_date': 'Nov 10 2021', 'Series_last_update_date': 'Jun 10 2023', 'Series_pubmed_id': '36882059', 'Series_summary': 'To see a possible effect of a physiological warm temperature shift (WTS, 35°C to 38.5°C) on Per2 transcription and translation in mammalian cells, we performed RNA-seq and Ribo-seq using dexamethasone-synchronized mouse embryonic fibroblast cells with or without WTS treatment at a circadain rising phase of Per2.', 'Series_overall_design': 'We used Dex-synchronized MEF cells treated with or without 2-h WTS. WTS was applied 24 h after Dex synchronization (i.e., circadian rising phase of Per2). Three independent experiments were performed.', 'Series_type': 'Other', 'Series_contributor': 'Miyake,,Takahito', 'Series_sample_id': 'GSM5684946', 'Series_contact_name': 'Masao,,Doi', 'Series_contact_email': 'doimasao@pharm.kyoto-u.ac.jp', 'Series_contact_phone': '81-75753-9555', 'Series_contact_department': 'Department of Systems Biology', 'Series_contact_institute': 'Kyoto University, Graduate School of Pharmaceutical Sciences', 'Series_contact_address': '46-29 Yoshida-Shimo-Adachi-cho, Sakyo-ku', 'Series_contact_city': 'Kyoto', 'Series_contact_zip/postal_code': '6068501', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE188nnn/GSE188529/suppl/GSE188529_RAW.tar', 'Series_platform_id': 'GPL21273', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA779283'}\n",
      "513 PRJNA779461 GSE188574 {'Series_title': 'A dual-activity topoisomerase complex regulates mRNA translation and turnover', 'Series_geo_accession': 'GSE188574', 'Series_status': 'Public on Jun 08 2022', 'Series_submission_date': 'Nov 10 2021', 'Series_last_update_date': 'Jul 14 2022', 'Series_pubmed_id': '35748872', 'Series_summary': 'Topoisomerase 3β (TOP3B) and TDRD3 form a dual-activity topoisomerase complex that interacts with FMRP and can change the topology of both DNA and RNA.  Here, we investigated the post-transcriptional influence of TOP3B and associated proteins on mRNA translation and turnover.  First, we discovered that in human HCT116 colon cancer cells, knock-out (KO) of TOP3B had similar effects on mRNA turnover and translation as did TDRD3-KO, while FMRP-KO resulted in rather distinct effects, indicating that TOP3B had stronger coordination with TDRD3 than FMRP in mRNA regulation.  Second, we identified TOP3B-bound mRNAs in HCT116 cells; we found that while TOP3B did not directly influence the stability or translation of most TOP3B target mRNAs, it stabilized a subset of target mRNAs but had a more complex effect on translation--enhancing for some mRNAs whereas reducing for others.  Interestingly, a point mutation that specifically disrupted TOP3B catalytic activity only partially recapitulated the effects of TOP3B-KO on mRNA stability and translation, suggesting that the impact of TOP3B on target mRNAs is only in part linked to its ability to change topology of mRNAs.  Collectively, our data suggest that TOP3B-TDRD3 can regulate mRNA translation and turnover by mechanisms that are dependent and independent of topoisomerase activity.', 'Series_overall_design': 'To identify the functions of the TOP3B-TDRD3 complex on post-transcriptional regulation in HCT116 cells, total 62 samples were uploaded, including PRO-seq, RNA-seq, Ribo-seq and eCLIP-seq samples from WT and KO/KI cells. Most of them contain two or three replications. For TOP3B eCLIP-seq, TOP3B-KO cells  were used as negative controls separately. For PRO-seq, RNA-seq and Ribo-seq, wild type cells were used as controls.', 'Series_type': 'Other', 'Series_contributor': 'Weidong,,Wang', 'Series_sample_id': 'GSM5685841', 'Series_contact_name': 'Weidong,,Wang', 'Series_contact_email': 'wangw@grc.nia.nih.gov', 'Series_contact_laboratory': 'Laboratory of Genetics and Genomics', 'Series_contact_department': 'National Institute on Aging', 'Series_contact_institute': 'Shuaikun Su', 'Series_contact_address': '251 Bayview Blvd', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21224', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE188nnn/GSE188574/suppl/GSE188574_RAW.tar', 'Series_platform_id': 'GPL30173', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP345494'}\n",
      "514 PRJNA780403 GSE188798 {'Series_title': 'Mapping translation in wild-type and METTL5 knockout (KO) mice', 'Series_geo_accession': 'GSE188798', 'Series_status': 'Public on Nov 19 2021', 'Series_submission_date': 'Nov 15 2021', 'Series_last_update_date': 'Feb 26 2022', 'Series_pubmed_id': '35033535', 'Series_summary': 'We performed ribosome profiling on wild-type and METTL5 KO mice to investigate translation changes due to loss of METTL5.', 'Series_overall_design': 'Comparing sequencing of ribosome-protected-fragments (RPFs, normalized to input RNA-seq) from wild type and METTL5 KO mice.', 'Series_type': 'Other', 'Series_contributor': 'Chuan,,He', 'Series_sample_id': 'GSM5690316', 'Series_contact_name': 'Caraline,,Sepich', 'Series_contact_email': 'csepich@uchicago.edu', 'Series_contact_laboratory': 'GCIS E307 (He lab)', 'Series_contact_department': 'Chemistry', 'Series_contact_institute': 'University of Chicago', 'Series_contact_address': '929 E 57th Street', 'Series_contact_city': 'Chicago', 'Series_contact_state': 'IL', 'Series_contact_zip/postal_code': '60637', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE188nnn/GSE188798/suppl/GSE188798_Sepich-Poore_METTL5_mouse_riboseq_input_merged_featurecount.txt.gz', 'Series_platform_id': 'GPL24247', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP346102'}\n",
      "515 PRJNA780800 GSE188929 {'Series_title': 'Ribosome profiling and RNA sequencing reveal genome-wide cellular translation and transcription regulation under osmotic stress in Lactobacillus rhamnosus ATCC 53103', 'Series_geo_accession': 'GSE188929', 'Series_status': 'Public on Dec 15 2021', 'Series_submission_date': 'Nov 16 2021', 'Series_last_update_date': 'Dec 15 2021', 'Series_pubmed_id': '34899662', 'Series_summary': 'To determine whether osmotic pressure affects the translation efficiency of Lactobacillus rhamnosus, the ribosome profiling assay was performed to analyze the changes in translation efficiency in L. rhamnosus ATCC 53103. Under osmotic stress, differentially expressed genes (DEGs) involved in fatty acid biosynthesis and metabolism, ribosome, and purine metabolism pathways were co-regulated with consistent expression direction at translation and transcription levels. DEGs involved in the biosynthesis of phenylalanine, tyrosine, and tryptophan, and the phosphotransferase system pathways also were co-regulated at translation and transcription levels, while they showed opposite expression direction at two levels. Moreover, DEGs involved in the two-component system, amino acid metabolism, and pyruvate metabolism pathways were only regulated at the transcription level. And DEGs involved in fructose and mannose metabolism were only regulated at the translation level. The translation efficiency of DEGs involved in the biosynthesis of amino acids was downregulated while in quorum sensing and PTS pathways was upregulated. In addition, the ribosome footprints accumulated in open reading frame regions resulted in impaired translation initiation and elongation under osmotic stress. In summary, L. rhamnosus ATCC 53103 could respond to osmotic stress by translation regulation and control the balance between survival and growth of cells by transcription and translation.', 'Series_overall_design': 'To systematically investigate the effect of osmotic stress on transcription and translation regulation, RNA-seq and ribosome profiling were analyzed in the same two sets of parallel populations of L. rhamnosus ATCC 53103 cells with two different concentrations of sodium lactate (control vs. 0.6 M). Each experiment was divided into control group (CG) and osmotic stress group (OS), respectively . The regulation of transcription and translation under osmotic stress were examined by deep sequencing of cellular total mRNAs and RPFs, respectively.', 'Series_type': 'Other', 'Series_contributor': 'Xuejing,,Fan', 'Series_sample_id': 'GSM5692040', 'Series_contact_name': 'Zhen,,Feng', 'Series_contact_email': 'fanxuejing@neau.edu.cn', 'Series_contact_institute': 'Northeast Agricultural University', 'Series_contact_address': 'Changjiang road 600', 'Series_contact_city': 'Harbin', 'Series_contact_state': 'Heilongjiang', 'Series_contact_zip/postal_code': '150030', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE188nnn/GSE188929/suppl/GSE188929_RAW.tar', 'Series_platform_id': 'GPL30959', 'Series_platform_taxid': '568703', 'Series_sample_taxid': '568703', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP346338'}\n",
      "516 PRJNA782500 GSE189306 {'Series_title': 'Deficiency of the RNA-binding protein Cth2 extends yeast replicative lifespan by alleviating its repressive effects on mitochondrial function', 'Series_geo_accession': 'GSE189306', 'Series_status': 'Public on Jul 18 2022', 'Series_submission_date': 'Nov 22 2021', 'Series_last_update_date': 'Aug 26 2022', 'Series_pubmed_id': '35858543', 'Series_summary': 'Iron dyshomeostasis contributes to aging, but little information is available about the molecular mechanisms. Here, we provide evidence that, in Saccharomyces cerevisiae, aging is associated with altered expression of genes involved in iron homeostasis. We further demonstrate that defects in the conserved mRNA-binding protein Cth2, which controls stability and translation of mRNAs encoding iron-containing proteins, increase lifespan by alleviating its repressive effects on mitochondrial function. Mutation of the conserved cysteine residue in Cth2 that inhibits its RNA-binding activity is sufficient to confer longevity, whereas Cth2 gain-of-function shortens replicative lifespan. Consistent with its function in RNA degradation, we demonstrate that Cth2 deficiency relieves Cth2-mediated post-transcriptional repression of nuclear-encoded components of the electron transport chain. Our findings uncover a major role of the RNA-binding protein Cth2 in the regulation of lifespan and suggest that modulation of iron starvation signaling can serve as a target for potential aging interventions.', 'Series_overall_design': 'RNA-Seq and Ribo-Seq analyses of replicatively aged (OLD) and young (YNG) wild-type and cth2 knockout strains', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Vyacheslav,M,Labunskyy', 'Series_sample_id': 'GSM5698693', 'Series_contact_name': 'Vyacheslav,,Labunskyy', 'Series_contact_email': 'vlabuns@bu.edu', 'Series_contact_laboratory': 'Labunskyy Lab', 'Series_contact_department': 'Department of Dermatology', 'Series_contact_institute': 'Boston University School of Medicine', 'Series_contact_address': '609 Albany St', 'Series_contact_city': 'Boston', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02118', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE189nnn/GSE189306/suppl/GSE189306_RAW.tar', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP347195'}\n",
      "517 PRJNA784790 GSE189854 {'Series_title': 'Ribo-Seq and RNA-seq sequencing of WT and Rpl3l-/- mouse hearts', 'Series_geo_accession': 'GSE189854', 'Series_status': 'Public on May 16 2022', 'Series_submission_date': 'Nov 30 2021', 'Series_last_update_date': 'Sep 08 2023', 'Series_pubmed_id': '36882085', 'Series_summary': 'RPL3L is a ribosomal protein expressed exclusively in adult straited muscle tissues, and a paralogue of the ubiquitously expressed RPL3. Here, we are looking into the effects of Rpl3l knockout on translation in adult mice hearts.', 'Series_overall_design': 'Ribo-SEQ and RNA-seq done on WT and Rpl3l-/- mouse hearts in biological triplicates.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Ivan,,Milenkovic', 'Series_sample_id': 'GSM5708612', 'Series_contact_name': 'Eva Maria,,Novoa', 'Series_contact_email': 'evamaria.novoa@gmail.com', 'Series_contact_phone': '+34933160260', 'Series_contact_laboratory': 'Epitranscriptomics and RNA Dynamics', 'Series_contact_institute': 'Center for Genomic Regulation', 'Series_contact_address': 'Dr. Aiguader 88', 'Series_contact_city': 'Barcelona', 'Series_contact_zip/postal_code': '08003', 'Series_contact_country': 'Spain', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE189nnn/GSE189854/suppl/GSE189854_RAW.tar', 'Series_citation': 'https://doi.org/10.1101/2021.12.04.471171', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA784790'}\n",
      "518 PRJNA787943 GSE190670 {'Series_title': 'The Polyamine-Hypusine Circuit Controls an Oncogenic Translational Program Necessary for MYC-Driven Lymphomagenesis [Ribo-seq]', 'Series_geo_accession': 'GSE190670', 'Series_status': 'Public on Dec 31 2022', 'Series_submission_date': 'Dec 10 2021', 'Series_last_update_date': 'Dec 31 2022', 'Series_summary': 'Hypusinated EiF5A likely impacts diverse cellular pathways through direct regulation of its translational targets or via secondary effects on other biological processes. We examined the global effects of knockdown of EiF5A and Dhps on the transcriptome and translational landscape of MYC-induced lymphoma.', 'Series_overall_design': 'To evaluate the effects of EiF5A or Dhps depletion on translation, Ribosome profiling analyses of Em-Myc; Rosa26-rtTA2 lymphoma cells that inducibly express sh-Ren, sh-EiF5A or sh-Dhps were performed along with RNA-seq.', 'Series_type': 'Other', 'Series_contributor': 'John,L,Cleveland', 'Series_sample_id': 'GSM5728629', 'Series_contact_name': 'Yonghong,,Zhang', 'Series_contact_email': 'zyhzhang@gmail.com', 'Series_contact_department': 'Biomedical Informatics', 'Series_contact_institute': 'Moffitt Cancer Center', 'Series_contact_address': '12902 USF Magnolia Drive', 'Series_contact_city': 'TAmpa', 'Series_contact_state': 'FL', 'Series_contact_zip/postal_code': '33612', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE190nnn/GSE190670/suppl/GSE190670_Read_count_raw.txt.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA787943'}\n",
      "519 PRJNA789247 GSE190980 {'Series_title': 'RNA-seq and Ribo-seq of Ythdc2KO and Ythdc2-WLA mutants', 'Series_geo_accession': 'GSE190980', 'Series_status': 'Public on Jan 20 2022', 'Series_submission_date': 'Dec 15 2021', 'Series_last_update_date': 'Mar 03 2022', 'Series_pubmed_id': '35058317', 'Series_summary': 'We sequenced RNA from testis of Ythdc2 mutant and wild type mice to compare gene expression level, and translation efficiency.', 'Series_overall_design': 'mRNA abundance and translation efficiency profiles by deep sequencing using Illumina Hiseq2500.', 'Series_type': 'Other', 'Series_contributor': 'Devanshi,,Jain', 'Series_sample_id': 'GSM5736939', 'Series_contact_name': 'Yuhki,,Saito', 'Series_contact_email': 'ysaito@rockefeller.edu', 'Series_contact_laboratory': 'Laboratory of Molecular Neuro-oncology', 'Series_contact_institute': 'The Rockefeller University', 'Series_contact_address': '1230 York avenue', 'Series_contact_city': 'New York', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '10065', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE190nnn/GSE190980/suppl/GSE190980_anota2seq_output.xlsx', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA789247'}\n",
      "520 PRJNA793438 GSE192846 {'Series_title': 'Determine the effects of QKI7 on translation efficiency via ribosome profiling (Ribo-seq)', 'Series_geo_accession': 'GSE192846', 'Series_status': 'Public on May 17 2023', 'Series_submission_date': 'Dec 31 2021', 'Series_last_update_date': 'Aug 16 2023', 'Series_pubmed_id': '37379838', 'Series_web_link': 'https://pubmed.ncbi.nlm.nih.gov/37379838/', 'Series_summary': 'N7-methylguanosine (m7G) modification, routinely occurring at the 5’ cap of mRNA or within tRNA and rRNA, also exists internally in mRNA. Although essential for mRNA translation as well as stress response, the “reader” protein for mRNA internal m7G modification is still unrevealed. Here, we reported that Quaking protein (QKI), especially QKI7, can selectively recognize the internal mRNA m7G decoration in the cytosol of various cell types. We identified over 1000 confident m7G-modified and QKI binding RNA targets with a conserved motif, “GANGAN (N=A/U/G)”. More strikingly, internal m7G reader QKI7 directly interacts with the SG core protein G3BP1 and can shuttle a subset of m7G-modified transcripts into SG mRNA pool under oxidative stress condition. Additionally, by sequestering mRNA within SGs, QKI7 modulates the translation efficiency of selected transcripts. Moreover, in line with the observation that doxorubicin triggers the assembly of SGs, QKI7 mediates the sensitivity of cancer cells to chemotherapy drug treatment.', 'Series_overall_design': 'Two 15-cm dishes empty vector or QKI7 overexpression U2OS cells were treated with 0.5mM NaAsO2 for 1 hour before harvesting the cells. The samples were subjected to ribosome profiling and both the input RNA and footprint RNAs were used for sequencing.', 'Series_type': 'Other', 'Series_contributor': 'Jianjun,,Chen', 'Series_sample_id': 'GSM5766807', 'Series_contact_name': 'Jianjun,,Chen', 'Series_contact_email': 'jianchen@coh.org', 'Series_contact_phone': '6262185591', 'Series_contact_laboratory': 'Chen Lab', 'Series_contact_department': 'Systems Biology', 'Series_contact_institute': 'Beckman Research Institute of City of Hope', 'Series_contact_address': '1218 S. 5th Ave., Room 2062', 'Series_contact_city': 'Monrovia', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '91016', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE192nnn/GSE192846/suppl/GSE192846_ribo.tpm.txt.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA793438'}\n",
      "521 PRJNA795419 GSE193200 {'Series_title': 'ribo-seq in control and U1 AMO treated HeLa cells', 'Series_geo_accession': 'GSE193200', 'Series_status': 'Public on Feb 27 2023', 'Series_submission_date': 'Jan 07 2022', 'Series_last_update_date': 'May 29 2023', 'Series_pubmed_id': '37224962', 'Series_summary': 'Control and U1 AMO were transfected into Hela cells, Ribosome-associated RNAs were purified and sequenced in Novaseq platform.To compare the translation efficiency, mRNA-seq were performed in parallel.', 'Series_overall_design': 'To understand the ribosome-binding profile of mRNAs in control and U1 AMO treated HeLa cells, we carried out Ribo-seq analysis in control and U1 AMO treated HeLa cells. To compare the translation efficiency, we performed mRNA-seq analysis in parallel.', 'Series_type': 'Other', 'Series_contributor': 'Chengguo,,Yao', 'Series_sample_id': 'GSM6347759', 'Series_contact_name': 'chengguo,,yao', 'Series_contact_email': 'yaochguo@mail.sysu.edu.cn', 'Series_contact_department': 'school of medicine', 'Series_contact_institute': 'sysu', 'Series_contact_address': 'No. 74 zhongshan 2rd road', 'Series_contact_city': 'guangzhou', 'Series_contact_zip/postal_code': '510080', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE193nnn/GSE193200/suppl/GSE193200_RAW.tar', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA795419'}\n",
      "522 PRJNA798445 GSE193945 {'Series_title': 'The integrated stress response protects microtubule dynamics to recover from proteotoxic stress', 'Series_geo_accession': 'GSE193945', 'Series_status': 'Public on Jul 15 2022', 'Series_submission_date': 'Jan 18 2022', 'Series_last_update_date': 'Jul 15 2022', 'Series_pubmed_id': '35758650', 'Series_summary': 'By performing ribosome profiling on squamous cell carcinoma stem cells (either control or S51A mutant) we found that the ISR translationally regulates a network of centrosomal proteins to help cellular recovery upon proteotoxic stress.', 'Series_overall_design': 'Ribosome profiling and total RNA sequencing were performed on control and S51A cells either vehicle-treated or treated with the proteosome inhibitor bortezomib for 4.5h', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Elaine,,Fuchs', 'Series_sample_id': 'GSM5824009', 'Series_contact_name': 'Douglas,,Barrows', 'Series_contact_email': 'dbarrows@rockefeller.edu', 'Series_contact_phone': '7749940981', 'Series_contact_institute': 'Rockefeller University', 'Series_contact_address': '1230 York Ave', 'Series_contact_city': 'New York', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '10065', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE193nnn/GSE193945/suppl/GSE193945_raw_counts_totalRNA_reseqMerge.csv.gz', 'Series_platform_id': 'GPL24247', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA798445'}\n",
      "523 PRJNA799348 GSE194172 {'Series_title': 'The Interplay Between Translational Efficiency, Poly(A) Tails, MicroRNAs, and Neuronal Activation', 'Series_geo_accession': 'GSE194172', 'Series_status': 'Public on Mar 16 2022', 'Series_submission_date': 'Jan 21 2022', 'Series_last_update_date': 'Mar 17 2022', 'Series_pubmed_id': '35273099', 'Series_summary': 'Neurons provide a rich setting for studying post-transcriptional control. Here, we investigate the landscape of translational control in neurons and search for mRNA features that explain differences in translational efficiency (TE), considering the interplay between TE, mRNA poly(A)-tail lengths, microRNAs, and neuronal activation. In neurons and brain tissues, TE correlates with tail length, and a few dozen mRNAs appear to undergo cytoplasmic polyadenylation upon light or chemical stimulation. However, the correlation between TE and tail length is modest, explaining <5% of TE variance, and even this modest relationship diminishes when accounting for other mRNA features. Thus, tail length appears to affect TE only minimally. Accordingly, miRNAs, which accelerate deadenylation of their mRNA targets, primarily influence target mRNA levels, with no detectable effect on either steady-state tail lengths or TE. Larger correlates with TE include codon composition and predicted mRNA folding energy. When combined in a model, the identified correlates explain 38–45% of TE variance. These results provide a framework for considering the relative impact of factors that contribute to translational control in neurons. They indicate that when examined in bulk, translational control in neurons largely resembles that of other types of post-embryonic cells. Thus, detection of more specialized control might require analyses that can distinguish translation occurring in neuronal processes from that occurring in cell bodies.', 'Series_overall_design': 'The experiments consist of 12 matched samples of ribosome profiling libraries (small RNA-seq and RPF reads, 24 files in total) from primary neuronal cortical culture or mouse tissues. These samples have matched PAL-seq V2 or TAIL-seq data. In addition, there are eight samples (light stimulated mouse visual cortex samples) that have PAL-seq V2 data but no matched ribosome profiling data.', 'Series_type': 'Other', 'Series_contributor': 'Timothy,J,Eisen', 'Series_sample_id': 'GSM5830123', 'Series_contact_name': 'Timothy,J,Eisen', 'Series_contact_email': 'timeisen@berkeley.edu', 'Series_contact_laboratory': 'John Kuriyan', 'Series_contact_department': 'QB3', 'Series_contact_institute': 'UC Berkeley', 'Series_contact_address': 'Stanley Hall', 'Series_contact_city': 'Berkeley', 'Series_contact_state': 'CALIFORNIA', 'Series_contact_zip/postal_code': '94720', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE194nnn/GSE194172/suppl/GSE194172_TailsTeAllFinal20220119.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA799348'}\n",
      "524 PRJNA801130 GSE195524 {'Series_title': 'Transcriptome and translatome of killing supernatant cultured human monocytes', 'Series_geo_accession': 'GSE195524', 'Series_status': 'Public on Jul 31 2023', 'Series_submission_date': 'Jan 27 2022', 'Series_last_update_date': 'Jul 31 2023', 'Series_summary': 'To investigate whether metoprolol treatment could affect the transcriptome and translatome of killing supernatant cultured human monocytes', 'Series_overall_design': 'PBMC were isolated from buffy coat of healthy blood donors by Ficoll gradients, and CD14+ monocytes were isolated from PBMC with anti-CD14 magnetic beads (Miltenyi Biotec). CD14+ monocytes were cultured with killing supernatant  for 5 days and treated with metoprolol or PBS prophylactically or conditionally and then were sent for both ribo-seq and RNA-seq', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Gang,,Xu', 'Series_sample_id': 'GSM5839944', 'Series_contact_name': 'Yuzhuo,,Yang', 'Series_contact_email': 'camilleyyz@gmail.com', 'Series_contact_institute': 'Tsinghua University', 'Series_contact_address': 'Zheng Yutong Building room D311', 'Series_contact_city': 'Beijing', 'Series_contact_zip/postal_code': '100084', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE195nnn/GSE195524/suppl/GSE195524_Readcount.xlsx', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA801130'}\n",
      "525 PRJNA801467 GSE195601 {'Series_title': 'A Single-copy Knock In Translating Ribosome ImmunoPrecipitation (SKI TRIP) tool kit for tissue specific profiling of actively translated mRNAs in C. elegans.', 'Series_geo_accession': 'GSE195601', 'Series_status': 'Public on Jan 31 2022', 'Series_submission_date': 'Jan 28 2022', 'Series_last_update_date': 'May 03 2023', 'Series_pubmed_id': '37056370', 'Series_web_link': 'https://doi.org/10.1016/j.crmeth.2023.100433', 'Series_summary': 'Translating Ribosome Affinity Purification (TRAP) methods have emerged as a powerful approach to profile actively translated transcripts in specific cell and tissue types.  Epitope tagged ribosomal subunits are expressed in defined cell populations and used to pull down ribosomes and their associated mRNAs, providing a snapshot of cell type-specific translation occurring in that space and time. Current TRAP toolkits available to the C. elegans community have been built using multi-copy arrays, randomly integrated in the genome.  Here we introduce a Single-copy Knock In Translating Ribosome ImmunoPrecipitation (SKI TRIP) tool kit, a collection of C. elegans strains engineered by CRISPR in which tissue specific expression of FLAG tagged ribosomal subunit protein RPL-22 is driven by cassettes present in single copy from defined sites in the genome. In depth characterization of the SKI TRIP strains and methodology shows that 3xFLAG tagged RPL-22 expressed from its endogenous locus or within defined cell types incorporates into actively translating ribosomes and can be used to efficiently and cleanly pull-down cell type specific transcripts without impacting overall mRNA translation or fitness of the animal. We propose SKI TRIP use for the study of processes that are acutely sensitive to changes in translation, such as aging.', 'Series_overall_design': 'RNAseq from C. elegans after SKI TRIP (IP) experiments (total RNA and IPed RNA in triplicates per strain) from strains MSD470=UBI, WBM1364=soma, WBM1471=neuroB, MSD473=neuroA, WBM1339=muscle, WBM1471=intestine', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'William,B,Mair', 'Series_sample_id': 'GSM5841661', 'Series_contact_name': 'Laura,Elisabeth,Wester', 'Series_contact_email': 'lwester@hsph.harvard.edu', 'Series_contact_institute': 'Harvard T.H. Chan School of Public', 'Series_contact_address': '677 Huntington Ave', 'Series_contact_city': 'BOSTON', 'Series_contact_state': 'Massachusetts', 'Series_contact_zip/postal_code': '02115', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE195nnn/GSE195601/suppl/GSE195601_WT.deseq2_results.tsv.gz', 'Series_platform_id': 'GPL26672', 'Series_platform_taxid': '6239', 'Series_sample_taxid': '6239', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA801467'}\n",
      "526 PRJNA801861 GSE195703 {'Series_title': 'RNA m1A methylation regulates glycolysis of cancer cells through modulating ATP5D [Ribo and RNA-seq]', 'Series_geo_accession': 'GSE195703', 'Series_status': 'Public on Feb 05 2022', 'Series_submission_date': 'Jan 29 2022', 'Series_last_update_date': 'Feb 05 2022', 'Series_summary': 'Studies on biological functions of RNA modifications such as N6-methyladenosine (m6A) in mRNA have sprung up in recent years, while the roles of N1-methyladenosine (m1A) in cancer progression remain largely unknown. We find m1A demethylase ALKBH3 can regulate the glycolysis of cancer cells via a demethylation activity dependent manner. Specifically, sequencing and functional studies confirm that ATP5D, one of the most important subunit of ATP synthase, is involved in m1A demethylase ALKBH3-regulated glycolysis of cancer cells. The m1A modified A71 at the Exon 1 of ATP5D negatively regulates its translation elongation via increasing the binding with YTHDF1/eRF1 complex, which facilitates the release of mRNA from ribosome complex. m1A also regulates mRNA stability of E2F1, which directly binds with ATP5D promoter to initiate its transcription. Targeted specific demethylation of ATP5D m1A by dm1ACRISPR system can significantly increase the expression of ATP5D and glycolysis of cancer cells. In vivo data confirm the roles of m1A/ATP5D in tumor growth and cancer progression. Our study for the first time reveals a crosstalk of mRNA m1A modification and cell metabolism, which expands the understanding of such interplays that are essential for cancer therapeutic application.', 'Series_overall_design': 'Ribo-seq and mRNA-seq in Human HeLa WT(n=2) and HeLa ALKBH3-KO(n=2), then caculated the quantity and efficiency of translation', 'Series_type': 'Other', 'Series_contributor': 'Guoyou,,Xie', 'Series_sample_id': 'GSM5848579', 'Series_contact_name': 'xie,,guoyou', 'Series_contact_email': 'xiegy7@mail2.sysu.edu.cn', 'Series_contact_phone': '13719173117', 'Series_contact_institute': '中山大学', 'Series_contact_address': '大学城中山大学', 'Series_contact_city': 'guangzhou', 'Series_contact_zip/postal_code': '511400', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE195nnn/GSE195703/suppl/GSE195703_mRNA-seq_All.fpkm.anno.xls.gz', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA801861'}\n",
      "527 PRJNA803443 GSE196124 {'Series_title': 'The translational landscape of direct cardiac reprogramming reveals a role of Ybx1 in repressing cardiac fate acquisition [Ribo-seq and RNA-seq]', 'Series_geo_accession': 'GSE196124', 'Series_status': 'Public on Aug 24 2023', 'Series_submission_date': 'Feb 04 2022', 'Series_last_update_date': 'Aug 24 2023', 'Series_summary': 'Direct reprogramming of fibroblasts into induced cardiomyocytes (iCMs) holds great promise for heart regeneration. Although great progress has been made in understanding the transcriptional and epigenetic mechanisms of iCM reprogramming, its translational regulation remains largely unexplored. Here we characterized the translational landscape of iCM reprogramming using integrative ribosome and transcriptomic profiling, and showed extensive translatome re-patterning during fibroblast conversion into iCM. Loss of function screening for translational regulators uncovered Ybx1 as a critical barrier to iCM induction. Knockdown of Ybx1 dramatically improved the efficiency and quality of iCM reprogramming. Mechanistically, Ybx1 directly bound the transcripts of Srf and Baf60c, both of which mediated, at least partially, the repressive effect of Ybx1 on iCM generation.and Depletion of Ybx1 de-repressed the translation of its targets including SRF and Baf60c. Interestingly, upon removing Ybx1, Tbx5 alone could reprogram fibroblasts into iCMs that exhibit classic iCM molecular features and reprogramming trajectories revealed by our single cell dual-omics. In sum, we presented here a global view of translatome dynamics of cardiac reprogramming and identified a novel translational barrier, Ybx1, to iCM generation. Removing this barrier allowed the single factor mediated iCM reprogramming.', 'Series_overall_design': 'Ribo-seq and RNA-seq libraries were generated on day 5 cardiac fibroblasts (CFs) infected with viral vector encoding the polycistronic Mef2c/Gata4/Tbx5 (MGT) and control cells, namely, the primary CFs', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Li,,Qian', 'Series_sample_id': 'GSM5860396', 'Series_contact_name': 'Yuchen,,Yang', 'Series_contact_email': 'yangych68@mail.sysu.edu.cn', 'Series_contact_institute': 'Sun Yat-sen University', 'Series_contact_address': 'Xingangxi Road 135', 'Series_contact_city': 'Guangzhou', 'Series_contact_state': 'Guangdong', 'Series_contact_zip/postal_code': '510275', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE196nnn/GSE196124/suppl/GSE196124_Day0_Day5_Riboseq_raw_count_matrix.csv.gz', 'Series_platform_id': 'GPL24247', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA803443'}\n",
      "528 PRJNA806946 GSE196709 {'Series_title': '5-Fluorouridine restores functional full-length p53 in TP53 nonsense mutant human tumor cells (Ribo-Seq)', 'Series_geo_accession': 'GSE196709', 'Series_status': 'Public on Dec 01 2022', 'Series_submission_date': 'Feb 14 2022', 'Series_last_update_date': 'Dec 01 2022', 'Series_pubmed_id': '36433934', 'Series_summary': 'The tumor suppressor gene TP53 is mutated in approximately half of all human tumors. Of those, around 10% are nonsense mutations that produce truncated and inactive p53 protein. Induction of translational readthrough is a promising approach for rescuing full-length p53 and thereby eliminate tumor cells with nonsense mutant TP53. To find novel nonsense mutant TP53 readthrough-inducing compounds with a tolerable toxicity profile, we performed an in silico screening of data at the National Cancer Institute database and identified 5-Fluorouracil (5-FU). We show here that 5-FU induces full-length p53 in human tumor cells carrying R213X nonsense mutant TP53 and that this activity is mediated by its metabolite 5-Fluorouridine (FUr). Ribo-seq analysis validated induction of translational readthrough by FUr. We also show that FUr is incorporated into RNA where it potentially allows base pairing of Arg tRNA at the R213X UGA premature termination codon. Full-length p53 rescued by FUr is transcriptionally active and triggers p53-dependent cell death. Moreover, treatment with 5-FU or FUr restores full-length p53 expression in TP53 R213X mutant human tumor xenografts in vivo. Our results suggest that induction of readthrough by 5-FU/FUr could contribute to therapeutic efficacy in patients with TP53 nonsense mutant tumors.', 'Series_overall_design': 'Ribo-seq analysis of readthrough in three conditions (Untreated, FUr, G418) in empty vector and R213X vector. Four replicates per group.', 'Series_type': 'Other', 'Series_contributor': 'Klas,G,Wiman', 'Series_sample_id': 'GSM5899121', 'Series_contact_name': 'Sebastiaan,,van Heesch', 'Series_contact_email': 'S.vanheesch@prinsesmaximacentrum.nl', 'Series_contact_laboratory': 'van Heesch', 'Series_contact_institute': 'Princess Maxima Center', 'Series_contact_address': 'Heidelberglaan 25', 'Series_contact_city': 'Utrecht', 'Series_contact_zip/postal_code': '3584 CS', 'Series_contact_country': 'Netherlands', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE196nnn/GSE196709/suppl/GSE196709_ribo_counts_matrix.txt.gz', 'Series_platform_id': 'GPL30173', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA806946'}\n",
      "529 PRJNA808115 GSE196982 {'Series_title': 'Eukaryotic initiation factor 5A2 regulates expression of antiviral genes', 'Series_geo_accession': 'GSE196982', 'Series_status': 'Public on Aug 01 2022', 'Series_submission_date': 'Feb 17 2022', 'Series_last_update_date': 'Oct 31 2022', 'Series_pubmed_id': '35358571', 'Series_summary': 'Using ribosome profiling and RNA-Sequencing, we reveal that eIF5A2 is functionally distinct from eIF5A1 and does not regulate transcript-specific or global protein synthesis. Instead, eIF5A2 knockout leads to decreased intrinsic antiviral gene expression.', 'Series_overall_design': 'SW480 cells were grown in complete media for a day before harvesting.', 'Series_type': 'Other', 'Series_contributor': 'Amy,S,Lee', 'Series_sample_id': 'GSM5906254', 'Series_contact_name': 'Amy,Si-Ying,Lee', 'Series_contact_department': 'Cancer Immunology and Virology', 'Series_contact_institute': 'DFCI', 'Series_contact_address': '450 Brookline Ave SM520', 'Series_contact_city': 'Boston', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02215', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE196nnn/GSE196982/suppl/GSE196982_count_table_merged.xlsx', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA808115'}\n",
      "530 PRJNA808741 GSE197060 {'Series_title': 'Development of nanoRibo-seq enables study of regulated translation by cortical neuron subtypes, showing uORF translation in synaptic-axonal genes', 'Series_geo_accession': 'GSE197060', 'Series_status': 'Public on Aug 25 2023', 'Series_submission_date': 'Feb 20 2022', 'Series_last_update_date': 'Nov 13 2023', 'Series_pubmed_id': '37624698', 'Series_web_link': 'https://www.cell.com/cell-reports/fulltext/S2211-1247(23)01006-9', 'Series_summary': 'Investigation of translation in rare cell types or subcellular contexts is challenging due to large input requirements for standard approaches. Here, we present “nanoRibo-seq” an optimized approach using 102- to 103-fold less input material than bulk approaches. nanoRibo-seq exhibits rigorous quality control features consistent with quantification of ribosome protected fragments with as few as 1,000 cells. We compare translatomes of two closely related cortical neuron subtypes, callosal projection neurons (CPN) and subcerebral projection neurons (SCPN), during their early postnatal development. We find that, while translational efficiency is highly correlated between CPN and SCPN, several dozen mRNAs are differentially translated. We further examine upstream open reading frame (uORF) translation and identify that mRNAs involved in synapse organization and axon development are highly enriched for uORF translation in both subtypes. nanoRibo-seq enables investigation of translational regulation of rare cell types in vivo and offers a flexible approach for globally quantifying translation from limited input material.', 'Series_overall_design': 'nanoRibo-seq: low input ribosome profiling', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'John,E,Froberg', 'Series_sample_id': 'GSM7453792', 'Series_contact_name': 'John,Edward,Froberg', 'Series_contact_email': 'jfroberg@fas.harvard.edu', 'Series_contact_phone': '8479770174', 'Series_contact_laboratory': 'Macklis Lab', 'Series_contact_department': 'Stem Cell & Regenerative Biology', 'Series_contact_institute': 'Harvard University', 'Series_contact_address': '7 Divinity Avenue', 'Series_contact_city': 'Cambridge', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02138', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE197nnn/GSE197060/suppl/GSE197060_sorted_neurons_three_utr_FC_matrix.csv.gz', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA808741'}\n",
      "531 PRJNA809314 GSE197174 {'Series_title': 'Intron-mediated induction of phenotypic heterogeneity', 'Series_geo_accession': 'GSE197174', 'Series_status': 'Public on Feb 23 2022', 'Series_submission_date': 'Feb 22 2022', 'Series_last_update_date': 'May 11 2022', 'Series_pubmed_id': '35444278', 'Series_summary': 'Introns are universally present in the nuclear genomes of eukaryotes. The budding yeast, an otherwise intron-poor species, preserves two sets of ribosomal protein (RP) genes differing primarily in their introns. Despite recent findings on the role of RP introns under stress and starvation, understanding the contribution of introns to ribosome regulation remains challenging. Here, combining isogrowth profiling with single-cell protein measurements, we found that introns can mediate inducible phenotypic heterogeneity conferring a clear fitness advantage. Osmotic stress leads to bimodal expression of the small ribosomal subunit protein Rps22B, mediated by an intron in the 5′ untranslated region of its transcript. The two resulting yeast subpopulations differ in their ability to cope with starvation. Low Rps22B protein levels resulted in prolonged survival under sustained starvation, while high Rps22B levels enabled cells to grow faster after transient starvation. Further, yeast growing at high sugar concentrations – similar to those in ripe grapes – exhibit bimodal Rps22B expression when approaching stationary phase. Differential intron-mediated regulation of RP genes thus provides a way to diversify the population when starvation looms in natural environments. Our findings reveal a new role for introns in inducing phenotypic heterogeneity in changing environments and suggest that duplicated RP genes in yeast contribute to resolving the evolutionary conflict between precise expression control and environmental responsiveness.', 'Series_overall_design': 'RNA samples from treatments with single stressors were obtained from cultures with automated re-inoculation every 8 hours, with a total incubation time of 24 hours, to keep cultures in exponential phase and assure the drug effects have taken place. Strains were grown in YPD with a drug, either in a concentration gradient to select samples closest to 50% of growth inhibition (LiCl, Fenpropimorph, and Myriocin), or in a single concentration (0.4 M NaCl and glucose 20%). Fenpropimorph (Sigma Aldrich cat. No. 36772) was tested in a 0.5-1.5 µM gradient, sequenced samples were treated with 0.85 µM Fenpropimorph. Myriocin (Sigma Aldrich cat. No. 476300) gradient encompassed 0.25-0.75 µg/ml; 0.32 µg/ml Myriocin samples were sequenced. For each sample of extracted RNA, both ribodepleted total RNA and mRNA Seq libraries were prepared at the Cologne Center for Genomics (CCG) and sequenced using 2×100bp paired-end reads.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Tobias,,Bollenbach', 'Series_sample_id': 'GSM5910771', 'Series_contact_name': 'Martin,,Lukačišin', 'Series_contact_laboratory': 'Bollenbach Lab', 'Series_contact_department': 'Institute for Biological Physics', 'Series_contact_institute': 'Universität zu Köln', 'Series_contact_address': 'Zülpicher Straße 47a', 'Series_contact_city': 'Köln', 'Series_contact_state': 'Deutschland', 'Series_contact_zip/postal_code': '50674', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE197nnn/GSE197174/suppl/GSE197174_countsIntrons.txt.gz', 'Series_platform_id': 'GPL27812', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA809314'}\n",
      "532 PRJNA809587 GSE197265 {'Series_title': 'Translatome and transcriptome co-profiling reveals a role of TPRXs in human zygotic genome activation', 'Series_geo_accession': 'GSE197265', 'Series_status': 'Public on Oct 08 2022', 'Series_submission_date': 'Feb 23 2022', 'Series_last_update_date': 'Oct 10 2022', 'Series_pubmed_id': '36074823', 'Series_web_link': 'https://www.science.org/doi/10.1126/science.abo7923', 'Series_summary': 'Translational regulation plays a critical role during oocyte-to-embryo transition including zygotic genome activation. However, the translatome during OET and molecular mechanisms underlying human ZGA remain largely uncharted. Here, we integrated ultra-low-input Ribo-seq with RNA-seq (R2-lite) to jointly profile the translatome and transcriptome from the same samples across 8 stages in human oocytes and early embryos. These data not only unveil conservation and divergence of the translation landscapes between human and mouse OET, but also identify critical regulators of human ZGA.', 'Series_overall_design': 'Ultra-low-input Ribo-seq(Ribo-lite) with RNA-seq (R2-lite) was performed on human oocytes and early embryos. RNA-seq was performed on human embryos with knockdown of transcription factors or under drug treatment.  RNA-seq and Stacc-seq were performed on hESC overexpressing transcription factors.', 'Series_type': 'Other', 'Series_contributor': 'Wei,,Xie', 'Series_sample_id': 'GSM6204108', 'Series_contact_name': 'Zhuoning,,Zou', 'Series_contact_email': 'zhuoningzou@gmail.com', 'Series_contact_phone': '18813108036', 'Series_contact_laboratory': 'Wei Xie', 'Series_contact_department': 'School of Life Sciences', 'Series_contact_institute': 'Tsinghua University', 'Series_contact_address': '30 Shuangqing Rd', 'Series_contact_city': 'Peking', 'Series_contact_zip/postal_code': '100084', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE197nnn/GSE197265/suppl/GSE197265_h_siTPRXL_d3_smartseq_fpkm.txt.gz', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA809587'}\n",
      "533 PRJNA810853 GSE197536 {'Series_title': 'Stress-induced transcriptional readthrough into neighboring genes is linked to intron retention', 'Series_geo_accession': 'GSE197536', 'Series_status': 'Public on Oct 27 2022', 'Series_submission_date': 'Feb 27 2022', 'Series_last_update_date': 'Jan 04 2023', 'Series_pubmed_id': '36505935', 'Series_summary': 'Exposure to certain stresses leads to readthrough transcription downstream of gene ends. Here we found that this phenomenon impacts the expression of genes located downstream to readthrough genes, whereby readthrough transcription proceeds into downstream genes, termed read-in genes. Using polyA-selected RNA-seq data from mouse fibroblasts, we identified widespread read-in in heat shock, oxidative and osmotic stress conditions. Read-in genes share distinctive genomic characteristics; they are extremely short, mainly due to less, shorter, introns, and they are highly GC rich. Furthermore, using ribosome footprint profiling we found that the translation of genes with high degrees of read-in is significantly reduced. Strikingly, read-in genes show extremely high levels of intron retention during stress, mostly in their first intron. While read-in genes share features that are generally associated with increased likelihood of intron retention, such as short introns and high GC content, intron retention in read-in genes during stress exceeds greatly beyond what is expected given their genomic properties. Finally, we found that first introns in read-in genes have weaker 5’ and 3’ splice sites. Our data portray a relationship between read-in and intron retention, suggesting it may have co-evolved to facilitate reduced translation of read-in genes during stress.', 'Series_overall_design': 'Induction of various types of stress in NIH3T3 cells, followed by transcriptome and translatome analysis.', 'Series_type': 'Other', 'Series_contributor': 'Reut,,Shalgi', 'Series_sample_id': 'GSM5919985', 'Series_contact_name': 'Reut,,Shalgi', 'Series_contact_email': 'reutshalgi@technion.ac.il', 'Series_contact_department': 'Faculty of Medicine', 'Series_contact_institute': 'Technion', 'Series_contact_address': '1 Efron st.', 'Series_contact_city': 'Haifa', 'Series_contact_zip/postal_code': '31096', 'Series_contact_country': 'Israel', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE197nnn/GSE197536/suppl/GSE197536_read_in_per_sample.txt.gz', 'Series_platform_id': 'GPL28457', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA810853'}\n",
      "534 PRJNA811788 GSE197735 {'Series_title': 'Frequent 4EBP1 amplification induces synthetic dependence on FGFR signaling in cancer', 'Series_geo_accession': 'GSE197735', 'Series_status': 'Public on Mar 05 2022', 'Series_submission_date': 'Mar 02 2022', 'Series_last_update_date': 'Jun 01 2022', 'Series_pubmed_id': '35626002', 'Series_summary': 'The eIF4E translation initiation factor has oncogenic properties and concordantly, the inhibitory eIF4E-binding protein (4EBP1) is considered a tumor suppressor. The exact molecular effects of 4EBP1 activation in cancer are still unknown. Surprisingly, 4EBP1 is a target of genomic copy number gains (Chr. 8p11) in breast and lung cancer. We notice that 4EBP1 gains are genetically linked to gains in neighboring genes including WHSC1L1 and FGFR1. Our results show that FGFR1 gains act to attenuate the function of 4EBP1 via PI3K mediated phosphorylation at Thr37/46, Ser65, and Thr70 sites. This implies that not 4EBP1 but instead FGFR1 is the genetic target of Chr. 8p11 gains in breast and lung cancer. Accordingly, these tumors show increased sensitivity to FGFR1 and PI3K inhibition and this is a therapeutic vulnerability through restoring the tumor-suppressive function of 4EBP1. Ribosome profiling reveals genes involved in insulin signaling, glucose metabolism, and inositol pathway to be the relevant translational targets of 4EBP1. These mRNAs are among the top 200 translation targets and are highly enriched for structure and sequence motifs in their 5’UTR that depends on the 4EBP1-EIF4E activity. In summary, we identify the translational targets of 4EBP1-EIF4E that facilitate the tumor suppressor function of 4EBP1 in cancer.', 'Series_overall_design': 'We performed ribosome footprinting and RNA sequencing on six samples of 293T 4A cells including three untreated samples (DMSO) and three samples that were collected 2 hrs after treatment with Doxycycline (2mg/ml). DMSO treated samples are indicated as CT and Dox treated samples are indicate as Dox.', 'Series_type': 'Other', 'Series_contributor': 'Kamini,,Singh', 'Series_sample_id': 'GSM5929279', 'Series_contact_name': 'Prathibha,,Mohan', 'Series_contact_email': 'prathibhamohanp@gmail.com', 'Series_contact_phone': '6468882528', 'Series_contact_laboratory': 'Wendel Lab', 'Series_contact_department': 'Cancer Biology and Genetics', 'Series_contact_institute': 'Memorial Sloan Kettering Cancer Center', 'Series_contact_address': '1275 York Avenue', 'Series_contact_city': 'New York', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '10010', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE197nnn/GSE197735/suppl/GSE197735_Project_5640ribdiff-output-WithGeneSymbols.txt.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA811788'}\n",
      "535 PRJNA812878 GSE197915 {'Series_title': 'METTL16 promotes translation and lung tumorigenesis by sequestering cytoplasmic eIF4E2', 'Series_geo_accession': 'GSE197915', 'Series_status': 'Public on Mar 07 2022', 'Series_submission_date': 'Mar 04 2022', 'Series_last_update_date': 'Oct 07 2023', 'Series_pubmed_id': '36840945', 'Series_web_link': 'https://linkinghub.elsevier.com/retrieve/pii/S2211-1247(23)00161-4', 'Series_summary': \"METTL16 was previously identified as an m6A methyltransferase. In this study, we validated the cytoplasmic location of METTL16 and explored its function in the cytoplasm. We found that METTL16 promoted translation by sequestering eIF4E2, a translation initiation repressor, from the 5' cap structure.\", 'Series_overall_design': 'Overall design: One sgRNA against METTL16 groups with two hybrid clones, one shRNAs against METTL16 groups, one shRNA against EIF4E2 groups with two bio-replicates, one sgNS group and two shNS group were subjected to ribosome profiling and both the input RNA and footprint RNAs were used for sequencing.', 'Series_type': 'Other', 'Series_contributor': 'Jun,,Zhang', 'Series_sample_id': 'GSM5932323', 'Series_contact_name': 'Zhang,,Jun', 'Series_contact_email': '3219030654@stu.cpu.edu.cn', 'Series_contact_phone': '15651718659', 'Series_contact_laboratory': 'zhoulab', 'Series_contact_institute': 'China Pharmaceutical University', 'Series_contact_address': 'longmiandadao', 'Series_contact_city': 'nanjing', 'Series_contact_state': 'jiangsu', 'Series_contact_zip/postal_code': '211198', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE197nnn/GSE197915/suppl/GSE197915_sheIF4E2-Ribo-RNA-tpm.xlsx', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA812878'}\n",
      "536 PRJNA816171 GSE198624 {'Series_title': 'Efficient detection of alternative spliced human proteome using translatome sequencing', 'Series_geo_accession': 'GSE198624', 'Series_status': 'Public on Jun 22 2022', 'Series_submission_date': 'Mar 14 2022', 'Series_last_update_date': 'Jun 23 2022', 'Series_pubmed_id': '35720116', 'Series_summary': 'Alternative splicing (AS) isoforms create numerous proteoforms, expanding the complexity of the genome. Highly similar sequences, incomplete reference databases and the insufficient sequence coverage of mass spectrometry limit the identification of AS proteoforms.In this work, we compared RNC-seq and Ribo-seq in the context of proteome identification, especially when identifying protein isoforms from AS. We also demonstrated that the single-molecule long read sequencing technique identified thousands of new splice variants and guided the MS identifications of new protein isoforms.', 'Series_overall_design': 'Getting translatome data of MHCC97H RNC and RPF.And we performed the direct full-length RNC-seq on MHCC97H using Nanopore MinION sequencer', 'Series_type': 'Other', 'Series_contributor': 'Yang,,Chen', 'Series_sample_id': 'GSM5953857', 'Series_contact_name': 'Gong,,Zhang', 'Series_contact_laboratory': 'Key Laboratory of Functional Protein Research of Guangdong Higher Education Institutes', 'Series_contact_institute': 'Jinan University', 'Series_contact_address': 'Jinan University, No.601, West Huangpu Avenue ，Guangzhou, Guangdong, China.', 'Series_contact_city': 'guangzhou', 'Series_contact_zip/postal_code': '510000', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE198nnn/GSE198624/suppl/GSE198624_RAW.tar', 'Series_platform_id': 'GPL24106', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA816171'}\n",
      "537 PRJNA818335 GSE199095 {'Series_title': 'Kaposi’s sarcoma-associated herpesvirus induces specialized ribosomes to efficiently translate viral lytic mRNAs', 'Series_geo_accession': 'GSE199095', 'Series_status': 'Public on Dec 14 2022', 'Series_submission_date': 'Mar 21 2022', 'Series_last_update_date': 'Feb 06 2023', 'Series_pubmed_id': '36653366', 'Series_summary': 'Historically, ribosomes have been viewed as unchanged homogeneous macromolecular machines with no intrinsic regulatory capacity for mRNA translation. However, an emerging concept is that heterogeneity of ribosomal composition exists, which can exert a regulatory function or specificity in translational control. This is supported by recent discoveries identifying compositionally distinct ‘specialized ribosomes’ that actively regulate mRNA translation. Viruses lack their own translational machinery and impose a high translational demand on the host cell during replication. Whilst viruses have evolved a variety of mechanisms to overcome this, we hypothesize that some viruses manipulate the host cell to produce specialized ribosomes to preferentially translate viral transcripts. Quantitative proteomic analysis has identified changes in the stoichiometry and composition of precursor ribosomal complexes during the switch from latent to lytic KSHV replication. Intriguingly, we demonstrate the enhanced association of ribosomal biogenesis factors BUD23 and NOC4L, and a previously uncharacterized KSHV lytic protein, ORF11, with small ribosomal subunit precursor complexes during lytic KSHV infection. Notably, BUD23 depletion resulted in significantly reduced viral gene expression and progression through the lytic cascade, culminating in a dramatic reduction of infectious virion production. Importantly, ribosome profiling demonstrated that BUD23 is essential for the reduced association of ribosomes with KSHV uORFs in late lytic genes, required for the efficient translation of the main open reading frame. Together our results provide new mechanistic insight into KSHV-mediated manipulation of cellular ribosome composition inducing a population of specialized ribosomes to facilitate efficient translation of viral mRNAs.', 'Series_overall_design': 'The final manuscript discusses KSHV data.', 'Series_type': 'Other', 'Series_contributor': 'Adrian,,Whitehouse', 'Series_sample_id': 'GSM5964231', 'Series_contact_name': 'James,,Murphy', 'Series_contact_email': 'bs12jcm@leeds.ac.uk', 'Series_contact_laboratory': 'Adrian Whitehouse', 'Series_contact_department': 'Faculty of Biological Sciences', 'Series_contact_institute': 'University of Leeds', 'Series_contact_address': 'Claredon Way', 'Series_contact_city': 'Leeds', 'Series_contact_state': 'West Yorkshire', 'Series_contact_zip/postal_code': 'LS2 9JT', 'Series_contact_country': 'United Kingdom', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE199nnn/GSE199095/suppl/GSE199095_kshv-RPFcounts-input4RiboRex.tsv.gz', 'Series_platform_id': 'GPL30065', 'Series_platform_taxid': '37296', 'Series_sample_taxid': '37296', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA818335'}\n",
      "538 PRJNA819859 GSE199387 {'Series_title': 'Ribosomal RNA 2’-O-methylation dynamics during development and differentiation impact cell fate decisions.', 'Series_geo_accession': 'GSE199387', 'Series_status': 'Public on Jun 15 2023', 'Series_submission_date': 'Mar 25 2022', 'Series_last_update_date': 'Sep 14 2023', 'Series_pubmed_id': '37473757', 'Series_web_link': 'https://www.sciencedirect.com/science/article/pii/S1534580723003052?via%3Dihub#app2', 'Series_summary': \"Variable rRNA 2'-O-me contributes to ribosome heterogeneity. We show that the rRNA 2'-O-me profile is dynamic in a tissue-specific manner during the directed differentiation of human embryonic stem cells (hESCs). 2'-O-me at position 26S:3904 transiently decreases at the neural progenitor (NPC) stage during neurogenesis. Knock-out of SNORD52, the guide of 28S:3904 2'-O-me, results in hESCs shifting to a NPC identity. We report that the lack of 28S:3904 methylation leads to an increased binding to the FMRP protein and translation of WNT pathway members.\", 'Series_overall_design': 'Ribosome profiling of H9 and H9SNORD52 knockout cells. H9 cells have rRNA 28S:Um3904 methylation whereas H9SNORD52 knockout cells do not have 28S:Um3904 methylation.', 'Series_type': 'Other', 'Series_contributor': 'Anders,H,Lund', 'Series_sample_id': 'GSM5971620', 'Series_contact_name': 'Anders,Henrik,Lund', 'Series_contact_email': 'anders.lund@bric.ku.dk', 'Series_contact_institute': 'University of Copenhagen,BRIC', 'Series_contact_address': 'Ole Maaløes Vej 5', 'Series_contact_city': 'Copenhagen', 'Series_contact_zip/postal_code': '2200', 'Series_contact_country': 'Denmark', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE199nnn/GSE199387/suppl/GSE199387_mean_nooffset_CPM_H9_WT_RP.bigWig', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA819859'}\n",
      "539 PRJNA822292 GSE199932 {'Series_title': 'A multi-omics integrative network map of maize.', 'Series_geo_accession': 'GSE199932', 'Series_status': 'Public on Oct 30 2022', 'Series_submission_date': 'Apr 01 2022', 'Series_last_update_date': 'Jun 03 2024', 'Series_pubmed_id': '36581701', 'Series_summary': 'These data include RNA-seq, circRNA-seq, and small RNA-seq of transcriptome, Ribo-seq of translatome and protein protein binary interactions by recombination-based library vs. library yeast-2-hybrid throughout the lifecycle of the maize inbred line B73.', 'Series_overall_design': 'We constructed a integrative network map involving co-expression, co-translation and interactome network.', 'Series_type': 'Other', 'Series_contributor': 'Jia,,Qian', 'Series_sample_id': 'GSM5994254', 'Series_contact_name': 'qian,lin,han', 'Series_contact_institute': 'Huazhong Agricultural University', 'Series_contact_address': 'Huazhong Agricultural University, No.1 Shizishan Street, Hongshan District', 'Series_contact_city': 'wuhan', 'Series_contact_state': 'hubei', 'Series_contact_zip/postal_code': '430070', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE199nnn/GSE199932/suppl/GSE199932_translatome.tpm.txt.gz', 'Series_platform_id': 'GPL29314', 'Series_platform_taxid': '4577', 'Series_sample_taxid': '4577', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA822292'}\n",
      "540 PRJNA822806 GSE200060 {'Series_title': 'A parasitic fungus employs mutated eIF4A to survive on rocaglate-synthesizing Aglaia plants', 'Series_geo_accession': 'GSE200060', 'Series_status': 'Public on Feb 20 2023', 'Series_submission_date': 'Apr 03 2022', 'Series_last_update_date': 'May 22 2023', 'Series_pubmed_id': '36852480', 'Series_summary': 'Plants often generate secondary metabolites with antifungal properties as defense mechanisms against parasites. Although some fungi may potentially overcome the barrier of antimicrobial compounds, only a limited number of examples and molecular mechanisms of resistance have been reported. Here, we found an Aglaia plant-parasitizing fungus that overcomes the toxicity of rocalgates, which are translation inhibitors synthesized by the plant, through an amino acid substitution in a translation initiation factor (eIF). De novo transcriptome assembly of the fungus revealed that eIF4A, a molecular target of rocaglates, replaces a critical amino acid in the rocaglate binding site. Moreover, genome-wide ribosome profiling harnessing a cucumber-infecting fungus, Colletotrichum orbiculare, demonstrated that the translational inhibitory effects of rocaglates were largely attenuated by the mutation found in the Aglaia parasite. The engineered Colletotrichum orbiculare showed a survival advantage on cucumber plants with rocaglates. Our study exemplifies a plant-fungus tug-of-war centered on secondary metabolites produced by host plants.', 'Series_overall_design': 'Ribosome profiling and RNA-Seq of Colletotrichum orbiculare treated with aglafoline', 'Series_type': 'Other', 'Series_contributor': 'Shintaro,,Iwasaki', 'Series_sample_id': 'GSM6858442', 'Series_contact_name': 'Shintaro,,Iwasaki', 'Series_contact_email': 'shintaro.iwasaki@riken.jp', 'Series_contact_laboratory': 'RNA Systems Biochemistry Laboratory', 'Series_contact_department': 'Center for Pioneering Research', 'Series_contact_institute': 'RIKEN', 'Series_contact_address': '2-1, Hirosawa', 'Series_contact_city': 'Wako', 'Series_contact_state': 'Saitama', 'Series_contact_zip/postal_code': '3510198', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE200nnn/GSE200060/suppl/GSE200060_RAW.tar', 'Series_platform_id': 'GPL32115', 'Series_platform_taxid': '5465', 'Series_sample_taxid': '5465', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA822806'}\n",
      "541 PRJNA822924 GSE200091 {'Series_title': 'Application of Translation Complex Profile sequencing (TCP-seq) to track the course of translational reprogramming in the exponentially growing culture of budding yeast (Saccharomyces cerevisiae, BY4741) subjected to glucose starvation for 10 minutes.', 'Series_geo_accession': 'GSE200091', 'Series_status': 'Public on Mar 31 2024', 'Series_submission_date': 'Apr 04 2022', 'Series_last_update_date': 'Aug 16 2024', 'Series_pubmed_id': '38721779', 'Series_summary': 'Conclusions: The stochastic disome signal, together with the other measurements sourced from TCP-seq and calculated as Stochastic Translation Efficiency (STE) measure, provide a link to the absolute translation initiation and protein biosynthesis rates. Using STE, it is possible to rank mRNAs by the absolute protein output and thus, characterise the ‘power’ of translation control elements across transcripts in a single setting or between different conditions. STE does not use bias-inducing normalisation to the RNA abundance or signals of different types and relies on self-normalised signal pairs. STE thus evades imprecisions arising from the different accessibility across mRNAs to translation machinery (such as in nucleus, phase-separated foci or condensates, etc.) and coincidental transcription- and stability-driven mRNA abundance alterations, and provides a robust measure suitable for investigating protein biosynthesis during rapidly changing transcriptome background, as occurs in cell stress response and reprogramming. With STE, we demonstrate that the prototypical example of translational control during yeast response to glucose starvation is more complex than previously thought and includes translational acceleration as well as re-adjustment of the RNA metabolism to suit new translational demands. More details can be uncovered in the future, as STE application is permissive to finer dynamics dissection and elucidation of very rapid cell responses at the level of RNA translation.', 'Series_overall_design': 'budding yeast TCP-seq libraries from naive and starved cells', 'Series_type': 'Other', 'Series_contributor': 'Nikolay,,Shirokikh', 'Series_sample_id': 'GSM6008520', 'Series_contact_name': 'Attila,,Horvath', 'Series_contact_email': 'attila.horvath@anu.edu.au', 'Series_contact_institute': 'The Australian National University', 'Series_contact_address': 'Garran Road 131', 'Series_contact_city': 'Canberra', 'Series_contact_state': 'ACT', 'Series_contact_zip/postal_code': '2601', 'Series_contact_country': 'Australia', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE200nnn/GSE200091/suppl/GSE200091_S10m_SSU_m.bed.gz', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA822924'}\n",
      "542 PRJNA822939 GSE200097 {'Series_title': 'Ribosome profiling and RNA-seq of human corneal epithelial cells upon mild osmotic stress', 'Series_geo_accession': 'GSE200097', 'Series_status': 'Public on Jul 07 2022', 'Series_submission_date': 'Apr 04 2022', 'Series_last_update_date': 'Oct 04 2022', 'Series_pubmed_id': '35858571', 'Series_summary': 'We present ribosome profiling(for activaly translated RNA) and RNA-seq (for total RNA) data for human corneal epithelial cells exposed to mild osmotic stress (500 mOsm) for 1h and 6h (additionally torin1 (mTOR inhibitor) or MeAIB (SNAT2 inhibitor were added for the last hour of treatment.', 'Series_overall_design': 'Ribosomal profiling and RNA-seq of human corneal epithelial cells. There are 4 replicates for each of the following conditions: control, 1h exposure to 500 mOsm, 6h exposure to 500 mOsm, 6h exposure to 500 mOsm + Torin1 for the last hour of treatment, 6h exposure to 500 mOsm + MeAIB for the last hour of treatment', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'M,,Hatzoglou', 'Series_sample_id': 'GSM6008635', 'Series_contact_name': 'Krzysztof,Jan,Szkop', 'Series_contact_email': 'krzysztof.szkop@ki.se', 'Series_contact_department': 'Department of Oncology and Pathology', 'Series_contact_institute': 'Karolinska Institutet', 'Series_contact_address': 'SciLifeLab, Tomtebodavägen 23', 'Series_contact_city': 'Stockholm', 'Series_contact_state': 'Solna', 'Series_contact_zip/postal_code': '171 65', 'Series_contact_country': 'Sweden', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE200nnn/GSE200097/suppl/GSE200097_raw_counts.txt.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA822939'}\n",
      "543 PRJNA824815 GSE200491 {'Series_title': 'Ribosome profiling and RNA-seq of an acute glucose starvation timecourse and 5 day growth course in S. cerevisiae', 'Series_geo_accession': 'GSE200491', 'Series_status': 'Public on Apr 11 2022', 'Series_submission_date': 'Apr 08 2022', 'Series_last_update_date': 'May 11 2022', 'Series_pubmed_id': '35491906', 'Series_summary': 'Ribosome profiling provides an opportunity to not only assess how the relative abundance of ribosome association with mRNAs changes in different conditions, but to look more closely at where along mRNAs those ribosomes bind. Here, we used ribosome profiling to calculate the ribosome polarity scores and changes in ribosome footprint read density in both aggregate and gene-specific ways. We profiled a time course of acute glucose starvation followed by glucose readdition and a multi-day growth course through the diauxic shift into stationary phase. We found that ribosome polarity became positive in postdiauxic shift conditions relative to log phase. In acute starvation, polarity shifted positive at our earliest time points but did not continue to do so at later time points. This is consistent with a read density analysis which demonstrated increased density on the 3’ half of genes after glucose starvation. Additionally, we performed ribosome profiling in samples that had glucose added back following acute starvation and observed a wave of new ribosome movement near the start codon and approximately 2,000 nucleotides downstream on open reading frames after one and five minutes of readdition, respectively. Our ribosome profiling analysis suggested that elongation slows during starvation which leads to a buildup of ribosomes on the 3’ halves of mRNAs. Further, it also indicated ribosomes previously built up can resume translation upon glucose readdition. We used reporter assays to corroborate these findings in vivo. Together, these results demonstrate how yeast regulate translation in response to glucose starvation.', 'Series_overall_design': 'mRNA and ribosome occupancy profiles of S. cerevisiae in actue glucose starvation, glucose readdition, and mutli-day growth conditions', 'Series_type': 'Other', 'Series_contributor': 'Molly,H,Cohn', 'Series_sample_id': 'GSM6035079', 'Series_contact_name': 'Brian,Matthew,Zid', 'Series_contact_email': 'zid@ucsd.edu', 'Series_contact_laboratory': 'Zid', 'Series_contact_department': 'Chemistry and Biochemistry', 'Series_contact_institute': 'University of California San Diego', 'Series_contact_address': '9500 Gilman Drive', 'Series_contact_city': 'La Jolla', 'Series_contact_state': 'California', 'Series_contact_zip/postal_code': '92093', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE200nnn/GSE200491/suppl/GSE200491_Raw_counts_and_polarity_per_gene.xlsx', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA824815'}\n",
      "544 PRJNA831863 GSE201484 {'Series_title': 'LncReader: identification of dual functional long non-coding RNAs using multi-head self-attention mechanism [Ribo-seq]', 'Series_geo_accession': 'GSE201484', 'Series_status': 'Public on Apr 26 2022', 'Series_submission_date': 'Apr 25 2022', 'Series_last_update_date': 'Dec 28 2022', 'Series_pubmed_id': '36575567', 'Series_summary': 'The dual functional lncRNAs have been intensively studied and identified to be involved in various fundamental cellular processes recently. It is essential to understand in which context when a dual functional lncRNA serves as a non-coding RNA or a template for coding peptide, particularly in some pathological conditions. However, apart from time consuming and cell type specific experiments, there is virtually no in-silico method for predicting the identity of dual functional lncRNAs. Here, we developed a deep-learning model with multi-head self-attention mechanism, LncReader, to identify dual functional lncRNAs based on their sequence, physicochemical and secondary structural features. Our data demonstrated that LncReader showed multiple advantage compared to various classical machine learning methods. Moreover, to obtain independent in-house datasets for robust testing, mass spectrometry proteomics combined with RNA-seq were applied in four leukemia cell lines. Remarkably, LncReader achieved the best performance among all these datasets. Therefore, LncReader provides a sophisticated and practical tool that enables fast dual functional lncRNAs identification.', 'Series_overall_design': 'Ribosome-seq of four leukemia cell lines (HL60, MOLM13, K562, THP_1), one replicate of each cell line.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Bohao,,Zou', 'Series_sample_id': 'GSM6065335', 'Series_contact_name': 'Tianyuan,,Liu', 'Series_contact_email': 'biocoder1024@hotmail.com', 'Series_contact_department': 'Bioinformatics Department', 'Series_contact_institute': 'Sourthern Medical University', 'Series_contact_address': 'Shatai South Road, Baiyun District', 'Series_contact_city': 'Guangzhou', 'Series_contact_zip/postal_code': '510515', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE201nnn/GSE201484/suppl/GSE201484_RAW.tar', 'Series_platform_id': 'GPL20795', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA831863'}\n",
      "545 PRJNA832792 GSE201732 {'Series_title': 'Circadian ribosome profiling reveals a role for the Period2 upstream opening reading frame in sleep', 'Series_geo_accession': 'GSE201732', 'Series_status': 'Public on May 05 2023', 'Series_submission_date': 'Apr 27 2022', 'Series_last_update_date': 'Dec 13 2023', 'Series_pubmed_id': '37769257', 'Series_web_link': 'https://www.pnas.org/doi/10.1073/pnas.2214636120', 'Series_summary': \"Many mammalian proteins have circadian cycles of production and degradation, but de novo transcription is only responsible for a small fraction of this rhythmicity. We used ribosome profiling to quantify RNA translation in liver from circadian-entrained mice transferred to constant darkness conditions over a 24-h period and compared translation levels to absolute protein production for 16 circadian proteins. We observed a delay between translation and peak protein levels for several circadian genes covering a wide range of translation efficiencies. We found extensive binding of ribosomes to upstream open reading frames (uORFs) in circadian mRNAs, including the core clock gene Period2 (Per2). Increased uORFs in 5' UTRs was associated with decreased ribosome binding in downstream ORFs and reduced expression of synthetic reporter constructs in vitro and in single cells. Mutation of the Per2 uORF increased luciferase and fluorescence reporter expression in 3T3 cells without altering phase or period. Genomic Per2 uORF mutation using CRISPR/Cas9 increased PER2 expression in PER2:Luc MEFs and reduced sleep in Per2 uORF mutant mice. These results suggest that post-transcriptional processes shape translation of mRNA transcripts, which can impact physiological rhythms and sleep.\", 'Series_overall_design': 'Circadian ribosome profiling of two C57BL/6N mouse livers from each timepoint CT00, CT04, CT08, CT12, CT16, CT20, and CT24', 'Series_type': 'Other', 'Series_contributor': 'Hiroki,R,Ueda', 'Series_sample_id': 'GSM6070367', 'Series_contact_name': 'Arthur,,Millius', 'Series_contact_email': 'arthur.millius@ifrec.osaka-u.ac.jp', 'Series_contact_phone': '+81-06-6879-8366', 'Series_contact_laboratory': 'Systems Immunology', 'Series_contact_department': 'iFReC', 'Series_contact_institute': 'Osaka University', 'Series_contact_address': 'iFReC 6F', 'Series_contact_city': 'Suita', 'Series_contact_state': 'Osaka', 'Series_contact_zip/postal_code': '565-0871', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE201nnn/GSE201732/suppl/GSE201732_Supplementary_Table_2_JTK_RiboProf_v4metaAlloc_.xlsx', 'Series_platform_id': 'GPL18480', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA832792'}\n",
      "546 PRJNA833301 GSE201845 {'Series_title': 'Subfunctionalized expression drives evolutionary retention of ribosomal protein paralogs Rps27 and Rps27l in vertebrates', 'Series_geo_accession': 'GSE201845', 'Series_status': 'Public on Jun 21 2023', 'Series_submission_date': 'Apr 29 2022', 'Series_last_update_date': 'Sep 11 2023', 'Series_pubmed_id': '37306301', 'Series_web_link': 'https://elifesciences.org/articles/78695', 'Series_summary': 'The formation of paralogs through gene duplication is a core evolutionary process. For paralogs that encode components of protein complexes such as the ribosome, a central question is whether they encode functionally distinct proteins, or whether they exist to maintain appropriate total expression of equivalent proteins. Here, we systematically tested evolutionary models of paralog function using the ribosomal protein paralogs Rps27 (eS27) and Rps27l (eS27L) as a case study. Evolutionary analysis suggests that Rps27 and Rps27l likely arose during whole-genome duplication(s) in a common vertebrate ancestor. We show that Rps27 and Rps27l have inversely correlated mRNA abundance across mouse cell types, with the highest Rps27 in lymphocytes and the highest Rps27l in mammary alveolar cells and hepatocytes. By endogenously tagging the Rps27 and Rps27l proteins, we demonstrate that Rps27- and Rps27l-ribosomes associate preferentially with different transcripts. Furthermore, murine Rps27 and Rps27l loss-of-function alleles are homozygous lethal at different developmental stages. However, strikingly, expressing Rps27 protein from the endogenous Rps27l locus or vice versa completely rescues loss-of-function lethality and yields mice with no detectable deficits. Together, these findings suggest that Rps27 and Rps27l are evolutionarily retained because their subfunctionalized expression patterns render both genes necessary to achieve the requisite total expression of two equivalent proteins across cell types. Our work represents the most in-depth characterization of a mammalian ribosomal protein paralog to date and highlights the importance of considering both protein function and expression when investigating paralogs.', 'Series_overall_design': 'Ribosome profiling of the total ribosome fraction and FLAG-immunoprecipitated (IP) ribosome fraction from Rps27-FLAG, Rps27l-FLAG, and untagged wild-type (WT) control mouse embryonic stem cells (mESCs). IP ribosome fractions from Rps27-FLAG and Rps27l-FLAG mESCs represent Rps27- and Rps27l-containing ribosomes, respectively. All samples were collected in triplicate.', 'Series_type': 'Other', 'Series_contributor': 'Adele,F,Xu', 'Series_sample_id': 'GSM6077078', 'Series_contact_name': 'Maria,,Barna', 'Series_contact_email': 'mbarna@stanford.edu', 'Series_contact_department': 'Genetics', 'Series_contact_institute': 'Stanford University', 'Series_contact_address': '240 Pasteur Dr, BMI 4500', 'Series_contact_city': 'Palo Alto', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94304', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE201nnn/GSE201845/suppl/GSE201845_reads_cds_v_samp.txt.gz', 'Series_platform_id': 'GPL21103', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA833301'}\n",
      "547 PRJNA837585 GSE202881 {'Series_title': 'Pyruvate Kinase M (PKM) binds ribosomes in a poly-ADP ribosylation dependent manner to induce translational stalling', 'Series_geo_accession': 'GSE202881', 'Series_status': 'Public on May 16 2022', 'Series_submission_date': 'May 12 2022', 'Series_last_update_date': 'May 16 2022', 'Series_summary': 'In light of the numerous studies identifying post-transcriptional regulators on the surface of the endoplasmic reticulum (ER), we ask whether there are factors that regulate compartment specific mRNA translation in human cells. Using a proteomic survey of spatially regulated polysome interacting proteins, we identified the glycolytic enzyme Pyruvate Kinase M (PKM) as a cytosolic (i.e., ER-excluded) polyribosome interactor and investigate how it influences mRNA translation. We discovered that the PKM-polysome interaction is directly regulated by ADP levels–providing a link between carbohydrate metabolism and mRNA translation. By performing enhanced crosslinking immunoprecipitation-sequencing (eCLIP-seq), we found that PKM crosslinks to mRNA sequences that are immediately downstream of regions that encode for lysine and glutamate enriched tracts. Additionally, PKM binding to ribosomes causes translational stalling near these lysine and glutamate encoding sequences. Lastly, we find that PKM recruitment to polysomes is dependent on poly-ADP ribosylation. Overall, our study uncovers a novel role for PKM in posttranscriptional gene regulation, linking cellular metabolism and mRNA translation, and provides the first reported evidence of nascent chains being co-translationally modified with poly-ADP ribose.', 'Series_overall_design': 'Ribosome profiling analysis of HEK293T depleted of PKM by RNAi', 'Series_type': 'Other', 'Series_contributor': 'Alexander,F,Palazzo', 'Series_sample_id': 'GSM6135058', 'Series_contact_name': 'Alexander,F,Palazzo', 'Series_contact_phone': '(416) 978-7234', 'Series_contact_laboratory': 'Palazzo Lab', 'Series_contact_department': 'Biochemistry', 'Series_contact_institute': 'University of Toronto', 'Series_contact_address': '661 University Ave.', 'Series_contact_city': 'Toronto', 'Series_contact_state': 'ON', 'Series_contact_zip/postal_code': 'M5G 1M1', 'Series_contact_country': 'Canada', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE202nnn/GSE202881/suppl/GSE202881_all_processed_data.csv.gz', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA837585'}\n",
      "548 PRJNA837608 GSE202900 {'Series_title': 'Synonymous codon usage regulates translation initiation', 'Series_geo_accession': 'GSE202900', 'Series_status': 'Public on May 16 2022', 'Series_submission_date': 'May 12 2022', 'Series_last_update_date': 'Jan 29 2024', 'Series_pubmed_id': '38096059', 'Series_summary': 'Ribo-seq with firefly luciferase optimality reporters to determine whether incomplete translation and/or inhibited translation initiation is responsible for the reduced ribosome occupancy on nonoptimal transcripts.', 'Series_overall_design': 'Performed Ribo-seq, with paired RNA-seq, on HEK293T cells transfected with either optimal or nonoptimal reporters. Experiments were performed in triplicate.', 'Series_type': 'Other', 'Series_contributor': 'Olivia,S,Rissland', 'Series_sample_id': 'GSM6135264', 'Series_contact_name': 'Olivia,,Rissland', 'Series_contact_email': 'olivia.rissland@cuanschutz.edu', 'Series_contact_laboratory': 'Rissland lab', 'Series_contact_department': 'BMG', 'Series_contact_institute': 'University Of CO Anschutz Medical Campus', 'Series_contact_address': '12801 East 17th Ave', 'Series_contact_city': 'Aurora', 'Series_contact_state': 'CO', 'Series_contact_zip/postal_code': '80045', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE202nnn/GSE202900/suppl/GSE202900_reporters.fa.gz', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA837608'}\n",
      "549 PRJNA838490 GSE203078 {'Series_title': 'Identification of DDX3X-dependent translation targets in the E11.5 mouse cortex', 'Series_geo_accession': 'GSE203078', 'Series_status': 'Public on May 18 2022', 'Series_submission_date': 'May 16 2022', 'Series_last_update_date': 'Mar 13 2024', 'Series_pubmed_id': '38273160', 'Series_summary': 'To identify DDX3X-dependent translation targets in the developing mouse cortex, we used E11.5 cortices from both sexes of wildtype and conditional Ddx3x knockout mice (driven by Emx1-Cre). We performed RNAseq and Riboseq on these samples in parallel.', 'Series_overall_design': 'Comparative gene expression profiling analysis of RNA-seq and Ribo-seq data for wildtype and Ddx3x conditional knockout mouse E11.5 cortex. The following are biological replicates: Female controls: F2, F3; Male controls: M1, M2, M3; cKO females: F1, F2, F3; cKO mals: M1, M2, M3', 'Series_type': 'Other', 'Series_contributor': 'Debra,,Silver', 'Series_sample_id': 'GSM6153513', 'Series_contact_name': 'Debra,,Silver', 'Series_contact_email': 'debra.silver@duke.edu', 'Series_contact_phone': '919-668-7909', 'Series_contact_department': 'Molecular Genetics and Microbiology', 'Series_contact_institute': 'Duke University School of Medicine', 'Series_contact_address': '224 CARL Building', 'Series_contact_city': 'Durham', 'Series_contact_state': 'North Carolina', 'Series_contact_zip/postal_code': '27710', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE203nnn/GSE203078/suppl/GSE203078_Supplemental_File_3.xlsx', 'Series_platform_id': 'GPL24247', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA838490'}\n",
      "550 PRJNA838843 GSE203147 {'Series_title': 'Multifarious translational regulation during replicative aging in yeast', 'Series_geo_accession': 'GSE203147', 'Series_status': 'Public on Sep 25 2022', 'Series_submission_date': 'May 17 2022', 'Series_last_update_date': 'Dec 25 2022', 'Series_pubmed_id': '36135663', 'Series_summary': 'Protein synthesis is strictly regulated during replicative aging in yeast, but global translational regulation during replicative aging is poorly characterized. To conduct ribosomal profiling during replicative aging, we collected a large number of dividing aged cells using a miniature chemostat aging device. Translational efficiency, defined as the number of ribosome footprints normalized to transcript abundance, was compared between young and aged cells for each gene. We identified more than 700 genes with changes greater than twofold during replicative aging. Increased translational efficiency was observed in genes involved in DNA repair and chromosome organization. Decreased translational efficiency was observed in genes encoding ribosome components, transposon Ty1 and Ty2 genes, transcription factor HAC1 genes associated with the unfolded protein response, genes involved in cell wall synthesis and assembly, and ammonium permease genes. Our results provide a global view of translational regulation during replicative aging, in which the pathways involved in various cell functions are translationally regulated and cause diverse phenotypic changes.', 'Series_overall_design': 'Ribosome profiling and RNA-Seq of Saccharomyces cerevisiae cells in the replicative aging using the MAD system', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Yoshikazu,,Ohya', 'Series_sample_id': 'GSM6160433', 'Series_contact_name': 'Shintaro,,Iwasaki', 'Series_contact_email': 'shintaro.iwasaki@riken.jp', 'Series_contact_laboratory': 'RNA Systems Biochemistry Laboratory', 'Series_contact_department': 'Center for Pioneering Research', 'Series_contact_institute': 'RIKEN', 'Series_contact_address': '2-1, Hirosawa', 'Series_contact_city': 'Wako', 'Series_contact_state': 'Saitama', 'Series_contact_zip/postal_code': '3510198', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE203nnn/GSE203147/suppl/GSE203147_RAW.tar', 'Series_platform_id': 'GPL26171', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA838843'}\n",
      "551 PRJNA847982 GSE205873 {'Series_title': 'Study the  global mRNA translation changes between SKBR cells overexpressing VIRMA and eGFP control', 'Series_geo_accession': 'GSE205873', 'Series_status': 'Public on Aug 03 2023', 'Series_submission_date': 'Jun 10 2022', 'Series_last_update_date': 'Aug 03 2023', 'Series_pubmed_id': '37208522', 'Series_summary': 'We then performed ribosome profiling analysis between SKBR cells overexpressing VIRMA FL and eGFP control', 'Series_overall_design': 'Comparative translation analysis of Poly-ribo-seq data for VIRMA-overexpressing SKBR3 cells and eGFP control', 'Series_type': 'Other', 'Series_contributor': 'Renhua,,Song', 'Series_sample_id': 'GSM6234767', 'Series_contact_name': 'RENHUA,,SONG', 'Series_contact_email': 'Renhua.song1989@gmail.com', 'Series_contact_phone': '61452667663', 'Series_contact_laboratory': 'Epigenetics and RNA Biology Program', 'Series_contact_institute': 'Centenary Institute', 'Series_contact_address': 'Charles Perkins Centre - D17, Level 4 West. The University of Sydney Johns Hopkins Drive', 'Series_contact_city': 'Camperdown', 'Series_contact_state': 'NSW', 'Series_contact_zip/postal_code': '2050', 'Series_contact_country': 'Australia', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE205nnn/GSE205873/suppl/GSE205873_Riboseq_SKBR3_counts.txt.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA847982'}\n",
      "552 PRJNA848829 GSE206040 {'Series_title': 'Developing High-Affinity Decoy Receptor Optimized for Multiple Myeloma and Diffuse Large B Cell Lymphoma Treatment [Ribo-seq]', 'Series_geo_accession': 'GSE206040', 'Series_status': 'Public on Jun 18 2022', 'Series_submission_date': 'Jun 13 2022', 'Series_last_update_date': 'Oct 26 2022', 'Series_pubmed_id': '35881112', 'Series_summary': 'This study demonstrates the dichotomous activities of APRIL and BAFF in MM and DLBCL, which can be safely targeted by an engineered fusion protein designed to trap both ligands with ultrahigh binding affinity.', 'Series_overall_design': 'Comparative ribosome and gene expression profiling analysis of RNA-seq data for U266 multiple myeloma cells and its knockdown derivatives (siBCMA).', 'Series_type': 'Other', 'Series_contributor': 'Amato,J,Giaccia', 'Series_sample_id': 'GSM6239423', 'Series_contact_name': 'Dadi,,Jiang', 'Series_contact_department': 'Radiation Oncology', 'Series_contact_institute': 'The University of Texas MD Anderson Cancer Center', 'Series_contact_address': '6565 MD Anderson Blvd, Unit 1052', 'Series_contact_city': 'Houston', 'Series_contact_state': 'TX', 'Series_contact_zip/postal_code': '77030', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE206nnn/GSE206040/suppl/GSE206040_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA848829'}\n",
      "553 PRJNA850795 GSE206437 {'Series_title': 'DUX4 orchestrates translational reprograming by broadly suppressing translation efficiency [Ribo-seq]', 'Series_geo_accession': 'GSE206437', 'Series_status': 'Public on Feb 01 2023', 'Series_submission_date': 'Jun 19 2022', 'Series_last_update_date': 'Feb 01 2023', 'Series_summary': 'DUX4, an embryonic transcription factor normally silenced in somatic tissues, was recently shown to reduce major histocompatibility complex I (MHC-I) and promote immune evasion in numerous cancer types. We report that transient expression of DUX4 results in the long-term suppression of interferon-gamma induction of MHC-I and the immunoproteasome. Brief expression of transcriptionally active DUX4 leads to a prolonged decrease in protein synthesis, altering the activity of translation initiation regulators eIF2a, eIF4E, 4EBP1, and translation elongation factor eEF2. Translational profiling identified mRNAs susceptible to DUX4-induced inhibition of translation initiation and elongation, including mRNAs encoding antigen presentation factors. We show that DUX4 orchestrates translational reprogramming by broadly attenuating protein synthesis while inducing the expression of RNAs encoding early embryonic proteins that are translationally up regulated and predicted to be less vulnerable to translational inhibition. Endogenous expression of DUX4 and its target genes in FSHD muscle cells and SuSa germinoma cells also correlates with suppression of protein synthesis and reduced MHC-I presentation. Our findings reveal a novel role of DUX4 in modulating translational control to reshape the cellular translatome with important implications for development and disease.', 'Series_overall_design': 'Comparative ribosome foortprinting RNA-seq analysis for MB135iDUX4 myoblasts: untreated, treated with IFNg, with a DUX4 pulse, or with a DUX4 pulse+IFNg and harvested at 68h.', 'Series_type': 'Other', 'Series_contributor': 'Chao-Jen,,Wong', 'Series_sample_id': 'GSM6253658', 'Series_contact_name': 'Stephen,,Tapscott', 'Series_contact_email': 'stapscot@fredhutch.org', 'Series_contact_laboratory': 'Tapscott', 'Series_contact_department': 'Human Biology', 'Series_contact_institute': 'Fred Hutch Cancer Research Center', 'Series_contact_address': '1100 Fairview N. Ave', 'Series_contact_city': 'Seattle', 'Series_contact_state': 'WASHINGTON', 'Series_contact_zip/postal_code': '98103', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE206nnn/GSE206437/suppl/GSE206437_rse_cds_by_gene.rda.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA850795'}\n",
      "554 PRJNA852784 GSE206931 {'Series_title': 'Multiomic characterization of WDR5 WIN site inhibition reveals actionable synergies for MLL-rearranged leukemia', 'Series_geo_accession': 'GSE206931', 'Series_status': 'Public on Jul 03 2023', 'Series_submission_date': 'Jun 25 2022', 'Series_last_update_date': 'May 24 2024', 'Series_pubmed_id': '38682900', 'Series_summary': 'In this study, we utilized a multi-omics approach to determine the transcriptional, translational, and proteomic responses to WIN Site inhibition in MLL-rearranged leukemia cells via RNA-Seq, Ribo-Seq, and quantitative proteomics, respectively. We also performed both a whole-genome targeting primary CRISPR screen and a secondary CRISPR screen targeting a currated collection of genes to determine genes important for sensitivity to WIN Site inhibition. These combined data sets rationally guided us in assembling a collection of compounds that, when combined with WIN Site inhibitor, synergistically inhibit growth of MLL-rearranged leukemia cells.', 'Series_overall_design': 'MV4;11 cells were treated with either DMSO or one of two unique WIN Site inhibitors before performing RNA-Seq to determine transcriptome alterations and Ribo-Seq to determine translatome alterations. CRISPR screens were performed in MV4;11 cells to determine genes altering sensitivity to multiple WIN Site inhibitors.', 'Series_type': 'Other', 'Series_contributor': 'Qi,,Liu', 'Series_sample_id': 'GSM7123577', 'Series_contact_name': 'Jing,,Wang', 'Series_contact_institute': 'Vanderbilt University Medical Center', 'Series_contact_address': '2220 Pierce Avenue', 'Series_contact_city': 'Nashville', 'Series_contact_state': 'TN', 'Series_contact_zip/postal_code': '37232', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE206nnn/GSE206931/suppl/GSE206931_RPF.count.txt.gz', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA852784'}\n",
      "555 PRJNA854905 GSE207347 {'Series_title': 'Transcriptional and Translational Dynamics of Zika and Dengue Virus Infection', 'Series_geo_accession': 'GSE207347', 'Series_status': 'Public on Jul 06 2022', 'Series_submission_date': 'Jul 01 2022', 'Series_last_update_date': 'Aug 03 2022', 'Series_pubmed_id': '35891396', 'Series_summary': 'Zika virus (ZIKV) and dengue virus (DENV) are members of the Flaviviridae family of RNA viruses and cause severe disease in humans. ZIKV and DENV share over 90% of their genome sequences, however the clinical features of Zika and dengue infections are very different reflecting tropism and cellular effects. Here, we used simultaneous RNA sequencing and ribosome footprinting to define the transcriptional and translational dynamics of ZIKV and DENV infection in human neuronal progenitor cells (hNPCs). The gene expression data showed induction of aminoacyl tRNA synthetases (ARS) and the translation-activating PIM1 kinase indicating an increase in RNA translation capacity. The data also reveal activation of different cell stress reponses, with ZIKV triggering a BACH1/2 redox program, and DENV activating the ETF/CHOP endoplasmatic reticulum (ER) stress program. The RNA translation data highlight activation of polyamine metabolism through changes in key enzymes and their regulators. This pathway is needed for eIF5A hypusination and has been implicated viral translation and replication. Concerning the viral RNA genomes, ribosome occupancy readily identifies highly translated open reading frames and a novel upstream ORF (uORF) in the DENV genome. Together, our data highlight both the cellular stress response and also the activation of RNA translation and polyamine metabolism during DENV and ZIKV infection.', 'Series_overall_design': 'We performed ribosome footprinting and RNA sequencing on human neuronal progenitor cells (hNPCs) cells either uninfected or infected with ZIKV or DENV  in two biological replicates that were collected 72 hours post infection. Unifected samples are indicated as A1 and B1, ZIKV infected samples are indicated as A2, B2 and DENV infected samples are indicated as A3 and B3.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Hans-Guido,,Wendel', 'Series_sample_id': 'GSM6284795', 'Series_contact_name': 'Kamini,,Singh', 'Series_contact_email': 'kamini.singh@einsteinmed.edu', 'Series_contact_phone': '7184302466', 'Series_contact_laboratory': 'Kamini Singh', 'Series_contact_department': 'Molecular Pharmacology', 'Series_contact_institute': 'Albert Einstein College of Medicine', 'Series_contact_address': '1300 Morris Park Ave, Golding 203', 'Series_contact_city': 'Bronx', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '10461', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE207nnn/GSE207347/suppl/GSE207347_ZIKV_DESeq2_result_name.txt.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA854905'}\n",
      "556 PRJNA858047 GSE208041 {'Series_title': 'Ribosome profiling (Ribo-Seq)  and mRNA sequencing (RNA-Seq) of Human leukemia monocytic cell line (THP-1) cells upon treatment with LPS and LPS+dexamethasone (Dex).', 'Series_geo_accession': 'GSE208041', 'Series_status': 'Public on Oct 06 2022', 'Series_submission_date': 'Jul 12 2022', 'Series_last_update_date': 'Nov 02 2022', 'Series_pubmed_id': '36284713', 'Series_summary': 'We then performed combined analysis of ribosome footprints (RPF) and mRNA abundance using the data obtained by Ribo-Seq and RNA-seq in THP-1 cells.', 'Series_overall_design': 'Integrative analysis of Ribo-Seq and RNA-Seq data from LPS and LPS+Dex treated THP-1 cells.', 'Series_type': 'Other', 'Series_contributor': 'Nina,H,Uhlenhaut', 'Series_sample_id': 'GSM6505556', 'Series_contact_name': 'Suhail,A.,Ansari', 'Series_contact_email': 'suhail.ansari@helmholtz-muenchen.de', 'Series_contact_laboratory': 'AG Uhlenhaut', 'Series_contact_department': 'Institute for Diabetes and Endocrinology (IDE)', 'Series_contact_institute': 'Helmholtz Zentrum München', 'Series_contact_address': 'Ingolstaedter Landstr.1', 'Series_contact_city': 'Neuherberg', 'Series_contact_zip/postal_code': '85764', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE208nnn/GSE208041/suppl/GSE208041_Thp1_raw_rna_counts.txt.gz', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA858047'}\n",
      "557 PRJNA858256 GSE208086 {'Series_title': 'Revealing the small proteome of Haloferax volcanii by ribosome profiling and small-protein optimised mass spectrometry', 'Series_geo_accession': 'GSE208086', 'Series_status': 'Public on Jan 18 2023', 'Series_submission_date': 'Jul 13 2022', 'Series_last_update_date': 'Aug 25 2023', 'Series_pubmed_id': '37223747', 'Series_summary': 'We present a combinatorial approach, integrating experimental data from small protein-optimized mass spectrometry (MS) and ribosome profiling (Ribo-seq), to generate a high confidence inventory of small proteins in the model archaeon Haloferax volcanii.', 'Series_overall_design': 'Ribo-seq and RNA-seq experiments to study the transcriptome and the translatome of H. volcanii', 'Series_type': 'Other', 'Series_contributor': 'Anita,,Marchfelder', 'Series_sample_id': 'GSM6336921', 'Series_contact_name': 'Rick,,Gelhausen', 'Series_contact_email': 'gelhausr@informatik.uni-freiburg.de', 'Series_contact_laboratory': 'Bioinformatics Group (AG Backofen)', 'Series_contact_department': 'Department of Computer Science', 'Series_contact_institute': 'Albert-Ludwigs-University Freiburg', 'Series_contact_address': 'Georges-Köhler-Allee 106', 'Series_contact_city': 'Freiburg', 'Series_contact_state': 'Baden Württemberg', 'Series_contact_zip/postal_code': '79110', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE208nnn/GSE208086/suppl/GSE208086_overview.xlsx', 'Series_citation': 'https://doi.org/10.1093/femsml/uqad001', 'Series_platform_id': 'GPL23691', 'Series_platform_taxid': '2246', 'Series_sample_taxid': '2246', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA858256'}\n",
      "558 PRJNA858420 GSE208116 {'Series_title': 'Evasion of antiviral innate immunity by SARS-CoV-2 NSP1 is exerted through H3K9 di-methylation', 'Series_geo_accession': 'GSE208116', 'Series_status': 'Public on Jul 12 2023', 'Series_submission_date': 'Jul 13 2022', 'Series_last_update_date': 'Jul 12 2023', 'Series_summary': 'The pathogen causing the current COVID-19 pandemic, the severe acute respiratory syndrome coronavirus (SARS-CoV-2), evades the innate immune machinery through the independent action of several viral proteins, including the nonstructural protein 1 (NSP1). NSP1 has multiple functions, but the relative contribution of NSP1-mediated translational repression, ribosome-proximate degradation of host mRNA, or other molecular mechanisms to viral immune evasion remains unclear. Here we combined several genome wide approaches, including RNA-seq, ribosome footprinting, and ChIP-seq to find that NSP1 predominantly affects transcription of immune-related genes. NSP1 expression in A549 cells induced Histone 3 Lysine 9 (H3K9) methylation of antiviral gene loci, leading to specific suppression of type I and type III interferon pathways. Treatment with the G9a/GLP H3K9 methyltransferase inhibitor UNC0638 reverses this suppression of antiviral genes and blocks viral replication after SARS-CoV2 infection of A549 cells. Our results call attention to epigenetic reprogramming induced by SARS-CoV2 and highlight the importance to identify innate factors regulating histone modification of gene loci targeted by SARS-CoV-2, with possible relevance to the understanding and therapy of other immunomodulatory diseases.', 'Series_overall_design': 'Herin several high throughput approaches including total RNA-Seq, Ribosome footprinting and ChIP-seq are combined to probe NSP1 function. Our results lead us to propose that NSP1 is a multifunctional protein that promotes innate antiviral immune evasion by inducing H3K9 dimethylation. Treatment with the specific H3K9 methyltransferases inhibitor UNC0638 attenuated suppression of antiviral genes by NSP1. In addition, UNC0638 treatment restored antiviral gene expression and blocked viral replication during SARS-CoV2 infection of human lung epithelial cells.', 'Series_type': 'Other', 'Series_contributor': 'Markus,,Hafner', 'Series_sample_id': 'GSM6337712', 'Series_contact_name': 'Markus,,Hafner', 'Series_contact_email': 'markus.hafner@nih.gov', 'Series_contact_laboratory': 'Laboratory of Muscle Stem Cells and Gene Regulation', 'Series_contact_department': 'NIAMS', 'Series_contact_institute': 'NIH', 'Series_contact_address': '50 South Drive', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '20892', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE208nnn/GSE208116/suppl/GSE208116_RAW.tar', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA858420'}\n",
      "559 PRJNA861947 GSE209633 {'Series_title': 'Ribosome profiling of 293T cells with or without miRNA inhibitors or non-cleaving shRNAs', 'Series_geo_accession': 'GSE209633', 'Series_status': 'Public on Jul 28 2022', 'Series_submission_date': 'Jul 25 2022', 'Series_last_update_date': 'Sep 27 2023', 'Series_pubmed_id': '37492926', 'Series_summary': 'Cells treated with NC control, miRNA inhibitors (miR-361-3p, 375-3p, 769-5p, and 4521), Scr control, or non-cleaving shNBD1s were harvested for ribosome profiling', 'Series_overall_design': 'Examination of translation status of miRNA inhibitors and non-cleaving shNBD1s', 'Series_type': 'Other', 'Series_contributor': 'Tadashi,,Yamamoto', 'Series_sample_id': 'GSM6381586', 'Series_contact_name': 'Hiroaki,,Sako', 'Series_contact_institute': 'Waseda University', 'Series_contact_address': '2-579-15', 'Series_contact_city': 'Tokorozawa', 'Series_contact_zip/postal_code': '359-1192', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE209nnn/GSE209633/suppl/GSE209633_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA861947'}\n",
      "560 PRJNA861982 GSE209648 {'Series_title': 'Distinct transcriptome, translatome, and proteome reprogramming during the oocyte-to-embryo transition', 'Series_geo_accession': 'GSE209648', 'Series_status': 'Public on Jun 06 2023', 'Series_submission_date': 'Jul 25 2022', 'Series_last_update_date': 'Sep 12 2023', 'Series_pubmed_id': '37443062', 'Series_summary': 'Oocyte-to-embryo transition plays a critical role in oocyte maturation and embryogenesis. It is a highly regulated process in part due to a transcription silenced period followed by zygotic genome activation. How transcriptome, translatome, and proteome interplay in this critical developmental window remains poorly understood. Utilizing a highly sensitive mass spectrometry, we obtained a high-quality proteome landscape spanning 10 stages, from the mouse full-grown oocyte (FGO) to blastocyst, using 100 oocytes/embryos at each stage. By integrative analysis with corresponding transcriptome and translatome, we found transcription and translation levels can not reflect protein abundance in most cases. From FGO to 4-cell embryos, proteomes are predominated by FGO-produced proteins, while the transcriptome and translatome are much more dynamic. FGO inherited proteins frequently persist after the corresponding transcripts are already downregulated or decayed. Improved concordance between protein and RNA is observed for genes starting translation only upon meiotic resumption (OET upregulated) or transcribed only in embryos, although the detected protein dynamics often lag behind transcription and translation. Concordance between protein and transcription/tranlation is associated with protein half-lives. Finally, a kinetic model well predicts protein dynamics when incorporating both the initial protein abundance in FGO and translation kinetics across developmental stages. In sum, our study reveals multilayer control of gene expression during oocyte maturation and embryogenesis.', 'Series_overall_design': 'By employing RNA-seq, Ribo-seq, and LC-MS/MS across 10 developmental stages from FGO to blastocyst, we investigated the protein, translation and transcription dynamics during oocyte maturation and early embryo development. We also invetigated the interplay between proteome and translatome or transcriptome.', 'Series_type': 'Other', 'Series_contributor': 'Wei,,Xie', 'Series_sample_id': 'GSM6381793', 'Series_contact_name': 'Wei,,Xie', 'Series_contact_email': 'xiewei121@tsinghua.edu.cn', 'Series_contact_department': 'THU-PKU Center for Life Sciences', 'Series_contact_institute': 'Tsinghua University', 'Series_contact_address': 'Zhongguancun street, Haidian', 'Series_contact_city': 'Beijing', 'Series_contact_zip/postal_code': '100084', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE209nnn/GSE209648/suppl/GSE209648_RAW.tar', 'Series_platform_id': 'GPL18480', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA861982'}\n",
      "561 PRJNA862900 GSE209902 {'Series_title': 'mRNA expression and translation regulated by the prion-like domain of Ilf3 in the mouse amygdala under chronic stress', 'Series_geo_accession': 'GSE209902', 'Series_status': 'Public on Feb 15 2023', 'Series_submission_date': 'Jul 27 2022', 'Series_last_update_date': 'Jun 14 2023', 'Series_pubmed_id': '36876121', 'Series_summary': 'Prion-like domain (PrLD) is a class of intrinsically disordered regions. Although its propensity to form condensates has been studied in the context of neurodegenerative diseases, the physiological role of PrLD remains unclear. Here, we investigated the physiological role of PrLD in NFAR2, generated by a splicing variant of the Ilf3 gene. Removal of the PrLD in mice affected the chronic water immersion and restraint stress (WIRS)-sensitive nuclear localization of NFAR2 in the amygdala, a fear-related brain region. Since NFAR2 regulates transcription and translation in the nucleus and cytoplasm, respectively, we investigated the effects of the PrLD deletion and WIRS on gene expression using RNA-seq and ribosome profiling.', 'Series_overall_design': 'RNA-seq and ribosome profiling in the amygdala from Ilf3+/+ and Ilf3∆PrLD/∆PrLD mice under WIRS and control conditions', 'Series_type': 'Other', 'Series_contributor': 'Nobuyuki,,Shiina', 'Series_sample_id': 'GSM6404274', 'Series_contact_name': 'Akira,,Yamashita', 'Series_contact_email': 'akiray92@nibb.ac.jp', 'Series_contact_institute': 'National Institute for Basic Biology', 'Series_contact_address': 'Nishigonaka 38, Myodaiji', 'Series_contact_city': 'Okazaki', 'Series_contact_zip/postal_code': '444-8585', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE209nnn/GSE209902/suppl/GSE209902_RAW.tar', 'Series_platform_id': 'GPL21103', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA862900'}\n",
      "562 PRJNA865860 GSE210503 {'Series_title': 'Ribosome profiling in Drosophila ovary enriched for late-stage oocytes (late-ovary)', 'Series_geo_accession': 'GSE210503', 'Series_status': 'Public on Dec 31 2022', 'Series_submission_date': 'Aug 03 2022', 'Series_last_update_date': 'Jun 22 2023', 'Series_pubmed_id': '37343092', 'Series_summary': 'To investigate the distribution of ribosomes in late-stage oocyte to infer their translational status.', 'Series_overall_design': 'Read patterns in Ribo_Seq libraries are used to infer local enrichment of ribosomes. Corresponding RNA_Seq libraries are used as control to filter out reads enrichment due to experimental artifact.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Elizabeth,R,Gavis', 'Series_sample_id': 'GSM6431493', 'Series_contact_name': 'Elizabeth,,Gavis', 'Series_contact_email': 'gavis@princeton.edu', 'Series_contact_institute': 'Princeton University', 'Series_contact_address': '401 Schultz Lab', 'Series_contact_city': 'Princeton', 'Series_contact_state': 'NJ', 'Series_contact_zip/postal_code': '08544', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE210nnn/GSE210503/suppl/GSE210503_RAW.tar', 'Series_platform_id': 'GPL17275', 'Series_platform_taxid': '7227', 'Series_sample_taxid': '7227', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA865860'}\n",
      "563 PRJNA871214 GSE211618 {'Series_title': 'The m6A Reader IGF2BP2 Regulates Glutamine Metabolism and Represents a Therapeutic Target in Acute Myeloid Leukemia [Ribo-seq]', 'Series_geo_accession': 'GSE211618', 'Series_status': 'Public on Nov 01 2022', 'Series_submission_date': 'Aug 19 2022', 'Series_last_update_date': 'Nov 01 2022', 'Series_pubmed_id': '36306790', 'Series_summary': 'We report that m6A reader IGF2BP2 participates in the regulation of glutamine metabolism in AML. To further clarify whether IGF2BP2 knockdown (KD) leads to translational inhibition, we perform Ribo-seq in control and IGF2BP2 KD cells.', 'Series_overall_design': 'MM6 cells infected with shNS or shIGF2BP2 (including shBP2B and shBP2D) were employed for ribo-seq.', 'Series_type': 'Other', 'Series_contributor': 'Jianjun,,Chen', 'Series_sample_id': 'GSM6481507', 'Series_contact_name': 'Hengyou,,Weng', 'Series_contact_email': 'weng_hengyou@gzlab.ac.cn', 'Series_contact_institute': 'Guangzhou Laboratory', 'Series_contact_address': 'Room 304, Building D, 190 Kaiyuan Ave,', 'Series_contact_city': 'Guangzhou', 'Series_contact_state': 'Guangdong', 'Series_contact_zip/postal_code': '510530', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE211nnn/GSE211618/suppl/GSE211618_Ribo-seq_normalized_read_count.xlsx', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA871214'}\n",
      "564 PRJNA871254 GSE211652 {'Series_title': 'Mycn regulates intestinal development through ribosomal biogenesis in a zebrafish model of Feingold syndrome 1 (RNA-seq and Ribo-seq)', 'Series_geo_accession': 'GSE211652', 'Series_status': 'Public on Aug 26 2022', 'Series_submission_date': 'Aug 19 2022', 'Series_last_update_date': 'Aug 26 2022', 'Series_summary': 'Conclusions: mTOR signaling was reduced in Mycn mutant.', 'Series_overall_design': 'wild-type or mycn zebrafish embryos at 72 hpf were harvested for RNA-seq and Ribo-seq.', 'Series_type': 'Other', 'Series_contributor': 'Yunfei,,Li', 'Series_sample_id': 'GSM6482085', 'Series_contact_name': 'Tao,,cheng', 'Series_contact_email': '11818118@zju.edu.cn', 'Series_contact_phone': '0571-88208276', 'Series_contact_department': 'Institute of genetics', 'Series_contact_institute': 'Zhejiang University', 'Series_contact_address': '886 Yuhangtang Road', 'Series_contact_city': 'Hang Zhou', 'Series_contact_state': 'Zhe Jiang', 'Series_contact_zip/postal_code': '310000', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE211nnn/GSE211652/suppl/GSE211652_translation_efficiency_matrix.csv.gz', 'Series_platform_id': 'GPL24995', 'Series_platform_taxid': '7955', 'Series_sample_taxid': '7955', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA871254'}\n",
      "565 PRJNA872137 GSE211793 {'Series_title': 'The Translation Initiation Factor Homolog, eif4e1c, Regulates Cardiomyocyte Metabolism and Proliferation During Heart Regeneration.', 'Series_geo_accession': 'GSE211793', 'Series_status': 'Public on Aug 25 2022', 'Series_submission_date': 'Aug 22 2022', 'Series_last_update_date': 'Sep 08 2023', 'Series_pubmed_id': '37306388', 'Series_summary': 'The\\u202feIF4E family of translation initiation factors bind 5’ methylated caps and act as the limiting-step for mRNA translation. The canonical eIF4E1A is required for cell viability, yet other related families exist and are all utilized in specific contexts or tissues. Here, we describe a new family called Eif4e1c that is ancestral to the canonical eIF4E1A in vertebrates, and for which we find roles during heart development and regeneration in zebrafish.\\u202fThe Eif4e1c family is present in all aquatic vertebrates but has been lost in all terrestrial species. A core group of amino acids shared over 500 million years of evolution from\\u202fray-finned fish to sharks\\u202fforms an evolutionarily conserved binding interface along the surface of the protein, suggesting Eif4e1c functions in a novel pathway.\\u202f Deletion of eif4e1c\\u202fin zebrafish caused growth deficits and impaired survival in juveniles.\\u202fMutants surviving to adulthood had fewer cardiomyocytes, while also showing reduced proliferative responses to cardiac injury. Ribosome profiling of mutant hearts demonstrated changes in translation efficiency of mRNA for genes known to regulate cardiomyocyte proliferation. Although\\u202feif4e1c\\u202fis expressed in many tissues and is the predominant cap-binding initiation homolog in embryos, its disruption had most notable impact on the heart and at juvenile stages. \\u202fOur findings reveal context-dependent requirements for translation initiation regulators during heart regeneration.', 'Series_overall_design': 'Ribosome profiling of adult uninjured eif4e1c zebrafish mutant hearts and adult uninjured EK zebrafish (wildtype) hearts.', 'Series_type': 'Other', 'Series_contributor': 'Eckhard,,Jankowsky', 'Series_sample_id': 'GSM6502520', 'Series_contact_name': 'Joseph,Aaron,Goldman', 'Series_contact_email': 'Aaron.goldman@osumc.edu', 'Series_contact_department': 'Biological Chemistry and Pharmacology', 'Series_contact_institute': 'The Ohio State University', 'Series_contact_address': '119 Rightmire Hall, 1060 Carmack Road', 'Series_contact_city': 'Columbus', 'Series_contact_state': 'Ohio', 'Series_contact_zip/postal_code': '43212', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE211nnn/GSE211793/suppl/GSE211793_summary_gene_count_RNA_table.txt.gz', 'Series_platform_id': 'GPL25922', 'Series_platform_taxid': '7955', 'Series_sample_taxid': '7955', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA872137'}\n",
      "566 PRJNA874814 GSE212288 {'Series_title': 'Ribosome profiling of MCMV', 'Series_geo_accession': 'GSE212288', 'Series_status': 'Public on Apr 26 2023', 'Series_submission_date': 'Aug 29 2022', 'Series_last_update_date': 'Apr 26 2023', 'Series_summary': 'Swiss murine embryonic fibroblasts (NIH-3T3) were infected with wild-type Smith strain mouse cytomegalovirus (MCMV) at a multiplicity of infection (MOI) of 10. Ribosome profiling was performed at various times during infection with minor modification to the protocol described in Stern-Ginossar N et al., Science 2012. The detailed protocol is described in Rutkowski et al, Nature commun 2015, and Whisnant et al., Nature commun. 2020. Translation start site profiling was performed by culturing cells in presence of Harringtonin or Lactimidomycin for 30 min prior to cell harvest.', 'Series_overall_design': 'Ribosome profiling was performed a 0, 2, 6, 12, 24 and 48 h post infection. Translation start site profiling was performed using Harringtonin (2µg/ml for for 30 min) or Lactimidomycin (50µM for 30 min) prior to cell lysis. Two biological replicates were analysed.', 'Series_type': 'Other', 'Series_contributor': 'Lars,,Dölken', 'Series_sample_id': 'GSM6523429', 'Series_contact_name': 'Florian,,Erhard', 'Series_contact_institute': 'Institute of Virology and Immunobiology, University of Wuerzburg', 'Series_contact_address': 'Versbacher Straße 7', 'Series_contact_city': 'Wuerzburg', 'Series_contact_zip/postal_code': '97078', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE212nnn/GSE212288/suppl/GSE212288_MCMV_orfs.tar.gz', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA874814'}\n",
      "567 PRJNA878797 GSE213023 {'Series_title': 'Total and dendritic ribosome profiling from primary mouse cortical neurons', 'Series_geo_accession': 'GSE213023', 'Series_status': 'Public on Mar 14 2024', 'Series_submission_date': 'Sep 09 2022', 'Series_last_update_date': 'Apr 10 2024', 'Series_pubmed_id': '38589584', 'Series_summary': 'Activity-dependent protein synthesis is critical for determining changes in dendritic proteomes underlying brain function, yet the mechanisms governing these changes are lacking. Here, we combined proximity-based labeling of dendritic transcriptome, translatome, and proteome to study the dynamics of RNA regulation in activated synapses. We discovered that depolarization leads to a switch from RNAs translated under basal conditions to new translation of previously unrecognized subsets of depolarization-dependent transcripts. Dynamically regulated RNAs bound by specific regulatory proteins, including NOVA1, FMRP, and ELAVLs, rapidly generate proteins with roles in mitochondrial regulation, RNA metabolism, translational control, and synaptic signaling in dendrites. Knockdowns of activity-induced dendritic RNAs altered neuronal physiology, underscoring how dynamic switches in the regulation of RNAs encoding coordinated sets of proteins underlie synaptic plasticity.', 'Series_overall_design': 'Four 150 mm cell culture dishes for resting and depolarized primary cortical neurons were combined per sample per replicate, 1/4th of which was spared for proteomics and rest for the ribosome profiling experiments. After a two-minute cycloheximide (Chx) treatment followed by a 30-minute biotin pulse, cells were quickly rinsed in ice-cold polysome gradient buffer (20 mM Tris pH7.5, 150 mM NaCl, 5 mM MgCl2, 1 mM DTT, and 100 μg/ml CHX), then scraped and lysed on plates on ice in 1 mL of ice-cold polysome lysis buffer (20 mM Tris pH7.5, 150 mM NaCl, 5 mM MgCl2 supplemented with 20 U/mL SUPERase-In RNase inhibitor, 24 U/mL Turbo DNase, 1 mM DTT, and 100 μg/ml CHX, 1x EDTA-free protease inhibitor. Lysates were clarified by centrifugation at 20,000 g for 2 minutes at 4°C, after which the supernatant was immediately loaded on to the cold 5 mL 7 kDa MWCO Zeba desalting column that was previously equilibrated with the ice-cold polysome gradient buffer. Lysates were subsequently flash-frozen or used for monosome fractionation. For monosome fractionation, lysates were added 5 mM CaCl2, incubated with micrococcal nuclease for 45 minutes at room temperature, quenched with 6.25 mM EGTA,\\xa0and loaded on sucrose gradients. Gradients were then spun for 2 hours at 41K rpm in an SW-41 rotor and fractionated using BioComp Gilson fraction collection. 10% of the monosome fraction was kept for input. Triton X-100 was added to the remaining monosomes to a final concentration of 0.05%. The biotinylated monosomes were isolated using 50 µl of MyOne streptavidin C1 magnetic Dynabeads that were prepared according to the manufacturer’s guidelines and washed twice with Buffer A, twice with Buffer B, and once with polysome gradient buffer (with DTT and CHX). The monosome and bead mix was nutated at 4°C overnight. The supernatant was removed the next day, and the beads were moved into a new tube with low salt wash buffer (20 mM Tris pH8, 150 mM NaCl, 5 mM MgCl2, 1 mM DTT, and 100 μg/ml CHX, and 0.1% Triton X-100), and consequently washed three times for 10 minutes at 4°C with high salt wash buffer (20 mM Tris pH8, 500 mM NaCl, 5 mM MgCl2, 1 mM DTT, 100 μg/ml CHX, and 0.1% Triton X-100).', 'Series_type': 'Other', 'Series_contributor': 'Robert,B,Darnell', 'Series_sample_id': 'GSM8192334', 'Series_contact_name': 'Caryn,R,Hale', 'Series_contact_email': 'chale@rockefeller.edu, carynhale@gmail.com', 'Series_contact_institute': 'Rockefeller University', 'Series_contact_address': '1230 York Avenue', 'Series_contact_city': 'New York', 'Series_contact_state': 'New York', 'Series_contact_zip/postal_code': '10065', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE213nnn/GSE213023/suppl/GSE213023_RAW.tar', 'Series_platform_id': 'GPL24247', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA878797'}\n",
      "568 PRJNA880902 GSE213472 {'Series_title': 'Ribosome dynamics following arginine deprivation in colon cancer cells', 'Series_geo_accession': 'GSE213472', 'Series_status': 'Public on Jan 06 2023', 'Series_submission_date': 'Sep 15 2022', 'Series_last_update_date': 'Apr 07 2023', 'Series_pubmed_id': '36608131', 'Series_summary': 'To investigate ribosome localization and gene expression dynamics during starvation in cancer cells, we performed RNA sequencing and ribosome profiling in RKO cells under fed and starved (12.5 uM arginine) conditions.', 'Series_overall_design': 'Cells were kept under fed (400 uM arginine) or starved (12.5 uM arginine) conditions for 24 hours. RNA sequencing and ribosome profiling were performed concurrently following previously published protocols (PMID 28579404)', 'Series_type': 'Other', 'Series_contributor': 'Dennis,,Hsu', 'Series_sample_id': 'GSM6586866', 'Series_contact_name': 'Dennis,,Hsu', 'Series_contact_institute': 'University of Pittsburgh', 'Series_contact_address': '5117 Centre Ave', 'Series_contact_city': 'Pittsburgh', 'Series_contact_state': 'PA', 'Series_contact_zip/postal_code': '15213', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE213nnn/GSE213472/suppl/GSE213472_rpf_count.txt.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA880902'}\n",
      "569 PRJNA885130 GSE214396 {'Series_title': 'Mechanisms of stop codon readthrough mitigation reveal principles of GCN1 mediated translational quality control', 'Series_geo_accession': 'GSE214396', 'Series_status': 'Public on May 15 2023', 'Series_submission_date': 'Sep 28 2022', 'Series_last_update_date': 'Aug 14 2023', 'Series_pubmed_id': '37339632', 'Series_summary': 'Readthrough into the 3′ untranslated region (3′UTR) of the mRNA results in the production of aberrant proteins. Metazoans efficiently clear readthrough proteins, but the underlying mechanisms remain unknown. Here, we show in C. elegans and mammalian cells that readthrough proteins are targeted by a coupled, two-level quality control pathway involving the BAG6 chaperone complex and the ribosome collision-sensing protein GCN1. Readthrough proteins with hydrophobic C-terminal extensions are recognized by SGTA-BAG6 and ubiquitylated by RNF126 for proteasomal degradation. Additionally, cotranslational mRNA decay initiated by GCN1 and CCR4/NOT limits the accumulation of readthrough products. Unexpectedly, selective ribosome profiling uncovered a general role of GCN1 in regulating translation dynamics when ribosomes encounter nonoptimal codons, a feature of 3′UTR sequences. Dysfunction of GCN1 results in mRNA and proteome imbalance, increasingly perturbing transmembrane proteins and collagens during aging. These results define GCN1 as a key factor acting during translation in maintaining protein homeostasis.', 'Series_overall_design': 'Selective Ribo-seq of GCN1 in C.elegans and HEK293 with respective input controls and additional mRNAseq', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'F.,U,Hartl', 'Series_sample_id': 'GSM6995358', 'Series_contact_name': 'Martin,,Mueller', 'Series_contact_email': 'mamueller@biochem.mpg.de', 'Series_contact_laboratory': 'Hartl', 'Series_contact_institute': 'MPI of Biochemistry', 'Series_contact_address': 'Am Klopferspitz 18', 'Series_contact_city': 'Martinsried/Planegg', 'Series_contact_zip/postal_code': '82152', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE214nnn/GSE214396/suppl/GSE214396_WT_halflives.csv.gz', 'Series_platform_id': 'GPL26672', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA885130'}\n",
      "570 PRJNA895461 GSE216831 {'Series_title': 'Decapping factor Dcp2 controls mRNA abundance and translation to adjust metabolism and filamentation to nutrient availability', 'Series_geo_accession': 'GSE216831', 'Series_status': 'Public on Feb 07 2023', 'Series_submission_date': 'Oct 28 2022', 'Series_last_update_date': 'Sep 27 2023', 'Series_pubmed_id': '37439347', 'Series_summary': 'Degradation of most yeast mRNAs involves decapping by Dcp1/Dcp2. DEAD-box protein Dhh1 has been implicated as an activator of decapping, and as a translational repressor, but their functions in cells are incompletely understood. We have analyzed these questions by a combination of ribosome profiling, RNA-Seq, CAGE analysis of capped mRNAs.', 'Series_overall_design': 'This study includes total 24 samples, 6 for ribosome profiling and 6 for RNA-Seq derived from duplicates of WT, dcp2D and dcp2-EE. Separately, total RNA of WT, dcp2, and xrn1 in duplicates,  were subjected to CAGE-Seq (6 samples) and total RNA-Seq (6 samples). All strains were grown in YPD at 30ºC till the OD600 ~0.6', 'Series_type': 'Other', 'Series_contributor': 'Alan,G,Hinnebusch', 'Series_sample_id': 'GSM6696337', 'Series_contact_name': 'Alan,G,Hinnebusch', 'Series_contact_email': 'alanh@mail.nih.gov', 'Series_contact_laboratory': 'Section on Nutrient Control of Gene Expression', 'Series_contact_department': 'Eunice Kennedy Shriver National Institute of Child Health and Human Development', 'Series_contact_institute': 'National Institutes of Health', 'Series_contact_address': '6 Center Drive', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'Maryland', 'Series_contact_zip/postal_code': '20892', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE216nnn/GSE216831/suppl/GSE216831_RAW.tar', 'Series_platform_id': 'GPL19756', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA895461'}\n",
      "571 PRJNA898352 GSE217287 {'Series_title': 'An mRNA processing pathway suppresses metastasis by governing translational control from the nucleus [Ribo-seq, RNA-seq]', 'Series_geo_accession': 'GSE217287', 'Series_status': 'Public on Feb 20 2023', 'Series_submission_date': 'Nov 04 2022', 'Series_last_update_date': 'Feb 20 2023', 'Series_summary': 'Cancer cells often co-opt post-transcriptional regulatory mechanisms to achieve pathologic expression of gene networks that drive metastasis. Translational control is a major regulatory hub in oncogenesis, however its effects on cancer progression remain poorly understood. To address this, we used ribosome profiling to compare genome-wide translation efficiencies of poorly and highly metastatic breast cancer cells and patient-derived xenografts. We developed novel regression-based methods to analyze ribosome profiling and alternative polyadenylation data, and identified HNRNPC as a translational controller of a specific mRNA regulon. Mechanistically, HNRNPC, in concert with PABPC4, binds near to poly(A) signals, thereby governing the alternative polyadenylation of a set of mRNAs. We found that HNRNPC and PABPC4 are downregulated in highly metastatic cells, which causes HNRNPC-bound mRNAs to undergo 3’ UTR lengthening and subsequently, translational repression. We showed that modulating HNRNPC expression impacts the metastatic capacity of breast cancer cells in xenograft mouse models. We also found that a small molecule, previously shown to induce a distal-to-proximal poly(A) site switching, counteracts the HNRNPC-PABPC4 driven deregulation of alternative polyadenylation and decreases the metastatic lung colonization by breast cancer cells in vivo.', 'Series_overall_design': 'Comparison of translation efficiencies in highly and poorly metastatic breast cancer cell lines', 'Series_type': 'Other', 'Series_contributor': 'Hani,,Goodarzi', 'Series_sample_id': 'GSM6710499', 'Series_contact_name': 'Hani,,Goodarzi', 'Series_contact_department': 'Biochemistry and Biophysics', 'Series_contact_institute': 'UCSF', 'Series_contact_address': '600 16th St, GH S312D', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE217nnn/GSE217287/suppl/GSE217287_sgHNRNPC_norm_count_rpf_rna.txt.gz', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA898352'}\n",
      "572 PRJNA898430 GSE217354 {'Series_title': 'IL-17/CXCL5 signaling within the oligovascular niche mediates human and mouse white matter injury (OPC RiboTAG)', 'Series_geo_accession': 'GSE217354', 'Series_status': 'Public on Nov 11 2022', 'Series_submission_date': 'Nov 04 2022', 'Series_last_update_date': 'Feb 10 2023', 'Series_pubmed_id': '36543124', 'Series_web_link': 'https://www.sciencedirect.com/science/article/pii/S2211124722017405?via%3Dihub', 'Series_summary': 'Cerebral small vessel disease and brain white matter injury are worsened by cardiovascular risk factors including obesity. Molecular pathways in cerebral endothelial cells activated by chronic cerebrovascular risk factors alter cell-cell signaling, blocking endogenous and post-ischemic white matter repair. Using cell-specific translating ribosome affinity purification(RiboTag) in white matter endothelia and oligodendrocyte progenitor cells (OPCs), we identify a coordinated interleukin-chemokine signaling cascade within the oligovascular niche of subcortical white matter that is triggered by diet-induced obesity (DIO).DIO induces IL-17B signaling that acts on the cerebral endothelia through IL-17Rb to increase both circulating and local endothelial expression of CXCL5. In white matter endothelia, CXCL5 promotes the association of OPCs with the vasculature and triggers OPC gene expression programs regulating cell migration through chemokine signaling. Targeted blockade of IL-17B reduced vessel-associated OPCs by reducing endothelial CXCL5 expression. In multiple human cohorts, blood levels of CXCL5 function as a diagnostic and prognostic biomarker of vascular cognitive impairment.', 'Series_overall_design': 'Comparative gene expression profiling from the white matter of Tie2-Cre:RiboTAG mice on either control fat or high fat diet. RiboTAG gene expression analysis performed as IP vs. input', 'Series_type': 'Other', 'Series_contributor': 'Jason,D,Hinman', 'Series_sample_id': 'GSM6715995', 'Series_contact_name': 'Jason,D,Hinman', 'Series_contact_email': 'JHinman@mednet.ucla.edu', 'Series_contact_laboratory': 'Hinman Lab', 'Series_contact_department': 'Neurology', 'Series_contact_institute': 'UCLA', 'Series_contact_address': '635 Charles E. Young Dr. South, Rm 415', 'Series_contact_city': 'Los Angeles', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '90095', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE217nnn/GSE217354/suppl/GSE217354_RAW.tar', 'Series_platform_id': 'GPL21103', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA898430'}\n",
      "573 PRJNA898432 GSE217355 {'Series_title': 'IL-17/CXCL5 signaling within the oligovascular niche mediates human and mouse white matter injury (Tie2-Cre RiboTAG)', 'Series_geo_accession': 'GSE217355', 'Series_status': 'Public on Nov 11 2022', 'Series_submission_date': 'Nov 04 2022', 'Series_last_update_date': 'Feb 10 2023', 'Series_pubmed_id': '36543124', 'Series_web_link': 'https://www.sciencedirect.com/science/article/pii/S2211124722017405?via%3Dihub', 'Series_summary': 'Cerebral small vessel disease and brain white matter injury are worsened by cardiovascular risk factors including obesity. Molecular pathways in cerebral endothelial cells activated by chronic cerebrovascular risk factors alter cell-cell signaling, blocking endogenous and post-ischemic white matter repair. Using cell-specific translating ribosome affinity purification(RiboTag) in white matter endothelia and oligodendrocyte progenitor cells (OPCs), we identify a coordinated interleukin-chemokine signaling cascade within the oligovascular niche of subcortical white matter that is triggered by diet-induced obesity (DIO).DIO induces IL-17B signaling that acts on the cerebral endothelia through IL-17Rb to increase both circulating and local endothelial expression of CXCL5. In white matter endothelia, CXCL5 promotes the association of OPCs with the vasculature and triggers OPC gene expression programs regulating cell migration through chemokine signaling. Targeted blockade of IL-17B reduced vessel-associated OPCs by reducing endothelial CXCL5 expression. In multiple human cohorts, blood levels of CXCL5 function as a diagnostic and prognostic biomarker of vascular cognitive impairment.', 'Series_overall_design': 'Comparative gene expression profiling from the white matter of Tie2-Cre:RiboTAG mice on either control fat or high fat diet. RiboTAG gene expression analysis performed as IP vs. input', 'Series_type': 'Other', 'Series_contributor': 'Jason,D,Hinman', 'Series_sample_id': 'GSM6716007', 'Series_contact_name': 'Jason,D,Hinman', 'Series_contact_email': 'JHinman@mednet.ucla.edu', 'Series_contact_laboratory': 'Hinman Lab', 'Series_contact_department': 'Neurology', 'Series_contact_institute': 'UCLA', 'Series_contact_address': '635 Charles E. Young Dr. South, Rm 415', 'Series_contact_city': 'Los Angeles', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '90095', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE217nnn/GSE217355/suppl/GSE217355_Tie2-Cre_RiboTAG_Complete_geneList_EdgeR.csv.gz', 'Series_platform_id': 'GPL21103', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA898432'}\n",
      "574 PRJNA903814 GSE218433 {'Series_title': 'A ubiquitous GC content signature underlies multimodal post-transcriptional regulation by DDX3X', 'Series_geo_accession': 'GSE218433', 'Series_status': 'Public on Aug 08 2023', 'Series_submission_date': 'Nov 21 2022', 'Series_last_update_date': 'Mar 13 2024', 'Series_pubmed_id': '38273160', 'Series_summary': 'The road from transcription to protein synthesis is paved with many obstacles, allowing for several modes of post-transcriptional regulation of gene expression. A fundamental player in mRNA biology, DDX3X is an RNA binding protein that regulates mRNA translation. By monitoring dynamics of mRNA abundance and translation following DDX3X depletion, we observe stabilization of translationally suppressed mRNA. We used interpretable statistical learning models to uncover GC content in the coding sequence as the major feature underlying RNA stabilization. GC content-related mRNA regulation is detectable in other studies, including hundreds of ENCODE datasets and recent work focusing on mRNA dynamics in the cell cycle. We provide further evidence for mRNA stabilization by detailed analysis of RNA-seq profiles in hundreds of samples, including a recently published Ddx3x conditional mouse model exhibiting cell cycle and neurogenesis defects. Our study identifies a ubiquitous feature underlying mRNA regulation and highlights the importance of quantifying multiple steps of the gene expression cascade, where RNA abundance and protein production are often uncoupled.', 'Series_overall_design': 'RNA-seq, Ribo-seq (Ribosome footprints) and SLAM-seq following DDX3X inactivation', 'Series_type': 'Other', 'Series_contributor': 'Lorenzo,,Calviello', 'Series_sample_id': 'GSM6744646', 'Series_contact_name': 'Lorenzo,,Calviello', 'Series_contact_email': 'calviello.l.bio@gmail.com', 'Series_contact_department': 'Berlin Institute for Medical Systems Biology', 'Series_contact_institute': 'Max Delbrueck Center for Molecular Medicine', 'Series_contact_address': 'Robert-Rössle-Straße 10, MDC, Haus 89', 'Series_contact_city': 'Berlin', 'Series_contact_zip/postal_code': '13125', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE218nnn/GSE218433/suppl/GSE218433_RAW.tar', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA903814'}\n",
      "575 PRJNA904347 GSE218580 {'Series_title': 'Adaptations in nucleus accumbens neuron subtypes mediate negative affective behaviors in fentanyl abstinence II', 'Series_geo_accession': 'GSE218580', 'Series_status': 'Public on Nov 28 2022', 'Series_submission_date': 'Nov 22 2022', 'Series_last_update_date': 'Dec 01 2022', 'Series_summary': 'We used RNAseq to evaluate baseline expression of NPAS+ VP translatomes in comparison to bulk VP translatomes.', 'Series_overall_design': 'We performed gene expression profiling in ribosome-associated mRNA from two different mouse genotypes (NPAS-Cre x Rpl22-HA or Rpl22-HA with Cre AAV in the VP) at baseline', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Seth,A,Ament', 'Series_sample_id': 'GSM6752834', 'Series_contact_name': 'Mahashweta,,Basu', 'Series_contact_email': 'MBasu@som.umaryland.edu', 'Series_contact_phone': '3019177729', 'Series_contact_institute': 'Institute for Genome Sciences', 'Series_contact_address': '670 W. Baltimore Street', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21201', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE218nnn/GSE218580/suppl/GSE218580_RAW.tar', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA904347'}\n",
      "576 PRJNA906596 GSE219022 {'Series_title': 'Ribosome profiling reveals the fine-tuned response of Escherichia coli to acid stress', 'Series_geo_accession': 'GSE219022', 'Series_status': 'Public on Sep 28 2023', 'Series_submission_date': 'Nov 29 2022', 'Series_last_update_date': 'Jan 03 2024', 'Series_pubmed_id': '37909716', 'Series_summary': 'The response to acidity is crucial for neutralophilic bacteria. Escherichia coli has a well characterized regulatory network to induce multiple defense mechanisms against excess of protons. Nevertheless, systemic studies of the transcriptional and translational reprogramming of E. coli to different acidic strengths have not yet been performed. Here, we used ribosome profiling and mRNA sequencing to determine the response of E. coli to pH 7.6, 5.8 and 4.4. Data were analyzed using the high-throughput HRIBO pipeline and previously undetected adaptations of E. coli to acid stress were found including up-regulation of glycerol catabolism and siderophore production, down-regulation of many membrane proteins and regulation by the transcriptional regulators YdeO, MhpR, IscR, and YdcI. Several examples of differential transcriptional and translational regulation of genes were identified as well as potential novel small open reading frames. These results expand the acid resistance network and provide new insights into the fine-tuned response of E. coli.', 'Series_overall_design': 'Comparison of genome-wide mRNA levels and translational activity at pH 7.6, 5.8 and 4.4 in LB medium.', 'Series_type': 'Other', 'Series_contributor': 'Kirsten,,Jung', 'Series_sample_id': 'GSM6765096', 'Series_contact_name': 'Rick,,Gelhausen', 'Series_contact_email': 'gelhausr@informatik.uni-freiburg.de', 'Series_contact_laboratory': 'Bioinformatics Group (AG Backofen)', 'Series_contact_department': 'Department of Computer Science', 'Series_contact_institute': 'Albert-Ludwigs-University Freiburg', 'Series_contact_address': 'Georges-Köhler-Allee 106', 'Series_contact_city': 'Freiburg', 'Series_contact_state': 'Baden Württemberg', 'Series_contact_zip/postal_code': '79110', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE219nnn/GSE219022/suppl/GSE219022_RAW.tar', 'Series_platform_id': 'GPL32899', 'Series_platform_taxid': '511145', 'Series_sample_taxid': '511145', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA906596'}\n",
      "577 PRJNA907315 GSE219172 {'Series_title': 'The post-translational modifications and translation-regulatory role of METTL14 in 5-fluorouracil-resistant colorectal cancer [Ribo-seq]', 'Series_geo_accession': 'GSE219172', 'Series_status': 'Public on Dec 01 2023', 'Series_submission_date': 'Dec 01 2022', 'Series_last_update_date': 'Dec 01 2023', 'Series_summary': '5-fluorouracil (5‑FU)  based chemotherapy is the first-line chemotherapy scheme for CRC, whereas acquired drug resistance poses a huge obstacle to curing CRC patients and the mechanism is still obscure. METTL14 is a core subunit of the m6A methyltransferase complex and has been reported to regulate the drug resistance of cancers. Here, we demonstrated that METTL14 is significantly downregulated in 5-FU-resistant CRC and METTL14 overexpression inhibits the proliferation and 5-FU resistance of CRC cells in vitro and in vivo. In addition, we found that MIB1 catalyzes the K63-linked ubiquitination of METTL14 and induces its degradation through the autolysosome pathway, while AURKA-mediated phosphorylation enhances METTL14 stability by inhibiting its ubiquitination. Moreover, we first discovered that METTL14 functions in 5-FU-resistant CRC cells by directly participating in the translation initiation of non-m6A-modified mRNAs independently of METTL3, rather than cooperating with METTL3 to methylate adenosine residues of targets. Further analysis indicated that METTL14 regulates the cell cycle process in 5-FU-resistant CRC cells. Collectively, our study not only identified METTL14 as a potential therapeutic target for 5-FU resistance in CRC but also revealed a novel translation-facilitating role of METTL14.', 'Series_overall_design': 'METTL14 overexpression plasmids and negative control plasmids were transfected into 5-FU-resistant HCT-8 cells, respectively. Then cells were treated with 100ug/mL CHX and implemented to the Ribosome-sequencing. Experiments were performed twice independently.', 'Series_type': 'Other', 'Series_contributor': 'Jian,,Qin', 'Series_sample_id': 'GSM6774936', 'Series_contact_name': 'Shukui,,Wang', 'Series_contact_email': 'sk_wang@njmu.edu.cn', 'Series_contact_institute': 'Nanjing First Hospital', 'Series_contact_address': 'No.68, Changle Road, Nanjing', 'Series_contact_city': 'Nanjing', 'Series_contact_zip/postal_code': '210006', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE219nnn/GSE219172/suppl/GSE219172_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA907315'}\n",
      "578 PRJNA913269 GSE221165 {'Series_title': 'Human Hematopoietic Stem Cell Vulnerability to Ferroptosis [Ribosome profiling]', 'Series_geo_accession': 'GSE221165', 'Series_status': 'Public on Feb 17 2023', 'Series_submission_date': 'Dec 16 2022', 'Series_last_update_date': 'May 19 2023', 'Series_pubmed_id': '36803603', 'Series_web_link': 'https://doi.org/10.1016/j.cell.2023.01.020', 'Series_summary': 'We performed ribosome profiling followed by NGS sequencing where only ribosome protected RNA was extracted to make the library, which mostly are in active translation', 'Series_overall_design': 'comparative ribosome profiling assay of MYSM1 or AAVS1 edited CD34+CD45RA-CD90+ cells with three biological replicates of each group', 'Series_type': 'Other', 'Series_contributor': 'Vijay,,Sankaran', 'Series_sample_id': 'GSM6846863', 'Series_contact_name': 'Jiawei,,Zhao', 'Series_contact_email': 'jiawei@broadinstitute.org', 'Series_contact_institute': \"Boston Children's Hospital\", 'Series_contact_address': '1 Blackfan Circle', 'Series_contact_city': 'Boston', 'Series_contact_state': 'Massachusetts', 'Series_contact_zip/postal_code': '02115', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE221nnn/GSE221165/suppl/GSE221165_RAW.tar', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA913269'}\n",
      "579 PRJNA918384 GSE222154 {'Series_title': 'Cpeb1b-mediated cytoplasmic polyadenylation of shha mRNA modulates zebrafish definitive hematopoiesis [RIBO-Seq]', 'Series_geo_accession': 'GSE222154', 'Series_status': 'Public on Feb 07 2023', 'Series_submission_date': 'Jan 04 2023', 'Series_last_update_date': 'Feb 09 2023', 'Series_summary': 'Ribosome profiling RNA sequencing in the low dose cpeb1b MO injected wt and cpeb1b mutant zebrafish embryos', 'Series_overall_design': 'Ribo-seq of 6 samples from the low dose cpeb1b MO injected wt zebrafish embryos and 6 samples from the low dose cpeb1b MO injected Cpeb1b mutant zebrafish embryos at 8 hpf, 16 hpf, and 22 hpf.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'J,,Heng', 'Series_sample_id': 'GSM6915732', 'Series_contact_name': 'Bao-Fa,,Sun', 'Series_contact_email': 'sunbf@big.ac.cn', 'Series_contact_institute': 'Beijing Institute of Genomics (BIG) of Chinese Academy of Sciences (CAS)', 'Series_contact_address': 'Da-Tun Road', 'Series_contact_city': 'Beijing', 'Series_contact_zip/postal_code': '100101', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE222nnn/GSE222154/suppl/GSE222154_RAW.tar', 'Series_platform_id': 'GPL24995', 'Series_platform_taxid': '7955', 'Series_sample_taxid': '7955', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA918384'}\n",
      "580 PRJNA918681 GSE222249 {'Series_title': 'Heterogeneous ribonucleoprotein interactions and impeded translational elongation in the respiratory tissue of SARS-CoV-2 pathology (Ribo-Seq)', 'Series_geo_accession': 'GSE222249', 'Series_status': 'Public on Dec 01 2023', 'Series_submission_date': 'Jan 05 2023', 'Series_last_update_date': 'Jan 16 2024', 'Series_pubmed_id': '37907741', 'Series_summary': 'Translational regulation in in vivo tissue environments during viral pathogenesis has hardly been scrutinized due to the lack of tissue translatomes upon viral infection despite a number of the translatome studies on virus-infected cells cultured in vitro. We constructed the first temporal profile of lung translatomes during SARS-CoV-2 pathogenesis by applying ribosome profiling (Ribo-seq) to a severe COVID-19 mouse model. Unexpectedly, we observed gradual accumulations of non-canonical Ribo-seq reads representing hitherto-unidentified ribonucleoproteins (RNPs) that are likely involved in impeded translational elongation in the infected tissues. Contemporarily developing ribosome heterogeneity with prominently deviated 5S rRNP association supported the malfunction of elongating ribosomes. The analyses of canonical Riboseq reads (ribosome footprints) highlighted two obstructive characteristics to host gene expression: attenuated translation for transcriptionally up-regulated genes including immune response genes and ribosome stalling on codons within transmembrane domain-coding regions. Our study elucidates hidden molecular features of gene regulation in vivo underlying SARS-CoV-2 pathogenesis.', 'Series_overall_design': 'Ribo-seq for lung tissues of K18-hACE2 transgenic mice infected by SARS-CoV-2. For the library construction, we divided conditions as two groups, the early set (Control for the early set, 1 day post-infection, 2 days post-infection) and late set (Control for the late set, 5 days post-infection, 7 days post-infection). For each condition, we prepared quadruplets.', 'Series_type': 'Other', 'Series_contributor': 'Jun,,Cho', 'Series_sample_id': 'GSM6918971', 'Series_contact_name': 'Hyeshik,,Chang', 'Series_contact_email': 'hyeshik@snu.ac.kr', 'Series_contact_laboratory': 'Hyeshik Chang Lab', 'Series_contact_department': 'School of Biological Sciences', 'Series_contact_institute': 'Seoul National University', 'Series_contact_address': 'Building 203 Room 525, School of Biological Sciences, Seoul National University, 1 Gwanak-ro, Gwanak-gu', 'Series_contact_city': 'Seoul', 'Series_contact_state': 'South Korea', 'Series_contact_zip/postal_code': '08826', 'Series_contact_country': 'South Korea', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE222nnn/GSE222249/suppl/GSE222249_RPF_count.csv.gz', 'Series_platform_id': 'GPL24247', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA918681'}\n",
      "581 PRJNA922045 GSE222415 {'Series_title': 'Effect of upregulation of LINE-1 retrotransposon on translation in the mouse testis', 'Series_geo_accession': 'GSE222415', 'Series_status': 'Public on May 25 2023', 'Series_submission_date': 'Jan 09 2023', 'Series_last_update_date': 'Aug 28 2023', 'Series_pubmed_id': '37307272', 'Series_summary': 'We used piRNA-deficient Maelstrom (Mael) knockout mice to investigate the effect of LINE-1 retrotransposon upregulation on translation efficiency (TE) in the germline. We collected Mael+/- and Mael-/- testes at postnatal day 16 (P16), when LINE-1 upregulation in Mael-/- spermatocytes is manifest, and performed ribosome profiling. RNA-seq analysis of the same samples was performed in parallel to evaluate the contribution of transcriptional changes to variations in footprint levels.', 'Series_overall_design': 'Comparative ribosome profiling analysis of P16 Mael-/- testes and their Mael+/- controls. Three biological replicates per genotype are reported.', 'Series_type': 'Other', 'Series_contributor': 'Alex,,Bortvin', 'Series_sample_id': 'GSM6923088', 'Series_contact_name': 'Alex,,Bortvin', 'Series_contact_email': 'bortvin@carnegiescience.edu', 'Series_contact_department': 'Embryology', 'Series_contact_institute': 'Carnegie Institution for Science', 'Series_contact_address': '3520 San Martin drive', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21218', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE222nnn/GSE222415/suppl/GSE222415_rna_mael_vs_het_DESeq2.txt.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA922045'}\n",
      "582 PRJNA925168 GSE223168 {'Series_title': 'Arabidopsis HOT3/eIF5B1 constrains rRNA RNAi by facilitating 18S rRNA maturation during translation initiation (Ribo-Seq)', 'Series_geo_accession': 'GSE223168', 'Series_status': 'Public on Apr 19 2023', 'Series_submission_date': 'Jan 18 2023', 'Series_last_update_date': 'Apr 21 2023', 'Series_pubmed_id': '37011204', 'Series_summary': 'Ribosome biogenesis is essential for protein synthesis in gene expression. Yeast eIF5B has been shown biochemically to facilitate 18S rRNA 3’ end maturation during late-40S ribosomal subunit assembly and gate the transition from translation initiation to elongation. But the effects of eIF5B have not been studied at the genome-wide level in any organism, and 18S rRNA 3’ end maturation is poorly understood in plants. Arabidopsis HOT3/eIF5B1 was found to promote development and heat-stress acclimation by translational regulation, but its molecular function remained unknown. Here, we show that HOT3 is a late-stage ribosome biogenesis factor that facilitates 18S rRNA 3’ end processing and is a translation initiation factor that globally impacts the transition from initiation to elongation. By developing and implementing 18S-ENDseq, we revealed previously unknown events in 18S rRNA 3’ end maturation or metabolism. We quantitatively defined new processing hotspots and identified adenylation as the prevalent non-templated RNA modification at the 3’ ends of pre-18S rRNAs. Aberrant 18S rRNA maturation in hot3 further activated RNAi to generate RDR1- and DCL2/4-dependent risiRNAs mainly from a 3’ portion of 18S rRNA. We further showed that risiRNAs in hot3 were predominantly localized in ribosome-free fractions not responsible for the 18S rRNA maturation or translation initiation defects in hot3. Our study uncovered the molecular function of HOT3/eIF5B1 in 18S rRNA maturation at the late-40S assembly stage and revealed the regulatory crosstalk among ribosome biogenesis, mRNA translation initiation, and siRNA biogenesis in plants.', 'Series_overall_design': 'Comparative translation profiling analysis of Ribo-seq data for inflorescence  of WT, hot3-2, hot3-3 and HOT-EYFP/hot3-2', 'Series_type': 'Other', 'Series_contributor': 'Xuemei,,Chen', 'Series_sample_id': 'GSM6940580', 'Series_contact_name': 'Chenjiang,,You', 'Series_contact_email': 'chenjiang.you@ucr.edu, youchenjiang2003@hotmail.com', 'Series_contact_institute': 'Fudan University', 'Series_contact_address': '2005 Songhu Rd', 'Series_contact_city': 'Shanghai', 'Series_contact_state': 'Shanghai', 'Series_contact_zip/postal_code': '200438', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE223nnn/GSE223168/suppl/GSE223168_RAW.tar', 'Series_platform_id': 'GPL23157', 'Series_platform_taxid': '3702', 'Series_sample_taxid': '3702', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA925168'}\n",
      "583 PRJNA926480 GSE223465 {'Series_title': 'eIF4F complex dynamics are important for the activation of the integrated stress response', 'Series_geo_accession': 'GSE223465', 'Series_status': 'Public on Jan 25 2023', 'Series_submission_date': 'Jan 23 2023', 'Series_last_update_date': 'Jul 01 2024', 'Series_pubmed_id': '38848692', 'Series_summary': 'In response to stress, eukaryotes activate the integrated stress response (ISR) via phosphorylation of eIF2α to promote the translation of pro-survival effector genes, such as GCN4 in yeast. Complementing the ISR is the Target of Rapamycin (TOR) pathway, which regulates eIF4E function. Here we probe translational control in the absence of eIF4E in Saccharomyces cerevisiae. Intriguingly, we find that loss of eIF4E leads to de-repression of GCN4 translation. In addition, we find that de-repression of GCN4 translation is neither accompanied by eIF2α phosphorylation nor reduction in initiator ternary complex. Our data suggest that when eIF4E levels are depleted, GCN4 translation is de-repressed via a unique mechanism that may involve faster scanning by the small ribosome subunit due to increased local concentration of eIF4A. Overall, our findings suggest that relative levels of eIF4F components are key to ribosome dynamics and may play important roles in translational control of gene expression.', 'Series_overall_design': 'Genome and Transcriptome Mapped ribosome profiling and RNA-seq reads. Genome mapping was done using STAR and analyzed using FeatureCounts and python scripts. Transcriptome mapping was done using Salmon and analyzed using DESeq2.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Hani,S,Zaher', 'Series_sample_id': 'GSM6956701', 'Series_contact_name': 'Hani,,Zaher', 'Series_contact_email': 'hzaher@wustl.edu', 'Series_contact_department': 'Biology', 'Series_contact_institute': 'Washington University in St. Louis', 'Series_contact_address': '1 Brookings Drive', 'Series_contact_city': 'St Louis', 'Series_contact_state': 'MO', 'Series_contact_zip/postal_code': '63130', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE223nnn/GSE223465/suppl/GSE223465_Multivariate_Analysis.xlsx', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA926480'}\n",
      "584 PRJNA926978 GSE223615 {'Series_title': 'Ribosome biogenesis controls\\xa0suture mesenchymal stem cell fate via selective regulation ofcomplement pathway in to prevent craniosynostosis\\xa0[Ribo-seq]', 'Series_geo_accession': 'GSE223615', 'Series_status': 'Public on Dec 20 2023', 'Series_submission_date': 'Jan 24 2023', 'Series_last_update_date': 'Dec 21 2023', 'Series_pubmed_id': '37977145', 'Series_summary': 'Suture mesenchymal stem cell (MSC) drives calvarial suture development, homeostasis, and regeneration. Its loss leads to craniosynostosis, a prevailing craniofacial disorder characterized by premature suture closure. Ribosome biogenesis, historically thought to be a static house-keeping process, is now known to have tissue-specific roles. However, the functional specificity of ribosome biogenesis in suture MSCs remains largely unexplored, hampering development of therapeutic strategies for craniofacial tissue regeneration. We genetically perturb ribosome biogenesis using\\xa0Snord118, a small nucleolar RNA (snoRNA) required for ribosomal RNA (rRNA) maturation. MSC specific conditional knockout (cKO) of\\xa0Snord118\\xa0in mice leads to p53 activation, cell death, mesenchymal and MSC loss, impaired osteogenic and osteoclastic activity, resulting in suture growth and craniosynostosis defects. Using our newly established human induced pluripotent stem cell (iPSC)-derived MSCs combined with ribosome profiling, we found that\\xa0SNORD118deficiency in MSCs causes global translation dysregulations and downregulation of complement pathway, a part of innate immune system with selective but poorly characterized physiological functions in craniofacial tissue homeostasis. Overall, ribosome biogenesis controls suture MSC fate via selective regulation of complement pathway.', 'Series_overall_design': 'We performed ribosome profiling in MSCs and its snord118 mutant heterozygote. Experiments were performed in two biological duplicates.', 'Series_type': 'Other', 'Series_contributor': 'Jianfu,,Chen', 'Series_sample_id': 'GSM6965662', 'Series_contact_name': 'Supawadee,,Jariyasakulroj', 'Series_contact_email': 'jariyasa@usc.edu', 'Series_contact_phone': '2135955126', 'Series_contact_institute': 'University of Southern California', 'Series_contact_address': 'Alcazar street', 'Series_contact_city': 'Los Angeles', 'Series_contact_zip/postal_code': '90033', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE223nnn/GSE223615/suppl/GSE223615_RAW.tar', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA926978'}\n",
      "585 PRJNA927820 GSE223730 {'Series_title': 'METTL3-mediated m6A modification controls splicing factor abundance and contributes to CLL progression [Ribo-seq]', 'Series_geo_accession': 'GSE223730', 'Series_status': 'Public on Mar 03 2023', 'Series_submission_date': 'Jan 25 2023', 'Series_last_update_date': 'Jun 08 2023', 'Series_pubmed_id': '37067905', 'Series_web_link': 'https://pubmed.ncbi.nlm.nih.gov/37067905/', 'Series_summary': 'RNA splicing dysregulation is a hallmark of cancers, promoting the onset and progression of disease. In chronic lymphocytic leukemia (CLL), spliceosome mutations leading to aberrant splicing occur in ~20% of patients. However, the underlying mechanism for splicing defects in spliceosome unmutated CLL cases remains elusive. Through an integrative transcriptomic and proteomic analysis, we discover proteins involved in RNA splicing are post-transcriptionally upregulated in CLL cells, resulting in splicing dysregulation. The abundance of splicing (factors) complexes is an independent risk factor and associated with poor prognosis in CLL. Moreover, increased splicing factor expression is highly correlated with METTL3, an RNA methyltransferase that modifies N6-methyladenosine (m6A) on mRNA. METTL3 is essential for cell growth in vitro and in vivo, and controls splicing factor protein expression in a methyltransferase-dependent manner through m6A modification mediated ribosome recycling and decoding process.  Our results uncover a novel regulatory axis of METTL3 for splicing dysregulation in CLL and highlight m6A modification as a major contributor to spliceosome mutation-independent splicing defects that lead to CLL progression.', 'Series_overall_design': 'Comparative riboseq data of METTL3 knock out vs Control in HG3 cell line', 'Series_type': 'Other', 'Series_contributor': 'Meiling,,Jin', 'Series_sample_id': 'GSM6976317', 'Series_contact_name': 'Lili,,Wang', 'Series_contact_email': 'lilwang@coh.org', 'Series_contact_laboratory': \"Wang's Lab\", 'Series_contact_department': 'Systems biology', 'Series_contact_institute': 'City of Hope', 'Series_contact_address': '1218 South Fifth Ave.', 'Series_contact_city': 'Monrovia', 'Series_contact_state': 'CALIFORNIA', 'Series_contact_zip/postal_code': '91016', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE223nnn/GSE223730/suppl/GSE223730_Riboseq_translation_efficiency.txt.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA927820'}\n",
      "586 PRJNA928320 GSE223755 {'Series_title': 'Boric acid intercepts 80S ribosome migration from AUG-stop by stabilizing eRF1 at the A site', 'Series_geo_accession': 'GSE223755', 'Series_status': 'Public on Sep 19 2023', 'Series_submission_date': 'Jan 26 2023', 'Series_last_update_date': 'Sep 20 2023', 'Series_summary': 'In response to environmental changes, cells flexibly and rapidly alter gene expression, through translational controls. In plants, the translation of NIP5;1, a boric acid diffusion facilitator, is upregulated through upstream open reading frames (uORFs) that comprise only AUG and stop codons, in response to a shortage of boric acid in the environment. However, the molecular details of how the minimum uORF controls the translation of the downstream main ORF, in a boric acid concentration-dependent manner, have remained unclear. Here, combining ribosome profiling, TCP-Seq, structural analysis with cryo-electron microscopy, and biochemical assays, we showed that the 80S ribosome (80S) assembled at AUG-stop migrates into the subsequent RNA segment, followed by downstream translation initiation, and that boric acid impedes this process by the stable confinement of eukaryotic release factor 1 (eRF1) on the 80S on AUG-stop. Our results provide molecular insight into translation regulation by a minimum and environment-responsive uORF.', 'Series_overall_design': 'Translation complex profile sequencing (TCP-seq)', 'Series_type': 'Other', 'Series_contributor': 'Toru,,Fujiwara', 'Series_sample_id': 'GSM6994860', 'Series_contact_name': 'Shintaro,,Iwasaki', 'Series_contact_email': 'shintaro.iwasaki@riken.jp', 'Series_contact_laboratory': 'RNA Systems Biochemistry Laboratory', 'Series_contact_department': 'Cluster for Pioneering Research', 'Series_contact_institute': 'RIKEN', 'Series_contact_address': '2-1 Hirosawa', 'Series_contact_city': 'Wako', 'Series_contact_zip/postal_code': '351-0198', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE223nnn/GSE223755/suppl/GSE223755_RAW.tar', 'Series_platform_id': 'GPL26697', 'Series_platform_taxid': '32630', 'Series_sample_taxid': '32630', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA928320'}\n",
      "587 PRJNA932446 GSE224774 {'Series_title': 'mRNA decapping activators Pat1 and Dhh1 regulate transcript abundance and translation to tune cellular responses to nutrient availability', 'Series_geo_accession': 'GSE224774', 'Series_status': 'Public on Sep 27 2023', 'Series_submission_date': 'Feb 07 2023', 'Series_last_update_date': 'Sep 28 2023', 'Series_pubmed_id': '37439347', 'Series_summary': 'Yeast mRNA decapping-activators Pat1 and Dhh1 have been implicated in repressing translation and mRNA degardation in glucose-deprived cells, but the specific mRNAs targeted by each factor and their functions in nutrient replete cells are poorly understood. To test this, we performed ribosome profiling and RNA-Seq in WT, pat1D and pat1Ddhh1D. Further, CAGE-Seq and Rpb1-ChIP Seq were employed to determine their role in decapping mediated degradation of mRNAs', 'Series_overall_design': 'This study includes total 26 samples, 9 for ribosome profiling from triplicates of WT, pat1D and pat1Ddhh1D; 3 from  triplicates of WT. Separately, the same total RNA dhh1D, and pat1D in duplicates,  were subjected to CAGE-Seq (4 samples) and total RNA-Seq (4 samples). pat1D and dhh1D in triplicates were subjected to Rpb1-ChIP seq (6 samples).  All strains were grown in YPD at 30ºC till the OD600 ~0.6', 'Series_type': 'Other', 'Series_contributor': 'Alan,G,Hinnebusch', 'Series_sample_id': 'GSM7031918', 'Series_contact_name': 'Alan,G,Hinnebusch', 'Series_contact_email': 'alanh@mail.nih.gov', 'Series_contact_laboratory': 'Section on Nutrient Control of Gene Expression', 'Series_contact_department': 'Eunice Kennedy Shriver National Institute of Child Health and Human Development', 'Series_contact_institute': 'National Institutes of Health', 'Series_contact_address': '6 Center Drive', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'Maryland', 'Series_contact_zip/postal_code': '20892', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE224nnn/GSE224774/suppl/GSE224774_RAW.tar', 'Series_platform_id': 'GPL27812', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA932446'}\n",
      "588 PRJNA933910 GSE225095 {'Series_title': 'Decoding of translation-regulating entities reveals heterogeneous translation deficiency patterns in cellular senescence', 'Series_geo_accession': 'GSE225095', 'Series_status': 'Public on Aug 31 2023', 'Series_submission_date': 'Feb 11 2023', 'Series_last_update_date': 'Aug 31 2023', 'Series_pubmed_id': '37547972', 'Series_summary': 'Cellular senescence constitutes a generally irreversible proliferation barrier, accompanied by macromolecular damage and metabolic rewiring. Several senescence types have been identified based on the initiating stimulus, such as replicative (RS), stress-induced (SIS) and oncogene-induced senescence (OIS). Reduced protein synthesis is considered a senescence hallmark, but whether this trait pertains to various senescence subtypes and if distinct molecular mechanisms are involved remain largely unknown. Here, we analyze large published or experimentally produced RNA-seq and Ribo-seq datasets to determine whether major translation-regulating entities such as ribosome stalling, the presence of uORFs/dORFs and IRES elements may differentially contribute to translation deficiency in senescence subsets. We show that translation-regulating mechanisms may not be directly relevant to RS, however uORFs are significantly enriched in SIS. Interestingly, ribosome stalling, uORF/dORF patterns and IRES elements comprise predominant mechanisms upon OIS, strongly correlating with Notch pathway activation. Our study provides for the first time evidence that major translation dysregulation mechanisms/patterns occur during cellular senescence, but at different rates depending on the stimulus type. The degree at which those mechanisms accumulate directly correlates with translation deficiency levels. Our thorough analysis contributes to elucidating crucial and so far unknown differences in the translation machinery and ribosome biogenesis between senescence subsets.', 'Series_overall_design': 'Translation efficiency changes upon different senescent stimuli versus normal proliferating conditions in IMR 90 cell line', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Vassilis,,Gorgoulis', 'Series_sample_id': 'GSM7039263', 'Series_contact_name': 'Vassilis,George,Gorgoulis', 'Series_contact_email': 'vgorg@med.uoa.gr', 'Series_contact_laboratory': 'Laboratory of Histology-Embryology Molecular Carcinogenesis Group', 'Series_contact_department': 'Medicine', 'Series_contact_institute': 'University of Athens', 'Series_contact_address': 'Mikras Asias 75', 'Series_contact_city': 'Athens', 'Series_contact_state': 'Athens', 'Series_contact_zip/postal_code': '11527', 'Series_contact_country': 'Greece', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE225nnn/GSE225095/suppl/GSE225095_counts_RNA_Ribo.txt.gz', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA933910'}\n",
      "589 PRJNA934078 GSE225140 {'Series_title': 'METTL3 mediated m6A mRNA methylation regulates neutrophil activation through targeting TLR4 signaling (Ribosome-Seq)', 'Series_geo_accession': 'GSE225140', 'Series_status': 'Public on Mar 31 2023', 'Series_submission_date': 'Feb 12 2023', 'Series_last_update_date': 'Jun 30 2023', 'Series_pubmed_id': '36920907', 'Series_web_link': '10.1016/j.celrep.2023.112259', 'Series_summary': 'We then performed gene expression profiling analysis using data obtained from Ribosome-seq of isolated neutrophil from METTL3f/f and METTL3f/f Lyzm-Cre mice .', 'Series_overall_design': 'Comparative gene expression profiling analysis of Ribosome-seq data and RNA-seq data for METTL3flox/flox neutrophil and METTL3flox/flox-Lyzmcre neutrophil.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Jing,,Tang', 'Series_sample_id': 'GSM7040012', 'Series_contact_name': 'Shuhua,,Luo', 'Series_contact_email': 'shuhualuo@126.com', 'Series_contact_institute': 'Guangdong Medical University', 'Series_contact_address': '2 Wenming East Roadan', 'Series_contact_city': 'Zhanjiang', 'Series_contact_state': 'guangdong', 'Series_contact_zip/postal_code': '524000', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE225nnn/GSE225140/suppl/GSE225140_genes.expression.annot.xls.gz', 'Series_platform_id': 'GPL24247', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA934078'}\n",
      "590 PRJNA935940 GSE225424 {'Series_title': 'Smooth muscle cell-specific translatome profiling of mouse atherosclerosis uncovers Itih4 as a novel SMC-enriched gene', 'Series_geo_accession': 'GSE225424', 'Series_status': 'Public on Feb 08 2024', 'Series_submission_date': 'Feb 16 2023', 'Series_last_update_date': 'Feb 08 2024', 'Series_pubmed_id': '38289873', 'Series_summary': 'The development of atherosclerosis, a chronic disease characterized by the buildup of plaque in arterial walls, involves the contribution of vascular smooth muscle cells (SMCs) and SMC-derived cells. To better understand the specific role of SMCs in atherosclerosis, a transgenic mouse line expressing EGFP-tagged ribosomal protein L10a (EGFP-L10a) under the SMC-specific αSMA promoter (SMCTRAP mice) was developed, enabling translating ribosome affinity purification followed by sequencing (TRAP-Seq). SMCTRAP mice were crossed with an atherosclerosis model (SMCTRAP-Athero mice). Using TRAP-Seq and bulk RNA-Seq, the aortas of 15-month-old SMCTRAP and SMCTRAP-Athero mice were analyzed to identify atherosclerosis-induced changes in the profiles of SMC ribosome-associated RNA. The results showed high enrichment of known SMC-specific genes in the TRAP fraction, indicating the specificity of the construct. The intersection of SMC-specific genes identified by TRAP-Seq, and atherosclerosis-induced genes identified by bulk RNA-Seq revealed the contribution of SMCs to the expression of several known disease-induced genes, as well as the identification of previously unlinked genes in atherosclerosis that warrant further investigation. These findings provide new insights into the role of SMCs in atherosclerosis and may offer potential targets for future treatments of this chronic disease.', 'Series_overall_design': 'Vascular smooth muscle cell ribosome-associated RNA was extracted using immunoprecipitation (IP) enrichment, along with bulk total RNA (Input), from the thoracic aorta of atherosclerosis model (SMCTRAP-Athero) and control mice (SMCTRAP). The extracted RNA was used to identify changes in the SMC ribosome-associated RNA profiles (translatomes) in atherosclerosis.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Minna,U,Kaikkonen', 'Series_sample_id': 'GSM7122456', 'Series_contact_name': 'Aarthi,,Ravindran', 'Series_contact_email': 'aarthi.ravindran@uef.fi', 'Series_contact_phone': '0465218505', 'Series_contact_institute': 'University of Eastern Finland', 'Series_contact_address': 'Neulaniementie 2, 70150 Kuopio', 'Series_contact_city': 'Kuopio', 'Series_contact_zip/postal_code': '70150', 'Series_contact_country': 'Finland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE225nnn/GSE225424/suppl/GSE225424_count_matrix.txt.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA935940'}\n",
      "591 PRJNA937159 GSE225683 {'Series_title': 'Rapamycin-Induced Feedback Activation of mRNA Translation in Pancreatic Cancer', 'Series_geo_accession': 'GSE225683', 'Series_status': 'Public on Feb 21 2023', 'Series_submission_date': 'Feb 21 2023', 'Series_last_update_date': 'Jun 20 2023', 'Series_pubmed_id': '36900235', 'Series_summary': \"We report mTOR inhibitor Rapamycin's specific and genome-wide effect on mRNA translation. Using ribosome footprinting in pancreatic cancer cells that lack 4EBP1 expression we establish the effect of mTOR-S6-dependent mRNA translation, and identify translation programs activated following mTOR inhibition.\", 'Series_overall_design': \"We performed ribosome footprinting and RNA sequencing on six samples of PANC1 cells: three control DMSO-treated biological replicates and three biological replicates collected 60 minutes after 50nM Rapamycin treatment. Control samples are labeled as 'CT', and Rapamycin-treated samples as 'RAPA'. Data from ribosome protected RNA are labeled as 'RF', and from total RNA as 'RNA'.\", 'Series_type': 'Other', 'Series_contributor': 'Kamini,,Singh', 'Series_sample_id': 'GSM7054187', 'Series_contact_name': 'Kamini,,Singh', 'Series_contact_email': 'kamini.singh@einsteinmed.edu', 'Series_contact_phone': '7184302466', 'Series_contact_laboratory': 'Kamini Singh', 'Series_contact_department': 'Molecular Pharmacology', 'Series_contact_institute': 'Albert Einstein College of Medicine', 'Series_contact_address': '1300 Morris Park Ave, Golding 203', 'Series_contact_city': 'Bronx', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '10461', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE225nnn/GSE225683/suppl/GSE225683_RiboDiff.output.PDAC_Rapamycin.tsv.gz', 'Series_citation': 'https://doi.org/10.3390/cancers15051444', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA937159'}\n",
      "592 PRJNA940173 GSE226436 {'Series_title': 'Ribosome rescue factor PELOTA modulates translation start site choice and protein isoform levels of transcription factor C/EBPα [riboprofiling]', 'Series_geo_accession': 'GSE226436', 'Series_status': 'Public on Mar 08 2023', 'Series_submission_date': 'Mar 01 2023', 'Series_last_update_date': 'Jun 06 2024', 'Series_pubmed_id': '38803235', 'Series_summary': 'Translation initiation at alternative start sites can dynamically control the synthesis of two or more functionally distinct protein isoforms from a single mRNA. Alternate isoforms of the hematopoietic transcription factor C/EBPα produced from different start sites exert opposing effects during myeloid cell development. This alternative initiation depends on sequence features of the CEBPA transcript, including a regulatory upstream open reading frame (uORF), but the molecular basis is not fully understood. Here we identify trans-acting factors that affect C/EBPα isoform choice using a sensitive and quantitative two-color fluorescence reporter coupled with CRISPRi screening. Our screen uncovered a role for the ribosome rescue factor PELOTA (PELO) in promoting expression of the longer C/EBPα isoform, by directly removing inhibitory unrecycled ribosomes and through indirect effects mediated by the mechanistic target of rapamycin (mTOR) kinase. Our work provides further mechanistic insights into coupling between ribosome recycling and translation reinitiation in regulation of a key transcription factor, with implications for normal hematopoiesis and leukemiagenesis.', 'Series_overall_design': 'To determine the impact of reduced PELO levels on ribosome occupancy transcriptome-wide and on a CEBPA, we established a CEBPA fluorescent reporter in K562 cells. We then depleted PELO by sgRNA (CRISPRi). We determined changes in translation on our reporter (and transcriptome-wide) by ribosome profiling using the nucleases, RNAse1 and P1. A nontargeting sgRNA was used as a control and both sgRNA conditions (nontargeting and PELO KD) were performed in triplicate. Samples from each condition and replicate were also taken for matched RNA-seq gene expression analysis.', 'Series_type': 'Other', 'Series_contributor': 'Lucas,,Ferguson', 'Series_sample_id': 'GSM7074866', 'Series_contact_name': 'Nicholas,T,Ingolia', 'Series_contact_email': 'ingolia@berkeley.edu', 'Series_contact_department': 'Molecular and Cell Biology', 'Series_contact_institute': 'University of California, Berkeley', 'Series_contact_address': '1 Barker Hall #3202', 'Series_contact_city': 'Berkeley', 'Series_contact_zip/postal_code': '94709', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE226nnn/GSE226436/suppl/GSE226436_pelokd_rnase1_ribo_cds_grch38_cnt_all.csv.gz', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA940173'}\n",
      "593 PRJNA944112 GSE227197 {'Series_title': 'Translational and Transcriptional control of gene expression during serum stimulation on human RPE-1 cells', 'Series_geo_accession': 'GSE227197', 'Series_status': 'Public on Mar 17 2024', 'Series_submission_date': 'Mar 13 2023', 'Series_last_update_date': 'Mar 17 2024', 'Series_summary': 'In animal cells, growth factor signaling promotes both cell cycle progression and cell growth. To understand how the cell cycle is coupled to growth, we examined translational activities in human Retinal Pigment Epithelial cells by whole-transcriptome ribosome profiling.', 'Series_overall_design': 'We performed Ribo-Seq and RNA-Seq on human RPE-1 cells that are serum starved for 18h, then serum stimulated for 5h, 10h, and 15h with 4 biological replicates each.', 'Series_type': 'Other', 'Series_contributor': 'Stephen,M,Pronovost', 'Series_sample_id': 'GSM7093049', 'Series_contact_name': 'Bruce,,Edgar', 'Series_contact_email': 'bruce.edgar@hci.utah.edu', 'Series_contact_department': 'Department of Oncological Sciences', 'Series_contact_institute': 'University of Utah', 'Series_contact_address': '2000 Cir of Hope Dr.', 'Series_contact_city': 'Salt Lake City', 'Series_contact_state': 'UT', 'Series_contact_zip/postal_code': '84112', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE227nnn/GSE227197/suppl/GSE227197_coregtf_counts.counts.txt.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA944112'}\n",
      "594 PRJNA945876 GSE227590 {'Series_title': 'Genome-wide Translational Response of Candida albicans to Fluconazole Treatment', 'Series_geo_accession': 'GSE227590', 'Series_status': 'Public on Aug 05 2023', 'Series_submission_date': 'Mar 17 2023', 'Series_last_update_date': 'Oct 26 2023', 'Series_pubmed_id': '37610232', 'Series_summary': 'Azoles are commonly used for the treatment of fungal infections and the ability of human fungal pathogens to rapidly respond to azole treatment is critical for the development of antifungal resistance. While the role of genetic mutations, chromosomal rearrangements and transcriptional mechanisms in azole resistance has been well-characterized, very little is known about post-transcriptional and translation mechanisms that drive this process. In addition, most previous genome-wide studies have focused on transcriptional responses to azole treatment, and likely serve as an inaccurate proxies due to extensive post-transcriptional and translational regulation. In this study we use ribosome profiling to provide the first picture of the global translational response of a major human fungal pathogen, Candida albicans, to treatment with fluconazole, one of the most widely used azole drugs. We identify sets of genes showing significantly altered translational efficiency (TE), including genes associated with a variety of biological processes such as the cell cycle, DNA repair, cell wall/cell membrane biosynthesis, transport, signaling, DNA- and RNA-binding activities and protein synthesis. Importantly, while there are similarities and differences among gene categories that are regulated by fluconazole at the translational vs. transcriptional levels, we observe very little overlap among individual genes controlled by these mechanisms. Our findings suggest that C. albicans possesses distinct translational mechanisms that are important for the response to antifungal treatment, which could eventually be targeted by novel antifungal therapies.', 'Series_overall_design': 'Examination of gene expression by RNA-seq and Ribo-seq in 1 strain of Candida albicans grown under 2 conditions (YEPD + 1 ug/mL fluconazole at  30°C  (Fluconazole)) and (YEPD + DMSO vehicle only control at 30°C (No drug)). There are four biological replicates for each strain grown under each condition (16 samples total).', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Saket,,Choudhary', 'Series_sample_id': 'GSM7103619', 'Series_contact_name': 'David,,Kadosh', 'Series_contact_email': 'kadosh@uthscsa.edu', 'Series_contact_phone': '(210) 567-3976', 'Series_contact_department': 'Department of Microbiology, Immunology & Molecular Genetics', 'Series_contact_institute': 'University of Texas Health Science Center at San Antonio', 'Series_contact_address': '7703 Floyd Curl Dr., MC: 7758', 'Series_contact_city': 'San Antonio', 'Series_contact_state': 'Texas', 'Series_contact_zip/postal_code': '78229', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE227nnn/GSE227590/suppl/GSE227590_ribo_rna_counts.tsv.gz', 'Series_platform_id': 'GPL27827', 'Series_platform_taxid': '5476', 'Series_sample_taxid': '5476', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA945876'}\n",
      "595 PRJNA947525 GSE227926 {'Series_title': 'Transfer RNA pools in human cells are controlled by selective gene expression [Ribo-seq]', 'Series_geo_accession': 'GSE227926', 'Series_status': 'Public on Oct 23 2023', 'Series_submission_date': 'Mar 22 2023', 'Series_last_update_date': 'Jan 22 2024', 'Series_pubmed_id': '38191669', 'Series_summary': 'Transfer RNAs are required for translating genetic information into protein sequence. The human genome contains hundreds of tRNA genes, many of which in multiple copies. How their expression is regulated to control functional tRNA levels is unknown. Here, we combined quantitative tRNA profiling and ChIP-Seq to measure tRNA expression upon differentiation of human induced pluripotent stem cells (hiPSC) into neuronal and cardiac cells. We find that tRNA transcript pools vary substantially, while the abundance of tRNAs with distinct anticodons, which governs decoding rates, is more stable among cell types. Mechanistically, RNA Polymerase III (Pol III) samples a wide range of tRNA genes in hiPSC and becomes constrained to a housekeeping subset upon differentiation. This is mediated by diminished mTOR signaling, which activates the Pol III repressor MAF1. Our data rationalize how tRNA anticodon pools are buffered in different cellular contexts and reveal that mTOR activity drives selective tRNA expression.', 'Series_overall_design': 'Human induce pluoripotent stem cells (hiPSC), and derived neural progenitors (NPC), mature neurons, and cardiomyocytes (CM) were cultured. Subsequent chromatin immunoprecipitation DNA-sequencing (ChIP-seq) for RNA Polymerase III subunit RPC1, TFIIIB subunit BRF1, histone modifications H3K4me3 and K3K27me3, ATAC-Seq, RNA-Seq, ribosome profiling followed by sequencing (Ribo-Seq) and tRNA-Seq datasets were generated to investigate the regulation of tRNA genes upon differentiation of hiPSCs.', 'Series_type': 'Other', 'Series_contributor': 'Danny,D,Nedialkova', 'Series_sample_id': 'GSM7111084', 'Series_contact_name': 'Danny,,Nedialkova', 'Series_contact_email': 'nedialkova@biochem.mpg.de', 'Series_contact_laboratory': 'Mechanisms of Protein Biogenesis', 'Series_contact_institute': 'Max Planck Institute for Biochemistry', 'Series_contact_address': 'Am Klopferspitz 18', 'Series_contact_city': 'Planegg', 'Series_contact_state': 'Bayern', 'Series_contact_zip/postal_code': '82152', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE227nnn/GSE227926/suppl/GSE227926_codonDT_all.csv.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA947525'}\n",
      "596 PRJNA950373 GSE228561 {'Series_title': 'Rewiring of RNA methylation by the oncometabolite fumarate in renal cell carcinoma [Ribosome_Profiling]', 'Series_geo_accession': 'GSE228561', 'Series_status': 'Public on Feb 15 2024', 'Series_submission_date': 'Mar 30 2023', 'Series_last_update_date': 'Feb 16 2024', 'Series_pubmed_id': '38328795', 'Series_summary': 'Metabolic reprogramming is a hallmark of cancer that facilitates changes in many adaptive biological processes. Mutations in the tricarboxylic acid (TCA) cycle enzyme fumarate hydratase (FH) lead to fumarate accumulation and cause hereditary leiomyomatosis and renal cell cancer (HLRCC). HLRCC is a rare, inherited disease characterized by the development of non-cancerous smooth muscle tumors of the uterus and skin, and an increased risk of a highly metastatic and aggressive form of kidney cancer. Fumarate has been shown to inhibit 2-oxyglutarate-dependent dioxygenases (2OGDDs) involved in the hydroxylation of HIF1α, as well as in DNA and histone demethylation. However, the link between fumarate accumulation and changes in RNA post-transcriptional modifications has not been defined. Here, we determine the consequences of fumarate accumulation on the activity of different members of the 2OGDD family targeting RNA modifications. By evaluating multiple RNA modifications in patient-derived HLRCC cell lines, we show that mutation of FH selectively alters the activity of demethylases acting upon N6-methyladenosine (m6A), while the demethylases acting upon N1-methyladenosine (m1A) and 5-formylcytosine (f5C) in mitochondrial RNA are unaffected. The observation that metabolites may modulate specific subsets of RNA-modifying enzymes offers new insights into their gene regulatory effects and potential strategies for therapeutic intervention.', 'Series_overall_design': 'Ribosome_Profiling', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Pedro,J,Batista', 'Series_sample_id': 'GSM7124093', 'Series_contact_name': 'Pedro,J,Batista', 'Series_contact_email': 'pedro.batista@nih.gov', 'Series_contact_phone': '3014356294', 'Series_contact_laboratory': 'Cell Biology', 'Series_contact_department': 'National Cancer Institute', 'Series_contact_institute': 'National Institutes of Health', 'Series_contact_address': '37 Convent Street Bldg 37', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '20892', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE228nnn/GSE228561/suppl/GSE228561_Riboseq_counts_table_R.txt.gz', 'Series_platform_id': 'GPL30173', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA950373'}\n",
      "597 PRJNA954953 GSE229572 {'Series_title': 'Tumor-specific circRNAs elicit anti-tumor immune response via encoding cryptic peptides [Ribo-Seq]', 'Series_geo_accession': 'GSE229572', 'Series_status': 'Public on Oct 08 2023', 'Series_submission_date': 'Apr 12 2023', 'Series_last_update_date': 'Jan 25 2024', 'Series_pubmed_id': '38093017', 'Series_summary': 'By using Ribo-seq, we report tumor special antigen from breast cancer tissues and normal tissues.', 'Series_overall_design': 'Ribo-seq of breast cancer tissues and normal tissues.', 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'Xiaofeng,,Zhu', 'Series_sample_id': 'GSM7165705', 'Series_contact_name': 'Zhu,,Xiaofeng', 'Series_contact_email': 'wangjw47@mail2.sysu.edu.cn', 'Series_contact_institute': 'Sun Yat-Sen University', 'Series_contact_address': '107 Yan Jinag West Road', 'Series_contact_city': 'Guangzhou', 'Series_contact_zip/postal_code': '510120', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE229nnn/GSE229572/suppl/GSE229572_count.utr.xlsx', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA954953'}\n",
      "598 PRJNA955270 GSE229672 {'Series_title': 'Threonine fuels glioblastoma through YRDC-mediated codon-biased translational reprogramming (Ribosome profiling)', 'Series_geo_accession': 'GSE229672', 'Series_status': 'Public on Jan 06 2024', 'Series_submission_date': 'Apr 13 2023', 'Series_last_update_date': 'Apr 11 2024', 'Series_pubmed_id': '38519786', 'Series_summary': 'To understand the underlying mechanism of YRDC, an enzyme catalysing the formation of N6-threonylcarbamoyladenosine (t6A) on ANN-decoding tRNAs, and the interplay between threonine and YRDC in glioblastoma stem cells (GSCs), we carried out comparative transcriptomic analysis in patient-derived GSC456 with or without YRDC deprivation and with or without threonine restriction (control media, 800 μM threonine; threonine-restricted media, 4 μM). To further investigate the translational regulation of YRDC and threonine in GSCs, we performed ribosome profiling in GSC456 with or without YRDC deprivation and with or without threonine restriction (control media, 800 μM threonine; threonine-restricted media, 4 μM).', 'Series_overall_design': 'Comparative gene expression profiling analysis of RNA-seq data and comparative ribosome footprint analysis of ribosome profiling data for GSC456 with or without YRDC deprivation  and with or without threonine restriction.', 'Series_type': 'Other', 'Series_contributor': 'Jeremy,N,Rich', 'Series_sample_id': 'GSM7170413', 'Series_contact_name': 'Xujia,,Wu', 'Series_contact_email': 'wuxujia.sysu@gmail.com', 'Series_contact_phone': '4128051540', 'Series_contact_laboratory': \"Jeremy N. Rich's lab\", 'Series_contact_department': 'Hillman Cancer Center', 'Series_contact_institute': 'University of Pittsburgh', 'Series_contact_address': '5117 Centre Avenue', 'Series_contact_city': 'Pittsburgh', 'Series_contact_state': 'Pennsylvania', 'Series_contact_zip/postal_code': '15232', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE229nnn/GSE229672/suppl/GSE229672_RFP-counts.txt.gz', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA955270'}\n",
      "599 PRJNA955811 GSE229777 {'Series_title': 'Ribosome profiling of Aim29-knockout S.cerevisiae cells', 'Series_geo_accession': 'GSE229777', 'Series_status': 'Public on Apr 17 2023', 'Series_submission_date': 'Apr 14 2023', 'Series_last_update_date': 'Apr 17 2023', 'Series_summary': 'The aim of this study was to gain insight into the ribosome occupancy on different genes and codons in AIM29 KO cells vs. wild-type cells', 'Series_overall_design': 'Ribosome profiling in S. cerevisiae AIM29 KO vs. wild type cells', 'Series_type': 'Other', 'Series_contributor': 'Colin,C,Wu', 'Series_sample_id': 'GSM7177581', 'Series_contact_name': 'Vladimir,,Denic', 'Series_contact_email': 'vdenic@mcb.harvard.edu', 'Series_contact_laboratory': 'Vladimir Denic', 'Series_contact_department': 'Molecular and Cellular Biology', 'Series_contact_institute': 'Harvard University', 'Series_contact_address': '52 Oxford St, Room 442', 'Series_contact_city': 'Cambridge', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02138', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE229nnn/GSE229777/suppl/GSE229777_dAIM29_vs_WT_bamCompare.bw', 'Series_platform_id': 'GPL33347', 'Series_platform_taxid': '580240', 'Series_sample_taxid': '580240', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA955811'}\n",
      "600 PRJNA956494 GSE229866 {'Series_title': 'The translatome of Glioblastoma', 'Series_geo_accession': 'GSE229866', 'Series_status': 'Public on Aug 01 2023', 'Series_submission_date': 'Apr 17 2023', 'Series_last_update_date': 'Aug 01 2023', 'Series_summary': 'To identify new potential therapeutic targets for the treatment of GB we performed ribosome profiling of glioblastoma sphere cultures (GSCs) under normal and radiotherapeutic conditions.', 'Series_overall_design': 'Glioma sphere cultures were left untreated (t0), or irradiated and harvested 30 min (t1) or 60 min (t2) after irradiation', 'Series_type': 'Other', 'Series_contributor': 'Bart,A,Westerman', 'Series_sample_id': 'GSM7179287', 'Series_contact_name': 'Bart A.,,Westerman', 'Series_contact_email': 'a.westerman@amsterdamumc.nl', 'Series_contact_phone': '0031(0)20 44 49073', 'Series_contact_laboratory': 'Neuro-oncology Research Group', 'Series_contact_department': 'CCA-Neurosurgery', 'Series_contact_institute': 'Cancer Center Amsterdam CCA', 'Series_contact_address': 'De Boelelaan 1117', 'Series_contact_city': 'Amsterdam', 'Series_contact_zip/postal_code': '1081 HZ', 'Series_contact_country': 'Netherlands', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE229nnn/GSE229866/suppl/GSE229866_ribo_count_matrix1.rds.gz', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA956494'}\n",
      "601 PRJNA956972 GSE230019 {'Series_title': 'Germ-cell specific eIF4E1b regulates maternal mRNA translation to ensure zygotic genome activation [Ribo-seq]', 'Series_geo_accession': 'GSE230019', 'Series_status': 'Public on Jun 08 2023', 'Series_submission_date': 'Apr 18 2023', 'Series_last_update_date': 'Sep 07 2023', 'Series_pubmed_id': '37257918', 'Series_summary': 'Translation of maternal mRNAs is detected before transcription of zygotic genes and is essential for mammalian embryo development. How certain maternal mRNAs are selected for translation instead of degradation and how this burst of translation affects zygotic genome activation remains unknown. Using gene-edited mice, we document that the oocyte-specific eukaryotic translation initiation factor 4E family member 1b (eIF4E1b) is the regulator of maternal mRNA expression that ensures subsequent reprogramming of the zygotic genome. In oocytes, eIF4E1b binds to transcripts encoding translation machinery proteins, chromatin remodelers and reprogramming factors to promote their translation in zygotes and protect them from degradation. The protein products are thought to establish an open chromatin landscape in one-cell zygotes to enable transcription of genes required for cleavage stage development. Our results define a program for rapid resetting of the zygotic epigenome that is regulated by maternal mRNA expression and provides new insights into the mammalian maternal-to-zygotic transition.', 'Series_overall_design': 'Low input ribo-seq profiling using eggs or embryos from Eif4e1b knockout or control female mice containing HA tagged Rpl22.', 'Series_type': 'Other', 'Series_contributor': 'Jurrien,,Dean', 'Series_sample_id': 'GSM7183985', 'Series_contact_name': 'Guanghui,,Yang', 'Series_contact_email': 'guanghui.yang@nih.gov', 'Series_contact_laboratory': 'Laboratory of Cellular and Developmental Biology', 'Series_contact_department': 'National Institute of Diabetes and Digestive and Kidney Diseases', 'Series_contact_institute': 'National Institutes of Health', 'Series_contact_address': '50 South Drive', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'Maryland', 'Series_contact_zip/postal_code': '20892', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE230nnn/GSE230019/suppl/GSE230019_gene_count_matrix.csv.gz', 'Series_platform_id': 'GPL24247', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA956972'}\n",
      "602 PRJNA957283 GSE230043 {'Series_title': 'Human SAMD9 is a Virus-Activatable Anticodon Nuclease Inhibiting Codon-Specific Protein Synthesis', 'Series_geo_accession': 'GSE230043', 'Series_status': 'Public on Jun 07 2023', 'Series_submission_date': 'Apr 19 2023', 'Series_last_update_date': 'Sep 06 2023', 'Series_pubmed_id': '37285440', 'Series_web_link': 'https://doi.org/10.1126/sciadv.adh8502', 'Series_summary': 'Transfer RNAs (tRNAs) are one of the most conserved components of protein synthesis machinery. An effective strategy for microbes to defend against viruses or competitors is to employ anticodon nucleases (ACNases) to deplete essential tRNAs and thereby shut off global protein synthesis, but this strategy has not been observed in multicellular eukaryotes. Here, we report that human SAMD9 is a virus-activatable ACNase that specifically depletes phenylalanine tRNA (tRNAPhe), causing codon-specific ribosomal pausing and inducing stress signaling. SAMD9 ACNase can be activated by poxvirus infection or by SAMD9 mutations associated with a spectrum of developmental or immunological disorders, implicating tRNAPhe deficiency as a major toxic effect in these human diseases and suggesting a new therapeutic strategy. The specificity of the ACNase, located to an N-terminal effector domain of SAMD9, is largely determined by a eukaryotic tRNAPhe specific 2’-O-methylation at the wobble position, making eukaryotic tRNAPhe a universal but specific target for SAMD9. The SAMD9 ACNase differs from the microbial ACNases in structure and substrate specificity, suggesting convergent evolution resulted in a universal immune defense strategy targeting tRNAs.', 'Series_overall_design': 'tRNA-seq, Ribo-seq and RNA-seq. For tRNA-seq, HeLa cells were uninfected or infected with either WT or vK1-C7- VACV for 8 hours. tRNAs were isolated from total RNAs resolved on a denaturing gel, untreated or treated with a tRNA demethylase and subjected to tRNA-seq. For Ribo-seq and RNA-seq,  BT20 cells transduced with SAMD9-R1293W were uninduced or induced for SAMD9-R1293W expression.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Yuanhui,,Mao', 'Series_sample_id': 'GSM7184371', 'Series_contact_name': 'Saori,,Uematsu', 'Series_contact_email': 'su64@cornell.edu', 'Series_contact_institute': 'Cornell University', 'Series_contact_address': '526 Campus Road, Cornell University', 'Series_contact_city': 'Ithaca', 'Series_contact_state': 'New York', 'Series_contact_zip/postal_code': '14853', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE230nnn/GSE230043/suppl/GSE230043_RPKM_TABLE.tsv.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA957283'}\n",
      "603 PRJNA957808 GSE230189 {'Series_title': 'Nuclear release of eIF1 globally increases translation initiation stringency during mitosis', 'Series_geo_accession': 'GSE230189', 'Series_status': 'Public on Dec 02 2023', 'Series_submission_date': 'Apr 20 2023', 'Series_last_update_date': 'Dec 04 2023', 'Series_summary': 'Regulated translation initiation has the potential to reshape the proteome, but conditions under which start codon selection is altered remain poorly defined. Here, using global translation initiation site profiling, we reveal widespread changes in start codon selection during the mammalian cell cycle. Low-efficiency initiation sites are preferentially repressed in mitosis, resulting in changes in the relative translation of thousands of annotated proteins, alternative translational isoforms, and uORFs. This increased mitotic translational stringency results from the cytoplasmic release of a nuclear pool of eIF1 upon nuclear envelope breakdown. Increased eIF1-40S ribosome interactions repress suboptimal initiation sites to rewire the mitotic proteome. Selectively depleting the nuclear pool of eIF1 eliminates changes to translational stringency during mitosis, resulting in substantially increased cell death following an extended mitotic delay induced by anti-mitotic chemotherapeutics. Thus, cells globally control translation initiation stringency to alter their physiology with critical roles during the mammalian cell cycle.', 'Series_overall_design': 'Paired translation initiation site profiling, elongating ribosome profiling, and RNA sequencing data for synchronized interphase, mitotic arrested, and cycling mitotic HeLa cells. Biological replicates were defined as separate cell synchronization, lysate preparation, and fractionation. N=2 biological replicates were performed each cell cycle stage.', 'Series_type': 'Other', 'Series_contributor': 'Iain,M,Cheeseman', 'Series_sample_id': 'GSM7191448', 'Series_contact_name': 'Jimmy,,Ly', 'Series_contact_laboratory': 'Iain Cheeseman', 'Series_contact_department': 'Biology', 'Series_contact_institute': 'Whitehead Institute', 'Series_contact_address': '455 main street', 'Series_contact_city': 'Cambridge', 'Series_contact_state': 'Massachusetts', 'Series_contact_zip/postal_code': '02142', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE230nnn/GSE230189/suppl/GSE230189_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA957808'}\n",
      "604 PRJNA967640 GSE231758 {'Series_title': 'Dysregulation of amino acid metabolism upon rapid depletion of cap-binding protein eIF4E [RNA-Seq]', 'Series_geo_accession': 'GSE231758', 'Series_status': 'Public on May 09 2023', 'Series_submission_date': 'May 05 2023', 'Series_last_update_date': 'Aug 10 2023', 'Series_pubmed_id': '37214807', 'Series_summary': 'Protein synthesis is metabolically costly, and the level of translation must match nutrient availability and cellular needs. Overall protein synthesis levels are modulated by regulating translation initiation. The cap-binding protein eIF4E—the earliest contact between mRNAs and the translation machinery—serves as one point of control, but its contributions to mRNA-specific translation regulation remain poorly understood. We acutely depleted eIF4E, which is essential in budding yeast, and observed surprisingly modest effects on cell growth and protein synthesis. Long-lived transcripts were downregulated, likely reflecting accelerated turnover, and the strongest gene-specific effects arose as secondary effects of reduced protein biosynthesis on amino acid pools. Futile cycles of amino acid synthesis and degradation were accompanied by translational activation of GCN4, which is typically induced by amino acid starvation. We further identified translational tuning of PCL5, a negative regulator of Gcn4, that provides a consistent protein-to-mRNA ratio under varying translation environments. This translational control depended in part on a uniquely long poly-(A) tract in the PCL5 5’ UTR and on poly-(A) binding protein. These results highlight the intricate interplay between translation, amino acid homeostasis, and gene regulation and uncover new layers of feedback control in cellular response to stress and nutrient availability.', 'Series_overall_design': 'To determine the impact of reduced eIF4E levels on ribosome occupancy transcriptome-wide, we established a eIF4E-AID (auxin inducible degron) yeast strain. We then depleted eIF4E by adding IAA for either 1 or 8 hours. We determined changes in translation and RNA abundance by ribosome profiling and RNA-seq, respectively. A DMSO treated sample was used as a control and both conditions (IAA and DMSO) were performed in duplicate.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Nicholas,J,McGlincy', 'Series_sample_id': 'GSM7300140', 'Series_contact_name': 'Paige,D,Diamond', 'Series_contact_email': 'paige.diamond@berkeley.edu', 'Series_contact_laboratory': 'Ingolia', 'Series_contact_department': 'MCB', 'Series_contact_institute': 'University of California - Berkeley', 'Series_contact_address': 'Barker Hall', 'Series_contact_city': 'Berkeley', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94720', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE231nnn/GSE231758/suppl/GSE231758_RAW.tar', 'Series_platform_id': 'GPL27812', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA967640'}\n",
      "605 PRJNA967742 GSE231820 {'Series_title': 'Circadian ribosome profiling reveals a role for the Period2 upstream opening reading frame in sleep.', 'Series_geo_accession': 'GSE231820', 'Series_status': 'Public on Sep 13 2023', 'Series_submission_date': 'May 05 2023', 'Series_last_update_date': 'Dec 13 2023', 'Series_pubmed_id': '37769257', 'Series_web_link': 'https://www.pnas.org/doi/10.1073/pnas.2214636120', 'Series_summary': \"Many mammalian proteins have circadian cycles of production and degradation, and many of these rhythms are altered post-transcriptionally. We used ribosome profiling to examine post-transcriptional control of circadian rhythms by quantifying RNA translation in the liver over a 24-h period from circadian-entrained mice transferred to constant darkness conditions and by comparing ribosome binding levels to protein levels for 16 circadian proteins. We observed large differences in ribosome binding levels compared to protein levels, and we observed delays between peak ribosome binding and peak protein abundance. We found extensive binding of ribosomes to upstream open reading frames (uORFs) in circadian mRNAs, including the core clock gene Period2 (Per2). An increase in the number of uORFs in the 5'UTR was associated with a decrease in ribosome binding in the main coding sequence and a reduction in expression of synthetic reporter constructs. Mutation of the Per2 uORF increased luciferase and fluorescence reporter expression in 3T3 cells without altering the phase or period and increased luciferase expression in PER2:LUC MEF cells. Mutation of the Per2 uORF in mice increased PER2 protein levels and reduced total sleep time compared to that in wild-type mice. These results suggest that post-transcriptional processes shape translation of mRNA transcripts, which can impact physiological rhythms and sleep.\", 'Series_overall_design': 'Ribosome profiling and RNA sequencing of three C57BL/6N mouse livers from wild-type and Per2uORF mutant mice', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Hiroki,R,Ueda', 'Series_sample_id': 'GSM7304551', 'Series_contact_name': 'Arthur,,Millius', 'Series_contact_email': 'arthur.millius@ifrec.osaka-u.ac.jp', 'Series_contact_phone': '+81-06-6879-8366', 'Series_contact_laboratory': 'Systems Immunology', 'Series_contact_department': 'iFReC', 'Series_contact_institute': 'Osaka University', 'Series_contact_address': 'iFReC 6F', 'Series_contact_city': 'Suita', 'Series_contact_state': 'Osaka', 'Series_contact_zip/postal_code': '565-0871', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE231nnn/GSE231820/suppl/GSE231820_DatasetS4.xlsx', 'Series_platform_id': 'GPL24247', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA967742'}\n",
      "606 PRJNA970317 GSE231964 {'Series_title': 'Human DDX6 regulates translation and decay of inefficiently translated mRNAs', 'Series_geo_accession': 'GSE231964', 'Series_status': 'Public on Jan 23 2024', 'Series_submission_date': 'May 08 2023', 'Series_last_update_date': 'Aug 08 2024', 'Series_pubmed_id': '38989862', 'Series_web_link': 'https://doi.org/10.7554/eLife.92426.1', 'Series_summary': 'Recent findings indicate that the translation elongation rate influences mRNA stability. One of the factors that has been implicated in this link between mRNA decay and translation speed is the yeast DEAD-box helicase Dhh1p. Here, we demonstrate that the human ortholog of Dhh1p, DDX6, triggers deadenylation-dependent decay of inefficiently translated mRNAs in human cells. DDX6 interacts with the ribosome through the Phe-Asp-Phe (FDF) motif in its RecA2 domain. Furthermore, RecA2-mediated interactions and ATPase activity are both required for DDX6 to destabilize inefficiently translated mRNAs. Using ribosome profiling and RNA sequencing, we identified two classes of endogenous mRNAs that are regulated in a DDX6-dependent manner. The identified targets are either translationally regulated or regulated at the steady-state-level and either exhibit signatures of poor overall translation or of locally reduced ribosome translocation rates. Transferring the identified sequence stretches into a reporter mRNA caused translation- and DDX6-dependent degradation of the reporter mRNA. In summary, these results identify DDX6 as a crucial regulator of mRNA translation and decay triggered by slow ribosome movement and provide insights into the mechanism by which DDX6 destabilizes inefficiently translated mRNAs.', 'Series_overall_design': 'Two cell types with two biological replicates analysed using RNA-Seq and Ribo-Seq', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Chung-Te,,Chang', 'Series_sample_id': 'GSM7306535', 'Series_contact_name': 'Ramona,,Weber', 'Series_contact_email': 'ramona.weber@uzh.ch', 'Series_contact_department': 'Institute for Regenerative Medicine', 'Series_contact_institute': 'University of Zurich', 'Series_contact_address': 'Wagistrasse 12', 'Series_contact_city': 'Schlieren', 'Series_contact_zip/postal_code': '8952', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE231nnn/GSE231964/suppl/GSE231964_readcounts_Ribo.txt.gz', 'Series_platform_id': 'GPL21290', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA970317'}\n",
      "607 PRJNA970321 GSE231968 {'Series_title': 'Ribosome profiling experiment characterising early response of Lactococcus Lactis infected in sk1 bacteriophage', 'Series_geo_accession': 'GSE231968', 'Series_status': 'Public on Feb 26 2024', 'Series_submission_date': 'May 08 2023', 'Series_last_update_date': 'Mar 15 2024', 'Series_pubmed_id': '38451091', 'Series_summary': 'Time-series ribosome profiling experiment. It consists of mock and sk1 bacteriophage infected samples of Lactococcus Lactis at three different timepoints; 2,5 and 15 minutes post infection', 'Series_overall_design': 'Time-series ribosome profiling experiment. infection', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Martina,M,Yordanova', 'Series_sample_id': 'GSM7306561', 'Series_contact_name': 'Martina,,Yordonova', 'Series_contact_department': 'School of Biochemistry and Cell Biology', 'Series_contact_institute': 'University College Cork', 'Series_contact_address': 'Western road', 'Series_contact_city': 'Cork', 'Series_contact_zip/postal_code': 'T12 XF62', 'Series_contact_country': 'Ireland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE231nnn/GSE231968/suppl/GSE231968_Supplementary_table.xls.gz', 'Series_platform_id': 'GPL23778', 'Series_platform_taxid': '1358', 'Series_sample_taxid': '1358', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA970321'}\n",
      "608 PRJNA971919 GSE232395 {'Series_title': \"Study Nocte's functions in transcription and translation in Drosophila eye discs\", 'Series_geo_accession': 'GSE232395', 'Series_status': 'Public on Nov 10 2023', 'Series_submission_date': 'May 12 2023', 'Series_last_update_date': 'Feb 09 2024', 'Series_pubmed_id': '38000373', 'Series_web_link': 'https://pubmed.ncbi.nlm.nih.gov/38000373/', 'Series_summary': \"Nocte is an RNA binding protein but its functions in Drosophila eye development is unknown. We found that knockdown of nocte by RNAi in Drosophila eyes leads to small and rough eye phenotypes. We detected nocte's functions at mRNA levels by doing RNA-seq with purified polyA mRNA. We detected nocte's functions in mRNA translation by doing Ribo-seq and total RNA-seq to calculate the translation efficiencies.\", 'Series_overall_design': 'For polyA RNA-seq, 20 pairs of late L3 eye discs were used in one repeat, and three repeats for WT and ey>nocte-RNAi eye discs were done. For Ribo-seq, 50 pairs of late L3 eye discs were used in one repeat, and three repeats for WT and ey>nocte-RNAi eye discs were done. The total RNAs from two repeats of WT and ey>nocte-RNAi samples for Ribo-seq were used for total RNA-seq. The samples in total RNA-seq were treated with rRNA depletion and submitted for RNA-seq.', 'Series_type': 'Other', 'Series_contributor': 'Weidong,,Wang', 'Series_sample_id': 'GSM7730633', 'Series_contact_name': 'Weidong,,Wang', 'Series_contact_email': 'wangw@grc.nia.nih.gov', 'Series_contact_phone': '4104548418', 'Series_contact_laboratory': 'LGG', 'Series_contact_institute': 'National Institute of Aging', 'Series_contact_address': '251 Bayview Blvd BRC, Suite 100 Rm10B014', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21224', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE232nnn/GSE232395/suppl/GSE232395_RAW.tar', 'Series_platform_id': 'GPL30203', 'Series_platform_taxid': '7227', 'Series_sample_taxid': '7227', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA971919'}\n",
      "609 PRJNA975827 GSE233297 {'Series_title': 'RNA-Seq and Ribo-Seq analyses of the study \"tRNA epitranscriptome determines pathogenicity of the opportunistic pathogen Pseudomonas aeruginosa\"', 'Series_geo_accession': 'GSE233297', 'Series_status': 'Public on Feb 18 2024', 'Series_submission_date': 'May 24 2023', 'Series_last_update_date': 'May 27 2024', 'Series_pubmed_id': '38451943', 'Series_summary': 'The success of bacterial pathogens depends on the coordinated expression of virulence determinants. Regulatory circuits that drive pathogenesis are complex, multilayered, and incompletely understood. Here, we reveal that alterations in tRNA modifications define pathogenic phenotypes in the opportunistic pathogen Pseudomonas aeruginosa. We demonstrate that the enzymatic activity of GidA leads to the introduction of a carboxymethylaminomethyl modification in selected tRNAs. Modifications at the wobble uridine base (cmnm5U34) of the anticodon drives translation of transcripts containing rare codons. Specifically, in P. aeruginosa the presence of GidA-dependent tRNA modifications modulates expression of genes encoding virulence regulators, leading to a cellular proteomic shift toward pathogenic and well-adapted physiological states. Our approach of profiling the consequences of chemical tRNA modifications is general in concept. It provides a paradigm of how environmentally driven tRNA modifications govern gene expression programs and regulate phenotypic outcomes responsible for bacterial adaption to challenging habitats prevailing in the host niche.', 'Series_overall_design': 'Pseudomonas aeruginosa UCBPP-PA14 mutant strains were planktonically cultured in LB to an OD600 of 0.4 (ribo seq experiment) or 2 (RNA seq experiment).domonas aeruginosa UCBPP-PA14 wild type and mutants strains cultivated in LB', 'Series_type': 'Other', 'Series_contributor': 'Susanne,,Haeussler', 'Series_sample_id': 'GSM7421817', 'Series_contact_name': 'Susanne,,Häußler', 'Series_contact_email': 'Susanne.Haeussler@helmholtz-hzi.de', 'Series_contact_department': 'Molecular Bacteriology', 'Series_contact_institute': 'Twincore', 'Series_contact_address': 'Feodor-Lynen-Str. 7', 'Series_contact_city': 'Hanover', 'Series_contact_zip/postal_code': '30625', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE233nnn/GSE233297/suppl/GSE233297_RAW.tar', 'Series_platform_id': 'GPL27892', 'Series_platform_taxid': '208963', 'Series_sample_taxid': '208963', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA975827'}\n",
      "610 PRJNA977673 GSE233736 {'Series_title': 'Nat10 is involved in myocardial remodeling through ac4C-mediated transcriptomic regulation', 'Series_geo_accession': 'GSE233736', 'Series_status': 'Public on Oct 31 2023', 'Series_submission_date': 'May 30 2023', 'Series_last_update_date': 'Nov 15 2023', 'Series_summary': 'Conclusions: Therefore, our data suggest that modulating epitranscriptomic processes, such as ac4C acetylation through NAT10, may be a promising therapeutic target against cardiac remodeling.', 'Series_overall_design': 'We performed ac4C-seq from human heart samples (six participants per group) and four mice groups (AAV9:CONi, AAV9:NAT10i, Ang II, and Ang II +AAV9:NAT10i, with three biological repeats per group) in Model 2 by Epibiotek (Guangzhou, China). SLAM-seq was performed to measure the effect of NAT10 on mRNA half-life transcriptome-wide in NMCMs and NMCFs (two biological repeats per group) with Epi? SLAM-seq Library Prep Kit (R202208-2, Epibiotek, China) by Epibiotek (Guangzhou, China) according to the manufacturer?s instructions. Ribo-seq was performed from three mice groups (AAV9:CONi, Ang II, and Ang II +AAV9:NAT10i, with three biological repeats per group) by Epibiotek (Guangzhou, China). RNA-seq was performed from human heart samples (six biological repeats per group) and four mice groups (AAV9:Coni, AAV9:NAT10i, Ang II, And Ang II +AAV9:NAT10i, With Three Biological Repeats Per Group) in Model 2 by Epibiotek (Guangzhou, China).', 'Series_type': 'Other', 'Series_contributor': 'Jing,,Shi', 'Series_sample_id': 'GSM7892737', 'Series_contact_name': 'Chuanxi,,Yang', 'Series_contact_email': '2205515@tongji.edu.cn', 'Series_contact_department': 'Department of Cardiology', 'Series_contact_institute': 'School of Medicine, Southeast University', 'Series_contact_address': '300 Guangzhou Road, 210029, Nanjing, PR China', 'Series_contact_city': 'Nanjing', 'Series_contact_state': 'China', 'Series_contact_zip/postal_code': '211300', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE233nnn/GSE233736/suppl/GSE233736_mRNA-seq_9_mouse_All.count.txt.gz', 'Series_platform_id': 'GPL24247', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA977673'}\n",
      "611 PRJNA982760 GSE234687 {'Series_title': 'A snoRNA–tRNA modification network governs codon-biased cellular states [RNAseq & Riboseq]', 'Series_geo_accession': 'GSE234687', 'Series_status': 'Public on Sep 21 2023', 'Series_submission_date': 'Jun 12 2023', 'Series_last_update_date': 'Dec 25 2023', 'Series_pubmed_id': '37792516', 'Series_web_link': 'https://pubmed.ncbi.nlm.nih.gov/37792516/', 'Series_summary': 'antioxidant. Loss of D97/D133 snoRNAs in human HEK293 cells reduced target tRNA levels and induced codon adaptation of the transcriptome and translatome. Both D97/D133 single and double knockouts in HEK293 cells suppress Met-enriched proliferation-related gene expression programs, including, translation, splicing, and mitochondrial energy metabolism, and promote Met-depleted programs related to development, differentiation, and morphogenesis. In a mouse embryonic stem cell model of development, knockdown and knockout of D97/D133 promote differentiation to mesoderm and endoderm fates, such as cardiomyocytes, without compromising pluripotency, consistent with the enhanced development-related gene expression programs in human cells. This work solves a decade-old mystery about orphan snoRNAs and reveals a function of snoRNAs in controlling the codon-biased dichotomous cellular states of proliferation and development.', 'Series_overall_design': 'Total RNA sequencing was performed to analyze comparative gene expression profiles. Small RNA sequencing was performed to analyze the tRNA fragmentation after loss of the snoRNA-guided 2’-O-methylation modifications on tRNA. Ribosome profiling sequencing (ribo-seq) was performed to analyze the cause of reduced translation after CRISPR knockout of the D97/D133 family of snoRNAs,', 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'Jianhui,,Bai', 'Series_sample_id': 'GSM7473345', 'Series_contact_name': 'Minjie,,Zhang', 'Series_contact_email': 'minjiezhang123@gmail.com', 'Series_contact_phone': '+86-18822329402', 'Series_contact_laboratory': 'Minjie Zhang', 'Series_contact_department': 'Department of Bioinformatics', 'Series_contact_institute': 'Tianjin Medical University', 'Series_contact_address': 'No22.Qixiangtai Rd.,Heping Dist', 'Series_contact_city': 'Tianjin', 'Series_contact_zip/postal_code': '300070', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE234nnn/GSE234687/suppl/GSE234687_totalRNAseq_mES_RPM.txt.gz', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA982760'}\n",
      "612 PRJNA985729 GSE235348 {'Series_title': 'Essential roles of RNA cap-proximal ribose methylation in mammalian embryonic development and fertility', 'Series_geo_accession': 'GSE235348', 'Series_status': 'Public on Jul 14 2023', 'Series_submission_date': 'Jun 20 2023', 'Series_last_update_date': 'Jul 14 2023', 'Series_pubmed_id': '37436893', 'Series_summary': 'Eukaryotic RNA pol II transcripts are capped at the 5′ end by the methylated guanosine (m7G) moiety. In higher eukaryotes, CMTR1 and CMTR2 catalyse cap-proximal ribose methylations on the first (cap1) and second (cap2) nucleotides, respectively. These modifications mark RNAs as ‘self’, blocking the activation of the innate immune response pathway. Here we show that loss of mouse Cmtr1 or Cmtr2 leads to embryonic lethality, with non-overlapping sets of transcripts being regulated, without activation of the interferon pathway. In contrast, Cmtr1 mutant adult mouse livers exhibit a chronic activation of the interferon pathway, with multiple interferon-stimulated genes (ISGs) being expressed. Conditional deletion of Cmtr1 in the male germline leads to infertility, while global translation is unaffected in the Cmtr1 mutant mouse liver and human cells. Thus, mammalian cap1 and cap2 modifications have an essential role in embryonic gene regulation that is independent of their established role in evading the innate immune system.', 'Series_overall_design': 'RNA sequencing was applied to study the gene expression differences between Cmtr1+/+ and Cmtr1-/- mouse embryos (E6.5 and E7.5), Cmtr1+/loxP; AlbCreERT2+/- (control) and Cmtr1-/loxP; AlbCreERT2+/- (Cmtr1 cKO) mouse livers, CMTR1+/+ and CMTR1-/- HAP1 cells, Cmtr2+/+ and Cmtr2-/- mouse embryos (E6.5 and E7.5). Ribosome profiling was used to assess translation efficiency differences between Cmtr1+/loxP; AlbCreERT2+/- (control) and Cmtr1-/loxP; AlbCreERT2+/- (Cmtr1 cKO) mouse livers, CMTR1+/+ and CMTR1-/- HAP1 cells.', 'Series_type': 'Other', 'Series_contributor': 'Ramesh,S,Pillai', 'Series_sample_id': 'GSM7500830', 'Series_contact_name': 'Ramesh,,Pillai', 'Series_contact_email': 'ramesh.pillai@unige.ch', 'Series_contact_department': 'Department of Molecular Biology', 'Series_contact_institute': 'University of Geneva', 'Series_contact_address': '30, Quai Ernest-Ansermet', 'Series_contact_city': 'Gneveva', 'Series_contact_zip/postal_code': 'CH-1211', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE235nnn/GSE235348/suppl/GSE235348_cKO_vs_ctrl_male_D6_deseq_sva.xlsx', 'Series_platform_id': 'GPL21103', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA985729'}\n",
      "613 PRJNA994913 GSE237412 {'Series_title': 'RNA helicase Ighmbp2 regulates THO complex to ensure cellular mRNA homeostasis', 'Series_geo_accession': 'GSE237412', 'Series_status': 'Public on Jan 18 2024', 'Series_submission_date': 'Jul 14 2023', 'Series_last_update_date': 'Apr 18 2024', 'Series_pubmed_id': '38368610', 'Series_summary': 'RNA helicases constitute a large protein family implicated in cellular RNA homeostasis and disease development. Here we show that the RNA helicase Ighmbp2, linked to the neuromuscular disorder SMARD1 (DSMA1) associates with polysomes and impacts on translation of cellular mRNAs containing short, GC-rich and highly structured 5’UTRs. Absence of Ighmbp2 causes ribosome stalling at the start codon of target mRNAs, leading to their reduced translation efficiency. The main mRNA targets of Ighmbp2-mediated regulation encode for components of the THO complex that links mRNA production and nuclear export. Accordingly, failure of Ighmbp2 regulation of the THO complex causes perturbations of the cellular transcriptome and its encoded proteome. Ablation of essential THO complex subunits phenocopies these perturbations. Thus, Ighmbp2 is an upstream regulator of the THO complex that affects cellular mRNA homeostasis. Of note, Ighmbp2 dependent regulation of the THO complex is also observed in astrocytes derived from SMARD1 patients, suggesting that de-regulated mRNA metabolism contributes to SMARD1 etiology and offers new avenues for developing alternative therapeutics to treating SMARD1.', 'Series_overall_design': \"To investigate the effect of Ighmbp2 on translation, we performed RNASeq and ribosome profiling after knockdown of Ighmbp2. Additionally, to confirm ability of Ighmbp2's targets THOC to phenocopy Ighmbp2 deficiency, we performed knockdown of THOC1 or THOC2 or both and compared the transcriptome of these cells to control and Ighmbp2 knockdown conditions.\", 'Series_type': 'Other', 'Series_contributor': 'Utz,,Fischer', 'Series_sample_id': 'GSM7611218', 'Series_contact_name': 'Florian,,Erhard', 'Series_contact_email': 'Florian.Erhard@uni-wuerzburg.de', 'Series_contact_department': 'Institut für Virologie und Immunbiologie', 'Series_contact_institute': 'Julius-Maximilians-Universität Würzburg', 'Series_contact_address': 'Versbacher Str. 7', 'Series_contact_city': 'Würzburg', 'Series_contact_zip/postal_code': '97078', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE237nnn/GSE237412/suppl/GSE237412_RNA.96h.counts.tsv.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA994913'}\n",
      "614 PRJNA109405 GSE13750 {'Series_title': 'Genome-Wide Analysis In Vivo of Translation with Nucleotide Resolution Using Ribosome Profiling', 'Series_geo_accession': 'GSE13750', 'Series_status': 'Public on Feb 12 2009', 'Series_submission_date': 'Nov 26 2008', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '19213877', 'Series_summary': 'Techniques for systematically monitoring protein translation have lagged far behind methods for measuring mRNA levels. Here we present a ribosome profiling strategy, based on deep sequencing of ribosome protected mRNA fragments, that enables genome-wide investigation of translation with sub-codon resolution. We used this technique to monitor translation in budding yeast under both rich and starvation conditions. These studies defined the protein sequences being translated and found extensive translational control both for determining absolute protein abundance and for responding to environmental stress. We also observed distinct phases during translation involving a large decrease in ribosome density going from early to late peptide elongation as well as wide-spread, regulated initiation at non-AUG codons. Ribosome profiling is readily adaptable to other organisms, making high-precision investigation of protein translation experimentally accessible.', 'Series_overall_design': 'Examine replicates of ribosome footprints and mRNA abundance in biological replicates of log-phase growth and acute amino acid starvation', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Sina,,Ghaemmaghami', 'Series_sample_id': 'GSM346122', 'Series_contact_name': 'Nicholas,T,Ingolia', 'Series_contact_email': 'ingolia@berkeley.edu, nick@ingolia.org', 'Series_contact_phone': '510 664 7071', 'Series_contact_laboratory': 'Ingolia', 'Series_contact_department': 'Molecular and Cell Biology', 'Series_contact_institute': 'University of California, Berkeley', 'Series_contact_address': '1 Barker Hall # 3202', 'Series_contact_city': 'Berkeley', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94720', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE13nnn/GSE13750/suppl/GSE13750_RAW.tar', 'Series_platform_id': 'GPL9134', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA109405'}\n",
      "615 PRJNA129385 GSE21992 {'Series_title': 'Analysis of HeLa cells after transfection with miR-1 or miR-155, by ribosome profiling and mRNA-Seq', 'Series_geo_accession': 'GSE21992', 'Series_status': 'Public on Aug 03 2010', 'Series_submission_date': 'May 25 2010', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '20703300', 'Series_summary': 'MicroRNAs (miRNAs) are endogenous ~22-nucleotide RNAs that mediate important gene-regulatory events by pairing to the mRNAs of protein-coding genes to direct their repression. Repression of these regulatory targets leads to decreased translational efficiency and/or decreased mRNA levels, but the relative contributions of these two outcomes have been largely unknown, particularly for endogenous targets expressed at low-to-moderate levels. Here, we use ribosome profiling to measure the overall effects on protein production and compare these to simultaneously measured effects on mRNA levels. For both ectopic and endogenous miRNA regulatory interactions, lowered mRNA levels account for most (≥84%) of the decreased protein production. These results show that changes in mRNA levels closely reflect the impact of miRNAs on gene expression and indicate that destabilization of target mRNAs is the predominant reason for reduced protein output.', 'Series_overall_design': 'mir1_summaryTable.txt: log2 fold changes (miR-1-transfected versus mock-transfected HeLa cells, 32hr).', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'D,P,Bartel', 'Series_sample_id': 'GSM546931', 'Series_contact_name': 'Huili,,Guo', 'Series_contact_email': 'hguo@nus.edu.sg', 'Series_contact_institute': 'National University of Singapore', 'Series_contact_address': '28 Medical Drive', 'Series_contact_city': 'Singapore', 'Series_contact_zip/postal_code': '117456', 'Series_contact_country': 'Singapore', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE21nnn/GSE21992/suppl/GSE21992_mir1_summaryTable.txt.gz', 'Series_platform_id': 'GPL9115', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA129385'}\n",
      "616 PRJNA129411 GSE22001 {'Series_title': 'Analysis of mir-223 knockout cultured neutrophils versus wild-type cultured neutrophils, by ribosome profiling and mRNA-Seq', 'Series_geo_accession': 'GSE22001', 'Series_status': 'Public on Aug 03 2010', 'Series_submission_date': 'May 26 2010', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '20703300', 'Series_summary': 'MicroRNAs (miRNAs) are endogenous ~22-nucleotide RNAs that mediate important gene-regulatory events by pairing to the mRNAs of protein-coding genes to direct their repression. Repression of these regulatory targets leads to decreased translational efficiency and/or decreased mRNA levels, but the relative contributions of these two outcomes have been largely unknown, particularly for endogenous targets expressed at low-to-moderate levels. Here, we use ribosome profiling to measure the overall effects on protein production and compare these to simultaneously measured effects on mRNA levels. For both ectopic and endogenous miRNA regulatory interactions, lowered mRNA levels account for most (≥84%) of the decreased protein production. These results show that changes in mRNA levels closely reflect the impact of miRNAs on gene expression and indicate that destabilization of target mRNAs is the predominant reason for reduced protein output.', 'Series_overall_design': 'mir223_summaryTable.txt: log2 fold changes (miR-223-knockout versus wild-type neutrophils).', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'D,P,Bartel', 'Series_sample_id': 'GSM546990', 'Series_contact_name': 'Huili,,Guo', 'Series_contact_email': 'hguo@nus.edu.sg', 'Series_contact_institute': 'National University of Singapore', 'Series_contact_address': '28 Medical Drive', 'Series_contact_city': 'Singapore', 'Series_contact_zip/postal_code': '117456', 'Series_contact_country': 'Singapore', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE22nnn/GSE22001/suppl/GSE22001_mir223_summaryTable.txt.gz', 'Series_platform_id': 'GPL9250', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA129411'}\n",
      "617 PRJNA145711 GSE31539 {'Series_title': 'Ribosome footprinting in the cytosol and endoplasmic reticulum', 'Series_geo_accession': 'GSE31539', 'Series_status': 'Public on Dec 20 2011', 'Series_submission_date': 'Aug 19 2011', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '22199352', 'Series_summary': 'In eukaryotic cells, the spatial regulation of protein expression is frequently conferred through the coupling of mRNA localization and the local control of translation. mRNA localization to the endoplasmic reticulum (ER) is a prominent example of such regulation and serves a ubiquitous role in segregating the synthesis of secretory and integral membrane proteins to the ER. Recent genomic and biochemical studies have now expanded this view to suggest a role for the ER in global protein synthesis. We have utilized cell fractionation and ribosome profiling to obtain a genomic survey of the subcellular organization of mRNA translation and report that ribosomal loading of mRNAs, a proxy for mRNA translation, is biased to the ER. Notably, ER-associated mRNAs encoding both cytosolic and topogenic signal-encoding proteins display similar ribosome loading densities, suggesting that ER-associated ribosomes serve a global role in mRNA translation. We propose that the partitioning of mRNAs and their translation between the cytosol and ER compartments may represent a novel mechanism for the post-transcriptional regulation of gene expression.', 'Series_overall_design': 'HEK293 cells were fractionated between the cytosol and endoplasmic reticulum. Within each fraction, ribosome footprints were generated and sequenced. In parallel, total mRNA was sequenced.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Christopher,V,Nicchitta', 'Series_sample_id': 'GSM782788', 'Series_contact_name': 'David,,Reid', 'Series_contact_laboratory': 'Nicchitta', 'Series_contact_department': 'Biochemistry', 'Series_contact_institute': 'Duke University', 'Series_contact_address': '436 Nanaline Duke Bldg., Box 3709 DUMC', 'Series_contact_city': 'Durham', 'Series_contact_state': 'NC', 'Series_contact_zip/postal_code': '27710', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE31nnn/GSE31539/suppl/GSE31539_processed_data.txt.gz', 'Series_platform_id': 'GPL13393', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA145711'}\n",
      "618 PRJNA148631 GSE33671 {'Series_title': 'Selective ribosome profiling reveals the co-translational chaperone action of trigger factor in vivo', 'Series_geo_accession': 'GSE33671', 'Series_status': 'Public on Dec 09 2011', 'Series_submission_date': 'Nov 14 2011', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '22153074', 'Series_summary': 'As nascent polypeptides exit ribosomes, they are engaged by a series of processing, targeting and folding factors. Here we present a selective ribosome profiling strategy that enables global monitoring of when these factors engage polypeptides in the complex cellular environment. Studies of the Escherichia coli chaperone Trigger Factor (TF) reveal that, while TF can interact with many polypeptides, β-barrel outer membrane proteins are the most prominent substrates. Loss of TF leads to broad outer membrane defects and premature, cotranslational protein translocation. While in vitro studies suggested that TF is prebound to ribosomes waiting for polypeptides to emerge from the exit channel, we find that in vivo TF engages ribosomes only after ~100 amino acids are translated. Moreover, excess TF interferes with cotranslational removal of the N-terminal formyl methionine. Our studies support a triaging model in which proper protein biogenesis relies on the fine-tuned, sequential engagement of processing, targeting ad folding factors.', 'Series_overall_design': 'Examination of translation in the Gram-negative bacterium Escherichia coli, as well as an analysis of the interactions between nascent chains and the molecular chaperone Trigger Factor.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Bernd,,Bukau', 'Series_sample_id': 'GSM832613', 'Series_contact_name': 'Eugene,,Oh', 'Series_contact_email': 'Eugene.oh@ucsf.edu', 'Series_contact_phone': '415-502-8089', 'Series_contact_fax': '415-514-2073', 'Series_contact_laboratory': 'Jonathan S Weissman', 'Series_contact_department': 'CMP', 'Series_contact_institute': 'UCSF', 'Series_contact_address': '1700 4th Street', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE33nnn/GSE33671/suppl/GSE33671_readme.txt', 'Series_platform_id': 'GPL10328', 'Series_platform_taxid': '562', 'Series_sample_taxid': '562', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA148631'}\n",
      "619 PRJNA149975 GSE34082 {'Series_title': 'High-resolution view of the yeast meiotic program revealed by ribosome profiling', 'Series_geo_accession': 'GSE34082', 'Series_status': 'Public on Jan 06 2012', 'Series_submission_date': 'Dec 01 2011', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '22194413', 'Series_summary': 'Meiosis is a complex developmental process that generates haploid cells from diploid progenitors. We measured mRNA abundance and protein production through yeast sporulation and found strong temporal control for most genes, achieved through both mRNA levels and translational regulation. Monitoring the timing of protein production revealed novel factors involved in recombination and helped to illuminate the molecular basis of the broad restructuring of meiotic cells. We also found a strong increase in noncanonical translation at short open reading frames (sORFs) on unannnotated transcripts and upstream regions of known transcripts (uORFs). Ribosome occupancy at near-cognate uORFs was associated with more efficient ORF translation; while some AUG uORFs, often on regulated leader extensions, acted comptetitively. This work reveals a pervasive role for meiotic translational control and great complexity in genomic coding.', 'Series_overall_design': 'Fine mapping of gene expression through meiosis reveals extensive regulation of protein synthesis and widespread non-canonical translation.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Jonathan,S,Weissman', 'Series_sample_id': 'GSM843783', 'Series_contact_name': 'Gloria,,Brar', 'Series_contact_email': 'gloria.brar@ucsf.edu', 'Series_contact_institute': 'UCSF', 'Series_contact_address': '1700 4th Street, BH404', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE34nnn/GSE34082/suppl/GSE34082_processed_files.tar.gz', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA149975'}\n",
      "620 PRJNA157105 GSE36892 {'Series_title': 'A unifying model for mTORC1-mediated regulation of mRNA translation', 'Series_geo_accession': 'GSE36892', 'Series_status': 'Public on May 03 2012', 'Series_submission_date': 'Mar 28 2012', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '22552098', 'Series_summary': 'Ribsome profiling analysis of mRNA translation in mouse cells under conditions of mTOR activiation or inhibition.', 'Series_overall_design': 'embryonic fibroblasts from 4EBP1/2 p53 mutants treated with Torin1', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'David,M,Sabatini', 'Series_sample_id': 'GSM904904', 'Series_contact_name': 'Carson,C,Thoreen', 'Series_contact_email': 'thoreen@wi.mit.edu', 'Series_contact_laboratory': 'David M. Sabatini', 'Series_contact_institute': 'Whitehead Institute', 'Series_contact_address': '9 Cambridge Center', 'Series_contact_city': 'Cambridge', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02142', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE36nnn/GSE36892/suppl/GSE36892_RAW.tar', 'Series_platform_id': 'GPL9250', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA157105'}\n",
      "621 PRJNA177721 GSE41605 {'Series_title': 'Decoding human cytomegalovirus using ribosome profiling', 'Series_geo_accession': 'GSE41605', 'Series_status': 'Public on Oct 17 2012', 'Series_submission_date': 'Oct 15 2012', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '23180859', 'Series_summary': 'The human cytomegalovirus (HCMV) genome was sequenced 20 years ago. However, like other complex viruses, our understanding of its protein coding potential is far from complete. Here, we use ribosome profiling and transcript analysis to experimentally define the HCMV translation products and follow their temporal expression. We identified several hundred previously unidentified open reading frames and confirmed a fraction by mass spectrometry. We found that regulated use of alternative transcript start sites plays a broad role in enabling tight temporal control of HCMV protein expression and allowing multiple distinct polypeptides to be generated from a single genomic locus. Our results reveal an unanticipated complexity to the HCMV coding capacity and illustrate the role of regulated changes in transcript start sites in generating this complexity.', 'Series_overall_design': \"The supplementary file 'GSE41605_merlin_final_orfs.bed.gz' includes the 751 ORFs of HCMV that were identified in this study.\", 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Jonathan,S,Weissman', 'Series_sample_id': 'GSM1020249', 'Series_contact_name': 'Noam,,Ginossar', 'Series_contact_institute': 'UCSF', 'Series_contact_address': '1700 4th street', 'Series_contact_city': 'San Francisco', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE41nnn/GSE41605/suppl/GSE41605_merlin_final_orfs.bed.gz', 'Series_platform_id': 'GPL16186', 'Series_platform_taxid': '10359', 'Series_sample_taxid': '10359', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP016143'}\n",
      "622 PRJNA213294 GSE49197 {'Series_title': 'Ribosome profiling reveals pervasive and regulated stop codon readthrough  in Drosophila melanogaster', 'Series_geo_accession': 'GSE49197', 'Series_status': 'Public on Dec 05 2013', 'Series_submission_date': 'Jul 25 2013', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '24302569', 'Series_summary': 'Ribosomes can read through stop codons in a regulated manner, elongating rather than terminating the nascent peptide. Stop codon readthrough is essential to diverse viruses, and phylogenetically predicted to occur in a few hundred genes in Drosophila melanogaster, but the importance of regulated readthrough in eukaryotes remains largely unexplored. Here, we present a ribosome profiling assay (deep sequencing of ribosome-protected mRNA fragments) for Drosophila melanogaster, and provide the first genome-wide experimental analysis of readthrough. Readthrough is far more pervasive than expected: the vast majority of readthrough events evolved within D. melanogaster and were not predicted phylogenetically. The resulting C-terminal protein extensions show evidence of selection, contain functional subcellular localization signals, and their readthrough is regulated, arguing for their importance. We further demonstrate that readthrough occurs in yeast and humans. Readthrough thus provides general mechanisms both to regulate gene expression and function, and to add plasticity to the proteome during evolution.', 'Series_overall_design': '12 samples of Drosophila ribosome profiling and poly(A)+ mRNA-seq, including technical replicates in S2 cells, and biological replicates of 0-2 hour embryos', 'Series_type': 'Other', 'Series_contributor': 'Jonathan,S,Weissman', 'Series_sample_id': 'GSM1195329', 'Series_contact_name': 'Joshua,Griffin,Dunn', 'Series_contact_email': 'joshua.g.dunn@gmail.com', 'Series_contact_laboratory': 'Weissman', 'Series_contact_department': 'Cellular & Molecular Pharmacology', 'Series_contact_institute': 'UCSF / HHMI', 'Series_contact_address': '1700 4th Street, BH404', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'California', 'Series_contact_zip/postal_code': '94114', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE49nnn/GSE49197/suppl/GSE49197_s2_150ab_pooled_rc.wig.gz', 'Series_platform_id': 'GPL13304', 'Series_platform_taxid': '7227', 'Series_sample_taxid': '7227', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP028243'}\n",
      "623 PRJNA230441 GSE52905 {'Series_title': 'Extensive oscillatory gene expression during C. elegans larval development [Ribosome footprinting]', 'Series_geo_accession': 'GSE52905', 'Series_status': 'Public on Mar 01 2014', 'Series_submission_date': 'Dec 03 2013', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '24440504', 'Series_summary': 'We report the presence of extensive, transcriptionally controlled oscillations in the C. elegans, developmental transcriptome. Furthermore, using ribosome profiling, we show that these oscillating transcripts are actively translated.', 'Series_overall_design': 'Ribosome-profiling analysis of a timecourse that was collected over C. elegans development', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Helge,,Großhans', 'Series_sample_id': 'GSM1277916', 'Series_contact_name': 'Dimos,,Gaidatzis', 'Series_contact_email': 'd.gaidatzis@fmi.ch', 'Series_contact_institute': 'Friedrich Miescher Institute', 'Series_contact_address': 'Maulbeerstrasse 66', 'Series_contact_city': 'Basel', 'Series_contact_zip/postal_code': '4058', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE52nnn/GSE52905/suppl/GSE52905_ce_geneExpression_Footprint_rawCounts.txt.gz', 'Series_platform_id': 'GPL13657', 'Series_platform_taxid': '6239', 'Series_sample_taxid': '6239', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP033430'}\n",
      "624 PRJNA265851 GSE62859 {'Series_title': 'Functional characterization of C. elegans Y-box binding proteins reveals tissue-specific functions and a critical role in the formation of polysomes (ribosome)', 'Series_geo_accession': 'GSE62859', 'Series_status': 'Public on Jan 08 2015', 'Series_submission_date': 'Oct 30 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25378320', 'Series_summary': 'We present a basic characterization of the function of Y-box binding proteins in C. elegans. Besides playing an important role for fertility in the germline (all four CEY proteins), we found that the presence of CEY-1 and CEY-4 is essential for the assembly of larger polysomes in the soma. We therefore performed ribosome-profiling in combination with total RNA sequencing in wild type and cey-1,-4 double mutant animals to globally compare mRNA levels and their translation status.', 'Series_overall_design': 'Ribosome-profiling analysis was peformed on wild type and cey-1,-4 mutant animals in duplicates. Four samples in total.', 'Series_type': 'Other', 'Series_contributor': 'Rafal,,Ciosk', 'Series_sample_id': 'GSM1534611', 'Series_contact_name': 'Dimos,,Gaidatzis', 'Series_contact_email': 'd.gaidatzis@fmi.ch', 'Series_contact_institute': 'Friedrich Miescher Institute', 'Series_contact_address': 'Maulbeerstrasse 66', 'Series_contact_city': 'Basel', 'Series_contact_zip/postal_code': '4058', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE62nnn/GSE62859/suppl/GSE62859_ce_geneExpression_Footprint_rawCounts.txt.gz', 'Series_platform_id': 'GPL18245', 'Series_platform_taxid': '6239', 'Series_sample_taxid': '6239', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP049414'}\n",
      "625 PRJNA269069 GSE63789 {'Series_title': 'Causal signals between codon bias, mRNA structure and the efficiency of translation and elongation', 'Series_geo_accession': 'GSE63789', 'Series_status': 'Public on Jan 05 2015', 'Series_submission_date': 'Dec 02 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25538139', 'Series_summary': 'Ribosome profiling data reports on the distribution of translating ribosomes, at steady-state, with codon-level resolution. We present a robust method to extract codon translation rates and protein synthesis rates from these data, and identify causal features associated with elongation and translation efficiency in physiological conditions in yeast. We show that neither elongation rate nor translational efficiency is improved by experimental manipulation of the abundance or body sequence of the rare AGG tRNA. Deletion of three of the four copies of the heavily used ACA tRNA shows a modest efficiency decrease that could be explained by other rate-reducing signals at gene start. This suggests that correlation between codon bias and efficiency arises as selection for codons to utilize translation machinery efficiently in highly translated genes. We also show a correlation between efficiency and RNA structure calculated both computationally and from recent structure probing data, as well as the Kozak initiation motif, which may comprise a mechanism to regulate initiation.', 'Series_overall_design': 'We test whether tRNA abundance affects elongation or translation efficiency by changing the tRNA levels through deletion or over expression and measuring the ribosomal dwell time at each codon using a robust statistical method that accounts for flow conservation.', 'Series_type': 'Other', 'Series_contributor': 'Daphne,,Koller', 'Series_sample_id': 'GSM1557451', 'Series_contact_name': 'Cristina,,Pop', 'Series_contact_department': 'Computer Science', 'Series_contact_institute': 'Stanford University', 'Series_contact_address': '353 Serra Mall', 'Series_contact_city': 'Stanford', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94305', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE63nnn/GSE63789/suppl/GSE63789_counts_wt_URA.txt.gz', 'Series_platform_id': 'GPL9377', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP050443'}\n",
      "626 PRJNA283425 GSE68705 {'Series_title': 'Genome-wide reprogramming of transcript architecture by temperature specifies the developmental states of the human pathogen Histoplasma [ribosome profiling]', 'Series_geo_accession': 'GSE68705', 'Series_status': 'Public on Jul 16 2015', 'Series_submission_date': 'May 08 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26177267', 'Series_summary': 'Eukaryotic cells integrate layers of gene regulation to coordinate complex cellular processes; however, mechanisms of post-transcriptional gene regulation remain poorly studied. The human fungal pathogen Histoplasma capsulatum (Hc) responds to environmental or host temperature by initiating unique transcriptional programs to specify multicellular (hyphae) or unicellular (yeast) developmental states that function in infectivity or pathogenesis, respectively. Here we used recent advances in next-generation sequencing to uncover a novel post-transcriptional mechanism of gene regulation in Hc developmental cell types. We found that ~2% percent of Hc transcripts exhibit 5’ leader sequences that differ markedly in length between morphogenetic states. Ribosome density and mRNA abundance measurements uncovered a class of differential leader transcripts that exhibit tight transcriptional and translational regulation. Further examination of these dually regulated genes revealed that some control Hc morphology and that their strict regulation is necessary for the pathogen to make appropriate developmental decisions in response to temperature.  This series contains ribosomal footprinting data for Hc G217B.', 'Series_overall_design': 'mRNA libraries were prepared with or without ribosome footprinting for two biological replicates each from cells grown in the yeast or hyphal phase (8 samples total).', 'Series_type': 'Other', 'Series_contributor': 'Anita,,Sil', 'Series_sample_id': 'GSM1679363', 'Series_contact_name': 'Mark,,Voorhies', 'Series_contact_email': 'mark.voorhies@ucsf.edu', 'Series_contact_laboratory': 'Anita Sil', 'Series_contact_department': 'Microbiology and Immunology', 'Series_contact_institute': 'University of California', 'Series_contact_address': '513 Parnassus, S472', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94143-0414', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE68nnn/GSE68705/suppl/GSE68705_Supplementary_File_27.tdt.txt.gz', 'Series_platform_id': 'GPL20173', 'Series_platform_taxid': '5037', 'Series_sample_taxid': '5037', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP058148'}\n",
      "627 PRJNA287112 GSE69906 {'Series_title': 'The transcription and translation landscapes during human cytomegalovirus infection reveal novel host-pathogen interactions', 'Series_geo_accession': 'GSE69906', 'Series_status': 'Public on Nov 02 2015', 'Series_submission_date': 'Jun 16 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26599541', 'Series_summary': 'During its long infection cycle, human cytomegalovirus (HCMV) extensively manipulates cellular gene expression to maintain conditions favorable for viral propagation. In order to reveal the signature of cellular genes that are manipulated by HCMV, we measured RNA abundance and rate of protein production through the course of HCMV infection. We characterized changes for most expressed cellular genes and although much of the regulation was transcriptional we uncover diverse and dynamic translational regulation for subsets of host genes, revealing unappreciated coordination in translational control that suggests common regulators', 'Series_overall_design': 'Ribosome profiling and mRNA-seq along HCMV infection', 'Series_type': 'Other', 'Series_contributor': 'Noam,,Stern-Ginossar', 'Series_sample_id': 'GSM1712284', 'Series_contact_name': 'Noam,,Stern-Ginossar', 'Series_contact_email': 'noam.stern-ginossar@weizmann.ac.il', 'Series_contact_laboratory': 'Noam Stern-Ginossar', 'Series_contact_department': 'Molecular Genetics', 'Series_contact_institute': 'Weizmann Institute', 'Series_contact_address': '234 herzel', 'Series_contact_city': 'Rehovot', 'Series_contact_zip/postal_code': '7610001', 'Series_contact_country': 'Israel', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE69nnn/GSE69906/suppl/GSE69906_RAW.tar', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP059531'}\n",
      "628 PRJNA318047 GSE80133 {'Series_title': 'Post-transcriptional regulation by the let-7 microRNA and the TRIM-NHL protein LIN41 [Ribosome footprinting]', 'Series_geo_accession': 'GSE80133', 'Series_status': 'Public on Jul 17 2017', 'Series_submission_date': 'Apr 11 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28111013', 'Series_summary': 'We perform RNA sequencing and ribosome profiling time course experiments to examine the effect of fully dysregulating all let-7 targets (in let-7(n2853) animals), partially dysregulating only LIN41 (in lin-41(xe11) animals) or fully dysregulating all let-7 targets while partially dysregulating LIN41 in lin-41(xe11); let-7(n2853) double mutant animals. We conclude that effects on gene expression in let-7 mutant animals are largely and quantitatively explained by dysregulation of LIN41 as its primary target. Furthermore, we identify direct LIN41 target genes regulated on the level of translation or mRNA abundance.', 'Series_overall_design': 'Ribosome profiling time course experiments sampling synchronized worm populations of different genetic backgrounds every two hours over the course of development from late L2/early L3 stage to late L4/Young adult stage.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Helge,,Großhans', 'Series_sample_id': 'GSM2113392', 'Series_contact_name': 'Dimos,,Gaidatzis', 'Series_contact_email': 'd.gaidatzis@fmi.ch', 'Series_contact_institute': 'Friedrich Miescher Institute', 'Series_contact_address': 'Maulbeerstrasse 66', 'Series_contact_city': 'Basel', 'Series_contact_zip/postal_code': '4058', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE80nnn/GSE80133/suppl/GSE80133_ce_geneExpression_riboM_rawCounts.txt.gz', 'Series_platform_id': 'GPL13657', 'Series_platform_taxid': '6239', 'Series_sample_taxid': '6239', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP073115'}\n",
      "629 PRJNA345383 GSE87614 {'Series_title': 'Translation initiation factor eIF4G1 preferentially binds yeast transcript leaders containing conserved oligo-uridine motifs', 'Series_geo_accession': 'GSE87614', 'Series_status': 'Public on May 29 2017', 'Series_submission_date': 'Oct 04 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28546148', 'Series_summary': 'Translational control of gene expression plays essential roles in cellular stress responses and organismal development by enabling rapid, selective, and localized control of protein production. Translational regulation depends on context-dependent differences in the protein output of mRNAs, but the key mRNA features that distinguish efficiently translated mRNAs are largely unknown. Here we comprehensively determined the RNA-binding preferences of the initiation factor eIF4G to assess whether this core translation initiation factor has intrinsic sequence preferences that contribute to preferential translation of specific mRNAs.', 'Series_overall_design': 'Ribosome footprint profiling of eIF4G depleted yeast cells', 'Series_type': 'Other', 'Series_contributor': 'Wendy,V,Gilbert', 'Series_sample_id': 'GSM2335497', 'Series_contact_name': 'Boris,,Zinshteyn', 'Series_contact_email': 'zinshteyn.boris@gmail.com', 'Series_contact_laboratory': 'Green', 'Series_contact_department': 'Molecular Biology and Genetics', 'Series_contact_institute': 'Johns Hopkins University School of Medicine', 'Series_contact_address': '725 N Wolfe St, PCTB 714', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'Maryland', 'Series_contact_zip/postal_code': '21205', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE87nnn/GSE87614/suppl/GSE87614_YPA_mRNA_offset8_rpkms.tsv.gz', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP090827'}\n",
      "630 PRJNA356775 GSE91068 {'Series_title': 'RNAseq and ribosome profiling of yeast cells grown under methionine restriction condition.', 'Series_geo_accession': 'GSE91068', 'Series_status': 'Public on Feb 24 2017', 'Series_submission_date': 'Dec 08 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28212626', 'Series_summary': 'We characterize the translational and transcriptional programs induced by MetR and investigate the potential mechanisms through which methionine regulates gene expression, using the budding yeast S. cerevisiae as the model organism. Using ribosomal profiling and RNA-seq, we systematically compared the translational and transcriptional profiles of cells growing in the normal and methionine restricted media. We observed a broad spectrum of gene expression changes in response to MetR, including hundreds of genes whose transcript level and/or translational efficiency changed significantly. These genes fall into specific functional classes that are informative of the physiological state of the cell under MetR. Analysis of ribosome loading patterns of genes with increased translational efficiency suggested mechanisms both similar and different from the canonical model of translational regulation by general amino acid starvation. Analysis of the genes with decreased translational efficiency added support to the thiolation model of translational regulation by methionine. Since MetR extends the lifespan of many species, the list of genes we identified in this study can be good candidates for studying the downstream effectors of lifespan extension.', 'Series_overall_design': 'The initial cell culture was incubated in 300 ml SD medium overnight to an OD600 0.8~1.0, then diluted by five fold using fresh SD media and incubated for another 4 hours under 30oC to an OD600 0.8~1.0. The sample was then divided equally into two aliquots. Cells were separated from the media by spin-down at 3000 g for 5 minutes and re-suspended in SD and MetR media respectively. All samples were incubated for another hour before harvesting. All the steps were carried out at 30°C. Raw sequences were obtained from Illumina Hiseq 2000.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Hao,,Li', 'Series_sample_id': 'GSM2420489', 'Series_contact_name': 'Jiashun,,Zheng', 'Series_contact_email': 'jiashun@genome.ucsf.edu', 'Series_contact_laboratory': 'Li Lab', 'Series_contact_department': 'Biochemistry and Biophysics', 'Series_contact_institute': 'UCSF', 'Series_contact_address': '1700 4th St', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE91nnn/GSE91068/suppl/GSE91068_RAW.tar', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP094800'}\n",
      "631 PRJNA386877 GSE98934 {'Series_title': 'Regulated Ire1-dependent mRNA decay requires no-go mRNA degradation to maintain endoplasmic reticulum homeostasis in S. pombe', 'Series_geo_accession': 'GSE98934', 'Series_status': 'Public on Sep 14 2017', 'Series_submission_date': 'May 15 2017', 'Series_last_update_date': 'Jul 25 2021', 'Series_pubmed_id': '28945192', 'Series_summary': 'The unfolded protein response (UPR) continually monitors the protein folding capacity of the endoplasmic reticulum (ER). In S. pombe, Ire1, an ER membrane-resident kinase/endoribonuclease, utilizes a mechanism of selective degradation of ER-targeted mRNAs (RIDD) to maintain ER homeostasis. Here, we used a genetic screen to identify factors critical to the Ire1-mediated UPR response in S. pombe and found several proteins, Dom34, Hbs1 and SkiX, previously implicated in ribosome rescue and the no-go-decay (NGD) pathway. Ribosome profiling in ER-stressed cells lacking these factors revealed that Ire1-mediated cleavage on ER-associated mRNAs results in ribosome stalling on the cleaved transcript and, ultimately in full mRNA degradation.  The process engages mechanisms of precise, iterated cleavage of the mRNAs and ribosome rescue. This clear signature allowed us to discover hundreds of novel mRNA targets of Ire1. Our results reveal that the UPR in S. pombe executes RIDD in an intricate interplay between Ire1, translation, and the NGD surveillance pathway, and establish a critical role for NGD in maintaining ER homeostasis.', 'Series_overall_design': \"16 samples are included in the study (8 mRNA-Seq samples and 8 ribosome footprinting samples). Read sizes were examined for lengths of 15-18 nt and 25-34 nt for all profiling samples and reads were aligned by 5' ends for mRNA-Seq and 3' ends for profiling.\", 'Series_type': 'Other', 'Series_contributor': 'Rachel,,Green', 'Series_sample_id': 'GSM2628054', 'Series_contact_name': 'Nicholas,R,Guydosh', 'Series_contact_email': 'nicholas.guydosh@nih.gov', 'Series_contact_laboratory': 'LBG', 'Series_contact_department': 'NIDDK', 'Series_contact_institute': 'National Institutes of Health', 'Series_contact_address': '8 Center Dr, Bldg 8, 220', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '20892', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE98nnn/GSE98934/suppl/GSE98934_RAW.tar', 'Series_platform_id': 'GPL17225', 'Series_platform_taxid': '4896', 'Series_sample_taxid': '4896', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP107240'}\n",
      "632 PRJNA421497 GSE107826 {'Series_title': 'Translational control of ERK signalling pathway by the mRNA cap-binding protein 4EHP', 'Series_geo_accession': 'GSE107826', 'Series_status': 'Public on Feb 20 2018', 'Series_submission_date': 'Dec 07 2017', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '29412140', 'Series_summary': 'microRNA (miRNA)-mediated gene silencing is commonly deregulated in a wide variety of diseases. Therefore, a better understanding of how miRNAs govern the translation and stability of mRNAs is of paramount importance. We recently demonstrated that the cap-binding protein 4EHP is recruited by the miRNA machinery to effect translational repression. However, the impact of 4EHP on regulation of endogenous mRNAs and its role in cell biology is debated. Herein, using the ribosome profiling assay, we identify a subset of endogenous mRNAs that are sensitive to translational repression by 4EHP. We show that expression of DUSP6, a phosphatase that reverses ERK1/2 phosphorylation, is regulated by 4EHP. This regulation, which occurs exclusively at the level of mRNA translation, without affecting the stability of Dusp6 mRNA, is engendered through 4EHP- and miRNA-dependent elements in Dusp6 3´UTR. Consequently, 4EHP protein controls ERK1/2 phosphorylation, promotes cell growth and inhibits apoptosis. Our data reveal a crucial role for translational control mechanisms in the regulation of the ERK pathway.', 'Series_overall_design': 'Ribosome profiling and mRNA-seq in 4EHP KO and WT MEFs.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Noam,,Stern-Ginossar', 'Series_sample_id': 'GSM2881067', 'Series_contact_name': 'Noam,,Stern-Ginossar', 'Series_contact_email': 'noam.stern-ginossar@weizmann.ac.il', 'Series_contact_laboratory': 'Noam Stern-Ginossar', 'Series_contact_department': 'Molecular Genetics', 'Series_contact_institute': 'Weizmann Institute', 'Series_contact_address': '234 herzel', 'Series_contact_city': 'Rehovot', 'Series_contact_zip/postal_code': '7610001', 'Series_contact_country': 'Israel', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE107nnn/GSE107826/suppl/GSE107826_RAW.tar', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP126348'}\n",
      "633 PRJNA515049 GSE125038 {'Series_title': 'Cycloheximide can distort measurements of mRNA levels and translation efficiency', 'Series_geo_accession': 'GSE125038', 'Series_status': 'Public on Mar 08 2019', 'Series_submission_date': 'Jan 14 2019', 'Series_last_update_date': 'Mar 28 2019', 'Series_pubmed_id': '30916348', 'Series_summary': 'Regulation of the efficiency with which an mRNA is translated into proteins represents a key mechanism for controlling gene expression. Such regulation impacts the number of actively translating ribosomes per mRNA molecule, referred to as translation efficiency (TE), which can be monitored using ribosome profiling and RNA-seq, or by evaluating the position of an mRNA in a polysome gradient. Here we show that in budding yeast, under nutrient limiting conditions, the commonly used translation inhibitor cycloheximide induces rapid transcriptional upregulation of hundreds of genes involved in ribosome biogenesis. Cycloheximide also prevents translation of these newly transcribed messages, leading to an apparent drop in TE of these genes under conditions that include key transitions of the yeast metabolic cycle, meiosis, and amino acid starvation; an effect which is abolished when cycloheximide pretreatment is omitted.  This response requires TORC1 signaling, and is modulated by the genetic background as well as the vehicle used to deliver the drug. The present work highlights an important caveat to the use of translation inhibitors when measuring TE, and will hopefully aid in future experimental design as well as interpretation of prior results', 'Series_overall_design': 'Paired RNA-seq and ribosome profiling sequencing data for 28 yeast samples in various growth conditions, either with or without cycloheximide pretreatment', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Jonathan,S,Weissman', 'Series_sample_id': 'GSM3561573', 'Series_contact_name': 'Daniel,A,Santos', 'Series_contact_laboratory': 'Jonathan Weissman', 'Series_contact_department': 'Cellular and Molecular Pharmacology', 'Series_contact_institute': 'UCSF', 'Series_contact_address': '1700 4th Street', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94153', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE125nnn/GSE125038/suppl/GSE125038_ymc_sample_info.txt.gz', 'Series_platform_id': 'GPL21656', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP180875'}\n",
      "634 PRJNA515197 GSE125086 {'Series_title': 'Ribosome queuing enables non-AUG translation to be resistant to multiple protein synthesis inhibitors', 'Series_geo_accession': 'GSE125086', 'Series_status': 'Public on Jun 05 2019', 'Series_submission_date': 'Jan 15 2019', 'Series_last_update_date': 'Oct 06 2023', 'Series_pubmed_id': '31171704', 'Series_summary': 'Aberrant translation initiation at non-AUG start codons is associated with multiple cancers and neurodegenerative diseases. Nevertheless, how non-AUG translation is regulated differently from canonical translation is poorly understood. We thus used start codon-selective\\xa0 reporters and ribosome profiling to characterize how translation from non-AUG start codons responds to protein synthesis inhibitors in human cells. These analyses surprisingly revealed that translation of non-AUG reporters and the endogenous GUG-encoded DAP5 (eIF4G2/p97) mRNA are resistant to cycloheximide (CHX), a translation inhibitor which slows but does not completely abrogate elongation. Our data suggest that slowly elongating ribosomes cause queuing of scanning pre-initiation complexes (PIC), preferentially enhancing otherwise poor recognition of non-AUG start codons. Consistent with this model, limiting PIC formation or scanning sensitizes non-AUG translation to CHX. Moreover, PIC queuing can cause translation from an AUG codon in a poor context to become less sensitive to CHX. We further find that non-AUG translation is resistant to other inhibitors that target ribosomes within the coding sequence, but not those targeting newly initiated ribosomes. In total, these data indicate that ribosome queuing enables mRNAs with poor initiation context, namely those from non-AUG start codons, to be resistant to pharmacological inhibitors.', 'Series_overall_design': 'Measuring translational efficiency genome-wide in response to prolonged cyclohexamide treatment', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Jeremy,,Wilusz', 'Series_sample_id': 'GSM3562593', 'Series_contact_name': 'Daniel,,Goldman', 'Series_contact_email': 'goldman@jhmi.edu', 'Series_contact_laboratory': 'Rachel Green', 'Series_contact_department': 'Molecular Biology and Genetics', 'Series_contact_institute': 'Johns Hopkins School of Medicine', 'Series_contact_address': '725 N. Wolfe St.', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'Maryland', 'Series_contact_zip/postal_code': '21205', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE125nnn/GSE125086/suppl/GSE125086_cds_rpkms.tsv.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP179458'}\n",
      "635 PRJNA554100 GSE134152 {'Series_title': 'RNAseq and ribosome profiling of yeast cells grown under glucose restriction condition.', 'Series_geo_accession': 'GSE134152', 'Series_status': 'Public on Jul 10 2020', 'Series_submission_date': 'Jul 11 2019', 'Series_last_update_date': 'Aug 24 2020', 'Series_pubmed_id': '32821821', 'Series_summary': 'We investigate the mechanism by which glucose restriction extends yeast replicative lifespan, using an approach that combines ribosomal profiling and RNA-seq. We systematically compared the translational and transcriptional profiles of cells grown in glucose restriction and normal media, uncovering groups of functionally related genes that are up or down regulated.', 'Series_overall_design': 'We used two different protocols to grow cell culture in SD vs. GR conditions.  1) protocol-1: Quick Dilution from SD (2% glucose) to GR (0.05% glucose): the initial cell culture was incubated in SD medium overnight to OD600 0.8~1.0, then diluted by 5 fold using SD media and incubated for another 4 hours. The sample was then divided equally into two aliquots. One was diluted  40 times by pre-warmed SD medium with 0% glucose to reach final 0.05% glucose concentration (the sample), and the other diluted 40 times by pre-warmed SD medium with 2% glucose (the control). Both samples were incubated for another hour before harvesting. All the steps were carried out at 30°C.  2) protocol-2: Spin-down and re-suspension from SD to GR : the initial cell culture was incubated in SD medium overnight to a OD600 0.8~1.0, then diluted by 5 fold using SD medium and incubated for another 4 hours.  The sample was then divided equally into 2 aliquots. Cells were separated from the media by spin-down at 2000 rcf for 5 minutes and re-suspended into SD and GR media respectively. All samples were incubated for for another hour before harvesting. All the steps were carried out at 30°C.  Ribosomal profiling experiments were carried out using the protocol developed by Ingolia et al.  Raw sequences were obtained from Illumina Hiseq 2000.', 'Series_type': 'Other', 'Series_contributor': 'Hao,,Li', 'Series_sample_id': 'GSM3938060', 'Series_contact_name': 'Hao,,Li', 'Series_contact_email': 'haoli@genome.ucsf.edu', 'Series_contact_phone': '4155142616', 'Series_contact_department': 'Biochemistry and Biophysics', 'Series_contact_institute': 'UCSF', 'Series_contact_address': '1700 4th St', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE134nnn/GSE134152/suppl/GSE134152_RAW.tar', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP214259'}\n",
      "636 PRJNA578565 GSE139131 {'Series_title': 'Selective Translation Complex Profiling in yeast and human Reveals Staged Initiation and Co-translational  Assembly of Initiation Factor Complexes [Human]', 'Series_geo_accession': 'GSE139131', 'Series_status': 'Public on Jun 26 2020', 'Series_submission_date': 'Oct 20 2019', 'Series_last_update_date': 'Sep 08 2020', 'Series_pubmed_id': '32589964', 'Series_summary': 'Translational control targeting mainly the initiation phase is central to the regulation of  gene expression. Understanding all of its aspects requires substantial technological  advancements. Here we modified yeast Translational Complex Profile sequencing  (TCP-seq), related to ribosome profiling, and adopted it for mammalian cells. Human  TCP-seq, capable of capturing footprints of 40S subunits (40Ses) in addition to 80S  ribosomes (80Ses), revealed that mammalian and yeast 40Ses distribute similarly  across 5’UTRs indicating considerable evolutionary conservation. We further  developed a variation called Selective TCP-seq (Sel-TCP-seq) enabling selection for  40Ses and 80Ses associated with an immuno-targeted factor in yeast and human. Sel-  TCP-seq demonstrated that eIF2 and eIF3 travel along 5’UTRs with scanning 40Ses to  successively dissociate upon start codon recognition. Manifesting the Sel-TCP-seq  versatility for gene expression studies, we also identified four initiating 48S  conformational intermediates, provided novel insights into ATF4 and GCN4 mRNA  translational control, and demonstrated co-translational assembly of initiation factor  complexes.', 'Series_overall_design': '2 replicates for 40S and 80S TCP-seq, 1 replicate for eIF3b bound 40S Sel-TCP-seq and mRNA', 'Series_type': 'Other', 'Series_contributor': 'Leos,S,Valasek', 'Series_sample_id': 'GSM4131413', 'Series_contact_name': 'Susan,,Wagner', 'Series_contact_email': 'susan.wagner@canberra.edu.au', 'Series_contact_laboratory': 'RNA Biology - Preiss Group', 'Series_contact_department': 'The John Curtin School of Medical Research', 'Series_contact_institute': 'Australian National University', 'Series_contact_address': '131 Garran Road', 'Series_contact_city': 'Canberra', 'Series_contact_state': 'ACT', 'Series_contact_zip/postal_code': '2601', 'Series_contact_country': 'Australia', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE139nnn/GSE139131/suppl/GSE139131_human_metagene_coverage_cap_-40+40.csv.gz', 'Series_platform_id': 'GPL20795', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP226389'}\n",
      "637 PRJNA578566 GSE139130 {'Series_title': 'Selective Translation Complex Profiling in yeast and human Reveals Staged Initiation and Co-translational  Assembly of Initiation Factor Complexes [Yeast]', 'Series_geo_accession': 'GSE139130', 'Series_status': 'Public on Jun 26 2020', 'Series_submission_date': 'Oct 20 2019', 'Series_last_update_date': 'Sep 08 2020', 'Series_pubmed_id': '32589964', 'Series_summary': 'Translational control targeting mainly the initiation phase is central to the regulation of  gene expression. Understanding all of its aspects requires substantial technological  advancements. Here we modified yeast Translational Complex Profile sequencing  (TCP-seq), related to ribosome profiling, and adopted it for mammalian cells. Human  TCP-seq, capable of capturing footprints of 40S subunits (40Ses) in addition to 80S  ribosomes (80Ses), revealed that mammalian and yeast 40Ses distribute similarly  across 5’UTRs indicating considerable evolutionary conservation. We further  developed a variation called Selective TCP-seq (Sel-TCP-seq) enabling selection for  40Ses and 80Ses associated with an immuno-targeted factor in yeast and human. Sel-  TCP-seq demonstrated that eIF2 and eIF3 travel along 5’UTRs with scanning 40Ses to  successively dissociate upon start codon recognition. Manifesting the Sel-TCP-seq  versatility for gene expression studies, we also identified four initiating 48S  conformational intermediates, provided novel insights into ATF4 and GCN4 mRNA  translational control, and demonstrated co-translational assembly of initiation factor  complexes.', 'Series_overall_design': '3 sets; set1: 3 replicates for 40S and 80S TCP-seq, 1 replicate for each eIF3a, eIF3c and eIF2beta bound 40S Sel-TCP-seq and 1 replicate each for eIF3a and eIF3c bound 80S Sel-TCP-seq (1-1, 1-2, 1-3); set2+3: 2 more replicates (in addition to set1) of 80S TCP-seq and eIF3a bound 80S Sel-TCP-seq (2-1, 3-1)', 'Series_type': 'Other', 'Series_contributor': 'Leos,S,Valasek', 'Series_sample_id': 'GSM4131407', 'Series_contact_name': 'Susan,,Wagner', 'Series_contact_email': 'susan.wagner@canberra.edu.au', 'Series_contact_laboratory': 'RNA Biology - Preiss Group', 'Series_contact_department': 'The John Curtin School of Medical Research', 'Series_contact_institute': 'Australian National University', 'Series_contact_address': '131 Garran Road', 'Series_contact_city': 'Canberra', 'Series_contact_state': 'ACT', 'Series_contact_zip/postal_code': '2601', 'Series_contact_country': 'Australia', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE139nnn/GSE139130/suppl/GSE139130_metagene_coverage_CAP_-40+40.tbl.txt.gz', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP226388'}\n",
      "638 PRJNA749454 GSE180766 {'Series_title': 'Queuosine-tRNA promotes sex-dependent learning and memory formation by maintaining codon-biased translation elongation speed.', 'Series_geo_accession': 'GSE180766', 'Series_status': 'Public on Aug 16 2023', 'Series_submission_date': 'Jul 24 2021', 'Series_last_update_date': 'Nov 15 2023', 'Series_pubmed_id': '37609797', 'Series_summary': 'performance.', 'Series_overall_design': 'Cytoplasmic RNA and Cytoplasmic RNA protected by ribosomes from RNase I (Ribonuclease I) cleavage, were sequenced. Small RNA libraries were performed according to the protocol using NEB NEXT Small RNA library Prep Set for Illumina (Multiplex Compatible) E7330, and size-selected (120-170 base pairs) before sequencing.', 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'Francesca,,Tuorto', 'Series_sample_id': 'GSM5470568', 'Series_contact_name': 'Francesca,,Tuorto', 'Series_contact_email': 'Francesca.Tuorto@medma.uni-heidelberg.de', 'Series_contact_department': 'Division of Biochemistry  Mannheim Institute for Innate Immunoscience (MI3)', 'Series_contact_institute': 'Deutsches Krebsforschungszentrum', 'Series_contact_address': 'Ludolf-Krehl-Str. 13-17', 'Series_contact_city': 'Mannheim', 'Series_contact_zip/postal_code': 'D-68167', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE180nnn/GSE180766/suppl/GSE180766_RAW.tar', 'Series_platform_id': 'GPL21626', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP329691'}\n",
      "639 PRJNA854095 GSE207189 {'Series_title': 'Ribosome profiling and mRNA seq demonstrate that insertion of common resistance cassettes in yeast drives aberrant transcription', 'Series_geo_accession': 'GSE207189', 'Series_status': 'Public on Nov 29 2022', 'Series_submission_date': 'Jun 29 2022', 'Series_last_update_date': 'Jan 05 2023', 'Series_pubmed_id': '36503721', 'Series_summary': 'Our study found that insertion of commonly used selection cassettes in yeast can drive aberrant transcription events and disrupt expression of neighboring genes. We performed ribosome profiling and mRNA-seq on yeast strains where ORFs had been replaced with common selection cassettes and observed aberrant transcripts. Aberrant transcripts positioned near adjacent genomic loci repressed protein synthesis of those genes by transcription interference (ablating normal length transcript production) coincident with translation of competitive uORFs on the long transcript which prevented translation of the main ORF seen by ribosome profiling.', 'Series_overall_design': 'We performed ribosome profiling and matched mRNA-seq on wild-type and dbp1∆::KANMX6 yeast of interest and observed the expression of loci near deleted ORFs', 'Series_type': 'Other', 'Series_contributor': 'Gloria,,Brar', 'Series_sample_id': 'GSM6281079', 'Series_contact_name': 'Gloria,Ann,Brar', 'Series_contact_email': 'gabrar@berkeley.edu', 'Series_contact_department': 'MCB', 'Series_contact_institute': 'UC Berkeley', 'Series_contact_address': '628 Barker Hall', 'Series_contact_city': 'Berkeley', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94720', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE207nnn/GSE207189/suppl/GSE207189_RAW.tar', 'Series_platform_id': 'GPL17582', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA854095'}\n",
      "640 PRJNA854601 GSE207267 {'Series_title': 'loss of DBP1 during meiosis in yeast', 'Series_geo_accession': 'GSE207267', 'Series_status': 'Public on Nov 29 2022', 'Series_submission_date': 'Jun 30 2022', 'Series_last_update_date': 'Jan 05 2023', 'Series_pubmed_id': '36503721', 'Series_summary': 'We used CRISPR/Cas9 to delete the DBP1 ORF in SK1 budding yeast cells. We then used ribosome profiling and mRNA-seq to observe gene expression profiles of wild-type and dbp1∆ cells during meiosis.', 'Series_overall_design': 'Cells were grown as described in Powers et al., 2022. Samples were collected 6h post transfer to SPO media. Two biological replicates are included for each genotype.', 'Series_type': 'Other', 'Series_contributor': 'Gloria,,Brar', 'Series_sample_id': 'GSM6283340', 'Series_contact_name': 'Gloria,Ann,Brar', 'Series_contact_email': 'gabrar@berkeley.edu', 'Series_contact_department': 'MCB', 'Series_contact_institute': 'UC Berkeley', 'Series_contact_address': '628 Barker Hall', 'Series_contact_city': 'Berkeley', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94720', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE207nnn/GSE207267/suppl/GSE207267_Wildtype_dbp1_footprintRPKM.txt.gz', 'Series_platform_id': 'GPL21656', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA854601'}\n",
      "641 PRJNA1002994 GSE240217 {'Series_title': 'The IGF2BP3-COPS7B axis facilitates mRNA translation to drive colorectal cancer progression', 'Series_geo_accession': 'GSE240217', 'Series_status': 'Public on Aug 07 2023', 'Series_submission_date': 'Aug 07 2023', 'Series_last_update_date': 'Nov 09 2023', 'Series_pubmed_id': '37560971', 'Series_summary': 'Many studies have provided valuable information about genomic and transcriptomic changes that occur in colorectal cancer (CRC). However, protein abundance cannot be reliably predicted by DNA alteration or mRNA expression, which can be partially attributed to post-transcriptional and/or translational regulation of gene expression. In this study, we identified increased translational efficiency (TE) as a hallmark of CRC by evaluating the transcriptomic and proteomic features of CRC patients, along with comparative transcriptomic and ribosome-protected mRNA analysis in colon epithelial cells and colon cancer cells. COPS7B was among the key genes that consistently showed both significant TE increase and protein elevation without transcriptional alteration in CRC. IGF2BP3 enhanced the TE of COPS7B mRNA to promote CRC growth and metastasis. COPS7B was found to be a component of the ribo-interactome that interacted with ribosomes to facilitate ribosome biogenesis and mRNA translation initiation. Collectively, this study revealed the proteomic features of CRC and highlighted elevated mRNA translation as a hallmark of CRC. The identification of the  IGF2BP3-COPS7B axis underlying the increased protein synthesis rate in CRC provided a  promising therapeutic target to treat this aggressive disease.', 'Series_overall_design': 'To investigate the TE changes of global mRNAs in three colon cancer cell lines (HCT116, HCT15 and SW620) compared with colon epithelial cells (841-CON) respectively through Ribo-seq', 'Series_type': 'Other', 'Series_contributor': 'Xiaobo,,Li', 'Series_sample_id': 'GSM7688396', 'Series_contact_name': 'Jing,,Tang', 'Series_contact_email': 'tangjing@hrbmu.edu.cn', 'Series_contact_institute': 'Harbin medical university', 'Series_contact_address': 'No. 157 Baojian Road, Nangang District', 'Series_contact_city': 'Harbin', 'Series_contact_state': 'Heilongjiang', 'Series_contact_zip/postal_code': '150081', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE240nnn/GSE240217/suppl/GSE240217_RAW.tar', 'Series_platform_id': 'GPL20795', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1002994'}\n",
      "642 PRJNA1035318 GSE246955 {'Series_title': 'Translationally inhibited mRNAs control cell movement as untranslated sequences [Ribo-Seq]', 'Series_geo_accession': 'GSE246955', 'Series_status': 'Public on Apr 20 2024', 'Series_submission_date': 'Nov 03 2023', 'Series_last_update_date': 'Apr 20 2024', 'Series_summary': 'Translation of localized mRNAs serves to regulate specific sites of protein synthesis and its functional activity, whereas translationally inhibited and untranslated mRNAs are compartmentalized for degradation or storage, with no recognized functional role. Here, however, we unexpectedly discover a group of functional, untranslated mRNA sequences that localize to integrin focal adhesions (FAs), dynamic multiprotein assemblies that govern cell movement. We show that specific mRNA sequences associate with native or mature FA functional states via messenger ribonucleoprotein (mRNP) complexes with the RNA and FA binding protein G3BP1, functioning to regulate cell migration. Mechanistically, the dimerizing propensity of the mRNA in G3BP1 mRNPs directly regulates FA protein function. Self-dimerizing mRNA sequences form large branched G3BP1 mRNPs, reducing FA protein turnover and extending adhesion lifetimes, inhibiting cell migration. In contrast, low self-dimerization promotes small more spheric G3BP1 mRNPs, facilitating FA complex dissolution and enhancing cell migration behaviors. Our findings define a previously unknown role for cytoplasmic mRNAs, challenging the notion that mRNAs are solely coding molecules and shedding light on their regulatory functions.', 'Series_overall_design': 'Ribosome profiling from whole cell or isolated focal adhesions from HUVECS (human umbilical vein endothelial cells) or HDFs (Human dermal fibroblasts). Ribosomes in active translation were isolated and the ribosome protected fragments were sequenced', 'Series_type': 'Other', 'Series_contributor': 'Stefania,,Nicoli', 'Series_sample_id': 'GSM7881398', 'Series_contact_name': 'Liana,,Boraas', 'Series_contact_email': 'liana.boraas@yale.edu', 'Series_contact_phone': '2037376480', 'Series_contact_institute': 'Yale University', 'Series_contact_address': '300 George Street, Room 752: Loria Center', 'Series_contact_city': 'New Haven', 'Series_contact_state': 'CT', 'Series_contact_zip/postal_code': '06511', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE246nnn/GSE246955/suppl/GSE246955_nicoli_adhesion_rpf_wc_run1_count_5_transcript_cds_v2.csv.gz', 'Series_platform_id': 'GPL24676', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1035318'}\n",
      "643 PRJNA1043570 GSE248349 {'Series_title': 'Sustained hyperglycemia specifically targets translation of mRNAs for insulin secretion', 'Series_geo_accession': 'GSE248349', 'Series_status': 'Public on Dec 01 2023', 'Series_submission_date': 'Nov 21 2023', 'Series_last_update_date': 'Feb 15 2024', 'Series_pubmed_id': '38032734', 'Series_summary': 'Pancreatic beta-cells are specialized for coupling glucose metabolism to insulin peptide production and secretion. Acute glucose exposure robustly and coordinately increases translation of proinsulin and proteins required for secretion of mature insulin peptide. By contrast, chronically elevated glucose levels that occur during diabetes impair beta-cell insulin secretion and have been shown experimentally to suppress insulin translation. Whether translation of other genes critical for insulin secretion are similarly downregulated by chronic high glucose is unknown. Here, we used high-throughput ribosome profiling and nascent proteomics in MIN6 insulinoma cells to elucidate the genome-wide impact of sustained high glucose on beta-cell mRNA translation. Prior to induction of ER stress or suppression of global translation, sustained high glucose suppressed glucose-stimulated insulin secretion and downregulated translation of not only insulin, but also of mRNAs related to insulin secretory granule formation, exocytosis, and metabolism-coupled insulin secretion. Translation of these mRNAs was also downregulated in primary rat and human islets following ex-vivo incubation with sustained high glucose and in an in vivo model of chronic mild hyperglycemia. Furthermore, translational downregulation decreased cellular abundance of these proteins. Our study uncovered a translational regulatory circuit during beta-cell glucose toxicity that impairs expression of proteins with critical roles in beta-cell function.', 'Series_overall_design': 'To determine effect of sustained high glucose on mRNA translation of pancreatic beta-cell, ribosome profiling was performed to measure mRNA translation efficiency based on the proportion of mRNA bound by the ribosome compared to total mRNA per gene. MIN6 cells were incubated in media with 5.5 mM or 25 mM glucose for 24 hours prior to ribosome profiling.', 'Series_type': 'Other', 'Series_contributor': 'Jean,E,Schaffer', 'Series_sample_id': 'GSM7911908', 'Series_contact_name': 'Jonathan,M,Dreyfuss', 'Series_contact_department': 'Bioinformatics & Biostatistics Core', 'Series_contact_institute': 'Joslin Diabetes Center', 'Series_contact_address': '1 Joslin Pl', 'Series_contact_city': 'Boston', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02215', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE248nnn/GSE248349/suppl/GSE248349_RPF_counts.xlsx', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1043570'}\n",
      "644 PRJNA176310 GSE41246 {'Series_title': 'Ribosome profiling in muscle reveals the translation landscape of the giant protein titin', 'Series_geo_accession': 'GSE41246', 'Series_status': 'Public on Sep 01 2015', 'Series_submission_date': 'Oct 01 2012', 'Series_last_update_date': 'May 15 2019', 'Series_summary': 'Conclusions: Our study represents the first detailed analysis of translation in an adult mammalian tissue generated by ribosome profiling technology. Current limitations to using ribosomal profiling in tissues include unknown perturbations to the dynamic state of translation despite rapidly harvested and snap-frozen samples. The uniform 5’ to 3’ coverage observed on individual large mRNAs and the ability to observe footprints on the extremely small phospholamban coding sequence, suggests that initiation and elongation were halted on similar time scales.  More detailed examination of the positional information within CDS region requires further understanding of the bias introduced during the library preparation steps for both RPF-and RNA-Seq, as well as local biases induced as translation is arrested.   Despite these qualifications, this initial view of active translation in muscle tissue highlights the potential for ribosome profiling to monitor the dynamic translation response to exercise, injury or disease pathology in animal models at a level of resolution not easily attainable with other quantitative approaches.', 'Series_overall_design': 'Heart and skeletal muscle ribosome-protected fragment and RNA-Seq profiles of 10-week old C57BL/6J male mice were generated by deep sequencing using the Illumina HiSeq 2000.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Robert,B,Weiss', 'Series_sample_id': 'GSM1012168', 'Series_contact_name': 'Robert,B,Weiss', 'Series_contact_email': 'bob.weiss@genetics.utah.edu', 'Series_contact_phone': '(801) 585-5606', 'Series_contact_laboratory': 'EIHG, Rm 2260B', 'Series_contact_department': 'Human Genetics', 'Series_contact_institute': 'University of Utah', 'Series_contact_address': '15 N 2030 E', 'Series_contact_city': 'Salt Lake City', 'Series_contact_state': 'UT', 'Series_contact_zip/postal_code': '84112', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE41nnn/GSE41246/suppl/GSE41246_TA_RPF-RNA_Abundance.txt.gz', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP015946'}\n",
      "645 PRJNA230057 GSE52799 {'Series_title': 'Polysome profiling and ribosome footprinting of Drosophila mature oocyte and activated egg', 'Series_geo_accession': 'GSE52799', 'Series_status': 'Public on Jun 12 2014', 'Series_submission_date': 'Nov 27 2013', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '24882012', 'Series_summary': 'We use mRNA-seq in combination with polysome profiling to determine translational status for all mRNAs in Drosophila mature oocytes and activated eggs. Puromycin-treated lysates are used as a negative control in polysome profiling experiments. Additionally, we use ribosome footprinting to globally measure translational efficiency of mRNAs in wild type mature oocytes as well as wild type and png mutant activated eggs.', 'Series_overall_design': 'Lysates of hand-dissected Drosophila mature oocytes  (containing ~540 μg of total RNA) were subjected to separation by velocity sedimentation through sucrose gradients. In this way, free mRNAs (present in RNPs fraction) or those comigrating with ribosomal subunits (40S or 60S+80S fractions) or with varying numbers of bound ribosomes (low polysomes (2-4 ribosomes), medium polysomes (5-9 ribosomes), and heavy polysomes (more than 10 ribosomes) can be separated based on their size and collected as sucrose gradient fractions. To compare quantitatively the levels of every mRNA across the polysome gradient fractions, we added 5ng of S. cerevisiae mRNA as an exogenous spike-in to each of the six fractions of interest: RNPs, 40S, 60S+80S, low polysomes, medium polysomes and heavy polysomes. RNA was extraced from these fractions, follwing proteinase K treatment, by hot acid phenol method.  In case of unfractionated lysates, RNA was extracted using TRIzol (Invitrogen) according to manufacturer’s instructions. mRNA-seq samples were prepared from 1 μg of total RNA (in case of sucrose gradient fractions and unfractionated lysates) and subject to Illumina based sequencing. Puromycin-treated lysates of mature oocytes or 0-2h Drosophila activated  eggs (containing ~540 μg of total RNA) were also subjected to separation by velocity sedimentation through sucrose gradients. Puromycin causes premature termination of elongating ribosomes and thus it can be used to determine whether the mRNAs co-sedimenting with the polysomal peaks (defined here as ≥5 ribosomes) were actively engaged in translation. As an independent approach to assess translation and obtain information on the position of ribosomes on mRNAs, we employed ribosome footprinting. In addition to analyzing the same samples, as by polysome profiling, we also analyzed png mutant activated eggs by ribosome footprinting. Ribosome footprint profiling measures the number of ribosome-protected fragments (RPFs) derived from the mRNAs of each gene, resulting in a singular value of translational efficiency (TE) for each gene (TE=RPF/RNA).', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Terry,L,Orr-Weaver', 'Series_sample_id': 'GSM1276400', 'Series_contact_name': 'Iva,,Kronja', 'Series_contact_laboratory': 'Terry Orr-Weaver', 'Series_contact_institute': 'Whitehead Institute for Biomedical Research', 'Series_contact_address': '9 Cambridge Center', 'Series_contact_city': 'Cambridge', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02142', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE52nnn/GSE52799/suppl/GSE52799_RAW.tar', 'Series_platform_id': 'GPL13304', 'Series_platform_taxid': '7227', 'Series_sample_taxid': '7227', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP033366'}\n",
      "646 PRJNA231673 GSE53313 {'Series_title': 'Determinants of ribosome density in yeast', 'Series_geo_accession': 'GSE53313', 'Series_status': 'Public on Dec 19 2013', 'Series_submission_date': 'Dec 13 2013', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '24476825', 'Series_summary': 'Here, we use ribosome-footprint profiing and mRNA-seq to determine the average ribosome density on each gene in S. cerevisiae. We then perform quantitative modeling to identify the molecular determinants of ribosome density.', 'Series_overall_design': 'Analysis of S. cerevisiae', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'David,P,Bartel', 'Series_sample_id': 'GSM1289257', 'Series_contact_name': 'David,E,Weinberg', 'Series_contact_email': 'david.weinberg@ucsf.edu', 'Series_contact_phone': '4155149723', 'Series_contact_laboratory': 'Weinberg', 'Series_contact_institute': 'University of California, San Francisco', 'Series_contact_address': '600 16th Street, GH S572F', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE53nnn/GSE53313/suppl/GSE53313_readMe.txt', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP034076'}\n",
      "647 PRJNA232649 GSE53693 {'Series_title': 'Identification of small ORFs in vertebrates using ribosome footprinting and evolutionary conservation', 'Series_geo_accession': 'GSE53693', 'Series_status': 'Public on Apr 04 2014', 'Series_submission_date': 'Dec 27 2013', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26896445', 'Series_summary': 'Identification of the coding elements in the genome is a fundamental step to understanding the building blocks of living systems. Short peptides (< 100 aa) have emerged as important regulators of development and physiology, but their identification has been limited by their size. We have leveraged the periodicity of ribosome movement on the mRNA to define actively translated ORFs by ribosome footprinting. This approach identifies several hundred translated small ORFs in zebrafish and human. Computational prediction of small ORFs from codon conservation patterns corroborates and extends these findings and identifies conserved sequences in zebrafish and human, suggesting functional peptide products (micropeptides). These results identify micropeptide‐encoding genes in vertebrates, providing an entry point to define their function in vivo.', 'Series_overall_design': 'Ribosome profiling experiments at five timepoints across zebrafish development in WT embryos', 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'Antonio,J,Giraldez', 'Series_sample_id': 'GSM1299109', 'Series_contact_name': 'Antonio,J.,Giraldez', 'Series_contact_email': 'antonio.giraldez@yale.edu', 'Series_contact_phone': '203 785 5423', 'Series_contact_laboratory': 'Giraldez Lab', 'Series_contact_department': 'Genetics', 'Series_contact_institute': 'Yale University', 'Series_contact_address': '333 Cedar Street', 'Series_contact_city': 'New Haven', 'Series_contact_state': 'CT', 'Series_contact_zip/postal_code': '06510', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE53nnn/GSE53693/suppl/GSE53693_allinfo_final_ORFs.txt.gz', 'Series_platform_id': 'GPL14875', 'Series_platform_taxid': '7955', 'Series_sample_taxid': '7955', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP034750'}\n",
      "648 PRJNA242628 GSE56148 {'Series_title': 'A novel IRES identified in DMD results in a functional N-truncated dystrophin, providing a potential route to therapy for patients with 5’ mutations.', 'Series_geo_accession': 'GSE56148', 'Series_status': 'Public on Aug 02 2014', 'Series_submission_date': 'Mar 24 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25108525', 'Series_summary': 'Conclusions: Our results provide a molecular explanation for the rescue of 5’ truncating mutations via a heretofore undescribed mechanism of post-transcriptional regulation of dystrophin expression. The presence of a glucocorticoid-inducible IRES within a highly conserved region of the DMD sequence strongly suggests a programmed role for alternate translation initiation, and ongoing efforts to understand the relevant cell lineage-specific and/or conditional activation signals will shed light on underlying mechanisms of IRES control and elucidate potentially novel functions of dystrophin.', 'Series_overall_design': 'Skeletal muscle ribosome-protected fragment and RNA-Seq profiles from a patient with an NM_004006:c.40_41delGA dystrophin mutation and a normal control were generated by deep sequencing using the Illumina HiSeq 2000.', 'Series_type': 'Other', 'Series_contributor': 'Kevin,M,Flanigan', 'Series_sample_id': 'GSM1356677', 'Series_contact_name': 'Robert,B,Weiss', 'Series_contact_email': 'bob.weiss@genetics.utah.edu', 'Series_contact_phone': '(801) 585-5606', 'Series_contact_laboratory': 'EIHG, Rm 2260B', 'Series_contact_department': 'Human Genetics', 'Series_contact_institute': 'University of Utah', 'Series_contact_address': '15 N 2030 E', 'Series_contact_city': 'Salt Lake City', 'Series_contact_state': 'UT', 'Series_contact_zip/postal_code': '84112', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE56nnn/GSE56148/suppl/GSE56148_NMED-A68360A_Fig1d_DATA.txt.gz', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP040550'}\n",
      "649 PRJNA258219 GSE60426 {'Series_title': 'mRNA destabilization is the dominant effect of mammalian microRNAs by the time substantial repression ensues (sequencing)', 'Series_geo_accession': 'GSE60426', 'Series_status': 'Public on Sep 25 2014', 'Series_submission_date': 'Aug 14 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25263593', 'Series_summary': 'MicroRNAs (miRNAs) regulate target mRNAs through a combination of translational repression and mRNA destabilization, with mRNA destabilization dominating at steady state in the few contexts examined globally. Here, we extend the global steady-state measurements to additional mammalian contexts and find that regardless of the miRNA, cell type, growth condition, or translational state, mRNA destabilization explains most (66%–>90%) miRNA-mediated repression. We also determine the relative dynamics of translational repression and mRNA destabilization for endogenous mRNAs as a miRNA is induced. Although translational repression occurs rapidly, its effect is relatively weak, such that by the time consequential repression ensues, the effect of mRNA destabilization dominates. These results imply that consequential miRNA-mediated repression is largely irreversible and provide other insights into the nature of miRNA-mediated regulation. They also simplify future studies, dramatically extending the known contexts and time points for which monitoring mRNA changes captures most of the direct miRNA effects.', 'Series_overall_design': '85 samples from a variety of cell types and species', 'Series_type': 'Third-party reanalysis', 'Series_contributor': 'David,P,Bartel', 'Series_sample_id': 'GSM1479653', 'Series_contact_name': 'Stephen,,Eichhorn', 'Series_contact_email': 'eichhorn@wi.mit.edu', 'Series_contact_laboratory': 'Bartel', 'Series_contact_department': 'Biology', 'Series_contact_institute': 'Whitehead Institute for Biomedical Research', 'Series_contact_address': '9 Cambridge Center', 'Series_contact_city': 'Cambridge', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02142', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE60nnn/GSE60426/suppl/GSE60426_miR-223_wt_neutrophil.txt.gz', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP045501'}\n",
      "650 PRJNA278009 GSE66809 {'Series_title': 'IFN-g Regulates mTORC1, Cellular Metabolism and mRNA Translation to Potentiate Inflammatory Macrophage Activation [RNA-Seq]', 'Series_geo_accession': 'GSE66809', 'Series_status': 'Public on Jun 29 2015', 'Series_submission_date': 'Mar 12 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26147685', 'Series_summary': 'IFN-g primes macrophages for enhanced inflammatory activation by TLRs and microbial killing, but little is known about the regulation of cell metabolism or mRNA translation during priming. We found that IFN-g regulates macrophage metabolism and translation in an integrated manner by targeting mTORC1 and MNK pathways that converge on the selective regulator of translation initiation eIF4E. Physiological downregulation of the central metabolic regulator mTORC1 by IFN-g was associated with autophagy and translational suppression of repressors of inflammation such as HES1. Genome-wide ribosome profiling in TLR2-stimulated macrophages revealed that IFN-g selectively modulates the macrophage translatome to promote inflammation, further reprogram metabolic pathways, and modulate protein synthesis. These results add IFN-g-mediated metabolic reprogramming and translational regulation as key components of classical inflammatory macrophage activation.', 'Series_overall_design': 'RPF and RNAseq libraries were generated from mock or IFN-g-primed human macrophages. Cells were stimulated with Pam3Cys and harvested at 4 hours. Libraries were generated using protocol modified from Illumina Truseq technology.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Lionel,B,Ivashkiv', 'Series_sample_id': 'GSM1632601', 'Series_contact_name': 'Xiaodi,,Su', 'Series_contact_email': 'xis2003@med.cornell.edu', 'Series_contact_laboratory': 'Ivashkiv Lab', 'Series_contact_department': 'Arthritis and Tissue Degeneration Program', 'Series_contact_institute': 'Hospital for Special Surgery', 'Series_contact_address': '535 East 70th Street', 'Series_contact_city': 'New York', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '10021', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE66nnn/GSE66809/suppl/GSE66809_IFNg_vs_Mock_RPF_RNAseq_Count.xlsx.gz', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP056098'}\n",
      "651 PRJNA286795 GSE69802 {'Series_title': 'Insights into the adaptive stress response of Arabidopsis thaliana to thermal stress by ribosomal profiling', 'Series_geo_accession': 'GSE69802', 'Series_status': 'Public on Oct 13 2016', 'Series_submission_date': 'Jun 11 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27724872', 'Series_summary': 'Environmental stress is detrimental to plants viability and requires an adequate reprogramming of cellular activities to maximize plant survival. We present a global analysis of the adaptive stress response of Arabidopsis thaliana to prolonged heat stress. We combine deep sequencing of RNA and ribosome protected fragments to provide genome wide map of adaptation to heat stress on at transcriptional and translational level. Our analysis shows that the genes with the highest upregulation upon heat stress are known heat-responsive gene, chaperons and other genes involved in protein folding control. Majority of these genes exhibits increase on both transcriptional and translational level. No translational inhibition or ribosome stalling was observed, which can be observed in the early thermal stress response, indicating that plants alter their cellular composition in order to adapt to the prolonged exposure to increased temperatures.', 'Series_overall_design': 'Comparing global transcriptional and translational control by RNA-Seq and Ribosome Profiling (mRNA-Seq of ribosome protected fragments – RPF).', 'Series_type': 'Other', 'Series_contributor': 'Ignatova,,Zoya', 'Series_sample_id': 'GSM1709065', 'Series_contact_name': 'Radoslaw,,Lukoszek', 'Series_contact_email': 'rlukoszek@gmail.com', 'Series_contact_institute': 'University of Potsdam', 'Series_contact_address': 'Karl-Liebknecht-Strasse 24-25', 'Series_contact_city': 'Potsdam', 'Series_contact_zip/postal_code': '14476', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE69nnn/GSE69802/suppl/GSE69802_RAW.tar', 'Series_platform_id': 'GPL13222', 'Series_platform_taxid': '3702', 'Series_sample_taxid': '3702', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP059391'}\n",
      "652 PRJNA288089 GSE70259 {'Series_title': 'The Ski complex binds 80S ribosomes for 3’ to 5’ decay of mRNA', 'Series_geo_accession': 'GSE70259', 'Series_status': 'Public on Feb 10 2017', 'Series_submission_date': 'Jun 25 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27980209', 'Series_summary': 'Cytoplasmic degradation of eukaryotic mRNAs in 3’ to 5’ direction is catalyzed by the exosome complex together with the Ski complex and, in the yeast Saccharomyces cerevisiae, the Ski7 protein. This exosome-Ski-system also degrades ribosome-associated aberrant mRNAs lacking a stop codon, a pathway called non-stop-mRNA decay (NSD). However, the interplay between the Ski components and the ribosome is unknown. Here, we report that the Ski complex can associate with ribosomes in vitro and in vivo independently of Ski7. Ribosome profiling suggests that this association is important for NSD but also for general mRNA turnover. A cryo-electron microscopy structure reveals that the Ski complex binds near the mRNA entry site of the 40S subunit, facilitating the threading of the mRNA into the Ski2 helicase. Collectively, these results demonstrate an unanticipated role for the Ski complex in mRNA decay of ribosome-associated mRNAs, mediated by a direct interaction with the translation machinery.', 'Series_overall_design': 'Ribosome profiling of 80S monosomes and in vivo pullouts of Ski complex associated 80S ribosomes. In vivo pullouts were performed using a yeast strain expressing C-terminally TAP (tandem affinity purification) tagged Ski3. Control mRNA from the total cell lysate and pull-down mRNA from the pullouts were also sequenced respectively.', 'Series_type': 'Other', 'Series_contributor': 'Roland,,Beckmann', 'Series_sample_id': 'GSM1722099', 'Series_contact_name': 'Markus,,Pech', 'Series_contact_email': 'pech@genzentrum.lmu.de', 'Series_contact_phone': '+49 89 218076907', 'Series_contact_laboratory': 'Roland Beckmann', 'Series_contact_department': 'Biochemistry', 'Series_contact_institute': 'Genzentrum, LMU', 'Series_contact_address': '25 Feodor-Lynen Str', 'Series_contact_city': 'Munich', 'Series_contact_zip/postal_code': '81377', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE70nnn/GSE70259/suppl/GSE70259_RAW.tar', 'Series_platform_id': 'GPL20615', 'Series_platform_taxid': '1247190', 'Series_sample_taxid': '1247190', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP059872'}\n",
      "653 PRJNA292861 GSE72064 {'Series_title': 'Genome-wide profilings of transcriptome and translatome in mouse hippocampi after contextual fear conditioning', 'Series_geo_accession': 'GSE72064', 'Series_status': 'Public on Oct 07 2015', 'Series_submission_date': 'Aug 14 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26430118', 'Series_summary': 'Memory stabilization after learning requires transcriptional and translational regulations in the brain, yet the temporal molecular changes following learning have not been explored at the genomic scale. We here employed ribosome profiling and RNA sequencing to quantify the translational status and transcript levels in mouse hippocampus following contextual fear conditioning. We identified 104 genes that are dynamically regulated. Intriguingly, our analysis revealed novel repressive regulations in the hippocampus: translational suppression of ribosomal protein-coding genes at basal state; learning-induced early translational repression of specific genes; and late persistent suppression of a subset of genes via inhibition of ESR1/ERα signaling. Further behavioral analyses revealed that Nrsn1, one of the newly identified genes undergoing rapid translational repression, can act as a memory suppressor gene. This study unveils the yet unappreciated importance of gene repression mechanisms in memory formation.', 'Series_overall_design': 'The application of ribosome profiling and RNA-seq techniques to mouse hippocampi tissues after contextual fear conditioning and to mouse hippocampal primary cultures.  Mouse ESCs were also examined.', 'Series_type': 'Other', 'Series_contributor': 'V,N,Kim', 'Series_sample_id': 'GSM1914374', 'Series_contact_name': 'Jun,,Cho', 'Series_contact_email': 'cjun0205@gmail.com', 'Series_contact_phone': '82-2-880-9120', 'Series_contact_laboratory': \"Narry Kim's lab\", 'Series_contact_department': 'School of Biological Sciences', 'Series_contact_institute': 'Seoul National University', 'Series_contact_address': 'Building 504 Room 509, School of Biological Sciences, Seoul National University, 1 Gwanangno, Gwanak-gu', 'Series_contact_city': 'Seoul', 'Series_contact_zip/postal_code': '08826', 'Series_contact_country': 'South Korea', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE72nnn/GSE72064/suppl/GSE72064_mES_RPF_RNA-seq.csv.gz', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP062407'}\n",
      "654 PRJNA296812 GSE73369 {'Series_title': 'Role of the Small Subunit Processome in the Maintenance of Pluripotent Stem Cells', 'Series_geo_accession': 'GSE73369', 'Series_status': 'Public on Sep 24 2015', 'Series_submission_date': 'Sep 23 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26443847', 'Series_summary': 'RNA-binding proteins (RBPs) play integral roles in gene regulation, yet only a small fraction of RBPs has been studied in the context of stem cells. We here apply RNA interference screen for RBPs in mouse embryonic stem cells (ESCs) and identify 16 RBPs involved in pluripotency maintenance. Interestingly, six identified RBPs including Krr1 and Ddx47 are part of a complex called Small Subunit Processome (SSUP) that mediates 18S rRNA biogenesis. The SSUP components are preferentially expressed in stem cells and enhance global translational rate, which is critical to sustain the protein levels of labile pluripotency factors such as Nanog and Esrrb. Furthermore, the SSUP proteins are required for efficient reprogramming of induced pluripotent stem cells. Our study uncovers the role of SSUP and the importance of translational control in stem cell fate decision.', 'Series_overall_design': 'R1 cells were transfected with 3 different control siRNAs and 3 different siRNAs targeting Krr1 for 48 hrs. Transcriptome profiles were generated by deep sequencing using illumina HiSeq 2000.', 'Series_type': 'Other', 'Series_contributor': 'Joha,,Park', 'Series_sample_id': 'GSM1891969', 'Series_contact_name': 'Joha,,Park', 'Series_contact_email': 'johapark@gmail.com', 'Series_contact_department': 'Biological Sciences', 'Series_contact_institute': 'Seoul National University', 'Series_contact_address': '1 Gwanak-ro, Gwanak-gu', 'Series_contact_city': 'Seoul', 'Series_contact_zip/postal_code': '08826', 'Series_contact_country': 'South Korea', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE73nnn/GSE73369/suppl/GSE73369_RPF-counts.txt.gz', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP064129'}\n",
      "655 PRJNA309995 GSE77292 {'Series_title': 'Ribosomal footprinting of CN34-Parental and CN34-LM1a', 'Series_geo_accession': 'GSE77292', 'Series_status': 'Public on Jun 07 2016', 'Series_submission_date': 'Jan 27 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27259150', 'Series_summary': 'We used Illlumina Artseq Ribo Profile to perform ribosome footprinting on the above cell lines', 'Series_overall_design': \"We followed the manufacturer's protocol: Cells were treated with cycloheximide (0.1 mg/ml) to inhibit translation; total RNA(tt) and ribosome-protected RNA fragments (RPF) were isolated and library preped for sequencing simultaneously.\", 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'S,F,Tavazoie', 'Series_sample_id': 'GSM2048211', 'Series_contact_name': 'Hani,,Goodarzi', 'Series_contact_department': 'Biochemistry and Biophysics', 'Series_contact_institute': 'UCSF', 'Series_contact_address': '600 16th St, GH S312D', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE77nnn/GSE77292/suppl/GSE77292_RAW.tar', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP068967'}\n",
      "656 PRJNA310015 GSE77315 {'Series_title': 'Ribosomal footprinting of MDA-Parental and MDA-LM2', 'Series_geo_accession': 'GSE77315', 'Series_status': 'Public on Jun 07 2016', 'Series_submission_date': 'Jan 27 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27259150', 'Series_summary': 'We used Illlumina Artseq Ribo Profile to perform ribosome footprinting on the above cell lines', 'Series_overall_design': \"We followed the manufacturer's protocol: Cells were treated with cycloheximide (0.1 mg/ml) to inhibit translation; total RNA(tt) and ribosome-protected RNA fragments (RPF) were isolated and library preped for sequencing simultaneously.\", 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'S,F,Tavazoie', 'Series_sample_id': 'GSM2049164', 'Series_contact_name': 'Hani,,Goodarzi', 'Series_contact_department': 'Biochemistry and Biophysics', 'Series_contact_institute': 'UCSF', 'Series_contact_address': '600 16th St, GH S312D', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE77nnn/GSE77315/suppl/GSE77315_RAW.tar', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP068977'}\n",
      "657 PRJNA310096 GSE77317 {'Series_title': 'Ribosomal footprinting of MDA_Ctrl and MDA_Arg overexpression cell lines', 'Series_geo_accession': 'GSE77317', 'Series_status': 'Public on Jun 07 2016', 'Series_submission_date': 'Jan 27 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27259150', 'Series_summary': 'We used Illlumina Artseq Ribo Profile to perform ribosome footprinting on the above cell lines', 'Series_overall_design': \"We followed the manufacturer's protocol: Cells were treated with cycloheximide (0.1 mg/ml) to inhibit translation; total RNA(tt) and ribosome-protected RNA fragments (RPF) were isolated and library preped for sequencing simultaneously.\", 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'S,F,Tavazoie', 'Series_sample_id': 'GSM2049176', 'Series_contact_name': 'Hani,,Goodarzi', 'Series_contact_department': 'Biochemistry and Biophysics', 'Series_contact_institute': 'UCSF', 'Series_contact_address': '600 16th St, GH S312D', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE77nnn/GSE77317/suppl/GSE77317_RAW.tar', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP069008'}\n",
      "658 PRJNA310107 GSE77347 {'Series_title': 'Ribosomal footprinting of MDA_Ctrl and MDA_Glu overexpression cell lines', 'Series_geo_accession': 'GSE77347', 'Series_status': 'Public on Jun 07 2016', 'Series_submission_date': 'Jan 28 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27259150', 'Series_summary': 'We used Illlumina Artseq Ribo Profile to perform ribosome footprinting on the above cell lines', 'Series_overall_design': \"We followed the manufacturer's protocol: Cells were treated with cycloheximide (0.1 mg/ml) to inhibit translation; total RNA and ribosome-protected RNA fragments were isolated and library preped for sequencing simultaneously.\", 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'S,F,Tavazoie', 'Series_sample_id': 'GSM2049927', 'Series_contact_name': 'Hani,,Goodarzi', 'Series_contact_department': 'Biochemistry and Biophysics', 'Series_contact_institute': 'UCSF', 'Series_contact_address': '600 16th St, GH S312D', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE77nnn/GSE77347/suppl/GSE77347_MDA-glu-2-TT_isoforms.fpkm_tracking.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP069011'}\n",
      "659 PRJNA314019 GSE78807 {'Series_title': 'Genome-wide profiling of mTOR-dependent translation in Oikopleura dioica', 'Series_geo_accession': 'GSE78807', 'Series_status': 'Public on Nov 08 2019', 'Series_submission_date': 'Mar 01 2016', 'Series_last_update_date': 'Dec 10 2019', 'Series_pubmed_id': '31783727', 'Series_summary': 'Protein synthesis is an energy-demanding process essential for cell proliferation and survival. Balancing the cost of protein synthesis with available resources has driven the evolution of its nutrient-dependent regulation. A central mechanism in this regulation is the repression of translation of the protein synthesis machinery during unfavorable growth conditions. This is mediated via mammalian target of rapamycin (mTOR), a master regulator of growth conserved from yeast to human. Despite extensive research, and the elucidation of a number of important factors, how mRNAs are translationally regulated by mTOR is still unclear. Repression depends on a 5’ Terminal Oligo Pyrimidine (TOP) motif which is conserved across vertebrates and present in Drosophila melanogaster. In Caenorhabditis elegans and the marine chordate Oikopleura dioica most TOP mRNAs are trans-spliced to a spliced leader. This results in the removal of the originally transcribed 5’ end and its replacement with a common short RNA sequence. In both species the 5’ end of the spliced leader is pyrimidine-enriched but does not meet strict requirements for a canonical TOP motif. How this affects the translational control of TOP mRNAs is unknown. Here, using transcriptome-wide ribosome profiling on whole animals treated with the mTOR inhibitor Torin 1, we show that trans-spliced TOP mRNAs in O. dioica are subject to mTOR-dependent translational control. We also show, using existing data, that trans-spliced transcripts in C. elegans are differentially translated upon recovery from starvation-induced developmental diapause. Together our results demonstrate that spliced leaders in metazoans are targets for mTOR-dependent translational control in response to nutrient availability. This indicates that trans-splicing in metazoans, the function of which has remained largely enigmatic, plays a key role in the coordinated translational regulation of growth-related genes. Moreover, our results reveal an innovative strategy for rapid evolution and developmental control of downstream targets of the ancient mTOR pathway.', 'Series_overall_design': 'Ribosome-protected RNA fragments and total RNA extracted from animals treated with mTOR inhibitor Torin 1 and control animals (DMSO) in three biological replicates (12 samples in total) sequenced on two lanes on each of two Illumina flow cells', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Eric,,Thompson', 'Series_sample_id': 'GSM2078105', 'Series_contact_name': 'Eric,,Thompson', 'Series_contact_email': 'Eric.Thompson@bio.uib.no', 'Series_contact_phone': '+47 55 58 43 46', 'Series_contact_institute': 'Sars International Centre for Marine Molecular Biology', 'Series_contact_address': 'Thormøhlensgt. 55', 'Series_contact_city': 'Bergen', 'Series_contact_zip/postal_code': 'N-5008', 'Series_contact_country': 'Norway', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE78nnn/GSE78807/suppl/GSE78807_count_matrix.txt.gz', 'Series_platform_id': 'GPL21546', 'Series_platform_taxid': '34765', 'Series_sample_taxid': '34765', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP071049'}\n",
      "660 PRJNA326506 GSE83616 {'Series_title': 'mRNA Poly(A)-tail Changes Specified by Deadenylation Broadly Reshape Translation in Drosophila Oocytes and Early Embryos', 'Series_geo_accession': 'GSE83616', 'Series_status': 'Public on Aug 08 2016', 'Series_submission_date': 'Jun 22 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27474798', 'Series_summary': 'Because maturing oocytes and early embryos lack transcription, posttranscriptional regulatory processes must control their development.  To better understand this control, we profiled translational efficiencies and poly(A)-tail lengths throughout Drosophila oocyte maturation and early embryonic development.  The correspondence between translational-efficiency changes and tail-length changes indicated that tail-length changes broadly reshape translational activity until gastrulation, when this coupling disappears.  Relative changes were largely retained in the absence of poly(A)-tail lengthening, which indicated that selective poly(A)-tail shortening primarily specifies the changes.  Many translational changes depended on PAN GU and Smaug, and both acted primarily through tail-length changes.  Our results also revealed tail-length–independent mechanisms of translational control that repressed translation regardless of tail-length changes during oocyte maturation, maintained translation despite tail-length shortening during oocyte maturation, and prevented detectable translation of bicoid and several other mRNAs before egg activation.  In addition to these fundamental insights, our results provide valuable resources for future studies.', 'Series_overall_design': '42 samples analyzed using RNA-seq, ribosome footprint profiling, and PAL-seq.', 'Series_type': 'Other', 'Series_contributor': 'David,P,Bartel', 'Series_sample_id': 'GSM2211007', 'Series_contact_name': 'Stephen,,Eichhorn', 'Series_contact_email': 'eichhorn@wi.mit.edu', 'Series_contact_laboratory': 'Bartel', 'Series_contact_department': 'Biology', 'Series_contact_institute': 'Whitehead Institute for Biomedical Research', 'Series_contact_address': '9 Cambridge Center', 'Series_contact_city': 'Cambridge', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02142', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE83nnn/GSE83616/suppl/GSE83616_dm6_unique_nooverlap_longest_final.txt.gz', 'Series_platform_id': 'GPL17275', 'Series_platform_taxid': '7227', 'Series_sample_taxid': '7227', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP076919'}\n",
      "661 PRJNA342004 GSE86466 {'Series_title': 'Proteomic and genomic characterization of a yeast model for Ogden syndrome [RNA-seq and Ribo-seq]', 'Series_geo_accession': 'GSE86466', 'Series_status': 'Public on Sep 07 2016', 'Series_submission_date': 'Sep 06 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '27668839', 'Series_summary': 'Naa10 is a Nα-terminal acetyltransferase that, in a complex with its auxiliary subunit Naa15, co-translationally acetylates the α-amino group of newly synthetized proteins as they emerge from the ribosome. Roughly 40-50% of the human proteome is acetylated by Naa10, rendering this an enzyme with one of the most broad substrate ranges known. Recently, we reported an X-linked disorder of infancy, Ogden syndrome, in two families harboring a c.109T>C (p.Ser37Pro) variant in NAA10. In the present study we performed in-depth characterization of a yeast model of Ogden syndrome. Stress tests and proteomic analyses suggest that the S37P mutation disrupts Naa10 function and reduces cellular fitness during heat shock, possibly due to dysregulation of chaperone expression and accumulation. Microarray and RNA-seq revealed a pseudo-diploid gene expression profile in ΔNaa10 cells, likely responsible for a mating defect. In conclusion, the data presented here further support the disruptive nature of the S37P/Ogden mutation and identify affected cellular processes potentially contributing to the severe phenotype seen in Ogden syndrome.', 'Series_overall_design': 'RNA-seq, microarray', 'Series_type': 'Other', 'Series_contributor': 'Gholson,J,Lyon', 'Series_sample_id': 'GSM2303103', 'Series_contact_name': 'Gholson,,Lyon', 'Series_contact_email': 'gholsonjlyon@gmail.com', 'Series_contact_phone': '646-872-1219', 'Series_contact_institute': 'Cold Spring Harbor Laboratory', 'Series_contact_address': '500 Sunnyside Blvd', 'Series_contact_city': 'Woodbury', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '11797', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE86nnn/GSE86466/suppl/GSE86466_RAW.tar', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP086703'}\n",
      "662 PRJNA349752 GSE89011 {'Series_title': 'Nat1 promotes translation of specific proteins that induce differentiation of mouse embryonic stem cells [ribosome profiling]', 'Series_geo_accession': 'GSE89011', 'Series_status': 'Public on Jan 06 2017', 'Series_submission_date': 'Oct 20 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28003464', 'Series_summary': 'Nat1 (also known as p97/Dap5/Eif4g2) is a ubiquitously expressed cytoplasmic protein that is homologous to the C-terminal two thirds of eukaryotic translation initiation factor 4G (also known as Eif4g1). We previously showed that Nat1-null mouse embryonic stem cells (mESCs) were resistant to differentiation. In the current study, we found that Nat1 and Eif4g1 share many binding proteins, such as Eif3s, Eif4s and ribosomal proteins. However, Nat1 did not bind to Eif4e or poly A binding proteins, which are critical for cap-dependent translation initiation. In contrast, Nat1 binds to Eif2s, Fmr and related proteins, and Prrc2 proteins more preferentially than does Eif4g1. We also found that Nat1-null mESCs possess a status partially similar to ground state, which is established in wild-type mES cells when treated with inhibitors of the ERK and GSK3 signaling pathways. In Nat1-null mESCs, the ERK pathway is suppressed even without inhibitors. Ribosomal profiling revealed that translation of Map3k3 and Sos1 is suppressed in the absence of Nat1. Forced expression of Map3k3 induced differentiation of Nat1-null mESCs. These data collectively showed that Nat1 is involved in translation of proteins that are required for cell differentiation.', 'Series_overall_design': 'Examining ribosome-protected mRNA fragments (RPF) and total RNA in replicate WT ESCs (n=2), Nat1+/- mESCs (n=3) and Nat1-null ESCs (n=3).', 'Series_type': 'Other', 'Series_contributor': 'K,,Takahashi', 'Series_sample_id': 'GSM2357227', 'Series_contact_name': 'Shinya,,Yamanaka', 'Series_contact_email': 'yamanaka@cira.kyoto-u.ac.jp', 'Series_contact_phone': '81-75-366-7041', 'Series_contact_institute': 'Center for iPS Cell Research and Applicaton (CiRA), Kyoto University', 'Series_contact_address': '53 Kawahara-machi, Shogoin, Sakyo-ku', 'Series_contact_city': 'Kyoto', 'Series_contact_zip/postal_code': '606-8507', 'Series_contact_country': 'Japan', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE89nnn/GSE89011/suppl/GSE89011_NormalizedCounts.txt.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP091889'}\n",
      "663 PRJNA350543 GSE89183 {'Series_title': 'Consequences of Ribosomal Protein Haploinsufficiency in Human Hematopoiesis', 'Series_geo_accession': 'GSE89183', 'Series_status': 'Public on Feb 25 2018', 'Series_submission_date': 'Oct 25 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '29551269', 'Series_summary': 'Ribosomal protein haploinsufficiency (RPH) underlies diverse human diseases with distinct and specific phenotypes, including Diamond-Blackfan anemia (DBA). Although multiple mechanisms have been proposed for the erythroid-specific hematopoietic defects observed in DBA, only recently has the role of selectively impaired translation been highlighted in these phenotypes. Exactly how and to what extent this impairment of translation occurs is currently unknown. Here, by identifying a novel DBA gene affecting ribosome biogenesis, we show that both RPH and impaired ribosome biogenesis (IRB) limit the availability of actively translating ribosomes, resulting in the hematopoietic and translational defects observed in DBA. Our results show that the selective impairment of translation is due to a quantitative defect, where ribosomes of invariant protein composition have a reduced abundance, rather than a qualitative defect, where a subset of ribosomes lack specific ribosomal proteins (RPs) and thus may have altered translational capacity. In RPH, we find that cellular RP homeostasis is largely maintained through translational co-regulation, and we identify a selective subset of transcripts that have impaired association with the ribosome. Surprisingly, these transcripts have short and unstructured 5’ UTRs and are highly abundant and efficiently translated in healthy human erythroid progenitors, suggesting that the impaired translation of a number of key transcripts, including GATA1, may underlie DBA. Overall, our study identifies mechanisms by which RPH and IRB affect mRNA translation, illuminating how these alterations can result in cell-type specific defects and cause human disease.', 'Series_overall_design': '(mRNA-seq) RNA-seq for control (shLuc), RPL5 knockdown (shRPL5), RPS19 knockdown (shRPS19), and TSR2 knockdown (shTSR2) was performed for two biological replicates of primary human erythroid cells at Day 7 of in vitro culture. (RPF-seq) Ribosome profiling was similarly performed for shLuc, shRPL5, and shRPS19 in the matching biological replicates.', 'Series_type': 'Other', 'Series_contributor': 'Vijay,G,Sankaran', 'Series_sample_id': 'GSM2360180', 'Series_contact_name': 'Vijay,G,Sankaran', 'Series_contact_email': 'sankaran@broadinstitute.org', 'Series_contact_institute': 'Broad Institute', 'Series_contact_address': '7 Cambridge Center', 'Series_contact_city': 'Cambridge', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02142', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE89nnn/GSE89183/suppl/GSE89183_Counts.txt.gz', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP092068'}\n",
      "664 PRJNA408071 GSE104033 {'Series_title': 'The dual methyltransferase METTL13 targets N-terminus and Lys55 of eEF1A and regulates codon-specific translation rates', 'Series_geo_accession': 'GSE104033', 'Series_status': 'Public on Jul 04 2018', 'Series_submission_date': 'Sep 20 2017', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '30143613', 'Series_summary': 'Eukaryotic elongation factor 1 alpha (eEF1A) delivers aminoacyl-tRNA to the ribosome and thereby plays a key role in protein synthesis. Human eEF1A is post-translationally methylated on several lysine residues as well as on the N-terminus and until recently the responsible methyltransferases were largely elusive. Using mass spectrometry based screens, gene targeted cells and in vitro enzymology we identify the human methyltransferase like proteins 13 (METTL13), which contains two distinct methyltransferase domains, as an enzyme catalyzing both dimethylation of Lys55 and trimethylation of the N-terminus of eEF1A. Moreover, through ribosome profiling we demonstrate that loss of METTL13 function alters translation dynamics and results in changed translation rate of specific codons. In line with the established descriptive nomenclature for this type of enzymes we suggest that METTL13 is redubbed eEF1A dual methyltransferase (eEF1A-DMT / EEF1ADMT).', 'Series_overall_design': 'ribosome profiling and poly(A) RNA sequencing of wildtype and MELL13-KO cells. Libraries were prepared as triplicates, except for wt ribosome profiling libraries that are duplicates only.', 'Series_type': 'Other', 'Series_contributor': 'Pål,Ø,Falnes', 'Series_sample_id': 'GSM2788011', 'Series_contact_name': 'Benedikt,Sebastian,Nilges', 'Series_contact_email': 'benedikt.nilges@mpi-muenster.mpg.de', 'Series_contact_department': 'RNA Biology Group', 'Series_contact_institute': 'Max Planck Institute for Molecular Biomedicine', 'Series_contact_address': 'Von-Esmarch-Straße 54', 'Series_contact_city': 'Muenster', 'Series_contact_zip/postal_code': '48149', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE104nnn/GSE104033/suppl/GSE104033_CountTable.csv.gz', 'Series_platform_id': 'GPL15456', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP118278'}\n",
      "665 PRJNA408093 GSE104043 {'Series_title': 'AGO CLIP reveals an activated network for acute regulation of brain glutamate homeostasis in ischemic stroke [dataset 4]', 'Series_geo_accession': 'GSE104043', 'Series_status': 'Public on Jun 14 2019', 'Series_submission_date': 'Sep 20 2017', 'Series_last_update_date': 'Jun 14 2019', 'Series_summary': 'Adult mouse cortical tissues after MCAO or sham surgery were analyzed by ribosomal profiling to assess differences in ribosome associated mRNA fragments in I/R injury model of stroke.', 'Series_overall_design': 'Mice were anesthetized briefly with 1.5–2% isoflurane.  A heat-blunted nylon suture (6/0) was inserted into the right external carotid artery and advanced until it obstructed the MCA together with the ligation of the common carotid artery for 35 min.  Regional cerebral blood flow (CBF, bregma coordinates: 2-mm posterior, 5-mm lateral) was continuously recorded by transcranial laser Doppler flowmetry from the induction of ischemia until 30 min after reperfusion.  Surgery success was determined by monitoring regional cerebral blood flow (CBF, bregma coordinates: 2-mm posterior, 5-mm lateral) using transcranial laser Doppler flowmetry, and required a residual CBF <15%, followed by >80% recovery within 10 min of reperfusion.   Following MCAO, mice were placed in temperature-controlled recovery cages for 3, 6, 12 or 24 hrs to prevent post-surgery hypothermia and sacrificed.  Sham animals were similarly anestheized, arteries exposed for surgical time period without filament insertion.  Cortical tissues from ipsilateral or contralateral hemispheres of MCAO or sham animals were dissected for MNase digestion, monosome purificaton, large ribosomal subunit (P0) immunoprecipitaton, RPF (ribosome protected fragment) extraction and library construction.  Each set of biological replicates represent animals from a distinct day of surgery.  Complete protocol detailed in referenced publication.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Mariko,,Kobayashi', 'Series_sample_id': 'GSM2788282', 'Series_contact_name': 'Mariko,,Kobayashi', 'Series_contact_email': 'mkobayashi@rockefeller.edu', 'Series_contact_laboratory': 'Robert B. Darnell', 'Series_contact_department': 'Molecular Neuro-Oncology', 'Series_contact_institute': 'Rockefeller University', 'Series_contact_address': '1230 York Avenue', 'Series_contact_city': 'New York', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '10065', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE104nnn/GSE104043/suppl/GSE104043_RPF_counts.txt.gz', 'Series_platform_id': 'GPL16417', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP118295'}\n",
      "666 PRJNA412370 GSE104329 {'Series_title': 'Ribosomal coverage with codon resulation in response to RPL12 siRNA treatment vs. no treatment', 'Series_geo_accession': 'GSE104329', 'Series_status': 'Public on May 17 2020', 'Series_submission_date': 'Sep 27 2017', 'Series_last_update_date': 'Jul 25 2021', 'Series_pubmed_id': '31657788', 'Series_summary': 'Ribosome Profiling was employed to learn about Ribosome A-site occupancies in response to uL11 siRNA treatment or scrambled siRNA treatment in Cystic Fibrosis Bronchial Epithelial (CFBE) cells.', 'Series_overall_design': 'Ribosome Profiling of cells 96h after siRNA transfection', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Eric,J,Sorscher', 'Series_sample_id': 'GSM2795500', 'Series_contact_name': 'Zoya,,Ignatova', 'Series_contact_email': 'zoya.ignatova@chemie.uni-hamburg.de', 'Series_contact_institute': 'Universitaet Hamburg', 'Series_contact_address': 'Martin -Luther-King-Platz 6', 'Series_contact_city': 'Hamburg', 'Series_contact_zip/postal_code': '20146', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE104nnn/GSE104329/suppl/GSE104329_RAW.tar', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP118987'}\n",
      "667 PRJNA412979 GSE104503 {'Series_title': 'Ribosome-profiling experiments for “Chronic repression by MAF1 supports futile RNA cycling as a mechanism for obesity resistance”', 'Series_geo_accession': 'GSE104503', 'Series_status': 'Public on Nov 04 2019', 'Series_submission_date': 'Oct 02 2017', 'Series_last_update_date': 'Jul 25 2021', 'Series_pubmed_id': '32686713', 'Series_summary': 'To characterize the translation defects in Maf1-/- samples, we used ribosome profiling on three WT and three Maf1-/- fed livers. Each sample was spiked with a fixed amount of Drosophila S2 Schneider cell material as an internal control for sample-to-sample normalization, and to allow detection of global changes in translation.', 'Series_overall_design': 'Two conditions (WT of Maf1 KO mice), three biological replicates per condition. For each sample, both Ribosome-protected fragments (RPFL) and total RNA fragments (TOTL) were sequenced.', 'Series_type': 'Other', 'Series_contributor': 'Nouria,,Hernandez', 'Series_sample_id': 'GSM2802547', 'Series_contact_name': 'Nicolo,,Riggi', 'Series_contact_laboratory': 'Institut universitaire de pathologie', 'Series_contact_department': 'Département de Pathologie Expérimentale', 'Series_contact_institute': 'CHUV', 'Series_contact_address': 'Bugnon 25', 'Series_contact_city': 'Lausanne', 'Series_contact_state': 'VD', 'Series_contact_zip/postal_code': '1011', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE104nnn/GSE104503/suppl/GSE104503_RAW.tar', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP119310'}\n",
      "668 PRJNA414090 GSE104884 {'Series_title': \"A to I editing of the 3'-UTR landscape regulates gene translation during retinal development\", 'Series_geo_accession': 'GSE104884', 'Series_status': 'Public on Oct 01 2023', 'Series_submission_date': 'Oct 12 2017', 'Series_last_update_date': 'Oct 01 2023', 'Series_summary': 'We used parallel ribosome profiling and RNA sequencing to investigate the relationship between RNA editing and translation. Through systematically profiling mouse tissues from different developmental stages, we found dynamic changes in A-to-I RNA editome during retinal development. Further analysis revealed that altered translational efficiency in retinal development correlates with dynamics of A-to-I RNA editome.', 'Series_overall_design': 'RNAseq and ribosome profiling were performed for 7 times during mouse retina development and two repllicates per time. A total of 28 barcoded libraries were pooled and single-end 51 bp and strand-specific pair-end 125bp sequencing was performed on an Illumina HiSeq 2500 for ribsome pforiling and RNAseq.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Zhi,,Xie', 'Series_sample_id': 'GSM2809456', 'Series_contact_name': 'Hongwei,,Wang', 'Series_contact_email': 'bioccwhw@126.com', 'Series_contact_phone': '+86 20 6667 7086', 'Series_contact_laboratory': 'XieLab', 'Series_contact_department': 'Zhongshan Ophthalmic Center', 'Series_contact_institute': 'Sun Yat-Sen University', 'Series_contact_address': 'No 7 Jinsui Rd, Research Building, Rm 301', 'Series_contact_city': 'Guangzhou', 'Series_contact_state': 'Guangdong', 'Series_contact_zip/postal_code': '510060', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE104nnn/GSE104884/suppl/GSE104884_RPKM_Matrix_Riboseq.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP119847'}\n",
      "669 PRJNA414611 GSE105082 {'Series_title': 'RNA G-quadruplexes mark repressive upstream open reading frames in human mRNAs', 'Series_geo_accession': 'GSE105082', 'Series_status': 'Public on Apr 30 2018', 'Series_submission_date': 'Oct 17 2017', 'Series_last_update_date': 'Jul 25 2021', 'Series_pubmed_id': '30591072', 'Series_summary': 'Translational control is a key determinant of protein abundance, which in turns defines the physiology and pathology of human cells. Initiation of translation is highly regulated in eukaryotes and is considered as the rate-limiting step of protein synthesis. mRNA secondary structures in 5’ untranslated region (UTR) and associated helicases have been characterised as key determinants of translation initiation. Nevertheless the transcriptome-wide contribution of non-canonical secondary structures, such as RNA G-quadruplexes (rG4s), to the translation of human mRNAs remains largely unappreciated. Here we use a ribosome profiling strategy to investigate the translational landscape associated to rG4s-containing mRNAs and the contribution of two rG4s-specialised DExH-box helicases, DHX9 and DHX36, to translation initiation in human cells. We show that rG4-forming sequences in 5’-UTR is associated with decreased translation efficiency which correlate with an increased ribosome density within the 5’-UTRs. We found that rG4s contribute to the translation of upstream open reading frames, and as a consequence, thwart the translation of the associated protein coding sequences (CDS). Depletion of the DHX36 and DHX9 helicases demonstrated that the formation of the rG4 structural motif rather than its nucleotide sequence mediate translation initiation. Our findings unveil a role for non-canonical structures in defining alternative 5’ starts for human mRNAs translation initiation.', 'Series_overall_design': '11 Ribo-Seq and RNA-Seq matched samples; 1 iCLIP libraries containing two barcoded replicates', 'Series_type': 'Other', 'Series_contributor': 'Shankar,,Balasubramanian', 'Series_sample_id': 'GSM2817700', 'Series_contact_name': 'Giovanni,,Marsico', 'Series_contact_email': 'persego@gmail.com', 'Series_contact_institute': 'CRUK Cambridge Institute', 'Series_contact_address': 'Robinson Way', 'Series_contact_city': 'Cambridge', 'Series_contact_zip/postal_code': 'CB2 0RE', 'Series_contact_country': 'United Kingdom', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE105nnn/GSE105082/suppl/GSE105082_RAW.tar', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP120157'}\n",
      "670 PRJNA417264 GSE106572 {'Series_title': 'Accurate Design of Translational Output by a Neural Network Model of Ribosome Distribution', 'Series_geo_accession': 'GSE106572', 'Series_status': 'Public on Nov 07 2017', 'Series_submission_date': 'Nov 06 2017', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '29967537', 'Series_summary': 'We present a ribosome profiling data set from S. cerevisiae as part of a larger study that models the elongation rate along transcripts as a function of local sequence neighborhoods. We then apply this model to design coding sequences that are optimized for translational output.', 'Series_overall_design': 'Two lanes of sequencing from one ribosome profiling experiment', 'Series_type': 'Other', 'Series_contributor': 'Liana,F,Lareau', 'Series_sample_id': 'GSM2842830', 'Series_contact_name': 'Robert,,Tunney', 'Series_contact_department': 'Computational Biology', 'Series_contact_institute': 'University of California, Berkeley', 'Series_contact_address': 'Barker Hall', 'Series_contact_city': 'Berkeley', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94709', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE106nnn/GSE106572/suppl/GSE106572_RAW.tar', 'Series_platform_id': 'GPL21656', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP124252'}\n",
      "671 PRJNA431807 GSE109734 {'Series_title': 'Xrn1 activates transcription, translation and decay of mRNAs encoding membrane proteins', 'Series_geo_accession': 'GSE109734', 'Series_status': 'Public on Feb 08 2019', 'Series_submission_date': 'Jan 26 2018', 'Series_last_update_date': 'Apr 02 2019', 'Series_pubmed_id': '30899024', 'Series_summary': 'To study the role of the exonuclease Xrn1 in translational control, we performed ribosome profiling and RNA-seq in Xrn1-depleted cells. By using an auxin-inducible degron, we were able to study immediate effects of Xrn1 depletion in translational control. Therefore, we could overcome experimental limitations associated to stable deletion mutants.', 'Series_overall_design': 'mRNA and ribosome footprints were obtained as described in (Ingolia, 2009) from BY25598 cells (Nishimura and Kanemaki, 2014) modified to express an Xrn1 protein tagged with an auxin inducible degron (AID). Libraries were obtained from cells treated with auxin for 30 minutes (t30) and compared from libraries from untreated controls (t0). After 30 minutes of treatment, Xrn1 protein was reduced to less than 10%. For the Xrn1-D208A mutant, a BY4741 background was used. Wild-type and mutant strains were grown to logarithmic phase and collected at approximately 0.5 OD.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Maria-Eugenia,,Gas', 'Series_sample_id': 'GSM2958784', 'Series_contact_name': 'Juana,,Díez', 'Series_contact_email': 'juana.diez@upf.edu', 'Series_contact_phone': '+34 933160862', 'Series_contact_laboratory': 'Molecular Virology Group', 'Series_contact_department': 'Department of Experimental and Health Sciences', 'Series_contact_institute': 'Universitat Pompeu Fabra', 'Series_contact_address': 'Carrer del Dr. Aiguader 88', 'Series_contact_city': 'Barcelona', 'Series_contact_state': 'Barcelona', 'Series_contact_zip/postal_code': '08003', 'Series_contact_country': 'Spain', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE109nnn/GSE109734/suppl/GSE109734_raw_counts_mRNA.tsv.gz', 'Series_platform_id': 'GPL19958', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP131527'}\n",
      "672 PRJNA475675 GSE115647 {'Series_title': 'The LIN28B-IMP1 post-transcriptional regulon has opposing effects on oncogenic  signaling in the intestine', 'Series_geo_accession': 'GSE115647', 'Series_status': 'Public on Dec 27 2018', 'Series_submission_date': 'Jun 12 2018', 'Series_last_update_date': 'Mar 27 2019', 'Series_pubmed_id': '30068703', 'Series_summary': 'Conclusions: Together, our data provide new evidence for the opposing effects of the LIN28B-IMP1 axis on post-transcriptional regulation of canonical WNT signaling, with implications in intestinal homeostasis, regeneration and tumorigenesis.', 'Series_overall_design': 'Ribosome-footprinting and RNA-seq samples from LIN28B OE SW480 cells and LIN28B OE+IMP1-/- knockout cells', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'A,K,Rustgi', 'Series_sample_id': 'GSM3186651', 'Series_contact_name': 'Premal,,Shah', 'Series_contact_email': 'premal.shah@rutgers.edu', 'Series_contact_laboratory': 'Shah Lab', 'Series_contact_department': 'Genetics', 'Series_contact_institute': 'Rutgers University', 'Series_contact_address': '145 Bevier Road, Rm 326', 'Series_contact_city': 'Piscataway', 'Series_contact_state': 'NJ', 'Series_contact_zip/postal_code': '08854', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE115nnn/GSE115647/suppl/GSE115647_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP150308'}\n",
      "673 PRJNA477799 GSE116221 {'Series_title': 'Charting DENR-dependent translation reinitiation uncovers predictive uORF features and links to circadian timekeeping via Clock', 'Series_geo_accession': 'GSE116221', 'Series_status': 'Public on Apr 01 2019', 'Series_submission_date': 'Jun 25 2018', 'Series_last_update_date': 'Jun 05 2019', 'Series_pubmed_id': '30982898', 'Series_summary': \"The non-canonical initiation factor DENR promotes translation reinitiation on uORF-containing mRNAs. Moreover, DENR depletion shortens circadian period in mouse fibroblasts, suggesting that uORF usage and reinitiation regulate clock function. To identify DENR-regulated translation events transcriptome-wide and, in particular, specific core clock transcripts affected by this mechanism, we have used ribosome profiling in DENR-deficient NIH3T3 cells. We found 240 transcripts with altered translation rate, and used linear regression analysis to extract uORF features predictive of DENR dependance. Among core clock genes, we identified Clock as a DENR target. Using Clock 5'UTR mutants, we mapped the specific uORF through which DENR acts to regulate CLOCK protein biosynthesis. Notably, these experiments identified an alternative downstream start codon, which likely represents the true CLOCK N-terminus. Our findings provide insights into uORF-mediated translational regulation that can regulate the mammalian circadian clock and gene expression at large.\", 'Series_overall_design': 'Denr knock-down or control NIH3t3 cells were grown to subconfluency in 15 cm plates/replicate. Cells were pre-treated with cycloheximide for 2 min, prior to lysate preparation. Two different sh-RNAs were used for knock-down experiments, and  Ribosome profiling (RPF-seq) and parallel RNA-seq libraries were prepared in triplicate for each. For control, non-targeting scramble sh-RNA in triplicate,  and Gfp-shRNA in duplicate was used.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'David,,Gatfield', 'Series_sample_id': 'GSM3214200', 'Series_contact_name': 'David,,Gatfield', 'Series_contact_email': 'david.gatfield@unil.ch', 'Series_contact_department': 'Center for Integrative Genomics', 'Series_contact_institute': 'University of Lausanne', 'Series_contact_address': 'Genopode', 'Series_contact_city': 'Lausanne', 'Series_contact_zip/postal_code': '1015', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE116nnn/GSE116221/suppl/GSE116221_RAW.tar', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP151259'}\n",
      "674 PRJNA486442 GSE118676 {'Series_title': 'Stoichiometric Production of Protein-Complex Components in Eukaryotes Without Widespread Feedback Regulation', 'Series_geo_accession': 'GSE118676', 'Series_status': 'Public on Dec 03 2018', 'Series_submission_date': 'Aug 17 2018', 'Series_last_update_date': 'Aug 15 2021', 'Series_pubmed_id': '30553725', 'Series_summary': 'Constituents of multi-protein complexes are required at well-defined levels relative to each other. However, it remains unknown whether eukaryotic cells typically produce precise amounts of subunits, or instead rely on degradation to mitigate imprecise production. Here we quantified the production rates of multi-protein complexes in single- and multi-cellular eukaryotes using ribosome profiling. By resolving read-mapping ambiguities present in a large fraction of ribosome footprints which often distorts quantitation accuracy in eukaryotes, we found that obligate components of multi-protein complexes are produced in proportion to their stoichiometry, indicating that their abundances are already precisely tuned at the synthesis level. By systematically interrogating the impact of gene dosage variations in budding yeast, we found a general lack of negative feedback regulation protecting the normally precise rates of subunit synthesis. These results reveal a core principle of proteome homeostasis and highlight the evolution towards quantitative control at every stage of the central dogma.', 'Series_overall_design': '13 ribosome profiling experiments in Saccharomyces cerevisiae', 'Series_type': 'Other', 'Series_contributor': 'Gene-Wei,,Li', 'Series_sample_id': 'GSM3336496', 'Series_contact_name': 'James,Christopher,Taggart', 'Series_contact_email': 'james_taggart@hms.harvard.edu', 'Series_contact_laboratory': 'Allon Klein', 'Series_contact_department': 'Systems Biology', 'Series_contact_institute': 'Harvard Medical School', 'Series_contact_address': '200 Longwood Avenue', 'Series_contact_city': 'Boston', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02115', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE118nnn/GSE118676/suppl/GSE118676_RAW.tar', 'Series_platform_id': 'GPL19756', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP158187'}\n",
      "675 PRJNA489895 GSE119681 {'Series_title': 'Ribosome profiling of selenoproteins in vivo reveals consequences of pathogenic Secisbp2 missense mutations', 'Series_geo_accession': 'GSE119681', 'Series_status': 'Public on Jul 26 2019', 'Series_submission_date': 'Sep 07 2018', 'Series_last_update_date': 'Oct 08 2019', 'Series_pubmed_id': '31350336', 'Series_summary': 'Re-coding of UGA codons as selenocysteine (Sec) codons in selenoproteins depends on a selenocysteine insertion sequence (SECIS) in the 3’ UTR of mRNAs of eukaryotic selenoproteins. SECIS-binding protein 2 (SECISBP2) increases the efficiency of this process. Pathogenic mutations in SECISBP2 reduce selenoprotein expression and lead to phenotypes associated with the reduction of deiodinase activities and selenoprotein N expression in humans. Two functions have been ascribed to SECISBP2: binding of SECIS elements in selenoprotein mRNAs and facilitation of co-translational Sec insertion. To separately probe both functions, we established here two mouse models carrying two pathogenic missense mutations in Secisbp2 previously identified in patients. We found that the C696R substitution in the RNAbinding domain abrogates SECIS binding and does not support selenoprotein translation above the level of a complete Secisbp2 null mutation. The R543Q missense substitution located in the selenocysteine insertion domain resulted in residual activity and caused reduced selenoprotein translation, as demonstrated by ribosomal profiling to determine the impact on UGA re-coding in individual selenoproteins. We found, however, that the R543Q variant is thermally unstable in vitro and completely degraded in the mouse liver in vivo, while being partially functional in the brain. The moderate impairment of selenoprotein expression in neurons led to astrogliosis and transcriptional induction of genes associated with immune responses. We conclude that differential SECISBP2 protein stability in individual cell types may dictate clinical phenotypes to a much greater extent than molecular interactions involving a mutated amino acid in SECISBP2 .', 'Series_overall_design': 'Ribosome profiling and RNA-sequencing of cortex samples from 2 Secisbp2 R543Q/fl; Camk-Cre pos mice and 2 control mice.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Michael,T,Howard', 'Series_sample_id': 'GSM3380729', 'Series_contact_name': 'Ulrich,,Schweizer', 'Series_contact_email': 'uschweiz@uni-bonn.de', 'Series_contact_laboratory': 'Schweizer Lab', 'Series_contact_department': 'IBMB', 'Series_contact_institute': 'Universität Bonn', 'Series_contact_address': 'Nussallee 11', 'Series_contact_city': 'Bonn', 'Series_contact_zip/postal_code': '53115', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE119nnn/GSE119681/suppl/GSE119681_RAW.tar', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP160398'}\n",
      "676 PRJNA492921 GSE120383 {'Series_title': 'Ribosome profiling of MCF10A cells with tRNA-Tyr-GUA depletion by shRNA relative to shControl cells', 'Series_geo_accession': 'GSE120383', 'Series_status': 'Public on Sep 25 2019', 'Series_submission_date': 'Sep 24 2018', 'Series_last_update_date': 'Sep 25 2019', 'Series_summary': 'We report the changes in ribosome protected fragments  as measured by ribosome profiling in cells with or without depletion of tRNA-Tyr-GUA', 'Series_overall_design': 'MCF10A cells with an shRNA against tRNA-Tyr-GUA or a control hairpin were subjected to ribosome profiling', 'Series_type': 'Other', 'Series_contributor': 'Sohail,F,Tavazoie', 'Series_sample_id': 'GSM3399570', 'Series_contact_name': 'Doowon,,Huh', 'Series_contact_email': 'dhuh01@rockefeller.edu', 'Series_contact_phone': '2123277220', 'Series_contact_laboratory': 'Tavazoie Lab', 'Series_contact_institute': 'Rockefeller University', 'Series_contact_address': '1230 York Avenue', 'Series_contact_city': 'New York', 'Series_contact_state': 'New York', 'Series_contact_zip/postal_code': '10065', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE120nnn/GSE120383/suppl/GSE120383_xtail_analysis_TE_shTyr.txt.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP162475'}\n",
      "677 PRJNA511999 GSE124428 {'Series_title': 'A tRNA modification balances carbon and nitrogen metabolism by regulating phosphate homeostasis, to couple metabolism to cell cycle progression.', 'Series_geo_accession': 'GSE124428', 'Series_status': 'Public on Dec 28 2018', 'Series_submission_date': 'Dec 27 2018', 'Series_last_update_date': 'Aug 21 2019', 'Series_pubmed_id': '31259691', 'Series_summary': 'Cells must appropriately sense and integrate multiple metabolic resources to commit to proliferation. Here, we report that cells regulate nitrogen (amino acid) and carbon metabolic homeostasis through tRNA U34-thiolation. Despite amino acid sufficiency, tRNA-thiolation deficient cells appear amino acid starved. In these cells, carbon flux towards nucleotide synthesis decreases, and trehalose synthesis increases, resulting in metabolic a starvation-signature. Thiolation mutants have only minor translation defects. However, these cells exhibit strongly decreased expression of phosphate homeostasis genes, mimicking a phosphate-limited state. Reduced phosphate enforces a metabolic switch, where glucose-6-phosphate is routed towards storage carbohydrates. Notably, trehalose synthesis, which releases phosphate and thereby restores phosphate availability, is central to this metabolic rewiring. Thus, cells use thiolated tRNAs to perceive amino acid sufficiency, and balance amino acid and carbon metabolic flux to maintain metabolic homeostasis, by controlling phosphate availability. These results further biochemical explain how phosphate availability determines a switch to a ‘starvation-state’.', 'Series_overall_design': 'Ribosome-footprinting and RNA-seq samples of  yeast CEN.PK, ∆uba4 and ∆ncs2 strains.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'S,,Laxman', 'Series_sample_id': 'GSM3533054', 'Series_contact_name': 'Premal,,Shah', 'Series_contact_email': 'premal.shah@rutgers.edu', 'Series_contact_laboratory': 'Shah Lab', 'Series_contact_department': 'Genetics', 'Series_contact_institute': 'Rutgers University', 'Series_contact_address': '145 Bevier Road, Rm 326', 'Series_contact_city': 'Piscataway', 'Series_contact_state': 'NJ', 'Series_contact_zip/postal_code': '08854', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE124nnn/GSE124428/suppl/GSE124428_barcode.tsv.gz', 'Series_platform_id': 'GPL19756', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP174607'}\n",
      "678 PRJNA532893 GSE129816 {'Series_title': 'Next Generation Sequencing Facilitates Quantitative Analysis of Wild Type, genome-edited MTOR(G4448A)-p.C1483Y, and Torin1-treated Wild type Translatomics', 'Series_geo_accession': 'GSE129816', 'Series_status': 'Public on Sep 11 2019', 'Series_submission_date': 'Apr 15 2019', 'Series_last_update_date': 'Sep 11 2019', 'Series_pubmed_id': '31483294', 'Series_summary': 'We performed rigorous ribosome profiling in genome-edited cell lines with human MTOR point mutations and Torin1 treated cell lines. By obtaining about 10 million bases of sequence cells, we generated genome-wide translatome and transcriptome data of genome-edited MTOR(G4448A)-p.C1483Y and Torin1 treated cell lines', 'Series_overall_design': 'Examination of transcriptome and translatome in 3 cell types.', 'Series_type': 'Other', 'Series_contributor': 'Jeong,H,Lee', 'Series_sample_id': 'GSM3722567', 'Series_contact_name': 'JangKeun,,Kim', 'Series_contact_email': 'jang1563@kaist.ac.kr', 'Series_contact_laboratory': 'Translational neurogenetics lab', 'Series_contact_department': 'Biomedical Science and Engineering Interdisciplinary Program', 'Series_contact_institute': 'KAIST', 'Series_contact_address': 'South Korea, Daejeon, Yuseonggu, Guseongdong KAIST E7 7108, Biomedical Research Center', 'Series_contact_city': 'Daejeon', 'Series_contact_state': 'YuseongGu', 'Series_contact_zip/postal_code': '34141', 'Series_contact_country': 'South Korea', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE129nnn/GSE129816/suppl/GSE129816_RAW.tar', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP192588'}\n",
      "679 PRJNA532897 GSE129818 {'Series_title': 'Next Generation Sequencing Facilitates Quantitative Analysis of Wild Type and FMCD mice Translatomics', 'Series_geo_accession': 'GSE129818', 'Series_status': 'Public on Sep 11 2019', 'Series_submission_date': 'Apr 15 2019', 'Series_last_update_date': 'Sep 11 2019', 'Series_pubmed_id': '31483294', 'Series_summary': 'We performed rigorous ribosome profiling in FMCD mouse models with human MTOR point mutations. By obtaining about 10 million bases of sequence from FACS sorted mutation carrying cells in FMCD, we generated genome-wide translatome and transcriptome data of cortical neuron of FMCD.', 'Series_overall_design': 'Examination of transcriptome and translatome in 3 mouse types.', 'Series_type': 'Other', 'Series_contributor': 'Jeong,H,Lee', 'Series_sample_id': 'GSM3722591', 'Series_contact_name': 'JangKeun,,Kim', 'Series_contact_email': 'jang1563@kaist.ac.kr', 'Series_contact_laboratory': 'Translational neurogenetics lab', 'Series_contact_department': 'Biomedical Science and Engineering Interdisciplinary Program', 'Series_contact_institute': 'KAIST', 'Series_contact_address': 'South Korea, Daejeon, Yuseonggu, Guseongdong KAIST E7 7108, Biomedical Research Center', 'Series_contact_city': 'Daejeon', 'Series_contact_state': 'YuseongGu', 'Series_contact_zip/postal_code': '34141', 'Series_contact_country': 'South Korea', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE129nnn/GSE129818/suppl/GSE129818_RAW.tar', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP192590'}\n",
      "680 PRJNA541924 GSE130898 {'Series_title': 'The RNA hairpin binder TRIM71 modulates alternative splicing by repressing Mbnl1 [RNA-seq & Ribo-seq]', 'Series_geo_accession': 'GSE130898', 'Series_status': 'Public on Aug 01 2019', 'Series_submission_date': 'May 08 2019', 'Series_last_update_date': 'Aug 05 2019', 'Series_pubmed_id': '31371437', 'Series_summary': 'We demonstrate that global effects of TRIM71 on the cellular transcriptome of mouse embryonic stem cells (mESCs) are largely attributable to its RNA binding activity. We observe that target repression by TRIM71 in mESCs occurs predominantly through transcript degradation. We identify a core set of targets consistently regulated by TRIM71 in different human and mouse cell lines.', 'Series_overall_design': 'RNA-seq of WT (10 replicates), TRIM71 KO (10 replicates), TRIM71 RING_mut (TRIM71RING; 5 replicates), and TRIM71 NHL_mut(TRIM71NHL, 5 replicates) mESC cells. RNA-seq of WT (8 replicates) and TRIM71 KO (8 replicates) Ne4C cells. RNA-seq of WT (3 replicates) and TRIM71 KO (3 replicates) Huh-7 cells. Ribosome profiling of WT (4 replicates), and TRIM71 KO (4 replicates) mES cells.', 'Series_type': 'Other', 'Series_contributor': 'Helge,,Großhans', 'Series_sample_id': 'GSM3755947', 'Series_contact_name': 'Panagiotis,,Papasaikas', 'Series_contact_email': 'panp80@gmail.com', 'Series_contact_phone': '+41791074582', 'Series_contact_institute': 'Friedrich Miescher Institute', 'Series_contact_address': '66 Maulbeerstrasse', 'Series_contact_city': 'Basel', 'Series_contact_zip/postal_code': 'CH-4058', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE130nnn/GSE130898/suppl/GSE130898_trim71_total_mRNA_counts.tab.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP197096'}\n",
      "681 PRJNA548784 GSE132703 {'Series_title': 'mTORC1 controls a dynamic program of TOP mRNA translation via LARP1', 'Series_geo_accession': 'GSE132703', 'Series_status': 'Public on Feb 12 2020', 'Series_submission_date': 'Jun 13 2019', 'Series_last_update_date': 'May 13 2020', 'Series_pubmed_id': '32094190', 'Series_summary': 'LARP1 has been proposed to control the translation of TOP mRNAs downstrteam of mTORC1. Here we used ribosome profiling to analyze transcriptome-wide changes in translation following mTOR inhibition in wild-type HEK-293T cells and cells where LARP1 (sgLARP1) or LARP1 and its homologue LARP1B (sgLARP1/1B) have been deleted using CRISPR/Cas9.', 'Series_overall_design': \"Please note that the 'larp1dko_*_RPF_rep*' processed data (e.g. DKO_DMSOa_RPF_unique.htseqcounts.txt) was generated together with the corresponding *reseq sample (e.g. GSM3889093 and GSM4146393) and is linked to the LP* sample records (e.g. GSM3889093 LP: larp1dko_dmso_RPF_rep1).\", 'Series_type': 'Other', 'Series_contributor': 'Carson,C,Thoreen', 'Series_sample_id': 'GSM4146396', 'Series_contact_name': 'Carson,C,Thoreen', 'Series_contact_email': 'carson.thoreen@yale.edu', 'Series_contact_phone': '2037377110', 'Series_contact_department': 'Cellular and Molecular Physiology', 'Series_contact_institute': 'Yale School of Medicine', 'Series_contact_address': '333 Cedar St. SHM B 145', 'Series_contact_city': 'New Haven', 'Series_contact_state': 'CT', 'Series_contact_zip/postal_code': '06437', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE132nnn/GSE132703/suppl/GSE132703_RAW.tar', 'Series_platform_id': 'GPL16791', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP201361'}\n",
      "682 PRJNA550038 GSE133125 {'Series_title': 'Ribosome profiling of Cryptococcus neoformans H99', 'Series_geo_accession': 'GSE133125', 'Series_status': 'Public on Jun 22 2019', 'Series_submission_date': 'Jun 21 2019', 'Series_last_update_date': 'Mar 16 2020', 'Series_pubmed_id': '32020195', 'Series_summary': 'We measured protein translation (by ribosome profiling) and RNA levels (by polyA-enriched RNA-seq) in Cryptococcus neoformans strain H99 and Cryptococcus neoformans strain JEC21. This is the first transcriptome-wide map of translation in this species complex.', 'Series_overall_design': 'Matched ribosome profiling and RNA-seq from 6 biological samples: 2x wild-type H99 and 2 deletion mutants (Ago1, Gwo1),  and 1x wild-type JEC21 and 1x deletion mutant (Ago1).', 'Series_type': 'Other', 'Series_contributor': 'Edward,,Wallace', 'Series_sample_id': 'GSM3900533', 'Series_contact_name': 'Edward,William Joseph,Wallace', 'Series_contact_email': 'edward.wallace@ed.ac.uk', 'Series_contact_department': 'Institute for Cell Biology, School of Biological Sciences', 'Series_contact_institute': 'University of Edinburgh', 'Series_contact_address': 'Waddington Building, Max Born Crescent', 'Series_contact_city': 'Edinburgh', 'Series_contact_zip/postal_code': 'EH9 3BF', 'Series_contact_country': 'United Kingdom', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE133nnn/GSE133125/suppl/GSE133125_ribosum_JEC21.txt.gz', 'Series_platform_id': 'GPL26812', 'Series_platform_taxid': '40410', 'Series_sample_taxid': '40410', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP202195'}\n",
      "683 PRJNA553160 GSE133934 {'Series_title': 'Ribosome profiling of GC7/vehicle-treated mouse primary B cells', 'Series_geo_accession': 'GSE133934', 'Series_status': 'Public on Jul 09 2019', 'Series_submission_date': 'Jul 08 2019', 'Series_last_update_date': 'Dec 05 2019', 'Series_pubmed_id': '31474573', 'Series_summary': 'Mouse primary B cells were treated with 10 uM GC7 or vehicle control and processed for ribosome profiling. During harvesting, samples were spiked with S. cerevisiae lysate containing stable polysomes.', 'Series_overall_design': 'Examination of ribosome-protected fragment abundances and distributions in drug-treated and control cells. N = 1.', 'Series_type': 'Other', 'Series_contributor': 'Jane,,Mellor', 'Series_sample_id': 'GSM3930365', 'Series_contact_name': 'Jack,Edward,Feltham', 'Series_contact_email': 'jack.feltham@bioch.ox.ac.uk', 'Series_contact_laboratory': 'Mellor', 'Series_contact_department': 'Biochemistry', 'Series_contact_institute': 'University of Oxford', 'Series_contact_address': 'South Parks Road', 'Series_contact_city': 'Oxford', 'Series_contact_state': 'Oxfordshire', 'Series_contact_zip/postal_code': 'OX1 3QU', 'Series_contact_country': 'United Kingdom', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE133nnn/GSE133934/suppl/GSE133934_RAW.tar', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP213616'}\n",
      "684 PRJNA556006 GSE134660 {'Series_title': 'The Dynamics of Cytoplasmic mRNA Metabolism', 'Series_geo_accession': 'GSE134660', 'Series_status': 'Public on Jan 01 2020', 'Series_submission_date': 'Jul 22 2019', 'Series_last_update_date': 'Jan 29 2020', 'Series_pubmed_id': '31902669', 'Series_summary': 'For all but a few mRNAs, the dynamics of metabolism are unknown. Here, we developed an experimental and analytical framework for examining these dynamics for mRNAs from thousands of genes. mRNAs of mouse fibroblasts exit the nucleus with diverse intragenic and intergenic poly(A)-tail lengths. Once in the cytoplasm, they have a broad (1000-fold) range of deadenylation rates, which correspond to cytoplasmic lifetimes. Indeed, degradation appears to occur primarily through deadenylation-linked mechanisms, with little contribution from endonucleolytic cleavage or deadenylation-independent decapping. Most mRNA molecules degrade only after their tail lengths fall below 25 nt. Decay rates of short-tailed mRNAs vary broadly (1000-fold) and are more rapid for short-tailed mRNAs that had previously undergone more rapid deadenylation. This coupling helps clear rapidly deadenylated mRNAs, enabling the large range in deadenylation rates to impart a similarly large range in stabilities.', 'Series_overall_design': 'The experiments consist of ribosome profiling libraries (small RNA-seq and RPF reads) from two microRNA-inducible 3T3 cell lines (miR-1 and miR-155) with or without microRNA induction. These samples were prepared for 5 and 6 metabolic-labeling time points for the miR-1 and miR-155 lines, respectively. In addition, a metabolic labeling/transcriptional shutoff experiment was performed which generated 12 sequences samples made with the NEXTflex RNA-seq kit. Finally, 38 PAL-seq samples from these time courses are also included to measure poly(A)-tail lengths.', 'Series_type': 'Other', 'Series_contributor': 'David,P,Bartel', 'Series_sample_id': 'GSM3962934', 'Series_contact_name': 'Timothy,J,Eisen', 'Series_contact_email': 'timeisen@berkeley.edu', 'Series_contact_laboratory': 'John Kuriyan', 'Series_contact_department': 'QB3', 'Series_contact_institute': 'UC Berkeley', 'Series_contact_address': 'Stanley Hall', 'Series_contact_city': 'Berkeley', 'Series_contact_state': 'CALIFORNIA', 'Series_contact_zip/postal_code': '94720', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE134nnn/GSE134660/suppl/GSE134660_README.txt', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP215938'}\n",
      "685 PRJNA556704 GSE134517 {'Series_title': 'A global overview of translation efficiency and ribosome occupancy at nucleotide resolution in HEK293 cells', 'Series_geo_accession': 'GSE134517', 'Series_status': 'Public on Oct 09 2019', 'Series_submission_date': 'Jul 19 2019', 'Series_last_update_date': 'Sep 17 2020', 'Series_pubmed_id': '31791371', 'Series_summary': 'Looking at ribosome distribution and translational efficiency in control conditions in HEK293 cell line. Combining this information with our other datasets from HEK293 cells has allowed us to look at ribosome occupancy of specific groups of mRNAs', 'Series_overall_design': 'Ribosome Profiling conducted in HEK293 cells treated with control siRNA. Ribo-zero treated total RNA for the same samples is also included.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Sarah,L,Gillen', 'Series_sample_id': 'GSM3954925', 'Series_contact_name': 'Sarah,Louise,Gillen', 'Series_contact_institute': 'Beatson Institute for Cancer Research', 'Series_contact_address': 'Beatson Institute, Garscube Estate, Switchback Road, Bearsden', 'Series_contact_city': 'Glasgow', 'Series_contact_zip/postal_code': 'G611BD', 'Series_contact_country': 'United Kingdom', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE134nnn/GSE134517/suppl/GSE134517_normalised_ribosome_occupancy_perTranscript.tsv.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP216422'}\n",
      "686 PRJNA562279 GSE136350 {'Series_title': 'Ribosome Footprinting (RiboSeq) analysis of mRNA translation in Dis3l2 knockout mESCs', 'Series_geo_accession': 'GSE136350', 'Series_status': 'Public on Apr 04 2020', 'Series_submission_date': 'Aug 26 2019', 'Series_last_update_date': 'Jun 08 2020', 'Series_pubmed_id': '32457326', 'Series_summary': 'Input mRNAs and ribosome protected fragments (RPFs) from Heterozygout (Control) and Dis3l2 knockout (KO) mESCs cultures were analyzed by TruSeq Ribo Profile (Illumina) library preparation and high throughput sequencing', 'Series_overall_design': 'mESCs were cultured in LIF/Serum condition. Input mRNAs and RPFs were isolated, ribosomal RNAs were depleted and cDNA libraries were prepared according to the TruSeq Ribo Profile pipeline (Illumina)', 'Series_type': 'Other', 'Series_contributor': 'Richard,I,Gregory', 'Series_sample_id': 'GSM4047150', 'Series_contact_name': 'Qi,,Liu', 'Series_contact_email': 'qi.liu2@childrens.harvard.edu', 'Series_contact_institute': \"Boston Children's Hospital\", 'Series_contact_address': 'Boston, 02115.USA', 'Series_contact_city': 'Boston', 'Series_contact_state': 'MASSACHUSETTS', 'Series_contact_zip/postal_code': '02115', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE136nnn/GSE136350/suppl/GSE136350_Supplemental_Table_2.xlsx', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP219880'}\n",
      "687 PRJNA566449 GSE137757 {'Series_title': \"Induction of an alternative 5' leader enhances translation of Inpp5e and resistance to oncolytic virus infection\", 'Series_geo_accession': 'GSE137757', 'Series_status': 'Public on Nov 30 2019', 'Series_submission_date': 'Sep 19 2019', 'Series_last_update_date': 'Apr 04 2023', 'Series_pubmed_id': '36958764', 'Series_summary': 'Residual cell-intrinsic innate immunity in cancer cells hampers infection with oncolytic viruses. Control of mRNA translation is an important feature of innate immunity, yet the identity of mRNA substrates that participate in host defences remain ill-defined. We characterized the translatome of resistant murine “4T1” breast cancer cells infected with three of the most clinically advanced oncolytic viruses: Herpes Simplex virus 1, Reovirus and Vaccinia virus. Common among all three infections were translationally de-repressed mRNAs including Inpp5e, encoding an inositol 5-phosphatase that modifies lipid second messenger signalling. We found that viral infection induced expression of an Inpp5e mRNA variant that lacks repressive upstream open reading frames (uORFs) within its 5’ leader and is efficiently translated. Furthermore, we show that INPP5E contributes to antiviral immunity by altering virus attachment. These findings uncover a role for translational control through alternative 5’ leader expression and assign an antiviral function to the ciliopathy gene INPP5E.', 'Series_overall_design': 'Ribosome Profiling (RPF-Seq and RNA -Seq) of murine 4T1 breast cancer cells infected with different oncolytic viruses', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Seyed,M,Jafarnejad', 'Series_sample_id': 'GSM4086611', 'Series_contact_name': 'Tommy,,Alain', 'Series_contact_email': 'tommy@arc.cheo.ca', 'Series_contact_institute': \"Children's Hospital of Eastern Ontario Research Institute\", 'Series_contact_address': '401 Smyth Road', 'Series_contact_city': 'Ottawa', 'Series_contact_state': 'Ontario', 'Series_contact_zip/postal_code': 'K1H 8L1', 'Series_contact_country': 'Canada', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE137nnn/GSE137757/suppl/GSE137757_RPKM_normalized_total_RNA_RPF_TE.xlsx', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP222582'}\n",
      "688 PRJNA576260 GSE138533 {'Series_title': 'Analysis of PDCD4 knockdown effects by high throughput sequencing of a human epithelial cell line', 'Series_geo_accession': 'GSE138533', 'Series_status': 'Public on Jan 20 2020', 'Series_submission_date': 'Oct 07 2019', 'Series_last_update_date': 'Mar 02 2020', 'Series_pubmed_id': '32066800', 'Series_summary': 'Purpose: examination of gene expression and translation changes in immortalized human epithelial cells (hTERT-RPE-1) induced by PDCD4 siRNA-knockdown.', 'Series_overall_design': 'Knockdown induction: hTERT-RPE-1 cells were transfected either with PDCD4-siRNA or control-siRNA (100 nM) and incubated for 24 h. Induced effects were analyzed by ribosome profiling and RNA-seq in triplicate (IlluminaSequencing).', 'Series_type': 'Other', 'Series_contributor': 'Sebastian,A,Leidel', 'Series_sample_id': 'GSM4110760', 'Series_contact_name': 'Astrid,,Haas', 'Series_contact_department': 'Biochemistry', 'Series_contact_institute': 'WWU Münster', 'Series_contact_address': 'Wilhelm-Klemm-Str.2', 'Series_contact_city': 'Münster', 'Series_contact_zip/postal_code': '48149', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE138nnn/GSE138533/suppl/GSE138533_siRNA_RPE_mRNA_resLA.txt.gz', 'Series_platform_id': 'GPL15456', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP224660'}\n",
      "689 PRJNA587516 GSE139910 {'Series_title': 'Neurons Release Serine to Support mRNA Translation in Pancreatic Cancer', 'Series_geo_accession': 'GSE139910', 'Series_status': 'Public on Nov 02 2020', 'Series_submission_date': 'Nov 04 2019', 'Series_last_update_date': 'Feb 01 2021', 'Series_pubmed_id': '33142117', 'Series_summary': 'In this dataset, we included sequencing data of total and ribosome protected fragments obtained from PATU-8902 cell lines grown in DMEM (2.78mM glucose, 4mM glutamine, 1mM pyruvate) + 10%dialyzed FBS + 1% Pen/Strep with or without 400uM Ser and 400uM Gly for 24 hours.', 'Series_overall_design': 'Ribosome profiling was performed in PATU-8902 cells grown with or without serine and glycine for 24hours.', 'Series_type': 'Other', 'Series_contributor': 'Alec,C,Kimmelman', 'Series_sample_id': 'GSM4802245', 'Series_contact_name': 'Robert,S,Banh', 'Series_contact_laboratory': 'Kimmelman Lab', 'Series_contact_department': 'Radiation Oncology', 'Series_contact_institute': 'New York University', 'Series_contact_address': '522 First Avenue, Smilow 907', 'Series_contact_city': 'New York', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '10016', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE139nnn/GSE139910/suppl/GSE139910_charging_output.tsv.gz', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP228560'}\n",
      "690 PRJNA607566 GSE145571 {'Series_title': 'Estimation of peptide elongation times from ribosome profiling spectra', 'Series_geo_accession': 'GSE145571', 'Series_status': 'Public on Apr 23 2021', 'Series_submission_date': 'Feb 19 2020', 'Series_last_update_date': 'Apr 25 2021', 'Series_pubmed_id': '33885812', 'Series_web_link': 'https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkab260/6246399?guestAccessKey=d3b4bb4e-f84e-4c09-8734-d9bae3dc91c1', 'Series_summary': 'We develop a model for ribosome-protected fragment (RPF) spectra that accounts for the local codon context-dependent variation of peptide elongation times and fragment processing biases. We use a Marquardt algorithm for non-linear regression with maximum likelihood (ML) like statistics to fit the RPF sequencing data. Taking advantage of the factorial character of the present model, we reconstruct the parameters adhering to an ideal, unbiased RPF spectrum.', 'Series_overall_design': 'Ribosome profiling of E. coli MG1655 cultured in rich medium', 'Series_type': 'Other', 'Series_contributor': 'Mans,,Eherenberg', 'Series_sample_id': 'GSM4321627', 'Series_contact_name': 'Zoya,,Ignatova', 'Series_contact_email': 'zoya.ignatova@chemie.uni-hamburg.de', 'Series_contact_institute': 'Universitaet Hamburg', 'Series_contact_address': 'Martin -Luther-King-Platz 6', 'Series_contact_city': 'Hamburg', 'Series_contact_zip/postal_code': '20146', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE145nnn/GSE145571/suppl/GSE145571_RAW.tar', 'Series_platform_id': 'GPL14548', 'Series_platform_taxid': '562', 'Series_sample_taxid': '562', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP250036'}\n",
      "691 PRJNA608263 GSE145795 {'Series_title': 'Alterations in the transcriptome and translatome profiles of Arabidopsis pollen germinated under heat stress', 'Series_geo_accession': 'GSE145795', 'Series_status': 'Public on Dec 15 2020', 'Series_submission_date': 'Feb 24 2020', 'Series_last_update_date': 'Mar 10 2021', 'Series_pubmed_id': '33459869', 'Series_summary': 'Plant reproduction is one key biological process very sensitive to heat stress and, as a consequence, enhanced global warming imposes serious threats to sustain food safety worldwide. In this work we have focused on the molecular impact that high temperature conditions impose on gene expression of Arabidopsis pollen germinated in vitro. We have used a high-resolution ribosome profiling technology to provide, for the first time, a comprehensive study of how both the transcriptome and the translatome of germinated pollen respond to the increase in temperature. Although heat shock responses operate properly under high temperature conditions, we have uncovered important alterations under elevated temperature regimes down-regulating essential processes linked to cation/proton exchange and to carbohydrate/cation symport transport. These alterations provide molecular explanations to the dramatic alterations of pollen tube growth under heat stress. Overall a high correlation between transcriptional and translational responses to high temperature was found, but specific regulations at the translational level are also present in pollen subjected to temperature challenging conditions.', 'Series_overall_design': 'Riboprofiling analysis was performed for Arabidopsis thaliana in vitro germinated pollen at permissive and restrictive temperatures (24°C 35°C respectively).', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Alejandro,,Ferrando', 'Series_sample_id': 'GSM4333064', 'Series_contact_name': 'Javier,,Forment', 'Series_contact_email': 'jforment@ibmcp.upv.es', 'Series_contact_laboratory': 'Servicio de Bioinformática', 'Series_contact_department': 'Biotecnología', 'Series_contact_institute': 'Instituto de Biología Molecular y Celular de Plantas, IBMCP (UPV-CSIC)', 'Series_contact_address': 'Ingeniero Fausto Elio, s/n', 'Series_contact_city': 'Valencia', 'Series_contact_state': 'Valencia', 'Series_contact_zip/postal_code': '46022', 'Series_contact_country': 'Spain', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE145nnn/GSE145795/suppl/GSE145795_counts_and_TPMs.tsv.gz', 'Series_platform_id': 'GPL21785', 'Series_platform_taxid': '3702', 'Series_sample_taxid': '3702', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP250465'}\n",
      "692 PRJNA609751 GSE146240 {'Series_title': 'Polysomes bypass a 50 nucleotide coding gap less efficiently than monosomes due to attenuation of an unstable 5’ mRNA stem loop stimulator and enhanced drop-off', 'Series_geo_accession': 'GSE146240', 'Series_status': 'Public on Jun 01 2020', 'Series_submission_date': 'Mar 02 2020', 'Series_last_update_date': 'Jun 01 2020', 'Series_pubmed_id': '32454154', 'Series_summary': 'Ribosome profiling analysis of phage T4 infected E. coli yielded protected mRNA fragments within the normal size range derived from ribosomes stalled at the take-off codon. Ribosomes at this position also yielded some 53 nucleotide fragments, 16 longer. These were due to protection of the nucleotides that form the 5’ stem loop.', 'Series_overall_design': \"Ribosome profiling of 'normal' and long protected fragments obtained from T4 infected E.coli or of E.coli with plasmid-borne cassette with the gene 60 sequence\", 'Series_type': 'Other', 'Series_contributor': 'John,F,Atkins', 'Series_sample_id': 'GSM4368685', 'Series_contact_name': 'John,,Atkins', 'Series_contact_email': 'atkins@genetics.utah.edu', 'Series_contact_department': 'School of Biochemistry and Cell Biology', 'Series_contact_institute': 'UCC', 'Series_contact_address': 'Western GateWay Building, Western Road, UCC, Cork', 'Series_contact_city': 'Cork', 'Series_contact_zip/postal_code': 'NA', 'Series_contact_country': 'Ireland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE146nnn/GSE146240/suppl/GSE146240_RAW.tar', 'Series_platform_id': 'GPL28207', 'Series_platform_taxid': '10665', 'Series_sample_taxid': '10665', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP251292'}\n",
      "693 PRJNA630709 GSE149970 {'Series_title': 'Ribosome Footprinting (RiboSeq) analysis of mRNA translation in MEF Wt cells overexpressing  METTL1+WDR4', 'Series_geo_accession': 'GSE149970', 'Series_status': 'Public on Oct 01 2020', 'Series_submission_date': 'May 06 2020', 'Series_last_update_date': 'Oct 01 2020', 'Series_summary': 'The RNA methyltransferase METTL1 catalyzes the N7-methylguanosine (m7G) modification of certain tRNAs, mRNAs, and miRNA precursors. However, the role of METTL1 and its cofactor WDR4 in cancer remains largely unexplored. Here we reveal the oncogenic role of METTL1/WDR4. METTL1 is frequently amplified and overexpressed in cancers and correlates with poor patient survival. METTL1 depletion in human cancer cells causes decreased abundance of m7G-modified tRNAs, altered cell cycle, and inhibits oncogenicity. Strikingly, METTL1/WDR4 overexpression induces oncogenic transformation and carcinogenesis. Mechanistically, we find increased abundance of a subset of m7G-modified tRNAs including tRNA-Arg(TCT), and increased translation of mRNAs enriched in the corresponding AGA codon including cell cycle regulators. Accordingly, expression of tRNA-Arg(TCT) is significantly elevated in many cancer types, correlates with patient survival, and overexpression of this tRNA enhances reporter gene expression and cell transformation. Thus, METTL1/WDR4-mediated m7G tRNA modification drives oncogenic transformation, thereby highlighting METTL1 as a promising cancer therapeutic target.', 'Series_overall_design': 'Examination of translatome in MEF Wt cells overexpressing METTL1+WDR4 compared to empty vector control, in duplicate, using Ribo-Seq  techique.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Richard,I,Gregory', 'Series_sample_id': 'GSM4519044', 'Series_contact_name': 'Qi,,Liu', 'Series_contact_email': 'qi.liu2@childrens.harvard.edu', 'Series_contact_institute': \"Boston Children's Hospital\", 'Series_contact_address': 'Boston, 02115.USA', 'Series_contact_city': 'Boston', 'Series_contact_state': 'MASSACHUSETTS', 'Series_contact_zip/postal_code': '02115', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE149nnn/GSE149970/suppl/GSE149970_MEF_Wt_METTL1_WDR4_TE_list.xlsx', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP260267'}\n",
      "694 PRJNA632889 GSE150601 {'Series_title': 'smORFer: a modular algorithm to detect small ORFs in prokaryotes', 'Series_geo_accession': 'GSE150601', 'Series_status': 'Public on May 12 2021', 'Series_submission_date': 'May 14 2020', 'Series_last_update_date': 'Jun 18 2021', 'Series_pubmed_id': '34125903', 'Series_summary': 'We developed a novel algorithm, smORFer, detecting smORFs (e.g. <50 codons) which performs with higher accuracy in prokaryotic organisms. smORFer considers structural features of genetic sequence along with in-register translation and using Fourier transform converts them into a measurable score to faithfully select smORFs. The algorithm is executed in a modular way and dependent on the data available different modules can be tested.', 'Series_overall_design': 'Ribosome profiling of Staphylococcus aureus cultured in complex TBS medium', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Zoya,,Ignatova', 'Series_sample_id': 'GSM4553032', 'Series_contact_name': 'Zoya,,Ignatova', 'Series_contact_email': 'zoya.ignatova@chemie.uni-hamburg.de', 'Series_contact_institute': 'Universitaet Hamburg', 'Series_contact_address': 'Martin -Luther-King-Platz 6', 'Series_contact_city': 'Hamburg', 'Series_contact_zip/postal_code': '20146', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE150nnn/GSE150601/suppl/GSE150601_RAW.tar', 'Series_platform_id': 'GPL19006', 'Series_platform_taxid': '1280', 'Series_sample_taxid': '1280', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP261663'}\n",
      "695 PRJNA633140 GSE150676 {'Series_title': 'eIF4A2 targets developmental potency and histone H3.3 transcripts for translational control of embryonic and cancer stemness [Ribo-seq]', 'Series_geo_accession': 'GSE150676', 'Series_status': 'Public on Apr 20 2022', 'Series_submission_date': 'May 15 2020', 'Series_last_update_date': 'Apr 20 2022', 'Series_pubmed_id': '35353581', 'Series_summary': 'Stemness is a defining feature in embryonic and cancer stem cells. How stemness is regulated at the mRNA translational initiation remains undefined. We carried out an RNAi screen for key translation initiation factors that maintain the stemness in mouse embryonic stem cells (ESCs). We identified eIF4A2 and defined its mechanistic action through Rps26-depleted and -containing ribosomes in translational initiation activation of mRNAs encoding pluripotency factors and H3.3 for embryonic and extraembryonic lineage repression, respectively. eIF4A2 also mediates translation initiation activation of Ddx6, which acts together with eIF4A2 to restrict the totipotent 2-cell (2C) transcription program in ESCs through Zscan4 mRNA degradation and translation repression. Knockdown of eIF4A2 disrupts ESC proteome causing the loss of stemness in ESCs as well as in human glioblastomas where eIF4A2 is highly enriched. Collectively, our study establishes an eIF4A2-mediated translation initiation control of stemness and provides insight into cancer therapeutics targeting the translation initiation machinery.', 'Series_overall_design': 'eIF4A2 ribosome profiling was investigated via Ribo-seq analysis in mouse embryonic stem cells.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Jianlong,,Wang', 'Series_sample_id': 'GSM4556260', 'Series_contact_name': 'Jianlong,,Wang', 'Series_contact_email': 'jw3925@cumc.columbia.edu', 'Series_contact_laboratory': 'Jianlong Wang', 'Series_contact_department': 'Department of Medicine', 'Series_contact_institute': 'Columbia University Irving Medical Center', 'Series_contact_address': '650 W. 168th St.', 'Series_contact_city': 'New York', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '10032', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE150nnn/GSE150676/suppl/GSE150676_RiboSeq_eif4a2_totalrna.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP261884'}\n",
      "696 PRJNA670703 GSE159869 {'Series_title': 'The biochemical basis of mitochondrial dysfunction in Zellweger Spectrum Disorder', 'Series_geo_accession': 'GSE159869', 'Series_status': 'Public on Jun 13 2021', 'Series_submission_date': 'Oct 22 2020', 'Series_last_update_date': 'Sep 13 2021', 'Series_pubmed_id': '34351705', 'Series_summary': 'Peroxisomal Biogenesis Disorders (PBDs) are genetic disorders of peroxisome biogenesis and metabolism that are characterized by profound developmental and neurological phenotypes. The most severe class of PBDs—Zellweger Spectrum Disorder (ZSD)—is caused by mutations in peroxin genes that result in both non-functional peroxisomes and mitochondrial dysfunction. It is unclear, however, how defective peroxisomes contribute to mitochondrial impairment. In order to understand the molecular basis of this inter-organellar relationship, we investigated the fate of peroxisomal mRNAs and proteins in ZSD model systems. We found that peroxins were still expressed and a subset of them accumulated on the mitochondrial membrane, which resulted in gross mitochondrial abnormalities and impaired mitochondrial metabolic function. We showed that overexpression of ATAD1, a mitochondrial quality control factor, was sufficient to rescue several aspects of mitochondrial function in human ZSD fibroblasts. Together, these data suggest that aberrant peroxisomal protein localization is necessary and sufficient for the devastating mitochondrial morphological and metabolic phenotypes in ZSDs.', 'Series_overall_design': '4 total S. cerevisiae samples with 1 RNA-seq and 1 Ribosome-footprint profiling entry for each sample', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Jared,,Rutter', 'Series_sample_id': 'GSM4849021', 'Series_contact_name': 'Jeff,,Morgan', 'Series_contact_email': 'jeff.morgan@biochem.utah.edu', 'Series_contact_laboratory': 'Jared Rutter', 'Series_contact_department': 'Biochemistry', 'Series_contact_institute': 'University of Utah', 'Series_contact_address': '15 N Medical Drive East', 'Series_contact_city': 'Salt Lake City', 'Series_contact_state': 'UT', 'Series_contact_zip/postal_code': '84112', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE159nnn/GSE159869/suppl/GSE159869_RAW.tar', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP288177'}\n",
      "697 PRJNA672952 GSE160326 {'Series_title': 'RNMT regulates the activated CD4 T cell transcriptome and translatome [Ribo-Seq]', 'Series_geo_accession': 'GSE160326', 'Series_status': 'Public on May 13 2021', 'Series_submission_date': 'Oct 28 2020', 'Series_last_update_date': 'Jun 21 2021', 'Series_pubmed_id': '34125914', 'Series_summary': 'We investigated the role of RNMT in T cells using an Rnmt conditional knockout mouse model. We report that the mRNA cap methyltransferase, RNMT, supports naïve T cell survival and activation-induced proliferation. We demonstrate that RNMT has gene-specific impacts in T cells, selectively regulating expression of terminal polypyrimidine tract (TOP) mRNAs which are targets of the m7G-cap binding protein LARP1. These ribosome footprinting experiments investigate the effect of Rnmt cKO on the activated CD4 T cell transcriptome and translatome.', 'Series_overall_design': 'Sequencing of rRNA-depleted RNA and ribosome protected fragments from 20 hour activated CD4 T cells from Rnmt fl/fl CD4cre (Rnmt cKO) and Rnmt fl/fl (control) mice.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Victoria,H,Cowling', 'Series_sample_id': 'GSM4871500', 'Series_contact_name': 'Alison,,Galloway', 'Series_contact_email': 'a.galloway@beatson.gla.ac.uk', 'Series_contact_laboratory': 'Victoria Cowling', 'Series_contact_institute': 'Beatson CRUK', 'Series_contact_address': 'Garscube Estate, Switchback Road', 'Series_contact_city': 'Glasgow', 'Series_contact_zip/postal_code': 'G61 1BD', 'Series_contact_country': 'United Kingdom', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE160nnn/GSE160326/suppl/GSE160326_Riboseq_analysis.csv.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP289492'}\n",
      "698 PRJNA680357 GSE162043 {'Series_title': 'Direct epitranscriptomic regulation of mammalian translation initiation through N4-acetylcytidine', 'Series_geo_accession': 'GSE162043', 'Series_status': 'Public on Jun 08 2022', 'Series_submission_date': 'Nov 23 2020', 'Series_last_update_date': 'Nov 29 2022', 'Series_pubmed_id': '35679869', 'Series_summary': 'mRNA function is influenced by modifications that modulate canonical nucleobase behavior. We show that a single modification mediates distinct impacts on mRNA translation in a position-dependent manner. While cytidine acetylation (ac4C) within protein-coding sequences stimulates translation, ac4C within 5’UTRs impacts protein synthesis at the level of initiation. 5’UTR acetylation promotes initiation at upstream sequences, competitively inhibiting annotated start codons. Acetylation further directly impedes initiation at optimal AUG contexts: ac4C within AUG-flanking Kozak sequences reduced initiation in base-resolved transcriptome-wide HeLa results, and in vitro utilizing substrates with site-specific ac4C incorporation. Cryo-EM of mammalian 80S initiation complexes revealed ac4C in the -1-position adjacent to an AUG start codon disrupts an interaction between C and hypermodified t6A at nucleotide 37 of the initiator tRNA. These findings demonstrate the impact of RNA modifications on nucleobase function at a molecular level and introduce mRNA acetylation as a factor regulating translation in a location-specific manner.', 'Series_overall_design': 'Detection of acetylated cytidine in RNA with a novel method (RedaC:T-seq), via chemical reduction of ac4C with sodium borohydride.  Ribo-Seq after Harringtonine treatment to analyze effects on translation initation. RNA sequencing of polysome fractions.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Shalini,,Oberdoerffer', 'Series_sample_id': 'GSM6203039', 'Series_contact_name': 'David,,Sturgill', 'Series_contact_email': 'davidsturgill@mail.nih.gov', 'Series_contact_phone': '240-760-6725', 'Series_contact_laboratory': 'LRBGE', 'Series_contact_department': 'NCI', 'Series_contact_institute': 'NIH', 'Series_contact_address': 'Building 41, Rm B622', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'MARYLAND', 'Series_contact_zip/postal_code': '20892', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE162nnn/GSE162043/suppl/GSE162043_RedaCTseq_ac4C_sites.txt.gz', 'Series_platform_id': 'GPL28443', 'Series_platform_taxid': '9986', 'Series_sample_taxid': '9986', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP293844'}\n",
      "699 PRJNA680364 GSE162050 {'Series_title': 'Ribosome profiling of H9 PUS7KO', 'Series_geo_accession': 'GSE162050', 'Series_status': 'Public on Dec 09 2021', 'Series_submission_date': 'Nov 23 2020', 'Series_last_update_date': 'Mar 24 2022', 'Series_pubmed_id': '35292784', 'Series_summary': 'Transfer RNA (tRNA)-derived fragments (tRF) are emerging small noncoding (nc) RNAs that, while commonly altered in cancer, have poorly defined roles in tumorigenesis. Here we show that pseudouridylation (Y) of a stem-cell-enriched tRF subtype, mTOG, selectively modulates malignant protein synthesis programs, thereby promoting differentiation of myelodysplastic syndrome (MDS) hematopoietic stem and progenitor cells (HSPC). Building on evidence that mTOG targets the polyadenylate-binding protein cytoplasmic 1 (PABPC1), we employed isotope exchange proteomics to reveal critical interactions between mTOG and functional RNA-recognition motif (RRM) domains in PABPC1. Mechanistically, this hinders the recruitment of the translational co-activator PABPC1-interacting protein 1 (PAIP1)3 and selectively represses translation of transcripts sharing pyrimidine-enriched sequences (PES) in their 5’UTR including 5’ terminal oligopyrimidine tracts (TOP) that encode protein machinery components, and are frequently altered in cancer4. Significantly, mTOG dysregulation leads to aberrant 5’PES mRNA translation in malignant MDS-HSPC and is clinically associated with leukemic transformation and decreased patient survival. Taken together, these results unveil a critical role for tRF and Y in difficult-to-treat subsets of MDS characterized by high risk of progression to acute myeloid leukemia.', 'Series_overall_design': 'Evaluation of hESC WT and PUS7-KO translatome using ribosome profiling', 'Series_type': 'Other', 'Series_contributor': 'Cristian,,Bellodi', 'Series_sample_id': 'GSM4932278', 'Series_contact_name': 'Phuong,,Cao Thi Ngoc', 'Series_contact_institute': 'Lund University', 'Series_contact_address': 'BMC, B1222b, Klinikgatan 26', 'Series_contact_city': 'Lund', 'Series_contact_zip/postal_code': 'SE-221 84', 'Series_contact_country': 'Sweden', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE162nnn/GSE162050/suppl/GSE162050_RAW.tar', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP293929'}\n",
      "700 PRJNA720951 GSE171780 {'Series_title': 'HMGB1 coordinates SASP-related chromatin folding and RNA homeostasis on the path to senescence [Ribo-seq]', 'Series_geo_accession': 'GSE171780', 'Series_status': 'Public on Apr 10 2021', 'Series_submission_date': 'Apr 09 2021', 'Series_last_update_date': 'Jul 12 2021', 'Series_pubmed_id': '34166567', 'Series_summary': 'Spatial organization and gene expression of mammalian chromosomes are maintained and regulated in conjunction with cell cycle progression. This is perturbed once cells enter senescence and the highly abundant HMGB1 protein is depleted from nuclei to act as an extracellular proinflammatory stimulus. Despite its physiological importance, we know little about the positioning of HMGB1 on chromatin and its nuclear roles. To address this, we mapped HMGB1 binding genome-wide in two primary cell lines. We integrated ChIP-seq and Hi-C with graph theory to uncover clustering of HMGB1-marked topological domains that harbour genes required for paracrine senescence. Using sCLIP and functional tests, we show that HMGB1 is also a bona fide RNA-binding protein (RBP) binding hundreds of mRNAs. It presents an interactome rich in RBPs implicated in senescence regulation. The mRNAs of many of these RBPs are directly bound by HMGB1 and regulate the availability of SASP-relevant transcripts. Our findings highlight a broader than hitherto assumed role for HMGB1 in coordinating chromatin folding and RNA homeostasis as part of a regulatory loop controlling cell-autonomous and paracrine senescence.', 'Series_overall_design': 'Single IMR90 isolates (I90-10 and -79, passage 5; Coriell Biorepository) were continuously passaged to replicative exhaustion. Three independent replicas of proliferating or senescent IMR90 were used.', 'Series_type': 'Other', 'Series_contributor': 'James,,Keane', 'Series_sample_id': 'GSM5234115', 'Series_contact_name': 'Argyris,,Papantonis', 'Series_contact_email': 'argyris.papantonis@med.uni-goettingen.de', 'Series_contact_department': 'Department of Pathology', 'Series_contact_institute': 'University Medical Center Goettingen', 'Series_contact_address': 'Robert-Koch-Str 40', 'Series_contact_city': 'Goettingen', 'Series_contact_zip/postal_code': '37075', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE171nnn/GSE171780/suppl/GSE171780_RAW.tar', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP314256'}\n",
      "701 PRJNA762342 GSE183923 {'Series_title': 'Effects of tRNA[Ser]Sec anticodon stem loop modifications on selenoprotein expression', 'Series_geo_accession': 'GSE183923', 'Series_status': 'Public on Nov 17 2021', 'Series_submission_date': 'Sep 10 2021', 'Series_last_update_date': 'Nov 17 2021', 'Series_pubmed_id': '34768885', 'Series_summary': 'Expression of selenoproteins requires the co-translational incorporation of selenocysteine (Sec) in response to an in-frame UGA codon. The machinery of UGA/Sec re-coding is complex and many factors affect the hierarchy of expression among selenoproteins, including modification of tRNA[Ser]Sec. Its hyper-modification in the anticodon stem loop is influenced by selenium bioavailability, and a mutation in adenosine 37 (A37) that abrogates isopentenylation, has a profound effect on selenoprotein expression in mice. Patients with mutations in tRNA-isopentenyl-transferase (TRIT1) show a severe neurological disorder and hence we wondered whether mutations in TRIT1 negatively affected the expression of selenoproteins. Fibroblasts from a patient carrying a pathogenic R323Q mutation in TRIT1 in homozygosity did not show decreased selenoprotein expression, although recombinant TRIT1R323Q had significantly reduced activity in vitro towards anticodon stem-loop substrates. We thus engineered mice conditionally deficient in Trit1 in hepatocytes and neurons. Selenoprotein expression as assessed by western blotting, 75Se metabolic labeling, and ribosomal profiling was not decreased despite the general reduction of N6-isopentenyl-adenosine in tRNAs. We show that 5-methylcarboxymethylation and 2’O-methylation of U34 occur independently of isopentenylation of A37 in tRNA[Ser]Sec. Reanalyzing previously published ribosomal profiling datasets, we demonstrate that (i) failure of 5-carboxymethylation at U34 is associated with reduced expression of GPX1, but not GPX4, and that (ii) FTSJ1 is not the elusive U34-2’O-methyltransferase involved in the methylation of tRNA[Ser]Sec.', 'Series_overall_design': 'Ribosome profiling of liver samples from 3 Trit1-deficient mice and 3 control mice', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Ulrich,,Schweizer', 'Series_sample_id': 'GSM5574432', 'Series_contact_name': 'Ulrich,,Schweizer', 'Series_contact_email': 'uschweiz@uni-bonn.de', 'Series_contact_laboratory': 'Schweizer Lab', 'Series_contact_department': 'IBMB', 'Series_contact_institute': 'Universität Bonn', 'Series_contact_address': 'Nussallee 11', 'Series_contact_city': 'Bonn', 'Series_contact_zip/postal_code': '53115', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE183nnn/GSE183923/suppl/GSE183923_rpf_counts_CDS.csv.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP336565'}\n",
      "702 PRJNA764064 GSE184322 {'Series_title': 'CRSPR/Cas9 gene knockout screen identifies colorectal cancer dependency on cryptic open reading frames', 'Series_geo_accession': 'GSE184322', 'Series_status': 'Public on Aug 06 2023', 'Series_submission_date': 'Sep 17 2021', 'Series_last_update_date': 'Aug 06 2023', 'Series_summary': 'Emerging evidence suggests that the cryptic translation beyond the annotated translatome may yield proteins with important function. However, the role and function mechanism of these cryptic ORFs in complex diseases such as cancer remain largely unknown. To fill this gap, we combined  ribosome profiling and CRISPR/Cas9 screen to systematically identify the colorectal cancer (CRC) dependency on cryptic ORFs.', 'Series_overall_design': 'CRISPR/Cas9 knockout screen and ribosome profiling to systematically discover nocanonical open reading frame encoded in long noncoding RNAs (lncRNAs) and explored their critical roles in colorectal cancer progression. RNA-Seq analysis the differernt expressed genes after SMMIP kockout or SMC1A knockdown. ChIP-Seq analysis SMC1A binding sites on genome.', 'Series_type': 'Other', 'Series_contributor': 'Yanjun,,Wei', 'Series_sample_id': 'GSM6356703', 'Series_contact_name': 'Yiwen,,Chen', 'Series_contact_email': 'YChen26@mdanderson.org', 'Series_contact_phone': '713-745-9283', 'Series_contact_department': 'Bioinformatics & Comp Biology', 'Series_contact_institute': 'UT MD Anderson Cancer Center', 'Series_contact_address': '1400 Pressler Street,  Unit 1410', 'Series_contact_city': 'Houston', 'Series_contact_state': 'TX', 'Series_contact_zip/postal_code': '77030', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE184nnn/GSE184322/suppl/GSE184322_SMC1A1P1.SMC1A1P2.pooled_peaks.narrowPeak.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA764064'}\n",
      "703 PRJNA770962 GSE185826 {'Series_title': 'Velcrin-induced selective cleavage of tRNALeu(TAA) by SLFN12 causes cancer cell death [Ribo-Seq]', 'Series_geo_accession': 'GSE185826', 'Series_status': 'Public on Sep 01 2022', 'Series_submission_date': 'Oct 13 2021', 'Series_last_update_date': 'Oct 29 2022', 'Series_pubmed_id': '36302897', 'Series_web_link': 'https://www.nature.com/articles/s41589-022-01170-9', 'Series_summary': 'Velcrin compounds kill cancer cells expressing high levels of phosphodiesterase 3A (PDE3A) and Schlafen family member 12 (SLFN12) by inducing complex formation between these two proteins, but the mechanism of cancer cell killing by the PDE3A–SLFN12 complex is not fully understood. Here, we report that the physiological substrate of SLFN12 RNase is tRNALeu(TAA). SLFN12 selectively digests tRNALeu(TAA), and velcrin treatment promotes the cleavage of tRNALeu(TAA) by inducing PDE3A–SLFN12 complex formation in vitro. We found that distinct sequences in the variable loop and acceptor stem of tRNALeu(TAA) are required for substrate digestion. Velcrin treatment of sensitive cells results in downregulation of tRNALeu(TAA), ribosome pausing at Leu-TTA codons and global inhibition of protein synthesis. Velcrin-induced cleavage of tRNALeu(TAA) by SLFN12 and the concomitant global inhibition of protein synthesis thus define a new mechanism of apoptosis initiation.', 'Series_overall_design': 'Ribosome profiling of Hela cells treated with DMSO control or DNMDP in duplicate', 'Series_type': 'Other', 'Series_contributor': 'Heidi,,Greulich', 'Series_sample_id': 'GSM5624313', 'Series_contact_name': 'Stephanie,,Hoyt', 'Series_contact_email': 'shoyt@broadinstitute.org', 'Series_contact_laboratory': 'Meyerson Group', 'Series_contact_department': 'Cancer Program', 'Series_contact_institute': 'Broad Institute of MIT and Harvard', 'Series_contact_address': '415 Main St', 'Series_contact_city': 'Cambridge', 'Series_contact_state': 'MA', 'Series_contact_zip/postal_code': '02142', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE185nnn/GSE185826/suppl/GSE185826_RAW.tar', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP341191'}\n",
      "704 PRJNA774895 GSE186639 {'Series_title': 'Translation efficiency changes measured using ribosome footprinting (Ribo-seq) in poorly and highly metastatic cancer cells', 'Series_geo_accession': 'GSE186639', 'Series_status': 'Public on Feb 20 2023', 'Series_submission_date': 'Oct 26 2021', 'Series_last_update_date': 'Feb 20 2023', 'Series_summary': 'Cancer cells often co-opt post-transcriptional regulatory mechanisms to achieve pathologic expression of gene networks that drive metastasis. Translational control is a major regulatory hub in oncogenesis, however its effects on cancer progression remain poorly understood. To address this, we used ribosome profiling to compare genome-wide translation efficiencies of poorly and highly metastatic breast cancer cells and patient-derived xenografts. We developed novel regression-based methods to analyze ribosome profiling and alternative polyadenylation data, and identified HNRNPC as a translational controller of a specific mRNA regulon. Mechanistically, HNRNPC, in concert with PABPC4, binds near to poly(A) signals, thereby governing the alternative polyadenylation of a set of mRNAs. We found that HNRNPC and PABPC4 are downregulated in highly metastatic cells, which causes HNRNPC-bound mRNAs to undergo 3’ UTR lengthening and subsequently, translational repression. We showed that modulating HNRNPC expression impacts the metastatic capacity of breast cancer cells in xenograft mouse models. We also found that a small molecule, previously shown to induce a distal-to-proximal poly(A) site switching, counteracts the HNRNPC-PABPC4 driven deregulation of alternative polyadenylation and decreases the metastatic lung colonization by breast cancer cells in vivo.', 'Series_overall_design': 'Comparison of translation efficiencies in highly and poorly metastatic breast cancer cell lines and PDXs', 'Series_type': 'Other', 'Series_contributor': 'Goodarzi,,Hani', 'Series_sample_id': 'GSM5658212', 'Series_contact_name': 'Hani,,Goodarzi', 'Series_contact_department': 'Biochemistry and Biophysics', 'Series_contact_institute': 'UCSF', 'Series_contact_address': '600 16th St, GH S312D', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186639/suppl/GSE186639_sgHNRNPC_norm_count_rpf_rna.txt.gz', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP343302'}\n",
      "705 PRJNA779948 GSE188692 {'Series_title': 'Ribo-Seq performed in parental or CPEB4 knock-out HeLa cells', 'Series_geo_accession': 'GSE188692', 'Series_status': 'Public on Aug 23 2022', 'Series_submission_date': 'Nov 12 2021', 'Series_last_update_date': 'Sep 27 2022', 'Series_pubmed_id': '36096941', 'Series_summary': \"Immediate early genes (IEGs) represent a unique class of genes with rapid induction kinetics and transient expression patterns, which requires IEG mRNAs to be short-lived. Here, we establish cytoplasmic polyadenylation element-binding protein 4 (CPEB4) as a major determinant of IEG mRNA instability. We identified human CPEB4 as an RNA-binding protein (RBP) with enhanced association to poly(A) RNA upon inhibition of class I histone deacetylases (HDACs), which is known to cause widespread degradation of poly(A)-containing mRNA. Photoactivatable ribonucleoside-enhanced crosslinking and immunoprecipitation (PAR-CLIP) analysis using endogenously tagged CBEP4 in HeLa cells revealed that CPEB4 preferentially binds to the 3' untranslated region (UTR) of IEG mRNAs, at U-rich sequence motifs located in close proximity to the poly(A) site. By transcriptome-wide mRNA decay measurements, we found that the strength of CPEB4 binding correlates with short mRNA half-lives, and that loss of CBEP4 expression leads to the stabilization of IEG mRNAs. Further, we demonstrate that CPEB4 mediates mRNA degradation by recruitment of the evolutionarily conserved CCR4-NOT complex, the major eukaryotic deadenylase. While CPEB4 is primarily known for its ability to stimulate cytoplasmic polyadenylation, our findings establish an additional function for CPEB4 as an RBP that enhances the degradation of short-lived IEG mRNAs.\", 'Series_overall_design': 'Preparation of ribosome-protected fragments (RPFs) followed by RNA-Seq to determine ribosome densities.', 'Series_type': 'Other', 'Series_contributor': 'Georg,,Stoecklin', 'Series_sample_id': 'GSM5688488', 'Series_contact_name': 'Johanna,Daniela,Schott', 'Series_contact_email': 'johanna.schott@medma.uni-heidelberg.de', 'Series_contact_laboratory': 'Stoecklin lab', 'Series_contact_department': 'Mannheim Institute for Innate Immunoscience', 'Series_contact_institute': 'Heidelberg University', 'Series_contact_address': 'Ludolf-Krehl-Str. 13-17', 'Series_contact_city': 'Mannheim', 'Series_contact_zip/postal_code': '68167', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE188nnn/GSE188692/suppl/GSE188692_RiboSeq_CPEB4_ko.csv.gz', 'Series_platform_id': 'GPL21697', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP345769'}\n",
      "706 PRJNA789542 GSE191048 {'Series_title': 'Engineered tRNAs suppress nonsense mutations in cells and in vivo', 'Series_geo_accession': 'GSE191048', 'Series_status': 'Public on Apr 04 2023', 'Series_submission_date': 'Dec 16 2021', 'Series_last_update_date': 'Jun 23 2023', 'Series_pubmed_id': '37258671', 'Series_summary': 'we introduce a strategy to repurpose sense-codon decoding tRNA into efficient suppressors of the three nonsense mutation-induced PTCs (UGA, UAG and UAA). The suppressor tRNAs restore function of a model and disease-related protein', 'Series_overall_design': 'Suppressor tRNAs are administered both in mice (intraveneous in liver or intratracheal in lung) or in immortalized cell culture and tissues/cells are collected and subjected to ribosome profiling (Ribo-Seq)', 'Series_type': 'Other', 'Series_contributor': 'Maher,,Alayyoubi', 'Series_sample_id': 'GSM5737996', 'Series_contact_name': 'Zoya,,Ignatova', 'Series_contact_email': 'zoya.ignatova@chemie.uni-hamburg.de', 'Series_contact_institute': 'Universitaet Hamburg', 'Series_contact_address': 'Martin -Luther-King-Platz 6', 'Series_contact_city': 'Hamburg', 'Series_contact_zip/postal_code': '20146', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE191nnn/GSE191048/suppl/GSE191048_RAW.tar', 'Series_platform_id': 'GPL21103', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA789542'}\n",
      "707 PRJNA798426 GSE193940 {'Series_title': 'An RNAi Screen of RNA Helicases Identifies eIF4A3 as a Regulator of Embryonic Stem Cell Identity [Ribo-seq]', 'Series_geo_accession': 'GSE193940', 'Series_status': 'Public on Dec 09 2022', 'Series_submission_date': 'Jan 18 2022', 'Series_last_update_date': 'Dec 10 2022', 'Series_pubmed_id': '36416264', 'Series_summary': 'RNA helicases are involved in multiple steps of RNA metabolism to direct their roles in gene expression, yet their functions in pluripotency control remain largely unexplored. Starting from an RNAi screen of RNA helicases, we identified that eIF4A3, a DEAD-box (Ddx) helicase component of the exon junction complex (EJC), is essential for the maintenance of embryonic stem cells (ESCs). Mechanistically, we show that eIF4A3 post-transcriptionally controls the pluripotency-related cell cycle regulators and that its depletion causes cellular differentiation via cell cycle dysregulation.  Specifically, eIF4A3 is required for the efficient nuclear export of Ccnb1 mRNA, which encodes Cyclin B1, a key component of the pluripotency-promoting pathway during cell cycle progression of ESCs. Our results reveal a previously unappreciated role of eIF4A3 and its associated EJC in the post-transcriptional cell cycle control in maintaining stem cell pluripotency.', 'Series_overall_design': 'eIF4A3 transcriptome profiling was investigated via RNA-seq analysis in mouse embryonic stem cells.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Jianlong,,Wang', 'Series_sample_id': 'GSM5823888', 'Series_contact_name': 'Jianlong,,Wang', 'Series_contact_laboratory': 'Jianlong Wang', 'Series_contact_department': 'Department of Medicine', 'Series_contact_institute': 'Columbia University Irving Medical Center', 'Series_contact_address': '650 W. 168th St.', 'Series_contact_city': 'New York', 'Series_contact_state': 'New York', 'Series_contact_zip/postal_code': '10032', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE193nnn/GSE193940/suppl/GSE193940_RAW.tar', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA798426'}\n",
      "708 PRJNA800783 GSE195480 {'Series_title': 'Systemic Delivery of an AAV9 Exon Skipping Vector Significantly Improves or Prevents Features of Duchenne Muscular Dystrophy in the Dup2 Mouse Model', 'Series_geo_accession': 'GSE195480', 'Series_status': 'Public on Jan 29 2022', 'Series_submission_date': 'Jan 26 2022', 'Series_last_update_date': 'Aug 24 2022', 'Series_pubmed_id': '35949298', 'Series_summary': 'Conclusions: The results demonstrate the robustness, durability, and safety of exon 2 skipping following delivery of scAAV9.U7snRNA.ACCA, supporting its clinical use.', 'Series_overall_design': 'Skeletal muscle mRNA and ribosome footprinting profiles from Dmd Dup2 mice treated with an AAV9 exon skipping vector', 'Series_type': 'Other', 'Series_contributor': 'Robert,,Weiss', 'Series_sample_id': 'GSM5838190', 'Series_contact_name': 'Robert,B,Weiss', 'Series_contact_email': 'bob.weiss@genetics.utah.edu', 'Series_contact_phone': '(801) 585-5606', 'Series_contact_laboratory': 'EIHG, Rm 2260B', 'Series_contact_department': 'Human Genetics', 'Series_contact_institute': 'University of Utah', 'Series_contact_address': '15 N 2030 E', 'Series_contact_city': 'Salt Lake City', 'Series_contact_state': 'UT', 'Series_contact_zip/postal_code': '84112', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE195nnn/GSE195480/suppl/GSE195480_ReadCounts-Dup2_TA_Unntreated-Treated_Top1000.txt.gz', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA800783'}\n",
      "709 PRJNA805181 GSE196483 {'Series_title': 'Ribosome profiling of Murine Embryonic Fibroblasts in the context of different Fbxo4 and hnRNPK status.', 'Series_geo_accession': 'GSE196483', 'Series_status': 'Public on Sep 21 2022', 'Series_submission_date': 'Feb 10 2022', 'Series_last_update_date': 'Dec 22 2022', 'Series_pubmed_id': '36329064', 'Series_summary': 'Background: SCF-Fbxo4 is suspected to regulate activity of RNA binding protein hnRNPK by non-degradive ubiquitylation. Research strategy: To determine the genomic output of SCF-Fbxo4 - hnRNPK interplay, murine embryonic fibroblasts (MEFs) with different status of hnRNPK/Fbxo4 were subjected to Ribosome profiling. Sequencing was performed on total RNA and RNA protected by ribosomes for each condition.', 'Series_overall_design': 'Applying RNAi strategy hnRNPK was depleted in either wild type MEFs or MEFs Fbxo(-/-). MEFs conditioned in 4 different ways: wt- control (I), Fbxo4 knock-out (II), Fbxo4 knock-out + hnRNPK RNAi knock-down (III) and RNAi hnRNPK knock-down only (IV) were subjected to sequencing of total RNA and Ribosome protected fragment.', 'Series_type': 'Other', 'Series_contributor': 'B,,Mucha', 'Series_sample_id': 'GSM5885029', 'Series_contact_name': 'J Alan,,Diehl', 'Series_contact_email': 'jad283@case.edu', 'Series_contact_phone': '2163680268', 'Series_contact_laboratory': 'Deihl Lab', 'Series_contact_department': 'Department of Biochemistry', 'Series_contact_institute': 'Case Western Reserve University', 'Series_contact_address': '2109 Adelbert Rd. SOM Wood/W402', 'Series_contact_city': 'Cleveland', 'Series_contact_state': 'Ohio', 'Series_contact_zip/postal_code': '44106', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE196nnn/GSE196483/suppl/GSE196483_table_norCount_RNAandRPF.txt.gz', 'Series_platform_id': 'GPL21626', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA805181'}\n",
      "710 PRJNA807729 GSE196927 {'Series_title': 'CRISPR/Cas9-mediated gene knockout screen uncovers novel open reading frames encoded in lncRNA regulating ER+ breast cancer progression', 'Series_geo_accession': 'GSE196927', 'Series_status': 'Public on Mar 18 2023', 'Series_submission_date': 'Feb 16 2022', 'Series_last_update_date': 'Mar 18 2023', 'Series_pubmed_id': '36856111', 'Series_summary': 'A large number of lncRNAs has been found aberrant expression in breast cancer and paly functional role in tumor progression. However, the role of small peptide hidden in lncRNA are largely unexplored.  In this study, we applied the CRISPR/Cas9 screen and ribosome profiling to systematically discover nocanonical open reading frame encoded in long noncoding RNAs (lncRNAs) and explored their critical roles in ER+ breast cancer.', 'Series_overall_design': 'CRISPR/Cas9 knockout screen and ribosome profiling to systematically discover nocanonical open reading frame encoded in long noncoding RNAs (lncRNAs) and explored their critical roles in ER-positive cancer progression.', 'Series_type': 'Genome binding/occupancy profiling by high throughput sequencing', 'Series_contributor': 'Wei,,Yanjun', 'Series_sample_id': 'GSM6612225', 'Series_contact_name': 'Yiwen,,Chen', 'Series_contact_email': 'YChen26@mdanderson.org', 'Series_contact_phone': '713-745-9283', 'Series_contact_department': 'Bioinformatics & Comp Biology', 'Series_contact_institute': 'UT MD Anderson Cancer Center', 'Series_contact_address': '1400 Pressler Street,  Unit 1410', 'Series_contact_city': 'Houston', 'Series_contact_state': 'TX', 'Series_contact_zip/postal_code': '77030', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE196nnn/GSE196927/suppl/GSE196927_MCF7.sgRNA.txt.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA807729'}\n",
      "711 PRJNA809655 GSE197276 {'Series_title': 'Identification of an RNA-binding region in 18S dimethyladenosine methyltransferase DIMT1 essential for proliferation of acute myeloid leukemia cells', 'Series_geo_accession': 'GSE197276', 'Series_status': 'Public on Apr 09 2024', 'Series_submission_date': 'Feb 23 2022', 'Series_last_update_date': 'Apr 09 2024', 'Series_summary': 'Aberrant assembly and function of ribosomes are implicated in cancers, including acute myeloid leukemia (AML). Here, we find that 18S rRNA methyltransferase DIMT1 is essential for AML proliferation through a non-catalytic function. We demonstrate that a positively charged cleft on DIMT1 plays an important role in RNA binding. Mutations at the identified cleft (5mutA-DIMT1) significantly weakens binding of DIMT1 to rRNA in vitro and in cells. Strikingly, the RNA-binding deficient 5mutA-DIMT1 is predominantly located in the nucleoplasm, in contrast with the primarily nucleolar localization of wild-type DIMT1. Furthermore, we demonstrate that rRNA binding is required for DIMT1 to undergo liquid-liquid phase separation in vitro, which likely promotes DIMT1’s nucleolar localization and its role in rRNA processing in cells. Importantly, re-expression of wild-type but not 5mutA-DIMT1 supports AML proliferation in competition-based cell proliferation assays. These results highlight the importance of targeting the catalytically independent RNA-binding activity of DIMT1 for therapy of AML.', 'Series_overall_design': 'Ribosome profiling followed by high-throughput sequencing (ribo-seq) in MOLM13C cells with DIMT1 or Rosa (a negative control) depletion.', 'Series_type': 'Other', 'Series_contributor': 'Hui,,Shen', 'Series_sample_id': 'GSM5912295', 'Series_contact_name': 'Fange,,Liu', 'Series_contact_email': 'liufg@pennmedicine.upenn.edu', 'Series_contact_institute': 'UPENN', 'Series_contact_address': '415 curie blvd, Clinical Research Building Rm 350', 'Series_contact_city': 'philadelphia', 'Series_contact_state': 'PA', 'Series_contact_zip/postal_code': '19104', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE197nnn/GSE197276/suppl/GSE197276_RPF.xlsx', 'Series_platform_id': 'GPL21697', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA809655'}\n",
      "712 PRJNA812862 GSE197909 {'Series_title': 'Profiling Mouse Brown and White Adipocytes to Identify Metabolically Relevant small ORFs and Functional Microproteins [Ribo-seq]', 'Series_geo_accession': 'GSE197909', 'Series_status': 'Public on Dec 13 2022', 'Series_submission_date': 'Mar 04 2022', 'Series_last_update_date': 'Nov 16 2023', 'Series_pubmed_id': '37874679', 'Series_summary': 'The absence of thousands of recently annotated small open reading frame (smORF)-encoded peptides and small proteins (microproteins) from databases has precluded their analysis in metabolism and metabolic disease. Given the outsized importance of small proteins and peptides in metabolism—insulin, leptin, amylin, glucagon, and glucagon-like peptide-1  (GLP-1)—microproteins are a potentially rich source of uncharacterized metabolic regulators. Here, we annotate smORFs in primary differentiated brown, white, and beige mouse adipose cells. Ribosome profiling (Ribo-Seq) detected a total of 3,877 unannotated smORFs. Analysis of RNA-Seq datasets revealed diet-regulated smORF expression in adipose tissues, and validated the adipose translation of the feeding-neuron marker gene Gm8773. Gm8773 encodes the mouse homolog of FAM237B, a neurosecretory protein that stimulates food intake and promotes weight gain in chickens. Testing of recombinant mFAM237B produced similar orexigenic activity in mice further supporting a role for FAM237B a metabolic regulator and part of the brain-adipose axis. Furthermore, we showed that data independent acquisition mass spectrometry (DIA-MS) proteomics can provide a sensitive, flexible, and quantitative platform for identifying microproteins by mass spectrometry. Using this system led to the detection of 58 microproteins from cell culture and an additional 25 from mouse serum. The proteomics data established the anti-inflammatory microprotein AW112010 as a circulating factor, and found that serum levels of a microprotein translated from a FRS2 uORF is elevated in older obese mice. Together, the data highlight the value of this database in examining understudied smORFs and microproteins in metabolic research and identifying additional regulators of metabolism.', 'Series_overall_design': 'total RNA and polyA RNA extracted for deep paired-end transcriptome profiling', 'Series_type': 'Other', 'Series_contributor': 'Christopher,A,Barnes', 'Series_sample_id': 'GSM5932283', 'Series_contact_name': 'April,Elizabeth,Williams', 'Series_contact_email': 'apriljack06@gmail.com, awilliams@salk.edu', 'Series_contact_phone': '7345461645', 'Series_contact_department': 'IGC', 'Series_contact_institute': 'Salk Institute for Biological Studies', 'Series_contact_address': '10010 N Torrey Pines Rd', 'Series_contact_city': 'San Diego', 'Series_contact_state': 'California', 'Series_contact_zip/postal_code': '92037', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE197nnn/GSE197909/suppl/GSE197909_Table1.xlsx', 'Series_platform_id': 'GPL21103', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA812862'}\n",
      "713 PRJNA819978 GSE199422 {'Series_title': 'Ncs2* mediates in vivo virulence of pathogenic yeast through sulphur modification of cytoplasmic transfer RNA', 'Series_geo_accession': 'GSE199422', 'Series_status': 'Public on Sep 05 2023', 'Series_submission_date': 'Mar 25 2022', 'Series_last_update_date': 'Sep 05 2023', 'Series_pubmed_id': '37462076', 'Series_summary': \"Fungal pathogens threaten ecosystems and human health. Understanding the molecular basis of their virulence is key to develop new treatment strategies. Here, we characterize NCS2*, a point mutation identified in a clinical baker's yeast isolate. Ncs2 is essential for 2-thiolation of tRNA and the NCS2* mutation leads to increased thiolation at body temperature. NCS2* yeast exhibits enhanced fitness when grown at elevated temperatures or when exposed to oxidative stress, inhibition of nutrient signalling, and cell-wall stress. Importantly, Ncs2* alters the interaction and stability of the thiolase complex likely mediated by nucleotide binding. The absence of 2-thiolation abrogates the in vivo virulence of pathogenic baker's yeast in infected mice. Finally, hypomodification triggers changes in colony morphology and hyphae formation in the common commensal pathogen Candida albicans resulting in decreased virulence in a human cell culture model. These findings demonstrate that 2-thiolation of tRNA acts as a key mediator of fungal virulence and reveal new mechanistic insights into the function of the highly conserved tRNA-thiolase complex.\", 'Series_overall_design': 'Wild type and 2-thiolation deficient Candida albicans strains were analysed by ribosome profiling at 30 °C, 37 °C and 41 °C.', 'Series_type': 'Other', 'Series_contributor': 'Sebastian,A,Leidel', 'Series_sample_id': 'GSM5972518', 'Series_contact_name': 'Sebastian,Andreas,Leidel', 'Series_contact_email': 'sebastian.leidel@unibe.ch', 'Series_contact_phone': '+41 31 631 42 96', 'Series_contact_laboratory': 'Research group for RNA Biochemistry', 'Series_contact_department': 'Department of Chemistry and Biochemistry', 'Series_contact_institute': 'University of Bern', 'Series_contact_address': 'Freiestrasse 3', 'Series_contact_city': 'Bern', 'Series_contact_state': 'Bern', 'Series_contact_zip/postal_code': '3012', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE199nnn/GSE199422/suppl/GSE199422_CanAlb_WT_and_NCS2_KO_raw_counts.txt.gz', 'Series_platform_id': 'GPL22403', 'Series_platform_taxid': '5476', 'Series_sample_taxid': '5476', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA819978'}\n",
      "714 PRJNA819979 GSE199421 {'Series_title': 'A highly conserved tRNA modification contributes to C. albicans filamentation and virulence', 'Series_geo_accession': 'GSE199421', 'Series_status': 'Public on Apr 04 2024', 'Series_submission_date': 'Mar 25 2022', 'Series_last_update_date': 'Jul 05 2024', 'Series_pubmed_id': '38587411', 'Series_web_link': 'https://journals.asm.org/doi/10.1128/spectrum.04255-22', 'Series_summary': 'tRNA modifications play important roles in maintaining translation accuracy in all domains of life. Disruptions in the tRNA modification machinery, especially of the anticodon stem loop, can be lethal for many bacteria and lead to a broad range of phenotypes in baker’s yeast. Very little is known about the function of tRNA modifications in host-pathogen interactions, where rapidly changing environments and stresses require fast adaptations. We found that two closely related fungal pathogens of humans, the highly pathogenic Candida albicans and its much less pathogenic sister species, Candida dubliniensis, differ in the function of a tRNA-modifying enzyme. This enzyme, Hma1, exhibits species\\xadspecific effects on the ability of the two fungi to grow in the hypha morphology, which is central to their virulence potential. We show that Hma1 has tRNA-threonylcarbamoyladenosine dehydratase activity, and its deletion alters ribosome occupancy, especially at 37°C—the body temperature of the human host. A C. albicans HMA1 deletion mutant also shows defects in adhesion to and invasion into human epithelial cells and shows reduced virulence in a fungal infection model. This links tRNA modifications to host-induced filamentation and virulence of one of the most important fungal pathogens of humans.', 'Series_overall_design': 'Candida albicans wild type and hma1 knockout strains were grown at 30°C and 37°C and analyzed by ribosome profiling to study the role of ct6A in translation.', 'Series_type': 'Other', 'Series_contributor': 'Sascha,,Brunke', 'Series_sample_id': 'GSM5972506', 'Series_contact_name': 'Sebastian,Andreas,Leidel', 'Series_contact_email': 'sebastian.leidel@unibe.ch', 'Series_contact_phone': '+41 31 631 42 96', 'Series_contact_laboratory': 'Research group for RNA Biochemistry', 'Series_contact_department': 'Department of Chemistry and Biochemistry', 'Series_contact_institute': 'University of Bern', 'Series_contact_address': 'Freiestrasse 3', 'Series_contact_city': 'Bern', 'Series_contact_state': 'Bern', 'Series_contact_zip/postal_code': '3012', 'Series_contact_country': 'Switzerland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE199nnn/GSE199421/suppl/GSE199421_CanAlb_HMA1_KO_and_WT_raw_counts.txt.gz', 'Series_platform_id': 'GPL22403', 'Series_platform_taxid': '5476', 'Series_sample_taxid': '5476', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA819979'}\n",
      "715 PRJNA887109 GSE214816 {'Series_title': 'Differential ribosome engagement profiling between undifferentiated and neuron-differentiated P19 cells', 'Series_geo_accession': 'GSE214816', 'Series_status': 'Public on Aug 04 2023', 'Series_submission_date': 'Oct 04 2022', 'Series_last_update_date': 'Sep 14 2023', 'Series_pubmed_id': '37636056', 'Series_summary': 'Translation is tightly regulated during differentiation. To gain insights into the mechanism regulating translational activity at the transcript level, we performed Ribosome Profiling of undifferentiated and neuron-differentiated P19 cells. Relative changes in translational activity between the undifferentiated and differentiated cells show an increase in the translational activity of the transcripts encoding for ribosomal proteins compared to the rest of the transcriptome. Thus, in this differentiation system, the biogenesis of the translational machinery is tightly controlled at the translational level.', 'Series_overall_design': 'P19 cells were differentiated into neuronal cells and then lyzed for material collection. Four biological replicates were collected before differentiation (D0), and four biological replicates were collected after six days of differentiation (D6). An RNA-seq library and a Ribosome Protected Fragment (RPF) library were generated for each sample. To estimate the change in ribosome engagement for each transcript between conditions, we compared the ratio of RFP reads to RNA-seq reads per transcript at D6 and D0.', 'Series_type': 'Other', 'Series_contributor': 'Marcos,,Morgan', 'Series_sample_id': 'GSM6616260', 'Series_contact_name': 'Marcos,,Morgan', 'Series_contact_institute': 'NIEHS', 'Series_contact_address': '111 TW Alexander Dr', 'Series_contact_city': 'Durham', 'Series_contact_zip/postal_code': '27709', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE214nnn/GSE214816/suppl/GSE214816_DTE_D6VSD0.csv.gz', 'Series_platform_id': 'GPL19057', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA887109'}\n",
      "716 PRJNA922087 GSE222433 {'Series_title': 'Circadian clock regulation by compartmentalized protein translation', 'Series_geo_accession': 'GSE222433', 'Series_status': 'Public on Jun 24 2023', 'Series_submission_date': 'Jan 09 2023', 'Series_last_update_date': 'Jun 26 2023', 'Series_summary': 'Cells exquisitely compartmentalize many biochemical reactions through phase-separated membrane-less organelles. Besides spatial organization, many biological processes are temporally regulated. While both transcription and translation oscillate in a rhythmic manner, the regulatory mechanism of the latter is understudied. Here we report that the translational regulation during circadian cycles is associated with oscillating phase separation controlled by Ataxin-2 and Ataxin-2L. Ataxin-2/2L forms rhythmic condensates through phase separation in the mammalian central clock suprachiasmatic nucleus to selectively concentrate the transcripts of the core clock genes and recruit the mRNA translation machinery. The rhythmic translational activation is abolished in cells and mouse circadian behavior is altered in the absence of Ataxin-2/2L. During aging or neurodegeneration, Ataxin-2/2L form large, irreversible puncta that no longer regulate translation. Overall, our results unveil Ataxin-2/2L’s role as the master regulators of rhythmic translation via a oscillating phase-separation, and explained the reason for their dysfunction in disease states.', 'Series_overall_design': 'Ribosome profiling of U2OS WT, Ataxin-2 KO, Ataxin-2L KO and DKO cells at 4 hours and 12 hours after synchronization with DXMS.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Chujie,,Sun', 'Series_sample_id': 'GSM7325155', 'Series_contact_name': 'Xuerui,,Yang', 'Series_contact_email': 'yangxuerui@tsinghua.edu.cn', 'Series_contact_department': 'School of Life Sciences', 'Series_contact_institute': 'Tsinghua University', 'Series_contact_address': 'Tsinghua University', 'Series_contact_city': 'Beijing', 'Series_contact_state': 'Beijing', 'Series_contact_zip/postal_code': '100084', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE222nnn/GSE222433/suppl/GSE222433_RAW.tar', 'Series_platform_id': 'GPL20795', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA922087'}\n",
      "717 PRJNA923627 GSE222825 {'Series_title': 'Ribo-seq of decitabine-treated WT HL-60 cells', 'Series_geo_accession': 'GSE222825', 'Series_status': 'Public on Jan 13 2023', 'Series_submission_date': 'Jan 13 2023', 'Series_last_update_date': 'Jan 13 2023', 'Series_summary': 'Ribolog was used to compare translational efficiency across conditions (https://github.com/goodarzilab/Ribolog). Briefly, Ribolog applies a logistic regression to model individual Ribo-seq and RNA-seq reads in order to provide estimates of logTER (i.e., logFC in TE) and its associated p-value across the coding transcriptome.', 'Series_overall_design': 'Ribosome profiling was performed as previously described (PMID: 28579404) on HL-60 cells treated with DMSO or decitabine.', 'Series_type': 'Other', 'Series_contributor': 'Abolfazl,,Arab', 'Series_sample_id': 'GSM6932805', 'Series_contact_name': 'Abolfazl,,Arab', 'Series_contact_email': 'abarbiology@gmail.com, abea@arcinstitute.org', 'Series_contact_laboratory': 'Gilbert Lab', 'Series_contact_institute': 'Arc Institute', 'Series_contact_address': '3160 Porter Dr', 'Series_contact_city': 'Palo Alto', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94304', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE222nnn/GSE222825/suppl/GSE222825_DAC_Ribo_seq_delta_te_table.txt.gz', 'Series_citation': 'Alex Y Ge, Abolfazl Arab, Raymond Dai, Albertas Navickas, Lisa Fish, Kristle Garcia, Hosseinali Asgharian, Hani Goodarzi, and Luke A Gilbert. A multiomics approach reveals RNA dynamics promote cellular sensitivity to DNA hypomethylation. bioRxiv 2022.12.14.518457; doi:10.1101/2022.12.14.518457', 'Series_platform_id': 'GPL20301', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA923627'}\n",
      "718 PRJNA956925 GSE229979 {'Series_title': 'Human SAMD9 is a Poxvirus-Activatable Anticodon Nuclease Inhibiting Codon-Specific Protein Synthesis', 'Series_geo_accession': 'GSE229979', 'Series_status': 'Public on Jun 07 2023', 'Series_submission_date': 'Apr 18 2023', 'Series_last_update_date': 'Sep 07 2023', 'Series_pubmed_id': '37285440', 'Series_web_link': 'https://www.science.org/doi/10.1126/sciadv.adh8502', 'Series_summary': 'As a defense strategy against viruses or competitors, some microbes employ anticodon nucleases (ACNases) to deplete essential tRNAs, effectively halting global protein synthesis. However, this mechanism has not been observed in multicellular eukaryotes. Here, we report that human SAMD9 is an ACNase that specifically cleaves phenylalanine tRNA (tRNAPhe), resulting in codon-specific ribosomal pausing and stress signaling. While SAMD9 ACNase activity is normally latent in cells, it can be activated by poxvirus infection or rendered constitutively active by SAMD9 mutations associated with various human disorders, revealing tRNAPhe depletion as an antiviral mechanism and a pathogenic condition in SAMD9 disorders. We identified the N-terminal effector domain of SAMD9 as the ACNase, with substrate specificity primarily determined by a eukaryotic tRNAPhe-specific 2’-O-methylation at the wobble position, making virtually all eukaryotic tRNAPhe susceptible to SAMD9 cleavage. Notably, the structure and substrate specificity of SAMD9 ACNase differ from known microbial ACNases, suggesting convergent evolution of a common immune defense strategy targeting tRNAs.', 'Series_overall_design': 'Ribo-seq and RNA-seq of uninfected HeLa cells as well as HeLa cells infected with vK1-C7- and WT VACV. HeLa cells in 2x150 mM dishes were left uninfected or infected with vK1-C7- or WT VACV WR in the absence or presence of AraC (50 ug/ml final) for 8 hours. The cells were treated with cycloheximide (0.1 mg/ml) for 1 min at 37oC. The cells were lysed with 800 ul mammalian lysis buffer. 100 ul of lysate was used for extracting total RNA, while 200 ul of lysate was used for ribosome foot printing.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Yuanhui,,Mao', 'Series_sample_id': 'GSM7182260', 'Series_contact_name': 'Yan,,Xiang', 'Series_contact_email': 'xiangy@uthscsa.edu', 'Series_contact_phone': '2105670884', 'Series_contact_laboratory': 'Yan Xiang', 'Series_contact_department': 'Microbiology, Immunology and Molecular Genetics', 'Series_contact_institute': 'UT Health San Antonio', 'Series_contact_address': '7703 FLOYD CURL DR', 'Series_contact_city': 'San Antonio', 'Series_contact_state': 'TX', 'Series_contact_zip/postal_code': '78229', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE229nnn/GSE229979/suppl/GSE229979_RAW.tar', 'Series_platform_id': 'GPL21290', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA956925'}\n",
      "719 PRJNA297288 GSE73553 {'Series_title': 'Circadian and feeding rhythms differentially affect rhythmic mRNA transcription and translation in mouse liver [Ribosome Profiling]', 'Series_geo_accession': 'GSE73553', 'Series_status': 'Public on Nov 10 2015', 'Series_submission_date': 'Sep 29 2015', 'Series_last_update_date': 'Jul 06 2021', 'Series_pubmed_id': '28475894', 'Series_summary': 'Diurnal oscillations of gene expression are a hallmark of rhythmic physiology across most living organisms. Such oscillations are controlled by the interplay between the circadian clock and feeding rhythms. While rhythmic mRNA accumulation has been extensively studied, comparatively less is known about their transcription and translation. Here, we quantified simultaneously temporal transcription, accumulation, and translation of mouse liver mRNAs under physiological light-dark conditions and ad libitum or night-restricted feeding in wild-type and Bmal1 deficient animals. We found that rhythmic transcription predominantly drives rhythmic mRNA accumulation and translation for a majority of genes. Comparison of wild-type and Bmal1 KO mice shows that circadian clock and feeding rhythms have broad impact on rhythmic genes expression, Bmal1 deletion having surprisingly more impact at the post-transcriptional level. Translation efficiency is differentially regulated during the diurnal cycle for genes with 5’-TOP sequences and for genes involved in mitochondrial activity and harboring a TISU motif. The increased translation efficiency of 5’-TOP and TISU genes is mainly driven by feeding rhythms but Bmal1 deletion impacts also amplitude and phase of translation, including TISU genes. Together this study emphasizes the complex interconnections between circadian and feeding rhythms at several steps ultimately determining rhythmic gene expression and translation.', 'Series_overall_design': 'RIbosome profiling from total RNA of mouse liver during the dirunal cycle. Time-series ribosome density of wild type (WT) and Bmal -/- mice under ad libitum and night restriced feeding regimen were generated by deep sequencing.', 'Series_type': 'Other', 'Series_contributor': 'Frédéric,,Gachon', 'Series_sample_id': 'GSM1897939', 'Series_contact_name': 'Frédéric,,Gachon', 'Series_contact_email': 'frederic.gachon@uq.edu.au', 'Series_contact_phone': '+41 791926057', 'Series_contact_department': 'Institute for Molecular Bioscience', 'Series_contact_institute': 'University Of Queensland', 'Series_contact_address': '306 Carmody Road', 'Series_contact_city': 'St Lucia', 'Series_contact_state': 'QLD', 'Series_contact_zip/postal_code': '4072', 'Series_contact_country': 'Australia', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP064304'}\n",
      "720 PRJNA510323 GSE123919 {'Series_title': 'Translational profiling of dorsal root ganglia and spinal cord in a mouse model of neuropathic pain', 'Series_geo_accession': 'GSE123919', 'Series_status': 'Public on Dec 18 2018', 'Series_submission_date': 'Dec 17 2018', 'Series_last_update_date': 'Mar 28 2019', 'Series_pubmed_id': '30906902', 'Series_web_link': 'https://doi.org/10.1016/j.ynpai.2018.04.001', 'Series_summary': 'Acute pain serves as a protective mechanism, guiding the organism away from actual or potential tissue injury. In contrast, chronic pain is a debilitating condition without any obvious physiological function. The transition to, and the maintenance of chronic pain require new gene expression to support biochemical and structural changes within the pain pathway. The regulation of gene expression at the level of mRNA translation has emerged as an important step in the control of protein expression in the cell. Recent studies show that signaling pathways upstream of mRNA translation, such as mTORC1 and ERK, are upregulated in chronic pain conditions, and their inhibition effectively alleviates pain in several animal models. Despite this progress, mRNAs whose translation is altered in chronic pain conditions remain largely unknown. Here, we performed genome-wide translational profiling of dorsal root ganglion (DRG) and spinal cord dorsal horn tissues in a mouse model of neuropathic pain, spared nerve injury (SNI), using the ribosome profiling technique. We identified distinct subsets of mRNAs that are differentially translated in response to nerve injury in both tissues. We discovered key converging upstream regulators and pathways linked to mRNA translational control and neuropathic pain. Our data are crucial for the understanding of mechanisms by which mRNA translation promotes persistent hypersensitivity after nerve injury.', 'Series_overall_design': 'Four conditions were analysed, with each condition having duplicate samples. That is, eight tissue samples were collected. The eight samples were lysed and the lysate was divided into two parts, one was used to prepare a ribosome footprint library and the other fraction was used to prepare whole mRNA library. Thus, we had 16 libraries in total for sequencing. The four conditions studied were: spinal_cord_SNI with spinal_cord_SHam as control and DRG_SNI with SRG_sham as control.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Arkady,,Khoutorsky', 'Series_sample_id': 'GSM3516891', 'Series_contact_name': 'Arkady,,Khoutrosky', 'Series_contact_email': 'arkady.khoutorsky@mcgill.ca', 'Series_contact_phone': '+1 514-398-5764', 'Series_contact_laboratory': 'Room 1207', 'Series_contact_department': 'Department of Anesthesia', 'Series_contact_institute': 'McGill University', 'Series_contact_address': '3655 Prom. Sir-William-Osler', 'Series_contact_city': 'Montreal', 'Series_contact_state': 'Quebec', 'Series_contact_zip/postal_code': 'H3G 1Y6', 'Series_contact_country': 'Canada', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE123nnn/GSE123919/suppl/GSE123919_readme.txt.gz', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP173586'}\n",
      "721 PRJNA548091 GSE132441 {'Series_title': 'Deficiency of mitoribosomal S10 protein affects translation and splicing in Arabidopsis mitochondria', 'Series_geo_accession': 'GSE132441', 'Series_status': 'Public on Oct 29 2019', 'Series_submission_date': 'Jun 10 2019', 'Series_last_update_date': 'Nov 21 2019', 'Series_pubmed_id': '31732734', 'Series_summary': 'The ribosome is not only a protein-making machine, but also a regulatory element in protein synthesis. This view is supported by our earlier data showing that Arabidopsis mitoribosomes altered due to the silencing of the nuclear RPS10 gene encoding mitochondrial ribosomal protein S10 differentially translate mitochondrial transcripts compared with the wild-type. Here, we used ribosome profiling to determine the contribution of transcriptional and translational control in the regulation of protein synthesis in rps10 mitochondria compared with the wild-type ones. Oxidative phosphorylation system proteins are preferentially synthesized in wild-type mitochondria but this feature is lost in the mutant. The rps10 mitoribosomes show slightly reduced translation efficiency of most respiration-related proteins and at the same time markedly more efficiently synthesize ribosomal proteins and MatR and TatC proteins. The mitoribosomes deficient in S10 protein protect shorter transcript fragments which exhibit a weaker 3-nucleotide periodicity compared with the wild-type. The decrease in the triplet periodicity is particularly drastic for genes containing introns. Notably, splicing is considerably less effective in the mutant, indicating an unexpected link between the deficiency of S10 and mitochondrial splicing. Thus, a shortage of the mitoribosomal S10 protein has wide-ranging consequences on mitochondrial gene expression.', 'Series_overall_design': 'Mitochondrial ribosome footprints from youngest leaves of 9-10 week-old wildtype or P2 and P3 phenotypes of heterozygous rps10 mutant', 'Series_type': 'Other', 'Series_contributor': 'Hanna,,Janska', 'Series_sample_id': 'GSM3863562', 'Series_contact_name': 'Artur,,Tomal', 'Series_contact_email': 'artur.tomal2@uwr.edu.pl', 'Series_contact_phone': '+480713756292', 'Series_contact_laboratory': 'Department of Cellular Molecular Biology', 'Series_contact_department': 'Faculty of Biotechnology', 'Series_contact_institute': 'University of Wroclaw', 'Series_contact_address': 'Fryderyka Joliot-Curie 14a', 'Series_contact_city': 'Wroclaw', 'Series_contact_zip/postal_code': '50-383', 'Series_contact_country': 'Poland', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE132nnn/GSE132441/suppl/GSE132441_RAW.tar', 'Series_platform_id': 'GPL17970', 'Series_platform_taxid': '3702', 'Series_sample_taxid': '3702', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP200966'}\n",
      "722 PRJNA552552 GSE133792 {'Series_title': 'RFP: The translational landscape of ground state pluripotency', 'Series_geo_accession': 'GSE133792', 'Series_status': 'Public on Mar 01 2020', 'Series_submission_date': 'Jul 03 2019', 'Series_last_update_date': 'Apr 03 2020', 'Series_pubmed_id': '32238817', 'Series_web_link': 'https://www.nature.com/articles/s41467-020-15449-9', 'Series_summary': 'Translational control plays a central role in regulation of gene expression and can lead to significant divergence between mRNA- and protein-abundance. Here we used genome-wide approaches combined with time-course analysis to measure the mRNA-abundance, mRNA-translation rate and protein expression during the transition of naïve-to-primed mouse embryonic stem cells (ESCs). We find that the ground state ESCs cultured with GSK3-, MEK-inhibitors and LIF (2iL) display higher ribosome density on a selective set of mRNAs. This set of mRNAs undergo  strong translational buffering to maintain stable protein expression levels in 2iL-ESCs. Importantly, we show that the global alteration of cellular proteome during the transition of naïve to primed pluripotency is largely accompanied by transcriptional rewiring. Thus, we provide a comprehensive and detailed overview of the global changes in gene expression in different states of ESCs and dissect the relative contributions of RNA-transcription, translation and regulation of protein stability in controlling protein abundance.', 'Series_overall_design': 'Analysis of genome wide mRNA translation by ribosome profiling and mRNA expression by RNA-seq. 9-conditions were used that include: 2iL, 2iLday1, 2iLday3, 2iLday7, SL, SLday1, SLday3, SLday7, EpiLSCs. Biological duplicates were used per condition; they are indicated as Rep1 and Rep2. In total 18 samples for RNA-seq and 18-samples for Ribosome profiling were generated.', 'Series_type': 'Other', 'Series_sample_id': 'GSM3927401', 'Series_contact_name': 'Hendrik,G,Stunnenberg', 'Series_contact_email': 'h.stunnenberg@ncmls.ru.nl', 'Series_contact_institute': 'Radboud University', 'Series_contact_address': 'Geert Grooteplein 30', 'Series_contact_city': 'Nijmegen', 'Series_contact_zip/postal_code': '6525GA', 'Series_contact_country': 'Netherlands', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE133nnn/GSE133792/suppl/GSE133792_RAW.tar', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP212919'}\n",
      "723 PRJNA597326 GSE142528 {'Series_title': 'Sulfur-dependent translational regulation via methylation multiplicity of 18S rRNA', 'Series_geo_accession': 'GSE142528', 'Series_status': 'Public on Mar 11 2021', 'Series_submission_date': 'Dec 23 2019', 'Series_last_update_date': 'Mar 11 2021', 'Series_pubmed_id': '33691096', 'Series_summary': 'N6-methyladenosine (m6A) is a conserved nucleoside modification that regulates many facets of RNA metabolism and cellular physiology. Using quantitative mass spectrometry, we find that the universally conserved tandem adenosines at the 3’ end of 18S rRNA (A1781 and A1782 in Saccharomyces cerevisiae), which were thought to be constitutively di-methylated (i.e. N6, N6-dimethyladenosine or m62A), are also mono-methylated (i.e. m6A). Although present at substoichiometric amounts, m6A at these positions increases significantly and specifically in response to sulfur starvation in both yeast and mammalian cells. Combining yeast genetics and ribosome-profiling, we further demonstrate that ribosomes bearing different numbers of methyl groups (zero, one, and two) at these tandem adenosines exhibit distinct translation profiles in a sulfur-dependent fashion. Our work thus reveals methylation multiplicity as a mechanism to regulate translation and also uncovers the functional importance of the ubiquitous m62A modification in 18S rRNA.', 'Series_overall_design': 'We performed ribosome profiling and RNA-seq on three Saccharomyces cerevisiae strains growing with and without methionine', 'Series_type': 'Other', 'Series_contributor': 'Benjamin,P,Tu', 'Series_sample_id': 'GSM4231366', 'Series_contact_name': 'Kuanqing,,Liu', 'Series_contact_email': 'jensliu1984921@gmail.com', 'Series_contact_laboratory': 'Benjamin P. Tu', 'Series_contact_department': 'Biochemistry', 'Series_contact_institute': 'UT Southwestern Medical Center', 'Series_contact_address': '5323 Harry Hines Blvd', 'Series_contact_city': 'Dallas', 'Series_contact_state': 'Texas', 'Series_contact_zip/postal_code': '75390', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE142nnn/GSE142528/suppl/GSE142528_rpkm_mrna_masked.txt.gz', 'Series_platform_id': 'GPL21656', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP238588'}\n",
      "724 PRJNA739680 GSE178576 {'Series_title': 'Targeting epithelial-mesenchymal plasticity via ribosome inhibition to reduce chemoresistant metastasis of breast cancer [bulk RNA-seq]', 'Series_geo_accession': 'GSE178576', 'Series_status': 'Public on Sep 01 2022', 'Series_submission_date': 'Jun 21 2021', 'Series_last_update_date': 'Oct 08 2024', 'Series_pubmed_id': '39259576', 'Series_summary': 'Using an EMT lineage-tracing model, we identified tumor cells in the transitioning phases of both EMT and MET. Further analyses revealed that the ribosome biogenesis (RiBi) pathway was specifically elevated in these transitioning cells, regardless of their destined phenotypical fates.', 'Series_overall_design': 'mRNA profiling of flow cytometry sorted RFP+, GFP+, and Double+ Tri-PyMT cells. 3 samples of each type of cells were sequenced.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Dingcheng,,Gao', 'Series_sample_id': 'GSM5393999', 'Series_contact_name': 'Dingcheng,,Gao', 'Series_contact_email': 'dig2009@med.cornell.edu', 'Series_contact_department': 'CT Surgery', 'Series_contact_institute': 'Weill Cornell Medical College', 'Series_contact_address': '1300 York Ave', 'Series_contact_city': 'New York', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '10065', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE178nnn/GSE178576/suppl/GSE178576_RGD_Bulk.txt.gz', 'Series_platform_id': 'GPL21103', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP324846'}\n",
      "725 PRJNA999250 GSE239419 {'Series_title': 'Reallocation of ribosomes as a disease mechanism of Amyotrophic Lateral Sclerosis', 'Series_geo_accession': 'GSE239419', 'Series_status': 'Public on Dec 06 2023', 'Series_submission_date': 'Jul 27 2023', 'Series_last_update_date': 'Dec 06 2023', 'Series_pubmed_id': '38026702', 'Series_summary': 'The molecular mechanism underlying Amyotrophic Lateral Sclerosis (ALS) is still largely unclear. The RNA binding protein TDP-43 is a major contributor and hallmark of the disease. Here, we show that in multiple TDP-43 models of ALS the alterations of the physiological crosstalk between free RNA/granules and polysomes lead to the global reorganisation of translational resources. We exploited in cellulo and in vivo models of TDP-43 proteinopathy and a unique tag-free miniaturised polysomal profiling coupled with RNA-Seq and biochemical assays with subcellular (i.e. cell body and axon) and compartment (i.e. RNA granules and polysomes) resolution. Using these models, we identified: i) a widespread axonal- and polysome-specific downregulation of TDP-43 target mRNAs, ii) the rewiring of mRNAs in granule and polysome compartments, iii) decrease in ribosome kinetics, iv) a robust ribosome engagement with residual mRNAs and v) downstream alterations in protein synthesis of TDP-43 non-target mRNAs. Irrespective of the ALS model used, we observed a snowball effect in RNA metabolism with redistribution of translational resources. Based on our results we propose a systems-oriented mechanism of disease pathogenesis, leveraging on the conceptual framework of reallocation of cellular resources.', 'Series_overall_design': 'In this study, we performed RNA-Seq of total, polysomal and free/granule-associated RNA from mouse primary cortical neurons control and expressing tRFP, exogenous TDP43-WT or TDP43-A315T to investigate the repertoires of defects caused by TDP-43 proteinopathy at subcellular (cell body vs axon) and compartment (RNA granules and polysomes) levels.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Gabriella,,Viero', 'Series_sample_id': 'GSM7664513', 'Series_contact_name': 'Fabio,,Lauria', 'Series_contact_email': 'fabio.lauria@ibf.cnr.it, fab.lauria.88@gmail.com', 'Series_contact_institute': 'Institute of Biophysics (Italian National Research Council)', 'Series_contact_address': 'Via alla Cascata 56', 'Series_contact_city': 'Povo (TN)', 'Series_contact_zip/postal_code': '38123', 'Series_contact_country': 'Italy', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE239nnn/GSE239419/suppl/GSE239419_reads_per_gene_table.txt.gz', 'Series_platform_id': 'GPL17021', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA999250'}\n",
      "726 PRJNA445182 GSE112186 {'Series_title': 'RPL12/uL11 phosphorylation regulates translation during mitosis [RIP-seq]', 'Series_geo_accession': 'GSE112186', 'Series_status': 'Public on May 03 2019', 'Series_submission_date': 'Mar 21 2018', 'Series_last_update_date': 'May 03 2019', 'Series_pubmed_id': '30220558', 'Series_summary': 'Emerging evidence indicates that heterogeneity in ribosome composition can give rise to specialized functions. Until now, research mainly focused on differences in core ribosomal proteins and associated factors. The impact of posttranslational modifications has not yet been studied systematically. Analyzing ribosome heterogeneity is challenging since individual proteins can be part of different subcomplexes (40S, 60S, 80S and polysomes). Here, we develop polysome proteome profiling to obtain unbiased proteomic maps across ribosomal subcomplexes. Our method combines extensive fractionation by sucrose gradient centrifugation with quantitative mass spectrometry. The high resolution of the profiles allows us to assign proteins to specific subcomplexes. Phosphoproteomics on the fractions reveals that phosphorylation of serine 38 in RPL12/uL11 -- a known mitotic CDK1 substrate -- is strongly depleted in polysomes. Follow-up experiments confirm that RPL12/uL11 phosphorylation regulates translation of specific subsets of mRNAs during mitosis. Together, our results show that posttranslational modification of ribosomal proteins can regulate translation.', 'Series_overall_design': 'Gene expression profiling of immunoprecipitated RPL12 protein-RNA complexes', 'Series_type': 'Other', 'Series_contributor': 'Matthias,,Selbach', 'Series_sample_id': 'GSM3059597', 'Series_contact_name': 'Katharina,,Schmidt-Bleek', 'Series_contact_email': 'katharina.schmidt-bleek@bih-charite.de', 'Series_contact_department': 'Julius Wolff Institute of Biomechanics and Musculoskeletal Regeneration', 'Series_contact_institute': 'Charité - Universitätsmedizin Berlin', 'Series_contact_address': 'Augustenburger Platz 1', 'Series_contact_city': 'Berlin', 'Series_contact_zip/postal_code': '13353', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE112nnn/GSE112186/suppl/GSE112186_RAW.tar', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP136214'}\n",
      "727 PRJNA870413 GSE211440 {'Series_title': 'PRRC2 proteins are translation factors that promote leaky scanning', 'Series_geo_accession': 'GSE211440', 'Series_status': 'Public on Feb 11 2023', 'Series_submission_date': 'Aug 17 2022', 'Series_last_update_date': 'May 12 2023', 'Series_pubmed_id': '36869665', 'Series_summary': 'Most animal mRNAs contain upstream Open Reading Frames (uORFs). These uORFs represent an impediment to translation of the main ORF since ribosomes usually bind the mRNA cap at the 5’ end and then scan for ORFs in a 5’-to-3’ fashion. One way for ribosomes to bypass uORFs is via leaky scanning, whereby the ribosome disregards the uORF start codon. Hence leaky scanning is an important post-transcriptional mechanism affecting gene expression. Few molecular factors regulating or facilitating this process are known. Here we identify the PRRC2 proteins PRRC2A, PRRC2B and PRRC2C as translation initiation factors. We find that they bind other eukaryotic translation initiation factors and preinitiation complexes and are enriched on ribosomes translating mRNAs with uORFs. We find that PRRC2 proteins promote translation of mRNAs containing uORFs by facilitating leaky scanning past uORFs. Since PRRC2 proteins have been associated with cancer, this provides a mechanistic starting point for understanding their physiological and pathophysiological roles.', 'Series_overall_design': 'Selective and total 40S and 80S ribosome footprinting in control versus PRRC2A+B+C knockdown condition. Selective Ribosome footprinting for PRRC2C. Also RNA-seq.', 'Series_type': 'Other', 'Series_contributor': 'Aurelio,,Teleman', 'Series_sample_id': 'GSM6470184', 'Series_contact_name': 'Jonathan,,Bohlen', 'Series_contact_laboratory': 'AG Teleman', 'Series_contact_department': 'B140', 'Series_contact_institute': 'DKFZ', 'Series_contact_address': 'Im Neuenheimer Feld 581', 'Series_contact_city': 'Heidelberg', 'Series_contact_zip/postal_code': '69123', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE211nnn/GSE211440/suppl/GSE211440_Master_Table_80S.xlsx', 'Series_platform_id': 'GPL21697', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA870413'}\n",
      "728 PRJNA661060 GSE157361 {'Series_title': 'Ebp1-selective ribosome profiling analysis of Neuro2a cells with Ebp1-immuno-precipitation', 'Series_geo_accession': 'GSE157361', 'Series_status': 'Public on Mar 23 2021', 'Series_submission_date': 'Sep 02 2020', 'Series_last_update_date': 'Jan 06 2023', 'Series_pubmed_id': '36482253', 'Series_summary': 'We perform Selective Ribosome Profiling (SeRP) analysis of mouse Neuro2a neuronal cultures with Ebp1-immuno-precipitation in biological duplicate to assess the translation-specific function of Ebp1', 'Series_overall_design': 'Each biological replicate analyzed for ribosome-protected mRNA fragments after RNAse digestion in both Ebp-ribosome immuno-precipitation conditions, and the total input ribosome pool, followed by by high-throughput sequencing.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Matthew,L,Kraushar', 'Series_sample_id': 'GSM4763684', 'Series_contact_name': 'Matthew,Lee,Kraushar', 'Series_contact_email': 'matthew.kraushar@molgen.mpg.de', 'Series_contact_department': 'Department of Computational Molecular Biology', 'Series_contact_institute': 'Max Planck Institute for Molecular Genetics', 'Series_contact_address': 'Ihnestrasse 63-73', 'Series_contact_city': 'Berlin', 'Series_contact_state': 'Berlin', 'Series_contact_zip/postal_code': '14195', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE157nnn/GSE157361/suppl/GSE157361_counts_table.csv.gz', 'Series_platform_id': 'GPL21626', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP279771'}\n",
      "729 PRJNA315441 GSE79306 {'Series_title': 'Gene expression profiles of brain endothelial cells during embryonic development at bulk and single-cell levels', 'Series_geo_accession': 'GSE79306', 'Series_status': 'Public on Jul 12 2017', 'Series_submission_date': 'Mar 16 2016', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '28698213', 'Series_summary': 'Conclusions: Our study provides further insights into the temporal and cellular complexity of the developing embryonic brain endothelium. Our data raises many questions and provides a basis for further studies aiming to unravel cellular and molecular mechanisms that underlie vascular development and differentiation in the CNS. We anticipate that additional investigation of the identified transcription factors is likely to be a fruitful approach in understanding and manipulating the development of BBB and to engineer more advanced blood brain barrier in vitro models.', 'Series_overall_design': 'TRAP-seq profiles for E11.5 till E17.5 wild type (WT) brain EC; E14.5 and E17.5 endothelial Ctnnb1 -/- brain EC; E14.5 head, heart, kidney, limb, liver and lung WT EC in triplicate, using Illumina HiSeq 2000. Single cell RNA-sequencing of E14.5 brain EC. RNA-sequenzing of HUVEC infected with either control, Foxf2, Foxq1, ZIC3 or three transcription factors combined in triplicate.', 'Series_type': 'Other', 'Series_contributor': 'Jan,M,Stenman', 'Series_sample_id': 'GSM2091335', 'Series_contact_name': 'Mike,,Hupe', 'Series_contact_email': 'Mike.Hupe@uni-wuerzburg.de', 'Series_contact_phone': '+499313184906', 'Series_contact_department': 'Department of Developmental Biochemistry', 'Series_contact_institute': 'University of Würzburg', 'Series_contact_address': 'Biocenter Am Hubland', 'Series_contact_city': 'Würzburg', 'Series_contact_zip/postal_code': 'D-97074', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE79nnn/GSE79306/suppl/GSE79306_deseq_E175_wt_brain_EC_vs_E175_KO_brain_EC.txt.gz', 'Series_platform_id': 'GPL13112', 'Series_platform_taxid': '10090', 'Series_sample_taxid': '10090', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP071876'}\n",
      "730 PRJNA362545 GSE93830 {'Series_title': 'Profiling Ssb-Nascent Chain Interactions Reveals Principles of Hsp70-Assisted Folding', 'Series_geo_accession': 'GSE93830', 'Series_status': 'Public on Jul 17 2017', 'Series_submission_date': 'Jan 19 2017', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '30158700', 'Series_summary': 'The yeast Hsp70 chaperone Ssb interacts with ribosomes and nascent chains to co-translationally assist protein folding. Here, we present a proteome-wide analysis of Hsp70 function during translation, based on in vivo selective ribosome profiling, that reveals mechanistic principles coordinating translation with chaperone-assisted protein folding. Ssb binds most cytosolic, nuclear, and mitochondrial proteins and a subset of ER proteins, supporting its general chaperone function. Position-resolved analysis of Ssb engagement reveals compartment- and protein-specific nascent chain binding profiles that are coordinated by emergence of positively charged peptide stretches enriched in aromatic amino acids. Ssbs’ function is temporally coordinated by RAC but independent from NAC. Analysis of ribosome footprint densities along orfs reveals that ribosomes translate faster at times of Ssb binding. This is coordinated by biases in mRNA secondary structure, and codon usage as well as the action of Ssb, suggesting chaperones may allow higher protein synthesis rates by actively coordinating protein synthesis with co-translational folding.', 'Series_overall_design': 'Investigating the interaction of Ssb1 and Ssb2 with nascent polypeptide chains in four different strain backgrounds (WT, ssb1/2∆, RAC∆, NAC∆) and the translation speed in WT and ssb1∆ssb2 cells (all in replicates).', 'Series_type': 'Other', 'Series_contributor': 'Bernd,,Bukau', 'Series_sample_id': 'GSM2463217', 'Series_contact_name': 'Kristina,,Döring', 'Series_contact_email': 'k.doering@zmbh.uni-heidelberg.de', 'Series_contact_laboratory': 'Bernd Bukau', 'Series_contact_department': 'ZMBH', 'Series_contact_institute': 'Heidelberg University', 'Series_contact_address': 'Im Neuenheimer Feld 282', 'Series_contact_city': 'Heidelberg', 'Series_contact_zip/postal_code': '69120', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE93nnn/GSE93830/suppl/GSE93830_RAW.tar', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP097127'}\n",
      "731 PRJNA399133 GSE102861 {'Series_title': 'RNA-seq profiling of the Tg(myo6b:GFP-2A-rpl10a-3xHA) zebrafish hair cell IP vs. whole fish IN', 'Series_geo_accession': 'GSE102861', 'Series_status': 'Public on Apr 24 2018', 'Series_submission_date': 'Aug 21 2017', 'Series_last_update_date': 'Jul 25 2021', 'Series_pubmed_id': '29765956', 'Series_summary': 'We have developed and tested the efficiency of the Tg(myo6b:GFP-2A-rpl10a-3xHA) zebrafish to specifically enrich for and evaluate the translatome of inner ear and lateral line sensory hair cells (IP) compared to the whole fish transcriptome (IN).  We show through RNA-seq that HA-tagged ribosome immunoprecipitation significantly enriches for RNA transcripts of known zebrafish hair cell expressed transcripts.', 'Series_overall_design': 'Tg(myo6b:GFP-2A-rpl10a-3xHA) were homogenized at 5dpf and the hair cell translatome was enriched for by HA-tagged ribosome immunoprecipitation, followed by RNA extraction. IP samples were compared to input RNA controls (IN) in independent biological triplicates.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Ronna,,Hertzano', 'Series_sample_id': 'GSM3143177', 'Series_contact_name': 'yang,,song', 'Series_contact_email': 'ysong@som.umaryland.edu', 'Series_contact_department': 'Department: Institute for Genome Sciences', 'Series_contact_institute': 'University of Maryland in Baltimore', 'Series_contact_address': 'HSF-3, 3rd Floor; 670 W Baltimore St.', 'Series_contact_city': 'Baltimore', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '21201', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE102nnn/GSE102861/suppl/GSE102861_RAW.tar', 'Series_platform_id': 'GPL21741', 'Series_platform_taxid': '7955', 'Series_sample_taxid': '7955', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP115900'}\n",
      "732 PRJNA479682 GSE116570 {'Series_title': 'Cotranslational assembly of protein complexes in eukaryotes revealed by ribosome profiling', 'Series_geo_accession': 'GSE116570', 'Series_status': 'Public on Jul 05 2018', 'Series_submission_date': 'Jul 03 2018', 'Series_last_update_date': 'Sep 04 2018', 'Series_pubmed_id': '30158700', 'Series_summary': 'Folding of newly synthesized proteins to the native state is a major challenge within the crowded cellular environment, since non-productive interactions can lead to misfolding, aggregation and degradation1. Cells cope with this challenge by coupling synthesis with polypeptide folding and by employing molecular chaperones to safeguard folding already cotranslationally2. However, little is known about the final step of folding, the assembly of polypeptides into complexes, although most of the cellular proteome forms oligomeric assemblies3. In prokaryotes, a proof-of-concept study showed that assembly of heterodimeric luciferase is an organized cotranslational process, facilitated by spatially confined translation of the subunits encoded on a polycistronic mRNA4. In eukaryotes, however, fundamental differences such as rarity of polycistronic mRNAs and different chaperone constellations raise the question whether assembly is also coordinated with translation. Here we provide a systematic and mechanistic analysis of protein complex assembly in eukaryotes using ribosome profiling. We determined the in vivo nascent subunits interactions of 12 hetero-oligomeric protein complexes of Saccharomyces cerevisiae at near-residue resolution. We find 9 complexes assemble cotranslationally; the 3 complexes that do not show cotranslational interactions are regulated by dedicated assembly chaperones5-7. Cotranslational assembly often occurs uni-directionally, with one fully synthesized subunit engaging its nascent partner subunit(s), thereby counteracting its aggregation propensity. The onset of cotranslational subunit association coincides sharply with full exposure of the nascent interaction domain at the ribosomal tunnel exit. The action of the ribosome-associated Hsp70 chaperone Ssb8 is coordinated with assembly. Ssb transiently engages partially synthesized interaction domains, then dissociates before the onset of partner subunit association, presumably to prevent premature assembly interactions. Our study shows that cotranslational subunit association is a prevalent mechanism for hetero-oligomers assembly in yeast and indicates that translation, folding and assembly of protein complexes are integrated processes in eukaryotes.', 'Series_overall_design': 'Investigating the interaction of 26 individual complex subunits with nascent polypeptide chains in yeast.', 'Series_type': 'Other', 'Series_contributor': 'Bernd,,Bukau', 'Series_sample_id': 'GSM3244291', 'Series_contact_name': 'Ayala,,Shiber', 'Series_contact_email': 'ayalashiber@technion.ac.il', 'Series_contact_institute': 'Technion, Israel institute of technology', 'Series_contact_address': 'Emerson building, Technion City', 'Series_contact_city': 'Haifa', 'Series_contact_zip/postal_code': '32000', 'Series_contact_country': 'Israel', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE116nnn/GSE116570/suppl/GSE116570_RAW.tar', 'Series_platform_id': 'GPL18330', 'Series_platform_taxid': '1247190', 'Series_sample_taxid': '1247190', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP152018'}\n",
      "733 PRJNA507750 GSE123166 {'Series_title': 'Selective ribosome profiling to study interactions of translating ribosomes in yeast', 'Series_geo_accession': 'GSE123166', 'Series_status': 'Public on Jul 10 2019', 'Series_submission_date': 'Nov 30 2018', 'Series_last_update_date': 'Oct 09 2019', 'Series_pubmed_id': '31332354', 'Series_summary': 'A number of enzymes, targeting factors and chaperones engage ribosomes to support fundamental steps of nascent protein maturation, including enzymatic processing, membrane targeting and co-translational folding. The selective ribosome profiling (SeRP) method is a new tool for studying the co-translational activity of maturation factors that provides proteome-wide information on a factor’s nascent interactome, the onset and duration of binding and the mechanisms controlling factor engagement. SeRP is based on the combination of two ribosome-profiling (RP) experiments, sequencing the ribosome-protected mRNA fragments from all ribosomes (total translatome) and the ribosome subpopulation engaged by the factor of interest (factor-bound translatome). We provide a detailed SeRP protocol, exemplified for the yeast Hsp70 chaperone Ssb (stress 70 B), for studying factor interactions with nascent proteins that is readily adaptable to identifying nascent interactomes of other co-translationally acting eukaryotic factors. The protocol provides general guidance for experimental design and optimization, as well as detailed instructions for cell growth and harvest, the isolation of (factor-engaged) monosomes, the generation of a cDNA library and data analysis. Experience in biochemistry and RNA handling, as well as basic programing knowledge, is necessary to perform SeRP. Execution of a SeRP experiment takes 8–10 working days, and initial data analysis can be completed within 1–2 d. This protocol is an extension of the originally developed protocol describing SeRP in bacteria.', 'Series_overall_design': 'Method description. Control mixing experiment to show that no post-lysis binding occurs.', 'Series_type': 'Other', 'Series_contributor': 'Günter,,Kramer', 'Series_sample_id': 'GSM3497875', 'Series_contact_name': 'Kristina,Rebekka,Doering', 'Series_contact_laboratory': 'AG Bukau', 'Series_contact_department': 'University of Heidelberg', 'Series_contact_institute': 'ZMBH', 'Series_contact_address': 'Im Neuenheimer Feld 282', 'Series_contact_city': 'Heidelberg', 'Series_contact_state': 'Baden-Württemberg', 'Series_contact_zip/postal_code': '69120', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE123nnn/GSE123166/suppl/GSE123166_RAW.tar', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP171612'}\n",
      "734 PRJNA769692 GSE185572 {'Series_title': 'Ribosome profiling reveals multiple roles of SecA in cotranslational protein export', 'Series_geo_accession': 'GSE185572', 'Series_status': 'Public on May 16 2022', 'Series_submission_date': 'Oct 08 2021', 'Series_last_update_date': 'Aug 15 2022', 'Series_pubmed_id': '35697696', 'Series_summary': 'SecA, an ATPase known to posttranslationally translocate secretory proteins across the bacterial plasma membrane, also interacts with ribosomes. Here, we used a combination of ribosome profiling methods to investigate the cotranslational actions of SecA in vivo. SecA scans translating ribosomes at the plasma membrane and cotranslationally engages large periplasmic loops on inner membrane proteins. In coordination with the proton motive force, SecA resolves the cytoplasmic accumulation of periplasmic loops during cotranslational protein translocation. SecA also associates with a subset of secretory proteins at an early stage of translation and mediates their cotranslational transport, whereas the chaperone trigger factor (TF) delays SecA engagement on other secretory proteins to impose a posttranslational mode of translocation. The hydrophobicity of signal sequence is a determinant of nascent protein triage between SecA and TF. Our results elucidate the principles of SecA-driven cotranslational protein translocation and reveal a hierarchical network of protein export pathways in bacteria.', 'Series_overall_design': 'Investigating the SecA-nascent chain interactions in four different strain backgrounds (WT, ∆tig, secA∆ZnBD, ∆secB) and in WT strain upon CCCP treatment. Monitoring cotranslational targeting events by fractionation-coupled ribosome profiling in WT strain with and without CCCP treatment (all in replicates).', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Shu-ou,,Shan', 'Series_sample_id': 'GSM5618622', 'Series_contact_name': 'Zikun,,Zhu', 'Series_contact_email': 'zzhu@caltech.edu', 'Series_contact_laboratory': 'Shan', 'Series_contact_department': 'CCE', 'Series_contact_institute': 'California Institute of Technology', 'Series_contact_address': '1200 E California Blvd', 'Series_contact_city': 'Pasadena', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '91125', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE185nnn/GSE185572/suppl/GSE185572_RAW.tar', 'Series_platform_id': 'GPL25368', 'Series_platform_taxid': '562', 'Series_sample_taxid': '562', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP340530'}\n",
      "735 PRJNA915314 GSE221651 {'Series_title': 'Dedicated chaperone at the ribosome safeguards the proteostasis network during eEF1A biogenesis', 'Series_geo_accession': 'GSE221651', 'Series_status': 'Public on Jan 16 2024', 'Series_submission_date': 'Dec 23 2022', 'Series_last_update_date': 'Feb 26 2024', 'Series_pubmed_id': '38360885', 'Series_summary': 'Cotranslational protein folding depends on general chaperones that engage highly diverse nascent chains at the ribosomes. Here we find that the universal cotranslational machinery adapts to accommodate the challenging biogenesis of abundantly expressed eukaryotic translation elongation factor 1A (eEF1A). During eEF1A synthesis, chaperone Chp1 is recruited to the ribosome with the help of the nascent polypeptide-associated complex (NAC), where it safeguards eEF1A biogenesis. Aberrant eEF1A production triggers instant proteolysis, widespread protein aggregation, activation of Hsf1 stress transcription and compromises cellular fitness. The expression of pathogenic eEF1A2 variants linked to epileptic-dyskinetic encephalopathy is protected by Chp1. Thus, eEF1A is a difficult to fold protein that necessitates dedicated folding factor Chp1 at the ribosomal tunnel exit to protect the eukaryotic cell from proteostasis collapse.', 'Series_overall_design': 'Selective Ribosome Profiling of Chp1 in S. cerevisiae (BY4741 Chp1-GFP strain, performed in two replicates, each consisting of an IP sample and a corresponding total translatome sample. As a negative control, IP and total translatome samples were prepared in WT untagged BY4741 cells also in two replicates). Ribosome profiling in S. cerevisiae (BY4741 WT or Chp1Δ cells, total translatomes in two replicates each)', 'Series_type': 'Other', 'Series_contributor': 'Michela,,Ciccarelli', 'Series_sample_id': 'GSM6892120', 'Series_contact_name': 'Claes,,Andreasson', 'Series_contact_email': 'claes.andreasson@su.se', 'Series_contact_laboratory': 'Claes Andreasson', 'Series_contact_department': 'Department of Molecular Biosciences', 'Series_contact_institute': 'The Wenner-Gren Institute, Stockholm University', 'Series_contact_address': 'Svante Arrhenius väg 20C', 'Series_contact_city': 'Stockholm', 'Series_contact_zip/postal_code': '106 91', 'Series_contact_country': 'Sweden', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE221nnn/GSE221651/suppl/GSE221651_RAW.tar', 'Series_platform_id': 'GPL26302', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA915314'}\n",
      "736 PRJNA163111 GSE37744 {'Series_title': 'The ribosome profiling strategy for monitoring translation in vivo by deep sequencing of ribosome-protected mRNA fragments', 'Series_geo_accession': 'GSE37744', 'Series_status': 'Public on Jul 26 2012', 'Series_submission_date': 'May 03 2012', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '22836135', 'Series_summary': 'Recent studies highlight the importance of translational control in determining protein abundance, underscoring the value of measuring gene expression at the level of translation. We present a protocol for genome-wide, quantitative analysis of in vivo translation by deep sequencing. This ribosome profiling approach maps the exact positions of ribosomes on transcripts by nuclease footprinting. The nuclease-protected mRNA fragments are converted into a DNA library suitable for deep sequencing using a strategy that minimizes bias. The abundance of different footprint fragments in deep sequencing data reports on the amount of translation of a gene. Additionally, footprints reveal the exact regions of the transcriptome that are translated. To better define translated reading frames, we describe an adaptation that reveals the sites of translation initiation by pre-treating cells with harringtonine to immobilize initiating ribosomes. The protocol we describe requires 5 - 7 days to generate a completed ribosome profiling sequencing library.', 'Series_overall_design': 'Ribosome profiling in cultured mammalian cells under three different footprinting conditions', 'Series_type': 'Other', 'Series_contributor': 'Jonathan,S,Weissman', 'Series_sample_id': 'GSM926676', 'Series_contact_name': 'Nicholas,T,Ingolia', 'Series_contact_email': 'ingolia@berkeley.edu, nick@ingolia.org', 'Series_contact_phone': '510 664 7071', 'Series_contact_laboratory': 'Ingolia', 'Series_contact_department': 'Molecular and Cell Biology', 'Series_contact_institute': 'University of California, Berkeley', 'Series_contact_address': '1 Barker Hall # 3202', 'Series_contact_city': 'Berkeley', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94720', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE37nnn/GSE37744/suppl/GSE37744_RAW.tar', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP012648'}\n",
      "737 PRJNA171135 GSE39561 {'Series_title': 'Genome-wide search for novel human uORFs and N-terminal protein extensions using ribosomal footprinting', 'Series_geo_accession': 'GSE39561', 'Series_status': 'Public on Aug 08 2012', 'Series_submission_date': 'Jul 23 2012', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '22879431', 'Series_summary': 'So far, the annotation of translation initiation sites (TISs) has been based mostly upon bioinformatics rather than experimental evidence. We adapted ribosomal footprinting to puromycin-treated cells to generate a transcriptome-wide map of TISs in a human monocytic cell line. A neural network was trained on the ribosomal footprints at previously annotated AUG translation initiation codons (TICs), and used for the ab initio prediction of TISs in 5062 transcripts with sufficient sequence coverage. Functional interpretation suggested 2994 novel upstream open reading frames (uORFs) in the 5´ UTR (924 AUG, 2070 near-cognate codons), 1406 uORFs overlapping with the coding sequence (116 AUG, 1290 near-cognate) and 546 N-terminal protein extensions (6 AUG, 540 near-cognate). The TIS detection method was validated on the basis of previously published alternative TISs and uORFs. On average, TICs in newly annotated TISs were significantly more conserved among primates than control codons, both for AUGs (p<10-10) and near-cognate codons (p=3.8×10-3). The derived transcriptome-wide map of novel candidate TISs will help to explain how human proteome diversity is influenced by alternative translation initiation and regulation.', 'Series_overall_design': 'Examination of translational initiation in human cell lines using ribosomal footprinting', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Mario,,Brosch', 'Series_sample_id': 'GSM971693', 'Series_contact_name': 'Mario,,Brosch', 'Series_contact_department': '1. Medical Department', 'Series_contact_institute': 'University Hospital Kiel', 'Series_contact_address': 'Schittenhelmstr. 12', 'Series_contact_city': 'Kiel', 'Series_contact_zip/postal_code': '24105', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE39nnn/GSE39561/suppl/GSE39561_THP-1_puro_plus-strand.bedGraph.gz', 'Series_platform_id': 'GPL9115', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP014542'}\n",
      "738 PRJNA205933 GSE47509 {'Series_title': 'High resolution view of bacteriophage lambda gene expression by ribosome profiling', 'Series_geo_accession': 'GSE47509', 'Series_status': 'Public on Jul 01 2013', 'Series_submission_date': 'May 30 2013', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '23812753', 'Series_summary': 'Bacteriophage lambda is one of the most extensively studied organisms, and has been a primary model for understanding basic modes of genetic regulation.  Here we examine the progress of lambda gene expression during phage development by ribosome profiling, and thereby provide a very high resolution view of lambda gene expression.  The known genes are expressed in a predictable fashion, authenticating the analysis.  But many previously unappreciated potential open reading frames become apparent in the expression analysis, revealing an unexpected complexity in the pattern of lambda gene function.', 'Series_overall_design': 'We chose temperature induction of the classic cI857 repressor mutation in a lysogen of E. coli MG1655 in order to synchronize the lytic process, sampling the lysogen and control non-lysogen both before and 2, 5, 10, and 20 minutes after shifting the temperature from 32° to 42°.  The last sample time was chosen to be before any significant cell lysis, but during the later stages of lytic gene expression. Total protected nucleotides within open reading frames were summed to determine the density of translation of each reading frame.  We take this number to indicate the overall rate of translation, although obviously this assumes that pauses in translation do not excessively affect the overall rate.  Since the expression level is not normalized for the copy number of the replicating phage DNA, it thus encompasses both the effect of DNA template availability on mRNA synthesis and the efficiency of utilization of messengers by ribosomes.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Jeffrey,W,Roberts', 'Series_sample_id': 'GSM1151308', 'Series_contact_name': 'Huifeng,,Jiang', 'Series_contact_email': 'jiang_hf@tib.cas.cn', 'Series_contact_institute': 'Tianjin Institute of Industrial Biotechnology', 'Series_contact_address': '#32XiQiDao Tianjin airport economic park', 'Series_contact_city': 'Tianjin', 'Series_contact_zip/postal_code': '300308', 'Series_contact_country': 'China', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE47nnn/GSE47509/suppl/GSE47509_RAW.tar', 'Series_platform_id': 'GPL15010', 'Series_platform_taxid': '511145', 'Series_sample_taxid': '511145', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP023499'}\n",
      "739 PRJNA251988 GSE58321 {'Series_title': 'Distinct stages of the translation elongation cycle revealed by sequencing ribosome-protected mRNA fragments', 'Series_geo_accession': 'GSE58321', 'Series_status': 'Public on Jun 10 2014', 'Series_submission_date': 'Jun 09 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '24842990', 'Series_summary': 'During translation elongation, the ribosome ratchets along its mRNA template, incorporating each new amino acid and translocating from one codon to the next. The elongation cycle requires dramatic structural rearrangements of the ribosome. We show here that deep sequencing of ribosome-protected mRNA fragments reveals not only the position of each ribosome but also, unexpectedly, its particular stage of the elongation cycle. Sequencing reveals two distinct populations of ribosome footprints, 28-30 nucleotides and 20-22 nucleotides long, representing translating ribosomes in distinct states, differentially stabilized by specific elongation inhibitors. We find that the balance of small and large footprints varies by codon and is correlated with translation speed. The ability to visualize conformational changes in the ribosome during elongation, at single-codon resolution, provides a new way to study the detailed kinetics of translation and a new probe with which to identify the factors that affect each step in the elongation cycle.', 'Series_overall_design': 'Ribosome profiling, or sequencing of ribosome-protected mRNA fragments, in yeast. We assay ribosome footprint sizes and positions in three conditions: untreated yeast (3 replicates) and yeast treated with translation inhibitors cycloheximide (2 replicates) and anisomycin (2 biological replicates, one technical replicate). We also treat yeast with 3-aminotriazole to measure the effect of limited histidine tRNAs on ribosome footprint size and distribution (two treatment durations).', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Patrick,O,Brown', 'Series_sample_id': 'GSM1406462', 'Series_contact_name': 'Liana,F,Lareau', 'Series_contact_email': 'lareau@berkeley.edu', 'Series_contact_department': 'California Institute for Quantitative Biosciences', 'Series_contact_institute': 'University of California, Berkeley', 'Series_contact_address': 'Barker Hall 424', 'Series_contact_city': 'Berkeley', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94720', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE58nnn/GSE58321/suppl/GSE58321_RAW.tar', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP043036'}\n",
      "740 PRJNA255669 GSE59573 {'Series_title': 'Translation Inhibitors Cause Abnormalities in Ribosome Profiling Experiments', 'Series_geo_accession': 'GSE59573', 'Series_status': 'Public on Aug 01 2014', 'Series_submission_date': 'Jul 18 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26261310', 'Series_summary': 'Ribosome profiling and high-throughput sequencing provide unprecedented opportunities for the analysis of mRNA translation. Using this novel method, several studies have demonstrated the widespread role of short upstream reading frames in translational control as well as slower elongation at the beginning of open reading frames in response to stress. Based on the initial studies, the importance of adding or omitting translation inhibitors, such as cycloheximide, was noted as it markedly affected ribosome coverage profiles. For that reason, many recent studies omitted translation inhibitors in the culture medium. Here, we investigate the influence of ranging cycloheximide concentrations on ribosome profiles in Saccharomyces cerevisiae and demonstrate that increasing the drug concentration can overcome some of the artifacts. We subjected cells to various manipulations and show that neither oxidative stress nor heat shock nor amino acid starvation affect translation elongation. Instead, the observations in the initial studies are the result of cycloheximide-inflicted artifacts. Likewise, we find little support for short upstream reading frames to be involved in wide-spread protein synthesis regulation under stress conditions. Our study highlights the need for better standardization of ribosome profiling methods.', 'Series_overall_design': 'Ranging concentrations of cycloheximide and various stress contitions were tested with Ribo-seq', 'Series_type': 'Other', 'Series_contributor': 'Vadim,,Gladyshev', 'Series_sample_id': 'GSM1439607', 'Series_contact_name': 'Maxim,,Gerashchenko', 'Series_contact_email': 'mgerashchenko@bwh.harvard.edu', 'Series_contact_department': 'Medicine', 'Series_contact_institute': \"Brigham and Women's Hospital\", 'Series_contact_address': '77 Louis Pasteur Ave, NRB 435', 'Series_contact_city': 'Boston', 'Series_contact_state': 'Massachusetts', 'Series_contact_zip/postal_code': '02115', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE59nnn/GSE59573/suppl/GSE59573_readme.txt', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP044649'}\n",
      "741 PRJNA260072 GSE61011 {'Series_title': 'Targeting and Plasticity of Mitochondrial Proteins Revealed by Proximity-Specific Ribosome Profiling', 'Series_geo_accession': 'GSE61011', 'Series_status': 'Public on Nov 10 2014', 'Series_submission_date': 'Sep 02 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25378625', 'Series_summary': 'Nearly all mitochondrial proteins are nuclear-encoded and are targeted to their mitochondrial destination from the cytosol. Here, we used proximity-specific ribosome profiling to comprehensively measure translation at the mitochondrial surface in yeast. The majority of inner membrane proteins were co-translationally targeted to mitochondria, reminiscent of proteins entering the endoplasmic reticulum (ER). Comparison between mitochondrial and ER localization demonstrated that the vast majority of proteins were targeted to a specific organelle. A prominent exception was the fumarate reductase Osm1, known to reside in mitochondria. We identified a conserved ER isoform of Osm1, which contributes to the oxidative protein folding capacity of the organelle. This dual localization was enabled by alternative translation initiation sites encoding distinct targeting signals. These findings highlight the exquisite in vivo specificity of organellar targeting mechanisms.', 'Series_overall_design': 'Ribosome profiling of whole cell or streptavidin-purified ribosomes biotinylated by cytosolic, ER or mitochondrially localized biotin ligase in yeast and mammalian cells', 'Series_type': 'Other', 'Series_contributor': 'Jonathan,S,Weissman', 'Series_sample_id': 'GSM1495504', 'Series_contact_name': 'Jonathan,,Weissman', 'Series_contact_institute': 'UCSF', 'Series_contact_address': '1700 4th Street', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE61nnn/GSE61011/suppl/GSE61011_TableS1.txt.gz', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP046015'}\n",
      "742 PRJNA260073 GSE61012 {'Series_title': 'Principles of ER Co-Translational Translocation Revealed by Proximity-Specific Ribosome Profiling', 'Series_geo_accession': 'GSE61012', 'Series_status': 'Public on Nov 10 2014', 'Series_submission_date': 'Sep 02 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '25378625', 'Series_summary': 'Localized protein synthesis is a fundamental mechanism for creating distinct subcellular environments. Here we developed a generalizable proximity-specific ribosome profiling strategy that enables global interrogation of translation in defined subcellular locations. We applied this approach to the endoplasmic reticulum (ER) in yeast and mammals. We observed the vast majority of secretory proteins to be co-translationally translocated, including substrates capable of post-translational insertion in vitro. Distinct translocon complexes engaged nascent chains at different points during synthesis. Whereas most proteins engaged the ER immediately following or even before signal sequence (SS) emergence, a class of Sec66-dependent proteins entered with a looped SS conformation. Finally, we observed rapid ribosome exchange into the cytosol following translation termination. These data provide insights into how distinct translocation mechanisms act in concert to promote efficient co-translational recruitment.', 'Series_overall_design': 'Ribosome profiling of whole cell or streptavidin-purified ribosomes biotinylated by cytosolic, ER or mitochondrially localized biotin ligase in yeast and mammalian cells', 'Series_type': 'Other', 'Series_contributor': 'Jonathan,S,Weissman', 'Series_sample_id': 'GSM1495536', 'Series_contact_name': 'Jonathan,,Weissman', 'Series_contact_institute': 'UCSF', 'Series_contact_address': '1700 4th Street', 'Series_contact_city': 'San Francisco', 'Series_contact_state': 'CA', 'Series_contact_zip/postal_code': '94158', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE61nnn/GSE61012/suppl/GSE61012_TableS6.txt.gz', 'Series_platform_id': 'GPL13821', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP046016'}\n",
      "743 PRJNA269272 GSE63858 {'Series_title': 'Ribosome profiling of E. coli K-12 MG1655 MOPS rich media with 0.2% glucose', 'Series_geo_accession': 'GSE63858', 'Series_status': 'Public on Jun 09 2015', 'Series_submission_date': 'Dec 04 2014', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26054770', 'Series_summary': 'Ribosome profiling was performed on E. coli wild type cells. All replicates were grown to an OD of ~0.4 in MOPS rich Media with 0.2% glucose supplemented, aerobically in shake flasks. Cultures were treated with chloramphenicol 2 min prior to harvest.', 'Series_overall_design': 'Two biological replicates', 'Series_type': 'Other', 'Series_contributor': 'Berhnard,O,Palsson', 'Series_sample_id': 'GSM1558531', 'Series_contact_name': 'Haythem,,Latif', 'Series_contact_laboratory': 'Latif', 'Series_contact_institute': 'University of California San Diego', 'Series_contact_address': '417 Powell-Focht Bioengineering Hall, 9500 Gilman Drive', 'Series_contact_city': 'La Jolla', 'Series_contact_state': 'California', 'Series_contact_zip/postal_code': '92093-0412', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE63nnn/GSE63858/suppl/GSE63858_RAW.tar', 'Series_platform_id': 'GPL17439', 'Series_platform_taxid': '511145', 'Series_sample_taxid': '511145', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP050548'}\n",
      "744 PRJNA275470 GSE65948 {'Series_title': 'Transcriptome-wide measurement of ribosomal occupancy by ribosome profiling', 'Series_geo_accession': 'GSE65948', 'Series_status': 'Public on Feb 12 2018', 'Series_submission_date': 'Feb 14 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26102273', 'Series_summary': 'Experiments for optimization of the ribosome profiling protocol using C. elegans lysates. We report that optimizing the cutting accuracy to a narrow window of 28-30 nucleotides when PAGE-purifying the ribosome-protected fragments (RPFs) significantly improves the quality of the RPF library. In addition, we find that purifying monosomes by sucrose gradient fractionation clearly removed more contaminating ribosomal RNA from the samples compared to the purification by gel filtration columns.', 'Series_overall_design': 'Experiment 1 (samples 1-4): Comparison between different methods to isolate monosomes and between different library preparations. Experiment 2 (samples 5-8): Optimization of the cutting accuracy to PAGE purify ribosome-protected fragments.', 'Series_type': 'Other', 'Series_contributor': 'Helge,,Grosshans', 'Series_sample_id': 'GSM1611598', 'Series_contact_name': 'Jieyi,,Xiong', 'Series_contact_institute': 'VIB-KULeuven Center for Cancer Biology', 'Series_contact_address': 'Herestraat 49, box 912', 'Series_contact_city': 'Leuven', 'Series_contact_zip/postal_code': '3000', 'Series_contact_country': 'Belgium', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE65nnn/GSE65948/suppl/GSE65948_RPF_CDS_RPKM.tsv.gz', 'Series_platform_id': 'GPL13657', 'Series_platform_taxid': '6239', 'Series_sample_taxid': '6239', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP055039'}\n",
      "745 PRJNA278007 GSE66808 {'Series_title': 'IFN-g Regulates mTORC1, Cellular Metabolism and mRNA Translation to Potentiate Inflammatory Macrophage Activation [miRNA-Seq]', 'Series_geo_accession': 'GSE66808', 'Series_status': 'Public on Jun 29 2015', 'Series_submission_date': 'Mar 12 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26147685', 'Series_summary': 'IFN-g primes macrophages for enhanced inflammatory activation by TLRs and microbial killing, but little is known about the regulation of cell metabolism or mRNA translation during priming. We found that IFN-g regulates macrophage metabolism and translation in an integrated manner by targeting mTORC1 and MNK pathways that converge on the selective regulator of translation initiation eIF4E. Physiological downregulation of the central metabolic regulator mTORC1 by IFN-g was associated with autophagy and translational suppression of repressors of inflammation such as HES1. Genome-wide ribosome profiling in TLR2-stimulated macrophages revealed that IFN-g selectively modulates the macrophage translatome to promote inflammation, further reprogram metabolic pathways, and modulate protein synthesis. These results add IFN-g-mediated metabolic reprogramming and translational regulation as key components of classical inflammatory macrophage activation.', 'Series_overall_design': 'microRNA-seq libraries were generated from mock or IFN-g-primed human macrophages. Cells were stimulated with or without Pam3Cys and harvested at 4 hours Libraries were generated using Illumina Truseq small RNA technology.', 'Series_type': 'Non-coding RNA profiling by high throughput sequencing', 'Series_contributor': 'Lionel,B,Ivashkiv', 'Series_sample_id': 'GSM1632593', 'Series_contact_name': 'Xiaodi,,Su', 'Series_contact_email': 'xis2003@med.cornell.edu', 'Series_contact_laboratory': 'Ivashkiv Lab', 'Series_contact_department': 'Arthritis and Tissue Degeneration Program', 'Series_contact_institute': 'Hospital for Special Surgery', 'Series_contact_address': '535 East 70th Street', 'Series_contact_city': 'New York', 'Series_contact_state': 'NY', 'Series_contact_zip/postal_code': '10021', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE66nnn/GSE66808/suppl/GSE66808_IFNg_vs_Mock_miRNA_cpm.xlsx.gz', 'Series_platform_id': 'GPL11154', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP056097'}\n",
      "746 PRJNA285458 GSE69414 {'Series_title': 'Ribosome profiling study of rli1 depeletion strain', 'Series_geo_accession': 'GSE69414', 'Series_status': 'Public on Jun 02 2015', 'Series_submission_date': 'Jun 01 2015', 'Series_last_update_date': 'May 15 2019', 'Series_pubmed_id': '26276635', 'Series_summary': \"ABCE1/Rli1 functions as a ribosome recycling factor in vitro, but has not been shown to be crucial for recycling in vivo. We use ribosome profiling and biochemistry to define the role of Rli1 in living yeast. When Rli1 levels were diminished, 80S ribosomes accumulated at stop codons and, surprisingly, in the adjoining 3'UTRs of most genes. While ribosomes did not show preference for any reading frame in the 3'UTR, their enrichment at stop codons and His codons after histidine starvation is consistent with aberrant 3'UTR translation. Predicted 3'UTR translation products were detected by Western analysis and mass spectrometry, and their small sizes indicate a reinitiation mechanism. Eliminating ribosome-rescue factor Dom34 dramatically increases 3'UTR ribosome occupancy in Rli1 depleted cells, indicating that Dom34 clears the bulk of unrecycled ribosomes. Thus, Rli1 is crucial for ribosome recycling in vivo and for overall gene expression as it controls ribosome homeostasis and 3'UTR translation.\", 'Series_overall_design': '10 biological samples are included in the study (2 mRNA-Seq samples and 8 ribosome footprinting samples). These include wild-type, rli1-depletion, rli1-depletion/dom34Δ, and RLI1 high-copy / dom34Δ strains. Samples are also included where cells were treated with 3-AT to starve for histidine.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Rachel,,Green', 'Series_sample_id': 'GSM1700894', 'Series_contact_name': 'Nicholas,R,Guydosh', 'Series_contact_email': 'nicholas.guydosh@nih.gov', 'Series_contact_laboratory': 'LBG', 'Series_contact_department': 'NIDDK', 'Series_contact_institute': 'National Institutes of Health', 'Series_contact_address': '8 Center Dr, Bldg 8, 220', 'Series_contact_city': 'Bethesda', 'Series_contact_state': 'MD', 'Series_contact_zip/postal_code': '20892', 'Series_contact_country': 'USA', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE69nnn/GSE69414/suppl/GSE69414_RAW.tar', 'Series_platform_id': 'GPL17342', 'Series_platform_taxid': '4932', 'Series_sample_taxid': '4932', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP058910'}\n",
      "747 PRJNA769267 GSE185489 {'Series_title': 'The dynamic rRNA ribomethylome drives stemness in acute myeloid leukemia [ribosome profiling]', 'Series_geo_accession': 'GSE185489', 'Series_status': 'Public on Nov 23 2022', 'Series_submission_date': 'Oct 07 2021', 'Series_last_update_date': 'Nov 24 2022', 'Series_pubmed_id': '36259929', 'Series_summary': 'We performed ribosome profiling to determine  the effect of FBL knockdown on mRNA translation in human leukemai cell Kasumi-1.', 'Series_overall_design': 'Kasumi-1 cells stably expressing doxcyxycline inducible control or FBL specific shRNA were used. Cells were treated with 100 ng/ml doxcyxycline for three days to induce the expression of shRNA. Cells were then collected for Ribosome profiling.', 'Series_type': 'Expression profiling by high throughput sequencing', 'Series_contributor': 'Carsten,,Müller-Tidow', 'Series_sample_id': 'GSM5616115', 'Series_contact_name': 'Christian,,Rohde', 'Series_contact_email': 'christian.rohde@uni-heidelberg.de', 'Series_contact_laboratory': 'Molecular Hematology and Oncology', 'Series_contact_institute': 'Heidelberg University', 'Series_contact_address': 'Im Neuenheimer Feld 410', 'Series_contact_city': 'Heidelberg', 'Series_contact_zip/postal_code': '69120', 'Series_contact_country': 'Germany', 'Series_supplementary_file': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE185nnn/GSE185489/suppl/GSE185489_cds_coverage.txt.gz', 'Series_platform_id': 'GPL18573', 'Series_platform_taxid': '9606', 'Series_sample_taxid': '9606', 'Series_relation': 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP340351'}\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "(748, 31)"
      ]
     },
     "execution_count": 83,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "geo_series_metadata = pd.DataFrame()\n",
    "\n",
    "for i, row in bioproject_gses.iterrows():\n",
    "    bioproject_id = row['BioProject']\n",
    "    gse_id = row['GEO']\n",
    "    \n",
    "    # Retrieve GEO Series metadata\n",
    "    series_metadata = download_and_parse_geo_soft(gse_id)\n",
    "    print(i, bioproject_id, gse_id, series_metadata)\n",
    "    \n",
    "    if series_metadata:\n",
    "        # Combine metadata into a single dictionary\n",
    "        combined_metadata = {**series_metadata}\n",
    "        \n",
    "        # Convert combined_metadata to DataFrame and concatenate\n",
    "        geo_series_metadata = pd.concat([geo_series_metadata, pd.DataFrame([combined_metadata])], ignore_index=True)\n",
    "\n",
    "geo_series_metadata.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of non-empty Series_pubmed_id entries: 658\n"
     ]
    }
   ],
   "source": [
    "non_empty_pubmed_ids_count = geo_series_metadata['Series_pubmed_id'].dropna().shape[0]\n",
    "print(f\"Number of non-empty Series_pubmed_id entries: {non_empty_pubmed_ids_count}\")"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
